<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8" /><script type="text/javascript">window.NREUM||(NREUM={}),__nr_require=function(e,n,t){function r(t){if(!n[t]){var o=n[t]={exports:{}};e[t][0].call(o.exports,function(n){var o=e[t][1][n];return r(o||n)},o,o.exports)}return n[t].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<t.length;o++)r(t[o]);return r}({1:[function(e,n,t){function r(){}function o(e,n,t){return function(){return i(e,[c.now()].concat(u(arguments)),n?null:this,t),n?void 0:this}}var i=e("handle"),a=e(2),u=e(3),f=e("ee").get("tracer"),c=e("loader"),s=NREUM;"undefined"==typeof window.newrelic&&(newrelic=s);var p=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],d="api-",l=d+"ixn-";a(p,function(e,n){s[n]=o(d+n,!0,"api")}),s.addPageAction=o(d+"addPageAction",!0),s.setCurrentRouteName=o(d+"routeName",!0),n.exports=newrelic,s.interaction=function(){return(new r).get()};var m=r.prototype={createTracer:function(e,n){var t={},r=this,o="function"==typeof n;return i(l+"tracer",[c.now(),e,t],r),function(){if(f.emit((o?"":"no-")+"fn-start",[c.now(),r,o],t),o)try{return n.apply(this,arguments)}finally{f.emit("fn-end",[c.now()],t)}}}};a("setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(e,n){m[n]=o(l+n)}),newrelic.noticeError=function(e){"string"==typeof e&&(e=new Error(e)),i("err",[e,c.now()])}},{}],2:[function(e,n,t){function r(e,n){var t=[],r="",i=0;for(r in e)o.call(e,r)&&(t[i]=n(r,e[r]),i+=1);return t}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],3:[function(e,n,t){function r(e,n,t){n||(n=0),"undefined"==typeof t&&(t=e?e.length:0);for(var r=-1,o=t-n||0,i=Array(o<0?0:o);++r<o;)i[r]=e[n+r];return i}n.exports=r},{}],4:[function(e,n,t){n.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(e,n,t){function r(){}function o(e){function n(e){return e&&e instanceof r?e:e?f(e,u,i):i()}function t(t,r,o,i){if(!d.aborted||i){e&&e(t,r,o);for(var a=n(o),u=m(t),f=u.length,c=0;c<f;c++)u[c].apply(a,r);var p=s[y[t]];return p&&p.push([b,t,r,a]),a}}function l(e,n){v[e]=m(e).concat(n)}function m(e){return v[e]||[]}function w(e){return p[e]=p[e]||o(t)}function g(e,n){c(e,function(e,t){n=n||"feature",y[t]=n,n in s||(s[n]=[])})}var v={},y={},b={on:l,emit:t,get:w,listeners:m,context:n,buffer:g,abort:a,aborted:!1};return b}function i(){return new r}function a(){(s.api||s.feature)&&(d.aborted=!0,s=d.backlog={})}var u="nr@context",f=e("gos"),c=e(2),s={},p={},d=n.exports=o();d.backlog=s},{}],gos:[function(e,n,t){function r(e,n,t){if(o.call(e,n))return e[n];var r=t();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(e,n,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return e[n]=r,r}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],handle:[function(e,n,t){function r(e,n,t,r){o.buffer([e],r),o.emit(e,n,t)}var o=e("ee").get("handle");n.exports=r,r.ee=o},{}],id:[function(e,n,t){function r(e){var n=typeof e;return!e||"object"!==n&&"function"!==n?-1:e===window?0:a(e,i,function(){return o++})}var o=1,i="nr@id",a=e("gos");n.exports=r},{}],loader:[function(e,n,t){function r(){if(!x++){var e=h.info=NREUM.info,n=d.getElementsByTagName("script")[0];if(setTimeout(s.abort,3e4),!(e&&e.licenseKey&&e.applicationID&&n))return s.abort();c(y,function(n,t){e[n]||(e[n]=t)}),f("mark",["onload",a()+h.offset],null,"api");var t=d.createElement("script");t.src="https://"+e.agent,n.parentNode.insertBefore(t,n)}}function o(){"complete"===d.readyState&&i()}function i(){f("mark",["domContent",a()+h.offset],null,"api")}function a(){return E.exists&&performance.now?Math.round(performance.now()):(u=Math.max((new Date).getTime(),u))-h.offset}var u=(new Date).getTime(),f=e("handle"),c=e(2),s=e("ee"),p=window,d=p.document,l="addEventListener",m="attachEvent",w=p.XMLHttpRequest,g=w&&w.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:w,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var v=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-1039.min.js"},b=w&&g&&g[l]&&!/CriOS/.test(navigator.userAgent),h=n.exports={offset:u,now:a,origin:v,features:{},xhrWrappable:b};e(1),d[l]?(d[l]("DOMContentLoaded",i,!1),p[l]("load",r,!1)):(d[m]("onreadystatechange",o),p[m]("onload",r)),f("mark",["firstbyte",u],null,"api");var x=0,E=e(4)},{}]},{},["loader"]);</script><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info={"beacon":"bam.nr-data.net","queueTime":0,"licenseKey":"cd1101aa94","agent":"","transactionName":"bgcDMkBZW0NZBkdcWVdNJxNcW0FZVwscV1dKB08CV1taQlkRXEdFAz0WFFNIRVVcOkVcU049BxNcWw==","applicationID":"6369297","errorBeacon":"bam.nr-data.net","applicationTime":460}</script>
    <title>Proposal: What is the function of FoxP in operant self-learning?</title>
	  <link rel="shortcut icon" href="https://think-lab.s3.amazonaws.com/s/img/favicon.ico" type="image/x-icon" />
    <meta property="og:image" content="https://think-lab.s3.amazonaws.com/s/img" />
    
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
  <link rel="stylesheet" href="https://code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css" />

<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/highlightjs/github.css" />
<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/css/css.css" />
<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/css/bootstrap-multiselect.css" />
<!-- <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'> -->

    
    
    
    
  
    <meta name="citation_doi" content="10.15363/thinklab.a8">
    <meta name="citation_title" content="What is the function of FoxP in operant self-learning? [proposal]">
    <meta name="citation_online_date" content="Nov. 18, 2015">
    <meta name="citation_author" content="Björn Brembs">
    <meta name="citation_journal_title" content="Thinklab">
    <meta name="citation_publisher" content="Thinklab">
  

    


	<!-- GoogleAnalytics -->
	<script>
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
	(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
	m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

	ga('create', 'UA-48550230-1', 'thinklab.com');
	ga('require', 'displayfeatures');
	
	ga('send', 'pageview');

	</script>






</head>
<body >
  <form id="form_action" method="post">
    <input type='hidden' name='csrfmiddlewaretoken' value='pQNiZQX4gF2vyFeXrnbUl9EmuTWA5MG1' />
</form>

	

	<div class="body-wrapper">

		


<!-- <div style="height:5px;background-image:url('https://think-lab.s3.amazonaws.com/s/img/top-bg.gif');"></div> -->
<div class="gradient">
	<div class="main-centered-div header-container">

<table class="full-width" style="height:60px;"><tr>
	<td style="width:200px;vertical-align:middle;padding-bottom:15px;">
		<a href="/" style="font-weight:normal;color:#212121;"><img src="https://think-lab.s3.amazonaws.com/s/img/logo.png" width="160"></a>
	</td><td style="vertical-align:middle;text-align:left;padding-top:6px;width:100%;padding-left:40px;">
		<ul id="not-search-block"  class="main-header">


			


				<li class="dropdown">
					<a class="dropdown-toggle" data-toggle="dropdown" href="#">Browse <span class="caret"></span></a>
					<ul class="dropdown-menu" role="menu">
						<li><a href="/proposals">Proposals</a></li>
						<li><a href="/projects">Projects</a></li>
						<li><a href="/publications">Article discussion</a></li>
						<li><a href="/discussion">All discussion</a></li>
					</ul>
				</li>
				<li><a href="/leaderboard">Leaderboard</a></li>

			
				<li><a href="/about">About</a></li>
			


		</ul>

    <ul id="search-block" class="main-header" style="display: none; margin-right: 30px;">
        <li class="full-width">
            <form method="get" action="/search">
			<div class="input-group full-width">
				<input name="q" type="text" class="form-control search-input">
			</div>
            </form>
        </li>
    </ul>

	</td><td style="vertical-align:middle;text-align:right;padding-top:6px; min-width: 1%; white-space: nowrap;">
		<ul class="main-header pull-right" style="padding-left: 0;">
    		
                <li><a href="/login" style="color:silver;">Login</a></li>
                <li style="margin-right:0"><a class="btn btn-default btn-sm btn-upper" href="/signup">Join</a></li>
            

        </ul>
    </ul>
	</td>
</tr></table>

<!-- <div id="global-search-div" class="main-centered-div" style="display:none;">
	<input type="text" value="" class="thread-search form-control" placeholder="Search" data-action="/discussion">
</div> -->

	</div>
</div>

		
	  



	  


		

		<div class="dark-head"><div class="main-centered-div">

			

	


	<div class="header-object-type"></div>

	
			<a class="btn btn-lg follow-link btn-follow pull-right active tt-top" style="margin-left:20px;" title="Follow project discussion" data-id="11" data-ctype="project" data-is-following="false">&nbsp;</a>
	

	<h2 class="header-object-name">What is the function of FoxP in operant self-learning?</h2>

	
	<div class="project-team">
		<a href='/u/brembs'>Björn Brembs</a>
		 <!-- &nbsp; &nbsp; doi:<a href="https://doi.org/10.15363/thinklab.11">10.15363/thinklab.11</a> -->
	</div>
	

		<!-- <div class="project-team">
			<b>DOI:</b> <a href="https://doi.org/10.15363/thinklab.11">10.15363/thinklab.11</a>
		</div> -->

		

	<div class="project-team">
		
		<div class="proposal-status proposal-status-lg proposal-status-open-for-review">Open for Review</div>
		
		
		
	</div>




	


	<!-- <div class="project-team">
		
	</div> -->

    <ul class="nav nav-tabs project-nav" style="margin-top:25px;">

		

		

    

		
	    
	        <li class="active"><a href="/p/dFoxPoperant/proposal">Proposal</a></li>
	    
	    
		

		<li><a href="/p/dFoxPoperant/discussion"><span class="nav-fig">3</span>Discussions</a></li>

		
			<li><a href="/p/dFoxPoperant/leaderboard"><span class="nav-fig">3</span>Reviewers</a></li>
		

		

		


        <li class="dropdown">
            <a class="dropdown-toggle" data-toggle="dropdown"><span style="opacity:0.5">More <span class="caret"></span></span></a>
            <ul class="dropdown-menu" style="font-size:13px;background-color:#819ec8">

				

				

            <li><a href="/p/dFoxPoperant/followers"><span class="nav-fig">2</span>Followers</a></li>
            </ul>
        </li>

			

    </ul>



		</div></div>
		
		<div class="main-centered-div" style="margin-top:40px;padding-bottom:340px">
	    

	
	<div class="alert alert-info"><div class="main-centered-div">This is a revision from Dec. 3, 2015, 9:18 a.m. <b><a href="/p/dFoxPoperant/proposal">Current version</a></b></div></div>
	

  

  
    
        
      
  

<!--  -->


  





    <table class="project-table"><tr>
      <td class="main-table-content">


        

        
			     <h1 class="page-title">Proposal</h1>
        

			  

<div class="document-body">

  <div id="document_body"><h1 id="project-description">Project Description</h1>

<h1 id="1-state-of-the-art-and-preliminary-work">1 State of the art and preliminary work</h1>

<p>The Forkhead Box P2 (FOXP2) transcription factor is the first gene discovered to be involved in the development of speech and language <span class="citation">[<a href="/doi/10.1038/35097076" class="citation" data-key="10.1038/35097076">1</a>, <a href="/doi/10.1038/nature01025" class="citation" data-key="10.1038/nature01025">2</a>]</span>. Recent studies in birds, mice and the fruit fly <em>Drosophila</em> corroborate the early hypothesis that the gene exerts this involvement via its conserved function in a particular form of operant conditioning: motor learning (or operant self-learning) <span class="citation">[<a href="/doi/10.1371/journal.pbio.0050321" class="citation" data-key="10.1371/journal.pbio.0050321">3</a>, <a href="/doi/10.1371/journal.pone.0100648" class="citation" data-key="10.1371/journal.pone.0100648">4</a>, <a href="/doi/10.1073/pnas.1414542111" class="citation" data-key="10.1073/pnas.1414542111">5</a>]</span>. This conserved function allows for a mechanistic investigation using invertebrate genetic model organisms such as <em>Drosophila</em>. Studies of these forms of learning in other model organisms such as transgenic mice, songbirds and the marine snail <em>Aplysia</em> suffer from experimental difficulties (few genetic tools in birds and snails, difficult physiology and complex brains in vertebrates), which can be complemented by the fly model system. The <em>Drosophila</em> FoxP orthologue, dFoxP, expresses three isoforms, isoA, isoB and an intron-retention isoform (isoIR; <a href="#dFoxPlocus">Figure 1</a>). The data suggest that isoB, in particular, may be specifically important for operant self-learning <span class="citation">[<a href="/doi/10.1371/journal.pone.0100648" class="citation" data-key="10.1371/journal.pone.0100648">4</a>]</span>. Therefore, we will focus our efforts on this isoform, but will include the other two isoforms, wherever possible.</p>

<a name="dFoxPlocus"></a><div class="figure" figure-id="dFoxPlocus"><div class="figure-content"><img src="http://think-lab.s3.amazonaws.com/m/figures/11.png"></div>
            <div class="signature-1">
                <div class="figure-title">Figure 1. The dFoxP locus and its isoforms</div>
                <div class="figure-description"><p>Above: Genomic structure of the dFoxP gene. Below: Transcribed isoforms. FH: Forkhead-Box Domain. I - Intron</p></div>
            </div>
        </div>

<p>Our laboratory has been investigating the involvement of the FoxP gene in <em>Drosophila</em> learning since about 2007. In these years we have been using available mutant and transgenic fly lines for our research and started developing our own antibodies (see poster publication at 1.1.2). We showed that FoxP mutants are impaired in operant self-learning but not operant world-learning <span class="citation">[<a href="/doi/10.1371/journal.pone.0100648" class="citation" data-key="10.1371/journal.pone.0100648">4</a>]</span>, providing a second component of the molecular process underlying operant self-learning (the first one being Protein Kinase C <span class="citation">[<a href="/doi/10.1016/j.cub.2008.07.041" class="citation" data-key="10.1016/j.cub.2008.07.041">6</a>]</span>). </p>

<p>For further mechanistic understanding of the processes modifying the neurons involved in operant self-learning, one needs to a) identify the neurons, b) characterize the role of FoxP/PKC in these neurons and c) understand the role activity in these neurons plays during learning. For PKC we have recently published work towards identifying the neurons where PKC is required (Colomb and Brembs, subm.). This work points to motorneurons as one cite of PKC-dependent plasticity. Given the data from other forms of self-learning, such as operant conditioning of the H-reflex (see, e.g., <span class="citation">[<a href="/doi/10.3389/fnint.2014.00025" class="citation" data-key="10.3389/fnint.2014.00025">7</a>, <a href="/doi/10.1146/annurev.neuro.24.1.807" class="citation" data-key="10.1146/annurev.neuro.24.1.807">8</a>, <a href="/doi/10.1016/S0166-2236%2897%2901133-8" class="citation" data-key="10.1016/S0166-2236(97)01133-8">9</a>]</span>), one would expect multiple sites of plasticity in operant self learning. In particular, work in vertebrate self-learning paradigms have implicated both cerebellar and basal ganglia structures. In fact, FOXP2 work in birds, mice and humans has focused almost exclusively on the basal ganglia. Hence, in order to elucidate FoxP function in operant self-learning, we need to find out if FoxP activity is required in the brain or in the ventral nerve chord of <em>Drosophila</em> for operant self-learning. However, as will be detailed below, we neither know the expression pattern of FoxP, nor do we have sufficient evidence that the available transgenic lines designed to specifically alter FoxP expression, affect FoxP.</p>

<p>Even in the light of recent public discussions (see, e.g., <span class="citation">[<a href="/doi/10.1038/527545a" class="citation" data-key="10.1038/527545a">10</a>, <a href="/doi/10.1038/518027a" class="citation" data-key="10.1038/518027a">11</a>, <a href="/doi/10.1038/521274a" class="citation" data-key="10.1038/521274a">12</a>, <a href="/doi/10.12688/f1000research.6894.1" class="citation" data-key="10.12688/f1000research.6894.1">13</a>]</span>), monoclonal antibodies remain the gold standard for reliably detecting a protein's expression pattern. However, such antibodies do not exist for dFoxP. It is a technical hurdle that the currently available FoxP mutant lines are insertion lines, which only affect the expression level of some of the isoforms. This makes it difficult, for instance, to evaluate the specificity of antibodies against FoxP. Despite the efforts of several laboratories around the world, including ours, there are now only polyclonal antibodies for dFoxP available (one published, see below, several unpublished), none of which satisfy basic specificity criteria. For example, in a collaboration with the laboratory of Prof. Dr. Schade at the Charit&#233;, Berlin, with decades of experience in generating highly specific polyclonal chicken sera (see publication at 1.1.2), we used four different peptides from the <em>Drosophila</em> FoxP gene and immunized chicken. Even after 10 months of immunization and subsequent affinity purification, the serum still detects several bands in a Western blot and none at the expected size (<a href="#ImmunizationStrategy">Figure 2</a>). Subsequent efforts in the laboratory of Constance Scharff were successful in generating antibodies against honeybee FoxP, but a parallel and analogous attempt failed for <em>Drosophila</em> (pers. comm.). Other laboratories working on FoxP also struggle with antibody-generation (e.g., Annette Schenk, Gero Miesenb&#246;ck, pers. comm.) and the only published report of wholemount immunofluorescence in the adult brain shows little overlap with the authors&#8217; own FoxP-GAL4 line and lacks documentation of standard validation procedures <span class="citation">[<a href="/doi/10.1016/j.bbr.2014.04.009" class="citation" data-key="10.1016/j.bbr.2014.04.009">14</a>]</span>, rendering the reliability of the expression reported from this antibody sub-optimal. Hence, in order to reliably detect the FoxP protein in wild type, mutant and transgenic <em>Drosophila</em>, monoclonal antibodies (ideally against all three isoforms) are required.</p>

<a name="ImmunizationStrategy"></a><div class="figure" figure-id="ImmunizationStrategy"><div class="figure-content"><img src="http://think-lab.s3.amazonaws.com/m/figures/12.png"></div>
            <div class="signature-1">
                <div class="figure-title">Figure 2. Immunization strategy and failure to generate specific IgY antibodies against FoxP protein</div>
                <div class="figure-description"><p>A - Location and sequence of the peptides used for chicken immunization. Immunogenic peptides were designed before the intron retention isoform was discovered.<br>B - Western blot of HEK cells either expressing a dFoxP isoform, tagged with a Flag tag (A, B), or no transgene (-). The anti-Flag antibody detects the isoforms at roughly the expected size, while the two affinity- purified IgYs (108, 109) do not detect any protein at the expected size (ellipsoid).</p></div>
            </div>
        </div>

<p>On the genomic level, there are efforts to generate both a complete dFoxP null-allele and isoform-specific alleles in other laboratories which are on track to be available well before the funding period of this proposal <span class="citation">[<a href="" class="citation" data-key="CastellsNobau2013">15</a>]</span>. Nevertheless, should these lines not be available, such lines are required for antibody characterization and we will generate them using CRSIPR/Cas9, an established technique in the collaborating Schneuwly laboratory.</p>

<p>In order to elucidate the role of FoxP in operant self-learning, it is necessary to manipulate FoxP action in a spatiotemporally controlled manner. However, the available transgenic lines are not reliable enough to allow for such manipulations. In our own publication, we show that the RNAi line we use, despite showing a clear behavioral phenotype that mimics the phenotype of the mutant line, does not show any evidence of a knock-down of FoxP on the mRNA level <span class="citation">[<a href="/doi/10.1371/journal.pone.0100648" class="citation" data-key="10.1371/journal.pone.0100648">4</a>]</span>. Hence, we tested all available RNAi lines for their knock-down using qPCR and found that none of the available lines appears to reduce the mRNA levels of any of the FoxP isoforms (<a href="#RNAiLine">Figure 3</a>). While still preliminary, these results replicate results from the laboratory of Annette Schenck (pers. comm) who used the pan-neuronal elav-GAL4 driver line. Two laboratories using different driver lines obtaining largely concordant results inspire confidence even if the results may be preliminary.</p>

<a name="RNAiLine"></a><div class="figure" figure-id="RNAiLine"><div class="figure-content"><img src="http://think-lab.s3.amazonaws.com/m/figures/13.png"></div>
            <div class="signature-1">
                <div class="figure-title">Figure 3. None of the available FoxP RNAi line significantly knocks down any known FoxP isoform</div>
                <div class="figure-description"><p>Preliminary results from quantitative real-time reverse transcriptase PCR on head homogenates from fly lines where the pan-neuronal nSyb-GAL4 line was driving the various available RNAi constructs. Note that the dicer;;nSybGal4 line already exhibits very low levels of the IR isoform, questioning the impression that the low levels of IR in 15735 may be due to knock-down. Numbers above columns - number of biological replicates.</p></div>
            </div>
        </div>

<p>As detailed in our paper, the issue of template mismatches predisposing the targeted mRNA to sequestration, rather than degradation is a long-known issue. Because of these mismatches, without highly specific antibodies it is impossible to determine if the available lines mimic the FoxP mutant phenotype because of off-target effects or because of their effect on FoxP translation. Moreover, at least in vertebrates, FoxP2 appears to be an exceedingly stable protein, such that genomic manipulation of the locus needs weeks to have an effect on the protein level <span class="citation">[<a href="" class="citation" data-key="French2015">16</a>]</span>. Thus, a reliable antibody is required to assess the effectiveness of any manipulation targeting FoxP expression. Even with a specific antibody available, it would be desirable to have at least one RNAi line that reliably knocks down FoxP mRNA for experiments where immunotechniques are not feasible and as a positive control when testing the already available lines for knock down on the protein level.</p>

<p>Traditionally, GAL4 lines derived from genomic promoter regions have not only served to mark and anatomically characterize important cells but also to manipulate these cells in order to study their function. The three available FoxP-GAL4 driver lines have been generated using, approx., 1, 1.5 and 2kb fragments of the dFoxP promotor region, respectively. All three lines differ widely in their expression pattern: The first published record of a FoxP-GAL4 expression pattern is from Lawton et al. <span class="citation">[<a href="/doi/10.1016/j.bbr.2014.04.009" class="citation" data-key="10.1016/j.bbr.2014.04.009">14</a>]</span>. The authors describe their expression pattern with &#8220;Particularly strong expression was evident in the protocerebral bridge&#8221;, while the second published report on such a fly line described their expression pattern thusly: &#8220;FoxP-GAL4&#8211;driven transgene expression was confined to two subsets of Kenyon cells (KCs), the principal intrinsic neurons of the mushroom bodies&#8221; <span class="citation">[<a href="/doi/10.1126/science.1252114" class="citation" data-key="10.1126/science.1252114">17</a>]</span>. The laboratory of Constance Scharff, with whom we collaborate, generated their own FoxP-GAL4 line with yet a third expression pattern (<a href="#ExpressionPattern">Figure 4</a>). The only way to determine which, if any, of the three FoxP-GAL4 lines mimics most closely the actual FoxP expression pattern, is to use highly specific, validated and thoroughly characterized antibodies for comparison and double-labelling. However, such experiments may also reveal that none of the expression patterns is accurate, as the existence of expression-altering P-Element insertions downstream of the FoxP gene suggest regulatory regions outside of the putative promotor region. These considerations are supported by FlyBase data indicating hotspots for jumu, sens, twi, Med, Ubx, D and dl downstream of the last FoxP exon (which one intuitively would have attributed to the downstream gene, hyd, if the insertions had had an effect on hyd expression, which they do not <span class="citation">[<a href="/doi/10.1371/journal.pone.0100648" class="citation" data-key="10.1371/journal.pone.0100648">4</a>]</span>). Moreover, there are multiple binding sites for known transcription factors (dl, twi, sens, inv), and different chromatin domains scattered throughout the FoxP gene region (FlyBase). Finally, in humans, an enhancer was recently discovered in the region downstream of the FOXP2 coding region <span class="citation">[<a href="/doi/10.1186/s13039-015-0173-0" class="citation" data-key="10.1186/s13039-015-0173-0">18</a>]</span>. All of these consideration make it appear highly likely that a substantial amount of FoxP gene regulation is carried by regulatory elements within and downstream of the coding region of the gene, in which case the promotor-fragment method cannot yield reliable GAL4 lines. Specific and sensitive antibodies are required to test any transgenic lines as to their faithful reproduction of the wildtype dFoxP expression pattern. </p>

<a name="ExpressionPattern"></a><div class="figure" figure-id="ExpressionPattern"><div class="figure-content"><img src="http://think-lab.s3.amazonaws.com/m/figures/14.png"></div>
            <div class="signature-1">
                <div class="figure-title">Figure 4. Expression pattern of FoxP-Gal4 in the line from the laboratory of Constance Scharff</div>
                <div class="figure-description"><p>Left: Three views of confocal stacks from anterior to posterior.<br>Right: Two details from the optic lobe (above) and the mushroom body (MB) calyx (below).</p></div>
            </div>
        </div>

<p>An alternative way to device a GAL4 driver line that faithfully targets FoxP-expressing neurons is to use RMCE-based gene-trapping targeting the Mi{MIC}FoxPMI09133 transposon inserted into the dFoxP gene at 3R:9728212 (FBti0154903). The regulatory motifs scattered throughout the gene region complicate any such gene-trapping approaches, emphasizing again the need for a reliable antibody to verify the gene trapping method. This option will be pursued in case our antibodies confirm our suspicion that none of the available FoxP-GAL4 lines replicate dFoxP expression. As outlined above, a reliable and well-characterized GAL4 driver for FoxP expressing neurons is a necessary condition for a mechanistic understanding of operant self-learning.</p>

<p>However, single driver lines may not be sufficient for a mechanistic understanding of dFoxP gene function, as we have found evidence that the different isoforms may serve different functions <span class="citation">[<a href="/doi/10.1371/journal.pone.0100648" class="citation" data-key="10.1371/journal.pone.0100648">4</a>]</span>. Moreover, evidence from vertebrates suggests that at least different FoxP family members form both homo- and hetero-dimers and even oligomers <span class="citation">[<a href="/doi/10.1002/pro.626" class="citation" data-key="10.1002/pro.626">19</a>, <a href="/doi/10.1016/j.str.2005.10.005" class="citation" data-key="10.1016/j.str.2005.10.005">20</a>, <a href="/doi/10.1093/intimm/dxm043" class="citation" data-key="10.1093/intimm/dxm043">21</a>]</span>. It is not known if the different dFoxP isoforms are expressed in the same neurons, nor if they form dimers or oligomers.</p>

<p>Using the by now tried-and-tested combination of operant self- and world-learning <span class="citation">[<a href="/doi/10.1371/journal.pone.0100648" class="citation" data-key="10.1371/journal.pone.0100648">4</a>, <a href="/doi/10.1016/j.beproc.2011.02.005" class="citation" data-key="10.1016/j.beproc.2011.02.005">22</a>, <a href="/doi/10.4161/cib.3.2.10334" class="citation" data-key="10.4161/cib.3.2.10334">23</a>, <a href="/doi/10.1016/j.cub.2009.06.014" class="citation" data-key="10.1016/j.cub.2009.06.014">24</a>, <a href="/doi/10.1016/j.cub.2008.07.041" class="citation" data-key="10.1016/j.cub.2008.07.041">6</a>]</span>, we will find out which of our manipulations of dFoxP expression have which effects specifically on operant self-learning. In particular, we will test if FoxP activity is required during adulthood or during development. We will test if FoxP activity is required in the brain or in the ventral nerve chord and in which neurons there. The identification of these neurons is necessary for a mechanistic understanding of this form of learning. Ultimately, we will study the mechanistic contribution of these neurons to operant self-learning by characterizing their functional connectivity to up- and downstream neurons within the circuit as well as their activity requirements by activating or silencing them during learning: what function do these neurons serve that makes their FoxP activity necessary for operant self-learning?</p>

<h2 id="11-project-related-publications">1.1 Project-related publications</h2>

<h3>1.1.1	Articles published by outlets with scientific quality assurance, book publications, and works accepted for publication but not yet published.</h3>

<ul><li>Mendoza E.; Colomb J.; Rybak J.; Pfl&#252;ger H.J.; Zars T.; Scharff C. and Brembs B. (2014): <em>Drosophila</em> FoxP mutants are deficient in operant self-learning. PLoS One  9(6): e100648 <span class="citation">[<a href="/doi/10.1371/journal.pone.0100648" class="citation" data-key="10.1371/journal.pone.0100648">4</a>]</span></li><li>Brembs, B. (2009): Mushroom-bodies regulate habit formation in <em>Drosophila</em>. Curr. Biol. 19(16): 1351&#8211;1355 <span class="citation">[<a href="/doi/10.1016/j.cub.2009.06.014" class="citation" data-key="10.1016/j.cub.2009.06.014">24</a>]</span></li><li>Brembs, B. and Plendl, W. (2008): Double dissociation of PKC and AC manipulations on operant and classical learning in <em>Drosophila</em>. Curr. Biol. 18(15): 1168-1171 <span class="citation">[<a href="/doi/10.1016/j.cub.2008.07.041" class="citation" data-key="10.1016/j.cub.2008.07.041">6</a>]</span></li></ul>

<h3>1.1.2	Other publications</h3>

<ul><li>Brembs B, Pauly D, Schade R, Mendoza E, Pfl&#252;ger J, Rybak J, Scharff C, Zars T (2010): The <em>Drosophila</em> FoxP gene is necessary for operant self-learning: implications for the evolutionary origins of language. Soc. Neurosci. Abstr., 704.7</li></ul>

<h3>1.1.3	Patents</h3>

<h4>1.1.3.1 Pending</h4>

<p>n.a.</p>

<h4>1.1.3.2 Issued</h4>

<p>n.a.</p>

<h1 id="2-objectives-and-work-programme">2 Objectives and work programme</h1>

<h2 id="21-anticipated-total-duration-of-the-project">2.1 Anticipated total duration of the project</h2>

<p>36 months</p>

<h2 id="22-objectives">2.2 Objectives</h2>

<p>This proposal has four specific objectives:</p>

<ol><li>Generate at least one RNAi line directed against dFoxP which shows significant knock-down on the mRNA level</li><li>Generate at least one highly specific monoclonal antibody against the FoxP gene product. </li><li>Generate a limited collection of genomic alterations to the dFoxP locus</li><li>Anatomical and behavioral characterization of a selected subset of available and all resulting fly lines. The behavioral experiments will detect deficits specifically in operant self-learning.</li></ol>

<p>Objectives 1-3 provide a thorough characterization of FoxP gene expression in <em>Drosophila</em>, as well as an initial characterization of the potential phenotypic consequences of altering FoxP expression during/after development. Hence, these objectives constitute a publishable research goal in its own right, in particular if the anatomical characterizations of objective 4 will be included as necessary. </p>

<p>Without at least some of the tools generated by the objectives 1-3, objective 4 cannot be completed. In fact, a thorough and reliable dissection of the mechanisms underlying operant self-learning is crucially dependent on the tools developed in objectives 1-3. However, in the face of catastrophic failure to reach some of these objectives, objectives 1 and 2 can, to some extent, complement the efforts from objective 3 and vice versa, such that some of the required experiments in objective 4 could take place. As such, the experimental design safeguards against such failures, but with the outcome of only partially reaching the goal of mechanistically explaining operant self-learning on a neuronal level. Characterizing existing lines, as well as identifying the temporal requirement of and the neurons with FoxP expression necessary for operant self-learning constitute readily achievable, minimal research aims that are publishable on their own and would be completed by a later research phase.<br>However, given that all the techniques we will use are established and have been used successfully in many other instances in the collaborating laboratories, there is currently no reason to expect such a failure. In fact, I anticipate objectives 1 and 3 to yield more possible manipulations than can be behaviorally characterized within the funding period and objective 2 to produce more than just one antibody. </p>

<h2 id="23-work-programme-incl-proposed-research-methods">2.3 Work programme incl. proposed research methods</h2>

<p>Objectives 1-3 on the one side and objective 4 on the other require radically different skill-sets and interests. While objectives 1-3 require traditional wet-lab bench work involving protein biochemistry and molecular biology, objective 4 requires the tools of behavioral neurobiology, which in the case of our laboratory also includes training in electronics and coding, above and beyond the behavioral neurobiology background. Commonly, wet-lab bench researchers are rarely trained (or are interested) in electronics/coding and vice versa, behavioral neurobiology students rarely have much experience or interest in molecular biology. Hence, the only way to reach a realistic applicant pool is to separate the two project areas and have two graduate students collaborate on this work programme. Moreover, the schedule for fly line and monoclonal antibody generation does not allow for additional behavioral experiments, even if a suitable applicant existed and could be identified. Therefore, the work programme is separated into these two project areas.</p>

<h3>2.3.1	Project Area Molecular Biology</h3>

<h4>2.3.1.1 dFoxP RNAi lines</h4>

<p>In collaboration with the Schneuwly laboratory <span class="citation">[<a href="/doi/10.1096/fj.05-5709com" class="citation" data-key="10.1096/fj.05-5709com">25</a>]</span>, we will use the short-hairpin technique to generate an RNAi line targeting the last exon of the dFoxP locus. In order to minimize the likelihood of mismatches between the RNAi construct and the target site (which we hypothesize to be responsible for the lack of mRNA knockdown in the currently available lines), we will sequence the relevant locus of our recipient strains and will bring all our driver lines into the same genetic background as the recipient line. In the course of this sequencing effort, we might find a fly strain without the polymorphisms we identified in the FoxP gene of the flies we used so far <span class="citation">[<a href="/doi/10.1371/journal.pone.0100648" class="citation" data-key="10.1371/journal.pone.0100648">4</a>]</span>. In this case, we will bring the 50200GD RNAi strain into this genetic background, as well as suitable driver lines. Without these mismatches to the 50200GD RNAi construct, we ought to be able to observe dFoxP mRNA knockdown. In the more likely case that no such strain can be identified, we will continue with generating three RNAi lines with the following target sequences:</p>

<table class="table markdown-table"><thead><tr><th>siRNA#</th><th>Position</th><th>SS Sequence</th><th>AS Sequence</th><th>Score</th><th>Corr_Scor</th><th></th></tr></thead><tbody><tr><td>1</td><td>389</td><td>GAUGAAUAUUCUACUAAUAUG</td><td>UAUUAGUAGAAUAUUCAUCGA</td><td>101.6</td><td>98.6</td></tr><tr><td>2</td><td>388</td><td>CGAUGAAUAUUCUACUAAUAU</td><td>AUUAGUAGAAUAUUCAUCGAU</td><td>100.6</td><td>97.6</td></tr><tr><td>3</td><td>354</td><td>CGGGUAUGAUCGAAAGUAACA</td><td>UUACUUUCGAUCAUACCCGAG</td><td>100.5</td><td>97.5</td></tr></tbody></table>

<p>We will then use both ubiquitous and pan-neuronal drivers in conjunction with qPCR to assess the effectiveness of these three RNAi lines. Should this part of the project take less than the anticipated time, we will use the same approach to specifically and individually target the other two dFoxP isoforms.</p>

<p>Thus, at the end of this project, we will have generated at least one RNAi line knocking down the mRNA of isoB, and potentially two more lines, one for each remaining isoform. At least the isoB line is required for characterizing any generated antibodies and as a positive control when characterizing the effects of the already available RNAi lines on the protein level. Together with the null alleles generated in other laboratories, these lines are a prerequisite for proper characterization of the monoclonal antibodies (2.3.1.2).</p>

<p>In the unlikely case the attempts from the other laboratories fail to generate null alleles, we will generate our own alleles using the CRISPR/Cas9 technique <span class="citation">[<a href="/doi/10.1534/g3.114.013979" class="citation" data-key="10.1534/g3.114.013979">26</a>]</span>, recently established in the Schneuwly laboratory.</p>

<h4>2.3.1.2 Anti-isoB monoclonal antibody</h4>

<p>In collaboration with the Pauly laboratory (University Hospital Regensburg), we will use the hybridoma technique and a multiplex screening assay to generate monoclonal antibodies directed specifically against dFoxP isoB <span class="citation">[<a href="/doi/10.1039/b911525k" class="citation" data-key="10.1039/b911525k">27</a>]</span>.  In parallel, different inbred and outbred mouse lines will be immunized, as epitope-specific immune reactions strongly depend on different major histocompatibility complexes. We will use recombinantly expressed full-length dFoxP-isoB with flag-tag as immunogens <span class="citation">[<a href="/doi/10.1371/journal.pone.0096371" class="citation" data-key="10.1371/journal.pone.0096371">28</a>]</span>. In our collaboration with the Scharff laboratory in Berlin, we have already generated expression vectors for all FoxP isoforms in order to express the entire protein sequence for mouse immunization. Animals with a high antigen-specific antibody titer will be used for hybridoma generation <span class="citation">[<a href="/doi/10.1039/b911525k" class="citation" data-key="10.1039/b911525k">27</a>]</span>. After the fusion of spleen cells with myeloma cells the hybridoma clones will be subsequently isolated in 80&#8211;100 96-well plates. Screening of up to 3000 hybridoma clones will be performed using an in-house cytometric bead array. In this suspension array technology, differently colored microbeads are covalently coupled to different antigens, thus allowing simultaneous ELISA detection of multiple antibodies from a minimal sample volume of 50 &#181;L <span class="citation">[<a href="/doi/10.1039/b911525k" class="citation" data-key="10.1039/b911525k">27</a>]</span>. We will test, at the same time, the reactivity of the hybridoma supernatants against microbeads coupled to a) dFoxP-isoA, b) dFoxP-isoB, c) dFoxP-isoIR, d) dFoxP-deficient <em>Drosophila</em> protein mixture, e) an irrelevant flag-tag protein and f) BSA as a negative control. Due to the specific requirements, multiple fusions and screens have to be performed to identify dFoxP-isoform specific monoclonal mouse antibodies. After positive isolation, the clones will be subcloned and expanded. Characterization of the hybridoma lines will be done in common immunological assays using ELISA, Western Blots and immunostainings (Work programme, 2.3.3).</p>

<p>Thus, at the end of this project, we will have generated at least one monoclonal antibody which recognizes isoB of dFoxP highly specifically. Potentially, we will have one additional antibody which binds to all isoforms. These antibodies will be generated so as to allow for utilization in both Western Blots and wholemount immunochemistry.</p>

<p>With this antibody we will then characterize the available RNAi lines as well as the available GAL4 lines. In the former case, we will quantify the effectiveness of these lines in Western blots. In the latter case, we will compare the GAL4 expression pattern with that of FoxP in whole mount double labelings. In addition, we will characterize the expression pattern of the eGFP reporter in the Mi{MIC} transposon inserted into the dFoxP gene at 3R:9728212 (FBti0154903). As the Hugo Bellen lab is currently generating a new series of MiMIC lines replacing those where the cassette failed to insert into an intron, it is possible that there will be such a new line available at the start of the project.</p>

<h4>2.3.1.3 Genome editing using MiMIC and RMCE</h4>

<p>In the case that none of the available FoxP-GAL4 lines faithfully reproduces the endogenous FoxP expression pattern, we will first generate a MiMIC-GAL4 line by using RMCE to insert a GAL4 gene trap cassette into the Mi{MIC} insertion in the fifth exon of the dFoxP locus (FBti0154903). After testing for any phenotype due to the insertion, the resulting GAL4 line will also be tested with various effectors for behavioral phenotypes.</p>

<p>Because the FoxP gene product is a transcription factor and hence expected to be localized in the cells&#8217; nucleus, the expression pattern of the protein is not very informative as to the neuropils in which the dFoxP neurons project. In order to characterize the neuronal structure and identity of FoxP isoB neurons, we will tag the last exon with a conditional CD8-GFP construct. Without expression of CD8-GFP, development of the fly will be wild type. When CD8-GFP expression is triggered after normal development, the construct will tether the protein to the membrane, not only labelling the entire neuron, but also preventing FoxP from entering the nucleus. The expression pattern of GFP (also in combination with immunochemistry for the FoxP protein) in these flies will reveal how successful this approach was. If the tether works, in addition to reporting the isoB neuronal identity and structure, this modification is predicted to also constitute an isoB-specific conditional mutant. We will test this prediction by subjecting these flies to behavioral experiments in operant self-learning. If, however, isoB forms di- or oligomers with other isoforms, we would expect a phenotype which is not specific to isoform B alone, e.g., the flies might show a flight deficit, as the mutant lines with compromised isoIR expression <span class="citation">[<a href="/doi/10.1371/journal.pone.0100648" class="citation" data-key="10.1371/journal.pone.0100648">4</a>]</span>. If there is no phenotype, the function of isoB appears to be cytosolic. If the GFP tag does not tether isoB to the membrane, we will replace exon 8 with the CD8-GFP construct either with a splice site or a basal promotor to generate an isoB specific mutant and membrane-bound reporter. Of course, different time-courses of CD8-GFP expression may thus also result in different phenotypes.</p>

<p>Should the first attempts to generate these flies be successful, we will use the same method to target the other isoforms.</p>

<p>The generation of the CD8-GFP tagged flies will first be attempted with a relatively simple RMCE-based approach <span class="citation">[<a href="/doi/10.1534/g3.114.014803" class="citation" data-key="10.1534/g3.114.014803">29</a>]</span>, where the MiMIC cassette (<a href="#GenomicEditing">Figure 5</a>a) is replaced with a cassette containing a portion of the genomic region of the dFoxP locus downstream of the Mi{MIC} insertion, with a conditional CD8-GFP construct inserted immediately after the last exon (in-frame). The resulting strain will have an altered dFoxP locus in that an attR site is now contained within exon five and that a conditional CD8-GFP construct is inserted in-frame at the very end of exon 8 (<a href="#GenomicEditing">Figure 5</a>b). Moreover, the right flanking region of the MiMIC cassette will now be pushed downstream by our insertion. As the attR site in exon 5 lies outside of the forkhead box and any potential regulatory regions downstream of the gene are contained in the genomic region we will insert, this manipulation should not alter dFoxP expression or function. However, as the Mi{MIC} insertion in line FBti0154903 is contained within a coding exon which is spliced into all FoxP isoforms, it is likely that this insertion alone already leads to a null mutation at the FoxP locus. Thus, we expect our RMCE approach to rescue any FoxP mutant phenotype the MiMIC strain might have. As the Hugo Bellen lab is currently generating a new series of MiMIC lines replacing those where the cassette failed to insert into an intron, it is possible that there will be such a new line available at the start of the project. We will test gene expression by Western blot as well as wholemount immunochemistry using the antibody developed in 2.3.1.2. We will test gene function using conditioning essays testing for operant self-learning as well as analyzing brain morphology. In the unlikely case that any of these tests reveal a deviation from wild type in the edited flies, we will manipulate the dFoxP genomic region using a more sophisticated approach. In this approach, again taking advantage of the available strain with the Mi{MIC} transposon inserted into the fifth exon of the dFoxP gene, we will use the fast and efficient gene knock-in technique described by Vilain et al. (2014) to additionally excise all unwanted flanking regions using double-strand-break-mediated homologous recombination triggered by transgenically expressed restriction enzymes. Should both approaches fail, we can fall back on the slightly more complicated method developed by <span class="citation">[<a href="/doi/10.1534/g3.114.013979" class="citation" data-key="10.1534/g3.114.013979">26</a>]</span> in nearby Munich to replace a suitable region of the FoxP locus with an insertion where the conditional CD8-GFP is attached to exon 8. In fact, by the time this project will commence, there will likely be a genome editing method published, which is even more effective and efficient than the ones cited above. In this case, we will of course use that method.</p>

<a name="GenomicEditing"></a><div class="figure" figure-id="GenomicEditing"><div class="figure-content"><img src="http://think-lab.s3.amazonaws.com/m/figures/15.png"></div>
            <div class="signature-1">
                <div class="figure-title">Figure 5. Genomic editing strategy</div>
                <div class="figure-description"><p>A &#8211; The location of the MiMIC transgene in the fifth exon and its components. Located downstream of dFoxP is the hyperplastic discs (hyd) gene and its transcription start site is marked &#8216;R&#8217;.<br>B &#8211; Resulting dFoxP gene region after RMCE. By replacing the MiMIC cassette via RMCE with the remainder of the dFoxP gene downstream of the MiMIC insertion, together with the conditional CD8-GFP tag, the contiguity of the FoxP gene is restored and the entire gene region until the hyd start site (R) is retained. The &#8216;surplus&#8217; genomic components of the FoxP gene pushed back by the insertion lack a promotor region and are hence expected to become non-functional.</p></div>
            </div>
        </div>

<h3>2.3.2	Project Area Behavioral Neurobiology</h3>

<h4>2.3.2.1 Screening available lines</h4>

<p>With no detectable knockdown on the mRNA level, it is possible that all reported behavioral phenotypes of RNAi-based dFoxP manipulations may stem from off-target or other, unknown effects. This hypothesis will be tested by screening all available RNAi lines targeting dFoxP, as well as a selection of RNAi lines targeting other, unrelated genes (e.g. rutabaga) for defects in operant self-learning <span class="citation">[<a href="/doi/10.1371/journal.pone.0100648" class="citation" data-key="10.1371/journal.pone.0100648">4</a>, <a href="/doi/10.4161/cib.3.2.10334" class="citation" data-key="10.4161/cib.3.2.10334">23</a>, <a href="/doi/10.1016/j.cub.2008.07.041" class="citation" data-key="10.1016/j.cub.2008.07.041">6</a>, <a href="/doi/10.1016/j.cub.2009.06.014" class="citation" data-key="10.1016/j.cub.2009.06.014">24</a>]</span>. The student will also test RNAi lines against the other known required protein class, the six Protein Kinase C (PKC) genes in <em>Drosophila</em>, as our results with these genes so far have been inconclusive (see attached manuscript). This comparison of RNAi lines will not only familiarize the student with this sort of experiments, but also yield insights into which RNAi lines show a self-learning phenotype and hence warrant further scrutiny. Once available, the RNAi line generated in this project will of course also be thoroughly characterized using this approach. For instance, in addition to pan-neuronal drivers, we will use the drivers identified by Julien Colomb in the PKC project (see attached manuscript) to investigate if the same neurons require both PKC and FoxP function for operant self-learning. With ever more specific driver lines, we will narrow down the minimal set of neurons where FoxP activity is required. All of the lines with a self-learning phenotype will also be tested for their phenotype when their expression is restricted to adulthood. <br>In case the lines generated in this project should not be available in time, we will use the expression pattern gathered from immunohistochemistry (2.3.2.2) to guide over- and misexpression experiments using the available UAS-FoxP line (FBtp0094414). The anatomical and behavioral consequences of these experiments will elucidate the role of potential dimer/oligomer formation in FoxP function, as well as further characterize the already established necessity of a wild type FoxP expression pattern for operant self-learning.<br>The outcome of this large set of experiments will reveal the specificity of FoxP isoform involvement as well as circumscribe the neuronal circuitry in which FoxP manipulations can affect operant self-learning. As such, these results are publishable by themselves.</p>

<h4>2.3.2.2 Anatomical characterization of GFP, GAL-4 expression in MiMIC lines and anti-body staining</h4>

<p>Using confocal microscopy, we will compare the staining patterns of the original MiMIC insertion line, the GAL4 MiMIC line, the GFP MiMIC line and the wholemount immunostaining using the monoclonal AB(s). We will compare these data to the expression patterns in all the available FoxP-GAL4 lines. Together with the behavioral results of the MiMIC lines, these data will identify the neurons involved in FoxP-dependent operant self-learning. These results will then be compared to the neurons where PKC is required for operant self-learning (see attached manuscript). Wild type, mutant and transgenic FoxP gene expression and anatomical phenotype results will be published separately before the behavioral characterization.</p>

<h4>2.3.2.3 Behavioral characterization of MiMIC strains</h4>

<p>First, the original MiMIC stock will be tested for the suspected mutant phenotype, both in behavior and in brain anatomy. The generated GAL4 and CD8-GFP manipulated lines will be tested for any phenotypes in operant self-learning and a selected set of other behavioral tasks (Buridan&#8217;s paradigm, spontaneous walking activity using PySolo, operant world-learning, phototaxis). Importantly, like the CD8-GFP line before 'activation', the GAL4 gene trap line should not show any behavioral phenoptype. If it does, similar techniques as described for inserting CD8-GFP need to be employed to generate a GAL4 line that lacks a mutant phenotype (while of course retaining the mutant GAL4 line for other, e.g., rescuing purposes). The CD8-GFP line will be tested for phenotypes both with GFP untranslated and after &#8216;activation&#8217;. The most effective time period for CD8-GFP activation will be determined according to the results obtained from timing RNAi expression (see 2.3.2.1). A behavioral phenotype in the inactivated line indicates that the second GFP version, with removed flanking regions needs to be produced (see 2.3.1.3). <br>The experiments here first validate that the transgenic lines do not exhibit any phenotype before explicit FoxP manipulation and then test which of the manipulations show what kind of effects on which behavior(s). The latter results will tell us if FoxP is required in the nucleus and provide evidence for or against the hypothesis of di-/oligomerization of FoxP isoforms. If FoxP is needed in the nucleus for operant self-learning, these results will also provide an independent validation of the RNAi experiments: the same driver lines must lead to the same behavioral defects. The results from the other behavioral tasks will indicate to which extent these FoxP manipulations affect only learning or also other behaviors.</p>

<h4>2.3.2.4 Behavioral characterization of flies with manipulated FoxP-positive neurons</h4>

<p>Using the newly generated and validated GAL4 line (2.3.2.3), we will silence and activate, respectively, the identified FoxP neurons in operant self- and world-learning experiments, using common effector lines that we use routinely in our laboratory (e.g., shibere, TrpA1, TNT, crimson, channelrhodopsin, halorhodopsin, etc.). These experiments will be the start to elucidate the temporal structure of necessity and sufficiency of activity in these circuits for operant self-learning. The results from these experiments will instruct us as to which subcellular manipulations need to be performed to provide further mechanistic insight into the precise role of these neurons in operant self-learning. We expect these experiments to be continued in a follow-up proposal, as a complete behavioral analysis of the role of these neurons will require an additional full funding period.</p>

<h3>2.3.3	Work programme</h3>

<p><img src="https://cloud.githubusercontent.com/assets/1437696/11292751/67e473aa-8f8d-11e5-89f9-47f540d5e17a.png" alt="image"></p>

<p>Red: Molecular Biology Project Area (2.3.1)<br>Green: Behavioral Neurobiology Project Area (2.3.2)</p>

<p><strong>Schedule details for the generation of monoclonal antibodies.</strong> Experience in the Pauly laboratory indicates that one year is a realistic timeframe:</p>

<table class="table markdown-table"><thead><tr><th></th><th>Step</th><th>Time</th></tr></thead><tbody><tr><td>1.</td><td>Antigen preparation</td><td>3 months</td></tr><tr><td>2.</td><td>Immunization</td><td>3 months</td></tr><tr><td>3.</td><td>Fusion</td><td>3 months</td></tr><tr><td>4.</td><td>Selection</td><td>3 months</td></tr></tbody></table>

<p>For the generation of monoclonal antibodies peptides or protein antigens have to be in native conformation and highly purified. Add least 200-500 &#181;g antigen will be used for immunization and hybridoma screening. At the end of this period, three months will be required to use this antibody to characterize the FoxP-RNAi lines in quantitative western blots.</p>

<h2 id="24-data-handling">2.4 Data handling</h2>

<p>Any and all sequence data will be made accessible in the respective databases at the point of generation. Fly lines will be made available immediately to any interested colleagues and deposited in stock collections at the latest upon publication, if not earlier. The antibodies will be made accessible via antibodies-online.com as well as by inter-laboratory transfer.<br>Behavioral data will be made accessible and citable via DOI at the point of collection via automated scripts. Currently, the data are deposited at <a href="http://figshare.com">FigShare</a>, but we are developing such functionality at the university library here in Regensburg as well. Once these developments are concluded, we will host all data and software code openly accessible with persistent identifiers in our university library.<br>Inasmuch as technically feasible, we will also register all our reagents, model organisms and tools with the Resource Identification Portal, obtaining RRIDs for all of them.</p>

<h2 id="25-other-information">2.5 Other information</h2>

<p>Please use this section for any additional information you feel is relevant which has not been provided elsewhere.</p>

<p>n.a.</p>

<h2 id="26-descriptions-of-proposed-investigations-involving-experiments-on-humans-human-materials-or-animals">2.6 Descriptions of proposed investigations involving experiments on humans, human materials or animals</h2>

<p>n.a.</p>

<h2 id="27-information-on-scientific-and-financial-involvement-of-international-cooperation-partners">2.7 Information on scientific and financial involvement of international cooperation partners</h2>

<p>n.a.</p>

<h1 id="3-bibliography">3 Bibliography</h1>

<p>See <a href="#references">references below</a></p>

<h1 id="4-requested-modulesfunds">4 Requested modules/funds</h1>

<p>Explain each item for each applicant (stating last name, first name).</p>

<h2 id="41-basic-module">4.1 Basic Module</h2>

<h3>4.1.1	Funding for Staff</h3>

<p>Two graduate students</p>

<h3>4.1.2	Direct Project Costs</h3>

<h4>4.1.2.1 Equipment up to &#8364;10,000, Software and Consumables</h4>

<h5>4.1.2.1.1. Fly line generation</h5>

<table class="table markdown-table"><thead><tr><th>Item</th><th>Cost</th></tr></thead><tbody><tr><td>Chemicals</td><td>3000</td></tr><tr><td>Fly stocks</td><td>500</td></tr><tr><td>Fly food and vials</td><td>2000</td></tr><tr><td>Immunohistochemical reagents</td><td>2000</td></tr><tr><td>Lab chemicals, reagents, pipettes</td><td>1500</td></tr><tr><td>Oligonucleotides, Enzymes, etc.</td><td>3000</td></tr></tbody></table>

<h5>4.1.2.1.2 Monoclonal antibody generation</h5>

<table class="table markdown-table"><thead><tr><th>Item</th><th>Cost</th></tr></thead><tbody><tr><td>Immunization (peptides)</td><td>2000</td></tr><tr><td>Recombinant protein expression/ purification</td><td>2000</td></tr><tr><td>Cell culture (IL-6, serum, hybridoma cloning factor)</td><td>4000</td></tr><tr><td>Multiplex array (Beads, SA-PE, coupling chemicals)</td><td>5000</td></tr><tr><td>Protein purification</td><td>500</td></tr><tr><td>Lab chemicals and reagents, pipettes</td><td>1750</td></tr></tbody></table>

<h5>4.1.2.1.3 Behavioral analysis of wildtype and transgenic fly lines</h5>

<table class="table markdown-table"><thead><tr><th>Item</th><th>Cost</th></tr></thead><tbody><tr><td>Fly food and vials</td><td>3000</td></tr></tbody></table>

<p>Total 29250&#8364;</p>

<h4>4.1.2.2 Travel Expenses</h4>

<p>Expenses for the graduate students and the applicant for one European (e.g. FENS Forum and G&#246;ttingen Neuroscience Meeting) and one international conference (e.g. SfN, USA) per year, i.e., 1500&#8364; each per year, or 13500&#8364; in total for the funding period.</p>

<h4>4.1.2.3 Visiting Researchers (excluding Mercator Fellows)</h4>

<p>n.a.</p>

<h4>4.1.2.4 Expenses for Laboratory Animals</h4>

<p>Mouse strains for monoclonal antibodies	300&#8364;</p>

<h4>4.1.2.5 Other Costs</h4>

<p>n.a.</p>

<h4>4.1.2.6 Project-related publication expenses</h4>

<p>Open Access author processing charges are covered by the University of Regensburg library. For all other project-related publication expenses: 750&#8364; per year, i.e., a total of 2250&#8364;.</p>

<h3>4.1.3	Instrumentation</h3>

<h4>4.1.3.1 Equipment exceeding Euro 10,000</h4>

<p>n.a.</p>

<h4>4.1.3.2 Major Instrumentation exceeding Euro 100,000</h4>

<p>n.a.</p>

<h2 id="42-module-temporary-position-for-funding">4.2 Module Temporary Position for Funding</h2>

<p>n.a.</p>

<h2 id="43-module-replacement-funding">4.3 Module Replacement Funding</h2>

<p>n.a.</p>

<h2 id="44-module-temporary-clinician-substitute">4.4 Module Temporary Clinician Substitute</h2>

<p>n.a.</p>

<h2 id="45-module-mercator-fellows">4.5 Module Mercator Fellows</h2>

<p>n.a.</p>

<h2 id="46-module-workshop-funding">4.6 Module Workshop Funding</h2>

<p>n.a.</p>

<h2 id="47-module-public-relations-funding">4.7 Module Public Relations Funding</h2>

<p>n.a.</p>

<h1 id="5-project-requirements">5 Project requirements</h1>

<h2 id="51-employment-status-information">5.1 Employment status information</h2>

<p>For each applicant, state the last name, first name, and employment status (including duration of contract and funding body, if on a fixed-term contract).</p>

<p>Bj&#246;rn Brembs is permanently employed.</p>

<h2 id="52-first-time-proposal-data">5.2 First-time proposal data</h2>

<p>Only if applicable: Last name, first name of first-time applicant</p>

<p>n.a.</p>

<h2 id="53-composition-of-the-project-group">5.3 Composition of the project group</h2>

<p>List only those individuals who will work on the project but will not be paid out of the project funds. State each person&#8217;s name, academic title, employment status, and type of funding.</p>

<p>Bj&#246;rn Brembs, Prof. Dr., permanently employed, university funding<br>Angelika K&#252;hn, Technician, permanently employed, university funding<br>NN graduate student<br>NN graduate student</p>

<h2 id="54-cooperation-with-other-researchers">5.4 Cooperation with other researchers</h2>

<h3>5.4.1	Researchers with whom you have agreed to cooperate on this project</h3>

<p>Prof. Dr. Stephan Schneuwly, Institute of Zoology, Department of Developmental Biology, Universit&#228;t Regensburg<br>Dr. Schneuwly is an expert in the techniques used to generate the fly lines proposed in this application. The Schneuwly laboratory has generated such lines themselves and the techniques proposed are established there. Besides training, the Schneuwly laboratory will also provide whatever equipment is not available in the Brembs laboratory.<br>Dr. Diana Pauly, Immune therapies for degenerative eye diseases, Department of Ophthalmology, University Hospital Regensburg<br>Dr. Pauly has almost ten years of experience generating monoclonal antibodies against a large variety of epitopes. Besides training the student in all the necessary techniques for antibody generation, the Pauly laboratory will provide all infrastructure not available in the Brembs laboratory.</p>

<h3>5.4.2	Researchers with whom you have collaborated scientifically within the past three years</h3>

<p>Prof. Dr. Troy Zars University of Missouri, Columbia, USA<br>Prof. Dr. Constance Scharff, Freie Universit&#228;t Berlin<br>Prof. Dr. Hans-Joachim Pfl&#252;ger, Freie Universit&#228;t Berlin</p>

<h2 id="55-scientific-equipment">5.5 Scientific equipment</h2>

<p>List larger instruments that will be available to you for the project. These may include large computer facilities if computing capacity will be needed. </p>

<p>&#8216;Wetlab&#8217; facilities for the generation of fly lines and antibodies, behavioral setups, including torque meter, for characterizing the generated fly lines behaviorally.</p>

<h2 id="56-project-relevant-cooperation-with-commercial-enterprises">5.6 Project-relevant cooperation with commercial enterprises</h2>

<p>If applicable, please note the guidelines contained in the EU&#8217;s Community Framework for State Aid for Research and Development and Innovation (2006/C 323/01) or contact your research institution in this regard.</p>

<p>n.a.</p>

<h2 id="57-project-relevant-participation-in-commercial-enterprises">5.7 Project-relevant participation in commercial enterprises</h2>

<p>Information on connections between the project and the production branch of the enterprise</p>

<p>n.a.</p>

<h1 id="6-additional-information">6 Additional information</h1>

<p>If applicable, please list proposals requesting major instrumentation and/or those previously submitted to a third party here.</p>

<p>n.a.</p>



</div>

  

  
  
</div>


		  </td>
	    <td class="sidebar project-sidebar">

  			








	<div class="details">Published</div>
	
		Nov. 18, 2015<br>
	
	<span style="font-size:12px;">(Last updated Jan. 13, 2016)</span><br>

	<a style="font-size:12px;" href="/doc/8/revisions">Revision history</a>




<div class="details">Views</div>
<span style="font-size:20px;">550</span>

<div class="details">Topics </div>
<a class='tag' href='/topics/crisprcas'>CRISPR/Cas</a><a class='tag' href='/topics/neurobiology'>Neurobiology</a><a class='tag' href='/topics/genome-editing'>Genome Editing</a><a class='tag' href='/topics/neurogenetics'>Neurogenetics</a><a class='tag' href='/topics/language'>Language</a><a class='tag' href='/topics/antibodies'>Antibodies</a><a class='tag' href='/topics/learning'>Learning</a><a class='tag' href='/topics/behavior'>Behavior</a><a class='tag' href='/topics/rnai'>RNAi</a><a class='tag' href='/topics/foxp'>FoxP</a><a class='tag' href='/topics/operant-learning'>Operant Learning</a><a class='tag' href='/topics/drosophila'>Drosophila</a>


<div class="details">Cite this as</div>
<span style="font-size:12px;">
Björn Brembs (2015) What is the function of FoxP in operant self-learning? [proposal]. <i>Thinklab.</i> doi:<a href='https://doi.org/10.15363/thinklab.a8'>10.15363/thinklab.a8</a>
</span>

<div class="details">License</div>
<p><a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0;width:88px;height:31px;" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a></p>


<br><br>

<div class="affix-container">
<div class="affix-box">


</div></div>


	    </td>
	  </tr></table>


		</div>


		
	    	<div class="footer">
<div style="background:#dedede;">
	<div class="main-centered-div text-center" style="padding:20px;">
		<h4 style="margin-bottom:0px;font-weight:bold;color:gray;">Feedback? Questions? Email us: <a href="/cdn-cgi/l/email-protection#650d0009090a25110d0c0b0e0904074b060a08"><span class="__cf_email__" data-cfemail="5f373a3333301f2b37363134333e3d713c3032">[email&#160;protected]</span><script data-cfhash='f9e31' type="text/javascript">/* <![CDATA[ */!function(t,e,r,n,c,a,p){try{t=document.currentScript||function(){for(t=document.getElementsByTagName('script'),e=t.length;e--;)if(t[e].getAttribute('data-cfhash'))return t[e]}();if(t&&(c=t.previousSibling)){p=t.parentNode;if(a=c.getAttribute('data-cfemail')){for(e='',r='0x'+a.substr(0,2)|0,n=2;a.length-n;n+=2)e+='%'+('0'+('0x'+a.substr(n,2)^r).toString(16)).slice(-2);p.replaceChild(document.createTextNode(decodeURIComponent(e)),c)}p.removeChild(t)}}catch(u){}}()/* ]]> */</script></a></h4>
	</div>
</div>
<div class="main-footer">
	<div class="main-centered-div">

<table class="full-width"><tr>
	<td class="footer-nav">

        <div class="foot-head">Explore</div>
        <a href="/proposals">Proposals</a><br>
        <a href="/projects">Projects</a><br>
        <a href="/publications">Article discussion</a><br>
        <a href="/discussion">All discussion</a><br>
        <a href="/topics">Topics</a><br>
        <!-- <a href="/funders">Funders</a><br> -->
        <a href="/funding">Funding opportunities</a><br>

	</td>
	<td class="footer-nav">

        <!-- <div class="foot-head">Help &amp; feedback</div>	 -->
        <!-- <div class="foot-head">Intro</div>
        <a href="/benefits/for-science">Why</a><br>
        <a href="/how-it-works">How it works</a><br>

<br> -->

        <div class="foot-head">Community</div>
        <a href="/leaderboard">Leaderboard</a><br>
        <a href="/p/meta">Thinklab Meta</a><br>
				<a href="/help/faq">Help</a><br>
				<!-- <a href="/new-project">Start a proposal</a><br> -->
        <!-- <a href="/benefits/proposal">Benefits</a><br> -->
        <!-- <a href="/funding">Get funded</a><br> -->
        <!-- <a href="/participate">Become a contributor</a><br> -->
        <!-- <a href="/for-funders">For science funders</a><br> -->
        <!-- <a href="/for-institutions">For institutions</a><br> -->

	</td>
	<td class="footer-nav">



        <div class="foot-head">About</div>
        <a href="/about">Overview</a><br>
        <a href="/benefits">Benefits</a><br>
        <a href="/how-it-works">How it works</a><br>
        <a href="/about/story">Our story</a><br>
        <a href="/blog">Our blog</a><br>
        <a href="/about/jobs">Join the team</a><br>
        <br>

        <!-- <div class="foot-head">Help</div>
        <a href="/help">Help</a><br>
		 -->


	</td>
	<td class="footer-nav">
		<a href="http://twitter.com/thinklab" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/twitter_com.png" width="40"/></a>
		<a href="https://www.facebook.com/pages/Thinklab/310895039088185" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/facebook_com.png" width="40"/></a>
		<a href="https://plus.google.com/115578983466800753402/" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/plus_google_com.png" width="40"/></a>
		<a href="https://www.linkedin.com/company/thinklab-co" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/linkedin_com.png" width="40"/></a>
		<br><br>
		<span class="light" style='font-size:12px;'>&copy; 2015 Thinklab</span>

	</td>

</tr></table>

</div>
</div>
</div>

		

	</div>


  <script></script>
  <script>
  var MEDIA_URL = 'https://think-lab.s3.amazonaws.com/m/';
  var STATIC_URL = 'https://think-lab.s3.amazonaws.com/s/';
  var page_is_authenticated = false;
  var page_page = '';
  var page_profile_id = undefined;
</script>



	<!-- Note different script type. MUST come before MathJax.js -->
	<script type="text/x-mathjax-config">
	    MathJax.Hub.Config({
	        "HTML-CSS": { preferredFont: "TeX", availableFonts: ["STIX","TeX"],
	                        scale: 110, linebreaks: { automatic:true },
	                        EqnChunk: (MathJax.Hub.Browser.isMobile ? 10 : 50) },
	        tex2jax: { inlineMath: [ ["$$", "$$"], ["\\(","\\)"] ],
	                    displayMath: [ ["$$$","$$$"], ["\\[", "\\]"] ],
	                    processEscapes: true,}
	            });

	</script>
    <script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>





  <script src="https://code.jquery.com/jquery-1.10.2.min.js"></script>
  <script src="https://code.jquery.com/ui/1.11.4/jquery-ui.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" integrity="sha384-0mSbJDEHialfmuBBQP6A4Qrprq5OVfW37PRR3j5ELqxss1yVqOtnepnHVP9aJ7xS" crossorigin="anonymous"></script>
  <script src="https://think-lab.s3.amazonaws.com/s/dist/main.min.js"></script>
  



  
	

    
    
<script>
$(function() {
  recordHit('document', 8);
});
</script>


    



    


<script>/* <![CDATA[ */(function(d,s,a,i,j,r,l,m,t){try{l=d.getElementsByTagName('a');t=d.createElement('textarea');for(i=0;l.length-i;i++){try{a=l[i].href;s=a.indexOf('/cdn-cgi/l/email-protection');m=a.length;if(a&&s>-1&&m>28){j=28+s;s='';if(j<m){r='0x'+a.substr(j,2)|0;for(j+=2;j<m&&a.charAt(j)!='X';j+=2)s+='%'+('0'+('0x'+a.substr(j,2)^r).toString(16)).slice(-2);j++;s=decodeURIComponent(s)+a.substr(j,m-j)}t.innerHTML=s.replace(/</g,'&lt;').replace(/\>/g,'&gt;');l[i].href='mailto:'+t.value}}catch(e){}}}catch(e){}})(document);/* ]]> */</script></body>
</html>
