<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8" /><script type="text/javascript">window.NREUM||(NREUM={}),__nr_require=function(e,n,t){function r(t){if(!n[t]){var o=n[t]={exports:{}};e[t][0].call(o.exports,function(n){var o=e[t][1][n];return r(o||n)},o,o.exports)}return n[t].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<t.length;o++)r(t[o]);return r}({1:[function(e,n,t){function r(){}function o(e,n,t){return function(){return i(e,[c.now()].concat(u(arguments)),n?null:this,t),n?void 0:this}}var i=e("handle"),a=e(2),u=e(3),f=e("ee").get("tracer"),c=e("loader"),s=NREUM;"undefined"==typeof window.newrelic&&(newrelic=s);var p=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],d="api-",l=d+"ixn-";a(p,function(e,n){s[n]=o(d+n,!0,"api")}),s.addPageAction=o(d+"addPageAction",!0),s.setCurrentRouteName=o(d+"routeName",!0),n.exports=newrelic,s.interaction=function(){return(new r).get()};var m=r.prototype={createTracer:function(e,n){var t={},r=this,o="function"==typeof n;return i(l+"tracer",[c.now(),e,t],r),function(){if(f.emit((o?"":"no-")+"fn-start",[c.now(),r,o],t),o)try{return n.apply(this,arguments)}finally{f.emit("fn-end",[c.now()],t)}}}};a("setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(e,n){m[n]=o(l+n)}),newrelic.noticeError=function(e){"string"==typeof e&&(e=new Error(e)),i("err",[e,c.now()])}},{}],2:[function(e,n,t){function r(e,n){var t=[],r="",i=0;for(r in e)o.call(e,r)&&(t[i]=n(r,e[r]),i+=1);return t}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],3:[function(e,n,t){function r(e,n,t){n||(n=0),"undefined"==typeof t&&(t=e?e.length:0);for(var r=-1,o=t-n||0,i=Array(o<0?0:o);++r<o;)i[r]=e[n+r];return i}n.exports=r},{}],4:[function(e,n,t){n.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(e,n,t){function r(){}function o(e){function n(e){return e&&e instanceof r?e:e?f(e,u,i):i()}function t(t,r,o,i){if(!d.aborted||i){e&&e(t,r,o);for(var a=n(o),u=m(t),f=u.length,c=0;c<f;c++)u[c].apply(a,r);var p=s[y[t]];return p&&p.push([b,t,r,a]),a}}function l(e,n){v[e]=m(e).concat(n)}function m(e){return v[e]||[]}function w(e){return p[e]=p[e]||o(t)}function g(e,n){c(e,function(e,t){n=n||"feature",y[t]=n,n in s||(s[n]=[])})}var v={},y={},b={on:l,emit:t,get:w,listeners:m,context:n,buffer:g,abort:a,aborted:!1};return b}function i(){return new r}function a(){(s.api||s.feature)&&(d.aborted=!0,s=d.backlog={})}var u="nr@context",f=e("gos"),c=e(2),s={},p={},d=n.exports=o();d.backlog=s},{}],gos:[function(e,n,t){function r(e,n,t){if(o.call(e,n))return e[n];var r=t();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(e,n,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return e[n]=r,r}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],handle:[function(e,n,t){function r(e,n,t,r){o.buffer([e],r),o.emit(e,n,t)}var o=e("ee").get("handle");n.exports=r,r.ee=o},{}],id:[function(e,n,t){function r(e){var n=typeof e;return!e||"object"!==n&&"function"!==n?-1:e===window?0:a(e,i,function(){return o++})}var o=1,i="nr@id",a=e("gos");n.exports=r},{}],loader:[function(e,n,t){function r(){if(!x++){var e=h.info=NREUM.info,n=d.getElementsByTagName("script")[0];if(setTimeout(s.abort,3e4),!(e&&e.licenseKey&&e.applicationID&&n))return s.abort();c(y,function(n,t){e[n]||(e[n]=t)}),f("mark",["onload",a()+h.offset],null,"api");var t=d.createElement("script");t.src="https://"+e.agent,n.parentNode.insertBefore(t,n)}}function o(){"complete"===d.readyState&&i()}function i(){f("mark",["domContent",a()+h.offset],null,"api")}function a(){return E.exists&&performance.now?Math.round(performance.now()):(u=Math.max((new Date).getTime(),u))-h.offset}var u=(new Date).getTime(),f=e("handle"),c=e(2),s=e("ee"),p=window,d=p.document,l="addEventListener",m="attachEvent",w=p.XMLHttpRequest,g=w&&w.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:w,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var v=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-1039.min.js"},b=w&&g&&g[l]&&!/CriOS/.test(navigator.userAgent),h=n.exports={offset:u,now:a,origin:v,features:{},xhrWrappable:b};e(1),d[l]?(d[l]("DOMContentLoaded",i,!1),p[l]("load",r,!1)):(d[m]("onreadystatechange",o),p[m]("onload",r)),f("mark",["firstbyte",u],null,"api");var x=0,E=e(4)},{}]},{},["loader"]);</script><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info={"beacon":"bam.nr-data.net","queueTime":0,"licenseKey":"cd1101aa94","agent":"","transactionName":"bgcDMkBZW0NZBkdcWVdNJxNcW0FZVwscV1dKB08CV1taQlkRXEdFAz0WFFNIRVVcOkVcU049BxNcWw==","applicationID":"6369297","errorBeacon":"bam.nr-data.net","applicationTime":394}</script>
    <title>Proposal: Microbial resistance to disinfectants in animal shelters</title>
	  <link rel="shortcut icon" href="https://think-lab.s3.amazonaws.com/s/img/favicon.ico" type="image/x-icon" />
    <meta property="og:image" content="https://think-lab.s3.amazonaws.com/s/img" />
    
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
  <link rel="stylesheet" href="https://code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css" />

<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/highlightjs/github.css" />
<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/css/css.css" />
<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/css/bootstrap-multiselect.css" />
<!-- <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'> -->

    
    
    
    
  
    <meta name="citation_doi" content="10.15363/thinklab.a4">
    <meta name="citation_title" content="Microbial resistance to disinfectants in animal shelters [proposal]">
    <meta name="citation_online_date" content="June 1, 2014">
    <meta name="citation_author" content="Holly Ganz, Jonathan Eisen, Leesa Li, Ruth Lee, David Coil, Katie Dahlhausen, Holly Bik">
    <meta name="citation_journal_title" content="Thinklab">
    <meta name="citation_publisher" content="Thinklab">
  

    


	<!-- GoogleAnalytics -->
	<script>
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
	(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
	m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

	ga('create', 'UA-48550230-1', 'thinklab.com');
	ga('require', 'displayfeatures');
	
	ga('send', 'pageview');

	</script>






</head>
<body >
  <form id="form_action" method="post">
    <input type='hidden' name='csrfmiddlewaretoken' value='pQNiZQX4gF2vyFeXrnbUl9EmuTWA5MG1' />
</form>

	

	<div class="body-wrapper">

		


<!-- <div style="height:5px;background-image:url('https://think-lab.s3.amazonaws.com/s/img/top-bg.gif');"></div> -->
<div class="gradient">
	<div class="main-centered-div header-container">

<table class="full-width" style="height:60px;"><tr>
	<td style="width:200px;vertical-align:middle;padding-bottom:15px;">
		<a href="/" style="font-weight:normal;color:#212121;"><img src="https://think-lab.s3.amazonaws.com/s/img/logo.png" width="160"></a>
	</td><td style="vertical-align:middle;text-align:left;padding-top:6px;width:100%;padding-left:40px;">
		<ul id="not-search-block"  class="main-header">


			


				<li class="dropdown">
					<a class="dropdown-toggle" data-toggle="dropdown" href="#">Browse <span class="caret"></span></a>
					<ul class="dropdown-menu" role="menu">
						<li><a href="/proposals">Proposals</a></li>
						<li><a href="/projects">Projects</a></li>
						<li><a href="/publications">Article discussion</a></li>
						<li><a href="/discussion">All discussion</a></li>
					</ul>
				</li>
				<li><a href="/leaderboard">Leaderboard</a></li>

			
				<li><a href="/about">About</a></li>
			


		</ul>

    <ul id="search-block" class="main-header" style="display: none; margin-right: 30px;">
        <li class="full-width">
            <form method="get" action="/search">
			<div class="input-group full-width">
				<input name="q" type="text" class="form-control search-input">
			</div>
            </form>
        </li>
    </ul>

	</td><td style="vertical-align:middle;text-align:right;padding-top:6px; min-width: 1%; white-space: nowrap;">
		<ul class="main-header pull-right" style="padding-left: 0;">
    		
                <li><a href="/login" style="color:silver;">Login</a></li>
                <li style="margin-right:0"><a class="btn btn-default btn-sm btn-upper" href="/signup">Join</a></li>
            

        </ul>
    </ul>
	</td>
</tr></table>

<!-- <div id="global-search-div" class="main-centered-div" style="display:none;">
	<input type="text" value="" class="thread-search form-control" placeholder="Search" data-action="/discussion">
</div> -->

	</div>
</div>

		
	  



	  


		

		<div class="dark-head"><div class="main-centered-div">

			

	


	<div class="header-object-type"></div>

	
			<a class="btn btn-lg follow-link btn-follow pull-right active tt-top" style="margin-left:20px;" title="Follow project discussion" data-id="3" data-ctype="project" data-is-following="false">&nbsp;</a>
	

	<h2 class="header-object-name">Microbial resistance to disinfectants in animal shelters</h2>

	
	<div class="project-team">
		<a href='/u/hollyganz'>Holly Ganz</a>, <a href='/u/jonathaneisen'>Jonathan Eisen</a>, <a href='/u/leesali'>Leesa Li</a>, <a href='/u/ruthlee'>Ruth Lee</a>, <a href='/u/davidcoil'>David Coil</a>, <a href='/u/katiedahlhausen'>Katie Dahlhausen</a>, <a href='/u/hollybik'>Holly Bik</a>
		 <!-- &nbsp; &nbsp; doi:<a href="https://doi.org/10.15363/thinklab.3">10.15363/thinklab.3</a> -->
	</div>
	

		<!-- <div class="project-team">
			<b>DOI:</b> <a href="https://doi.org/10.15363/thinklab.3">10.15363/thinklab.3</a>
		</div> -->

		

	<div class="project-team">
		
		<div class="proposal-status proposal-status-lg proposal-status-awaiting-funder">Awaiting Funder</div>
		
		
		
	</div>




	


	<!-- <div class="project-team">
		
	</div> -->

    <ul class="nav nav-tabs project-nav" style="margin-top:25px;">

		

		

    

		
	    
	        <li class="active"><a href="/p/MicrobeResist/proposal">Proposal</a></li>
	    
	    
		

		<li><a href="/p/MicrobeResist/discussion"><span class="nav-fig">8</span>Discussions</a></li>

		
			<li><a href="/p/MicrobeResist/leaderboard"><span class="nav-fig">7</span>Reviewers</a></li>
		

		

		


        <li class="dropdown">
            <a class="dropdown-toggle" data-toggle="dropdown"><span style="opacity:0.5">More <span class="caret"></span></span></a>
            <ul class="dropdown-menu" style="font-size:13px;background-color:#819ec8">

				

				

            <li><a href="/p/MicrobeResist/followers"><span class="nav-fig">16</span>Followers</a></li>
            </ul>
        </li>

			

    </ul>



		</div></div>
		
		<div class="main-centered-div" style="margin-top:40px;padding-bottom:340px">
	    

	

  

  
    
        
      
  

<!--  -->


  





    <table class="project-table"><tr>
      <td class="main-table-content">


        

        
			     <h1 class="page-title">Proposal</h1>
        

			  

<div class="document-body">

  <div id="document_body"><h1 id="abstract">Abstract</h1>

<p>Each year animal shelters provide essential care for millions of animals, many of which enter the shelter without receiving any prior veterinary care. Animal shelter housing typically has high occupancy of highly stressed, potentially immunosuppressed animals. Consequently disease transmission is a major concern. To protect both animal and human health, animals are vaccinated and treated with antibiotics; additionally most animal housing units are cleaned daily with disinfectants. The use of antibiotics and disinfectants may promote the spread of resistance traits in resident bacterial populations. Here we propose a study to characterize bacterial and fungal community composition and the presence of antibiotic and disinfectant resistance genes in the microbiota residing on surfaces in an animal shelter.</p>

<h1 id="objectives">Objectives</h1>

<ol><li><p>We will use amplicon sequencing to characterize the composition of bacterial and fungal communities occurring on surfaces in an animal shelter. By identifying what bacteria and fungi occur here, we can begin to understand sources for these communities (such as animal host, soil, air, water) and identify potential effects that different environmental factors may have on microbial persistence in these building reservoirs (such as use of disinfectants, proximity to windows, moisture, sunlight). These findings can help to inform practices within animal shelters such as the use of ventilation and disinfectants to protect animal health.</p></li><li><p>We will use metagenomic sequencing to identify the presence of genes associated with antimicrobial and antibiotic resistance as well as biofilm formation in these communities. This part of the study provides an important link between describing what microbes are present on building surfaces and identifying what traits might help them to persist in such relatively extreme environments. This study will help in the development of strategies to manage the development of antimicrobial and antibiotic resistance in an intensive housing environment.</p></li></ol>

<h1 id="background">Background</h1>

<p>Animal shelters house a high density of stressed animals, providing the opportunity for the emergence and transmission of infectious diseases <span class="citation">[<a href="https://doi.org/10.1177/0300985813511129" class="citation" data-key="10.1177/0300985813511129">1</a>]</span>. The US Humane Society estimates that between 6 and 8 million companion animals enter animal shelters in the United States each year. Many of these animals have not received any prior veterinary care or vaccination. Numerous pathogens occur in the animal shelter environment, including beta-hemolytic Streptococci, feline herpesvirus I, canine distemper virus, canine influenza virus, feline calicivirus, parvoviruses and feline infectious peritonitis  <span class="citation">[<a href="https://doi.org/10.1177/0300985813511129" class="citation" data-key="10.1177/0300985813511129">1</a>]</span>. Fungal infections such as ringworm are also of major concern <span class="citation">[<a href="https://doi.org/10.1016/j.prevetmed.2010.11.001" class="citation" data-key="10.1016/j.prevetmed.2010.11.001">2</a>]</span>. To minimize disease outbreaks in this intensive housing environment, shelter staff use vaccination, antibiotics and disinfectants to protect both animal and human health.</p>

<p>The widespread use of antimicrobial products is accelerating the spread of antibiotic and antimicrobial resistance. Drug resistant bacterial strains have been identified in animal shelters: bacteria with multidrug resistance were isolated from both dog kennels <span class="citation">[<a href="https://doi.org/10.1016/j.theriogenology.2012.05.033" class="citation" data-key="10.1016/j.theriogenology.2012.05.033">3</a>]</span> and cat housing areas <span class="citation">[<a href="https://doi.org/10.1016/j.rvsc.2005.10.004" class="citation" data-key="10.1016/j.rvsc.2005.10.004">4</a>]</span>. Increased biocide resistance may cause increased antibiotic resistance because resistance genes can be located in close proximity and can share common resistance mechanisms (e.g., <span class="citation">[<a href="https://doi.org/10.1128/AAC.45.2.428-432.2001" class="citation" data-key="10.1128/AAC.45.2.428-432.2001">5</a>]</span>, <span class="citation">[<a href="https://doi.org/10.1046/j.1365-2672.92.5s1.4.x" class="citation" data-key="10.1046/j.1365-2672.92.5s1.4.x">6</a>]</span>, <span class="citation">[<a href="https://doi.org/10.1016/j.tim.2006.02.006" class="citation" data-key="10.1016/j.tim.2006.02.006">7</a>]</span>, <span class="citation">[<a href="https://doi.org/10.1016/j.vetmic.2011.02.014" class="citation" data-key="10.1016/j.vetmic.2011.02.014">8</a>]</span>, <span class="citation">[<a href="https://doi.org/10.1128/AAC.49.1.454-456.2005" class="citation" data-key="10.1128/AAC.49.1.454-456.2005">9</a>]</span>). Furthermore, antibiotic resistance can be transmitted from animals treated with antibiotics to humans working in close proximity to them <span class="citation">[<a href="https://doi.org/10.1089/fpd.2006.0066" class="citation" data-key="10.1089/fpd.2006.0066">10</a>]</span>.  </p>

<p>Shelter animals and staff may acquire drug resistant bacterial strains in shelters and carry them into local communities. According to the ASPCA about 2.7 million companion animals are adopted from shelters each year. Many people live in close contact with their pets; studies of dog owners found that they share both gastrointestinal bacteria <span class="citation">[<a href="https://doi.org/10.1017/S0022172400064597" class="citation" data-key="10.1017/S0022172400064597">11</a>]</span> and skin bacteria <span class="citation">[<a href="https://doi.org/10.7554/eLife.00458" class="citation" data-key="10.7554/eLife.00458">12</a>]</span> with their dogs. Our pets harbor and shed diverse microbial communities that may shape the microbial composition of their human companions <span class="citation">[<a href="https://doi.org/10.7554/eLife.00458" class="citation" data-key="10.7554/eLife.00458">12</a>]</span> through both direct and indirect contact. Although most research has focused on the effects of human occupancy in buildings, animal skin, hair, fecal material, saliva and ectoparasites such as fleas and mites are likely to contribute significantly to the microbial communities associated with indoor surfaces and air <span class="citation">[<a href="https://doi.org/10.1111/j.1365-2222.1994.tb03323.x" class="citation" data-key="10.1111/j.1365-2222.1994.tb03323.x">13</a>]</span>, <span class="citation">[<a href="https://doi.org/10.1016/j.jaci.2010.05.042" class="citation" data-key="10.1016/j.jaci.2010.05.042">14</a>]</span>, <span class="citation">[<a href="https://doi.org/10.1186/gb-2013-14-2-202" class="citation" data-key="10.1186/gb-2013-14-2-202">15</a>]</span>. </p>

<p>Here we propose a pilot study to characterize the microbial communities occurring on surfaces in an animal shelter. First we will use phylogenetically informative markers to characterize the diversity of bacteria and fungi occurring in these samples. Then we will sequence the metagenome for a subset of these samples to identify genes that may promote persistence in this environment, including those that affect the ability to form biofilms to colonize surfaces, such as amyloid fibers called curli promote adhesion on surfaces and human epithelial cells (found in Enterbacteriaceae), as well as resistance to antimicrobial disinfectants and antibiotics. </p>

<h1 id="research-plan">Research Plan</h1>

<p>We will focus on the holding area for dogs in an open intake municipal animal shelter. The dog holding area contains 52 individual indoor single compartment kennels, 80&#8221; long x 40&#8221; wide x 80&#8221; high. Kennel floors are concrete and walls are ceramic tile covered with concrete block. Kennels are arranged linearly with one central aisle and 2 side aisles. The ventilation system draws air from the outside into the the holding area and exhaust is pulled out by ceiling fans. The kennel also has open windows and doors in summer. Dogs brought into the shelter are subject to a five day holding period and during this period they cannot be walked. Each kennel is cleaned daily to remove animal waste with accelerated hydrogen peroxide sprayed by a pressure washer. 	</p>

<p>We will collect surface samples from throughout the dog holding area. Surface samples will be collected from the floors, walls and ledges using a 10cm x 10cm grid and sterile, dry cotton swabs (Puritan brand). We collect three samples from each kennel (52 kennels x 3 surfaces = 156 samples). We will record whether a kennel was currently occupied. We will also collect samples from the walkways, drainage system, window ledges and outside each entrance to the dog holding area. Samples will be stored at -80 &#176;C until DNA is extracted.</p>

<p>DNA will be extracted using MoBio PowerSoil kits with an additional lysis step to maximize extraction efficiency for low biomass, swab samples. We will use sterile scissors to cut the cotton head of the swab directly into provided PowerBead tubes containing solution C1. The tubes will be incubated at 65 &#176;C for 10 minutes and vortexed horizontally for 10 minutes. Subsequently, we will follow the standard kit protocol. DNA samples will be stored at -80&#176;C prior to preparing Illumina libraries for sequencing. </p>

<p>DNA isolated from surface samples will be characterized for bacterial diversity based on phylogenetically informative markers in the V4 region of 16S rRNA following the methods of <span class="citation">[<a href="https://doi.org/10.1038/nmeth.f.303" class="citation" data-key="10.1038/nmeth.f.303">16</a>]</span>. Fungal diversity will be characterized using the internal transcribed spacer (ITS) region 1 of the nuclear ribosomal coding cistron <span class="citation">[<a href="https://doi.org/10.1073/pnas.1117018109" class="citation" data-key="10.1073/pnas.1117018109">17</a>]</span>. Samples will be sequenced in multiplexed Illumina MiSeq lanes (using paired-end 250 base pair sequencing). Sequence data will be entered into the standard Quantitative Insights in Microbial Ecology (QIIME 1.5.0) workflow. Each sample will be characterized for taxonomic composition (number and abundance). For presence/absence analyses, representative operational taxonomic units (OTUs) will be clustered at the &gt;97% identity level and an OTU table will be constructed using QIIME&#8217;s pick_otus_through_otu_table.py script. We will compare beta diversity using Fast UniFrac scores <span class="citation">[<a href="https://doi.org/10.1038/ismej.2009.97" class="citation" data-key="10.1038/ismej.2009.97">18</a>]</span> visualized with principle coordinates analysis. We will compare the temporal sequence of establishment in each kitten and assess temporal changes in microbiome diversity using various nonparametric multivariate methods, discriminant function analysis and association networks. We will use SourceTracker <span class="citation">[<a href="https://doi.org/10.1038/nmeth.1650" class="citation" data-key="10.1038/nmeth.1650">19</a>]</span> to identify potential sources of bacteria by comparing bacterial OTUs identified in samples from outside, soil, canine and human. </p>

<p>We will sequence the metagenome for a subset of these samples to detect antimicrobial and antibiotic resistance in bacteria occurring on surfaces in the animal shelter environment. Because we expect to have a low amount of input DNA for each sample, we will use the Illumina Nextera XT library preparation kit. We will select, label with barcodes and pool up to 24 samples for sequencing. We will use PhyloSift <span class="citation">[<a href="https://doi.org/10.7717/peerj.243" class="citation" data-key="10.7717/peerj.243">20</a>]</span> to perform phylogeny-driven Bayesian hypothesis tests for the presence of bacteria, fungi, viruses and other organisms occurring in these samples. The BacMet database for antibacterial biocide and metal resistance genes will be used to screen our metagenomic data for resistance genes as well as genes associated with biofilm formation <span class="citation">[<a href="https://doi.org/10.1093/nar/gkt1252" class="citation" data-key="10.1093/nar/gkt1252">21</a>]</span>.</p>

<h1 id="team-resources">Team &amp; Resources</h1>

<p>Jonathan Eisen, Primary Investigator, is an expert in microbial genomics and an advocate for open science.<br>Holly Ganz, Project Scientist, is a microbial ecologist, who uses genetic and statistical methods to characterize the diversity of microorganisms that live in the environment and in association with plants and animals. Currently she is leading the Kitten Microbiome Project, a citizen science project on the development of the microbiome in neonatal kittens. <br>David Coil, Project Scientist. mentors undergraduate research projects in microbial ecology and genomics in the Eisen Lab. Co-investigator for Project MERCURRI, a citizen science project focused on bacteria living in the International Space Station. Also performs public outreach in Microbiology through the microBEnet project, which is focused on communication in the context of studies of microbiology of the built environment. <br>Katherine Dahlhausen, Graduate Student Researcher, Biophysics Program, studies how bacteria tolerate and respond to DNA damage.<br>Ruth Lee and Leesa Li, Undergraduate Student Researchers, will provide laboratory assistance.</p>

<p>Available resources: &#8232;1. The UC Davis Genome Center integrates experimental and computational approaches to address key problems at the forefront of genomics. The Center is housed in a new research building with state-of-the-art computational and laboratory facilities. The DNA Technologies Core of the Genome Center provides both high throughput sequencing analysis, including Illumina HiSeq and MiSeq.&#8232; 2. Dr. Eisen's laboratory occupies roughly 1000 sq. ft. of wet laboratory space in the Genome and Biomedical Sciences Facility. This laboratory includes all the necessary equipment to perform PCR and qPCR, Qubit (for DNA quantification) and microscopy. Dr. Eisen also has office and cubicle space for computational staff, students and postdoctoral fellows that is directly adjacent to the laboratory. The space can comfortably accommodate eight bench scientists and four computational scientists and can accommodate undergraduate researchers.</p></div>

  

    <h1 id="budget-items">Budget Items &nbsp; </h1>
    <div>
      <div class="budget-doughnut">
        <canvas id="budget-doughnut" width="185" height="185"></canvas>
      </div>
      <div id="budget-doughnut-legend" class="legend"></div>
    </div>
    <div style="clear: both;"></div>

  

  
    <h1 id="references">References</h1>
    <table class="references table-td-5"><tr><td><a href='/doi/10.1177/0300985813511129' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>1.</div></td><td><div class='doi-body' data-key='10.1177/0300985813511129'><a href='/doi/10.1177/0300985813511129' class='pub-title'>Common and Emerging Infectious Diseases in the Animal Shelter</a><div class='pub-body'>P. A. Pesavento, B. G. Murphy (2013) <i>Veterinary Pathology</i>. doi:<a class='pub-doi' href='https://doi.org/10.1177/0300985813511129'>10.1177/0300985813511129</a></div></div></td></tr><tr><td><a href='/doi/10.1016/j.prevetmed.2010.11.001' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>2.</div></td><td><div class='doi-body' data-key='10.1016/j.prevetmed.2010.11.001'><a href='/doi/10.1016/j.prevetmed.2010.11.001' class='pub-title'>A needs-assessment and demographic survey of infection-control and disease awareness in western US animal shelters</a><div class='pub-body'>Kay K. Steneroden, Ashley E. Hill, M.D. Salman (2011) <i>Preventive Veterinary Medicine</i>. doi:<a class='pub-doi' href='https://doi.org/10.1016/j.prevetmed.2010.11.001'>10.1016/j.prevetmed.2010.11.001</a></div></div></td></tr><tr><td><a href='/doi/10.1016/j.theriogenology.2012.05.033' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>3.</div></td><td><div class='doi-body' data-key='10.1016/j.theriogenology.2012.05.033'><a href='/doi/10.1016/j.theriogenology.2012.05.033' class='pub-title'>Antimicrobial resistance in bacteria from breeding dogs housed in kennels with differing neonatal mortality and use of antibiotics</a><div class='pub-body'>C. Milani, M. Corrò, M. Drigo, A. Rota (2012) <i>Theriogenology</i>. doi:<a class='pub-doi' href='https://doi.org/10.1016/j.theriogenology.2012.05.033'>10.1016/j.theriogenology.2012.05.033</a></div></div></td></tr><tr><td><a href='/doi/10.1016/j.rvsc.2005.10.004' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>4.</div></td><td><div class='doi-body' data-key='10.1016/j.rvsc.2005.10.004'><a href='/doi/10.1016/j.rvsc.2005.10.004' class='pub-title'>Acquired antimicrobial resistance in the intestinal microbiota of diverse cat populations</a><div class='pub-body'>H. Moyaert, E.M. De Graef, F. Haesebrouck, A. Decostere (2006) <i>Research in Veterinary Science</i>. doi:<a class='pub-doi' href='https://doi.org/10.1016/j.rvsc.2005.10.004'>10.1016/j.rvsc.2005.10.004</a></div></div></td></tr><tr><td><a href='/doi/10.1128/AAC.45.2.428-432.2001' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>5.</div></td><td><div class='doi-body' data-key='10.1128/AAC.45.2.428-432.2001'><a href='/doi/10.1128/AAC.45.2.428-432.2001' class='pub-title'>Cross-Resistance between Triclosan and Antibiotics in Pseudomonas aeruginosa Is Mediated by Multidrug Efflux Pumps: Exposure of a Susceptible Mutant Strain to Triclosan Selects nfxB Mutants Overexpressing MexCD-OprJ</a><div class='pub-body'>R. Chuanchuen, K. Beinlich, T. T. Hoang, A. Becher, R. R. Karkhoff-Schweizer, H. P. Schweizer (2001) <i>Antimicrobial Agents and Chemotherapy</i>. doi:<a class='pub-doi' href='https://doi.org/10.1128/AAC.45.2.428-432.2001'>10.1128/AAC.45.2.428-432.2001</a></div></div></td></tr><tr><td><a href='/doi/10.1046/j.1365-2672.92.5s1.4.x' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>6.</div></td><td><div class='doi-body' data-key='10.1046/j.1365-2672.92.5s1.4.x'><a href='/doi/10.1046/j.1365-2672.92.5s1.4.x' class='pub-title'>Active efflux, a common mechanism for biocide and antibiotic resistance</a><div class='pub-body'>S.B. Levy (2002) <i>J Appl Microbiol</i>. doi:<a class='pub-doi' href='https://doi.org/10.1046/j.1365-2672.92.5s1.4.x'>10.1046/j.1365-2672.92.5s1.4.x</a></div></div></td></tr><tr><td><a href='/doi/10.1016/j.tim.2006.02.006' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>7.</div></td><td><div class='doi-body' data-key='10.1016/j.tim.2006.02.006'><a href='/doi/10.1016/j.tim.2006.02.006' class='pub-title'>Co-selection of antibiotic and metal resistance</a><div class='pub-body'>Craig Baker-Austin, Meredith S. Wright, Ramunas Stepanauskas, J.V. McArthur (2006) <i>Trends in Microbiology</i>. doi:<a class='pub-doi' href='https://doi.org/10.1016/j.tim.2006.02.006'>10.1016/j.tim.2006.02.006</a></div></div></td></tr><tr><td><a href='/doi/10.1016/j.vetmic.2011.02.014' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>8.</div></td><td><div class='doi-body' data-key='10.1016/j.vetmic.2011.02.014'><a href='/doi/10.1016/j.vetmic.2011.02.014' class='pub-title'>Zinc resistance of Staphylococcus aureus of animal origin is strongly associated with methicillin resistance</a><div class='pub-body'>Lina M. Cavaco, Henrik Hasman, Frank M. Aarestrup, Members of MRSA-CG:, Jaap A. Wagenaar, Haitske Graveland, Kees Veldman, Dik Mevius, Alexandra Fetsch, Bernd-Alois Tenhagen, M. Concepcion Porrero, Lucas Dominguez, Sophie A. Granier, Eric Jouy, Patrick Butaye, Eva Kaszanyitzky, Adam Dán, Jacek Zmudzki, Antonio Battisti, Alessia Franco, Stefan Schwarz, Montserrat Gutierrez, J. Scott Weese, Shengui Cui, Constança Pomba (2011) <i>Veterinary Microbiology</i>. doi:<a class='pub-doi' href='https://doi.org/10.1016/j.vetmic.2011.02.014'>10.1016/j.vetmic.2011.02.014</a></div></div></td></tr><tr><td><a href='/doi/10.1128/AAC.49.1.454-456.2005' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>9.</div></td><td><div class='doi-body' data-key='10.1128/AAC.49.1.454-456.2005'><a href='/doi/10.1128/AAC.49.1.454-456.2005' class='pub-title'>Relationship between Copper, Glycopeptide, and Macrolide Resistance among Enterococcus faecium Strains Isolated from Pigs in Denmark between 1997 and 2003</a><div class='pub-body'>H. Hasman, F. M. Aarestrup (2004) <i>Antimicrobial Agents and Chemotherapy</i>. doi:<a class='pub-doi' href='https://doi.org/10.1128/AAC.49.1.454-456.2005'>10.1128/AAC.49.1.454-456.2005</a></div></div></td></tr><tr><td><a href='/doi/10.1089/fpd.2006.0066' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>10.</div></td><td><div class='doi-body' data-key='10.1089/fpd.2006.0066'><a href='/doi/10.1089/fpd.2006.0066' class='pub-title'>Antibiotic Resistance in Bacteria Associated with Food Animals: A United States Perspective of Livestock Production</a><div class='pub-body'>Alan G. Mathew, Robin Cissell, S. Liamthong (2007) <i>Foodborne Pathogens and Disease</i>. doi:<a class='pub-doi' href='https://doi.org/10.1089/fpd.2006.0066'>10.1089/fpd.2006.0066</a></div></div></td></tr><tr><td><a href='/doi/10.1017/S0022172400064597' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>11.</div></td><td><div class='doi-body' data-key='10.1017/S0022172400064597'><a href='/doi/10.1017/S0022172400064597' class='pub-title'>Distribution of multilocus genotypes of Escherichia coli within and between host families</a><div class='pub-body'>Dominique A. Caugant, B. R. Levin, R. K. Selander (1984) <i>Journal of Hygiene</i>. doi:<a class='pub-doi' href='https://doi.org/10.1017/S0022172400064597'>10.1017/S0022172400064597</a></div></div></td></tr><tr><td><a href='/doi/10.7554/eLife.00458' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>12.</div></td><td><div class='doi-body' data-key='10.7554/eLife.00458'><a href='/doi/10.7554/eLife.00458' class='pub-title'>Cohabiting family members share microbiota with one another and with their dogs</a><div class='pub-body'>Se Jin Song, Christian Lauber, Elizabeth K Costello, Catherine A Lozupone, Gregory Humphrey, Donna Berg-Lyons, J Gregory Caporaso, Dan Knights, Jose C Clemente, Sara Nakielny, Jeffrey I Gordon, Noah Fierer, Rob Knight (2013) <i>eLife</i>. doi:<a class='pub-doi' href='https://doi.org/10.7554/eLife.00458'>10.7554/eLife.00458</a></div></div></td></tr><tr><td><a href='/doi/10.1111/j.1365-2222.1994.tb03323.x' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>13.</div></td><td><div class='doi-body' data-key='10.1111/j.1365-2222.1994.tb03323.x'><a href='/doi/10.1111/j.1365-2222.1994.tb03323.x' class='pub-title'>House dust mite and cat allergen in different indoor environments</a><div class='pub-body'>A. CUSTOVIC, S. C. O. TAGGART, A. WOODCOCK (1994) <i>Clinical &amp; Experimental Allergy</i>. doi:<a class='pub-doi' href='https://doi.org/10.1111/j.1365-2222.1994.tb03323.x'>10.1111/j.1365-2222.1994.tb03323.x</a></div></div></td></tr><tr><td><a href='/doi/10.1016/j.jaci.2010.05.042' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>14.</div></td><td><div class='doi-body' data-key='10.1016/j.jaci.2010.05.042'><a href='/doi/10.1016/j.jaci.2010.05.042' class='pub-title'>Man&#39;s best friend? The effect of pet ownership on house dust microbial communities</a><div class='pub-body'>Kei E. Fujimura, Christine C. Johnson, Dennis R. Ownby, Michael J. Cox, Eoin L. Brodie, Suzanne L. Havstad, Edward M. Zoratti, Kimberley J. Woodcroft, Kevin R. Bobbitt, Ganesa Wegienka, Homer A. Boushey, Susan V. Lynch (2010) <i>Journal of Allergy and Clinical Immunology</i>. doi:<a class='pub-doi' href='https://doi.org/10.1016/j.jaci.2010.05.042'>10.1016/j.jaci.2010.05.042</a></div></div></td></tr><tr><td><a href='/doi/10.1186/gb-2013-14-2-202' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>15.</div></td><td><div class='doi-body' data-key='10.1186/gb-2013-14-2-202'><a href='/doi/10.1186/gb-2013-14-2-202' class='pub-title'>Studying the microbiology of the indoor environment</a><div class='pub-body'>Scott T Kelley, Jack A Gilbert (2013) <i>Genome Biol</i>. doi:<a class='pub-doi' href='https://doi.org/10.1186/gb-2013-14-2-202'>10.1186/gb-2013-14-2-202</a></div></div></td></tr><tr><td><a href='/doi/10.1038/nmeth.f.303' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>16.</div></td><td><div class='doi-body' data-key='10.1038/nmeth.f.303'><a href='/doi/10.1038/nmeth.f.303' class='pub-title'>QIIME allows analysis of high-throughput community sequencing data</a><div class='pub-body'>J Gregory Caporaso, Justin Kuczynski, Jesse Stombaugh, Kyle Bittinger, Frederic D Bushman, Elizabeth K Costello, Noah Fierer, Antonio Gonzalez Peña, Julia K Goodrich, Jeffrey I Gordon, Gavin A Huttley, Scott T Kelley, Dan Knights, Jeremy E Koenig, Ruth E Ley, Catherine A Lozupone, Daniel McDonald, Brian D Muegge, Meg Pirrung, Jens Reeder, Joel R Sevinsky, Peter J Turnbaugh, William A Walters, Jeremy Widmann, Tanya Yatsunenko, Jesse Zaneveld, Rob Knight (2010) <i>Nat Meth</i>. doi:<a class='pub-doi' href='https://doi.org/10.1038/nmeth.f.303'>10.1038/nmeth.f.303</a></div></div></td></tr><tr><td><a href='/doi/10.1073/pnas.1117018109' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>17.</div></td><td><div class='doi-body' data-key='10.1073/pnas.1117018109'><a href='/doi/10.1073/pnas.1117018109' class='pub-title'>Nuclear ribosomal internal transcribed spacer (ITS) region as a universal DNA barcode marker for Fungi</a><div class='pub-body'>C. L. Schoch, K. A. Seifert, S. Huhndorf, V. Robert, J. L. Spouge, C. A. Levesque, W. Chen, E. Bolchacova, K. Voigt, P. W. Crous, A. N. Miller, M. J. Wingfield, M. C. Aime, K.-D. An, F.-Y. Bai, R. W. Barreto, D. Begerow, M.-J. Bergeron, M. Blackwell, T. Boekhout, M. Bogale, N. Boonyuen, A. R. Burgaz, B. Buyck, L. Cai, Q. Cai, G. Cardinali, P. Chaverri, B. J. Coppins, A. Crespo, P. Cubas, C. Cummings, U. Damm, Z. W. de Beer, G. S. de Hoog, R. Del-Prado, B. Dentinger, J. Dieguez-Uribeondo, P. K. Divakar, B. Douglas, M. Duenas, T. A. Duong, U. Eberhardt, J. E. Edwards, M. S. Elshahed, K. Fliegerova, M. Furtado, M. A. Garcia, Z.-W. Ge, G. W. Griffith, K. Griffiths, J. Z. Groenewald, M. Groenewald, M. Grube, M. Gryzenhout, L.-D. Guo, F. Hagen, S. Hambleton, R. C. Hamelin, K. Hansen, P. Harrold, G. Heller, C. Herrera, K. Hirayama, Y. Hirooka, H.-M. Ho, K. Hoffmann, V. Hofstetter, F. Hognabba, P. M. Hollingsworth, S.-B. Hong, K. Hosaka, J. Houbraken, K. Hughes, S. Huhtinen, K. D. Hyde, T. James, E. M. Johnson, J. E. Johnson, P. R. Johnston, E. B. G. Jones, L. J. Kelly, P. M. Kirk, D. G. Knapp, U. Koljalg, G. M. Kovacs, C. P. Kurtzman, S. Landvik, S. D. Leavitt, A. S. Liggenstoffer, K. Liimatainen, L. Lombard, J. J. Luangsa-ard, H. T. Lumbsch, H. Maganti, S. S. N. Maharachchikumbura, M. P. Martin, T. W. May, A. R. McTaggart, A. S. Methven, W. Meyer, J.-M. Moncalvo, S. Mongkolsamrit, L. G. Nagy, R. H. Nilsson, T. Niskanen, I. Nyilasi, G. Okada, I. Okane, I. Olariaga, J. Otte, T. Papp, D. Park, T. Petkovits, R. Pino-Bodas, W. Quaedvlieg, H. A. Raja, D. Redecker, T. L. Rintoul, C. Ruibal, J. M. Sarmiento-Ramirez, I. Schmitt, A. Schussler, C. Shearer, K. Sotome, F. O. P. Stefani, S. Stenroos, B. Stielow, H. Stockinger, S. Suetrong, S.-O. Suh, G.-H. Sung, M. Suzuki, K. Tanaka, L. Tedersoo, M. T. Telleria, E. Tretter, W. A. Untereiner, H. Urbina, C. Vagvolgyi, A. Vialle, T. D. Vu, G. Walther, Q.-M. Wang, Y. Wang, B. S. Weir, M. Weiss, M. M. White, J. Xu, R. Yahr, Z. L. Yang, A. Yurkov, J.-C. Zamora, N. Zhang, W.-Y. Zhuang, D. Schindel (2012) <i>Proceedings of the National Academy of Sciences</i>. doi:<a class='pub-doi' href='https://doi.org/10.1073/pnas.1117018109'>10.1073/pnas.1117018109</a></div></div></td></tr><tr><td><a href='/doi/10.1038/ismej.2009.97' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>18.</div></td><td><div class='doi-body' data-key='10.1038/ismej.2009.97'><a href='/doi/10.1038/ismej.2009.97' class='pub-title'>Fast UniFrac: facilitating high-throughput phylogenetic analyses of microbial communities including analysis of pyrosequencing and PhyloChip data</a><div class='pub-body'>Micah Hamady, Catherine Lozupone, Rob Knight (2009) <i>The ISME Journal</i>. doi:<a class='pub-doi' href='https://doi.org/10.1038/ismej.2009.97'>10.1038/ismej.2009.97</a></div></div></td></tr><tr><td><a href='/doi/10.1038/nmeth.1650' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>19.</div></td><td><div class='doi-body' data-key='10.1038/nmeth.1650'><a href='/doi/10.1038/nmeth.1650' class='pub-title'>Bayesian community-wide culture-independent microbial source tracking</a><div class='pub-body'>Dan Knights, Justin Kuczynski, Emily S Charlson, Jesse Zaneveld, Michael C Mozer, Ronald G Collman, Frederic D Bushman, Rob Knight, Scott T Kelley (2011) <i>Nat Meth</i>. doi:<a class='pub-doi' href='https://doi.org/10.1038/nmeth.1650'>10.1038/nmeth.1650</a></div></div></td></tr><tr><td><a href='/doi/10.7717/peerj.243' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>20.</div></td><td><div class='doi-body' data-key='10.7717/peerj.243'><a href='/doi/10.7717/peerj.243' class='pub-title'>PhyloSift: phylogenetic analysis of genomes and metagenomes</a><div class='pub-body'>Aaron E. Darling, Guillaume Jospin, Eric Lowe, Frederick A. Matsen, Holly M. Bik, Jonathan A. Eisen (2014) <i>PeerJ</i>. doi:<a class='pub-doi' href='https://doi.org/10.7717/peerj.243'>10.7717/peerj.243</a></div></div></td></tr><tr><td><a href='/doi/10.1093/nar/gkt1252' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>21.</div></td><td><div class='doi-body' data-key='10.1093/nar/gkt1252'><a href='/doi/10.1093/nar/gkt1252' class='pub-title'>BacMet: antibacterial biocide and metal resistance genes database</a><div class='pub-body'>C. Pal, J. Bengtsson-Palme, C. Rensing, E. Kristiansson, D. G. J. Larsson (2013) <i>Nucleic Acids Research</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/nar/gkt1252'>10.1093/nar/gkt1252</a></div></div></td></tr></table>
  
  
</div>


		  </td>
	    <td class="sidebar project-sidebar">

  			



	








	<div class="details">Published</div>
	
		June 1, 2014<br>
	
	<span style="font-size:12px;">(Last updated Jan. 13, 2016)</span><br>

	<a style="font-size:12px;" href="/doc/4/revisions">Revision history</a>




<div class="details">Views</div>
<span style="font-size:20px;">565</span>

<div class="details">Topics </div>
<a class='tag' href='/topics/disinfectant-resistance'>Disinfectant Resistance</a><a class='tag' href='/topics/fungi'>Fungi</a><a class='tag' href='/topics/metagenomics'>Metagenomics</a><a class='tag' href='/topics/bacteria'>Bacteria</a><a class='tag' href='/topics/biofilm-communities'>Biofilm Communities</a><a class='tag' href='/topics/antibiotic-resistance'>Antibiotic Resistance</a>


<div class="details">Cite this as</div>
<span style="font-size:12px;">
Holly Ganz, Jonathan Eisen, Leesa Li, Ruth Lee, David Coil, Katie Dahlhausen, Holly Bik (2014) Microbial resistance to disinfectants in animal shelters [proposal]. <i>Thinklab.</i> doi:<a href='https://doi.org/10.15363/thinklab.a4'>10.15363/thinklab.a4</a>
</span>

<div class="details">License</div>
<p><a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0;width:88px;height:31px;" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a></p>


<br><br>

<div class="affix-container">
<div class="affix-box">


    <!-- <h3>Contents</h3> -->

	<div id="document_nav">
    <ul class="nav document-menu">
        
            <li class="level-1 active"><a href="#abstract">Abstract</a></li>
        
            <li class="level-1"><a href="#objectives">Objectives</a></li>
        
            <li class="level-1"><a href="#background">Background</a></li>
        
            <li class="level-1"><a href="#research-plan">Research Plan</a></li>
        
            <li class="level-1"><a href="#team-resources">Team &amp; Resources</a></li>
        
            <li class="level-"><a href="#budget-items">Budget Items</a></li>
        
            <li class="level-"><a href="#references">References</a></li>
        
    </ul>
	</div>



</div></div>


	    </td>
	  </tr></table>


		</div>


		
	    	<div class="footer">
<div style="background:#dedede;">
	<div class="main-centered-div text-center" style="padding:20px;">
		<h4 style="margin-bottom:0px;font-weight:bold;color:gray;">Feedback? Questions? Email us: <a href="/cdn-cgi/l/email-protection#d1b9b4bdbdbe91a5b9b8bfbabdb0b3ffb2bebc"><span class="__cf_email__" data-cfemail="98f0fdf4f4f7d8ecf0f1f6f3f4f9fab6fbf7f5">[email&#160;protected]</span><script data-cfhash='f9e31' type="text/javascript">/* <![CDATA[ */!function(t,e,r,n,c,a,p){try{t=document.currentScript||function(){for(t=document.getElementsByTagName('script'),e=t.length;e--;)if(t[e].getAttribute('data-cfhash'))return t[e]}();if(t&&(c=t.previousSibling)){p=t.parentNode;if(a=c.getAttribute('data-cfemail')){for(e='',r='0x'+a.substr(0,2)|0,n=2;a.length-n;n+=2)e+='%'+('0'+('0x'+a.substr(n,2)^r).toString(16)).slice(-2);p.replaceChild(document.createTextNode(decodeURIComponent(e)),c)}p.removeChild(t)}}catch(u){}}()/* ]]> */</script></a></h4>
	</div>
</div>
<div class="main-footer">
	<div class="main-centered-div">

<table class="full-width"><tr>
	<td class="footer-nav">

        <div class="foot-head">Explore</div>
        <a href="/proposals">Proposals</a><br>
        <a href="/projects">Projects</a><br>
        <a href="/publications">Article discussion</a><br>
        <a href="/discussion">All discussion</a><br>
        <a href="/topics">Topics</a><br>
        <!-- <a href="/funders">Funders</a><br> -->
        <a href="/funding">Funding opportunities</a><br>

	</td>
	<td class="footer-nav">

        <!-- <div class="foot-head">Help &amp; feedback</div>	 -->
        <!-- <div class="foot-head">Intro</div>
        <a href="/benefits/for-science">Why</a><br>
        <a href="/how-it-works">How it works</a><br>

<br> -->

        <div class="foot-head">Community</div>
        <a href="/leaderboard">Leaderboard</a><br>
        <a href="/p/meta">Thinklab Meta</a><br>
				<a href="/help/faq">Help</a><br>
				<!-- <a href="/new-project">Start a proposal</a><br> -->
        <!-- <a href="/benefits/proposal">Benefits</a><br> -->
        <!-- <a href="/funding">Get funded</a><br> -->
        <!-- <a href="/participate">Become a contributor</a><br> -->
        <!-- <a href="/for-funders">For science funders</a><br> -->
        <!-- <a href="/for-institutions">For institutions</a><br> -->

	</td>
	<td class="footer-nav">



        <div class="foot-head">About</div>
        <a href="/about">Overview</a><br>
        <a href="/benefits">Benefits</a><br>
        <a href="/how-it-works">How it works</a><br>
        <a href="/about/story">Our story</a><br>
        <a href="/blog">Our blog</a><br>
        <a href="/about/jobs">Join the team</a><br>
        <br>

        <!-- <div class="foot-head">Help</div>
        <a href="/help">Help</a><br>
		 -->


	</td>
	<td class="footer-nav">
		<a href="http://twitter.com/thinklab" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/twitter_com.png" width="40"/></a>
		<a href="https://www.facebook.com/pages/Thinklab/310895039088185" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/facebook_com.png" width="40"/></a>
		<a href="https://plus.google.com/115578983466800753402/" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/plus_google_com.png" width="40"/></a>
		<a href="https://www.linkedin.com/company/thinklab-co" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/linkedin_com.png" width="40"/></a>
		<br><br>
		<span class="light" style='font-size:12px;'>&copy; 2015 Thinklab</span>

	</td>

</tr></table>

</div>
</div>
</div>

		

	</div>


  <script></script>
  <script>
  var MEDIA_URL = 'https://think-lab.s3.amazonaws.com/m/';
  var STATIC_URL = 'https://think-lab.s3.amazonaws.com/s/';
  var page_is_authenticated = false;
  var page_page = 'document_home';
  var page_profile_id = undefined;
</script>



	<!-- Note different script type. MUST come before MathJax.js -->
	<script type="text/x-mathjax-config">
	    MathJax.Hub.Config({
	        "HTML-CSS": { preferredFont: "TeX", availableFonts: ["STIX","TeX"],
	                        scale: 110, linebreaks: { automatic:true },
	                        EqnChunk: (MathJax.Hub.Browser.isMobile ? 10 : 50) },
	        tex2jax: { inlineMath: [ ["$$", "$$"], ["\\(","\\)"] ],
	                    displayMath: [ ["$$$","$$$"], ["\\[", "\\]"] ],
	                    processEscapes: true,}
	            });

	</script>
    <script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>





  <script src="https://code.jquery.com/jquery-1.10.2.min.js"></script>
  <script src="https://code.jquery.com/ui/1.11.4/jquery-ui.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" integrity="sha384-0mSbJDEHialfmuBBQP6A4Qrprq5OVfW37PRR3j5ELqxss1yVqOtnepnHVP9aJ7xS" crossorigin="anonymous"></script>
  <script src="https://think-lab.s3.amazonaws.com/s/dist/main.min.js"></script>
  



  
	

    
    
<script>
$(function() {
  recordHit('document', 4);
});
</script>


    


<script src="https://think-lab.s3.amazonaws.com/s/js/Chart.min.js"></script>
<script src="https://think-lab.s3.amazonaws.com/s/js/charts.js"></script>
<script>
	init_doughnut('budget-doughnut', [{"value": 550, "label": "MoBio PowerSoil DNA extraction kit (200 preps)"}, {"value": 175, "label": "PCR reagents"}, {"value": 875, "label": "Nextera XT library preparation kit"}, {"value": 2400, "label": "llumina MiSeq PE 250 2 runs with QC"}], 'budget-doughnut-legend');
</script>


    


<script>/* <![CDATA[ */(function(d,s,a,i,j,r,l,m,t){try{l=d.getElementsByTagName('a');t=d.createElement('textarea');for(i=0;l.length-i;i++){try{a=l[i].href;s=a.indexOf('/cdn-cgi/l/email-protection');m=a.length;if(a&&s>-1&&m>28){j=28+s;s='';if(j<m){r='0x'+a.substr(j,2)|0;for(j+=2;j<m&&a.charAt(j)!='X';j+=2)s+='%'+('0'+('0x'+a.substr(j,2)^r).toString(16)).slice(-2);j++;s=decodeURIComponent(s)+a.substr(j,m-j)}t.innerHTML=s.replace(/</g,'&lt;').replace(/\>/g,'&gt;');l[i].href='mailto:'+t.value}}catch(e){}}}catch(e){}})(document);/* ]]> */</script></body>
</html>
