<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8" /><script type="text/javascript">window.NREUM||(NREUM={}),__nr_require=function(e,n,t){function r(t){if(!n[t]){var o=n[t]={exports:{}};e[t][0].call(o.exports,function(n){var o=e[t][1][n];return r(o||n)},o,o.exports)}return n[t].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<t.length;o++)r(t[o]);return r}({1:[function(e,n,t){function r(){}function o(e,n,t){return function(){return i(e,[c.now()].concat(u(arguments)),n?null:this,t),n?void 0:this}}var i=e("handle"),a=e(2),u=e(3),f=e("ee").get("tracer"),c=e("loader"),s=NREUM;"undefined"==typeof window.newrelic&&(newrelic=s);var p=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],d="api-",l=d+"ixn-";a(p,function(e,n){s[n]=o(d+n,!0,"api")}),s.addPageAction=o(d+"addPageAction",!0),s.setCurrentRouteName=o(d+"routeName",!0),n.exports=newrelic,s.interaction=function(){return(new r).get()};var m=r.prototype={createTracer:function(e,n){var t={},r=this,o="function"==typeof n;return i(l+"tracer",[c.now(),e,t],r),function(){if(f.emit((o?"":"no-")+"fn-start",[c.now(),r,o],t),o)try{return n.apply(this,arguments)}finally{f.emit("fn-end",[c.now()],t)}}}};a("setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(e,n){m[n]=o(l+n)}),newrelic.noticeError=function(e){"string"==typeof e&&(e=new Error(e)),i("err",[e,c.now()])}},{}],2:[function(e,n,t){function r(e,n){var t=[],r="",i=0;for(r in e)o.call(e,r)&&(t[i]=n(r,e[r]),i+=1);return t}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],3:[function(e,n,t){function r(e,n,t){n||(n=0),"undefined"==typeof t&&(t=e?e.length:0);for(var r=-1,o=t-n||0,i=Array(o<0?0:o);++r<o;)i[r]=e[n+r];return i}n.exports=r},{}],4:[function(e,n,t){n.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(e,n,t){function r(){}function o(e){function n(e){return e&&e instanceof r?e:e?f(e,u,i):i()}function t(t,r,o,i){if(!d.aborted||i){e&&e(t,r,o);for(var a=n(o),u=m(t),f=u.length,c=0;c<f;c++)u[c].apply(a,r);var p=s[y[t]];return p&&p.push([b,t,r,a]),a}}function l(e,n){v[e]=m(e).concat(n)}function m(e){return v[e]||[]}function w(e){return p[e]=p[e]||o(t)}function g(e,n){c(e,function(e,t){n=n||"feature",y[t]=n,n in s||(s[n]=[])})}var v={},y={},b={on:l,emit:t,get:w,listeners:m,context:n,buffer:g,abort:a,aborted:!1};return b}function i(){return new r}function a(){(s.api||s.feature)&&(d.aborted=!0,s=d.backlog={})}var u="nr@context",f=e("gos"),c=e(2),s={},p={},d=n.exports=o();d.backlog=s},{}],gos:[function(e,n,t){function r(e,n,t){if(o.call(e,n))return e[n];var r=t();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(e,n,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return e[n]=r,r}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],handle:[function(e,n,t){function r(e,n,t,r){o.buffer([e],r),o.emit(e,n,t)}var o=e("ee").get("handle");n.exports=r,r.ee=o},{}],id:[function(e,n,t){function r(e){var n=typeof e;return!e||"object"!==n&&"function"!==n?-1:e===window?0:a(e,i,function(){return o++})}var o=1,i="nr@id",a=e("gos");n.exports=r},{}],loader:[function(e,n,t){function r(){if(!x++){var e=h.info=NREUM.info,n=d.getElementsByTagName("script")[0];if(setTimeout(s.abort,3e4),!(e&&e.licenseKey&&e.applicationID&&n))return s.abort();c(y,function(n,t){e[n]||(e[n]=t)}),f("mark",["onload",a()+h.offset],null,"api");var t=d.createElement("script");t.src="https://"+e.agent,n.parentNode.insertBefore(t,n)}}function o(){"complete"===d.readyState&&i()}function i(){f("mark",["domContent",a()+h.offset],null,"api")}function a(){return E.exists&&performance.now?Math.round(performance.now()):(u=Math.max((new Date).getTime(),u))-h.offset}var u=(new Date).getTime(),f=e("handle"),c=e(2),s=e("ee"),p=window,d=p.document,l="addEventListener",m="attachEvent",w=p.XMLHttpRequest,g=w&&w.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:w,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var v=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-1039.min.js"},b=w&&g&&g[l]&&!/CriOS/.test(navigator.userAgent),h=n.exports={offset:u,now:a,origin:v,features:{},xhrWrappable:b};e(1),d[l]?(d[l]("DOMContentLoaded",i,!1),p[l]("load",r,!1)):(d[m]("onreadystatechange",o),p[m]("onload",r)),f("mark",["firstbyte",u],null,"api");var x=0,E=e(4)},{}]},{},["loader"]);</script><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info={"beacon":"bam.nr-data.net","queueTime":0,"licenseKey":"cd1101aa94","agent":"","transactionName":"bgcDMkBZW0NZBkdcWVdNJxNcW0FZVwscV1dKB08QW11CQxYTWlBBZgEOC19dW0QCEVtHU1gG","applicationID":"6369297","errorBeacon":"bam.nr-data.net","applicationTime":320}</script>
    <title>Data nomenclature: naming and abbreviating our network types | Thinklab</title>
	  <link rel="shortcut icon" href="https://think-lab.s3.amazonaws.com/s/img/favicon.ico" type="image/x-icon" />
    <meta property="og:image" content="https://think-lab.s3.amazonaws.com/s/img" />
    
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
  <link rel="stylesheet" href="https://code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css" />

<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/highlightjs/github.css" />
<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/css/css.css" />
<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/css/bootstrap-multiselect.css" />
<!-- <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'> -->

    
    
    
    
  
    <meta name="citation_doi" content="10.15363/thinklab.d162">
    <meta name="citation_title" content="Data nomenclature: naming and abbreviating our network types">
    <meta name="citation_online_date" content="Feb. 17, 2016">
    <meta name="citation_author" content="Daniel Himmelstein, Lars Juhl Jensen, Pouya Khankhanian">
    <meta name="citation_journal_title" content="Thinklab">
    <meta name="citation_publisher" content="Thinklab">
  

    


	<!-- GoogleAnalytics -->
	<script>
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
	(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
	m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

	ga('create', 'UA-48550230-1', 'thinklab.com');
	ga('require', 'displayfeatures');
	
	ga('send', 'pageview');

	</script>






</head>
<body >
  <form id="form_action" method="post">
    <input type='hidden' name='csrfmiddlewaretoken' value='ypamUdfXu4FnASUyTHPbTrQWcdTFmVrB' />
</form>

	

	<div class="body-wrapper">

		


<!-- <div style="height:5px;background-image:url('https://think-lab.s3.amazonaws.com/s/img/top-bg.gif');"></div> -->
<div class="gradient">
	<div class="main-centered-div header-container">

<table class="full-width" style="height:60px;"><tr>
	<td style="width:200px;vertical-align:middle;padding-bottom:15px;">
		<a href="/" style="font-weight:normal;color:#212121;"><img src="https://think-lab.s3.amazonaws.com/s/img/logo.png" width="160"></a>
	</td><td style="vertical-align:middle;text-align:left;padding-top:6px;width:100%;padding-left:40px;">
		<ul id="not-search-block"  class="main-header">


			


				<li class="dropdown">
					<a class="dropdown-toggle" data-toggle="dropdown" href="#">Browse <span class="caret"></span></a>
					<ul class="dropdown-menu" role="menu">
						<li><a href="/proposals">Proposals</a></li>
						<li><a href="/projects">Projects</a></li>
						<li><a href="/publications">Article discussion</a></li>
						<li><a href="/discussion">All discussion</a></li>
					</ul>
				</li>
				<li><a href="/leaderboard">Leaderboard</a></li>

			
				<li><a href="/about">About</a></li>
			


		</ul>

    <ul id="search-block" class="main-header" style="display: none; margin-right: 30px;">
        <li class="full-width">
            <form method="get" action="/search">
			<div class="input-group full-width">
				<input name="q" type="text" class="form-control search-input">
			</div>
            </form>
        </li>
    </ul>

	</td><td style="vertical-align:middle;text-align:right;padding-top:6px; min-width: 1%; white-space: nowrap;">
		<ul class="main-header pull-right" style="padding-left: 0;">
    		
                <!-- <li><a href="/login" style="color:silver;">Login</a></li>
                <li style="margin-right:0"><a class="btn btn-default btn-sm btn-upper" href="/signup">Join</a></li> -->
            

        </ul>
    </ul>
	</td>
</tr></table>

<!-- <div id="global-search-div" class="main-centered-div" style="display:none;">
	<input type="text" value="" class="thread-search form-control" placeholder="Search" data-action="/discussion">
</div> -->

	</div>
</div>

		
	  



	  


		

		<div class="dark-head"><div class="main-centered-div">

			

		</div></div>
		
		<div class="main-centered-div" style="margin-top:40px;padding-bottom:340px">
	    

<div style="margin-top:-20px; margin-bottom:20px;">

    <a class="side-box" href="/p/rephetio" style="background-color:#ebfdff;"><div class="side-type">Project:</div> <div class="side-title">Rephetio: Repurposing drugs on a hetnet [rephetio]</div></a><br>



</div>

<table style="margin-top:0px;"><tr>
	<td class="thread-table-content">

		<table class="full-width">
			
<tr class="comment-tr target-hl" id="1" data-comment-id="802">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="802"></a>

    

			<h2 class="page-title" style="margin-bottom:5px;margin-top:0px !important;">Data nomenclature: naming and abbreviating our network types</h2>

			<div style="margin-top:5px;color:gray;margin-bottom:20px;font-size:13px;">
				<a style="font-weight:normal;color:#212121" href="/u/dhimmel"><strong>Daniel Himmelstein</strong></a>
				<span class="is-researcher">Researcher</span>
				&nbsp; <a href="#1" class="light">Feb. 17, 2016</a>
				
			</div>

      

          <hr>
		

    <p>We've created a <a href="http://thinklab.com/discussion/one-network-to-rule-them-all/102" title="Preliminary hetnet release">preliminary network</a> with 10 types of nodes (metanodes) and 27 types of edges (metaedges). Now an important detail is naming node and edge types appropriately.</p>

<p>For each metanode and metaedge, we also need abbreviations. We use the abbreviations to make writing out complete paths less cumbersome. For example, in our previous project, we <a href="https://doi.org/10.1371/journal.pcbi.1004259.s010" title="S1 Table. Features · Hetnet-Based Prioritization of Disease Associations">abbreviated</a> the <code>Gene - interaction - Gene - expression - Tissue - localization - Disease</code> path to <code>GiGeTlD</code> <span class="citation">[<a href="https://doi.org/10.1371/journal.pcbi.1004259.s010" class="citation" data-key="10.1371/journal.pcbi.1004259.s010">1</a>]</span>.</p>

<p>We have several conventions for naming and abbreviations, but they haven't been publicly explained or discussed. This discussion is now home to these topics.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="2" data-comment-id="804">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="804"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#2" class="light">Feb. 17, 2016</a>
          
        </span>
      </div>

    

    <h1>Naming according to parts of speech</h1>

<p>According to <a href="https://en.wikipedia.org/w/index.php?title=Entity%E2%80%93relationship_model&amp;oldid=704204795#Mapping_natural_language" title="Mapping natural language · Entity–relationship model · Wikipedia">Chen's rules of thumb</a>, we should use parts of speech as follows <span class="citation">[<a href="https://doi.org/10.1016/s0169-023x(97)00017-7" class="citation" data-key="10.1016/s0169-023x(97)00017-7">1</a>]</span>:</p>

<ul><li><em>common nouns</em> for node labels (types)</li><li><em>proper nouns</em> for node names</li><li><em>transitive verbs</em> for relationship (edge) types</li><li><em>intransitive verbs</em> for property (attribute) types</li><li><em>adjectives</em> for node properties</li><li><em>adverbs</em> for relationship properties</li></ul>

<p>I'm not convinced about the last three, since our properties (data attributes for nodes and relationships) are often highly technical. However, I think we should adhere to the first three rules when possible.</p>

<p>Our node labels are already common nouns. Our node names are already proper nouns. However, we were using common nouns for relationship types. Thus, I switched to transitive verbs for relationship types (<a href="https://github.com/dhimmel/integrate/commit/8ca7c9e971ce5a85c7729b3f1df7db54beb19d18">commit</a>). The table below shows the noun (old) and verb (new) relationship type.</p>

<table class="table markdown-table"><thead><tr><th>Source</th><th>Target</th><th>Metaedge (noun)</th><th>Metaedge (verb)</th></tr></thead><tbody><tr><td>compound</td><td>gene</td><td>binding</td><td>binds</td></tr><tr><td>compound</td><td>side effect</td><td>causation</td><td>causes</td></tr><tr><td>compound</td><td>gene</td><td>downregulation</td><td>downregulates</td></tr><tr><td>compound</td><td>disease</td><td>indication</td><td>palliates</td></tr><tr><td>compound</td><td>compound</td><td>similarity</td><td>resembles</td></tr><tr><td>compound</td><td>disease</td><td>indication</td><td>treats</td></tr><tr><td>compound</td><td>gene</td><td>upregulation</td><td>upregulates</td></tr><tr><td>disease</td><td>gene</td><td>association</td><td>associates</td></tr><tr><td>disease</td><td>gene</td><td>downregulation</td><td>downregulates</td></tr><tr><td>disease</td><td>anatomy</td><td>localization</td><td>localizes</td></tr><tr><td>disease</td><td>symptom</td><td>presence</td><td>presents</td></tr><tr><td>disease</td><td>disease</td><td>similarity</td><td>resembles</td></tr><tr><td>disease</td><td>gene</td><td>upregulation</td><td>upregulates</td></tr><tr><td>gene</td><td>anatomy</td><td>downregulation</td><td>downregulates</td></tr><tr><td>gene</td><td>gene</td><td>evolution</td><td>evolves</td></tr><tr><td>gene</td><td>anatomy</td><td>expression</td><td>expresses</td></tr><tr><td>gene</td><td>gene</td><td>interaction</td><td>interacts</td></tr><tr><td>gene</td><td>biological process</td><td>participation</td><td>participates</td></tr><tr><td>gene</td><td>cellular component</td><td>participation</td><td>participates</td></tr><tr><td>gene</td><td>molecular function</td><td>participation</td><td>participates</td></tr><tr><td>gene</td><td>pathway</td><td>participation</td><td>participates</td></tr><tr><td>gene</td><td>perturbation</td><td>regulation</td><td>regulates</td></tr><tr><td>gene</td><td>anatomy</td><td>upregulation</td><td>upregulates</td></tr><tr><td>gene</td><td>gene</td><td>knockdown downregulation</td><td>knockdown downregulates</td></tr><tr><td>gene</td><td>gene</td><td>knockdown upregulation</td><td>knockdown upregulates</td></tr><tr><td>gene</td><td>gene</td><td>overexpression downregulation</td><td>overexpression downregulates</td></tr><tr><td>gene</td><td>gene</td><td>overexpression upregulation</td><td>overexpression upregulates</td></tr></tbody></table>

<p>In several cases, switching from noun to verb cut out several characters — a welcome occurrence. Switching relationship types to verbs also makes sense as part of our <a href="http://thinklab.com/discussion/using-the-neo4j-graph-database-for-hetnets/112" class="citation thread-citation" data-key="10.15363/thinklab.d112">migration to neo4j</a>. The neo4j convention is to use verbs for relationship types. In fact, a neo4j company <a href="https://www.graphstory.com/elements-of-a-graph-database">explains relationships</a> by saying:</p>

<blockquote><p>Where nodes can be thought of as nouns, relationships can be thought of as verbs.</p></blockquote>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="3" data-comment-id="814">
  <td class="comment-profile-avatar">
		<a href="/u/larsjuhljensen"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/125_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="814"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/larsjuhljensen">Lars Juhl Jensen</a>
				
				<span class="light">&nbsp;<a href="#3" class="light">Feb. 18, 2016</a>
          
        </span>
      </div>

    

    <p>The compound-gene associations are not intuitive to me. I assume that when, for example, a compound downregulates a gene, it is supposed to mean that the compound inhibits the protein product encoded by the gene. However, if read at face value, it would mean that the compound binds to something else that through some signaling results in down-regulation of the gene (i.e. less transcription).</p>

<p>The gene-gene association "evolves" is bit of a misnomer, I think. Unless you are looking at ancestral genes, one gene will not have evolved from another gene. Rather two genes will share ancestry. In that case, the term "homology" is would be much clearer. Also, you probably want to be able to distinguish between orthologs and paralogs in your network.</p>

<p>Are the gene-anatomy relationships not backwards? I can understand what it means that means that the liver "upregulates" a gene (I assume it means that the gene is higher expressed in the liver than elsewhere). But I cannot comprehend what it would mean that a gene upregulates the liver.</p>

<p>Same goes for gene-pertubation relationships. I can understand that a pertubation regulates a gene, but how can a gene regulate a pertubation? And why is this type of association not divided into up- and down-regulation like everything else?</p>

<p>I am not entirely sure how useful the "knockdown downregulates" etc. types are. Usually "knockdown downregulates" would be interpreted to mean "upregulates" etc.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="4" data-comment-id="815">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="815"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#4" class="light">Feb. 18, 2016</a>
          
        </span>
      </div>

    

    <blockquote><p>I assume that when, for example, a compound downregulates a gene, it is supposed to mean that the compound inhibits the protein product encoded by the gene. However, if read at face value, it would mean that the compound binds to something else that through some signaling results in down-regulation of the gene (i.e. less transcription).</p></blockquote>

<p>Your face value interpretation is correct. Compound–downregulates–Gene means the compound decreases the transcriptional expression of the gene. We <a href="http://thinklab.com/discussion/computing-consensus-transcriptional-profiles-for-compounds-using-lincs/43#6" class="citation thread-citation" data-key="10.15363/thinklab.d43">extracted these relationships</a> from LINCS L1000.</p>

<blockquote><p>The gene-gene association "evolves" is bit of a misnomer</p></blockquote>

<p>I agree, "evolves" is not good. This edge signifies <a href="http://thinklab.com/discussion/selecting-informative-erc-evolutionary-rate-covariation-values-between-genes/57" class="citation thread-citation" data-key="10.15363/thinklab.d57">evolutionary rate covariation</a> <span class="citation">[<a href="https://doi.org/10.1371/journal.pgen.1004967" class="citation" data-key="10.1371/journal.pgen.1004967">1</a>]</span>. It's a mouthful, and I don't know the best way to shorten and verbify it. Perhaps "covaries" is an improvement?</p>

<blockquote><p>Are the gene-anatomy relationships not backwards? … Same goes for gene-pertubation relationships.</p></blockquote>

<p>Great point. We should present these edges in subject-verb-object order. I have switched the default orientation of the confusing metaedges (<a href="https://github.com/dhimmel/integrate/commit/3354a4cbb36d184f46e78831fa0f605ff92e7637)" title="GitHub · dhimmel/integrate @ 3354a4cbb36d184f46e78831fa0f605ff92e7637">commit</a>). In practice the object-verb-subject order may still arise, for example when representing paths.</p>

<blockquote><p>I am not entirely sure how useful the "knockdown downregulates" etc. types are. Usually "knockdown downregulates" would be interpreted to mean "upregulates" etc.</p></blockquote>

<p>I will look into collapsing:</p>

<ul><li><em>knockdown downregulates</em> with <em>overexpression upregulates</em> to create an <em>upregulates</em> edge</li><li><em>knockdown upregulates</em> with <em>overexpression downregulates</em> to create a <em>downregulates</em> edge</li></ul>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="5" data-comment-id="1004">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="1004"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#5" class="light">Feb. 22, 2016</a>
          
        </span>
      </div>

    

    <h1>Indication terminology</h1>

<p>We've been referred to when a drug treats a disease as an "<a href="https://en.wikipedia.org/w/index.php?title=Indication_(medicine)&amp;oldid=703054912" title="Wikipedia · Indication (medicine)">indication</a>". While readers with a medical background understand the term, others find "indication" confusing. </p>

<p>Now <a href="http://thinklab.com/discussion/expert-curation-of-our-indication-catalog-for-disease-modifying-treatments/95" class="citation thread-citation" data-key="10.15363/thinklab.d95">we've split our indications</a> into two categories: disease-modifying and symptomatic. Additionally, we've switched to <a href="#2">using verbs</a> to describe relationships. </p>

<p>Given these factors, I chose "treats" for disease-modifying indications and "palliates" for symptomatic indications. This terminology aligns with a recent repurposing study <span class="citation">[<a href="https://doi.org/10.1038/ncomms10331" class="citation" data-key="10.1038/ncomms10331">1</a>]</span>, which refers to</p>

<blockquote><p>distinguishing non-causative and palliative from causative and effective treatments</p></blockquote>

<p>While readers may not be familiar with the term palliates, it has an applicable and precise <a href="http://www.oxforddictionaries.com/us/definition/american_english/palliate">definition</a> (making lookup easier):</p>

<blockquote><p>Make (a disease or its symptoms) less severe or unpleasant without removing the cause</p></blockquote>

<p><a href="/u/pouyakhankhanian" class="username">@pouyakhankhanian</a>, do you think the treats/palliates terminology makes sense?</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="6" data-comment-id="1061">
  <td class="comment-profile-avatar">
		<a href="/u/pouyakhankhanian"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/188_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="1061"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/pouyakhankhanian">Pouya Khankhanian</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#6" class="light">Feb. 23, 2016</a>
          
        </span>
      </div>

    

    <p>I certainly agree with maintaining the terminology consistent with prior studies. I think the terms "indication and "palliates" are well defined as you describe. My only concern is the use of the word "treat" to mean "disease-modifying" as opposed to symptom management, especially since it is very common to use the phrase "treat symptoms". </p>

<p>If there are other prior studies that use alternate terminology, it might be best to align with those. Otherwise, I would think the two goals are (1) maintain previous terminology and (2) make sure to define our terminology very clearly.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="7" data-comment-id="1093">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="1093"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#7" class="light">Feb. 24, 2016</a>
          
        </span>
      </div>

    

    <p>I'm not sure the phrase "drug X treats symptom Y" is that problematic, since symptom Y is the sentence's subject rather than a disease. I agree that we should maintain existing terminology, but I'm not finding much guidance in the literature.</p>

<p>Potential alternatives to "treats" for representing disease-modifying indications are: modifies, medicates, indicates, remedies, ameliorates, betters, improves, corrects, affects, alleviates, repairs, and cures. <a href="/u/pouyakhankhanian" class="username">@pouyakhankhanian</a>, do you prefer any of these verbs to "treats"?</p>

<p>And regardless of which term we pick, we'll make sure to define each relationship type.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="8" data-comment-id="1280">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="1280"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#8" class="light">April 18, 2016</a>
          
        </span>
      </div>

    

    <h1>Hetionet v1.0 type nomenclature</h1>

<p>We've settled on a final type nomenclature for Hetionet v1.0 (our hetnet for this project). See the following tables:</p>

<ul><li><a href="https://github.com/dhimmel/hetionet/blob/31e6cb3162dee8a06085709e730380b52278e32a/hetnet/neo4j/labels.tsv" title="Hetionet v1.0 metanode table"><strong>Metanodes</strong></a> where <code>metanode</code> is the primary name, <code>abbreviation</code> is the 1–2 letter abbreviation, and <code>label</code> is the Neo4j node label.</li><li><a href="https://github.com/dhimmel/hetionet/blob/31e6cb3162dee8a06085709e730380b52278e32a/describe/edges/metaedge-styles.tsv" title="Hetionet v1.0 styled metaedge table"><strong>Metaedges</strong></a> where <code>metaedge</code> is the primary name, <code>unicode_metaedge</code> is a styled version of the primary name, <code>standard_metaedge</code> is the primary edge orientation, and <code>inverted</code> indicates the non-primary edge orientation. The remaining columns are <code>abbreviation</code>, <code>standard_abbreviation</code>, <code>source</code>, and <code>target</code>. </li><li><a href="https://github.com/dhimmel/hetionet/blob/31e6cb3162dee8a06085709e730380b52278e32a/hetnet/neo4j/types.tsv" title="Hetionet v1.0 Neo4j relationship type mapping"><strong>Neo4j relationship types</strong></a> where <code>metaedge</code> is the primary name, <code>rel_type</code> is Neo4j relationship type, and <code>direction</code> notes whether edges are bidirectional (<code>both</code>) or directed (<code>forward</code> or <code>backward</code>).</li></ul>

<h2>Neo4j type nomenclature</h2>

<p>We conform to the Neo4j style of CamelCase labels and ALL_CAPS relationship types. In addition, Neo4j relationship types are appended with metaedge standard abbreviations. This adds source/target-metanode awareness to relationship types and <a href="https://thinklab.com/discussion/using-the-neo4j-graph-database-for-hetnets/112#6" class="citation thread-citation" data-key="10.15363/thinklab.d112">enables optimized queries</a>.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

    		
		</table>

    

    &nbsp;<br>
    <!-- <a class="btn btn-primary btn-lg" href="/signup?next=/discussion/data-nomenclature-naming-and-abbreviating-our-network-types/162&suppress_message=1" style>Join to Reply</a> -->


		<table class="references table-td-5" style="display:none;"><tr><td><a href='/doi/10.1371/journal.pcbi.1004259.s010' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>1.</div></td><td><div class='doi-body' data-key='10.1371/journal.pcbi.1004259.s010'>None</div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d102' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>2.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d102'><a href='/d/102' class='pub-title'>One network to rule them all</a><div class='pub-body'>Daniel Himmelstein, Lars Juhl Jensen (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d102'>10.15363/thinklab.d102</a></div></div></td></tr><tr><td><a href='/doi/10.1016/s0169-023x(97)00017-7' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>3.</div></td><td><div class='doi-body' data-key='10.1016/s0169-023x(97)00017-7'><a href='/doi/10.1016/s0169-023x(97)00017-7' class='pub-title'>English, Chinese and ER diagrams</a><div class='pub-body'>Peter Pin-Shan Chen (1997) <i>Data &amp; Knowledge Engineering</i>. doi:<a class='pub-doi' href='https://doi.org/10.1016/s0169-023x(97)00017-7'>10.1016/s0169-023x(97)00017-7</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d112' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>4.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d112'><a href='/d/112' class='pub-title'>Using the neo4j graph database for hetnets</a><div class='pub-body'>Daniel Himmelstein (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d112'>10.15363/thinklab.d112</a></div></div></td></tr><tr><td><a href='/doi/10.1371/journal.pgen.1004967' class='pub-comments tt-right' title='Article comments'>3</a></td><td><div class='pub-order'>5.</div></td><td><div class='doi-body' data-key='10.1371/journal.pgen.1004967'><a href='/doi/10.1371/journal.pgen.1004967' class='pub-title'>Evolutionary Signatures amongst Disease Genes Permit Novel Methods for Gene Prioritization and Construction of Informative Gene-Based Networks</a><div class='pub-body'>Nolan Priedigkeit, Nicholas Wolfe, Nathan L. Clark (2015) <i>PLOS Genetics</i>. doi:<a class='pub-doi' href='https://doi.org/10.1371/journal.pgen.1004967'>10.1371/journal.pgen.1004967</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d43' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>6.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d43'><a href='/d/43' class='pub-title'>Computing consensus transcriptional profiles for LINCS L1000 perturbations</a><div class='pub-body'>Daniel Himmelstein, Caty Chung (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d43'>10.15363/thinklab.d43</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d57' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>7.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d57'><a href='/d/57' class='pub-title'>Selecting informative ERC (evolutionary rate covariation) values between genes</a><div class='pub-body'>Daniel Himmelstein, Raghavendran Partha (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d57'>10.15363/thinklab.d57</a></div></div></td></tr><tr><td><a href='/doi/10.1038/ncomms10331' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>8.</div></td><td><div class='doi-body' data-key='10.1038/ncomms10331'><a href='/doi/10.1038/ncomms10331' class='pub-title'>Network-based in silico drug efficacy screening</a><div class='pub-body'>Emre Guney, Jörg Menche, Marc Vidal, Albert-László Barábasi (2016) <i>Nature Communications</i>. doi:<a class='pub-doi' href='https://doi.org/10.1038/ncomms10331'>10.1038/ncomms10331</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d95' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>9.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d95'><a href='/d/95' class='pub-title'>Expert curation of our indication catalog for disease-modifying treatments</a><div class='pub-body'>Daniel Himmelstein, Pouya Khankhanian, Chrissy Hessler (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d95'>10.15363/thinklab.d95</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d112' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>10.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d112'><a href='/d/112' class='pub-title'>Using the neo4j graph database for hetnets</a><div class='pub-body'>Daniel Himmelstein (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d112'>10.15363/thinklab.d112</a></div></div></td></tr></table>

    


	</td>
	<td class="thread-sidebar">


			

      <!-- <a class="btn btn-default follow-link btn-follow btn-lg is-following tt-top full-width" title="Follow discussion" data-id="162" data-ctype="thread" data-is-following="false">&nbsp;</a> -->

      
        <div class="status-box text-center"><span style="">Status:</span> 
  Completed

</div>
      

      
        
      


				

					<div class="details">Views</div>
					<span style="font-size:20px;">164</span>

        

				
					<div class="details">Topics</div>
					<div style="margin-bottom:15px;">
		            <a href="/topics/nomenclature" class="tag">Nomenclature</a><a href="/topics/drug-repurposing" class="tag">Drug Repurposing</a><a href="/topics/hetnets" class="tag">Hetnets</a>
					</div>
				


		
			<div class="details">Referenced by</div>
			<div style="font-size:12px;"><ul style="padding-left:16px !important;">
			
				<li>
					
					Research report: <a href='/p/rephetio/report'>Rephetio: Repurposing drugs on a hetnet</a>
				</li>
			
				<li>
					
					<a href='/d/115' class='thread-title'>Assessing the informativeness of features</a>
				</li>
			
				<li>
					
					<a href='/d/171' class='thread-title'>Positive correlations between knockdown and overexpression profiles from LINCS L1000</a>
				</li>
			
				</ul>
			</div>
		

		
			<div class="details">Cite this as</div>
			<span style="font-size:12px;">
				Daniel Himmelstein, Lars Juhl Jensen, Pouya Khankhanian (2016) Data nomenclature: naming and abbreviating our network types. <i>Thinklab.</i> doi:<a href='https://doi.org/10.15363/thinklab.d162'>10.15363/thinklab.d162</a>
			</span>
		

		<div class="details">License</div>
		<p><a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0;width:88px;height:31px;" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a></p>

		<div class="details">Share</div>
		
			
				<a class="share-button fb" onclick="javascript:void(window.open('https://www.facebook.com/sharer/sharer.php?u=https%3A//thinklab.com/discussion/data-nomenclature-naming-and-abbreviating-our-network-types/162','','width=600,height=460,menubar=no,status=no'));" href="#"></a>
<a class="share-button twitter" onclick="javascript:void(window.open('https://twitter.com/share?text=Data%20nomenclature%3A%20naming%20and%20abbreviating%20our%20network%20types&url=https%3A//thinklab.com/discussion/data-nomenclature-naming-and-abbreviating-our-network-types/162','','width=600,height=300,menubar=no,status=no'));" href="#"></a>
<a class="share-button google" onclick="javascript:void(window.open('https://plus.google.com/share?url=https%3A//thinklab.com/discussion/data-nomenclature-naming-and-abbreviating-our-network-types/162','','width=510,height=460,menubar=no,status=no'));" href="#"></a>

			
		

	</td>
</tr></table>


		</div>


		
	    	<div class="footer">
<!-- <div style="background:#dedede;">
	<div class="main-centered-div text-center" style="padding:20px;">
		<h4 style="margin-bottom:0px;font-weight:bold;color:gray;">Feedback? Questions? Email us: <a href="mailto:hello@thinklab.com">hello@thinklab.com</a></h4>
	</div>
</div> -->
<div class="main-footer">
	<div class="main-centered-div">

<table class="full-width"><tr>
	<td class="footer-nav">

        <div class="foot-head">Explore</div>
        <a href="/proposals">Proposals</a><br>
        <a href="/projects">Projects</a><br>
        <a href="/publications">Article discussion</a><br>
        <a href="/discussion">All discussion</a><br>
        <a href="/topics">Topics</a><br>
        <!-- <a href="/funders">Funders</a><br> -->
        <a href="/funding">Funding opportunities</a><br>

	</td>
	<td class="footer-nav">

        <!-- <div class="foot-head">Help &amp; feedback</div>	 -->
        <!-- <div class="foot-head">Intro</div>
        <a href="/benefits/for-science">Why</a><br>
        <a href="/how-it-works">How it works</a><br>

<br> -->

        <div class="foot-head">Community</div>
        <a href="/leaderboard">Leaderboard</a><br>
        <a href="/p/meta">Thinklab Meta</a><br>
				<a href="/help/faq">Help</a><br>
				<!-- <a href="/new-project">Start a proposal</a><br> -->
        <!-- <a href="/benefits/proposal">Benefits</a><br> -->
        <!-- <a href="/funding">Get funded</a><br> -->
        <!-- <a href="/participate">Become a contributor</a><br> -->
        <!-- <a href="/for-funders">For science funders</a><br> -->
        <!-- <a href="/for-institutions">For institutions</a><br> -->

	</td>
	<td class="footer-nav">



        <div class="foot-head">About</div>
        <a href="/about">Overview</a><br>
        <a href="/benefits">Benefits</a><br>
        <a href="/how-it-works">How it works</a><br>
        <a href="/about/story">Our story</a><br>
        <a href="/blog">Our blog</a><br>
        <a href="/about/jobs">Join the team</a><br>
        <br>

        <!-- <div class="foot-head">Help</div>
        <a href="/help">Help</a><br>
		 -->


	</td>
	<td class="footer-nav">
		<!-- <a href="http://twitter.com/thinklab" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/twitter_com.png" width="40"/></a>
		<a href="https://www.facebook.com/pages/Thinklab/310895039088185" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/facebook_com.png" width="40"/></a>
		<a href="https://plus.google.com/115578983466800753402/" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/plus_google_com.png" width="40"/></a>
		<a href="https://www.linkedin.com/company/thinklab-co" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/linkedin_com.png" width="40"/></a> -->
		<br><br>
		<span class="light" style='font-size:12px;'>&copy; 2015 Thinklab</span>

	</td>

</tr></table>

</div>
</div>
</div>

		

	</div>


  <script></script>
  <script>
  var MEDIA_URL = 'https://think-lab.s3.amazonaws.com/m/';
  var STATIC_URL = 'https://think-lab.s3.amazonaws.com/s/';
  var page_is_authenticated = false;
  var page_page = 'discussion';
  var page_profile_id = undefined;
</script>



	<!-- Note different script type. MUST come before MathJax.js -->
	<script type="text/x-mathjax-config">
	    MathJax.Hub.Config({
	        "HTML-CSS": { preferredFont: "TeX", availableFonts: ["STIX","TeX"],
	                        scale: 110, linebreaks: { automatic:true },
	                        EqnChunk: (MathJax.Hub.Browser.isMobile ? 10 : 50) },
	        tex2jax: { inlineMath: [ ["$$", "$$"], ["\\(","\\)"] ],
	                    displayMath: [ ["$$$","$$$"], ["\\[", "\\]"] ],
	                    processEscapes: true,}
	            });

	</script>
    <script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>





  <script src="https://code.jquery.com/jquery-1.10.2.min.js"></script>
  <script src="https://code.jquery.com/ui/1.11.4/jquery-ui.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" integrity="sha384-0mSbJDEHialfmuBBQP6A4Qrprq5OVfW37PRR3j5ELqxss1yVqOtnepnHVP9aJ7xS" crossorigin="anonymous"></script>
  <script src="https://think-lab.s3.amazonaws.com/s/dist/main.min.js"></script>
  



  
	
  
  
  
<script>
$(function() {
  recordHit('thread', 162);
});
</script>



</body>
</html>
