<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8" /><script type="text/javascript">window.NREUM||(NREUM={}),__nr_require=function(e,n,t){function r(t){if(!n[t]){var o=n[t]={exports:{}};e[t][0].call(o.exports,function(n){var o=e[t][1][n];return r(o||n)},o,o.exports)}return n[t].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<t.length;o++)r(t[o]);return r}({1:[function(e,n,t){function r(){}function o(e,n,t){return function(){return i(e,[c.now()].concat(u(arguments)),n?null:this,t),n?void 0:this}}var i=e("handle"),a=e(2),u=e(3),f=e("ee").get("tracer"),c=e("loader"),s=NREUM;"undefined"==typeof window.newrelic&&(newrelic=s);var p=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],d="api-",l=d+"ixn-";a(p,function(e,n){s[n]=o(d+n,!0,"api")}),s.addPageAction=o(d+"addPageAction",!0),s.setCurrentRouteName=o(d+"routeName",!0),n.exports=newrelic,s.interaction=function(){return(new r).get()};var m=r.prototype={createTracer:function(e,n){var t={},r=this,o="function"==typeof n;return i(l+"tracer",[c.now(),e,t],r),function(){if(f.emit((o?"":"no-")+"fn-start",[c.now(),r,o],t),o)try{return n.apply(this,arguments)}finally{f.emit("fn-end",[c.now()],t)}}}};a("setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(e,n){m[n]=o(l+n)}),newrelic.noticeError=function(e){"string"==typeof e&&(e=new Error(e)),i("err",[e,c.now()])}},{}],2:[function(e,n,t){function r(e,n){var t=[],r="",i=0;for(r in e)o.call(e,r)&&(t[i]=n(r,e[r]),i+=1);return t}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],3:[function(e,n,t){function r(e,n,t){n||(n=0),"undefined"==typeof t&&(t=e?e.length:0);for(var r=-1,o=t-n||0,i=Array(o<0?0:o);++r<o;)i[r]=e[n+r];return i}n.exports=r},{}],4:[function(e,n,t){n.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(e,n,t){function r(){}function o(e){function n(e){return e&&e instanceof r?e:e?f(e,u,i):i()}function t(t,r,o,i){if(!d.aborted||i){e&&e(t,r,o);for(var a=n(o),u=m(t),f=u.length,c=0;c<f;c++)u[c].apply(a,r);var p=s[y[t]];return p&&p.push([b,t,r,a]),a}}function l(e,n){v[e]=m(e).concat(n)}function m(e){return v[e]||[]}function w(e){return p[e]=p[e]||o(t)}function g(e,n){c(e,function(e,t){n=n||"feature",y[t]=n,n in s||(s[n]=[])})}var v={},y={},b={on:l,emit:t,get:w,listeners:m,context:n,buffer:g,abort:a,aborted:!1};return b}function i(){return new r}function a(){(s.api||s.feature)&&(d.aborted=!0,s=d.backlog={})}var u="nr@context",f=e("gos"),c=e(2),s={},p={},d=n.exports=o();d.backlog=s},{}],gos:[function(e,n,t){function r(e,n,t){if(o.call(e,n))return e[n];var r=t();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(e,n,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return e[n]=r,r}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],handle:[function(e,n,t){function r(e,n,t,r){o.buffer([e],r),o.emit(e,n,t)}var o=e("ee").get("handle");n.exports=r,r.ee=o},{}],id:[function(e,n,t){function r(e){var n=typeof e;return!e||"object"!==n&&"function"!==n?-1:e===window?0:a(e,i,function(){return o++})}var o=1,i="nr@id",a=e("gos");n.exports=r},{}],loader:[function(e,n,t){function r(){if(!x++){var e=h.info=NREUM.info,n=d.getElementsByTagName("script")[0];if(setTimeout(s.abort,3e4),!(e&&e.licenseKey&&e.applicationID&&n))return s.abort();c(y,function(n,t){e[n]||(e[n]=t)}),f("mark",["onload",a()+h.offset],null,"api");var t=d.createElement("script");t.src="https://"+e.agent,n.parentNode.insertBefore(t,n)}}function o(){"complete"===d.readyState&&i()}function i(){f("mark",["domContent",a()+h.offset],null,"api")}function a(){return E.exists&&performance.now?Math.round(performance.now()):(u=Math.max((new Date).getTime(),u))-h.offset}var u=(new Date).getTime(),f=e("handle"),c=e(2),s=e("ee"),p=window,d=p.document,l="addEventListener",m="attachEvent",w=p.XMLHttpRequest,g=w&&w.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:w,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var v=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-1044.min.js"},b=w&&g&&g[l]&&!/CriOS/.test(navigator.userAgent),h=n.exports={offset:u,now:a,origin:v,features:{},xhrWrappable:b};e(1),d[l]?(d[l]("DOMContentLoaded",i,!1),p[l]("load",r,!1)):(d[m]("onreadystatechange",o),p[m]("onload",r)),f("mark",["firstbyte",u],null,"api");var x=0,E=e(4)},{}]},{},["loader"]);</script><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info={"beacon":"bam.nr-data.net","queueTime":0,"licenseKey":"cd1101aa94","agent":"","transactionName":"bgcDMkBZW0NZBkdcWVdNJxNcW0FZVwscV1dKB08QW11CQxYTWlBBZgEOC19dW0QCEVtHU1gG","applicationID":"6369297","errorBeacon":"bam.nr-data.net","applicationTime":513}</script>
    <title>MSigDB licensing | Thinklab</title>
	  <link rel="shortcut icon" href="https://think-lab.s3.amazonaws.com/s/img/favicon.ico" type="image/x-icon" />
    <meta property="og:image" content="https://think-lab.s3.amazonaws.com/s/img" />
    
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
  <link rel="stylesheet" href="https://code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css" />

<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/highlightjs/github.css" />
<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/css/css.css" />
<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/css/bootstrap-multiselect.css" />
<!-- <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'> -->

    
    
    
    
  
    <meta name="citation_doi" content="10.15363/thinklab.d108">
    <meta name="citation_title" content="MSigDB licensing">
    <meta name="citation_online_date" content="Sept. 28, 2015">
    <meta name="citation_author" content="Daniel Himmelstein">
    <meta name="citation_journal_title" content="Thinklab">
    <meta name="citation_publisher" content="Thinklab">
  

    


	<!-- GoogleAnalytics -->
	<script>
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
	(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
	m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

	ga('create', 'UA-48550230-1', 'thinklab.org');
	ga('require', 'displayfeatures');
	
	ga('send', 'pageview');

	</script>






</head>
<body >
  <form id="form_action" method="post">
    <input type='hidden' name='csrfmiddlewaretoken' value='JC3hnVSWzcwBOS226QTq2eGyHXj1YAM6' />
</form>

	

	<div class="body-wrapper">

		


<!-- <div style="height:5px;background-image:url('https://think-lab.s3.amazonaws.com/s/img/top-bg.gif');"></div> -->
<div class="gradient">
	<div class="main-centered-div header-container">

<table class="full-width" style="height:60px;"><tr>
	<td style="width:200px;vertical-align:middle;padding-bottom:15px;">
		<a href="/" style="font-weight:normal;color:#212121;"><img src="https://think-lab.s3.amazonaws.com/s/img/logo.png" width="160"></a>
	</td><td style="vertical-align:middle;text-align:left;padding-top:6px;width:100%;padding-left:40px;">
		<ul id="not-search-block"  class="main-header">


			


				<li class="dropdown">
					<a class="dropdown-toggle" data-toggle="dropdown" href="#">Browse <span class="caret"></span></a>
					<ul class="dropdown-menu" role="menu">
						<li><a href="/proposals">Proposals</a></li>
						<li><a href="/projects">Projects</a></li>
						<li><a href="/publications">Article discussion</a></li>
						<li><a href="/discussion">All discussion</a></li>
					</ul>
				</li>
				<li><a href="/leaderboard">Leaderboard</a></li>

			
				<li><a href="/about">About</a></li>
			


		</ul>

    <ul id="search-block" class="main-header" style="display: none; margin-right: 30px;">
        <li class="full-width">
            <form method="get" action="/search">
			<div class="input-group full-width">
				<input name="q" type="text" class="form-control search-input">
			</div>
            </form>
        </li>
    </ul>

	</td><td style="vertical-align:middle;text-align:right;padding-top:6px; min-width: 1%; white-space: nowrap;">
		<ul class="main-header pull-right" style="padding-left: 0;">
    		
                <!-- <li><a href="/login" style="color:silver;">Login</a></li>
                <li style="margin-right:0"><a class="btn btn-default btn-sm btn-upper" href="/signup">Join</a></li> -->
            

        </ul>
    </ul>
	</td>
</tr></table>

<!-- <div id="global-search-div" class="main-centered-div" style="display:none;">
	<input type="text" value="" class="thread-search form-control" placeholder="Search" data-action="/discussion">
</div> -->

	</div>
</div>

		
	  



	  


		

		<div class="dark-head"><div class="main-centered-div">

			

		</div></div>
		
		<div class="main-centered-div" style="margin-top:40px;padding-bottom:340px">
	    

<div style="margin-top:-20px; margin-bottom:20px;">

    <a class="side-box" href="/p/rephetio" style="background-color:#ebfdff;"><div class="side-type">Project:</div> <div class="side-title">Rephetio: Repurposing drugs on a hetnet [rephetio]</div></a><br>



</div>

<table style="margin-top:0px;"><tr>
	<td class="thread-table-content">

		<table class="full-width">
			
<tr class="comment-tr target-hl" id="1" data-comment-id="439">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="439"></a>

    

			<h2 class="page-title" style="margin-bottom:5px;margin-top:0px !important;">MSigDB licensing</h2>

			<div style="margin-top:5px;color:gray;margin-bottom:20px;font-size:13px;">
				<a style="font-weight:normal;color:#212121" href="/u/dhimmel"><strong>Daniel Himmelstein</strong></a>
				<span class="is-researcher">Researcher</span>
				&nbsp; <a href="#1" class="light">Sept. 28, 2015</a>
				
			</div>

      

          <hr>
		

    <p>We currently rely on <a href="http://www.broadinstitute.org/gsea/msigdb/index.jsp">MSigDB</a> <span class="citation">[<a href="https://doi.org/10.1073/pnas.0506580102" class="citation" data-key="10.1073/pnas.0506580102">1</a>, <a href="https://doi.org/10.1093/bioinformatics/btr260" class="citation" data-key="10.1093/bioinformatics/btr260">2</a>]</span>, the Molecular Signatures Database, for perturbation gene sets and pathways. Since the <a href="https://github.com/dhimmel/integrate/blob/afe1c049e4a088bc266533332e6e00499e3d2e20/licenses/custom/MSigDB.asciidoc">license</a> is highly restrictive, we have emailed the creators with the below message. We will post any updates regarding MSigDB licensing or permissions on this discussion.</p>

<hr>

<p>Greetings MSigDB Team,</p>

<p>I am a graduate student at UCSF, and I have been using <a href="http://www.broadinstitute.org/gsea/msigdb/index.jsp">MSigDB</a> for my research. My project aims to predict new uses for existing drugs by integrating many different types of biomedical information. Recently, the issue of database copyright and licensing <a href="http://thinklab.com/discussion/integrating-resources-with-disparate-licensing-into-an-open-network/107" class="citation thread-citation" data-key="10.15363/thinklab.d107">came up</a>, and we are now trying to ensure that we have sufficient permissions for each of the 28 databases we're integrating.</p>

<p>I was surprised to learn of MSigDB's restrictive <a href="http://www.broadinstitute.org/cancer/software/gsea/wiki/index.php/License_info">license</a> that forbids redistribution, especially given the projects <a href="http://grantome.com/grant/NIH/R01-CA121941-06A1">public funding</a>. Currently, several resources I have created may be non-compliant with the license:</p>

<p>My GitHub repository (<a href="https://github.com/dhimmel/msigdb"><code>dhimmel/msigdb</code></a>) for parsing the MSigDB database contains:</p>

<ul><li>unmodified MSigDB downloads</li><li>a reformatted version of the underlying data</li></ul>

<p>My GitHub repository (<a href="https://github.com/dhimmel/pathways"><code>dhimmel/pathways</code></a>) for combining pathway databases contains:</p>

<ul><li>the reformatted version of C2:CP from <code>dhimmel/msigdb</code> and a derived dataset containing pathways from other resources</li></ul>

<p>My GitHub repository (<a href="https://github.com/dhimmel/integrate"><code>dhimmel/integrate</code></a>) for integrating many resources into a single network contains:</p>

<ul><li>the majority of MSigDB 5.0 C2:CP and C2:CGP data stored as network nodes and edges.</li></ul>

<p>My <a href="http://het.io/disease-genes/downloads/">website</a> for a previous project <span class="citation">[<a href="https://doi.org/10.1371/journal.pcbi.1004259" class="citation" data-key="10.1371/journal.pcbi.1004259">3</a>]</span> contains:</p>

<ul><li>a network download formatted as in <code>dhimmel/integrate</code>, but containing data from most collections in MSigDB version 3.0.</li></ul>

<p>The public availability of the aforementioned resources is important so others can reproduce and build off of our work. Thus, we request permission for our current redistribution and derivative works of MSigDB. Ideally, we could be granted permission to release MSigDB data under a <a href="https://creativecommons.org/licenses/">Creative Commons license</a> without a No Derivatives restriction. Applying a CC license would lessen the burden on downstream users.</p>

<p>Thanks for your consideration. Our research is academic in nature, and we suspect it falls under the intended use of MSigDB but perhaps not the license.</p>

<p>Finally, we're performing our project using an open science platform called Thinklab. I've <a href="#1">posted a copy</a> of this email on Thinklab and will update the discussion with our progress. Alternatively, feel free to respond via Thinklab rather than email. By detailing each step of our research process publicly, we're hoping to create a valuable resource and explore a more holistic and collaborative medium of publication.</p>

<p>Sincerely,<br>Daniel</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="2" data-comment-id="760">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="760"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#2" class="light">Feb. 19, 2016</a>
          
        </span>
      </div>

    

    <p>After an additional round of emails on February 4, 2016, Jill Mesirov got back to me. Dr. Mesirov was the principal investigator for the MSigDB project while at the Broad but has since moved to UCSD. She mentioned that they received permission to distribute certain parts of the database but that they did not receive permission to pass on those rights. She also added Helga Thorvaldsdottir — the MSigDB project manager at the Broad — to the conversation.</p>

<p>I responded with the following message:</p>

<blockquote><p>Dear Dr. Mesirov et al,</p><p>Thanks for the reply and involving Helga. Hopefully, we can now locate the appropriate parties to handle our request.</p><p>I had guessed that non-transferable distribution rights were part of the issue. I appreciate wanting to build the most comprehensive resource, even when that necessitates stricter licensing.</p><p>Do you know which resources forbid downstream distribution? Perhaps we could be given permission to redistribute the unencumbered portions of the database? And for encumbered portions, we could seek the needed additional permissions from the content owners.</p><p>We feel that distribution fulfills an important scientific need. We're integrating over 30 resources into a single network that we envision becoming a widely used community dataset. Much like MSigDB did with gene sets, our network will enable novel analyses that are only possible once the data has been unified into a single resource. Additionally, forbidding distribution has troubling consequences for reproducibility. See for example <a href="http://wpo.st/NUj91" title="What happened when a group of researchers tried to repeat a headline-grabbing study">this instance</a> where data copyright interfered with replication.</p><p>Given these considerations, we would appreciate help in finding a solution that allows us to distribute MSigDB data, even if only a subset of the database.</p><p>Best,<br>Daniel</p></blockquote>

<p>In short, I asked if they could look into granting us permission to distribute the unencumbered portions of the database. Ms. Thorvaldsdottir responded that they will be meeting with the IP/Licensing team to discuss my request.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="3" data-comment-id="1214">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="1214"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#3" class="light">April 3, 2016</a>
          
        </span>
      </div>

    

    <h1>Removing MSigDB from the Rephetio project</h1>

<p>I removed MSigDB from our project, since we haven't been able to resolve the licensing issues. It's been 186 days since we initially contacted the MSigDB team and 53 days since we were told that the IP/Licensing team had been notified and a meeting scheduled. Unfortunately, we can't wait any longer.</p>

<p>There are a few distinctions that make MSigDB distinct from <a href="http://thinklab.com/discussion/integrating-resources-with-disparate-licensing-into-an-open-network/107#14" class="citation thread-citation" data-key="10.15363/thinklab.d107">other resources</a> where permission is pending but we continue to include. MSigDB is the <a href="https://github.com/dhimmel/integrate/blob/daefe6e3e9a44b9fdc85cb79cee597927f119559/licenses/README.md" title="Table of licenses for all sources">only resource</a> with a license that explicitly forbids distribution. Additionally, the MSigDB website requires <a href="http://software.broadinstitute.org/gsea/register.jsp">registration</a>, although accessing the database by URL bypasses registration.</p>

<p>Registration makes the license into a legally binding agreement. Essentially, the registration acts as a contract, which can place additional restrictions beyond copyright. As an aside, I therefore find it misleading that the <a href="http://software.broadinstitute.org/gsea/msigdb">website</a> states:</p>

<blockquote><p>Registration is free. Its only purpose is to help us track usage for reports to our funding agencies.</p></blockquote>

<p>Specifically, <a href="https://github.com/dhimmel/integrate/blob/daefe6e3e9a44b9fdc85cb79cee597927f119559/licenses/custom/MSigDB.asciidoc" title="MSigDB License">the license</a> contains several troubling components. First is a reporting requirement for modifications and bug fixes:</p>

<blockquote><p>modifications and BUG FIXES shall be provided to MIT promptly upon their creation.</p></blockquote>

<p>While this reporting requirement only applies to the program, which we don't use, a different reporting requirement applies to database:</p>

<blockquote><p>As consideration for the licenses granted in this Agreement, LICENSEE agrees to provide … a written evaluation of the PROGRAM and the DATABASE, including a description of its functionality or problems and areas for further improvement in the PROGRAM or the DATABASE.</p></blockquote>

<p>The license is very clear that distribution is prohibited. In fact, uploading the database to a private cloud service appears to violate the license:</p>

<blockquote><p>2.2 No Sublicensing or Additional Rights. In no event shall LICENSEE sublicense or distribute, in whole or in part, the PROGRAM, modifications, BUG FIXES, or the DATABASE, without prior permission from MIT. LICENSEE agrees not to allow any non-employee of LICENSEE to access, view, or use the PROGRAM or the DATABASE, unless such person is an independent contractor performing services on behalf of LICENSEE. LICENSEE agrees not to put the PROGRAM or the DATABASE on a network, server, or other similar technology that may be accessed by any individual other than the LICENSEE.</p></blockquote>

<p>As a result, using MSigDB as part of extensible open science project is not possible.</p>

<h2>Remedial action</h2>

<p>I removed MSigDB from our hetnet (<a href="https://github.com/dhimmel/integrate/commit/daefe6e3e9a44b9fdc85cb79cee597927f119559">commit</a>). Our hetnet no longer contains perturbation gene sets, which were from the <code>C2:CGP</code> collection. I <a href="http://thinklab.com/discussion/adding-pathway-resources-to-your-network/72#11" class="citation thread-citation" data-key="10.15363/thinklab.d72">replaced</a> the MSigDB pathways (<code>C2:CP</code> collection) using <a href="http://www.pathwaycommons.org/">Pathway Commons</a>. Both Pathway Commons and MSigDB include data from Reactome and the PID, but by going through Pathway Commons we were able to release those resources as CC BY and CC0.</p>

<p>I deleted my GitHub repository, formerly <a href="https://github.com/dhimmel/msigdb"><code>dhimmel/msigdb</code></a>, for converting the database into a single user-friendly TSV. Our <a href="http://het.io/disease-genes/downloads/">website</a> for a previous project contains a hetnet including MSigDB 3.0, which I posted before being aware of the licensing issue. Since this hetnet is the foundation of our prior study <span class="citation">[<a href="https://doi.org/10.1371/journal.pcbi.1004259" class="citation" data-key="10.1371/journal.pcbi.1004259">1</a>]</span>, taking it offline would be problematic for reproducibility and destructive to science. Therefore, I am not taking down this dataset unless specifically requested.</p>

<p>Finally, I removed the following two paragraphs from our project report draft. I'll let them serve as a eulogy:</p>

<blockquote><p>Pathways were extracted by combining human pathways from <a href="http://www.wikipathways.org/">WikiPathways</a> <span class="citation">[<a href="https://doi.org/10.1093/nar/gkv1024" class="citation" data-key="10.1093/nar/gkv1024">2</a>, <a href="https://doi.org/10.1371/journal.pbio.0060184" class="citation" data-key="10.1371/journal.pbio.0060184">3</a>]</span> and MSigDB version 5.1 <span class="citation">[<a href="https://doi.org/10.1073/pnas.0506580102" class="citation" data-key="10.1073/pnas.0506580102">4</a>, <a href="https://doi.org/10.1093/bioinformatics/btr260" class="citation" data-key="10.1093/bioinformatics/btr260">5</a>]</span>. Perturbations were extracted from MSigDB. Each perturbation corresponds to a differential expression experiment of a chemical or genetic perturbation.</p><p><em>Perturbation–regulates–Gene</em> edges are from MSigDB <span class="citation">[<a href="https://doi.org/10.1073/pnas.0506580102" class="citation" data-key="10.1073/pnas.0506580102">4</a>, <a href="https://doi.org/10.1093/bioinformatics/btr260" class="citation" data-key="10.1093/bioinformatics/btr260">5</a>]</span>. These edges represent groups of genes that responded in the same direction to a chemical or genetic perturbation. Our previous project found this indiscriminate, automated, and high-throughput method produced gene sets that together were highly informative for predicting disease–gene associations <span class="citation">[<a href="https://doi.org/10.1371/journal.pcbi.1004743" class="citation" data-key="10.1371/journal.pcbi.1004743">6</a>]</span>.</p></blockquote>

<h2>Reflections</h2>

<p>One issue at play is the restrictive licensing of resources that MSigDB integrates. For example, KEGG <a href="https://github.com/dhimmel/integrate/blob/daefe6e3e9a44b9fdc85cb79cee597927f119559/licenses/custom/KEGG.md">requires</a> academic services to obtain a <a href="http://www.bioinformatics.jp/docs/subscription_organizational.pdf">subscription</a>, which stipulates that:</p>

<blockquote><p>Your Product or Service must not allow Your users to obtain KEGG FTP Data, except in small quantities.</p></blockquote>

<p>Hence, portions of the MSigDB dataset do have to be licensed to forbid redistribution. Nevertheless, MIT went beyond these upstream restrictions when writing the MSigDB license. First, much of the database could likely be released openly. Second, I find the reporting requirements extreme. It's possible that MIT had a financial motivation when writing the license, as it states:</p>

<blockquote><p>LICENSEE agrees that neither the PROGRAM nor the DATABASE shall be used as the basis of a commercial software or hardware product</p></blockquote>

<p>I think the MSigDB team deserves credit for making their comprehensive compilation of gene sets public. However, given the <a href="http://grantome.com/grant/NIH/R01-CA121941-10" title="See Related projects on Grantome">extent of public funding</a> this project has received, I question whether it's ethical for MIT to apply such a problematic license.</p>

<p>Finally I'm not deflecting responsibility: I'm the one who included a resource whose license forbids redistribution. While scientists are often poorly informed on the legality of data reuse, I think it's important to take responsibility for educating yourself. Going forward I will address licensing issues before using a new dataset to avoid similar problems.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

    		
		</table>

    

    &nbsp;<br>
    <!-- <a class="btn btn-primary btn-lg" href="/signup?next=/discussion/msigdb-licensing/108&suppress_message=1" style>Join to Reply</a> -->


		<table class="references table-td-5" style="display:none;"><tr><td><a href='/doi/10.1073/pnas.0506580102' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>1.</div></td><td><div class='doi-body' data-key='10.1073/pnas.0506580102'><a href='/doi/10.1073/pnas.0506580102' class='pub-title'>Gene set enrichment analysis: A knowledge-based approach for interpreting genome-wide expression profiles</a><div class='pub-body'>A. Subramanian, P. Tamayo, V. K. Mootha, S. Mukherjee, B. L. Ebert, M. A. Gillette, A. Paulovich, S. L. Pomeroy, T. R. Golub, E. S. Lander, J. P. Mesirov (2005) <i>Proceedings of the National Academy of Sciences</i>. doi:<a class='pub-doi' href='https://doi.org/10.1073/pnas.0506580102'>10.1073/pnas.0506580102</a></div></div></td></tr><tr><td><a href='/doi/10.1093/bioinformatics/btr260' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>2.</div></td><td><div class='doi-body' data-key='10.1093/bioinformatics/btr260'><a href='/doi/10.1093/bioinformatics/btr260' class='pub-title'>Molecular signatures database (MSigDB) 3.0</a><div class='pub-body'>A. Liberzon, A. Subramanian, R. Pinchback, H. Thorvaldsdottir, P. Tamayo, J. P. Mesirov (2011) <i>Bioinformatics</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/bioinformatics/btr260'>10.1093/bioinformatics/btr260</a></div></div></td></tr><tr><td><a href='/doi/10.1371/journal.pcbi.1004259' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>3.</div></td><td><div class='doi-body' data-key='10.1371/journal.pcbi.1004259'><a href='/doi/10.1371/journal.pcbi.1004259' class='pub-title'>Heterogeneous Network Edge Prediction: A Data Integration Approach to Prioritize Disease-Associated Genes</a><div class='pub-body'>Daniel S. Himmelstein, Sergio E. Baranzini (2015) <i>PLOS Computational Biology</i>. doi:<a class='pub-doi' href='https://doi.org/10.1371/journal.pcbi.1004259'>10.1371/journal.pcbi.1004259</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d107' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>4.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d107'><a href='/d/107' class='pub-title'>Integrating resources with disparate licensing into an open network</a><div class='pub-body'>Daniel Himmelstein, Lars Juhl Jensen, MacKenzie Smith, Katie Fortney, Caty Chung (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d107'>10.15363/thinklab.d107</a></div></div></td></tr><tr><td><a href='/doi/10.1371/journal.pcbi.1004259' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>5.</div></td><td><div class='doi-body' data-key='10.1371/journal.pcbi.1004259'><a href='/doi/10.1371/journal.pcbi.1004259' class='pub-title'>Heterogeneous Network Edge Prediction: A Data Integration Approach to Prioritize Disease-Associated Genes</a><div class='pub-body'>Daniel S. Himmelstein, Sergio E. Baranzini (2015) <i>PLOS Computational Biology</i>. doi:<a class='pub-doi' href='https://doi.org/10.1371/journal.pcbi.1004259'>10.1371/journal.pcbi.1004259</a></div></div></td></tr><tr><td><a href='/doi/10.1093/nar/gkv1024' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>6.</div></td><td><div class='doi-body' data-key='10.1093/nar/gkv1024'><a href='/doi/10.1093/nar/gkv1024' class='pub-title'>WikiPathways: capturing the full diversity of pathway knowledge</a><div class='pub-body'>Martina Kutmon, Anders Riutta, Nuno Nunes, Kristina Hanspers, Egon L. Willighagen, Anwesha Bohler, Jonathan Mélius, Andra Waagmeester, Sravanthi R. Sinha, Ryan Miller, Susan L. Coort, Elisa Cirillo, Bart Smeets, Chris T. Evelo, Alexander R. Pico (2015) <i>Nucleic Acids Res</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/nar/gkv1024'>10.1093/nar/gkv1024</a></div></div></td></tr><tr><td><a href='/doi/10.1371/journal.pbio.0060184' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>7.</div></td><td><div class='doi-body' data-key='10.1371/journal.pbio.0060184'><a href='/doi/10.1371/journal.pbio.0060184' class='pub-title'>WikiPathways: Pathway Editing for the People</a><div class='pub-body'>Alexander R. Pico, Thomas Kelder, Martijn P. van Iersel, Kristina Hanspers, Bruce R. Conklin, Chris Evelo (2008) <i>Plos Biol</i>. doi:<a class='pub-doi' href='https://doi.org/10.1371/journal.pbio.0060184'>10.1371/journal.pbio.0060184</a></div></div></td></tr><tr><td><a href='/doi/10.1073/pnas.0506580102' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>8.</div></td><td><div class='doi-body' data-key='10.1073/pnas.0506580102'><a href='/doi/10.1073/pnas.0506580102' class='pub-title'>Gene set enrichment analysis: A knowledge-based approach for interpreting genome-wide expression profiles</a><div class='pub-body'>A. Subramanian, P. Tamayo, V. K. Mootha, S. Mukherjee, B. L. Ebert, M. A. Gillette, A. Paulovich, S. L. Pomeroy, T. R. Golub, E. S. Lander, J. P. Mesirov (2005) <i>Proceedings of the National Academy of Sciences</i>. doi:<a class='pub-doi' href='https://doi.org/10.1073/pnas.0506580102'>10.1073/pnas.0506580102</a></div></div></td></tr><tr><td><a href='/doi/10.1093/bioinformatics/btr260' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>9.</div></td><td><div class='doi-body' data-key='10.1093/bioinformatics/btr260'><a href='/doi/10.1093/bioinformatics/btr260' class='pub-title'>Molecular signatures database (MSigDB) 3.0</a><div class='pub-body'>A. Liberzon, A. Subramanian, R. Pinchback, H. Thorvaldsdottir, P. Tamayo, J. P. Mesirov (2011) <i>Bioinformatics</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/bioinformatics/btr260'>10.1093/bioinformatics/btr260</a></div></div></td></tr><tr><td><a href='/doi/10.1371/journal.pcbi.1004743' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>10.</div></td><td><div class='doi-body' data-key='10.1371/journal.pcbi.1004743'><a href='/doi/10.1371/journal.pcbi.1004743' class='pub-title'>Ten Simple Rules for Selecting a Bio-ontology</a><div class='pub-body'>James Malone, Robert Stevens, Simon Jupp, Tom Hancocks, Helen Parkinson, Cath Brooksbank (2016) <i>PLOS Computational Biology</i>. doi:<a class='pub-doi' href='https://doi.org/10.1371/journal.pcbi.1004743'>10.1371/journal.pcbi.1004743</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d107' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>11.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d107'><a href='/d/107' class='pub-title'>Integrating resources with disparate licensing into an open network</a><div class='pub-body'>Daniel Himmelstein, Lars Juhl Jensen, MacKenzie Smith, Katie Fortney, Caty Chung (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d107'>10.15363/thinklab.d107</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d72' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>12.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d72'><a href='/d/72' class='pub-title'>Adding pathway resources to your network</a><div class='pub-body'>Alexander Pico, Daniel Himmelstein (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d72'>10.15363/thinklab.d72</a></div></div></td></tr></table>

    


	</td>
	<td class="thread-sidebar">


			

      <!-- <a class="btn btn-default follow-link btn-follow btn-lg is-following tt-top full-width" title="Follow discussion" data-id="108" data-ctype="thread" data-is-following="false">&nbsp;</a> -->

      
        <div class="status-box text-center"><span style="">Status:</span> 
  Completed

</div>
      

      
        

          <div class="details">
            
            Labels
          </div>

          
            <a href='/p/rephetio/discussion?q=[licensing]' class='pl pl-sidebar ' style='background-color:#6b6b6b'><span class='glyphicon glyphicon-tag'></span>&nbsp; licensing</a>
          
        
      


				

					<div class="details">Views</div>
					<span style="font-size:20px;">196</span>

        

				
					<div class="details">Topics</div>
					<div style="margin-bottom:15px;">
		            <a href="/topics/copyright" class="tag">Copyright</a><a href="/topics/permission" class="tag">Permission</a><a href="/topics/licensing" class="tag">Licensing</a><a href="/topics/mit" class="tag">MIT</a><a href="/topics/msigdb" class="tag">MSigDB</a>
					</div>
				


		
			<div class="details">Referenced by</div>
			<div style="font-size:12px;"><ul style="padding-left:16px !important;">
			
				<li>
					
					<a href='/d/107' class='thread-title'>Integrating resources with disparate licensing into an open network</a>
				</li>
			
				<li>
					
					Research report: <a href='/p/rephetio/report'>Rephetio: Repurposing drugs on a hetnet</a>
				</li>
			
				<li>
					
					<a href='/d/213' class='thread-title'>Sounding the alarm on DrugBank’s new license and terms of use</a>
				</li>
			
				<li>
					
					<a href='/d/72' class='thread-title'>Adding pathway resources to your network</a>
				</li>
			
				</ul>
			</div>
		

		
			<div class="details">Cite this as</div>
			<span style="font-size:12px;">
				Daniel Himmelstein (2015) MSigDB licensing. <i>Thinklab.</i> doi:<a href='https://doi.org/10.15363/thinklab.d108'>10.15363/thinklab.d108</a>
			</span>
		

		<div class="details">License</div>
		<p><a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0;width:88px;height:31px;" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a></p>

		<div class="details">Share</div>
		
			
				<a class="share-button fb" onclick="javascript:void(window.open('https://www.facebook.com/sharer/sharer.php?u=https%3A//thinklab.org/discussion/msigdb-licensing/108','','width=600,height=460,menubar=no,status=no'));" href="#"></a>
<a class="share-button twitter" onclick="javascript:void(window.open('https://twitter.com/share?text=MSigDB%20licensing&url=https%3A//thinklab.org/discussion/msigdb-licensing/108','','width=600,height=300,menubar=no,status=no'));" href="#"></a>
<a class="share-button google" onclick="javascript:void(window.open('https://plus.google.com/share?url=https%3A//thinklab.org/discussion/msigdb-licensing/108','','width=510,height=460,menubar=no,status=no'));" href="#"></a>

			
		

	</td>
</tr></table>


		</div>


		
	    	<div class="footer">
<!-- <div style="background:#dedede;">
	<div class="main-centered-div text-center" style="padding:20px;">
		<h4 style="margin-bottom:0px;font-weight:bold;color:gray;">Feedback? Questions? Email us: <a href="mailto:hello@thinklab.com">hello@thinklab.com</a></h4>
	</div>
</div> -->
<div class="main-footer">
	<div class="main-centered-div">

<table class="full-width"><tr>
	<td class="footer-nav">

        <div class="foot-head">Explore</div>
        <a href="/proposals">Proposals</a><br>
        <a href="/projects">Projects</a><br>
        <a href="/publications">Article discussion</a><br>
        <a href="/discussion">All discussion</a><br>
        <a href="/topics">Topics</a><br>
        <!-- <a href="/funders">Funders</a><br> -->
        <a href="/funding">Funding opportunities</a><br>

	</td>
	<td class="footer-nav">

        <!-- <div class="foot-head">Help &amp; feedback</div>	 -->
        <!-- <div class="foot-head">Intro</div>
        <a href="/benefits/for-science">Why</a><br>
        <a href="/how-it-works">How it works</a><br>

<br> -->

        <div class="foot-head">Community</div>
        <a href="/leaderboard">Leaderboard</a><br>
        <a href="/p/meta">Thinklab Meta</a><br>
				<a href="/help/faq">Help</a><br>
				<!-- <a href="/new-project">Start a proposal</a><br> -->
        <!-- <a href="/benefits/proposal">Benefits</a><br> -->
        <!-- <a href="/funding">Get funded</a><br> -->
        <!-- <a href="/participate">Become a contributor</a><br> -->
        <!-- <a href="/for-funders">For science funders</a><br> -->
        <!-- <a href="/for-institutions">For institutions</a><br> -->

	</td>
	<td class="footer-nav">



        <div class="foot-head">About</div>
        <a href="/about">Overview</a><br>
        <a href="/benefits">Benefits</a><br>
        <a href="/how-it-works">How it works</a><br>
        <a href="/about/story">Our story</a><br>
        <a href="/blog">Our blog</a><br>
        
        <br>

        <!-- <div class="foot-head">Help</div>
        <a href="/help">Help</a><br>
		 -->


	</td>
	<td class="footer-nav">
		<!-- <a href="http://twitter.com/thinklab" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/twitter_com.png" width="40"/></a>
		<a href="https://www.facebook.com/pages/Thinklab/310895039088185" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/facebook_com.png" width="40"/></a>
		<a href="https://plus.google.com/115578983466800753402/" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/plus_google_com.png" width="40"/></a>
		<a href="https://www.linkedin.com/company/thinklab-co" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/linkedin_com.png" width="40"/></a> -->
		<br><br>
		<span class="light" style='font-size:12px;'>&copy; 2015 Thinklab</span>

	</td>

</tr></table>

</div>
</div>
</div>

		

	</div>


  <script></script>
  <script>
  var MEDIA_URL = 'https://think-lab.s3.amazonaws.com/m/';
  var STATIC_URL = 'https://think-lab.s3.amazonaws.com/s/';
  var page_is_authenticated = false;
  var page_page = 'discussion';
  var page_profile_id = undefined;
</script>



	<!-- Note different script type. MUST come before MathJax.js -->
	<script type="text/x-mathjax-config">
	    MathJax.Hub.Config({
	        "HTML-CSS": { preferredFont: "TeX", availableFonts: ["STIX","TeX"],
	                        scale: 110, linebreaks: { automatic:true },
	                        EqnChunk: (MathJax.Hub.Browser.isMobile ? 10 : 50) },
	        tex2jax: { inlineMath: [ ["$$", "$$"], ["\\(","\\)"] ],
	                    displayMath: [ ["$$$","$$$"], ["\\[", "\\]"] ],
	                    processEscapes: true,}
	            });

	</script>
    <script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>





  <script src="https://code.jquery.com/jquery-1.10.2.min.js"></script>
  <script src="https://code.jquery.com/ui/1.11.4/jquery-ui.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" integrity="sha384-0mSbJDEHialfmuBBQP6A4Qrprq5OVfW37PRR3j5ELqxss1yVqOtnepnHVP9aJ7xS" crossorigin="anonymous"></script>
  <script src="https://think-lab.s3.amazonaws.com/s/dist/main.min.js"></script>
  



  
	
  
  
  
<script>
$(function() {
  recordHit('thread', 108);
});
</script>



</body>
</html>
