<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8" /><script type="text/javascript">window.NREUM||(NREUM={}),__nr_require=function(e,n,t){function r(t){if(!n[t]){var o=n[t]={exports:{}};e[t][0].call(o.exports,function(n){var o=e[t][1][n];return r(o||n)},o,o.exports)}return n[t].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<t.length;o++)r(t[o]);return r}({1:[function(e,n,t){function r(){}function o(e,n,t){return function(){return i(e,[c.now()].concat(u(arguments)),n?null:this,t),n?void 0:this}}var i=e("handle"),a=e(2),u=e(3),f=e("ee").get("tracer"),c=e("loader"),s=NREUM;"undefined"==typeof window.newrelic&&(newrelic=s);var p=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],d="api-",l=d+"ixn-";a(p,function(e,n){s[n]=o(d+n,!0,"api")}),s.addPageAction=o(d+"addPageAction",!0),s.setCurrentRouteName=o(d+"routeName",!0),n.exports=newrelic,s.interaction=function(){return(new r).get()};var m=r.prototype={createTracer:function(e,n){var t={},r=this,o="function"==typeof n;return i(l+"tracer",[c.now(),e,t],r),function(){if(f.emit((o?"":"no-")+"fn-start",[c.now(),r,o],t),o)try{return n.apply(this,arguments)}finally{f.emit("fn-end",[c.now()],t)}}}};a("setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(e,n){m[n]=o(l+n)}),newrelic.noticeError=function(e){"string"==typeof e&&(e=new Error(e)),i("err",[e,c.now()])}},{}],2:[function(e,n,t){function r(e,n){var t=[],r="",i=0;for(r in e)o.call(e,r)&&(t[i]=n(r,e[r]),i+=1);return t}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],3:[function(e,n,t){function r(e,n,t){n||(n=0),"undefined"==typeof t&&(t=e?e.length:0);for(var r=-1,o=t-n||0,i=Array(o<0?0:o);++r<o;)i[r]=e[n+r];return i}n.exports=r},{}],4:[function(e,n,t){n.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(e,n,t){function r(){}function o(e){function n(e){return e&&e instanceof r?e:e?f(e,u,i):i()}function t(t,r,o,i){if(!d.aborted||i){e&&e(t,r,o);for(var a=n(o),u=m(t),f=u.length,c=0;c<f;c++)u[c].apply(a,r);var p=s[y[t]];return p&&p.push([b,t,r,a]),a}}function l(e,n){v[e]=m(e).concat(n)}function m(e){return v[e]||[]}function w(e){return p[e]=p[e]||o(t)}function g(e,n){c(e,function(e,t){n=n||"feature",y[t]=n,n in s||(s[n]=[])})}var v={},y={},b={on:l,emit:t,get:w,listeners:m,context:n,buffer:g,abort:a,aborted:!1};return b}function i(){return new r}function a(){(s.api||s.feature)&&(d.aborted=!0,s=d.backlog={})}var u="nr@context",f=e("gos"),c=e(2),s={},p={},d=n.exports=o();d.backlog=s},{}],gos:[function(e,n,t){function r(e,n,t){if(o.call(e,n))return e[n];var r=t();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(e,n,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return e[n]=r,r}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],handle:[function(e,n,t){function r(e,n,t,r){o.buffer([e],r),o.emit(e,n,t)}var o=e("ee").get("handle");n.exports=r,r.ee=o},{}],id:[function(e,n,t){function r(e){var n=typeof e;return!e||"object"!==n&&"function"!==n?-1:e===window?0:a(e,i,function(){return o++})}var o=1,i="nr@id",a=e("gos");n.exports=r},{}],loader:[function(e,n,t){function r(){if(!x++){var e=h.info=NREUM.info,n=d.getElementsByTagName("script")[0];if(setTimeout(s.abort,3e4),!(e&&e.licenseKey&&e.applicationID&&n))return s.abort();c(y,function(n,t){e[n]||(e[n]=t)}),f("mark",["onload",a()+h.offset],null,"api");var t=d.createElement("script");t.src="https://"+e.agent,n.parentNode.insertBefore(t,n)}}function o(){"complete"===d.readyState&&i()}function i(){f("mark",["domContent",a()+h.offset],null,"api")}function a(){return E.exists&&performance.now?Math.round(performance.now()):(u=Math.max((new Date).getTime(),u))-h.offset}var u=(new Date).getTime(),f=e("handle"),c=e(2),s=e("ee"),p=window,d=p.document,l="addEventListener",m="attachEvent",w=p.XMLHttpRequest,g=w&&w.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:w,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var v=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-1044.min.js"},b=w&&g&&g[l]&&!/CriOS/.test(navigator.userAgent),h=n.exports={offset:u,now:a,origin:v,features:{},xhrWrappable:b};e(1),d[l]?(d[l]("DOMContentLoaded",i,!1),p[l]("load",r,!1)):(d[m]("onreadystatechange",o),p[m]("onload",r)),f("mark",["firstbyte",u],null,"api");var x=0,E=e(4)},{}]},{},["loader"]);</script><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info={"beacon":"bam.nr-data.net","queueTime":0,"licenseKey":"cd1101aa94","agent":"","transactionName":"bgcDMkBZW0NZBkdcWVdNJxNcW0FZVwscV1dKB08QW11CQxYTWlBBZgEOC19dW0QCEVtHU1gG","applicationID":"6369297","errorBeacon":"bam.nr-data.net","applicationTime":554}</script>
    <title>Assessing the effectiveness of our hetnet permutations | Thinklab</title>
	  <link rel="shortcut icon" href="https://think-lab.s3.amazonaws.com/s/img/favicon.ico" type="image/x-icon" />
    <meta property="og:image" content="https://think-lab.s3.amazonaws.com/s/img" />
    
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
  <link rel="stylesheet" href="https://code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css" />

<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/highlightjs/github.css" />
<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/css/css.css" />
<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/css/bootstrap-multiselect.css" />
<!-- <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'> -->

    
    
    
    
  
    <meta name="citation_doi" content="10.15363/thinklab.d178">
    <meta name="citation_title" content="Assessing the effectiveness of our hetnet permutations">
    <meta name="citation_online_date" content="Feb. 25, 2016">
    <meta name="citation_author" content="Daniel Himmelstein">
    <meta name="citation_journal_title" content="Thinklab">
    <meta name="citation_publisher" content="Thinklab">
  

    


	<!-- GoogleAnalytics -->
	<script>
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
	(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
	m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

	ga('create', 'UA-48550230-1', 'thinklab.org');
	ga('require', 'displayfeatures');
	
	ga('send', 'pageview');

	</script>






</head>
<body >
  <form id="form_action" method="post">
    <input type='hidden' name='csrfmiddlewaretoken' value='JC3hnVSWzcwBOS226QTq2eGyHXj1YAM6' />
</form>

	

	<div class="body-wrapper">

		


<!-- <div style="height:5px;background-image:url('https://think-lab.s3.amazonaws.com/s/img/top-bg.gif');"></div> -->
<div class="gradient">
	<div class="main-centered-div header-container">

<table class="full-width" style="height:60px;"><tr>
	<td style="width:200px;vertical-align:middle;padding-bottom:15px;">
		<a href="/" style="font-weight:normal;color:#212121;"><img src="https://think-lab.s3.amazonaws.com/s/img/logo.png" width="160"></a>
	</td><td style="vertical-align:middle;text-align:left;padding-top:6px;width:100%;padding-left:40px;">
		<ul id="not-search-block"  class="main-header">


			


				<li class="dropdown">
					<a class="dropdown-toggle" data-toggle="dropdown" href="#">Browse <span class="caret"></span></a>
					<ul class="dropdown-menu" role="menu">
						<li><a href="/proposals">Proposals</a></li>
						<li><a href="/projects">Projects</a></li>
						<li><a href="/publications">Article discussion</a></li>
						<li><a href="/discussion">All discussion</a></li>
					</ul>
				</li>
				<li><a href="/leaderboard">Leaderboard</a></li>

			
				<li><a href="/about">About</a></li>
			


		</ul>

    <ul id="search-block" class="main-header" style="display: none; margin-right: 30px;">
        <li class="full-width">
            <form method="get" action="/search">
			<div class="input-group full-width">
				<input name="q" type="text" class="form-control search-input">
			</div>
            </form>
        </li>
    </ul>

	</td><td style="vertical-align:middle;text-align:right;padding-top:6px; min-width: 1%; white-space: nowrap;">
		<ul class="main-header pull-right" style="padding-left: 0;">
    		
                <!-- <li><a href="/login" style="color:silver;">Login</a></li>
                <li style="margin-right:0"><a class="btn btn-default btn-sm btn-upper" href="/signup">Join</a></li> -->
            

        </ul>
    </ul>
	</td>
</tr></table>

<!-- <div id="global-search-div" class="main-centered-div" style="display:none;">
	<input type="text" value="" class="thread-search form-control" placeholder="Search" data-action="/discussion">
</div> -->

	</div>
</div>

		
	  



	  


		

		<div class="dark-head"><div class="main-centered-div">

			

		</div></div>
		
		<div class="main-centered-div" style="margin-top:40px;padding-bottom:340px">
	    

<div style="margin-top:-20px; margin-bottom:20px;">

    <a class="side-box" href="/p/rephetio" style="background-color:#ebfdff;"><div class="side-type">Project:</div> <div class="side-title">Rephetio: Repurposing drugs on a hetnet [rephetio]</div></a><br>



</div>

<table style="margin-top:0px;"><tr>
	<td class="thread-table-content">

		<table class="full-width">
			
<tr class="comment-tr target-hl" id="1" data-comment-id="1135">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="1135"></a>

    

			<h2 class="page-title" style="margin-bottom:5px;margin-top:0px !important;">Assessing the effectiveness of our hetnet permutations</h2>

			<div style="margin-top:5px;color:gray;margin-bottom:20px;font-size:13px;">
				<a style="font-weight:normal;color:#212121" href="/u/dhimmel"><strong>Daniel Himmelstein</strong></a>
				<span class="is-researcher">Researcher</span>
				&nbsp; <a href="#1" class="light">Feb. 25, 2016</a>
				
			</div>

      

          <hr>
		

    <p>We've previously discussed <a href="http://thinklab.com/discussion/permuting-hetnets-and-implementing-randomized-edge-swaps-in-cypher/136#1" class="citation thread-citation" data-key="10.15363/thinklab.d136">what hetnet permutation is and why we do it</a>. To permute a hetnet, we go through each relationship type (metaedge) and repeatedly swap the target nodes of two random relationships (edges). This strategy is called <code>XSwap</code> <span class="citation">[<a href="https://doi.org/10.1137/1.9781611972795.67" class="citation" data-key="10.1137/1.9781611972795.67">1</a>]</span>.</p>

<p>We looked into performing the <a href="http://thinklab.com/discussion/permuting-hetnets-and-implementing-randomized-edge-swaps-in-cypher/136#2" class="citation thread-citation" data-key="10.15363/thinklab.d136">permutation in neo4j using cypher</a>, but decided to stick with our <a href="https://github.com/dhimmel/hetio/blob/a6c8a286ec0c7367673970c6ddda06cd47733034/hetio/permute.py#L7" title="hetio.permute.permute_graph · dhimmel/hetio on GitHub">python implementation</a> since cypher's cost planner currently lacks the needed abilities.</p>

<h2>Implementation specifics</h2>

<p>We closely followed the parameters from our previous study <span class="citation">[<a href="https://doi.org/10.1371/journal.pcbi.1004259" class="citation" data-key="10.1371/journal.pcbi.1004259">2</a>]</span> and did the following:</p>

<ul><li>We created 5 permuted hetnets. The first permutated hetnet was created from the unpermuted hetnet; the second permutated hetnet was created from the first permutated hetnet; and so on until the fifth permutated hetnet was created from the fourth permutated hetnet. This iterative strategy is referred to as a Markov chain.</li><li>To create each permuted hetnet, we separately permuted each metaedge. For a given metaedge, we attempted <em>n</em> XSwaps where <em>n</em> equals four times the number of edges (<code>multipler = 4</code>).</li><li>Xswaps can be unsuccessful for several reasons. The same edge could have been randomly selected twice (referred to as <code>same_edge</code>). One or both of the potential new edges may already exist (<code>duplicate</code> or <code>undirected_duplicate</code> for select cases where a biderectional edge connects two nodes of the same type). One or both of the potential new edges may connect a node to itself (<code>self_loop</code>). In these instances, no swap is performed. In the future, we may switch to stopping a completing permutation after a certain number of successes rather than attempts.</li></ul>

<h2>Assessing permutation effectiveness</h2>

<p>For each permutation and each metaedge, we measure the progress of the randomization at 10 points (<a href="https://github.com/dhimmel/integrate/blob/ebd71cd2157d26e52646b5b483f5c70293a84f71/data/permuted/stats.tsv">dataset</a>). The measure we're primarily interested in is the percent of edges that are unchanged after a permutation (<code>unchanged</code>).</p>

<p>We find that the percent of unchanged edges varies by metaedge (<a href="http://nbviewer.jupyter.org/github/dhimmel/integrate/blob/ebd71cd2157d26e52646b5b483f5c70293a84f71/data/permuted/evaluate-permutations.ipynb#How-many-permutations-are-needed-to-randomize-an-edge">notebook cell 4</a>). It appears that we could safely reduce our multiplier from 4 to 2.5 and still generate permuted networks that are maximally diversified from their predecessor.</p>

<p>Of concern are metaedges where a high percentage of the edges do not change. This occurred when a high percentage of swaps resulted in already existing edges (<a href="https://github.com/dhimmel/integrate/blob/ebd71cd2157d26e52646b5b483f5c70293a84f71/data/permuted/evaluate-permutations.ipynb">notebook cell 6</a>). Particularly troublesome was the <em>Anatomy–expresses–Gene</em> edge where almost all attempts yielded duplicated edges and only ~10% of edges changed from a permutation. I'm now inclined to revisit our <a href="http://thinklab.com/discussion/processing-bgee-for-tissue-specific-gene-presence-and-overunder-expression/124#2" class="citation thread-citation" data-key="10.15363/thinklab.d124">previous observation</a> that we're being too permissive regarding expression edge inclusion.</p>

<p>Metaedges whose edges do not change from permutation are limited in informativeness. Such edges hold little information besides their degree contribution to the nodes they connect. In the context of our expression edge, the problem is visible in the node degree distribution: most anatomies express 0 genes while a minority of anatomies express an extremely high number of genes (<a href="http://nbviewer.jupyter.org/github/dhimmel/integrate/blob/ebd71cd2157d26e52646b5b483f5c70293a84f71/viz/degrees.pdf#page=10">see the <code>anatomy - expresses - gene</code> panel on page 10</a>).</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="2" data-comment-id="1172">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="1172"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#2" class="light">March 11, 2016</a>
          
        </span>
      </div>

    

    <h1>Improved randomization of expression edges</h1>

<p>We <a href="http://thinklab.com/discussion/processing-bgee-for-tissue-specific-gene-presence-and-overunder-expression/124#6" class="citation thread-citation" data-key="10.15363/thinklab.d124">updated our method</a> for extracting <em>Anatomy–expresses–Gene</em> relationships from Bgee. This update <a href="https://github.com/dhimmel/integrate/commit/d68b823bf2167e7ab7f0e784d1280200c33fb3bf#diff-c849eed0ccfe917ca2fceb4f57045444R3">reduced the number</a> of expression edges in our hetnet from 1,006,278 to 526,407. The number of genes with an expression edge went from 18,147 to 18,094. The number of anatomies with an expression edge went from 256 to 241.</p>

<p>The permutation of expression edges increased in effectiveness. Now ~25% of expression edges (as opposed to ~10% <a href="#1">previously</a>) <a href="http://nbviewer.jupyter.org/github/dhimmel/integrate/blob/d68b823bf2167e7ab7f0e784d1280200c33fb3bf/data/permuted/evaluate-permutations.ipynb#How-many-permutations-are-needed-to-randomize-an-edge">change in</a> a permuted hetnet. And this is in spite of fewer attempted swaps per permutation: I decreased the multiplier from 4 to 3 to reduce runtime.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="3" data-comment-id="1509">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="1509"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#3" class="light">July 8, 2017</a>
          
        </span>
      </div>

    

    <h1>Comparison to BiRewire</h1>

<p>BiRewire is an R <a href="http://www.bioconductor.org/packages/release/bioc/html/BiRewire.html">Bioconductor package</a> for bipartite network randomization (<a href="https://github.com/Bioconductor-mirror/BiRewire">source code</a>, <a href="http://www.ebi.ac.uk/~iorio/BiRewire">homepage</a>) <span class="citation">[<a href="https://doi.org/10.1093/bioinformatics/btu474" class="citation " data-key="10.1093/bioinformatics/btu474">1</a>, <a href="https://doi.org/10.1186/s12859-016-1402-1" class="citation " data-key="10.1186/s12859-016-1402-1">2</a>]</span>. Like our implementation, BiRewire performs randomization through edge swaps (i.e. XSwap in <span class="citation">[<a href="https://doi.org/10.1137/1.9781611972795.67" class="citation " data-key="10.1137/1.9781611972795.67">3</a>]</span>), which they refer to as the "switching-steps".</p>

<p>BiRewire also accounts for edge-sign, which is essentially a second dimension of direction, that can be used to represent a binary edge attribute, such as activation versus inhibition <span class="citation">[<a href="https://doi.org/10.1186/s12859-016-1402-1" class="citation " data-key="10.1186/s12859-016-1402-1">2</a>]</span>. From my understanding, you could engineer sign-awareness by permuting sign-positive and sign-negative edges separately. In other words, sign-positive edges can only be swapped with other sign-positive edges. In the <code>hetio</code> <a href="https://github.com/dhimmel/hetio/blob/30c6dbb18a17c05d71cb909cf57af7372e4d4908/hetio/permute.py">randomization implementation</a>, we perform edge swaps only within metaedge type. We also make sure to respect directionality when present.</p>

<p>BiRewire also adopts a Markov chain approach, where permutation is done iteratively: the first permutation is created from the unpermuted hetnet, and the second permutation is created from the first permutation. BiRewire describes the conceptual appeal of this approach <span class="citation">[<a href="https://doi.org/10.1186/s12859-016-1402-1" class="citation " data-key="10.1186/s12859-016-1402-1">2</a>]</span>:</p>

<blockquote><p>In <span class="citation">[<a href="https://doi.org/10.1093/bioinformatics/btu474" class="citation " data-key="10.1093/bioinformatics/btu474">1</a>]</span> we prove that, if executing a sufficiently large number of [switching-steps], the [switching-algorithm] can efficiently simulate samplings from the uniform distribution of all the possible bipartite networks with predefined node sets and prescribed node degrees. Therefore it can be used to obtain a rewired version of a network B that it is, on average, no more similar to <em>B</em> than are similar to each other two bipartite networks <em>B₁</em> and <em>B₂</em> sampled from the real uniform distribution of all the possible bipartite networks with the same node sets and node degrees of <em>B</em>.</p></blockquote>

<p>The BiRewire team did some nice analytical work relating to how many swaps need to be performed by each iteration of the Markov chain <span class="citation">[<a href="https://doi.org/10.1093/bioinformatics/btu474" class="citation " data-key="10.1093/bioinformatics/btu474">1</a>]</span>:</p>

<blockquote><p>We analytically derived an approximate lower bound for the number of switching-steps to be performed by the switching-algorithm</p></blockquote>

<div class="math">$$$
N=\frac{N′}{200(1-d)} \ln⁡{\frac{(1-d)N′}{100}}
$$$</div>

<blockquote><p>where d is the edge density of the original network, defined as the ratio between  |E|  and the number of edges of a fully connected bipartite graph with the same number of nodes in the two classes.</p></blockquote>

<p>We took an empirical rather than analytical approach to ensure that we performed sufficient edge swaps (see above comments). However, we can compare our number of swaps to the minimum suggested by the BiRewire team for the most difficult metaedge to randomize (<em>Anatomy–expresses–Gene</em>). Using our <a href="https://github.com/dhimmel/integrate/blob/d68b823bf2167e7ab7f0e784d1280200c33fb3bf/data/permuted/stats.tsv#L181">empirical randomization statistics</a>, we apply the formula above in Python:</p>

<pre><code class="python">import math
N_prime = 526407
d = N_prime / (18094 * 241)
N = N_prime / (200 * (1 - d)) * math.log(0.01 * (1 - d) * N_prime)
n_swaps = 1579220 * (1 - 0.90)
assert N &lt;= n_swaps
print(round(N), round(n_swaps))</code></pre>

<p>The minimum <em>N</em> (number of edge swaps) suggested by the BiRewire formula is 25264, whereas we performed approximately 158,000. Given that I'm not sure all of the assumptions of BiRewire's formulation apply to our case (e.g. "same number of nodes in the two classes"), I still recommend empirically investigating the decay curve of percent of edges changed, rather than relying exclusively on the formula. But nice to see the complementary research into network randomization!</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

    		
		</table>

    

    &nbsp;<br>
    <!-- <a class="btn btn-primary btn-lg" href="/signup?next=/discussion/assessing-the-effectiveness-of-our-hetnet-permutations/178&suppress_message=1" style>Join to Reply</a> -->


		<table class="references table-td-5" style="display:none;"><tr><td><a href='/doi/10.1137/1.9781611972795.67' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>1.</div></td><td><div class='doi-body' data-key='10.1137/1.9781611972795.67'><a href='/doi/10.1137/1.9781611972795.67' class='pub-title'>Randomization Techniques for Graphs</a><div class='pub-body'>Sami Hanhijärvi, Gemma C. Garriga, Kai Puolamäki (2009) <i>Proceedings of the 2009 SIAM International Conference on Data Mining</i>. doi:<a class='pub-doi' href='https://doi.org/10.1137/1.9781611972795.67'>10.1137/1.9781611972795.67</a></div></div></td></tr><tr><td><a href='/doi/10.1371/journal.pcbi.1004259' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>2.</div></td><td><div class='doi-body' data-key='10.1371/journal.pcbi.1004259'><a href='/doi/10.1371/journal.pcbi.1004259' class='pub-title'>Heterogeneous Network Edge Prediction: A Data Integration Approach to Prioritize Disease-Associated Genes</a><div class='pub-body'>Daniel S. Himmelstein, Sergio E. Baranzini (2015) <i>PLOS Computational Biology</i>. doi:<a class='pub-doi' href='https://doi.org/10.1371/journal.pcbi.1004259'>10.1371/journal.pcbi.1004259</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d136' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>3.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d136'><a href='/d/136' class='pub-title'>Permuting hetnets and implementing randomized edge swaps in cypher</a><div class='pub-body'>Daniel Himmelstein (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d136'>10.15363/thinklab.d136</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d136' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>4.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d136'><a href='/d/136' class='pub-title'>Permuting hetnets and implementing randomized edge swaps in cypher</a><div class='pub-body'>Daniel Himmelstein (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d136'>10.15363/thinklab.d136</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d124' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>5.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d124'><a href='/d/124' class='pub-title'>Processing Bgee for tissue-specific gene presence and over/under-expression</a><div class='pub-body'>Daniel Himmelstein, Frederic Bastian (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d124'>10.15363/thinklab.d124</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d124' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>6.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d124'><a href='/d/124' class='pub-title'>Processing Bgee for tissue-specific gene presence and over/under-expression</a><div class='pub-body'>Daniel Himmelstein, Frederic Bastian (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d124'>10.15363/thinklab.d124</a></div></div></td></tr><tr><td><a href='/doi/10.1093/bioinformatics/btu474' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>7.</div></td><td><div class='doi-body' data-key='10.1093/bioinformatics/btu474'><a href='/doi/10.1093/bioinformatics/btu474' class='pub-title'>Fast randomization of large genomic datasets while preserving alteration counts</a><div class='pub-body'>Andrea Gobbi, Francesco Iorio, Kevin J. Dawson, David C. Wedge, David Tamborero, Ludmil B. Alexandrov, Nuria Lopez-Bigas, Mathew J. Garnett, Giuseppe Jurman, Julio Saez-Rodriguez (2014) <i>Bioinformatics</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/bioinformatics/btu474'>10.1093/bioinformatics/btu474</a></div></div></td></tr><tr><td><a href='/doi/10.1186/s12859-016-1402-1' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>8.</div></td><td><div class='doi-body' data-key='10.1186/s12859-016-1402-1'><a href='/doi/10.1186/s12859-016-1402-1' class='pub-title'>Efficient randomization of biological networks while preserving functional characterization of individual nodes</a><div class='pub-body'>Francesco Iorio, Marti Bernardo-Faura, Andrea Gobbi, Thomas Cokelaer, Giuseppe Jurman, Julio Saez-Rodriguez (2016) <i>BMC Bioinformatics</i>. doi:<a class='pub-doi' href='https://doi.org/10.1186/s12859-016-1402-1'>10.1186/s12859-016-1402-1</a></div></div></td></tr><tr><td><a href='/doi/10.1137/1.9781611972795.67' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>9.</div></td><td><div class='doi-body' data-key='10.1137/1.9781611972795.67'><a href='/doi/10.1137/1.9781611972795.67' class='pub-title'>Randomization Techniques for Graphs</a><div class='pub-body'>Sami Hanhijärvi, Gemma C. Garriga, Kai Puolamäki (2009) <i>Proceedings of the 2009 SIAM International Conference on Data Mining</i>. doi:<a class='pub-doi' href='https://doi.org/10.1137/1.9781611972795.67'>10.1137/1.9781611972795.67</a></div></div></td></tr></table>

    


	</td>
	<td class="thread-sidebar">


			

      <!-- <a class="btn btn-default follow-link btn-follow btn-lg is-following tt-top full-width" title="Follow discussion" data-id="178" data-ctype="thread" data-is-following="false">&nbsp;</a> -->

      
        <div class="status-box text-center"><span style="">Status:</span> 
  Completed

</div>
      

      
        
      


				

					<div class="details">Views</div>
					<span style="font-size:20px;">78</span>

        

				
					<div class="details">Topics</div>
					<div style="margin-bottom:15px;">
		            <a href="/topics/permutation" class="tag">Permutation</a><a href="/topics/xswap" class="tag">XSwap</a><a href="/topics/hetio" class="tag">Hetio</a><a href="/topics/hetnets" class="tag">Hetnets</a><a href="/topics/network-randomization" class="tag">Network Randomization</a>
					</div>
				


		
			<div class="details">Referenced by</div>
			<div style="font-size:12px;"><ul style="padding-left:16px !important;">
			
				<li>
					
					Research report: <a href='/p/rephetio/report'>Rephetio: Repurposing drugs on a hetnet</a>
				</li>
			
				<li>
					
					<a href='/d/115' class='thread-title'>Assessing the informativeness of features</a>
				</li>
			
				<li>
					
					<a href='/d/214' class='thread-title'>Introducing the Residual Degree Weighted Path Count (R-DWPC)</a>
				</li>
			
				<li>
					
					<a href='/d/187' class='thread-title'>Estimating the complexity of hetnet traversal</a>
				</li>
			
				<li>
					
					<a href='/d/201' class='thread-title'>Network Edge Prediction: Estimating the prior</a>
				</li>
			
				<li>
					
					<a href='/d/124' class='thread-title'>Processing Bgee for tissue-specific gene presence and over/under-expression</a>
				</li>
			
				</ul>
			</div>
		

		
			<div class="details">Cite this as</div>
			<span style="font-size:12px;">
				Daniel Himmelstein (2016) Assessing the effectiveness of our hetnet permutations. <i>Thinklab.</i> doi:<a href='https://doi.org/10.15363/thinklab.d178'>10.15363/thinklab.d178</a>
			</span>
		

		<div class="details">License</div>
		<p><a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0;width:88px;height:31px;" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a></p>

		<div class="details">Share</div>
		
			
				<a class="share-button fb" onclick="javascript:void(window.open('https://www.facebook.com/sharer/sharer.php?u=https%3A//thinklab.org/discussion/assessing-the-effectiveness-of-our-hetnet-permutations/178','','width=600,height=460,menubar=no,status=no'));" href="#"></a>
<a class="share-button twitter" onclick="javascript:void(window.open('https://twitter.com/share?text=Assessing%20the%20effectiveness%20of%20our%20hetnet%20permutations&url=https%3A//thinklab.org/discussion/assessing-the-effectiveness-of-our-hetnet-permutations/178','','width=600,height=300,menubar=no,status=no'));" href="#"></a>
<a class="share-button google" onclick="javascript:void(window.open('https://plus.google.com/share?url=https%3A//thinklab.org/discussion/assessing-the-effectiveness-of-our-hetnet-permutations/178','','width=510,height=460,menubar=no,status=no'));" href="#"></a>

			
		

	</td>
</tr></table>


		</div>


		
	    	<div class="footer">
<!-- <div style="background:#dedede;">
	<div class="main-centered-div text-center" style="padding:20px;">
		<h4 style="margin-bottom:0px;font-weight:bold;color:gray;">Feedback? Questions? Email us: <a href="mailto:hello@thinklab.com">hello@thinklab.com</a></h4>
	</div>
</div> -->
<div class="main-footer">
	<div class="main-centered-div">

<table class="full-width"><tr>
	<td class="footer-nav">

        <div class="foot-head">Explore</div>
        <a href="/proposals">Proposals</a><br>
        <a href="/projects">Projects</a><br>
        <a href="/publications">Article discussion</a><br>
        <a href="/discussion">All discussion</a><br>
        <a href="/topics">Topics</a><br>
        <!-- <a href="/funders">Funders</a><br> -->
        <a href="/funding">Funding opportunities</a><br>

	</td>
	<td class="footer-nav">

        <!-- <div class="foot-head">Help &amp; feedback</div>	 -->
        <!-- <div class="foot-head">Intro</div>
        <a href="/benefits/for-science">Why</a><br>
        <a href="/how-it-works">How it works</a><br>

<br> -->

        <div class="foot-head">Community</div>
        <a href="/leaderboard">Leaderboard</a><br>
        <a href="/p/meta">Thinklab Meta</a><br>
				<a href="/help/faq">Help</a><br>
				<!-- <a href="/new-project">Start a proposal</a><br> -->
        <!-- <a href="/benefits/proposal">Benefits</a><br> -->
        <!-- <a href="/funding">Get funded</a><br> -->
        <!-- <a href="/participate">Become a contributor</a><br> -->
        <!-- <a href="/for-funders">For science funders</a><br> -->
        <!-- <a href="/for-institutions">For institutions</a><br> -->

	</td>
	<td class="footer-nav">



        <div class="foot-head">About</div>
        <a href="/about">Overview</a><br>
        <a href="/benefits">Benefits</a><br>
        <a href="/how-it-works">How it works</a><br>
        <a href="/about/story">Our story</a><br>
        <a href="/blog">Our blog</a><br>
        
        <br>

        <!-- <div class="foot-head">Help</div>
        <a href="/help">Help</a><br>
		 -->


	</td>
	<td class="footer-nav">
		<!-- <a href="http://twitter.com/thinklab" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/twitter_com.png" width="40"/></a>
		<a href="https://www.facebook.com/pages/Thinklab/310895039088185" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/facebook_com.png" width="40"/></a>
		<a href="https://plus.google.com/115578983466800753402/" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/plus_google_com.png" width="40"/></a>
		<a href="https://www.linkedin.com/company/thinklab-co" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/linkedin_com.png" width="40"/></a> -->
		<br><br>
		<span class="light" style='font-size:12px;'>&copy; 2015 Thinklab</span>

	</td>

</tr></table>

</div>
</div>
</div>

		

	</div>


  <script></script>
  <script>
  var MEDIA_URL = 'https://think-lab.s3.amazonaws.com/m/';
  var STATIC_URL = 'https://think-lab.s3.amazonaws.com/s/';
  var page_is_authenticated = false;
  var page_page = 'discussion';
  var page_profile_id = undefined;
</script>



	<!-- Note different script type. MUST come before MathJax.js -->
	<script type="text/x-mathjax-config">
	    MathJax.Hub.Config({
	        "HTML-CSS": { preferredFont: "TeX", availableFonts: ["STIX","TeX"],
	                        scale: 110, linebreaks: { automatic:true },
	                        EqnChunk: (MathJax.Hub.Browser.isMobile ? 10 : 50) },
	        tex2jax: { inlineMath: [ ["$$", "$$"], ["\\(","\\)"] ],
	                    displayMath: [ ["$$$","$$$"], ["\\[", "\\]"] ],
	                    processEscapes: true,}
	            });

	</script>
    <script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>





  <script src="https://code.jquery.com/jquery-1.10.2.min.js"></script>
  <script src="https://code.jquery.com/ui/1.11.4/jquery-ui.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" integrity="sha384-0mSbJDEHialfmuBBQP6A4Qrprq5OVfW37PRR3j5ELqxss1yVqOtnepnHVP9aJ7xS" crossorigin="anonymous"></script>
  <script src="https://think-lab.s3.amazonaws.com/s/dist/main.min.js"></script>
  



  
	
  
  
  
<script>
$(function() {
  recordHit('thread', 178);
});
</script>



</body>
</html>
