<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8" /><script type="text/javascript">window.NREUM||(NREUM={}),__nr_require=function(e,n,t){function r(t){if(!n[t]){var o=n[t]={exports:{}};e[t][0].call(o.exports,function(n){var o=e[t][1][n];return r(o||n)},o,o.exports)}return n[t].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<t.length;o++)r(t[o]);return r}({1:[function(e,n,t){function r(){}function o(e,n,t){return function(){return i(e,[c.now()].concat(u(arguments)),n?null:this,t),n?void 0:this}}var i=e("handle"),a=e(2),u=e(3),f=e("ee").get("tracer"),c=e("loader"),s=NREUM;"undefined"==typeof window.newrelic&&(newrelic=s);var p=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],d="api-",l=d+"ixn-";a(p,function(e,n){s[n]=o(d+n,!0,"api")}),s.addPageAction=o(d+"addPageAction",!0),s.setCurrentRouteName=o(d+"routeName",!0),n.exports=newrelic,s.interaction=function(){return(new r).get()};var m=r.prototype={createTracer:function(e,n){var t={},r=this,o="function"==typeof n;return i(l+"tracer",[c.now(),e,t],r),function(){if(f.emit((o?"":"no-")+"fn-start",[c.now(),r,o],t),o)try{return n.apply(this,arguments)}finally{f.emit("fn-end",[c.now()],t)}}}};a("setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(e,n){m[n]=o(l+n)}),newrelic.noticeError=function(e){"string"==typeof e&&(e=new Error(e)),i("err",[e,c.now()])}},{}],2:[function(e,n,t){function r(e,n){var t=[],r="",i=0;for(r in e)o.call(e,r)&&(t[i]=n(r,e[r]),i+=1);return t}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],3:[function(e,n,t){function r(e,n,t){n||(n=0),"undefined"==typeof t&&(t=e?e.length:0);for(var r=-1,o=t-n||0,i=Array(o<0?0:o);++r<o;)i[r]=e[n+r];return i}n.exports=r},{}],4:[function(e,n,t){n.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(e,n,t){function r(){}function o(e){function n(e){return e&&e instanceof r?e:e?f(e,u,i):i()}function t(t,r,o,i){if(!d.aborted||i){e&&e(t,r,o);for(var a=n(o),u=m(t),f=u.length,c=0;c<f;c++)u[c].apply(a,r);var p=s[y[t]];return p&&p.push([b,t,r,a]),a}}function l(e,n){v[e]=m(e).concat(n)}function m(e){return v[e]||[]}function w(e){return p[e]=p[e]||o(t)}function g(e,n){c(e,function(e,t){n=n||"feature",y[t]=n,n in s||(s[n]=[])})}var v={},y={},b={on:l,emit:t,get:w,listeners:m,context:n,buffer:g,abort:a,aborted:!1};return b}function i(){return new r}function a(){(s.api||s.feature)&&(d.aborted=!0,s=d.backlog={})}var u="nr@context",f=e("gos"),c=e(2),s={},p={},d=n.exports=o();d.backlog=s},{}],gos:[function(e,n,t){function r(e,n,t){if(o.call(e,n))return e[n];var r=t();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(e,n,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return e[n]=r,r}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],handle:[function(e,n,t){function r(e,n,t,r){o.buffer([e],r),o.emit(e,n,t)}var o=e("ee").get("handle");n.exports=r,r.ee=o},{}],id:[function(e,n,t){function r(e){var n=typeof e;return!e||"object"!==n&&"function"!==n?-1:e===window?0:a(e,i,function(){return o++})}var o=1,i="nr@id",a=e("gos");n.exports=r},{}],loader:[function(e,n,t){function r(){if(!x++){var e=h.info=NREUM.info,n=d.getElementsByTagName("script")[0];if(setTimeout(s.abort,3e4),!(e&&e.licenseKey&&e.applicationID&&n))return s.abort();c(y,function(n,t){e[n]||(e[n]=t)}),f("mark",["onload",a()+h.offset],null,"api");var t=d.createElement("script");t.src="https://"+e.agent,n.parentNode.insertBefore(t,n)}}function o(){"complete"===d.readyState&&i()}function i(){f("mark",["domContent",a()+h.offset],null,"api")}function a(){return E.exists&&performance.now?Math.round(performance.now()):(u=Math.max((new Date).getTime(),u))-h.offset}var u=(new Date).getTime(),f=e("handle"),c=e(2),s=e("ee"),p=window,d=p.document,l="addEventListener",m="attachEvent",w=p.XMLHttpRequest,g=w&&w.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:w,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var v=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-1044.min.js"},b=w&&g&&g[l]&&!/CriOS/.test(navigator.userAgent),h=n.exports={offset:u,now:a,origin:v,features:{},xhrWrappable:b};e(1),d[l]?(d[l]("DOMContentLoaded",i,!1),p[l]("load",r,!1)):(d[m]("onreadystatechange",o),p[m]("onload",r)),f("mark",["firstbyte",u],null,"api");var x=0,E=e(4)},{}]},{},["loader"]);</script><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info={"beacon":"bam.nr-data.net","queueTime":0,"licenseKey":"cd1101aa94","agent":"","transactionName":"bgcDMkBZW0NZBkdcWVdNJxNcW0FZVwscV1dKB08QW11CQxYTWlBBZgEOC19dW0QCEVtHU1gG","applicationID":"6369297","errorBeacon":"bam.nr-data.net","applicationTime":1023}</script>
    <title>Text as a resource for network population? | Thinklab</title>
	  <link rel="shortcut icon" href="https://think-lab.s3.amazonaws.com/s/img/favicon.ico" type="image/x-icon" />
    <meta property="og:image" content="https://think-lab.s3.amazonaws.com/s/img" />
    
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
  <link rel="stylesheet" href="https://code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css" />

<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/highlightjs/github.css" />
<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/css/css.css" />
<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/css/bootstrap-multiselect.css" />
<!-- <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'> -->

    
    
    
    
  
    <meta name="citation_doi" content="10.15363/thinklab.d48">
    <meta name="citation_title" content="Text as a resource for network population?">
    <meta name="citation_online_date" content="April 2, 2015">
    <meta name="citation_author" content="Benjamin Good, Daniel Himmelstein">
    <meta name="citation_journal_title" content="Thinklab">
    <meta name="citation_publisher" content="Thinklab">
  

    


	<!-- GoogleAnalytics -->
	<script>
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
	(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
	m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

	ga('create', 'UA-48550230-1', 'thinklab.org');
	ga('require', 'displayfeatures');
	
	ga('send', 'pageview');

	</script>






</head>
<body >
  <form id="form_action" method="post">
    <input type='hidden' name='csrfmiddlewaretoken' value='JC3hnVSWzcwBOS226QTq2eGyHXj1YAM6' />
</form>

	

	<div class="body-wrapper">

		


<!-- <div style="height:5px;background-image:url('https://think-lab.s3.amazonaws.com/s/img/top-bg.gif');"></div> -->
<div class="gradient">
	<div class="main-centered-div header-container">

<table class="full-width" style="height:60px;"><tr>
	<td style="width:200px;vertical-align:middle;padding-bottom:15px;">
		<a href="/" style="font-weight:normal;color:#212121;"><img src="https://think-lab.s3.amazonaws.com/s/img/logo.png" width="160"></a>
	</td><td style="vertical-align:middle;text-align:left;padding-top:6px;width:100%;padding-left:40px;">
		<ul id="not-search-block"  class="main-header">


			


				<li class="dropdown">
					<a class="dropdown-toggle" data-toggle="dropdown" href="#">Browse <span class="caret"></span></a>
					<ul class="dropdown-menu" role="menu">
						<li><a href="/proposals">Proposals</a></li>
						<li><a href="/projects">Projects</a></li>
						<li><a href="/publications">Article discussion</a></li>
						<li><a href="/discussion">All discussion</a></li>
					</ul>
				</li>
				<li><a href="/leaderboard">Leaderboard</a></li>

			
				<li><a href="/about">About</a></li>
			


		</ul>

    <ul id="search-block" class="main-header" style="display: none; margin-right: 30px;">
        <li class="full-width">
            <form method="get" action="/search">
			<div class="input-group full-width">
				<input name="q" type="text" class="form-control search-input">
			</div>
            </form>
        </li>
    </ul>

	</td><td style="vertical-align:middle;text-align:right;padding-top:6px; min-width: 1%; white-space: nowrap;">
		<ul class="main-header pull-right" style="padding-left: 0;">
    		
                <!-- <li><a href="/login" style="color:silver;">Login</a></li>
                <li style="margin-right:0"><a class="btn btn-default btn-sm btn-upper" href="/signup">Join</a></li> -->
            

        </ul>
    </ul>
	</td>
</tr></table>

<!-- <div id="global-search-div" class="main-centered-div" style="display:none;">
	<input type="text" value="" class="thread-search form-control" placeholder="Search" data-action="/discussion">
</div> -->

	</div>
</div>

		
	  



	  


		

		<div class="dark-head"><div class="main-centered-div">

			

		</div></div>
		
		<div class="main-centered-div" style="margin-top:40px;padding-bottom:340px">
	    

<div style="margin-top:-20px; margin-bottom:20px;">

    <a class="side-box" href="/p/rephetio" style="background-color:#ebfdff;"><div class="side-type">Project:</div> <div class="side-title">Rephetio: Repurposing drugs on a hetnet [rephetio]</div></a><br>



</div>

<table style="margin-top:0px;"><tr>
	<td class="thread-table-content">

		<table class="full-width">
			
<tr class="comment-tr target-hl" id="1" data-comment-id="100">
  <td class="comment-profile-avatar">
		<a href="/u/b_good"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/48_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="100"></a>

    

			<h2 class="page-title" style="margin-bottom:5px;margin-top:0px !important;">Text as a resource for network population?</h2>

			<div style="margin-top:5px;color:gray;margin-bottom:20px;font-size:13px;">
				<a style="font-weight:normal;color:#212121" href="/u/b_good"><strong>Benjamin Good</strong></a>
				
				&nbsp; <a href="#1" class="light">April 3, 2015</a>
				
			</div>

      

          <hr>
		

    <p>You say elsewhere that you want to avoid bias - that you want to work basically form experimental measurements as much as possible.  This has some merit, but it also seems like there must be quite a lot of value out there in the space of biased knowledge.. Some of that bias will be real signal.  Would be great to execute an experiment to empirically test the impact of bringing in prior knowledge (e.g. via text mining techniques for relation extraction).  Does it make it better or worse at the task at hand?</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="2" data-comment-id="120">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="120"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#2" class="light">April 8, 2015</a>
          
        </span>
      </div>

    

    <blockquote><p>there must be quite a lot of value out there in the space of biased knowledge</p></blockquote>

<p>You are correct. In our current proposal, we do use text mining for the symptom-disease edges and tissue-disease edges. We also rely on literature curation for compound-target binding and protein interactions.</p>

<blockquote><p>Would be great to execute an experiment to empirically test the impact of bringing in prior knowledge</p></blockquote>

<p>I agree, we should fit a model that excludes all knowledge-biased domains. I reckon this model's performance on known indications will be drastically inferior. The worry with predicting known indications with known biology is that your testing performance becomes nearly perfect. However, your novel predictions are not interesting — they would be readily apparent to a pharmacologist.</p>

<p>The more text mining data you include the larger your gap between testing performance and generative performance (see our <a href="http://thinklab.com/discussion/evaluation-framework/47#113">discussion on evaluation</a>). Therefore, I like the following workflow:</p>

<ol><li>Start with high-throughput resources that are not affected by knowledge bias (a.k.a. study bias)</li><li>If the algorithm performs significantly better than random, explore the top predictions.</li><li>If performance is mediocre, add a biased resource that provides orthogonal information (information not already included from a systematic resource)</li><li>Repeat.</li></ol>

<p>One final note to help explain the insidiousness of the knowledge bias. In <a href="http://ctdbase.org/">CTD</a> <span class="citation">[<a href="https://doi.org/10.1093/nar/gku935" class="citation" data-key="10.1093/nar/gku935">1</a>]</span>, curators may read that <code>compound_X</code> treats <code>disease_X</code> and also targets <code>gene_X</code> and therefore add interactions between all three entities to their knowledgebase. In reality the study hasn't proven that <code>gene_X</code> is associated with <code>disease_X</code> but this relationship was still extracted. This happens on a macro-scale across the entire compendium of published literature: specific network vicinities become well studied and the resulting disease-gene-compound triangles are more a result of attention rather than noteworthy biology.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="3" data-comment-id="365">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="365"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#3" class="light">Aug. 8, 2015</a>
          
        </span>
      </div>

    

    <h1>Knowledge biased and unbiased edges</h1>

<p>Thanks to <a href="/u/b_good" class="username">@b_good</a>'s suggestion for text mining and curated databases, we have incorporated several edges that are subject to knowledge bias.</p>

<p>Text mining edges include:</p>

<ul><li><em>Disease — causation — Symptom</em> edges from MEDLINE <a href="http://thinklab.com/discussion/mining-knowledge-from-medline-articles-and-their-indexed-mesh-terms/67#4">topic cooccurrence</a></li><li><em>Disease — similarity — Disease</em> edges from MEDLINE <a href="http://thinklab.com/discussion/disease-similarity-from-medline-topic-cooccurrence/93" class="citation thread-citation" data-key="10.15363/thinklab.d93">topic cooccurrence</a></li><li><em>Disease — localization — Anatomy</em> edges from MEDLINE <a href="http://thinklab.com/discussion/mining-knowledge-from-medline-articles-and-their-indexed-mesh-terms/67#5">topic cooccurrence</a></li></ul>

<p>Literature curation edges include:</p>

<ul><li><em>Compound — target — Gene</em> edges from <a href="http://thinklab.com/discussion/protein-target-carrier-transporter-and-enzyme-interactions-in-drugbank/65" class="citation thread-citation" data-key="10.15363/thinklab.d65">DrugBank</a> and <a href="http://thinklab.com/discussion/integrating-drug-target-information-from-bindingdb/53" class="citation thread-citation" data-key="10.15363/thinklab.d53">BindingDB</a></li><li><em>Gene — interaction — Gene</em> edges from <a href="http://thinklab.com/discussion/creating-a-catalog-of-protein-interactions/85" class="citation thread-citation" data-key="10.15363/thinklab.d85">curated databases</a></li><li><em>Gene — membership — Pathway</em> edges from <a href="http://thinklab.com/discussion/adding-pathway-resources-to-your-network/72" class="citation thread-citation" data-key="10.15363/thinklab.d72">WikiPathways and MSigDB</a></li><li><em>Gene — membership — GO Domain</em> edges from the <a href="http://thinklab.com/discussion/compiling-gene-ontology-annotations-into-an-easy-to-use-format/39#8">Gene Ontology</a> annotations</li><li><em>Gene — function — Disease</em> edges from the <a href="http://thinklab.com/discussion/functional-disease-annotations-for-genes-using-doaf/94" class="citation thread-citation" data-key="10.15363/thinklab.d94">DOAF</a></li></ul>

<p>And we have several edges from systematic technologies that are not subject to knowledge biases.</p>

<ul><li><em>Disease — expression — Anatomy</em> edges from <a href="http://thinklab.com/discussion/tissue-specific-gene-expression-resources/81#8">Bgee</a> and <a href="http://thinklab.com/discussion/the-tissues-resource-for-the-tissue-specificity-of-genes/91" class="citation thread-citation" data-key="10.15363/thinklab.d91">TISSUES</a></li><li><em>Disease — up/down-regulation — Gene</em> edges from <a href="http://thinklab.com/discussion/stargeo-expression-signatures-for-disease-using-crowdsourced-geo-annotation/96" class="citation thread-citation" data-key="10.15363/thinklab.d96">STARGEO</a></li><li><em>Gene — membership — Perturbation Gene Set</em> edges from MSigDB</li><li><em>Gene — interaction — Gene</em> edges from <a href="http://thinklab.com/discussion/creating-a-catalog-of-protein-interactions/85#7">Y2H experiments</a></li><li><em>Gene — evolution — Gene</em> edges from <a href="http://thinklab.com/discussion/selecting-informative-erc-evolutionary-rate-covariation-values-between-genes/57" class="citation thread-citation" data-key="10.15363/thinklab.d57">evolutionary rate covariation</a></li><li><em>Gene — knowdown up/down-reglulation — Gene</em> edges from <a href="http://thinklab.com/discussion/suggestions-for-additional-information-types/22#6">LINCS L1000</a></li><li><em>Compound — up/down-regulation — Gene</em> from <a href="http://thinklab.com/discussion/computing-consensus-transcriptional-profiles-for-compounds-using-lincs/43#6">LINCS L1000</a></li></ul>

<p>Thus, for each edge we will create an <code>unbiased</code> attribute which takes a <code>True</code> or <code>False</code> value. <strong>Using our network masking feature, we can easily switch between using the whole network or only the knowledge-unbiased portion.</strong></p>

<p>Some metaedges will contain a mix of biased and unbiased edges. For example, protein interactions based on their source database. When both a biased and unbiased source contribute an edge, we will give precedence to the unbiased designation.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
<li id="note-166" class="inline-note toggle-parent" data-note-id="166">
	<p><a href="/u/b_good">Benjamin Good</a>: I'm wondering about your relation ontology here.  Can you really distinguish a causal relation from co-occurrence information?  Do the names on these edges actually matter based on how you are using the network ?  If the meaning of the relations is really important, I think there are other text-mining approaches that you should look into.  If not, the co-occurrence stuff should be fine.  Really curious about this experiment..  Would also like to see how the result of other text-ming approaches would influence the outcome.  e.g. would it change things if you swapped in the relations from semmedDB ?
		
	</p>
	
	<table style="display:none;"></table>
</li>

				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="4" data-comment-id="412">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="412"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#4" class="light">Aug. 20, 2015</a>
          
        </span>
      </div>

    

    <blockquote><p>Do the names on these edges actually matter based on how you are using the network?</p></blockquote>

<p><a href="/u/b_good" class="username">@b_good</a>, for predicting the probability of efficacy of a compound–disease pair, the metaedge names do not matter. The <a href="http://het.io/hnep">algorithm</a> only considers the structure of the network. The names are used to assist with interpretability. For example, the <em>CtGad</em> feature (capturing when a compound targets genes that are associated with the disease) may be predictive. In this case, we would conclude that disease-associated genes are informative for repurposing. If what <a href="http://thinklab.com/discussion/processing-disgenet-for-disease-gene-relationships/105#6" class="citation thread-citation" data-key="10.15363/thinklab.d105">we call</a> an 'association' is actually some other type of relationship, then the interpretation that associations are influential will be unfounded.</p>

<p>When we have multiple metaedges between the same metanodes, we hope there is a difference in the type of information encoded. Otherwise, we would be better off having only a single metaedge. For example, we included <a href="http://thinklab.com/discussion/integrating-drug-target-information-from-bindingdb/53#5" class="citation thread-citation" data-key="10.15363/thinklab.d53">binding</a> and <a href="http://thinklab.com/discussion/protein-target-carrier-transporter-and-enzyme-interactions-in-drugbank/65#1" class="citation thread-citation" data-key="10.15363/thinklab.d65">target</a> edges between compounds and genes. It is unclear whether merging these edges would be beneficial, because it's difficult to know how they differ. Therefore, a good understanding what information each metaedge captures will assist with metagraph design. Accurate metaedge names can help with understanding edge content and therefore network design decisions.</p>

<blockquote><p>Would also like to see how the result of other text-ming approaches would influence the outcome. e.g. would it change things if you swapped in the relations from semmedDB?</p></blockquote>

<p>Currently, I am happy with the quality of our <a href="http://thinklab.com/discussion/mining-knowledge-from-medline-articles-and-their-indexed-mesh-terms/67" class="citation thread-citation" data-key="10.15363/thinklab.d67">MEDLINE topic cooccurrence</a> approach. I assume that you highlight <a href="http://skr3.nlm.nih.gov/SemMedDB/">SemMedDB</a> <span class="citation">[<a href="https://doi.org/10.1093/bioinformatics/bts591" class="citation" data-key="10.1093/bioinformatics/bts591">1</a>]</span> because it has the ability to extract the type of relationship. I agree this could be a valuable addition. However, in the interest of time, this will most likely have to wait till a successive project.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

    		
		</table>

    

    &nbsp;<br>
    <!-- <a class="btn btn-primary btn-lg" href="/signup?next=/discussion/text-as-a-resource-for-network-population/48&suppress_message=1" style>Join to Reply</a> -->


		<table class="references table-td-5" style="display:none;"><tr><td><a href='/doi/10.1093/nar/gku935' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>1.</div></td><td><div class='doi-body' data-key='10.1093/nar/gku935'><a href='/doi/10.1093/nar/gku935' class='pub-title'>The Comparative Toxicogenomics Database&#39;s 10th year anniversary: update 2015</a><div class='pub-body'>A. P. Davis, C. J. Grondin, K. Lennon-Hopkins, C. Saraceni-Richards, D. Sciaky, B. L. King, T. C. Wiegers, C. J. Mattingly (2014) <i>Nucleic Acids Research</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/nar/gku935'>10.1093/nar/gku935</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d47' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>2.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d47'><a href='/d/47' class='pub-title'>Evaluation framework</a><div class='pub-body'>Benjamin Good, Daniel Himmelstein (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d47'>10.15363/thinklab.d47</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d67' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>3.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d67'><a href='/d/67' class='pub-title'>Mining knowledge from MEDLINE articles and their indexed MeSH terms</a><div class='pub-body'>Daniel Himmelstein, Alex Pankov (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d67'>10.15363/thinklab.d67</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d93' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>4.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d93'><a href='/d/93' class='pub-title'>Disease similarity from MEDLINE topic cooccurrence</a><div class='pub-body'>Daniel Himmelstein (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d93'>10.15363/thinklab.d93</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d67' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>5.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d67'><a href='/d/67' class='pub-title'>Mining knowledge from MEDLINE articles and their indexed MeSH terms</a><div class='pub-body'>Daniel Himmelstein, Alex Pankov (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d67'>10.15363/thinklab.d67</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d65' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>6.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d65'><a href='/d/65' class='pub-title'>Protein (target, carrier, transporter, and enzyme) interactions in DrugBank</a><div class='pub-body'>Daniel Himmelstein, Sabrina Chen (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d65'>10.15363/thinklab.d65</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d53' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>7.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d53'><a href='/d/53' class='pub-title'>Integrating drug target information from BindingDB</a><div class='pub-body'>Daniel Himmelstein, Mike Gilson (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d53'>10.15363/thinklab.d53</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d85' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>8.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d85'><a href='/d/85' class='pub-title'>Creating a catalog of protein interactions</a><div class='pub-body'>Daniel Himmelstein, Dexter Hadley, Alexey Strokach (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d85'>10.15363/thinklab.d85</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d72' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>9.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d72'><a href='/d/72' class='pub-title'>Adding pathway resources to your network</a><div class='pub-body'>Alexander Pico, Daniel Himmelstein (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d72'>10.15363/thinklab.d72</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d39' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>10.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d39'><a href='/d/39' class='pub-title'>Compiling Gene Ontology annotations into an easy-to-use format</a><div class='pub-body'>Daniel Himmelstein, Casey Greene, Venkat Malladi, Frederic Bastian (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d39'>10.15363/thinklab.d39</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d94' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>11.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d94'><a href='/d/94' class='pub-title'>Functional disease annotations for genes using DOAF</a><div class='pub-body'>Daniel Himmelstein (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d94'>10.15363/thinklab.d94</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d81' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>12.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d81'><a href='/d/81' class='pub-title'>Tissue-specific gene expression resources</a><div class='pub-body'>Daniel Himmelstein, Frederic Bastian (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d81'>10.15363/thinklab.d81</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d91' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>13.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d91'><a href='/d/91' class='pub-title'>The TISSUES resource for the tissue-specificity of genes</a><div class='pub-body'>Daniel Himmelstein, Lars Juhl Jensen (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d91'>10.15363/thinklab.d91</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d96' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>14.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d96'><a href='/d/96' class='pub-title'>STARGEO: expression signatures for disease using crowdsourced GEO annotation</a><div class='pub-body'>Daniel Himmelstein, Frederic Bastian, Dexter Hadley, Casey Greene (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d96'>10.15363/thinklab.d96</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d85' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>15.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d85'><a href='/d/85' class='pub-title'>Creating a catalog of protein interactions</a><div class='pub-body'>Daniel Himmelstein, Dexter Hadley, Alexey Strokach (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d85'>10.15363/thinklab.d85</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d57' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>16.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d57'><a href='/d/57' class='pub-title'>Selecting informative ERC (evolutionary rate covariation) values between genes</a><div class='pub-body'>Daniel Himmelstein, Raghavendran Partha (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d57'>10.15363/thinklab.d57</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d22' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>17.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d22'><a href='/d/22' class='pub-title'>Suggestions for additional information types?</a><div class='pub-body'>Daniel Himmelstein, Venkat Malladi, Alessandro Didonna, Alexey Strokach (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d22'>10.15363/thinklab.d22</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d43' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>18.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d43'><a href='/d/43' class='pub-title'>Computing consensus transcriptional profiles for LINCS L1000 perturbations</a><div class='pub-body'>Daniel Himmelstein, Caty Chung (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d43'>10.15363/thinklab.d43</a></div></div></td></tr><tr><td><a href='/doi/10.1093/bioinformatics/bts591' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>19.</div></td><td><div class='doi-body' data-key='10.1093/bioinformatics/bts591'><a href='/doi/10.1093/bioinformatics/bts591' class='pub-title'>SemMedDB: a PubMed-scale repository of biomedical semantic predications</a><div class='pub-body'>H. Kilicoglu, D. Shin, M. Fiszman, G. Rosemblat, T. C. Rindflesch (2012) <i>Bioinformatics</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/bioinformatics/bts591'>10.1093/bioinformatics/bts591</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d105' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>20.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d105'><a href='/d/105' class='pub-title'>Processing DisGeNET for disease-gene relationships</a><div class='pub-body'>Daniel Himmelstein, janet piñero (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d105'>10.15363/thinklab.d105</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d53' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>21.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d53'><a href='/d/53' class='pub-title'>Integrating drug target information from BindingDB</a><div class='pub-body'>Daniel Himmelstein, Mike Gilson (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d53'>10.15363/thinklab.d53</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d65' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>22.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d65'><a href='/d/65' class='pub-title'>Protein (target, carrier, transporter, and enzyme) interactions in DrugBank</a><div class='pub-body'>Daniel Himmelstein, Sabrina Chen (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d65'>10.15363/thinklab.d65</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d67' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>23.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d67'><a href='/d/67' class='pub-title'>Mining knowledge from MEDLINE articles and their indexed MeSH terms</a><div class='pub-body'>Daniel Himmelstein, Alex Pankov (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d67'>10.15363/thinklab.d67</a></div></div></td></tr></table>

    


	</td>
	<td class="thread-sidebar">


			

      <!-- <a class="btn btn-default follow-link btn-follow btn-lg is-following tt-top full-width" title="Follow discussion" data-id="48" data-ctype="thread" data-is-following="false">&nbsp;</a> -->

      
        <div class="status-box text-center"><span style="">Status:</span> 
  Accepted

</div>
      

      
        
      


				

					<div class="details">Views</div>
					<span style="font-size:20px;">102</span>

        

				
					<div class="details">Topics</div>
					<div style="margin-bottom:15px;">
		            <a href="/topics/natural-language-processing" class="tag">Natural Language Processing</a><a href="/topics/prior-knowledge" class="tag">Prior Knowledge</a><a href="/topics/relation-extraction" class="tag">Relation Extraction</a><a href="/topics/bias" class="tag">Bias</a>
					</div>
				


		
			<div class="details">Referenced by</div>
			<div style="font-size:12px;"><ul style="padding-left:16px !important;">
			
				<li>
					
					<a href='/doc/7/review' class='thread-title'><span class='inbox-icon tt-top glyphicon glyphicon-flash'></span>Report Review: Rephetio: Repurposing drugs on a hetnet</a>
				</li>
			
				<li>
					
					<a href='/d/102' class='thread-title'>One network to rule them all</a>
				</li>
			
				<li>
					
					<a href='/d/227' class='thread-title'>Brainstorming future directions for Hetionet</a>
				</li>
			
				<li>
					
					<a href='/d/67' class='thread-title'>Mining knowledge from MEDLINE articles and their indexed MeSH terms</a>
				</li>
			
				<li>
					
					<a href='/d/85' class='thread-title'>Creating a catalog of protein interactions</a>
				</li>
			
				<li>
					
					<a href='/d/91' class='thread-title'>The TISSUES resource for the tissue-specificity of genes</a>
				</li>
			
				<li>
					
					<a href='/d/52' class='thread-title'>Human Symptom Disease Network-MeSH ID Matching</a>
				</li>
			
				</ul>
			</div>
		

		
			<div class="details">Cite this as</div>
			<span style="font-size:12px;">
				Benjamin Good, Daniel Himmelstein (2015) Text as a resource for network population?. <i>Thinklab.</i> doi:<a href='https://doi.org/10.15363/thinklab.d48'>10.15363/thinklab.d48</a>
			</span>
		

		<div class="details">License</div>
		<p><a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0;width:88px;height:31px;" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a></p>

		<div class="details">Share</div>
		
			
				<a class="share-button fb" onclick="javascript:void(window.open('https://www.facebook.com/sharer/sharer.php?u=https%3A//thinklab.org/discussion/text-as-a-resource-for-network-population/48','','width=600,height=460,menubar=no,status=no'));" href="#"></a>
<a class="share-button twitter" onclick="javascript:void(window.open('https://twitter.com/share?text=Text%20as%20a%20resource%20for%20network%20population%3F&url=https%3A//thinklab.org/discussion/text-as-a-resource-for-network-population/48','','width=600,height=300,menubar=no,status=no'));" href="#"></a>
<a class="share-button google" onclick="javascript:void(window.open('https://plus.google.com/share?url=https%3A//thinklab.org/discussion/text-as-a-resource-for-network-population/48','','width=510,height=460,menubar=no,status=no'));" href="#"></a>

			
		

	</td>
</tr></table>


		</div>


		
	    	<div class="footer">
<!-- <div style="background:#dedede;">
	<div class="main-centered-div text-center" style="padding:20px;">
		<h4 style="margin-bottom:0px;font-weight:bold;color:gray;">Feedback? Questions? Email us: <a href="mailto:hello@thinklab.com">hello@thinklab.com</a></h4>
	</div>
</div> -->
<div class="main-footer">
	<div class="main-centered-div">

<table class="full-width"><tr>
	<td class="footer-nav">

        <div class="foot-head">Explore</div>
        <a href="/proposals">Proposals</a><br>
        <a href="/projects">Projects</a><br>
        <a href="/publications">Article discussion</a><br>
        <a href="/discussion">All discussion</a><br>
        <a href="/topics">Topics</a><br>
        <!-- <a href="/funders">Funders</a><br> -->
        <a href="/funding">Funding opportunities</a><br>

	</td>
	<td class="footer-nav">

        <!-- <div class="foot-head">Help &amp; feedback</div>	 -->
        <!-- <div class="foot-head">Intro</div>
        <a href="/benefits/for-science">Why</a><br>
        <a href="/how-it-works">How it works</a><br>

<br> -->

        <div class="foot-head">Community</div>
        <a href="/leaderboard">Leaderboard</a><br>
        <a href="/p/meta">Thinklab Meta</a><br>
				<a href="/help/faq">Help</a><br>
				<!-- <a href="/new-project">Start a proposal</a><br> -->
        <!-- <a href="/benefits/proposal">Benefits</a><br> -->
        <!-- <a href="/funding">Get funded</a><br> -->
        <!-- <a href="/participate">Become a contributor</a><br> -->
        <!-- <a href="/for-funders">For science funders</a><br> -->
        <!-- <a href="/for-institutions">For institutions</a><br> -->

	</td>
	<td class="footer-nav">



        <div class="foot-head">About</div>
        <a href="/about">Overview</a><br>
        <a href="/benefits">Benefits</a><br>
        <a href="/how-it-works">How it works</a><br>
        <a href="/about/story">Our story</a><br>
        <a href="/blog">Our blog</a><br>
        <a href="/about/jobs">Join the team</a><br>
        <br>

        <!-- <div class="foot-head">Help</div>
        <a href="/help">Help</a><br>
		 -->


	</td>
	<td class="footer-nav">
		<!-- <a href="http://twitter.com/thinklab" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/twitter_com.png" width="40"/></a>
		<a href="https://www.facebook.com/pages/Thinklab/310895039088185" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/facebook_com.png" width="40"/></a>
		<a href="https://plus.google.com/115578983466800753402/" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/plus_google_com.png" width="40"/></a>
		<a href="https://www.linkedin.com/company/thinklab-co" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/linkedin_com.png" width="40"/></a> -->
		<br><br>
		<span class="light" style='font-size:12px;'>&copy; 2015 Thinklab</span>

	</td>

</tr></table>

</div>
</div>
</div>

		

	</div>


  <script></script>
  <script>
  var MEDIA_URL = 'https://think-lab.s3.amazonaws.com/m/';
  var STATIC_URL = 'https://think-lab.s3.amazonaws.com/s/';
  var page_is_authenticated = false;
  var page_page = 'discussion';
  var page_profile_id = undefined;
</script>



	<!-- Note different script type. MUST come before MathJax.js -->
	<script type="text/x-mathjax-config">
	    MathJax.Hub.Config({
	        "HTML-CSS": { preferredFont: "TeX", availableFonts: ["STIX","TeX"],
	                        scale: 110, linebreaks: { automatic:true },
	                        EqnChunk: (MathJax.Hub.Browser.isMobile ? 10 : 50) },
	        tex2jax: { inlineMath: [ ["$$", "$$"], ["\\(","\\)"] ],
	                    displayMath: [ ["$$$","$$$"], ["\\[", "\\]"] ],
	                    processEscapes: true,}
	            });

	</script>
    <script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>





  <script src="https://code.jquery.com/jquery-1.10.2.min.js"></script>
  <script src="https://code.jquery.com/ui/1.11.4/jquery-ui.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" integrity="sha384-0mSbJDEHialfmuBBQP6A4Qrprq5OVfW37PRR3j5ELqxss1yVqOtnepnHVP9aJ7xS" crossorigin="anonymous"></script>
  <script src="https://think-lab.s3.amazonaws.com/s/dist/main.min.js"></script>
  



  
	
  
  
  
<script>
$(function() {
  recordHit('thread', 48);
});
</script>



</body>
</html>
