<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8" /><script type="text/javascript">window.NREUM||(NREUM={}),__nr_require=function(e,n,t){function r(t){if(!n[t]){var o=n[t]={exports:{}};e[t][0].call(o.exports,function(n){var o=e[t][1][n];return r(o||n)},o,o.exports)}return n[t].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<t.length;o++)r(t[o]);return r}({1:[function(e,n,t){function r(){}function o(e,n,t){return function(){return i(e,[c.now()].concat(u(arguments)),n?null:this,t),n?void 0:this}}var i=e("handle"),a=e(2),u=e(3),f=e("ee").get("tracer"),c=e("loader"),s=NREUM;"undefined"==typeof window.newrelic&&(newrelic=s);var p=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],d="api-",l=d+"ixn-";a(p,function(e,n){s[n]=o(d+n,!0,"api")}),s.addPageAction=o(d+"addPageAction",!0),s.setCurrentRouteName=o(d+"routeName",!0),n.exports=newrelic,s.interaction=function(){return(new r).get()};var m=r.prototype={createTracer:function(e,n){var t={},r=this,o="function"==typeof n;return i(l+"tracer",[c.now(),e,t],r),function(){if(f.emit((o?"":"no-")+"fn-start",[c.now(),r,o],t),o)try{return n.apply(this,arguments)}finally{f.emit("fn-end",[c.now()],t)}}}};a("setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(e,n){m[n]=o(l+n)}),newrelic.noticeError=function(e){"string"==typeof e&&(e=new Error(e)),i("err",[e,c.now()])}},{}],2:[function(e,n,t){function r(e,n){var t=[],r="",i=0;for(r in e)o.call(e,r)&&(t[i]=n(r,e[r]),i+=1);return t}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],3:[function(e,n,t){function r(e,n,t){n||(n=0),"undefined"==typeof t&&(t=e?e.length:0);for(var r=-1,o=t-n||0,i=Array(o<0?0:o);++r<o;)i[r]=e[n+r];return i}n.exports=r},{}],4:[function(e,n,t){n.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(e,n,t){function r(){}function o(e){function n(e){return e&&e instanceof r?e:e?f(e,u,i):i()}function t(t,r,o,i){if(!d.aborted||i){e&&e(t,r,o);for(var a=n(o),u=m(t),f=u.length,c=0;c<f;c++)u[c].apply(a,r);var p=s[y[t]];return p&&p.push([b,t,r,a]),a}}function l(e,n){v[e]=m(e).concat(n)}function m(e){return v[e]||[]}function w(e){return p[e]=p[e]||o(t)}function g(e,n){c(e,function(e,t){n=n||"feature",y[t]=n,n in s||(s[n]=[])})}var v={},y={},b={on:l,emit:t,get:w,listeners:m,context:n,buffer:g,abort:a,aborted:!1};return b}function i(){return new r}function a(){(s.api||s.feature)&&(d.aborted=!0,s=d.backlog={})}var u="nr@context",f=e("gos"),c=e(2),s={},p={},d=n.exports=o();d.backlog=s},{}],gos:[function(e,n,t){function r(e,n,t){if(o.call(e,n))return e[n];var r=t();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(e,n,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return e[n]=r,r}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],handle:[function(e,n,t){function r(e,n,t,r){o.buffer([e],r),o.emit(e,n,t)}var o=e("ee").get("handle");n.exports=r,r.ee=o},{}],id:[function(e,n,t){function r(e){var n=typeof e;return!e||"object"!==n&&"function"!==n?-1:e===window?0:a(e,i,function(){return o++})}var o=1,i="nr@id",a=e("gos");n.exports=r},{}],loader:[function(e,n,t){function r(){if(!x++){var e=h.info=NREUM.info,n=d.getElementsByTagName("script")[0];if(setTimeout(s.abort,3e4),!(e&&e.licenseKey&&e.applicationID&&n))return s.abort();c(y,function(n,t){e[n]||(e[n]=t)}),f("mark",["onload",a()+h.offset],null,"api");var t=d.createElement("script");t.src="https://"+e.agent,n.parentNode.insertBefore(t,n)}}function o(){"complete"===d.readyState&&i()}function i(){f("mark",["domContent",a()+h.offset],null,"api")}function a(){return E.exists&&performance.now?Math.round(performance.now()):(u=Math.max((new Date).getTime(),u))-h.offset}var u=(new Date).getTime(),f=e("handle"),c=e(2),s=e("ee"),p=window,d=p.document,l="addEventListener",m="attachEvent",w=p.XMLHttpRequest,g=w&&w.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:w,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var v=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-1044.min.js"},b=w&&g&&g[l]&&!/CriOS/.test(navigator.userAgent),h=n.exports={offset:u,now:a,origin:v,features:{},xhrWrappable:b};e(1),d[l]?(d[l]("DOMContentLoaded",i,!1),p[l]("load",r,!1)):(d[m]("onreadystatechange",o),p[m]("onload",r)),f("mark",["firstbyte",u],null,"api");var x=0,E=e(4)},{}]},{},["loader"]);</script><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info={"beacon":"bam.nr-data.net","queueTime":0,"licenseKey":"cd1101aa94","agent":"","transactionName":"bgcDMkBZW0NZBkdcWVdNJxNcW0FZVwscV1dKB08QW11CQxYTWlBBZgEOC19dW0QCEVtHU1gG","applicationID":"6369297","errorBeacon":"bam.nr-data.net","applicationTime":381}</script>
    <title>Tissue Node | Thinklab</title>
	  <link rel="shortcut icon" href="https://think-lab.s3.amazonaws.com/s/img/favicon.ico" type="image/x-icon" />
    <meta property="og:image" content="https://think-lab.s3.amazonaws.com/s/img" />
    
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
  <link rel="stylesheet" href="https://code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css" />

<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/highlightjs/github.css" />
<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/css/css.css" />
<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/css/bootstrap-multiselect.css" />
<!-- <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'> -->

    
    
    
    
  
    <meta name="citation_doi" content="10.15363/thinklab.d41">
    <meta name="citation_title" content="Tissue Node">
    <meta name="citation_online_date" content="March 19, 2015">
    <meta name="citation_author" content="Venkat Malladi, Daniel Himmelstein, Chris Mungall">
    <meta name="citation_journal_title" content="Thinklab">
    <meta name="citation_publisher" content="Thinklab">
  

    


	<!-- GoogleAnalytics -->
	<script>
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
	(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
	m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

	ga('create', 'UA-48550230-1', 'thinklab.org');
	ga('require', 'displayfeatures');
	
	ga('send', 'pageview');

	</script>






</head>
<body >
  <form id="form_action" method="post">
    <input type='hidden' name='csrfmiddlewaretoken' value='JC3hnVSWzcwBOS226QTq2eGyHXj1YAM6' />
</form>

	

	<div class="body-wrapper">

		


<!-- <div style="height:5px;background-image:url('https://think-lab.s3.amazonaws.com/s/img/top-bg.gif');"></div> -->
<div class="gradient">
	<div class="main-centered-div header-container">

<table class="full-width" style="height:60px;"><tr>
	<td style="width:200px;vertical-align:middle;padding-bottom:15px;">
		<a href="/" style="font-weight:normal;color:#212121;"><img src="https://think-lab.s3.amazonaws.com/s/img/logo.png" width="160"></a>
	</td><td style="vertical-align:middle;text-align:left;padding-top:6px;width:100%;padding-left:40px;">
		<ul id="not-search-block"  class="main-header">


			


				<li class="dropdown">
					<a class="dropdown-toggle" data-toggle="dropdown" href="#">Browse <span class="caret"></span></a>
					<ul class="dropdown-menu" role="menu">
						<li><a href="/proposals">Proposals</a></li>
						<li><a href="/projects">Projects</a></li>
						<li><a href="/publications">Article discussion</a></li>
						<li><a href="/discussion">All discussion</a></li>
					</ul>
				</li>
				<li><a href="/leaderboard">Leaderboard</a></li>

			
				<li><a href="/about">About</a></li>
			


		</ul>

    <ul id="search-block" class="main-header" style="display: none; margin-right: 30px;">
        <li class="full-width">
            <form method="get" action="/search">
			<div class="input-group full-width">
				<input name="q" type="text" class="form-control search-input">
			</div>
            </form>
        </li>
    </ul>

	</td><td style="vertical-align:middle;text-align:right;padding-top:6px; min-width: 1%; white-space: nowrap;">
		<ul class="main-header pull-right" style="padding-left: 0;">
    		
                <!-- <li><a href="/login" style="color:silver;">Login</a></li>
                <li style="margin-right:0"><a class="btn btn-default btn-sm btn-upper" href="/signup">Join</a></li> -->
            

        </ul>
    </ul>
	</td>
</tr></table>

<!-- <div id="global-search-div" class="main-centered-div" style="display:none;">
	<input type="text" value="" class="thread-search form-control" placeholder="Search" data-action="/discussion">
</div> -->

	</div>
</div>

		
	  



	  


		

		<div class="dark-head"><div class="main-centered-div">

			

		</div></div>
		
		<div class="main-centered-div" style="margin-top:40px;padding-bottom:340px">
	    

<div style="margin-top:-20px; margin-bottom:20px;">

    <a class="side-box" href="/p/rephetio" style="background-color:#ebfdff;"><div class="side-type">Project:</div> <div class="side-title">Rephetio: Repurposing drugs on a hetnet [rephetio]</div></a><br>



    <a class="side-box" href="/doi/10.1186/gb-2012-13-1-r5" style="background-color:#f0ffec;"><div class="side-type">Publication:</div> <div class="side-title">Uberon, an integrative multi-species anatomy ontology</div></a><br>

</div>

<table style="margin-top:0px;"><tr>
	<td class="thread-table-content">

		<table class="full-width">
			
<tr class="comment-tr target-hl" id="1" data-comment-id="84">
  <td class="comment-profile-avatar">
		<a href="/u/vsmalladi"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/s/img/p_default_medium.png' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="84"></a>

    

			<h2 class="page-title" style="margin-bottom:5px;margin-top:0px !important;">Tissue Node</h2>

			<div style="margin-top:5px;color:gray;margin-bottom:20px;font-size:13px;">
				<a style="font-weight:normal;color:#212121" href="/u/vsmalladi"><strong>Venkat Malladi</strong></a>
				
				&nbsp; <a href="#1" class="light">March 19, 2015</a>
				
			</div>

      

          <hr>
		

    <p>Instead of using <a href="http://www.brenda-enzymes.info/ontology.php?ontology_id=3">Brenda Tissue Ontology</a>, I would suggest using <a href="http://uberon.org">Uberon</a> for anatomy which incorporates <a href="http://cellontology.org/">CL</a>.These ontologies provide greater breadth of anatomy and cell types for various species. In addition, I think that with integration with Disease Ontology into <a href="http://www.ebi.ac.uk/efo">EFO</a> you can add more data sources to this network as well as links to further experiments. </p>

<p>ENCODE makes use of Uberon, you can read more  <span class="citation">[<a href="https://doi.org/10.1093/database/bav010" class="citation" data-key="10.1093/database/bav010">1</a>]</span></p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
<li id="note-59" class="inline-note toggle-parent" data-note-id="59">
	<p><a href="/u/dhimmel">Daniel Himmelstein</a>: <span><a href="/u/vsmalladi" class="username">@vsmalladi</a>, you may consider switching the associated publication to the Uberon paper <span class="citation">[<a href="https://doi.org/10.1186/gb-2012-13-1-r5" class="citation" data-key="10.1186/gb-2012-13-1-r5">1</a>]</span> and move the ENCODE use-case paper <span class="citation">[<a href="https://doi.org/10.1093/database/bav010" class="citation" data-key="10.1093/database/bav010">2</a>]</span> as an <a href="http://thinklab.com/help/writing-in-markdown">inline citation</a>.</span>
		
	</p>
	
	<table style="display:none;"><tr><td><a href='/doi/10.1186/gb-2012-13-1-r5' class='pub-comments tt-right' title='Article comments'>16</a></td><td><div class='pub-order'>1.</div></td><td><div class='doi-body' data-key='10.1186/gb-2012-13-1-r5'><a href='/doi/10.1186/gb-2012-13-1-r5' class='pub-title'>Uberon, an integrative multi-species anatomy ontology</a><div class='pub-body'>Christopher J Mungall, Carlo Torniai, Georgios V Gkoutos, Suzanna E Lewis, Melissa A Haendel (2012) <i>Genome Biol</i>. doi:<a class='pub-doi' href='https://doi.org/10.1186/gb-2012-13-1-r5'>10.1186/gb-2012-13-1-r5</a></div></div></td></tr><tr><td><a href='/doi/10.1093/database/bav010' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>2.</div></td><td><div class='doi-body' data-key='10.1093/database/bav010'><a href='/doi/10.1093/database/bav010' class='pub-title'>Ontology application and use at the ENCODE DCC</a><div class='pub-body'>V. S. Malladi, D. T. Erickson, N. R. Podduturi, L. D. Rowe, E. T. Chan, J. M. Davidson, B. C. Hitz, M. Ho, B. T. Lee, S. Miyasato, G. R. Roe, M. Simison, C. A. Sloan, J. S. Strattan, F. Tanaka, W. J. Kent, J. M. Cherry, E. L. Hong (2015) <i>Database</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/database/bav010'>10.1093/database/bav010</a></div></div></td></tr></table>
</li>

<li id="note-60" class="inline-note toggle-parent" data-note-id="60">
	<p><a href="/u/dhimmel">Daniel Himmelstein</a>: <span><a href="/u/jspauld" class="username">@jspauld</a>, see the citation display failure in the previous note.</span>
		
	</p>
	
	<table style="display:none;"></table>
</li>

<li id="note-61" class="inline-note toggle-parent" data-note-id="61">
	<p><a href="/u/jspauld">Jesse Spaulding</a>: <span><a href="/u/dhimmel" class="username">@dhimmel</a> Yeah, the inline citations don't work in the notes right now. I suspect it will be rare that people will try to use them in notes given that most of the substantive conversation is intended to take place in comments. So, for the time being I think I'll wait and see how much demand there will be. [<strong>Edit:</strong> After speaking with <a href="/u/dhimmel" class="username">@dhimmel</a> offline he has convinced me this was a bug. Inline note citations now work!]</span>
		
	</p>
	
	<table style="display:none;"></table>
</li>

<li id="note-63" class="inline-note toggle-parent" data-note-id="63">
	<p><a href="/u/jspauld">Jesse Spaulding</a>: With regard to the associated publication — at the present moment this cannot be changed by the user. <a href="/u/vsmalladi" class="username">@vsmalladi</a>, if you agree it makes sense to change it please just let me know and I'll take care of it.
		
	</p>
	
	<table style="display:none;"></table>
</li>

<li id="note-64" class="inline-note toggle-parent" data-note-id="64">
	<p><a href="/u/vsmalladi">Venkat Malladi</a>: <span><a href="/u/jspauld" class="username">@jspauld</a> yes please update the citation</span>
		
	</p>
	
	<table style="display:none;"></table>
</li>

<li id="note-66" class="inline-note toggle-parent" data-note-id="66">
	<p><a href="/u/jspauld">Jesse Spaulding</a>: Updated. If you'd like to do an inline citation to the original DOI you were referencing just insert the following in your markdown: <code>[@10.1093/database/bav010]</code>
		
	</p>
	
	<table style="display:none;"><tr><td><a href='/doi/10.1093/database/bav010' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>1.</div></td><td><div class='doi-body' data-key='10.1093/database/bav010'><a href='/doi/10.1093/database/bav010' class='pub-title'>Ontology application and use at the ENCODE DCC</a><div class='pub-body'>V. S. Malladi, D. T. Erickson, N. R. Podduturi, L. D. Rowe, E. T. Chan, J. M. Davidson, B. C. Hitz, M. Ho, B. T. Lee, S. Miyasato, G. R. Roe, M. Simison, C. A. Sloan, J. S. Strattan, F. Tanaka, W. J. Kent, J. M. Cherry, E. L. Hong (2015) <i>Database</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/database/bav010'>10.1093/database/bav010</a></div></div></td></tr></table>
</li>

				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="2" data-comment-id="86">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="86"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#2" class="light">March 20, 2015</a>
          
        </span>
      </div>

    

    <p>Hi <a href="/u/vsmalladi" class="username">@vsmalladi</a>, thanks for the <a href="https://uberon.github.io/">Uberon</a> <span class="citation">[<a href="https://doi.org/10.1186/gb-2012-13-1-r5" class="citation" data-key="10.1186/gb-2012-13-1-r5">1</a>]</span> suggestion.</p>

<p>The project has good documentation, a nice user interface, and is <a href="https://github.com/obophenotype/uberon/commits/master">actively maintained</a> — three important features when choosing an ontology (and areas where the <a href="http://www.brenda-enzymes.info/ontology.php?ontology_id=3">BRENDA Tissue Ontology</a> (BTO) <span class="citation">[<a href="https://doi.org/10.1093/nar/gkq968" class="citation" data-key="10.1093/nar/gkq968">2</a>]</span> sometimes lags behind.</p>

<p>In our previous network <span class="citation">[<a href="https://doi.org/10.1371/journal.pcbi.1004259" class="citation" data-key="10.1371/journal.pcbi.1004259">3</a>]</span>, we had under 100 tissues and only used BTO as a common vocabulary. Since the current project is in an early stage, it is difficult to know whether we will take full advantage of the rich structure and cross-referencing provided by next-generation ontologies. However, I agree it makes sense to build an extensible and forward-thinking network and Uberon will assist in these pursuits. We also prefer ontologies that will have the widest adoption and are free of restrictive licensing. Do you know Uberon's license?</p>

<p>I noticed Uberon includes <a href="https://github.com/obophenotype/uberon/wiki/inter-anatomy-ontology-bridge-ontologies">mappings to other ontologies</a>, called <em>bridges</em>. A BTO bridge doesn't currently exist, but perhaps we could contribute one for the 77 BTO terms (<a href="http://het.io/disease-genes/downloads/files/expression.txt.gz">download link</a>) used in our disease network.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="3" data-comment-id="88">
  <td class="comment-profile-avatar">
		<a href="/u/vsmalladi"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/s/img/p_default_medium.png' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="88"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/vsmalladi">Venkat Malladi</a>
				
				<span class="light">&nbsp;<a href="#3" class="light">March 20, 2015</a>
          
        </span>
      </div>

    

    <p>Hi <a href="/u/dantericci" class="username">@dantericci</a>, I believe there is no restrictive licensing and is open to use. </p>

<p>As for adpotion, they have many big projects and consotrium adopting use of Uberon. Such as EBI, <a href="http://uberon.github.io/about/adopters.html">other adoptors</a>.</p>

<p>Uberon does have cross-references to BTO, so I don't think we need to make a bridge. </p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
<li id="note-67" class="inline-note toggle-parent" data-note-id="67">
	<p><a href="/u/jspauld">Jesse Spaulding</a>: FYI, it looks like you mentioned the wrong person here. I'm assuming you meant <a href="/u/dhimmel" class="username">@dhimmel</a> .. Any idea how that happened?
		
	</p>
	
	<table style="display:none;"></table>
</li>

<li id="note-68" class="inline-note toggle-parent" data-note-id="68">
	<p><a href="/u/vsmalladi">Venkat Malladi</a>: <span><a href="/u/jspauld" class="username">@jspauld</a> I believe this was just user error. </span>
		
	</p>
	
	<table style="display:none;"></table>
</li>

				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="4" data-comment-id="89">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="89"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#4" class="light">March 20, 2015</a>
          
        </span>
      </div>

    

    <p>I <a href="http://thinklab.com/p/rephetio/plan/compare/cf0af88368d84640e6face839932af64a0963f5f/438864fde53c372dfa654dd59674f4bfc6cf249c">updated the proposal</a> to replace BTO with Uberon.</p>

<p>As for the BTO mapping, I didn't see <a href="https://github.com/obophenotype/uberon/tree/master/bridge">any bridges on the GitHub</a> and the <a href="http://purl.obolibrary.org/obo/uberon/bridge/uberon-bridge-to-bto.owl">BTO bridge download</a> produces an error. Am I looking in the wrong place or are the BTO mappings not populated yet?</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="5" data-comment-id="90">
  <td class="comment-profile-avatar">
		<a href="/u/vsmalladi"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/s/img/p_default_medium.png' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="90"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/vsmalladi">Venkat Malladi</a>
				
				<span class="light">&nbsp;<a href="#5" class="light">March 20, 2015</a>
          
        </span>
      </div>

    

    <p><a href="/u/dhimmel" class="username">@dhimmel</a> Within the <a href="http://berkeleybop.org/ontologies/uberon.owl">Uberon file</a> there are DbXref's. An example is for </p>

<pre><code>&lt;owl:Class rdf:about="http://purl.obolibrary.org/obo/UBERON_0000002"&gt;
    &lt;rdfs:label rdf:datatype="http://www.w3.org/2001/XMLSchema#string"&gt;uterine cervix&lt;/rdfs:label&gt;
    &lt;oboInOwl:hasDbXref rdf:datatype="http://www.w3.org/2001/XMLSchema#string"&gt;BTO:0001421&lt;/oboInOwl:hasDbXref&gt;
    &lt;oboInOwl:hasDbXref rdf:datatype="http://www.w3.org/2001/XMLSchema#string"&gt;BTO:0002249&lt;/oboInOwl:hasDbXref&gt;</code></pre>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="6" data-comment-id="230">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="230"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#6" class="light">May 21, 2015</a>
          
        </span>
      </div>

    

    <p>Hi <a href="/u/vsmalladi" class="username">@vsmalladi</a>, we have proceeded with Uberon and incorporated the MeSH cross-references. Specifically, we <a href="http://thinklab.com/d/67#229">identified</a> disease–anatomy localization using literature mining.</p>

<p>We would like a way to restrict terms to structures in humans. Does anyone know how to implement a species filter?</p>

<p>We would also like to incorporate the Cell Ontology (CL) for cell information <span class="citation">[<a href="https://doi.org/10.1186/gb-2005-6-2-r21" class="citation" data-key="10.1186/gb-2005-6-2-r21">1</a>]</span>. However, there is a <a href="https://code.google.com/p/cell-ontology/issues/detail?id=146#c2">MeSH xref issue</a> that will need to be remedied first.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="7" data-comment-id="232">
  <td class="comment-profile-avatar">
		<a href="/u/vsmalladi"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/s/img/p_default_medium.png' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="232"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/vsmalladi">Venkat Malladi</a>
				
				<span class="light">&nbsp;<a href="#7" class="light">May 22, 2015</a>
          
        </span>
      </div>

    

    <p><a href="/u/dhimmel" class="username">@dhimmel</a> You can restrict structures by NCBI taxon ID Human would be Taxon:9606</p>

<p>Can you elaborate on the MeSH issue? I might be able to help. </p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="8" data-comment-id="234">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="234"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#8" class="light">May 23, 2015</a>
          
        </span>
      </div>

    

    <h1>Restricting to human terms</h1>

<p><a href="/u/vsmalladi" class="username">@vsmalladi</a>, thanks for the pointer. In the <a href="//berkeleybop.org/ontologies/uberon/ext.obo">obo</a> header I see:<br></p>

<p></p><pre><code class="no-highlight hljs">treat-xrefs-as-reverse-genus-differentia: DHBA part_of NCBITaxon:9606
treat-xrefs-as-reverse-genus-differentia: EHDAA2 part_of NCBITaxon:9606
treat-xrefs-as-reverse-genus-differentia: FMA part_of NCBITaxon:9606
treat-xrefs-as-reverse-genus-differentia: HBA part_of NCBITaxon:9606
treat-xrefs-as-reverse-genus-differentia: HsapDv part_of NCBITaxon:9606</code></pre>

<p>Therefore I speculate the best way to identify human applicable terms would be to identify all terms with a cross-reference to the above resources and all broader terms in the hierarchy. Is that what you suggest?</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="9" data-comment-id="235">
  <td class="comment-profile-avatar">
		<a href="/u/vsmalladi"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/s/img/p_default_medium.png' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="235"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/vsmalladi">Venkat Malladi</a>
				
				<span class="light">&nbsp;<a href="#9" class="light">May 26, 2015</a>
          
        </span>
      </div>

    

    <p><a href="/u/dhimmel" class="username">@dhimmel</a> </p>

<p>Actually what you want to do is for each term filter on <code>present_in_taxon</code>:</p>

<p>   property_value: present_in_taxon NCBITaxon:7777</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="10" data-comment-id="237">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="237"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#10" class="light">June 2, 2015</a>
          
        </span>
      </div>

    

    <p><a href="/u/vsmalladi" class="username">@vsmalladi</a>, <a href="http://berkeleybop.org/ontologies/uberon/ext.obo">the obo</a> only contains <code>relationship: present_in_taxon NCBITaxon:9606</code> for four terms. Is there a more extensive listing of <code>present_in_taxon</code> relationships that you are aware of?</p>

<p>Meanwhile the <a href="https://github.com/dhimmel/uberon/blob/50c311f3d15744e8c559ea76178bdd7542f6d16b/data/mesh-map.tsv">results</a> of the <code>treat-xrefs-as-reverse-genus-differentia</code> method look satisfactory. Terms were annotated as human (<code>in_human = 1</code>) if they or any terms they subsumed (along <code>is_a</code>, <code>part_of</code>, and <code>develops_from</code> relationships) contained a cross-reference to any of the following resources: <code>DHBA</code>, <code>EHDAA2</code>, <code>FMA</code>, <code>HBA</code>, and <code>HsapDv</code> (<a href="https://github.com/dhimmel/uberon/blob/50c311f3d15744e8c559ea76178bdd7542f6d16b/mesh-map.ipynb">notebook</a>). Most terms where <code>in_human = 0</code> are not appropriate for humans.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="11" data-comment-id="276">
  <td class="comment-profile-avatar">
		<a href="/u/chrismungall"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/s/img/p_default_medium.png' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="276"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/chrismungall">Chris Mungall</a>
				
				<span class="light">&nbsp;<a href="#11" class="light">June 17, 2015</a>
          
        </span>
      </div>

    

    <p>So you probably don't want structures that are <em>uniquely</em> human or that evolved after the human-chimp common ancestor - there are probably only a handful of these, e.g. certain minor glands and brain regions.</p>

<p>You probably want structures that are typically present in humans but not necessarily absent from other species. There are two ways to go about answering this, based on your tolerance for accidentally including a non-human structure vs accidentally excluding a human structure.</p>

<ol><li>List structures that exclude those that are known not to be found in human</li><li>List structures for which there is evidence that the structure is found in humans.</li></ol>

<p>Currently we are well geared up for answering (1), but you have to tolerate the occasional inclusion of some obscure brain region that was only actually observed in macaques or mice. We call these 'taxon modules'. We are not well geared up for (2) but this could be prioritized. Using cross-references to DHBA, EHDAA2, FMA, HBA, and HsapDv is a good start but you may still miss some things.</p>

<p>Either way, you need to do something more specific than just look up the direct properties of the class. You need inference over both the anatomical graph, and the taxonomy graph. For (1) you need to make use of negative evidence, which intuitively 'reverses the flow' of inference. So if a larval stage is never found in amniotes, then any structure that necessarily exists at the larval stage is never found in any descendant of amniotes.</p>

<p>You'd be better using an owl reasoner or some existing tooling for this.</p>

<p>Some background on the taxon axioms:<br><a href="https://github.com/obophenotype/uberon/wiki/Taxon-constraints">https://github.com/obophenotype/uberon/wiki/Taxon-constraints</a></p>

<p>Hmm the taxon subsets files could do with better documentation:</p>

<p><a href="http://uberon.github.io/downloads.html#subsets">http://uberon.github.io/downloads.html#subsets</a></p>

<p>(these follow (1) above) </p>

<p>We should really make a ready-made human subset here. It would probably be more popular than the Aves one.</p>

<p>These are built using <a href="https://github.com/owlcollab/owltools">owltools</a></p>

<p></p><pre><code class="no-highlight hljs">owltools --use-catalog ext.owl--reasoner elk --make-species-subset -t NCBITaxon:9606 --remove-dangling --assert-inferred-subclass-axioms --useIsInferred --remove-dangling --set-ontology-id $(OBO)/uberon/subsets/human.owl -o human.owl</code></pre>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="12" data-comment-id="279">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="279"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#12" class="light">June 18, 2015</a>
          
        </span>
      </div>

    

    <h1>Human constraint: <em>positive</em> versus <em>no negative</em> evidence</h1>

<p><a href="/u/chrismungall" class="username">@chrismungall</a>, could not have hoped for a more informative response!</p>

<p>I am going to refer to your two methods as:</p>

<ol><li>no negative evidence, which should include all human structures and some non-human structures</li><li>positive evidence, which should include some human structures and exclude all non-human structures</li></ol>

<p>I created a comparison of the two methods for Uberon terms, using <a href="/u/fbastian" class="username">@fbastian</a>'s <a href="https://github.com/obophenotype/uberon/issues/703#issuecomment-113131156">implementation</a> of <a href="/u/chrismungall" class="username">@chrismungall</a>'s owltools command <a href="#276">above</a> for (1) and my <a href="https://github.com/dhimmel/uberon/blob/08cbf5beef80a68cf880a459a49097e39afade08/human-constraint.ipynb">implementation</a> of <em>positive evidence</em> for (2). My implementation has not been vetted, and if there is an owltools command for this functionality, we should switch. </p>

<p>I created two tsv files: one with <a href="https://github.com/dhimmel/uberon/blob/08cbf5beef80a68cf880a459a49097e39afade08/data/human-constraint.tsv">all uberon terms</a> and another with only <a href="https://github.com/dhimmel/uberon/blob/08cbf5beef80a68cf880a459a49097e39afade08/data/mesh-map.tsv">MeSH-mapping uberon terms</a>.</p>

<p>Since I'm primarily concerned with terms in MeSH, I looked through MeSH-mapping uberon terms with no positive evidence (<code>positive_evidence == 0</code>) and also no negative evidence (<code>no_negative_evidence == 1</code>). Looking through this subset, there were a few notable terms where we would like negative evidence to exist:</p>

<table class="table markdown-table"><thead><tr><th>uberon_id</th><th>uberon_name</th></tr></thead><tbody><tr><td>UBERON:0013196</td><td>strand of wool</td></tr><tr><td>UBERON:0002415</td><td>tail</td></tr><tr><td>UBERON:0007113</td><td>venom</td></tr><tr><td>UBERON:0005079</td><td>eggshell</td></tr><tr><td>UBERON:0001011</td><td>hemolymph</td></tr><tr><td>UBERON:0006378</td><td>strand of vibrissa hair</td></tr><tr><td>UBERON:0004758</td><td>salt gland</td></tr><tr><td>UBERON:0011123</td><td>stifle joint</td></tr></tbody></table>

<p>However, overall the <em>no negative evidence</em> method appeared to have higher accuracy than the <em>positive evidence</em> method. Therefore, we will proceed using <em>no negative evidence</em>, which should improve over time as Uberon matures.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="13" data-comment-id="286">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="286"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#13" class="light">June 19, 2015</a>
          
        </span>
      </div>

    

    <h1>Nomenclature</h1>

<p>We have been referring to the metanode (node type) for uberon as Tissue. Is Tissue a misnomer? Does Tissue encompass all uberon nodes? If we add CL terms under the same metanode, what term can we use that encompass uberon terms and cell types?</p>

<p>A single word is preferred to a compound term. Some options I can think of are</p>

<ul><li>Tissue</li><li>Anatomy</li><li>Structure</li><li>Anatomical Structure</li></ul>

<p><a href="/u/chrismungall" class="username">@chrismungall</a>, do you have an inclination?</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="14" data-comment-id="287">
  <td class="comment-profile-avatar">
		<a href="/u/chrismungall"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/s/img/p_default_medium.png' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="287"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/chrismungall">Chris Mungall</a>
				
				<span class="light">&nbsp;<a href="#14" class="light">June 19, 2015</a>
          
        </span>
      </div>

    

    <p>tissue is too specific, but people will still know what you mean. Uberon follows the CARO upper level ontology:</p>

<ul><li><a href="http://purl.obolibrary.org/obo/CARO_0030000">CARO:0030000</a> ! biological entity<ul><li><a href="http://purl.obolibrary.org/obo/CARO_0000000">CARO:0000000</a> ! anatomical entity<ul><li><a href="http://purl.obolibrary.org/obo/CARO_0000006">CARO:0000006</a> ! material anatomical entity<ul><li><a href="http://purl.obolibrary.org/obo/CARO_0000003">CARO:0000003</a> ! connected anatomical structure (aka anatomical structure)<ul><li><a href="http://purl.obolibrary.org/obo/CARO_0000013">CARO:0000013</a> ! cell </li><li><a href="http://purl.obolibrary.org/obo/CARO_0010000">CARO:0010000</a> ! multicellular anatomical structure<ul><li><a href="http://purl.obolibrary.org/obo/CARO_0000043">CARO:0000043</a> ! tissue  </li><li><a href="http://purl.obolibrary.org/obo/CARO_0020004">CARO:0020004</a> ! organ  </li><li><a href="http://purl.obolibrary.org/obo/CARO_0000043">CARO:0000043</a> ! tissue  </li></ul></li></ul></li></ul></li></ul></li></ul></li></ul>

<p>So formally speaking 'anatomical structure' would exclude non-material entities such as lumens, 2D boundaries... but you should never have gene expression in any of these sites</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
<li id="note-134" class="inline-note toggle-parent" data-note-id="134">
	<p><a href="/u/dhimmel">Daniel Himmelstein</a>: Links are broken
		
	</p>
	
	<table style="display:none;"></table>
</li>

				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="15" data-comment-id="295">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="295"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#15" class="light">June 20, 2015</a>
          
        </span>
      </div>

    

    <blockquote><p>formally speaking 'anatomical structure' would exclude non-material entities such as lumens, 2D boundaries</p></blockquote>

<p>Nodes that don't have expression may still be connected via a disease localization edge. These edges are currently <a href="http://thinklab.com/discussion/mining-knowledge-from-medline-articles-and-their-indexed-mesh-terms/67#229">created via MEDLINE cooccurrence</a> and can connect any uberon or CL terms that map to MeSH.</p>

<p>I am leaning towards calling the metanode <em>Anatomy</em>. We would end up with metapaths (path types) like: <strong>C</strong>ompound–<strong>t</strong>arget–<strong>G</strong>ene–<strong>e</strong>xpression–<strong>A</strong>natomy–<strong>l</strong>ocalization–<strong>D</strong>isease (abbreviated as <em>CtGeAlD</em>). This metapath refers to when a compound targets a gene that is expressed in an anatomy/tissue/cell-type where the disease is localized. Does that seem like a misuse of the word Anatomy?</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="16" data-comment-id="349">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="349"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#16" class="light">Aug. 3, 2015</a>
          
        </span>
      </div>

    

    <h1>Set of anatomy nodes</h1>

<p>We have settled on 402 Uberon terms to use as our anatomy vocabulary (<a href="https://github.com/dhimmel/uberon/blob/134f23479186abba03ba340fc6dc90e16c781920/data/hetio-slim.tsv">tsv</a>, <a href="https://github.com/dhimmel/uberon/blob/134f23479186abba03ba340fc6dc90e16c781920/process.ipynb">notebook</a>).</p>

<p>We included terms that met the following conditions:</p>

<ul><li>were in the <code>uberon_slim</code> subset.</li><li>were not in the <code>non_informative</code>, <code>upper_level</code>, or <code>grouping_class</code> subsets. See this related <a href="https://github.com/obophenotype/uberon/issues/1133">GitHub issue</a>.</li><li>contained a MeSH cross-reference</li><li>were human-relevant based on the <a href="#12">no negative evidence</a> standard.</li></ul>

<p>We chose a restrictive subset of Uberon terms because the vast extent of tissue-specific gene expression edges can become computationally troubling. We did not include cell types from the Cell Ontology because this ontology lags behind Uberon in terms of subset assignments, cross-references, and documentation.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

    		
		</table>

    

    &nbsp;<br>
    <!-- <a class="btn btn-primary btn-lg" href="/signup?next=/discussion/tissue-node/41&suppress_message=1" style>Join to Reply</a> -->


		<table class="references table-td-5" style="display:none;"><tr><td><a href='/doi/10.1093/database/bav010' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>1.</div></td><td><div class='doi-body' data-key='10.1093/database/bav010'><a href='/doi/10.1093/database/bav010' class='pub-title'>Ontology application and use at the ENCODE DCC</a><div class='pub-body'>V. S. Malladi, D. T. Erickson, N. R. Podduturi, L. D. Rowe, E. T. Chan, J. M. Davidson, B. C. Hitz, M. Ho, B. T. Lee, S. Miyasato, G. R. Roe, M. Simison, C. A. Sloan, J. S. Strattan, F. Tanaka, W. J. Kent, J. M. Cherry, E. L. Hong (2015) <i>Database</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/database/bav010'>10.1093/database/bav010</a></div></div></td></tr><tr><td><a href='/doi/10.1186/gb-2012-13-1-r5' class='pub-comments tt-right' title='Article comments'>16</a></td><td><div class='pub-order'>2.</div></td><td><div class='doi-body' data-key='10.1186/gb-2012-13-1-r5'><a href='/doi/10.1186/gb-2012-13-1-r5' class='pub-title'>Uberon, an integrative multi-species anatomy ontology</a><div class='pub-body'>Christopher J Mungall, Carlo Torniai, Georgios V Gkoutos, Suzanna E Lewis, Melissa A Haendel (2012) <i>Genome Biol</i>. doi:<a class='pub-doi' href='https://doi.org/10.1186/gb-2012-13-1-r5'>10.1186/gb-2012-13-1-r5</a></div></div></td></tr><tr><td><a href='/doi/10.1093/database/bav010' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>3.</div></td><td><div class='doi-body' data-key='10.1093/database/bav010'><a href='/doi/10.1093/database/bav010' class='pub-title'>Ontology application and use at the ENCODE DCC</a><div class='pub-body'>V. S. Malladi, D. T. Erickson, N. R. Podduturi, L. D. Rowe, E. T. Chan, J. M. Davidson, B. C. Hitz, M. Ho, B. T. Lee, S. Miyasato, G. R. Roe, M. Simison, C. A. Sloan, J. S. Strattan, F. Tanaka, W. J. Kent, J. M. Cherry, E. L. Hong (2015) <i>Database</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/database/bav010'>10.1093/database/bav010</a></div></div></td></tr><tr><td><a href='/doi/10.1093/database/bav010' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>4.</div></td><td><div class='doi-body' data-key='10.1093/database/bav010'><a href='/doi/10.1093/database/bav010' class='pub-title'>Ontology application and use at the ENCODE DCC</a><div class='pub-body'>V. S. Malladi, D. T. Erickson, N. R. Podduturi, L. D. Rowe, E. T. Chan, J. M. Davidson, B. C. Hitz, M. Ho, B. T. Lee, S. Miyasato, G. R. Roe, M. Simison, C. A. Sloan, J. S. Strattan, F. Tanaka, W. J. Kent, J. M. Cherry, E. L. Hong (2015) <i>Database</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/database/bav010'>10.1093/database/bav010</a></div></div></td></tr><tr><td><a href='/doi/10.1186/gb-2012-13-1-r5' class='pub-comments tt-right' title='Article comments'>16</a></td><td><div class='pub-order'>5.</div></td><td><div class='doi-body' data-key='10.1186/gb-2012-13-1-r5'><a href='/doi/10.1186/gb-2012-13-1-r5' class='pub-title'>Uberon, an integrative multi-species anatomy ontology</a><div class='pub-body'>Christopher J Mungall, Carlo Torniai, Georgios V Gkoutos, Suzanna E Lewis, Melissa A Haendel (2012) <i>Genome Biol</i>. doi:<a class='pub-doi' href='https://doi.org/10.1186/gb-2012-13-1-r5'>10.1186/gb-2012-13-1-r5</a></div></div></td></tr><tr><td><a href='/doi/10.1093/nar/gkq968' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>6.</div></td><td><div class='doi-body' data-key='10.1093/nar/gkq968'><a href='/doi/10.1093/nar/gkq968' class='pub-title'>The BRENDA Tissue Ontology (BTO): the first all-integrating ontology of all organisms for enzyme sources</a><div class='pub-body'>M. Gremse, A. Chang, I. Schomburg, A. Grote, M. Scheer, C. Ebeling, D. Schomburg (2010) <i>Nucleic Acids Research</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/nar/gkq968'>10.1093/nar/gkq968</a></div></div></td></tr><tr><td><a href='/doi/10.1371/journal.pcbi.1004259' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>7.</div></td><td><div class='doi-body' data-key='10.1371/journal.pcbi.1004259'><a href='/doi/10.1371/journal.pcbi.1004259' class='pub-title'>Heterogeneous Network Edge Prediction: A Data Integration Approach to Prioritize Disease-Associated Genes</a><div class='pub-body'>Daniel S. Himmelstein, Sergio E. Baranzini (2015) <i>PLOS Computational Biology</i>. doi:<a class='pub-doi' href='https://doi.org/10.1371/journal.pcbi.1004259'>10.1371/journal.pcbi.1004259</a></div></div></td></tr><tr><td><a href='/doi/10.1186/gb-2005-6-2-r21' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>8.</div></td><td><div class='doi-body' data-key='10.1186/gb-2005-6-2-r21'>None</div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d67' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>9.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d67'><a href='/d/67' class='pub-title'>Mining knowledge from MEDLINE articles and their indexed MeSH terms</a><div class='pub-body'>Daniel Himmelstein, Alex Pankov (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d67'>10.15363/thinklab.d67</a></div></div></td></tr></table>

    


	</td>
	<td class="thread-sidebar">


			

      <!-- <a class="btn btn-default follow-link btn-follow btn-lg is-following tt-top full-width" title="Follow discussion" data-id="41" data-ctype="thread" data-is-following="false">&nbsp;</a> -->

      
        <div class="status-box text-center"><span style="">Status:</span> 
  Completed

</div>
      

      
        
      


				

					<div class="details">Views</div>
					<span style="font-size:20px;">181</span>

        

				
					<div class="details">Topics</div>
					<div style="margin-bottom:15px;">
		            <a href="/topics/ontologies" class="tag">Ontologies</a>
					</div>
				


		
			<div class="details">Referenced by</div>
			<div style="font-size:12px;"><ul style="padding-left:16px !important;">
			
				<li>
					
					Research report: <a href='/p/rephetio/report'>Rephetio: Repurposing drugs on a hetnet</a>
				</li>
			
				<li>
					
					<a href='/d/22' class='thread-title'>Suggestions for additional information types?</a>
				</li>
			
				<li>
					
					<a href='/d/124' class='thread-title'>Processing Bgee for tissue-specific gene presence and over/under-expression</a>
				</li>
			
				<li>
					
					<a href='/d/81' class='thread-title'>Tissue-specific gene expression resources</a>
				</li>
			
				<li>
					
					<a href='/d/82' class='thread-title'>Processing GTEx for tissue-specific gene expression</a>
				</li>
			
				</ul>
			</div>
		

		
			<div class="details">Cite this as</div>
			<span style="font-size:12px;">
				Venkat Malladi, Daniel Himmelstein, Chris Mungall (2015) Tissue Node. <i>Thinklab.</i> doi:<a href='https://doi.org/10.15363/thinklab.d41'>10.15363/thinklab.d41</a>
			</span>
		

		<div class="details">License</div>
		<p><a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0;width:88px;height:31px;" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a></p>

		<div class="details">Share</div>
		
			
				<a class="share-button fb" onclick="javascript:void(window.open('https://www.facebook.com/sharer/sharer.php?u=https%3A//thinklab.org/discussion/tissue-node/41','','width=600,height=460,menubar=no,status=no'));" href="#"></a>
<a class="share-button twitter" onclick="javascript:void(window.open('https://twitter.com/share?text=Tissue%20Node&url=https%3A//thinklab.org/discussion/tissue-node/41','','width=600,height=300,menubar=no,status=no'));" href="#"></a>
<a class="share-button google" onclick="javascript:void(window.open('https://plus.google.com/share?url=https%3A//thinklab.org/discussion/tissue-node/41','','width=510,height=460,menubar=no,status=no'));" href="#"></a>

			
		

	</td>
</tr></table>


		</div>


		
	    	<div class="footer">
<!-- <div style="background:#dedede;">
	<div class="main-centered-div text-center" style="padding:20px;">
		<h4 style="margin-bottom:0px;font-weight:bold;color:gray;">Feedback? Questions? Email us: <a href="mailto:hello@thinklab.com">hello@thinklab.com</a></h4>
	</div>
</div> -->
<div class="main-footer">
	<div class="main-centered-div">

<table class="full-width"><tr>
	<td class="footer-nav">

        <div class="foot-head">Explore</div>
        <a href="/proposals">Proposals</a><br>
        <a href="/projects">Projects</a><br>
        <a href="/publications">Article discussion</a><br>
        <a href="/discussion">All discussion</a><br>
        <a href="/topics">Topics</a><br>
        <!-- <a href="/funders">Funders</a><br> -->
        <a href="/funding">Funding opportunities</a><br>

	</td>
	<td class="footer-nav">

        <!-- <div class="foot-head">Help &amp; feedback</div>	 -->
        <!-- <div class="foot-head">Intro</div>
        <a href="/benefits/for-science">Why</a><br>
        <a href="/how-it-works">How it works</a><br>

<br> -->

        <div class="foot-head">Community</div>
        <a href="/leaderboard">Leaderboard</a><br>
        <a href="/p/meta">Thinklab Meta</a><br>
				<a href="/help/faq">Help</a><br>
				<!-- <a href="/new-project">Start a proposal</a><br> -->
        <!-- <a href="/benefits/proposal">Benefits</a><br> -->
        <!-- <a href="/funding">Get funded</a><br> -->
        <!-- <a href="/participate">Become a contributor</a><br> -->
        <!-- <a href="/for-funders">For science funders</a><br> -->
        <!-- <a href="/for-institutions">For institutions</a><br> -->

	</td>
	<td class="footer-nav">



        <div class="foot-head">About</div>
        <a href="/about">Overview</a><br>
        <a href="/benefits">Benefits</a><br>
        <a href="/how-it-works">How it works</a><br>
        <a href="/about/story">Our story</a><br>
        <a href="/blog">Our blog</a><br>
        <a href="/about/jobs">Join the team</a><br>
        <br>

        <!-- <div class="foot-head">Help</div>
        <a href="/help">Help</a><br>
		 -->


	</td>
	<td class="footer-nav">
		<!-- <a href="http://twitter.com/thinklab" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/twitter_com.png" width="40"/></a>
		<a href="https://www.facebook.com/pages/Thinklab/310895039088185" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/facebook_com.png" width="40"/></a>
		<a href="https://plus.google.com/115578983466800753402/" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/plus_google_com.png" width="40"/></a>
		<a href="https://www.linkedin.com/company/thinklab-co" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/linkedin_com.png" width="40"/></a> -->
		<br><br>
		<span class="light" style='font-size:12px;'>&copy; 2015 Thinklab</span>

	</td>

</tr></table>

</div>
</div>
</div>

		

	</div>


  <script></script>
  <script>
  var MEDIA_URL = 'https://think-lab.s3.amazonaws.com/m/';
  var STATIC_URL = 'https://think-lab.s3.amazonaws.com/s/';
  var page_is_authenticated = false;
  var page_page = 'discussion';
  var page_profile_id = undefined;
</script>



	<!-- Note different script type. MUST come before MathJax.js -->
	<script type="text/x-mathjax-config">
	    MathJax.Hub.Config({
	        "HTML-CSS": { preferredFont: "TeX", availableFonts: ["STIX","TeX"],
	                        scale: 110, linebreaks: { automatic:true },
	                        EqnChunk: (MathJax.Hub.Browser.isMobile ? 10 : 50) },
	        tex2jax: { inlineMath: [ ["$$", "$$"], ["\\(","\\)"] ],
	                    displayMath: [ ["$$$","$$$"], ["\\[", "\\]"] ],
	                    processEscapes: true,}
	            });

	</script>
    <script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>





  <script src="https://code.jquery.com/jquery-1.10.2.min.js"></script>
  <script src="https://code.jquery.com/ui/1.11.4/jquery-ui.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" integrity="sha384-0mSbJDEHialfmuBBQP6A4Qrprq5OVfW37PRR3j5ELqxss1yVqOtnepnHVP9aJ7xS" crossorigin="anonymous"></script>
  <script src="https://think-lab.s3.amazonaws.com/s/dist/main.min.js"></script>
  



  
	
  
  
  
<script>
$(function() {
  recordHit('thread', 41);
});
</script>



</body>
</html>
