<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8" /><script type="text/javascript">window.NREUM||(NREUM={}),__nr_require=function(e,n,t){function r(t){if(!n[t]){var o=n[t]={exports:{}};e[t][0].call(o.exports,function(n){var o=e[t][1][n];return r(o||n)},o,o.exports)}return n[t].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<t.length;o++)r(t[o]);return r}({1:[function(e,n,t){function r(){}function o(e,n,t){return function(){return i(e,[c.now()].concat(u(arguments)),n?null:this,t),n?void 0:this}}var i=e("handle"),a=e(2),u=e(3),f=e("ee").get("tracer"),c=e("loader"),s=NREUM;"undefined"==typeof window.newrelic&&(newrelic=s);var p=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],d="api-",l=d+"ixn-";a(p,function(e,n){s[n]=o(d+n,!0,"api")}),s.addPageAction=o(d+"addPageAction",!0),s.setCurrentRouteName=o(d+"routeName",!0),n.exports=newrelic,s.interaction=function(){return(new r).get()};var m=r.prototype={createTracer:function(e,n){var t={},r=this,o="function"==typeof n;return i(l+"tracer",[c.now(),e,t],r),function(){if(f.emit((o?"":"no-")+"fn-start",[c.now(),r,o],t),o)try{return n.apply(this,arguments)}finally{f.emit("fn-end",[c.now()],t)}}}};a("setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(e,n){m[n]=o(l+n)}),newrelic.noticeError=function(e){"string"==typeof e&&(e=new Error(e)),i("err",[e,c.now()])}},{}],2:[function(e,n,t){function r(e,n){var t=[],r="",i=0;for(r in e)o.call(e,r)&&(t[i]=n(r,e[r]),i+=1);return t}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],3:[function(e,n,t){function r(e,n,t){n||(n=0),"undefined"==typeof t&&(t=e?e.length:0);for(var r=-1,o=t-n||0,i=Array(o<0?0:o);++r<o;)i[r]=e[n+r];return i}n.exports=r},{}],4:[function(e,n,t){n.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(e,n,t){function r(){}function o(e){function n(e){return e&&e instanceof r?e:e?f(e,u,i):i()}function t(t,r,o,i){if(!d.aborted||i){e&&e(t,r,o);for(var a=n(o),u=m(t),f=u.length,c=0;c<f;c++)u[c].apply(a,r);var p=s[y[t]];return p&&p.push([b,t,r,a]),a}}function l(e,n){v[e]=m(e).concat(n)}function m(e){return v[e]||[]}function w(e){return p[e]=p[e]||o(t)}function g(e,n){c(e,function(e,t){n=n||"feature",y[t]=n,n in s||(s[n]=[])})}var v={},y={},b={on:l,emit:t,get:w,listeners:m,context:n,buffer:g,abort:a,aborted:!1};return b}function i(){return new r}function a(){(s.api||s.feature)&&(d.aborted=!0,s=d.backlog={})}var u="nr@context",f=e("gos"),c=e(2),s={},p={},d=n.exports=o();d.backlog=s},{}],gos:[function(e,n,t){function r(e,n,t){if(o.call(e,n))return e[n];var r=t();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(e,n,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return e[n]=r,r}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],handle:[function(e,n,t){function r(e,n,t,r){o.buffer([e],r),o.emit(e,n,t)}var o=e("ee").get("handle");n.exports=r,r.ee=o},{}],id:[function(e,n,t){function r(e){var n=typeof e;return!e||"object"!==n&&"function"!==n?-1:e===window?0:a(e,i,function(){return o++})}var o=1,i="nr@id",a=e("gos");n.exports=r},{}],loader:[function(e,n,t){function r(){if(!x++){var e=h.info=NREUM.info,n=d.getElementsByTagName("script")[0];if(setTimeout(s.abort,3e4),!(e&&e.licenseKey&&e.applicationID&&n))return s.abort();c(y,function(n,t){e[n]||(e[n]=t)}),f("mark",["onload",a()+h.offset],null,"api");var t=d.createElement("script");t.src="https://"+e.agent,n.parentNode.insertBefore(t,n)}}function o(){"complete"===d.readyState&&i()}function i(){f("mark",["domContent",a()+h.offset],null,"api")}function a(){return E.exists&&performance.now?Math.round(performance.now()):(u=Math.max((new Date).getTime(),u))-h.offset}var u=(new Date).getTime(),f=e("handle"),c=e(2),s=e("ee"),p=window,d=p.document,l="addEventListener",m="attachEvent",w=p.XMLHttpRequest,g=w&&w.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:w,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var v=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-1044.min.js"},b=w&&g&&g[l]&&!/CriOS/.test(navigator.userAgent),h=n.exports={offset:u,now:a,origin:v,features:{},xhrWrappable:b};e(1),d[l]?(d[l]("DOMContentLoaded",i,!1),p[l]("load",r,!1)):(d[m]("onreadystatechange",o),p[m]("onload",r)),f("mark",["firstbyte",u],null,"api");var x=0,E=e(4)},{}]},{},["loader"]);</script><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info={"beacon":"bam.nr-data.net","queueTime":0,"licenseKey":"cd1101aa94","agent":"","transactionName":"bgcDMkBZW0NZBkdcWVdNJxNcW0FZVwscV1dKB08QW11CQxYTWlBBZgEOC19dW0QCEVtHU1gG","applicationID":"6369297","errorBeacon":"bam.nr-data.net","applicationTime":679}</script>
    <title>Positive correlations between knockdown and overexpression profiles from LINCS L1000 | Thinklab</title>
	  <link rel="shortcut icon" href="https://think-lab.s3.amazonaws.com/s/img/favicon.ico" type="image/x-icon" />
    <meta property="og:image" content="https://think-lab.s3.amazonaws.com/s/img" />
    
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
  <link rel="stylesheet" href="https://code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css" />

<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/highlightjs/github.css" />
<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/css/css.css" />
<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/css/bootstrap-multiselect.css" />
<!-- <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'> -->

    
    
    
    
  
    <meta name="citation_doi" content="10.15363/thinklab.d171">
    <meta name="citation_title" content="Positive correlations between knockdown and overexpression profiles from LINCS L1000">
    <meta name="citation_online_date" content="Feb. 26, 2016">
    <meta name="citation_author" content="Daniel Himmelstein, Casey Greene, Lars Juhl Jensen">
    <meta name="citation_journal_title" content="Thinklab">
    <meta name="citation_publisher" content="Thinklab">
  

    


	<!-- GoogleAnalytics -->
	<script>
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
	(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
	m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

	ga('create', 'UA-48550230-1', 'thinklab.org');
	ga('require', 'displayfeatures');
	
	ga('send', 'pageview');

	</script>






</head>
<body >
  <form id="form_action" method="post">
    <input type='hidden' name='csrfmiddlewaretoken' value='JC3hnVSWzcwBOS226QTq2eGyHXj1YAM6' />
</form>

	

	<div class="body-wrapper">

		


<!-- <div style="height:5px;background-image:url('https://think-lab.s3.amazonaws.com/s/img/top-bg.gif');"></div> -->
<div class="gradient">
	<div class="main-centered-div header-container">

<table class="full-width" style="height:60px;"><tr>
	<td style="width:200px;vertical-align:middle;padding-bottom:15px;">
		<a href="/" style="font-weight:normal;color:#212121;"><img src="https://think-lab.s3.amazonaws.com/s/img/logo.png" width="160"></a>
	</td><td style="vertical-align:middle;text-align:left;padding-top:6px;width:100%;padding-left:40px;">
		<ul id="not-search-block"  class="main-header">


			


				<li class="dropdown">
					<a class="dropdown-toggle" data-toggle="dropdown" href="#">Browse <span class="caret"></span></a>
					<ul class="dropdown-menu" role="menu">
						<li><a href="/proposals">Proposals</a></li>
						<li><a href="/projects">Projects</a></li>
						<li><a href="/publications">Article discussion</a></li>
						<li><a href="/discussion">All discussion</a></li>
					</ul>
				</li>
				<li><a href="/leaderboard">Leaderboard</a></li>

			
				<li><a href="/about">About</a></li>
			


		</ul>

    <ul id="search-block" class="main-header" style="display: none; margin-right: 30px;">
        <li class="full-width">
            <form method="get" action="/search">
			<div class="input-group full-width">
				<input name="q" type="text" class="form-control search-input">
			</div>
            </form>
        </li>
    </ul>

	</td><td style="vertical-align:middle;text-align:right;padding-top:6px; min-width: 1%; white-space: nowrap;">
		<ul class="main-header pull-right" style="padding-left: 0;">
    		
                <!-- <li><a href="/login" style="color:silver;">Login</a></li>
                <li style="margin-right:0"><a class="btn btn-default btn-sm btn-upper" href="/signup">Join</a></li> -->
            

        </ul>
    </ul>
	</td>
</tr></table>

<!-- <div id="global-search-div" class="main-centered-div" style="display:none;">
	<input type="text" value="" class="thread-search form-control" placeholder="Search" data-action="/discussion">
</div> -->

	</div>
</div>

		
	  



	  


		

		<div class="dark-head"><div class="main-centered-div">

			

		</div></div>
		
		<div class="main-centered-div" style="margin-top:40px;padding-bottom:340px">
	    

<div style="margin-top:-20px; margin-bottom:20px;">

    <a class="side-box" href="/p/rephetio" style="background-color:#ebfdff;"><div class="side-type">Project:</div> <div class="side-title">Rephetio: Repurposing drugs on a hetnet [rephetio]</div></a><br>



</div>

<table style="margin-top:0px;"><tr>
	<td class="thread-table-content">

		<table class="full-width">
			
<tr class="comment-tr target-hl" id="1" data-comment-id="1051">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="1051"></a>

    

			<h2 class="page-title" style="margin-bottom:5px;margin-top:0px !important;">Positive correlations between knockdown and overexpression profiles from LINCS L1000</h2>

			<div style="margin-top:5px;color:gray;margin-bottom:20px;font-size:13px;">
				<a style="font-weight:normal;color:#212121" href="/u/dhimmel"><strong>Daniel Himmelstein</strong></a>
				<span class="is-researcher">Researcher</span>
				&nbsp; <a href="#1" class="light">Feb. 26, 2016</a>
				
			</div>

      

          <hr>
		

    <p><a href="/u/alessandrodidonna" class="username">@alessandrodidonna</a> suggested adding RNA interference data, so we <a href="http://thinklab.com/discussion/suggestions-for-additional-information-types/22#6" class="citation thread-citation" data-key="10.15363/thinklab.d22">incorporated genetic perturbation relationships</a> from LINCS L1000. The L1000 project measures how the expression of 978 genes (called landmark genes) changes in response to perturbation. Here we are focusing on gene knockdown (shRNA) and gene overexpression perturbations.</p>

<p>We <a href="http://thinklab.com/discussion/computing-consensus-transcriptional-profiles-for-compounds-using-lincs/43#6" class="citation thread-citation" data-key="10.15363/thinklab.d43">computed consensus transcriptional profiles</a> for knockdown (<a href="http://support.lincscloud.org/hc/en-us/articles/202216073-Perturbation-Types" title="lincscloud Perturbation Types"><code>pert_type = trt_sh</code></a>) and overexpression (<code>pert_type = trt_oe</code>) perturbations. For each gene perturbation, we end up with a vector of 978 z-scores representing the change in expression of each landmark gene. Using a Bonferroni cutoff to correct for the 978 comparisons, we identify the significantly upregulated and downregulated genes for each perturbation. Using this approach, we generate four relationship types for our network:</p>

<ol><li>Gene → knockdown downregulates → Gene</li><li>Gene → knockdown upregulates → Gene</li><li>Gene → overexpression downregulates → Gene</li><li>Gene → overexpression upregulates → Gene</li></ol>

<p>In a <a href="http://thinklab.com/discussion/data-nomenclature-naming-and-abbreviating-our-network-types/162#3" class="citation thread-citation" data-key="10.15363/thinklab.d162">separate discussion</a>, <a href="/u/larsjuhljensen" class="username">@larsjuhljensen</a> commented:</p>

<blockquote><p>I am not entirely sure how useful the "knockdown downregulates" etc. types are. Usually "knockdown downregulates" would be interpreted to mean "upregulates" etc.</p></blockquote>

<p>In other words, shouldn't we combine relationship types 1 &amp; 4 above into "Gene → upregulates → Gene" and 2 &amp; 3 into "Gene → downregulates → Gene"? To investigate whether this makes sense, I looked into whether knockdown and overexpression profiles for the same gene were anticorrelated. Does knocking down a gene have the opposite trascriptional effect as overexpressing it?</p>

<p>The results were surprising (<a href="https://github.com/dhimmel/lincs/blob/00c55f95ead78bec72b9c7255f38b512c4a3da30/binarize-consensi.ipynb">notebook</a>). Knockdown and overexpression of the same gene resulted in positively correlated transcriptional profiles 65.0% of the time. And if we correlate the knockdown of a random gene with the overexpression of a different random gene, we see a positive correlation 65.3% of the time. In summary, the transcriptional profiles of knocking down and overexpressing genes are more often than not positively correlated. And profiles for the same gene show no more correlation or anticorrelation than profiles for two different genes. Hmm.</p>

<p><img src="https://github.com/dhimmel/lincs/raw/00c55f95ead78bec72b9c7255f38b512c4a3da30/viz/knockdown-overexpression-corr.png" alt="Violinplots of correlation distributions"></p>

<p>What could cause this counterintuitive finding?</p>

<ul><li>We could have a mistake in our code. Does anyone know of a gold standard for genetic perturbations that we could compare to?</li><li>By looking only at the 978 landmark genes, we are overlooking the crucial genes and instead picking up on a general perturbation response.</li><li>Gene regulation is a non-linear process.</li><li>Our method of analysis or the LINCS L1000 data may be limitated.</li></ul>

<p>Does anyone, specifically those with gene expression experience (<a href="/u/caseygreene" class="username">@caseygreene</a>, <a href="/u/larsjuhljensen" class="username">@larsjuhljensen</a>, <a href="/u/fbastian" class="username">@fbastian</a>), have any insight on what might be happening? I'll also reach out to the L1000 team.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
<li id="note-210" class="inline-note toggle-parent" data-note-id="210">
	<p><a href="/u/dhimmel">Daniel Himmelstein</a>: <p>I removed the following sentence, since it is not true:</p>

<blockquote><p>Unfortunately, none of the perturbed genes were in the landmark set, so I can't detect whether the perturbations are actually affecting their target genes in the desired direction.</p></blockquote>
		
	</p>
	
	<table style="display:none;"></table>
</li>

				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="2" data-comment-id="1137">
  <td class="comment-profile-avatar">
		<a href="/u/caseygreene"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/22_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="1137"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/caseygreene">Casey Greene</a>
				
				<span class="light">&nbsp;<a href="#2" class="light">Feb. 26, 2016</a>
          
        </span>
      </div>

    

    <p>Quick thoughts: Is there a specific set of genes driving the positive correlation? Maybe perturbations in general lead to some change in, for example, growth rate? What, specifically, is your high correlation measuring? Is it possible that highly expressed genes tend to remain highly expressed, or did you transform the data in some way to normalize gene expression across conditions per gene.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
<li id="note-207" class="inline-note toggle-parent" data-note-id="207">
	<p><a href="/u/larsjuhljensen">Lars Juhl Jensen</a>: Exactly - we think alike. What I refer to as "not a happy cell" is usually some mix of stress response and reduced growth rate, which also results in a change in the distribution of cells across cell-cycle phases.
		
	</p>
	
	<table style="display:none;"></table>
</li>

<li id="note-208" class="inline-note toggle-parent" data-note-id="208">
	<p><a href="/u/caseygreene">Casey Greene</a>: <a href="/u/larsjuhljensen" class="username">@larsjuhljensen</a> Indeed! My guess right now is that or no gene-wise normalization (e.g. the most highly expressed genes are still the most highly expressed).
		
	</p>
	
	<table style="display:none;"></table>
</li>

<li id="note-209" class="inline-note toggle-parent" data-note-id="209">
	<p><a href="/u/larsjuhljensen">Lars Juhl Jensen</a>: I assume that the expression values here are already ratios between perturbed and non-perturbed. Otherwise, I would put my money on it being a normalization artifact, but in that case I would expect a much stronger correlation than what is observed.
		
	</p>
	
	<table style="display:none;"></table>
</li>

				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="3" data-comment-id="1139">
  <td class="comment-profile-avatar">
		<a href="/u/larsjuhljensen"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/125_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="1139"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/larsjuhljensen">Lars Juhl Jensen</a>
				
				<span class="light">&nbsp;<a href="#3" class="light">Feb. 26, 2016</a>
          
        </span>
      </div>

    

    <p>Messing about with cells always tends to induce some degree of stress-induced global expression changes. This is the case pretty much no matter which perturbation you do to the cells, including overexpression of some gene, knockdown of some gene, cell-cycle synchronization, centrifugation, increasing temperature, decreasing temperature, etc.</p>

<p>My guess is that the small positive correlation you see is caused by small changes in expression of a large number of genes, which you could summarize as "not a happy cell".</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="4" data-comment-id="1140">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="1140"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#4" class="light">Feb. 26, 2016</a>
          
        </span>
      </div>

    

    <blockquote><p>Did you transform the data in some way to normalize gene expression across conditions per gene.</p></blockquote>

<p><a href="/u/caseygreene" class="username">@caseygreene</a>, our profiles contain <em>z</em>-scores measuring the differential expression for 978 genes. The profiles (called consensus signatures in <a href="http://support.lincscloud.org/hc/en-us/articles/202099616-Signature-Generation-and-Analysis-L1000-">L1000 terminology</a>) are at the CONSENSUS stage in the following pipeline:</p>

<p><img src="http://support.lincscloud.org/hc/en-us/article_attachments/200733106/data_flow.png" alt="Processing of Broad LINCS data"></p>

<p>The <em>z</em>-scores compare a gene's expression level in cells given the perturbation to cells without the perturbation (controls). I believe the controls account for the non-specific disturbances caused by delivering the molecular payload, but will confirm.</p>

<p>Now I will look into the following questions:</p>

<ul><li><a href="/u/larsjuhljensen" class="username">@larsjuhljensen</a>: My guess is that the small positive correlation you see is caused by small changes in expression of a large number of genes.</li><li><a href="/u/caseygreene" class="username">@caseygreene</a>: Is there a specific set of genes driving the positive correlation?</li></ul>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="5" data-comment-id="1167">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="1167"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#5" class="light">March 9, 2016</a>
          
        </span>
      </div>

    

    <h1>Update with workshop findings</h1>

<p>I recently led a Systems Pharmacology workshop for first-year graduate students. <a href="http://thinklab.com/discussion/workshop-to-analyze-lincs-data-for-the-systems-pharmacology-course-at-ucsf/181" class="citation thread-citation" data-key="10.15363/thinklab.d181">We analyzed</a> the L1000 genetic perturbation data with the goal of shedding light on the issues in this discussion. The workshop was based on significant dysregulation due to knockdown or overexpression from <a href="http://thinklab.com/discussion/computing-consensus-transcriptional-profiles-for-lincs-l1000-perturbations/43#7" class="citation thread-citation" data-key="10.15363/thinklab.d43"><code>dhimmel/lincs v2.0</code></a> <span class="citation">[<a href="https://doi.org/10.5281/zenodo.47223" class="citation" data-key="10.5281/zenodo.47223">1</a>, <a href="https://doi.org/10.6084/m9.figshare.3085426" class="citation" data-key="10.6084/m9.figshare.3085426">2</a>]</span>. Compared to <code>v1.0</code> (what the <a href="#1">leadoff post</a> was based on), <code>v2.0</code> adds dysregulation scores for imputed genes.</p>

<p>See the <a href="http://thinklab.com/discussion/workshop-to-analyze-lincs-data-for-the-systems-pharmacology-course-at-ucsf/181#14" class="citation thread-citation" data-key="10.15363/thinklab.d181">summary of our findings</a>. In short, certain genes responded in the same direction to a large number of perturbations. For example, <em>RPS4Y1</em> was frequently downregulated and <em>MCOLN1</em> was frequently upregulated, regardless of which gene was perturbed in which direction.</p>

<p><a href="/u/larsjuhljensen" class="username">@larsjuhljensen</a> noted:</p>

<blockquote><p>Reassuring to see that things behave the way I would expect. This should make it fairly easy to derive a scoring scheme that extracts only associations that are specifically associated with a small number of perturbations, as opposed to associated with any perturbation.</p></blockquote>

<p>I think Lars makes a great suggestion, worthy of investigation. However, due to time constraints, we will have to postpone this analysis for a future undertaking.</p>

<h1>Proposed quick fix</h1>

<p>Currently, I'm leaning towards collapsing all four types of regulation into a single relationship type (<em>Gene → regulates → Gene</em>), which means perturbation of the source gene significantly dysregulated the target gene. In other words, we'll take the union of the four <a href="#1">aforementioned</a> regulation relationships.</p>

<p>Our <em>DWPC</em> method for quantifying the connectivity between two nodes downweights paths through high degree nodes <span class="citation">[<a href="https://doi.org/10.1371/journal.pcbi.1004259" class="citation" data-key="10.1371/journal.pcbi.1004259">3</a>]</span>. Thus the pervasively dysregulated genes should not be too problematic.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

    		
		</table>

    

    &nbsp;<br>
    <!-- <a class="btn btn-primary btn-lg" href="/signup?next=/discussion/positive-correlations-between-knockdown-and-overexpression-profiles-from-lincs-l1000/171&suppress_message=1" style>Join to Reply</a> -->


		<table class="references table-td-5" style="display:none;"><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d22' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>1.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d22'><a href='/d/22' class='pub-title'>Suggestions for additional information types?</a><div class='pub-body'>Daniel Himmelstein, Venkat Malladi, Alessandro Didonna, Alexey Strokach (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d22'>10.15363/thinklab.d22</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d43' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>2.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d43'><a href='/d/43' class='pub-title'>Computing consensus transcriptional profiles for LINCS L1000 perturbations</a><div class='pub-body'>Daniel Himmelstein, Caty Chung (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d43'>10.15363/thinklab.d43</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d162' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>3.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d162'><a href='/d/162' class='pub-title'>Data nomenclature: naming and abbreviating our network types</a><div class='pub-body'>Daniel Himmelstein, Lars Juhl Jensen, Pouya Khankhanian (2016) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d162'>10.15363/thinklab.d162</a></div></div></td></tr><tr><td><a href='/doi/10.5281/zenodo.47223' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>4.</div></td><td><div class='doi-body' data-key='10.5281/zenodo.47223'><a href='/doi/10.5281/zenodo.47223' class='pub-title'>dhimmel/lincs v2.0: Refined consensus signatures from LINCS L1000</a><div class='pub-body'>Daniel Himmelstein, Leo Brueggeman, Sergio Baranzini (2016) <i>Zenodo</i>. doi:<a class='pub-doi' href='https://doi.org/10.5281/zenodo.47223'>10.5281/zenodo.47223</a></div></div></td></tr><tr><td><a href='/doi/10.6084/m9.figshare.3085426' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>5.</div></td><td><div class='doi-body' data-key='10.6084/m9.figshare.3085426'><a href='/doi/10.6084/m9.figshare.3085426' class='pub-title'>Consensus signatures for LINCS L1000 perturbations</a><div class='pub-body'>Daniel Himmelstein, Leo Brueggeman, Sergio Baranzini (2016) <i>Figshare</i>. doi:<a class='pub-doi' href='https://doi.org/10.6084/m9.figshare.3085426'>10.6084/m9.figshare.3085426</a></div></div></td></tr><tr><td><a href='/doi/10.1371/journal.pcbi.1004259' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>6.</div></td><td><div class='doi-body' data-key='10.1371/journal.pcbi.1004259'><a href='/doi/10.1371/journal.pcbi.1004259' class='pub-title'>Heterogeneous Network Edge Prediction: A Data Integration Approach to Prioritize Disease-Associated Genes</a><div class='pub-body'>Daniel S. Himmelstein, Sergio E. Baranzini (2015) <i>PLOS Computational Biology</i>. doi:<a class='pub-doi' href='https://doi.org/10.1371/journal.pcbi.1004259'>10.1371/journal.pcbi.1004259</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d181' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>7.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d181'><a href='/d/181' class='pub-title'>Workshop to analyze LINCS data for the Systems Pharmacology course at UCSF</a><div class='pub-body'>Daniel Himmelstein, Kathleen Keough, Misha Vysotskiy, Jeffrey Kim, Beau Norgeot, Julia Cluceru, Marjorie Imperial, Emmalyn Chen, Jasleen Sodhi, Elizabeth Levy (2016) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d181'>10.15363/thinklab.d181</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d43' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>8.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d43'><a href='/d/43' class='pub-title'>Computing consensus transcriptional profiles for LINCS L1000 perturbations</a><div class='pub-body'>Daniel Himmelstein, Caty Chung (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d43'>10.15363/thinklab.d43</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d181' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>9.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d181'><a href='/d/181' class='pub-title'>Workshop to analyze LINCS data for the Systems Pharmacology course at UCSF</a><div class='pub-body'>Daniel Himmelstein, Kathleen Keough, Misha Vysotskiy, Jeffrey Kim, Beau Norgeot, Julia Cluceru, Marjorie Imperial, Emmalyn Chen, Jasleen Sodhi, Elizabeth Levy (2016) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d181'>10.15363/thinklab.d181</a></div></div></td></tr></table>

    


	</td>
	<td class="thread-sidebar">


			

      <!-- <a class="btn btn-default follow-link btn-follow btn-lg is-following tt-top full-width" title="Follow discussion" data-id="171" data-ctype="thread" data-is-following="false">&nbsp;</a> -->

      
        <div class="status-box text-center"><span style="">Status:</span> 
  Open

</div>
      

      
        
      


				

					<div class="details">Views</div>
					<span style="font-size:20px;">269</span>

        

				
					<div class="details">Topics</div>
					<div style="margin-bottom:15px;">
		            <a href="/topics/lincs" class="tag">LINCS</a><a href="/topics/l1000" class="tag">L1000</a><a href="/topics/upregulation" class="tag">Upregulation</a><a href="/topics/downregulation" class="tag">Downregulation</a><a href="/topics/shrnas" class="tag">shRNAs</a><a href="/topics/knockdown" class="tag">Knockdown</a><a href="/topics/overexpression" class="tag">Overexpression</a>
					</div>
				


		
			<div class="details">Referenced by</div>
			<div style="font-size:12px;"><ul style="padding-left:16px !important;">
			
				<li>
					
					Research report: <a href='/p/rephetio/report'>Rephetio: Repurposing drugs on a hetnet</a>
				</li>
			
				<li>
					
					<a href='/d/185' class='thread-title'>Assessing the imputation quality of gene expression in LINCS L1000</a>
				</li>
			
				<li>
					
					<a href='/d/181' class='thread-title'>Workshop to analyze LINCS data for the Systems Pharmacology course at UCSF</a>
				</li>
			
				</ul>
			</div>
		

		
			<div class="details">Cite this as</div>
			<span style="font-size:12px;">
				Daniel Himmelstein, Casey Greene, Lars Juhl Jensen (2016) Positive correlations between knockdown and overexpression profiles from LINCS L1000. <i>Thinklab.</i> doi:<a href='https://doi.org/10.15363/thinklab.d171'>10.15363/thinklab.d171</a>
			</span>
		

		<div class="details">License</div>
		<p><a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0;width:88px;height:31px;" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a></p>

		<div class="details">Share</div>
		
			
				<a class="share-button fb" onclick="javascript:void(window.open('https://www.facebook.com/sharer/sharer.php?u=https%3A//thinklab.org/discussion/positive-correlations-between-knockdown-and-overexpression-profiles-from-lincs-l1000/171','','width=600,height=460,menubar=no,status=no'));" href="#"></a>
<a class="share-button twitter" onclick="javascript:void(window.open('https://twitter.com/share?text=Positive%20correlations%20between%20knockdown%20and%20overexpression%20profiles%20from%20LINCS%20L1000&url=https%3A//thinklab.org/discussion/positive-correlations-between-knockdown-and-overexpression-profiles-from-lincs-l1000/171','','width=600,height=300,menubar=no,status=no'));" href="#"></a>
<a class="share-button google" onclick="javascript:void(window.open('https://plus.google.com/share?url=https%3A//thinklab.org/discussion/positive-correlations-between-knockdown-and-overexpression-profiles-from-lincs-l1000/171','','width=510,height=460,menubar=no,status=no'));" href="#"></a>

			
		

	</td>
</tr></table>


		</div>


		
	    	<div class="footer">
<!-- <div style="background:#dedede;">
	<div class="main-centered-div text-center" style="padding:20px;">
		<h4 style="margin-bottom:0px;font-weight:bold;color:gray;">Feedback? Questions? Email us: <a href="mailto:hello@thinklab.com">hello@thinklab.com</a></h4>
	</div>
</div> -->
<div class="main-footer">
	<div class="main-centered-div">

<table class="full-width"><tr>
	<td class="footer-nav">

        <div class="foot-head">Explore</div>
        <a href="/proposals">Proposals</a><br>
        <a href="/projects">Projects</a><br>
        <a href="/publications">Article discussion</a><br>
        <a href="/discussion">All discussion</a><br>
        <a href="/topics">Topics</a><br>
        <!-- <a href="/funders">Funders</a><br> -->
        <a href="/funding">Funding opportunities</a><br>

	</td>
	<td class="footer-nav">

        <!-- <div class="foot-head">Help &amp; feedback</div>	 -->
        <!-- <div class="foot-head">Intro</div>
        <a href="/benefits/for-science">Why</a><br>
        <a href="/how-it-works">How it works</a><br>

<br> -->

        <div class="foot-head">Community</div>
        <a href="/leaderboard">Leaderboard</a><br>
        <a href="/p/meta">Thinklab Meta</a><br>
				<a href="/help/faq">Help</a><br>
				<!-- <a href="/new-project">Start a proposal</a><br> -->
        <!-- <a href="/benefits/proposal">Benefits</a><br> -->
        <!-- <a href="/funding">Get funded</a><br> -->
        <!-- <a href="/participate">Become a contributor</a><br> -->
        <!-- <a href="/for-funders">For science funders</a><br> -->
        <!-- <a href="/for-institutions">For institutions</a><br> -->

	</td>
	<td class="footer-nav">



        <div class="foot-head">About</div>
        <a href="/about">Overview</a><br>
        <a href="/benefits">Benefits</a><br>
        <a href="/how-it-works">How it works</a><br>
        <a href="/about/story">Our story</a><br>
        <a href="/blog">Our blog</a><br>
        <a href="/about/jobs">Join the team</a><br>
        <br>

        <!-- <div class="foot-head">Help</div>
        <a href="/help">Help</a><br>
		 -->


	</td>
	<td class="footer-nav">
		<!-- <a href="http://twitter.com/thinklab" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/twitter_com.png" width="40"/></a>
		<a href="https://www.facebook.com/pages/Thinklab/310895039088185" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/facebook_com.png" width="40"/></a>
		<a href="https://plus.google.com/115578983466800753402/" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/plus_google_com.png" width="40"/></a>
		<a href="https://www.linkedin.com/company/thinklab-co" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/linkedin_com.png" width="40"/></a> -->
		<br><br>
		<span class="light" style='font-size:12px;'>&copy; 2015 Thinklab</span>

	</td>

</tr></table>

</div>
</div>
</div>

		

	</div>


  <script></script>
  <script>
  var MEDIA_URL = 'https://think-lab.s3.amazonaws.com/m/';
  var STATIC_URL = 'https://think-lab.s3.amazonaws.com/s/';
  var page_is_authenticated = false;
  var page_page = 'discussion';
  var page_profile_id = undefined;
</script>



	<!-- Note different script type. MUST come before MathJax.js -->
	<script type="text/x-mathjax-config">
	    MathJax.Hub.Config({
	        "HTML-CSS": { preferredFont: "TeX", availableFonts: ["STIX","TeX"],
	                        scale: 110, linebreaks: { automatic:true },
	                        EqnChunk: (MathJax.Hub.Browser.isMobile ? 10 : 50) },
	        tex2jax: { inlineMath: [ ["$$", "$$"], ["\\(","\\)"] ],
	                    displayMath: [ ["$$$","$$$"], ["\\[", "\\]"] ],
	                    processEscapes: true,}
	            });

	</script>
    <script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>





  <script src="https://code.jquery.com/jquery-1.10.2.min.js"></script>
  <script src="https://code.jquery.com/ui/1.11.4/jquery-ui.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" integrity="sha384-0mSbJDEHialfmuBBQP6A4Qrprq5OVfW37PRR3j5ELqxss1yVqOtnepnHVP9aJ7xS" crossorigin="anonymous"></script>
  <script src="https://think-lab.s3.amazonaws.com/s/dist/main.min.js"></script>
  



  
	
  
  
  
<script>
$(function() {
  recordHit('thread', 171);
});
</script>



</body>
</html>
