<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8" /><script type="text/javascript">window.NREUM||(NREUM={}),__nr_require=function(e,n,t){function r(t){if(!n[t]){var o=n[t]={exports:{}};e[t][0].call(o.exports,function(n){var o=e[t][1][n];return r(o||n)},o,o.exports)}return n[t].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<t.length;o++)r(t[o]);return r}({1:[function(e,n,t){function r(){}function o(e,n,t){return function(){return i(e,[c.now()].concat(u(arguments)),n?null:this,t),n?void 0:this}}var i=e("handle"),a=e(2),u=e(3),f=e("ee").get("tracer"),c=e("loader"),s=NREUM;"undefined"==typeof window.newrelic&&(newrelic=s);var p=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],d="api-",l=d+"ixn-";a(p,function(e,n){s[n]=o(d+n,!0,"api")}),s.addPageAction=o(d+"addPageAction",!0),s.setCurrentRouteName=o(d+"routeName",!0),n.exports=newrelic,s.interaction=function(){return(new r).get()};var m=r.prototype={createTracer:function(e,n){var t={},r=this,o="function"==typeof n;return i(l+"tracer",[c.now(),e,t],r),function(){if(f.emit((o?"":"no-")+"fn-start",[c.now(),r,o],t),o)try{return n.apply(this,arguments)}finally{f.emit("fn-end",[c.now()],t)}}}};a("setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(e,n){m[n]=o(l+n)}),newrelic.noticeError=function(e){"string"==typeof e&&(e=new Error(e)),i("err",[e,c.now()])}},{}],2:[function(e,n,t){function r(e,n){var t=[],r="",i=0;for(r in e)o.call(e,r)&&(t[i]=n(r,e[r]),i+=1);return t}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],3:[function(e,n,t){function r(e,n,t){n||(n=0),"undefined"==typeof t&&(t=e?e.length:0);for(var r=-1,o=t-n||0,i=Array(o<0?0:o);++r<o;)i[r]=e[n+r];return i}n.exports=r},{}],4:[function(e,n,t){n.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(e,n,t){function r(){}function o(e){function n(e){return e&&e instanceof r?e:e?f(e,u,i):i()}function t(t,r,o,i){if(!d.aborted||i){e&&e(t,r,o);for(var a=n(o),u=m(t),f=u.length,c=0;c<f;c++)u[c].apply(a,r);var p=s[y[t]];return p&&p.push([b,t,r,a]),a}}function l(e,n){v[e]=m(e).concat(n)}function m(e){return v[e]||[]}function w(e){return p[e]=p[e]||o(t)}function g(e,n){c(e,function(e,t){n=n||"feature",y[t]=n,n in s||(s[n]=[])})}var v={},y={},b={on:l,emit:t,get:w,listeners:m,context:n,buffer:g,abort:a,aborted:!1};return b}function i(){return new r}function a(){(s.api||s.feature)&&(d.aborted=!0,s=d.backlog={})}var u="nr@context",f=e("gos"),c=e(2),s={},p={},d=n.exports=o();d.backlog=s},{}],gos:[function(e,n,t){function r(e,n,t){if(o.call(e,n))return e[n];var r=t();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(e,n,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return e[n]=r,r}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],handle:[function(e,n,t){function r(e,n,t,r){o.buffer([e],r),o.emit(e,n,t)}var o=e("ee").get("handle");n.exports=r,r.ee=o},{}],id:[function(e,n,t){function r(e){var n=typeof e;return!e||"object"!==n&&"function"!==n?-1:e===window?0:a(e,i,function(){return o++})}var o=1,i="nr@id",a=e("gos");n.exports=r},{}],loader:[function(e,n,t){function r(){if(!x++){var e=h.info=NREUM.info,n=d.getElementsByTagName("script")[0];if(setTimeout(s.abort,3e4),!(e&&e.licenseKey&&e.applicationID&&n))return s.abort();c(y,function(n,t){e[n]||(e[n]=t)}),f("mark",["onload",a()+h.offset],null,"api");var t=d.createElement("script");t.src="https://"+e.agent,n.parentNode.insertBefore(t,n)}}function o(){"complete"===d.readyState&&i()}function i(){f("mark",["domContent",a()+h.offset],null,"api")}function a(){return E.exists&&performance.now?Math.round(performance.now()):(u=Math.max((new Date).getTime(),u))-h.offset}var u=(new Date).getTime(),f=e("handle"),c=e(2),s=e("ee"),p=window,d=p.document,l="addEventListener",m="attachEvent",w=p.XMLHttpRequest,g=w&&w.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:w,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var v=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-1039.min.js"},b=w&&g&&g[l]&&!/CriOS/.test(navigator.userAgent),h=n.exports={offset:u,now:a,origin:v,features:{},xhrWrappable:b};e(1),d[l]?(d[l]("DOMContentLoaded",i,!1),p[l]("load",r,!1)):(d[m]("onreadystatechange",o),p[m]("onload",r)),f("mark",["firstbyte",u],null,"api");var x=0,E=e(4)},{}]},{},["loader"]);</script><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info={"beacon":"bam.nr-data.net","queueTime":0,"licenseKey":"cd1101aa94","agent":"","transactionName":"bgcDMkBZW0NZBkdcWVdNJxNcW0FZVwscV1dKB08QW11CQxYTWlBBZgEOC19dW0QCEVtHU1gG","applicationID":"6369297","errorBeacon":"bam.nr-data.net","applicationTime":1413}</script>
    <title>Integrating resources with disparate licensing into an open network | Thinklab</title>
	  <link rel="shortcut icon" href="https://think-lab.s3.amazonaws.com/s/img/favicon.ico" type="image/x-icon" />
    <meta property="og:image" content="https://think-lab.s3.amazonaws.com/s/img" />
    
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
  <link rel="stylesheet" href="https://code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css" />

<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/highlightjs/github.css" />
<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/css/css.css" />
<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/css/bootstrap-multiselect.css" />
<!-- <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'> -->

    
    
    
    
  
    <meta name="citation_doi" content="10.15363/thinklab.d107">
    <meta name="citation_title" content="Integrating resources with disparate licensing into an open network">
    <meta name="citation_online_date" content="Aug. 28, 2015">
    <meta name="citation_author" content="Daniel Himmelstein, Lars Juhl Jensen, MacKenzie Smith, Katie Fortney, Caty Chung">
    <meta name="citation_journal_title" content="Thinklab">
    <meta name="citation_publisher" content="Thinklab">
  

    


	<!-- GoogleAnalytics -->
	<script>
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
	(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
	m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

	ga('create', 'UA-48550230-1', 'thinklab.com');
	ga('require', 'displayfeatures');
	
	ga('send', 'pageview');

	</script>






</head>
<body >
  <form id="form_action" method="post">
    <input type='hidden' name='csrfmiddlewaretoken' value='pQNiZQX4gF2vyFeXrnbUl9EmuTWA5MG1' />
</form>

	

	<div class="body-wrapper">

		


<!-- <div style="height:5px;background-image:url('https://think-lab.s3.amazonaws.com/s/img/top-bg.gif');"></div> -->
<div class="gradient">
	<div class="main-centered-div header-container">

<table class="full-width" style="height:60px;"><tr>
	<td style="width:200px;vertical-align:middle;padding-bottom:15px;">
		<a href="/" style="font-weight:normal;color:#212121;"><img src="https://think-lab.s3.amazonaws.com/s/img/logo.png" width="160"></a>
	</td><td style="vertical-align:middle;text-align:left;padding-top:6px;width:100%;padding-left:40px;">
		<ul id="not-search-block"  class="main-header">


			


				<li class="dropdown">
					<a class="dropdown-toggle" data-toggle="dropdown" href="#">Browse <span class="caret"></span></a>
					<ul class="dropdown-menu" role="menu">
						<li><a href="/proposals">Proposals</a></li>
						<li><a href="/projects">Projects</a></li>
						<li><a href="/publications">Article discussion</a></li>
						<li><a href="/discussion">All discussion</a></li>
					</ul>
				</li>
				<li><a href="/leaderboard">Leaderboard</a></li>

			
				<li><a href="/about">About</a></li>
			


		</ul>

    <ul id="search-block" class="main-header" style="display: none; margin-right: 30px;">
        <li class="full-width">
            <form method="get" action="/search">
			<div class="input-group full-width">
				<input name="q" type="text" class="form-control search-input">
			</div>
            </form>
        </li>
    </ul>

	</td><td style="vertical-align:middle;text-align:right;padding-top:6px; min-width: 1%; white-space: nowrap;">
		<ul class="main-header pull-right" style="padding-left: 0;">
    		
                <li><a href="/login" style="color:silver;">Login</a></li>
                <li style="margin-right:0"><a class="btn btn-default btn-sm btn-upper" href="/signup">Join</a></li>
            

        </ul>
    </ul>
	</td>
</tr></table>

<!-- <div id="global-search-div" class="main-centered-div" style="display:none;">
	<input type="text" value="" class="thread-search form-control" placeholder="Search" data-action="/discussion">
</div> -->

	</div>
</div>

		
	  



	  


		

		<div class="dark-head"><div class="main-centered-div">

			

		</div></div>
		
		<div class="main-centered-div" style="margin-top:40px;padding-bottom:340px">
	    

<div style="margin-top:-20px; margin-bottom:20px;">

    <a class="side-box" href="/p/rephetio" style="background-color:#ebfdff;"><div class="side-type">Project:</div> <div class="side-title">Rephetio: Repurposing drugs on a hetnet [rephetio]</div></a><br>



</div>

<table style="margin-top:0px;"><tr>
	<td class="thread-table-content">

		<table class="full-width">
			
<tr class="comment-tr target-hl" id="1" data-comment-id="417">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="417"></a>

    

			<h2 class="page-title" style="margin-bottom:5px;margin-top:0px !important;">Integrating resources with disparate licensing into an open network</h2>

			<div style="margin-top:5px;color:gray;margin-bottom:20px;font-size:13px;">
				<a style="font-weight:normal;color:#212121" href="/u/dhimmel"><strong>Daniel Himmelstein</strong></a>
				<span class="is-researcher">Researcher</span>
				&nbsp; <a href="#1" class="light">679 days ago</a>
				
			</div>

      

          <hr>
		

    <h1>Network overview</h1>

<p>We recently released the <a href="http://thinklab.com/discussion/one-network-to-rule-them-all/102" class="citation thread-citation" data-key="10.15363/thinklab.d102">first version</a> of our network containing 10 node types and 27 edge types. The network contains data (nodes and edges) extracted from <a href="https://github.com/dhimmel/integrate/tree/0e343d8745a98757c86e73f645b41353f52b82b5/licenses">28 resources</a>. Many of these 28 resources have themselves compiled data from disparately-licensed resources. In addition:</p>

<ul><li>12 lack any licensing information</li><li>10 use standard licenses</li><li>6 use custom licenses</li><li>3 resources are publication supplements</li><li>6 forbid commercial use</li><li>2 forbid any redistribution of the data</li></ul>

<h1>Why an open network</h1>

<p>We <a href="http://thinklab.com/discussion/enabling-reproducibility-and-reuse/23#6" class="citation thread-citation" data-key="10.15363/thinklab.d23">are committed</a> to performing an open project, where all code, data, analyses, and results are maximally reproducible and reusable. The foundation of our research is that datasets are more informative when placed in a broader context. Through integration, we create a resource that is more informative and versatile than the 28 separate sources.</p>

<p>However, data integration is challenging and time intensive. Thus far, our integration effort consists of an 8 month time investment, 41 Thinklab <a href="http://thinklab.com/p/rephetio/discussion">discussions</a>, and 35 GitHub <a href="https://github.com/dhimmel?tab=repositories">repositories</a>. By making our network public and extensible, other researchers can avoid this laborious process while harnessing the benefits of integration.</p>

<h1>The licensing problem</h1>

<p>We initially released our network under the <a href="https://creativecommons.org/publicdomain/zero/1.0/">CC0</a> (public domain) license, but <a href="/u/larsjuhljensen" class="username">@larsjuhljensen</a> <a href="http://thinklab.com/discussion/one-network-to-rule-them-all/102#2" class="citation thread-citation" data-key="10.15363/thinklab.d102">pointed out</a> that this may violate many sources' licensing. While we used only publicly available resources — funded primarily by the public — many resources are burdened by restrictive licenses. We now must integrate data with incompatible licenses that require legal expertise to understand and operate in jurisdiction dependent manners.</p>

<h1>Compliance and caveats</h1>

<p>We are seeking expert advice on how to proceed. We would like to achieve the following:</p>

<ul><li>a network that is publicly available in full and maximally unrestricted</li><li>public domain findings. Foremost, unencumbered predictions of drug efficacy</li><li>legal compliance</li><li>normative compliance that respects the intent of the data creators</li><li>minimal pruning of the current network to preserve our investment</li></ul>

<p>We plan to add node/edge-specific attribution and license information to our network, but will await expert advice before proceeding.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="2" data-comment-id="420">
  <td class="comment-profile-avatar">
		<a href="/u/larsjuhljensen"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/125_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="420"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/larsjuhljensen">Lars Juhl Jensen</a>
				
				<span class="light">&nbsp;<a href="#2" class="light">679 days</a>
          
        </span>
      </div>

    

    <p>You may also want to consider splitting the network into multiple files. For example, you may have a base file that includes, for example, only public domain and CC-BY content. The edges with SA and/or NC clauses could be in "add-on" files. This partially avoids the problem that your complete network file becomes subject to the lowest common denominator.</p>

<p>Having multiple files will allow people to "pick their poison", so to speak. If they need the most permissive license, they will get a less complete network. If they want the most complete network, they will have to live with a less permissive license.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
<li id="note-178" class="inline-note toggle-parent" data-note-id="178">
	<p><a href="/u/dhimmel">Daniel Himmelstein</a>: Great suggestion. This way users can avoid having to subset the network and reconcile various licenses.
		
	</p>
	
	<table style="display:none;"></table>
</li>

				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="3" data-comment-id="421">
  <td class="comment-profile-avatar">
		<a href="/u/larsjuhljensen"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/125_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="421"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/larsjuhljensen">Lars Juhl Jensen</a>
				
				<span class="light">&nbsp;<a href="#3" class="light">679 days</a>
          
        </span>
      </div>

    

    <p>Regarding the 12 that completely lack any licensing information, I would contact the authors. For academic databases/datasets, this is usually due to people not knowing that when it comes to copyright, the default is "all rights reserved". Academics often put things on the internet, thinking that this makes it "public domain". If you ask them, they will likely be happy to put a CC0 waiver or CC-BY license on it.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="4" data-comment-id="423">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="423"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#4" class="light">671 days</a>
          
        </span>
      </div>

    

    <h1>Workflow details</h1>

<p>Our data workflow consists of three major stages. Each stage invokes various <a href="http://www.smashingmagazine.com/2011/06/understanding-copyright-and-licenses/">aspects of copyright</a> as described below</p>

<h2>1) Resource processing</h2>

<p>Most resources require processing before they can be added to the network. Common steps include terminology conversion, quality control, subsetting, and record merging.</p>

<p>Our general procedure is to create a public GitHub repository for each resource (examples <a href="https://github.com/dhimmel/diseases">1</a>, <a href="https://github.com/dhimmel/SIDER4">2</a>, <a href="https://github.com/dhimmel/lincs">3</a>, <a href="https://github.com/dhimmel/uberon">4</a>, <a href="https://github.com/dhimmel/drugbank">5</a>). Separate repositories help keep the project modular and reusable. Each repository contains a <code>download</code> directory where we store the unmodified input. Having the local copy is important for reproducibility because the original download location may become unavailable or serve an updated dataset. Therefore, our <code>download</code> directory redistributes unmodified data.</p>

<p>Next, we process data from the <code>download</code> directory and save the resulting datasets in the <code>data</code> directory. The processing steps generally change the database model and field names and include a substantial portion of the original data. However, the original data has usually been transformed in some regard.</p>

<p><strong>Proposed action</strong>: apply the source's license to the contents of <code>download</code>. For the contents of <code>data</code>, apply either the source's license or CC0 if the underlying data is not <a href="http://www.bitlaw.com/copyright/database.html">subject to copyright</a> or the derivative work qualifies as fair use. Resources without a license or that explicitly forbid redistribution are problematic. We propose contacting the creators of these resources for permission or licensing clarification. Components in these repositories that do not derive from protected resources will be released as CC0.</p>

<h2>2) Integrative network</h2>

<p>Our <a href="https://github.com/dhimmel/integrate"><code>integrate</code></a> repository combines the resource-specific data from stage 1 into a single network. The <a href="https://github.com/dhimmel/integrate/tree/master/compile"><code>compile</code></a> directory merges resources with the same type of information. The creation of the network is performed by <a href="https://github.com/dhimmel/integrate/blob/master/integrate.ipynb"><code>integrate.ipynb</code></a>. We have <a href="https://github.com/dhimmel/integrate/tree/master/licenses">compiled</a> the licenses for each resource. The network is saved as text files in the <a href="https://github.com/dhimmel/integrate/tree/master/data"><code>data</code></a> directory with <a href="https://github.com/dhimmel/integrate/blob/master/data/hetnet.json.gz"><code>hetnet.json.gz</code></a> as the main release. In this integrated network, the database model and field names from the original resource are not present, just derived data.</p>

<p><strong>Proposed action</strong>: Adopt a per node/edge licensing framework. Identify which nodes and edges, if any, are eligible for CC0 release. CC0 release may be possible if the creators chose a permissive license or give us permission, the network is fair use, or if the underlying content is not subject to copyright.</p>

<h2>3) Network analyses</h2>

<p>Next, we use the integrated network from stage 2 for data mining. The purpose of the data mining is to evaluate methods, extract insights, and make predictions. As an example, see <a href="https://github.com/dhimmel/snplentiful">this analysis</a> <span class="citation">[<a href="https://doi.org/10.5281/zenodo.30105" class="citation" data-key="10.5281/zenodo.30105">1</a>]</span> of the network that <a href="/u/caseygreene" class="username">@caseygreene</a> and I recently did for a separate project.</p>

<p>Here, it is crucial that findings from analyses on the network are fair use and can be placed in the public domain. Since, the network contains data with <a href="https://wiki.creativecommons.org/wiki/Wiki/cc_license_compatibility">incompatible</a> licenses such as CC-BY-SA and CC-BY-NC, data mining will be impossible if not considered fair use. In the US, <a href="http://www.arl.org/storage/documents/TDM-5JUNE2015.pdf">precedent</a> implies our network analyses qualify as fair use. </p>

<p><strong>Proposed action</strong>: Identify whether our network analyses qualify as fair use and whether our results can be released as CC0. Evaluate when and if we are subject to European copyright laws, which are less favorable for content users.</p>

<h1>Expert feedback requested</h1>

<p>We are seeking expert advice. Specifically, are the proposed actions compliant with copyright law? Regarding the three stages, are we on the right track? Will network analyses count as fair use?</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="5" data-comment-id="427">
  <td class="comment-profile-avatar">
		<a href="/u/mackenziesmith"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/135_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="427"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/mackenziesmith">MacKenzie Smith</a>
				
				<span class="light">&nbsp;<a href="#5" class="light">668 days</a>
          
        </span>
      </div>

    

    <p>Your analysis of the situation looks great — you've correctly described the difficulty of combining incompatible licenses and the data they cover, and the potential of fair use (for extracting data subsets and data mining) for what you're trying to do. And for the datasets that lack a license, you know that in many cases they aren't protected by copyright so you're free to do what you want. Federal government agencies are notorious for refusing to assign licenses or rights waivers to the data they release, claiming that everything they have and do is in the public domain and we should all just know that, so sometimes no license means you're fine. Your goal of making it clear to users what rights and licenses apply to which datasets is laudable.</p>

<p>The one thing I didn't see you covering is liability. I can't figure out who actually owns the work that you're doing — you want to put it in the public domain, which is great, but do you personally have the right to do that? Are you working on a grant project or employed by a university that might claim "ownership" of your results? This is usually dealt with by the licenses. Apache open source software licenses include the language "Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License." Even CC licenses include language like "No warranties are given. The license may not give you all of the permissions necessary for your intended use. For example, other rights such as publicity, privacy, or moral rights may limit how you use the material." So if you're using CC0 wherever you can, you might want a separate statement of warranty (or lack thereof) unless you want to be liable, or implicate your institution, if you do accidentally screw up (easy enough to do, in such a complex project, even when you've done everything you can). </p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="6" data-comment-id="429">
  <td class="comment-profile-avatar">
		<a href="/u/larsjuhljensen"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/125_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="429"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/larsjuhljensen">Lars Juhl Jensen</a>
				
				<span class="light">&nbsp;<a href="#6" class="light">668 days</a>
          
        </span>
      </div>

    

    <p>Regarding the problem of incompatible licenses, it is very important that you are clear on the difference between redistribution and data mining.</p>

<p>You write that "Since, the network contains data with incompatible licenses such as CC-BY-SA and CC-BY-NC, data mining will be impossible if not considered fair use". This is to my knowledge simply not true. There is no problem whatsoever in combining material from these incompatible licenses and mining it in any way that you want. The reason is that copyright purely has to do with how you are allow to redistribute things. And if the data mining leads to some results that are substantially different and not effectively a copy of the original material, there is also no problem in redistributing the results.</p>

<p>The problem comes when you want to make what is effectively a meta-resource that combines material from a lot of databases and redistributes it. In this case, you are redistributing something that is effectively a reformatted version of the material. In my opinion, your network falls squarely in that category</p>

<p>However, the solution is very simple. As I have suggested before, you can split the network into subnetworks, that are all redistributed under their respective licenses. You can bundle everything CC-BY-SA in one file and redistribute it under CC-BY-SA. You can bundle everything CC-BY-NC in another file and redistribute it under CC-BY-NC. And as described above, nothing prevents anyone in the world from legally downloading both files, combining them, and mining the data as they please.</p>

<p>To make it simpler, let me make an analogy from the world of text mining, where the situation is somewhat more clearcut, since there is no doubt that articles are subject to copyright law. I can download some articles under CC-BY-SA and some others under CC-BY-NC. I can run text mining on all of them despite the licenses being incompatible, and I can redistribute the results of my efforts under any license I please, because the results are my results, which are not simply a reformatting of the original text. However, I cannot take all the articles, combine them into a text corpus, and release it under CC0.</p>

<p>Caveats: I am not a lawyer, this does constitute legal advice etc.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
<li id="note-184" class="inline-note toggle-parent" data-note-id="184">
	<p><a href="/u/dhimmel">Daniel Himmelstein</a>: I am interested in subnetworks for user convenience — if a user wants only CC-BY-SA content, then the subnetwork saves them time. However, I disagree that subnetworks are a substitute for a complete network with mixed licensing. Merging networks will be burdensome to my hypothetical user who is interested only in analyzing rather than redistributing the network. Additionally, I am not sure the entire network can be represented by copyright uniform subnetworks. For example, DrugBank <a href="https://github.com/dhimmel/integrate/blob/0e343d8745a98757c86e73f645b41353f52b82b5/licenses/custom/DrugBank.md">forbids</a> commercial reuse. However, our drug–gene binding edges derive from the CC-BY-SA ChEMBL resource. Thus the CC-BY-SA subnetwork would contain edges whose nodes cannot be included.
		
	</p>
	
	<table style="display:none;"></table>
</li>

				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="7" data-comment-id="430">
  <td class="comment-profile-avatar">
		<a href="/u/larsjuhljensen"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/125_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="430"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/larsjuhljensen">Lars Juhl Jensen</a>
				
				<span class="light">&nbsp;<a href="#7" class="light">668 days</a>
          
        </span>
      </div>

    

    <p><a href="/u/mackenziesmith" class="username">@mackenziesmith</a> makes a very good point about liability, which in my opinion is why you should not attempt to take copyrighted material, claim fair use under US law, and slap a CC0 waiver on it.</p>

<p>Imagine someone in Europe were to download your network, assume that everything was free of copyright (which is what CC0 effectively promises), take all the SIDER data, and redistribute it under CC0. Since SIDER is covered by European <em>sui generis</em> database rights, they could get sued and would likely lose. Subsequently, they could choose to sue you for liabilities.</p>

<p>Caveats: I am not a lawyer, this does constitute legal advice etc.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="8" data-comment-id="428">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="428"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#8" class="light">667 days</a>
          
        </span>
      </div>

    

    <h1>Who owns the created work</h1>

<p>After some background <a href="http://www.d.umn.edu/~pschoff/documents/ElliotR05WhoOwnsScientificDatapdf.pdf">reading</a> <span class="citation">[<a href="https://doi.org/10.1087/0953151053584984" class="citation" data-key="10.1087/0953151053584984">1</a>]</span> and <a href="https://www.youtube.com/playlist?list=PLYTiwx6hV33tXEfueCk5k7xyLePcqj4XK">video watching</a>, who owns the work we're creating is not straightforward.</p>

<p>I am a graduate student at UCSF and my PI, <a href="/u/sergiobaranzini" class="username">@sergiobaranzini</a>, is a professor at UCSF. I am largely, but not completely, funded by an NSF Graduate Research Fellowship whose conditions <a href="http://www.nsf.gov/pubs/2015/nsf15597/nsf15597.htm">state</a>:</p>

<blockquote><p>The National Science Foundation claims no rights to any inventions or writings that might result from its fellowship or traineeship grants.</p></blockquote>

<h2>Copyright and the UC</h2>

<p>The UC's 1992 <a href="http://copyright.universityofcalifornia.edu/resources/copyright-ownership.html">'copyright ownership' policy</a> stipulates ownership by category of work. Several of these categories may apply:</p>

<ul><li><p>academic appointee originator ownership of "scholarly/aesthetic work"</p><blockquote><p>A scholarly/aesthetic work is a work originated by a designated academic appointee resulting from independent academic effort. Ownership of copyrights to scholarly/aesthetic works shall reside with the designated academic appointee originator, unless they are also sponsored works or contracted facilities works, or unless the designated academic appointee agrees to participate in a project which has special provisions on copyright ownership pursuant to Section V.C. of this Policy.</p></blockquote></li><li><p>originator ownership of "personal work"</p><blockquote><p>A personal work is a work that is prepared outside the course and scope of University employment (except for permissible non-University consulting activities) without the use of University Resources. Ownership of copyrights to Personal works shall reside with the originator.</p></blockquote></li><li><p>originator ownership of "student work":</p><blockquote><p>A student work is a work produced by a registered student without the use of University funds (other than Student Financial Aid), that is produced outside any University employment, and is not a sponsored, contracted facilities, or commissioned work. Ownership of copyrights to student works shall reside with the originator.</p></blockquote></li><li><p>university ownership of "institutional work":</p><blockquote><p>Except as otherwise provided in this Policy, the University shall own all copyrights to works made by University employees in the course and scope of their employment and shall own all copyrights to works made with the use of University resources.</p></blockquote></li></ul>

<p>Therefore, UC's asserted ownership is dependent on which categories our work falls under. Additional guidance <a href="http://copyright.universityofcalifornia.edu/ownership/works-created-at-uc.html">states</a>:</p>

<blockquote><p>University staff who create works within the scope of their employment generally do not own the copyright to the work. A work prepared by an employee within the scope of his or her employment is considered a "work made for hire."  When a work qualifies as a work made for hire, the employer or commissioning party is considered its author. Under UC policies, some written works created by certain categories of UC faculty, graduate students, and staff are considered works made for hire. </p></blockquote>

<p>Thus, the University's assertion of ownership may be contradicted by the <a href="http://chronicle.com/article/Employees-or-Not-/145573/">strong argument</a> that graduate students, such as myself, are not employees and do not produce "work made for hire". Furthermore, the policies and guidelines are outdated and not well tailored towards the collaborative, digital, online, and open approach our project takes. The work I perform goes beyond the sole purposes of studentship, employment, and institutional work. And the academic community has established norms and precedent for allowing creators to transfer copyright and choose licensing — the foremost examples being academic publishing and open source software contribution.</p>

<h2>Data and the UC</h2>

<p>The UC <a href="http://copyright.universityofcalifornia.edu/resources/copyright-ownership.html">'copyright ownership' policy</a> explicitly states that it only:</p>

<blockquote><p>addresses ownership of copyright; it does not address ownership or access to the underlying research results or data, as covered in Academic Personnel Manual Section 020. </p></blockquote>

<p>The Academic Personnel Manual <a href="http://www.ucop.edu/academic-personnel-programs/_files/apm/apm-020.pdf">Section 020</a>, dated in 1953, provides little clarification:</p>

<blockquote><p>All such research shall be conducted so as to be as generally useful as possible. To this end, the right of publication is reserved by the University. The University may itself publish the material or may authorize, in any specific case, a member or members of the faculty to publish it through some recognized scientific or professional medium of publication. A report detailing the essential data and presenting the final results must be filed with the University. Notebooks and other original records of the research are the property of the University.</p></blockquote>

<p>Outside of official policy, UC appears to claim ownership of results and data. Quoting from a <a href="https://youtu.be/QQOEG_PyRWY">talk</a> by <a href="/u/mackenziesmith" class="username">@mackenziesmith</a>:</p>

<blockquote><p>The University of California posits that it actually has a contractual obligation to maintain the ownership of all research data produced from grant funded projects by any researcher at UC, especially federally funded grants. So they claim that the university owns the data.</p></blockquote>

<p>Additionally, a <a href="http://ucsd.libguides.com/c.php?g=90957&amp;p=585144">UCSD guide</a> states:</p>

<blockquote><ul><li>Data produced by UC researchers belong to the Regents of the University of California.</li><li>To promote sharing and unlimited use of your data, make your data available under a Creative Commons <a href="http://creativecommons.org/choose/zero">CC0 Declaration</a>.</li></ul></blockquote>

<p>These seemingly contradictory statements imply that UC may own the data but that its creators are free to release it into the public domain.</p>

<h2>Resolutions</h2>

<p>We are looking for suggested courses of action to address the ambiguity and potential multiplicity of claims regarding ownership. Two possible actions are:</p>

<ul><li>applying a <em>without warranty</em> clause to our licensing to limit our liability.</li><li>identifying all potential parties that may claim ownership and request permission to release the work as freely as possible given the <a href="#4">aforementioned considerations</a>.</li></ul>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="9" data-comment-id="431">
  <td class="comment-profile-avatar">
		<a href="/u/mackenziesmith"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/135_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="431"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/mackenziesmith">MacKenzie Smith</a>
				
				<span class="light">&nbsp;<a href="#9" class="light">667 days</a>
          
        </span>
      </div>

    

    <p>At a workshop I organized at UC Davis last year — Data Rights and Data Wrongs — senior counsel from the UC Office of General Counsel (i.e., the university's lawyers) was very clear that UC retains ownership rights to original data as the official 'grantee' and to insure compliance with federal laws for research conduct, etc. I think the relevant policy is here <a href="http://www.ucop.edu/raohome/cgmemos/84-31.html">http://www.ucop.edu/raohome/cgmemos/84-31.html</a> (old but still in effect). So I think your assessment is right that UC asserts ownership but allows you to release the data under reasonable terms, including CC0. However most of the data you're working with isn't original to you, so what UC 'owns' is your own findings and if you did something wrong, the university is liable to some extent. </p>

<p>Of course, finding all the rights holders and getting their explicit permission to do what you're doing would be ideal, but is that practical? Do you even know who holds the rights to all the data sources? I disagree with the point that you can't rely on Fair Use and release your results under a CC0 waiver — I believe that's what Fair Use is for, if it's truly transformative — but you might want to be explicit about the waiver of liability. Especially given how gray the area you're working in is, legally speaking.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
<li id="note-181" class="inline-note toggle-parent" data-note-id="181">
	<p><a href="/u/larsjuhljensen">Lars Juhl Jensen</a>: The big problem I see here is the term "results". If the results are actually new results, then I agree that they can be released whichever way you like. But if your "results" are in fact other people's databases mapped to different identifiers, bundled, and reformatted in JSON format, then claiming fair use is in my opinion a very risky proposition.
		
	</p>
	
	<table style="display:none;"></table>
</li>

<li id="note-182" class="inline-note toggle-parent" data-note-id="182">
	<p><a href="/u/dhimmel">Daniel Himmelstein</a>: <a href="/u/larsjuhljensen" class="username">@larsjuhljensen</a> and <a href="/u/mackenziesmith" class="username">@mackenziesmith</a>, I think we may all be on the same page. Results from <em>network analyses</em> are truly transformative and thus eligible for CC0 licensing due to fair use. However, the <em>resource processing</em> and <em>integrative network</em> <a href="#4">steps</a>, which distribute unmodified downloads as well as network-coerced versions of databases, should generally transmit the source licensing.
		
	</p>
	
	<table style="display:none;"></table>
</li>

<li id="note-183" class="inline-note toggle-parent" data-note-id="183">
	<p><a href="/u/larsjuhljensen">Lars Juhl Jensen</a>: <a href="/u/dhimmel" class="username">@dhimmel</a>, we almost agree. The one point where I disagree is that, in my opinion, fair use has nothing to do with it. If you do a network analysis that truly produces new results (e.g. predicting new edges based on the imported ones), then those edges are yours. You are free to do with them whatever you want, not because of fair use, but simply because you are the original creator :-)
		
	</p>
	
	<table style="display:none;"></table>
</li>

				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="10" data-comment-id="438">
  <td class="comment-profile-avatar">
		<a href="/u/katiefortney"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/137_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="438"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/katiefortney">Katie Fortney</a>
				
				<span class="light">&nbsp;<a href="#10" class="light">654 days</a>
          
        </span>
      </div>

    

    <p>Hi all,<br>I am <strong>a</strong> lawyer, but not <strong>your</strong> lawyer (or UC’s lawyer), and this isn’t legal advice. Also, I’m not yet familiar with the data sources or the project at a high level of detail - but here’s what I can say about the general issues.</p>

<h1>I. U.S. law</h1>

<h2>A. Layers of copyright</h2>

<ol><li>I notice that you’ve generally got a single assessment of copyright/licensing issues associated with each data source. I could see each one having up to three. For instance, you could have facts that both the original distributor and the downstream user agree are in the public domain - layer 1, you can do anything with those if you’ve extracted them and rearranged them. They could be collected and shared in a database that’s licensed under something like CC BY-SA, and the terms of that license would need to be followed when distributing the whole database, or parts of it, in such a way that you were copying &amp; distributing the licensor’s copyrightable arrangement/selection/original authorship. The database is layer 2. Then you might have special software created by the data distributor to access and manipulate the data and/or the database, and that might be licensed separately, either with a CC license or with an open source software license like MIT or BSD. That’s layer 3. Without being an expert on these particular databases, I’m guessing 2 and 3 are often going to be the same thing, but it’s best not to just assume that.</li><li>If no license terms are posted, the underlying facts are in the public domain, and any copyrightable expression like software, or creative arrangement, is copyright default’s “all rights reserved.”</li><li>Why am I bothering to spell this out? Depending on the terms of these things and how you want to use/redistribute them, it’s possible that something like the GSEA/MIT terms that look really restrictive may not be a hurdle. I read that one to limit what you can do with layers 2 (“the DATABASE”) and 3 (“the PROGRAM”), but less so layer 1. If you’re committed to redistribution of layer 2 wholesale, then yeah, we’ve got barriers.</li></ol>

<h2>B. Particular licenses</h2>

<ol><li>Software licenses are more commonly used for software than CC licenses are, although either would theoretically work. UC recommends BSD and MIT licenses in particular, because they don’t say anything about patents.</li><li>There’s some interesting stuff in the fine print of the CC licenses that might be helpful. For instance, the SA requirement has to be retained by the original material, and has to be attached to any “Adapted Material.” But not every use of a work is “Adapted Material.” Compilations generally aren’t an adaptation, so maybe there’s some creative thinking to be done around that. Attribution requirements can be satisfied in “any reasonable manner based on the medium, means, and context,” and maybe we could do some thinking about what’s a reasonable manner in <em>this</em> context.</li></ol>

<h2>C. The CC0 dedication</h2>

<ol><li>Like the CC licenses, the CC0 dedication only applies to … what it <em>can</em> apply to. Just the things the licensor has the ability to waive rights to. Here’s the language:<br><em>Affirmer disclaims responsibility for clearing rights of other persons that may apply to the Work or any use thereof, including without limitation any person's Copyright and Related Rights in the Work. Further, Affirmer disclaims responsibility for obtaining any necessary consents, permissions or other rights required for any use of the Work.</em><br>On the bright side this means that theoretically, you can just release your own layer/contributions/authorship as CC0, without affecting the things you reference or incorporate. Unfortunately, this isn’t so helpful for downstream users who have to try to figure out what the CC0 applies to and what other rights are lurking there. Lots of explanation, labeling, help pages, etc. can be useful if people read them.</li></ol>

<h2>D. UC and data “ownership”</h2>

<ol><li>I’m going to keep putting “ownership” in quotes until something official explains to me, to my satisfaction, exactly what UC is claiming to own. The APM policy they seem to rely on from the 50s talks about records, like notebooks. Data can only be owned to the extent there’s intellectual property involved, like patent, copyright, or trade secrets. If none of those are present, there’s nothing to own. There may be contractual restrictions about what you can or must do with something, that you’ve agreed to as part of an employment agreement or a grant agreement, but that’s a different animal, and will be more explicit than the automatic rights involved in copyright.</li><li>Depending on how this project is funded I think any copyrightable work here - the software, for instance - could be student work, personal work, or institutional work, under the 1992 Copyright Ownership Policy. It’s unlikely to be a scholarly/aesthetic work because of the definition of “designated academic appointee,” but I don’t know who the co-authors are.</li><li>UC’s lawyers - OGC and general counsel - will generally weigh in to assess legal risk to the university or disposition of university intellectual property. They will not/cannot provide advice about liability to an individual, or assessment of their personal intellectual property.</li><li>Each campus has a designated authority who is authorized to approve licensing decisions and the like on that campus. I believe UCSF’s is Karin Immergluck. In my experience, if we get to a place where we decide “well, this project includes copyrights owned by UCSF, but we want to license them CC BY or dedicate them to the public domain,” an email to the relevant campus person explaining the rationale (and preferably why this isn’t something the university would make money off of) results in a quick approval.</li></ol>

<h2>E. Contracts</h2>

<ol><li>U.S. copyright law includes all kinds of rights for users, including fair use, and the fact that certain things are in the public domain. But you can sign a contract giving away any of these rights. To the extent that you have to agree to restrictive terms to get access to a data set, those terms may effectively limit your rights to reuse even factual data. It’s like when libraries sign a license for a ProQuest database and promise not to make any copies of newspaper articles from the 1800s.</li></ol>

<h1>II. International law</h1>

<h2>A. Database protection generally</h2>

<p>Lots of countries protect a database, but not the underlying facts, with copyright law. I see you found the Bitlaw page on this, which is where I would have sent you.</p>

<h2>B. European database directive</h2>

<p>I’ve never had occasion to deal with this before, but there’s a parallel thing in some countries like Italy for, e.g. digitizing old manuscripts. Limited protection as an incentive to create the thing or make it accessible. It sounds like enough of a pain that it’s probably worth figuring out which of the proposed sources are covered. That may be time consuming and difficult - so, something for further discussion/research.</p>

<h2>C. International liability for potential copyright infringement</h2>

<p>This is a tricky issue, and a fun subject for law review articles. Most of them revolve around <em>selling</em> things internationally, for a couple reasons. First, that’s when you’re likely to make people mad enough to bother with suing you. Second, there are jurisdictional issues about how much you have to do in a country to subject yourself to a lawsuit there. All I can say is that internet plus free distribution doesn’t automatically equal global legal risk. But that may not matter much because...</p>

<h1>II. There’s law, and then there’s politics.</h1>

<p>If we were looking at hundreds of sources, contacting them individually would be a horrible thing to contemplate. With a couple dozen, it might be worth it to put together a form letter to let people know about the project, to avoid burning bridges with current colleagues and potential future collaborators. This could address the things these folks are most likely to be concerned about: what is this project doing with the data sources? How will downstream users be able to tell the source of the data? What things will facilitate or burden commercial use? And there could be a few different versions depending on the legal assessment of the underlying rights and which ones the project implicates - maybe a letter to US sources is more of an FYI, and one to European sources asks them to reply granting permission. Or maybe if the project really wants <em>everything</em> to be as open as possible, you just actually get permission from everyone to a release of some version of their data, in this context, under your chosen license. Just because they make it available to the world under, e.g., CC BY-SA doesn’t mean they can’t make it available to you under different terms. There are options. None of them are as easy as “just use it,” but if people have tried to restrict how their stuff is used you have to decide the relative value you place on maximizing your rights under the law vs. maintaining goodwill.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
<li id="note-186" class="inline-note toggle-parent" data-note-id="186">
	<p><a href="/u/larsjuhljensen">Lars Juhl Jensen</a>: Thanks a lot for also pointing out the politics aspect. In my opinion, the risk of actually getting sued in academia is probably fairly low. However, if you were to systematically take resources with restrictive licenses, integrate them, and redistribute the complete data under CC0, you would almost for sure be burning bridges.
		
	</p>
	
	<table style="display:none;"></table>
</li>

				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="11" data-comment-id="440">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="440"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#11" class="light">648 days</a>
          
        </span>
      </div>

    

    <h1>Mixed copyright licensing</h1>

<p>As explained <a href="#4">above</a>, we have created resources (mostly GitHub repositories) that contain content with varying licenses and restrictions. Therefore, we need to:</p>

<ul><li>license different files from the same repository under different licenses</li><li>license different portions within a single file under different licenses</li></ul>

<p>It appears that there is not a rigid formula for how to specify mixed copyright. I found a few examples including the <a href="https://github.com/neo4j/neo4j/blob/5f991933fa531f7dd901d4b6570fe78d73f8bb3c/README.asciidoc">neo4j source code</a> and a <a href="https://github.com/lucidv01d/samasy/blob/32148fdd01c1993a991787d23f4c851ff26f7c01/LICENSE">license</a> the <a href="https://ita.ucsf.edu/">UCSF Office of Innovation, Technology &amp; Alliances</a> created for my classmate.</p>

<p>In the later case, my classmate asked the ITA to assist him in creating an open source license. As <a href="/u/mackenziesmith" class="username">@mackenziesmith</a> predicted, UC asserted ownership of the content and forbid any for-profit usage. As an aside, I am highly confident that UC does not own my work, because it is not <em>work made for hire</em>, and I never agreed to any transfer of ownership.</p>

<h2>Proposed license for the SIDER4 repository</h2>

<p>SIDER 4 is a resource <a href="http://thinklab.com/discussion/extracting-side-effects-from-sider-4/97" class="citation thread-citation" data-key="10.15363/thinklab.d97">we're using</a> for drug side effects. I propose the <a href="https://github.com/dhimmel/SIDER4/blob/45d0ba626e406ae3ce6f8f503f09d5af9b4b7b63/LICENSE.md">following license</a> for the repository:</p>

<blockquote><p>SIDER 4 data is <a href="http://sideeffects.embl.de/download/">released</a> under a <a href="http://creativecommons.org/licenses/by-nc-sa/4.0/">CC-BY-NC-SA</a> license. Therefore, all redistributed and derived content from SIDER 4 is <a href="http://creativecommons.org/licenses/by-nc-sa/4.0/">CC-BY-NC-SA</a>. All original content is released under <a href="https://creativecommons.org/publicdomain/zero/1.0/">CC0</a>.</p><p>Accordingly, the following files are <a href="http://creativecommons.org/licenses/by-nc-sa/4.0/">CC-BY-NC-SA</a>:</p><ul><li><code>download/meddra_all_indications.tsv.gz</code></li><li><code>download/meddra_all_se.tsv.gz</code></li><li><code>download/meddra_freq.tsv.gz</code></li><li><code>data/indication.tsv</code></li><li><code>data/side-effects.tsv</code></li></ul><p><strong>Disclaimer</strong>: The repository is provided "as is", without warranty of any kind, express or implied, including but not limited to the warranties of merchantability, fitness for a particular purpose and noninfringement. In no event shall the authors or copyright holders be liable for any claim, damages or other liability, whether in an action of contract, tort or otherwise, arising from, out of or in connection with the repository or the use or other dealings in the repository.</p></blockquote>

<p>We added the disclaimer to limit our liability as <a href="#4">suggested</a> by <a href="/u/mackenziesmith" class="username">@mackenziesmith</a>. Does the proposed license seem adequate? Is it clear? <a href="/u/katiefortney" class="username">@katiefortney</a>, any suggestions?</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
<li id="note-193" class="inline-note toggle-parent" data-note-id="193">
	<p><a href="/u/katiefortney">Katie Fortney</a>: Generally with a CC license you should say how the author wants to be credited - if not specific as to manner, than at least with a name of person or organization. That's all so far. I'll keep an eye out now that I'm back in the office...
		
	</p>
	
	<table style="display:none;"></table>
</li>

				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="12" data-comment-id="445">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="445"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#12" class="light">646 days</a>
          
        </span>
      </div>

    

    <h1>Chronicling licensing and permission requests</h1>

<p>Inspired by the story of Max Haeussler <span class="citation">[<a href="https://doi.org/10.1038/483134a" class="citation" data-key="10.1038/483134a">1</a>]</span> who <a href="http://text.soe.ucsc.edu/progress.html">publicly documented</a> his permission requests to publishers to text mine their corpora, I will be chronicling our licensing efforts that require contact. We will therefore release summaries and statistics pertaining to three types of requests:</p>

<ul><li>permission requests to resources with licenses that <strong>forbid redistribution or derivatives</strong>. We have begun by posting our requests to <a href="http://thinklab.com/discussion/msigdb-licensing/108" class="citation thread-citation" data-key="10.15363/thinklab.d108">MSigDB</a> and <a href="http://thinklab.com/discussion/lincs-l1000-licensing/110" class="citation thread-citation" data-key="10.15363/thinklab.d110">LINCS L1000</a>.</li><li>requests to post licenses for resources <strong>without licensing information</strong>. Resources for which we could not find license information are <a href="https://github.com/dhimmel/integrate/tree/0e343d8745a98757c86e73f645b41353f52b82b5/licenses">available here</a>. We have already emailed the creators of these resources and will report back with progress.</li><li>for datasets obtained from <strong>publication supplements</strong>, license clarification or permission requests to the journal.</li></ul>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
<li id="note-187" class="inline-note toggle-parent" data-note-id="187">
	<p><a href="/u/dhimmel">Daniel Himmelstein</a>: Yesterday, we <a href="http://thinklab.com/discussion/incomplete-interactome-licensing/111" class="citation thread-citation" data-key="10.15363/thinklab.d111">sent out</a> our first permission request for a publication <span class="citation">[<a href="https://doi.org/10.1126/science.1257601" class="citation" data-key="10.1126/science.1257601">1</a>]</span> supplement.
		
	</p>
	
	<table style="display:none;"><tr><td><a href='/doi/10.1126/science.1257601' class='pub-comments tt-right' title='Article comments'>5</a></td><td><div class='pub-order'>1.</div></td><td><div class='doi-body' data-key='10.1126/science.1257601'><a href='/doi/10.1126/science.1257601' class='pub-title'>Uncovering disease-disease relationships through the incomplete interactome</a><div class='pub-body'>J. Menche, A. Sharma, M. Kitsak, S. D. Ghiassian, M. Vidal, J. Loscalzo, A.-L. Barabasi (2015) <i>Science</i>. doi:<a class='pub-doi' href='https://doi.org/10.1126/science.1257601'>10.1126/science.1257601</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d111' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>2.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d111'><a href='/discussion/incomplete-interactome-licensing/111' class='pub-title'>Incomplete Interactome licensing</a><div class='pub-body'>Daniel Himmelstein (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d111'>10.15363/thinklab.d111</a></div></div></td></tr></table>
</li>

				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="13" data-comment-id="462">
  <td class="comment-profile-avatar">
		<a href="/u/cchung"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/s/img/p_default_medium.png' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="462"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/cchung">Caty Chung</a>
				
				<span class="light">&nbsp;<a href="#13" class="light">638 days</a>
          
        </span>
      </div>

    

    <p>Can you add into the <a href="https://github.com/dhimmel/integrate/blob/0e343d8745a98757c86e73f645b41353f52b82b5/licenses/README.md">table</a> the corporate or institutional affiliation of the project and the funding agency to each of the data sources?</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
<li id="note-191" class="inline-note toggle-parent" data-note-id="191">
	<p><a href="/u/dhimmel">Daniel Himmelstein</a>: I've <a href="https://github.com/dhimmel/integrate/blob/7ef64533f0822fb5728ab4c1d88dc39f3345dcc8/licenses/README.md">added institutional affiliations</a>. I did not specify affiliations for community driven projects or multi-affiliated projects. In general, is funding information available? Where can I find it?
		
	</p>
	
	<table style="display:none;"></table>
</li>

<li id="note-192" class="inline-note toggle-parent" data-note-id="192">
	<p><a href="/u/cchung">Caty Chung</a>: Funding is available on the project site or the citation.  For example <a href="http://disease-ontology.org/about/">DO</a>, their last <a href="http://nar.oxfordjournals.org/content/early/2014/10/27/nar.gku1011.full.pdf?keytype=ref&amp;ijkey=Ul8AlMyerFSf0rP">publication</a>, the funding are from several grants from NIH, EMBL and the department of energy. </p>

<p>When sources are looking into licensing or terms of use, do they bind to the affiliation, funding or make the best educated guess?
		
	</p>
	
	<table style="display:none;"></table>
</li>

<li id="note-214" class="inline-note toggle-parent" data-note-id="214">
	<p><a href="/u/dhimmel">Daniel Himmelstein</a>: I <a href="https://github.com/dhimmel/integrate/blob/145810796f0729d1ed5255bcb83c658490a198f9/licenses/README.md">updated the table</a> to include funding information. Without seeing the specific contractual arrangements between funders and Universities and between Universities and their researchers, it's difficult to know whether there are any binding obligations regarding data licensing.
		
	</p>
	
	<table style="display:none;"></table>
</li>

				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="14" data-comment-id="1176">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="1176"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#14" class="light">479 days</a>
          
        </span>
      </div>

    

    <h1>Data licensing and compliance report</h1>

<p>As a refresher, we released an initial version of our network built from publicly-availabe resources. I had assumed that as long as a resource was public, we could use it for our research. In addition, we're committed to open science — releasing our network and intermediate data, both for reproducibility and to allow others to build off of our research. However, as <a href="/u/larsjuhljensen" class="username">@larsjuhljensen</a> <a href="http://thinklab.com/discussion/one-network-to-rule-them-all/102#2" class="citation thread-citation" data-key="10.15363/thinklab.d102">pointed out</a>, legal issues arise when using public data that isn't specifically licensed to permit reuse.</p>

<p>It has now been 212 days since Lar's alert and 199 days since I started this discussion seeking expert advice. Here I'll report on the strategy we chose. Our goals were: to bring us into compliance with copyright law and license agreements; to respect the intent of resource creators; to preserve our sunk time investment; and to retain the scientific value of our network. Unfortunately, no one solution satisfied every objective. We were left to choose between several imperfect ways forward.</p>

<h2>Compliance efforts</h2>

<p>First, I compiled the <a href="https://github.com/dhimmel/integrate/blob/145810796f0729d1ed5255bcb83c658490a198f9/licenses/README.md">licenses for all of the resources</a> we included in our network. Of the 28 resources we integrated, only 12 had licenses that <a href="http://opendefinition.org/licenses/0">met</a> the <a href="http://opendefinition.org/od/2.1/en/" title="Open Definition 2.1">criteria for open knowledge</a>. As a result, our project would not be a possibility under a paradigm of absolute compliance.</p>

<p>Resources fell into four categories regarding their licensing:</p>

<ol><li>Resources that are in the <strong>public domain</strong>.</li><li>Resources with a license that <strong>allows</strong> use, redistribution, and modification.</li><li>Resources with a license that <strong>forbids</strong> use, redistribution, or modification.</li><li>Resources that <strong>do not have</strong> a license.</li></ol>

<p>While I retrospectively assigned these categories while writing this post, the approach we pursued for a given resource aligned with its category. We approached category 1 &amp; 2 resources by specifying their license wherever we use, redistribute, or modify them. We approached category 3 &amp; 4 resources by requesting permission from their creators or owners. I attempted attribution for all resources, regardless of category, to maintain data provenance.</p>

<h3>Category 1 &amp; 2 resources</h3>

<p>There were 4 <strong>category 1</strong> resources — Entrez Gene, MEDLINE, LabeledIn, and MeSH — all due to US federal Government creations <a href="https://en.wikipedia.org/w/index.php?title=Copyright_status_of_work_by_the_U.S._government&amp;oldid=708981516">not being entitled</a> to copyright protection. These resources were easy to integrate: I could proceed without restriction and released derivative works under CC0.</p>

<p>There were 14 <strong>category 2</strong> resources. If the resource uses a standard license, such as a license by <a href="https://creativecommons.org/licenses/">Creative Commons</a> or <a href="http://opendatacommons.org/licenses/">Open Data Commons</a>, I used the same license including version for redistribution and derivative works. Examples include Disease Ontology, DISEASES, Gene Ontology, TISSUES, Uberon, WikiPathways, BindingDB, DisGeNET. If the resource used a custom license, then I applied a Creative Commons license that abided by the custom stipulations. For example, <a href="https://creativecommons.org/licenses/by/4.0/">CC BY 4.0</a> for custom licenses that require attribution — GWAS Catalog &amp; LINCS L1000 — and <a href="https://creativecommons.org/licenses/by-nc/4.0/">CC BY-NC 4.0</a> for custom licenses that forbid commercial use or specify academic use only — DrugBank.</p>

<p>I embedded licensing into the network as node/relationship properties. Therefore, users can filter to retain only specific licenses when querying or parsing our network. Prior to the network stage when data for each resource still resides in separate repositories, I specified licensing via a <code>LICENSE.md</code> file or a section in the <code>README.md</code> file.</p>

<h3>Category 3 &amp; 4 resources</h3>

<p>Originally I identified 3 <strong>category 3</strong> resources — MSigDB, Incomplete Interactome, LINCS L1000. I chronicled these permission requests on <em>Thinklab</em>. Through our permission requests, we learned that the <a href="https://doi.org/10.15363/thinklab.d111">Incomplete Interactome was</a> actually category 4 and <a href="https://doi.org/10.15363/thinklab.d110">LINCS L1000 was</a> actually category 2. Our permission request to MSigDB <a href="http://doi.org/10.15363/thinklab.d108">is ongoing</a>.</p>

<p>There were 9 <strong>category 4</strong> resources — ADEPTUS, Bgee, DOAF, ehrlink, ERC, hetio-dag, Incomplete Interactome, Human Interactome Database, STARGEO. Since I am the creator of hetio-dag and our STARGEO analysis, these resources did not require any action. For the remaining resources, I sent permission requests.</p>

<p>For category 3 &amp; 4 resources, I opted to continue including the resource in our network regardless of whether we affirmatively received permission. I deemed these resources too critical from a scientific perspective to justify their removal. Several factors shaped my decision: many scientists who post their data assume it will automatically be reusable; the resources were publicly funded with the intent to be used for science; copyright may not apply if our network is fair use or the underlying data is factual; and reuse of scientific data despite all rights reserved is prevalent throughout academia.</p>

<p>There are several unpleasant consequences to my decision to include category 3 &amp; 4 works. First, I risk the legal consequences of infringement. Second, we could have to purge content from our network if a data creator/owner requests that we discontinue use of their resource. Third, anyone who wants to use or build off of our network will have to revisit the same issues we're facing here.</p>

<h4>Permission requests by outcome</h4>

<p>For category 3 &amp; 4 resources, I requested permission to use the resource for our project. I've organized my requests into four outcomes:</p>

<ul><li>EXST — We received a response referring us to an existing license. In the four instances, we had overlooked the license because it was difficult to find or unclear whether it applied.</li><li>PERM — We received a response granting us permission to use the resource. In both cases, the authors granted their permission but acknowledged that they may not be the rights holder.</li><li>INC — We received an inconclusive response. In all three cases, the authors indicated they would take licensing actions which have yet to happen.</li><li>NORESP — No response.</li></ul>

<p>Each resource for which we requested permissions is below. Days indicates the time till first response. When present, public documentation of our request is linked to in Contact Method. The table is sorted by outcome and then by days.</p>

<table class="table markdown-table"><thead><tr><th>Resource</th><th>Outcome</th><th>Days</th><th>Contact Method</th></tr></thead><tbody><tr><td><a href="http://uberon.org">Uberon</a></td><td>EXST</td><td>0</td><td><a href="https://github.com/obophenotype/uberon/issues/1139">GitHub Issue</a></td></tr><tr><td><a href="http://www.ncbi.nlm.nih.gov/gene">Entrez Gene</a></td><td>EXST</td><td>2</td><td>helpdesk</td></tr><tr><td><a href="http://www.lincscloud.org/l1000/">LINCS L1000</a></td><td>EXST</td><td>16</td><td><a href="https://doi.org/10.15363/thinklab.d110">email</a></td></tr><tr><td><a href="https://www.ebi.ac.uk/gwas/">GWAS Catalog</a></td><td>EXST</td><td>19</td><td>email</td></tr><tr><td><a href="http://science.sciencemag.org/content/suppl/2015/02/18/347.6224.1257601.DC1">Incomplete Interactome</a></td><td>PERM</td><td>0</td><td><a href="https://doi.org/10.15363/thinklab.d111">email</a></td></tr><tr><td><a href="http://csb.pitt.edu/erc_analysis/Methods.php">Evolutionary Rate Covariation</a></td><td>PERM</td><td>16</td><td>email</td></tr><tr><td><a href="http://doa.nubic.northwestern.edu/pages/search.php">DOAF</a></td><td>INC</td><td>2</td><td>email</td></tr><tr><td><a href="http://bgee.org">Bgee</a></td><td>INC</td><td>9</td><td>email/<a href="https://doi.org/10.15363/thinklab.d82#15">note</a></td></tr><tr><td><a href="http://software.broadinstitute.org/gsea/msigdb">MSigDB</a></td><td>INC</td><td>129</td><td><a href="https://doi.org/10.15363/thinklab.d108">email</a></td></tr><tr><td><a href="http://interactome.dfci.harvard.edu/H_sapiens/index.php?page=download">Human Interactome Database</a></td><td>NORESP</td><td>189+</td><td>email</td></tr><tr><td><a href="http://acgt.cs.tau.ac.il/adeptus/">ADEPTUS</a></td><td>NORESP</td><td>198+</td><td>email</td></tr></tbody></table>

<h2>Conclusion</h2>

<p>We've gone to great lengths and invested substantial time in complying with data copyright and licensing. However, under a strict interpretation, our project may infringe upon the rights of publicly-funded scholarly resources.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
<li id="note-216" class="inline-note toggle-parent" data-note-id="216">
	<p><a href="/u/larsjuhljensen">Lars Juhl Jensen</a>: If I were you, I would just delete Human Interactome Database and ADEPTUS from the network, unless they are crucial. Considering that over half a year has gone by without them responding, it seems likely that you will get permission, so I would remove them to not have legally questionable data in my resource.
		
	</p>
	
	<table style="display:none;"></table>
</li>

<li id="note-218" class="inline-note toggle-parent" data-note-id="218">
	<p><a href="/u/dhimmel">Daniel Himmelstein</a>: ADEPTUS is no longer included (indicated by ‡ <a href="https://github.com/dhimmel/integrate/blob/145810796f0729d1ed5255bcb83c658490a198f9/licenses/README.md">here</a>). The Human Interactome Database (<a href="http://interactome.dfci.harvard.edu/H_sapiens/index.php?page=download">HID</a>) is important because it's the only <a href="http://thinklab.com/discussion/creating-a-catalog-of-protein-interactions/85#9" class="citation thread-citation" data-key="10.15363/thinklab.d85">resource we include</a> of systematic (unbiased by knowledge) interactions. The HID — having received many millions in NIH funding (see <a href="http://grantome.com/grant/NIH/U01-HG001715-16">a</a>, <a href="http://grantome.com/grant/NIH/U41-HG001715-17">b</a>, &amp; <a href="http://grantome.com/grant/NIH/R01-HG001715-13">c</a>) to create its dataset — provides a unique resource that is commonly reused despite it's lack of a license. I will follow up on my initial emails to the HID.
		
	</p>
	
	<table style="display:none;"><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d85' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>1.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d85'><a href='/discussion/creating-a-catalog-of-protein-interactions/85' class='pub-title'>Creating a catalog of protein interactions</a><div class='pub-body'>Daniel Himmelstein, Dexter Hadley, Alexey Strokach (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d85'>10.15363/thinklab.d85</a></div></div></td></tr></table>
</li>

<li id="note-221" class="inline-note toggle-parent" data-note-id="221">
	<p><a href="/u/dhimmel">Daniel Himmelstein</a>: We <a href="http://thinklab.com/discussion/the-adeptus-resource-for-expression-signatures-of-disease/101#2" class="citation thread-citation" data-key="10.15363/thinklab.d101">received permission</a> to use ADEPTUS. The ADEPTUS row in the permission request table now becomes PERM after 213 days.
		
	</p>
	
	<table style="display:none;"><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d101' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>1.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d101'><a href='/discussion/the-adeptus-resource-for-expression-signatures-of-disease/101' class='pub-title'>The ADEPTUS resource for expression signatures of disease</a><div class='pub-body'>Daniel Himmelstein (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d101'>10.15363/thinklab.d101</a></div></div></td></tr></table>
</li>

<li id="note-225" class="inline-note toggle-parent" data-note-id="225">
	<p><a href="/u/dhimmel">Daniel Himmelstein</a>: MSigDB — the only remaining category 3 resource — has now <a href="http://thinklab.com/discussion/msigdb-licensing/108#3" class="citation thread-citation" data-key="10.15363/thinklab.d108">been removed</a>.
		
	</p>
	
	<table style="display:none;"><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d108' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>1.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d108'><a href='/discussion/msigdb-licensing/108' class='pub-title'>MSigDB licensing</a><div class='pub-body'>Daniel Himmelstein (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d108'>10.15363/thinklab.d108</a></div></div></td></tr></table>
</li>

				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="15" data-comment-id="1375">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="1375"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#15" class="light">236 days</a>
          
        </span>
      </div>

    

    <h1>Additional references</h1>

<p>I'll try to keep the following list up to date with webpages or papers I come across that provide relevant data licensing information.</p>

<ul><li><strong>Legal confusion threatens to slow data science</strong> <span class="citation">[<a href="https://doi.org/10.1038/536016a" class="citation" data-key="10.1038/536016a">1</a>]</span> — which discusses our licensing struggles in creating Hetionet</li><li><strong>Sharing Research Data and Intellectual Property Law: A Primer</strong> <span class="citation">[<a href="https://doi.org/10.1371/journal.pbio.1002235" class="citation" data-key="10.1371/journal.pbio.1002235">2</a>]</span></li><li><strong>Legal Interoperability of Research Data: Principles and Implementation Guidelines</strong> <span class="citation">[<a href="https://doi.org/10.5281/zenodo.162241" class="citation" data-key="10.5281/zenodo.162241">3</a>]</span></li><li><strong>Who “owns” your data?</strong> by <a href="/u/katiefortney" class="username">@katiefortney</a> <a href="http://osc.universityofcalifornia.edu/2016/09/who-owns-your-data/">published on</a> the University of California, Office of Scholarly Communication Blog.</li><li><strong>Create guidelines for OBO maintainers who want to be included in Wikidata</strong> — <a href="https://github.com/OBOFoundry/OBOFoundry.github.io/issues/285">Issue #285 on the OBOFoundry GitHub</a> discussing licensing options for biomedical ontologies.</li></ul>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="16" data-comment-id="1490">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="1490"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#16" class="light">155 days</a>
          
        </span>
      </div>

    

    <h1>Final resource counts</h1>

<p>Since stats help provide context, we often mention how many resources Hetionet integrates. Our <a href="https://github.com/dhimmel/integrate/blob/725f4e4b4a737cfb15abe55ef36386c23e1c4f1f/licenses/README.md">licensing table</a> lists 31 resources, but 2 were removed. Hence, we claim Hetionet v1.0 integrates 29 resources. This is the number of resources that directly contributed data that was encoded as nodes or edges in the hetnet.</p>

<p><strong>Caveats:</strong> The number 29 underestimates the extent of integration required for a project such as Rephetio. First, we used additional resources, such as <a href="https://thinklab.com/discussion/unifying-drug-vocabularies/40#4" class="citation thread-citation" data-key="10.15363/thinklab.d40">UniChem</a>, to help standardize and integrate these 29 resources. Additionally, several resources were themselves compilations of other resources, such as <a href="https://thinklab.com/discussion/integrating-drug-target-information-from-bindingdb/53" class="citation thread-citation" data-key="10.15363/thinklab.d53">BindingDB</a> and the <a href="https://thinklab.com/discussion/creating-a-catalog-of-protein-interactions/85#9" class="citation thread-citation" data-key="10.15363/thinklab.d85">resources for protein-interactions</a>. Finally, we rely on several other databases to interpret our findings, such as <a href="https://thinklab.com/discussion/prediction-in-epilepsy/224#10" class="citation thread-citation" data-key="10.15363/thinklab.d224">ATC Codes</a> and <a href="https://thinklab.com/discussion/using-entrez-gene-as-our-gene-vocabulary/34#13" class="citation thread-citation" data-key="10.15363/thinklab.d34">HGNC Gene Families</a>.</p>

<p>Nonetheless here are the 31 resources divided by their copyright and licensing situation. Note that this stratification requires some subjectivity. In other words, I used my best judgement to help simplify complex legal considerations into 6 categories. The categories are slightly different <a href="#14">than above</a>. Furthermore, this post reflects the <a href="https://thinklab.com/discussion/msigdb-licensing/108#3" class="citation thread-citation" data-key="10.15363/thinklab.d108">removal of MSigDB</a> and the <a href="https://thinklab.com/discussion/adding-pathway-resources-to-your-network/72#11" class="citation thread-citation" data-key="10.15363/thinklab.d72">corresponding addition</a> of two pathway resources (Reactome &amp; Pathway Interaction Database).</p>

<h2>5 public domain resources</h2>

<p>Five resources were created by the United States Government. Hence I consider them not subject to copyright and part of the public domain. Nodes and edges from these resources are CC0 licensed in Hetionet v1.0.</p>

<ol><li>Entrez Gene</li><li>LabeledIn</li><li>MEDLINE</li><li>MeSH</li><li>Pathway Interaction Database</li></ol>

<p><strong>Caveats:</strong> The public domain status of these resources is complicated. None of them adopted a CC0 license to unambiguously place them in the public domain, including outside of the United States. Furthermore, they often come with custom legal statements and terms of use. See for example, the MeSH <a href="https://github.com/dhimmel/integrate/blob/725f4e4b4a737cfb15abe55ef36386c23e1c4f1f/licenses/custom/MeSH.md">Memorandum of Understanding</a>. These custom terms make one question whether these are actually public domain resources. It's a mess. <a href="/u/andrewsu" class="username">@andrewsu</a> is a leading expert on this mess and reform efforts.</p>

<h2>12 openly licensed resources</h2>

<p>Twelve resources had licenses that met the <a href="http://opendefinition.org/od/2.1/en/">Open Definition version 2.1</a>, which is summarized as:</p>

<blockquote><p>Knowledge is open if anyone is free to access, use, modify, and share it — subject, at most, to measures that preserve provenance and openness.</p></blockquote>

<p>Those resources are:</p>

<ol><li>Disease Ontology</li><li>DISEASES</li><li>DrugCentral</li><li>Gene Ontology</li><li>GWAS Catalog</li><li>Reactome</li><li>LINCS L1000</li><li>TISSUES</li><li>Uberon</li><li>WikiPathways</li><li>BindingDB</li><li>DisGeNET</li></ol>

<p><strong>Caveats:</strong> Not all of these resources used a standard license that <a href="http://opendefinition.org/licenses/">officially conforms</a> with the Open Definition. Therefore, I used my best judgement whether custom license terms were compatible with open licensing. For the most part, nodes and edges from these resources are openly licensed in Hetionet v1.0.</p>

<h2>4 resources that allow non-commercial reuse</h2>

<p>Four resources had licenses that allowed non-commercial reuse only. Nodes and edges from these resources use the least-restrictive compatible Creative Commons license in Hetionet v1.0.</p>

<ol><li>DrugBank 4.2</li><li>MEDI</li><li>PREDICT</li><li>SIDER 4</li></ol>

<p>Besides DrugBank, these resources did use standard Creative Commons licenses, which while not being open are at least legally straightforward. And DrugBank <a href="https://thinklab.com/discussion/sounding-the-alarm-on-drugbanks-new-license-and-terms-of-use/213#10" class="citation thread-citation" data-key="10.15363/thinklab.d213">switched to a Creative Commons license</a> part of the way through Project Rephetio, based in part on our feedback.</p>

<h2>9 unlicensed resources</h2>

<p>Nine resources did not have a license. For the most part, nodes and edges from these resources don't have a license attribute in Hetionet v1.0.</p>

<ol><li>ADEPTUS (removed)</li><li>Bgee</li><li>DOAF</li><li>ehrlink</li><li>Evolutionary Rate Covariation</li><li>hetio-dag</li><li>Incomplete Interactome</li><li>Human Interactome Database</li><li>STARGEO</li></ol>

<p><strong>Caveats:</strong> In these cases, I believe the researchers generally put the data online for others to use but are unaware of the legal barriers to data reuse. Or in other instances, they would like to openly license their work but are not the data owners or are unsure of the legal considerations of doing so.</p>

<h2>MSigDB explicitly forbids redistribution</h2>

<p>Ultimately, one resource explicitly forbid redistribution.</p>

<ol><li>MSigDB (removed)</li></ol>

<h2>Conclusion</h2>

<p><span class="math">$$5 + 12 + 4 + 9 + 1 - 2 = 29$$</span></p>

<p>Phew!</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

    		
		</table>

    

    &nbsp;<br>
    <a class="btn btn-primary btn-lg" href="/signup?next=/discussion/integrating-resources-with-disparate-licensing-into-an-open-network/107&suppress_message=1" style>Join to Reply</a>


		<table class="references table-td-5" style="display:none;"><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d102' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>1.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d102'><a href='/discussion/one-network-to-rule-them-all/102' class='pub-title'>One network to rule them all</a><div class='pub-body'>Daniel Himmelstein, Lars Juhl Jensen (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d102'>10.15363/thinklab.d102</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d23' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>2.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d23'><a href='/discussion/enabling-reproducibility-and-reuse/23' class='pub-title'>Enabling reproducibility and reuse</a><div class='pub-body'>Jesse Spaulding, Daniel Himmelstein, Casey Greene, Benjamin Good (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d23'>10.15363/thinklab.d23</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d102' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>3.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d102'><a href='/discussion/one-network-to-rule-them-all/102' class='pub-title'>One network to rule them all</a><div class='pub-body'>Daniel Himmelstein, Lars Juhl Jensen (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d102'>10.15363/thinklab.d102</a></div></div></td></tr><tr><td><a href='/doi/10.5281/zenodo.30105' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>4.</div></td><td><div class='doi-body' data-key='10.5281/zenodo.30105'><a href='/doi/10.5281/zenodo.30105' class='pub-title'>Highly connected genes contain more SNPs</a><div class='pub-body'>Daniel Himmelstein, Casey Greene (2015) <i>Zenodo</i>. doi:<a class='pub-doi' href='https://doi.org/10.5281/zenodo.30105'>10.5281/zenodo.30105</a></div></div></td></tr><tr><td><a href='/doi/10.1087/0953151053584984' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>5.</div></td><td><div class='doi-body' data-key='10.1087/0953151053584984'><a href='/doi/10.1087/0953151053584984' class='pub-title'>Who owns scientific data? The impact of intellectual property rights on the scientific publication chain</a><div class='pub-body'>Roger Elliott (2005) <i>Learned Publishing</i>. doi:<a class='pub-doi' href='https://doi.org/10.1087/0953151053584984'>10.1087/0953151053584984</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d97' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>6.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d97'><a href='/discussion/extracting-side-effects-from-sider-4/97' class='pub-title'>Extracting side effects from SIDER 4</a><div class='pub-body'>Daniel Himmelstein (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d97'>10.15363/thinklab.d97</a></div></div></td></tr><tr><td><a href='/doi/10.1038/483134a' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>7.</div></td><td><div class='doi-body' data-key='10.1038/483134a'><a href='/doi/10.1038/483134a' class='pub-title'>Trouble at the text mine</a><div class='pub-body'>Richard Van Noorden (2012) <i>Nature</i>. doi:<a class='pub-doi' href='https://doi.org/10.1038/483134a'>10.1038/483134a</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d108' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>8.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d108'><a href='/discussion/msigdb-licensing/108' class='pub-title'>MSigDB licensing</a><div class='pub-body'>Daniel Himmelstein (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d108'>10.15363/thinklab.d108</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d110' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>9.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d110'><a href='/discussion/lincs-l1000-licensing/110' class='pub-title'>LINCS L1000 licensing</a><div class='pub-body'>Daniel Himmelstein (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d110'>10.15363/thinklab.d110</a></div></div></td></tr><tr><td><a href='/doi/10.1126/science.1257601' class='pub-comments tt-right' title='Article comments'>5</a></td><td><div class='pub-order'>10.</div></td><td><div class='doi-body' data-key='10.1126/science.1257601'><a href='/doi/10.1126/science.1257601' class='pub-title'>Uncovering disease-disease relationships through the incomplete interactome</a><div class='pub-body'>J. Menche, A. Sharma, M. Kitsak, S. D. Ghiassian, M. Vidal, J. Loscalzo, A.-L. Barabasi (2015) <i>Science</i>. doi:<a class='pub-doi' href='https://doi.org/10.1126/science.1257601'>10.1126/science.1257601</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d111' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>11.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d111'><a href='/discussion/incomplete-interactome-licensing/111' class='pub-title'>Incomplete Interactome licensing</a><div class='pub-body'>Daniel Himmelstein (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d111'>10.15363/thinklab.d111</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d102' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>12.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d102'><a href='/discussion/one-network-to-rule-them-all/102' class='pub-title'>One network to rule them all</a><div class='pub-body'>Daniel Himmelstein, Lars Juhl Jensen (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d102'>10.15363/thinklab.d102</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d85' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>13.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d85'><a href='/discussion/creating-a-catalog-of-protein-interactions/85' class='pub-title'>Creating a catalog of protein interactions</a><div class='pub-body'>Daniel Himmelstein, Dexter Hadley, Alexey Strokach (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d85'>10.15363/thinklab.d85</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d101' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>14.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d101'><a href='/discussion/the-adeptus-resource-for-expression-signatures-of-disease/101' class='pub-title'>The ADEPTUS resource for expression signatures of disease</a><div class='pub-body'>Daniel Himmelstein (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d101'>10.15363/thinklab.d101</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d108' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>15.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d108'><a href='/discussion/msigdb-licensing/108' class='pub-title'>MSigDB licensing</a><div class='pub-body'>Daniel Himmelstein (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d108'>10.15363/thinklab.d108</a></div></div></td></tr><tr><td><a href='/doi/10.1038/536016a' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>16.</div></td><td><div class='doi-body' data-key='10.1038/536016a'><a href='/doi/10.1038/536016a' class='pub-title'>Legal confusion threatens to slow data science</a><div class='pub-body'>Simon Oxenham (2016) <i>Nature</i>. doi:<a class='pub-doi' href='https://doi.org/10.1038/536016a'>10.1038/536016a</a></div></div></td></tr><tr><td><a href='/doi/10.1371/journal.pbio.1002235' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>17.</div></td><td><div class='doi-body' data-key='10.1371/journal.pbio.1002235'><a href='/doi/10.1371/journal.pbio.1002235' class='pub-title'>Sharing Research Data and Intellectual Property Law: A Primer</a><div class='pub-body'>Michael W. Carroll (2015) <i>PLOS Biology</i>. doi:<a class='pub-doi' href='https://doi.org/10.1371/journal.pbio.1002235'>10.1371/journal.pbio.1002235</a></div></div></td></tr><tr><td><a href='/doi/10.5281/zenodo.162241' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>18.</div></td><td><div class='doi-body' data-key='10.5281/zenodo.162241'><a href='/doi/10.5281/zenodo.162241' class='pub-title'>Legal Interoperability Of Research Data: Principles And Implementation Guidelines</a><div class='pub-body'>RDA-CODATA Legal Interoperability Interest Group (2016) <i>Zenodo</i>. doi:<a class='pub-doi' href='https://doi.org/10.5281/zenodo.162241'>10.5281/zenodo.162241</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d40' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>19.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d40'><a href='/discussion/unifying-drug-vocabularies/40' class='pub-title'>Unifying drug vocabularies</a><div class='pub-body'>Daniel Himmelstein (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d40'>10.15363/thinklab.d40</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d53' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>20.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d53'><a href='/discussion/integrating-drug-target-information-from-bindingdb/53' class='pub-title'>Integrating drug target information from BindingDB</a><div class='pub-body'>Daniel Himmelstein, Mike Gilson (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d53'>10.15363/thinklab.d53</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d85' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>21.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d85'><a href='/discussion/creating-a-catalog-of-protein-interactions/85' class='pub-title'>Creating a catalog of protein interactions</a><div class='pub-body'>Daniel Himmelstein, Dexter Hadley, Alexey Strokach (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d85'>10.15363/thinklab.d85</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d224' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>22.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d224'><a href='/discussion/prediction-in-epilepsy/224' class='pub-title'>Prediction in epilepsy</a><div class='pub-body'>Pouya Khankhanian, Daniel Himmelstein (2016) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d224'>10.15363/thinklab.d224</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d34' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>23.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d34'><a href='/discussion/using-entrez-gene-as-our-gene-vocabulary/34' class='pub-title'>Using Entrez Gene as our gene vocabulary</a><div class='pub-body'>Daniel Himmelstein, Casey Greene, Alexander Pico (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d34'>10.15363/thinklab.d34</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d108' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>24.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d108'><a href='/discussion/msigdb-licensing/108' class='pub-title'>MSigDB licensing</a><div class='pub-body'>Daniel Himmelstein (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d108'>10.15363/thinklab.d108</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d72' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>25.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d72'><a href='/discussion/adding-pathway-resources-to-your-network/72' class='pub-title'>Adding pathway resources to your network</a><div class='pub-body'>Alexander Pico, Daniel Himmelstein (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d72'>10.15363/thinklab.d72</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d213' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>26.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d213'><a href='/discussion/sounding-the-alarm-on-drugbanks-new-license-and-terms-of-use/213' class='pub-title'>Sounding the alarm on DrugBank’s new license and terms of use</a><div class='pub-body'>Daniel Himmelstein, Katie Fortney, Craig Knox, Christopher Southan (2016) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d213'>10.15363/thinklab.d213</a></div></div></td></tr></table>

    


	</td>
	<td class="thread-sidebar">


			

      <a class="btn btn-default follow-link btn-follow btn-lg is-following tt-top full-width" title="Follow discussion" data-id="107" data-ctype="thread" data-is-following="false">&nbsp;</a>

      
        <div class="status-box text-center"><span style="">Status:</span> 
  In Progress

</div>
      

      
        

          <div class="details">
            
            Labels
          </div>

          
            <a href='/p/rephetio/discussion?q=[licensing]' class='pl pl-sidebar ' style='background-color:#6b6b6b'><span class='glyphicon glyphicon-tag'></span>&nbsp; licensing</a>
          
        
      


				

					<div class="details">Views</div>
					<span style="font-size:20px;">517</span>

        

				
					<div class="details">Topics</div>
					<div style="margin-bottom:15px;">
		            <a href="/topics/open-science" class="tag">Open science</a><a href="/topics/open-data" class="tag">Open Data</a><a href="/topics/license" class="tag">License</a><a href="/topics/permission" class="tag">Permission</a><a href="/topics/licensing" class="tag">Licensing</a><a href="/topics/copyright" class="tag">Copyright</a><a href="/topics/creative-commons" class="tag">Creative Commons</a><a href="/topics/intellectual-property" class="tag">Intellectual Property</a>
					</div>
				


		
			<div class="details">Referenced by</div>
			<div style="font-size:12px;"><ul style="padding-left:16px !important;">
			
				<li>
					
					<a href='/discussion/tracking-project-reuse-citation-and-publicity/113' class='thread-title'>Tracking project reuse, citation, and publicity</a>
				</li>
			
				<li>
					
					<a href='/discussion/one-network-to-rule-them-all/102' class='thread-title'>One network to rule them all</a>
				</li>
			
				<li>
					
					<a href='/discussion/brainstorming-future-directions-for-hetionet/227' class='thread-title'>Brainstorming future directions for Hetionet</a>
				</li>
			
				<li>
					
					Research report: <a href='/p/rephetio/report'>Rephetio: Repurposing drugs on a hetnet</a>
				</li>
			
				<li>
					
					<a href='/discussion/sounding-the-alarm-on-drugbanks-new-license-and-terms-of-use/213' class='thread-title'>Sounding the alarm on DrugBank’s new license and terms of use</a>
				</li>
			
				<li>
					
					<a href='/discussion/creating-a-catalog-of-protein-interactions/85' class='thread-title'>Creating a catalog of protein interactions</a>
				</li>
			
				<li>
					
					<a href='/discussion/msigdb-licensing/108' class='thread-title'>MSigDB licensing</a>
				</li>
			
				<li>
					
					<a href='/discussion/the-adeptus-resource-for-expression-signatures-of-disease/101' class='thread-title'>The ADEPTUS resource for expression signatures of disease</a>
				</li>
			
				<li>
					
					Project: <a href='/p/rephetio'>Rephetio: Repurposing drugs on a hetnet</a>
				</li>
			
				<li>
					
					<a href='/discussion/incomplete-interactome-licensing/111' class='thread-title'>Incomplete Interactome licensing</a>
				</li>
			
				<li>
					
					<a href='/discussion/lincs-l1000-licensing/110' class='thread-title'>LINCS L1000 licensing</a>
				</li>
			
				<li>
					
					<a href='/discussion/processing-gtex-for-tissue-specific-gene-expression/82' class='thread-title'>Processing GTEx for tissue-specific gene expression</a>
				</li>
			
				</ul>
			</div>
		

		
			<div class="details">Cite this as</div>
			<span style="font-size:12px;">
				Daniel Himmelstein, Lars Juhl Jensen, MacKenzie Smith, Katie Fortney, Caty Chung (2015) Integrating resources with disparate licensing into an open network. <i>Thinklab.</i> doi:<a href='https://doi.org/10.15363/thinklab.d107'>10.15363/thinklab.d107</a>
			</span>
		

		<div class="details">License</div>
		<p><a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0;width:88px;height:31px;" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a></p>

		<div class="details">Share</div>
		
			
				<a class="share-button fb" onclick="javascript:void(window.open('https://www.facebook.com/sharer/sharer.php?u=https%3A//thinklab.com/discussion/integrating-resources-with-disparate-licensing-into-an-open-network/107','','width=600,height=460,menubar=no,status=no'));" href="#"></a>
<a class="share-button twitter" onclick="javascript:void(window.open('https://twitter.com/share?text=Integrating%20resources%20with%20disparate%20licensing%20into%20an%20open%20network&url=https%3A//thinklab.com/discussion/integrating-resources-with-disparate-licensing-into-an-open-network/107','','width=600,height=300,menubar=no,status=no'));" href="#"></a>
<a class="share-button google" onclick="javascript:void(window.open('https://plus.google.com/share?url=https%3A//thinklab.com/discussion/integrating-resources-with-disparate-licensing-into-an-open-network/107','','width=510,height=460,menubar=no,status=no'));" href="#"></a>

			
		

	</td>
</tr></table>


		</div>


		
	    	<div class="footer">
<div style="background:#dedede;">
	<div class="main-centered-div text-center" style="padding:20px;">
		<h4 style="margin-bottom:0px;font-weight:bold;color:gray;">Feedback? Questions? Email us: <a href="/cdn-cgi/l/email-protection#08606d646467487c6061666364696a266b6765"><span class="__cf_email__" data-cfemail="ee868b828281ae9a86878085828f8cc08d8183">[email&#160;protected]</span><script data-cfhash='f9e31' type="text/javascript">/* <![CDATA[ */!function(t,e,r,n,c,a,p){try{t=document.currentScript||function(){for(t=document.getElementsByTagName('script'),e=t.length;e--;)if(t[e].getAttribute('data-cfhash'))return t[e]}();if(t&&(c=t.previousSibling)){p=t.parentNode;if(a=c.getAttribute('data-cfemail')){for(e='',r='0x'+a.substr(0,2)|0,n=2;a.length-n;n+=2)e+='%'+('0'+('0x'+a.substr(n,2)^r).toString(16)).slice(-2);p.replaceChild(document.createTextNode(decodeURIComponent(e)),c)}p.removeChild(t)}}catch(u){}}()/* ]]> */</script></a></h4>
	</div>
</div>
<div class="main-footer">
	<div class="main-centered-div">

<table class="full-width"><tr>
	<td class="footer-nav">

        <div class="foot-head">Explore</div>
        <a href="/proposals">Proposals</a><br>
        <a href="/projects">Projects</a><br>
        <a href="/publications">Article discussion</a><br>
        <a href="/discussion">All discussion</a><br>
        <a href="/topics">Topics</a><br>
        <!-- <a href="/funders">Funders</a><br> -->
        <a href="/funding">Funding opportunities</a><br>

	</td>
	<td class="footer-nav">

        <!-- <div class="foot-head">Help &amp; feedback</div>	 -->
        <!-- <div class="foot-head">Intro</div>
        <a href="/benefits/for-science">Why</a><br>
        <a href="/how-it-works">How it works</a><br>

<br> -->

        <div class="foot-head">Community</div>
        <a href="/leaderboard">Leaderboard</a><br>
        <a href="/p/meta">Thinklab Meta</a><br>
				<a href="/help/faq">Help</a><br>
				<!-- <a href="/new-project">Start a proposal</a><br> -->
        <!-- <a href="/benefits/proposal">Benefits</a><br> -->
        <!-- <a href="/funding">Get funded</a><br> -->
        <!-- <a href="/participate">Become a contributor</a><br> -->
        <!-- <a href="/for-funders">For science funders</a><br> -->
        <!-- <a href="/for-institutions">For institutions</a><br> -->

	</td>
	<td class="footer-nav">



        <div class="foot-head">About</div>
        <a href="/about">Overview</a><br>
        <a href="/benefits">Benefits</a><br>
        <a href="/how-it-works">How it works</a><br>
        <a href="/about/story">Our story</a><br>
        <a href="/blog">Our blog</a><br>
        <a href="/about/jobs">Join the team</a><br>
        <br>

        <!-- <div class="foot-head">Help</div>
        <a href="/help">Help</a><br>
		 -->


	</td>
	<td class="footer-nav">
		<a href="http://twitter.com/thinklab" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/twitter_com.png" width="40"/></a>
		<a href="https://www.facebook.com/pages/Thinklab/310895039088185" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/facebook_com.png" width="40"/></a>
		<a href="https://plus.google.com/115578983466800753402/" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/plus_google_com.png" width="40"/></a>
		<a href="https://www.linkedin.com/company/thinklab-co" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/linkedin_com.png" width="40"/></a>
		<br><br>
		<span class="light" style='font-size:12px;'>&copy; 2015 Thinklab</span>

	</td>

</tr></table>

</div>
</div>
</div>

		

	</div>


  <script></script>
  <script>
  var MEDIA_URL = 'https://think-lab.s3.amazonaws.com/m/';
  var STATIC_URL = 'https://think-lab.s3.amazonaws.com/s/';
  var page_is_authenticated = false;
  var page_page = 'discussion';
  var page_profile_id = undefined;
</script>



	<!-- Note different script type. MUST come before MathJax.js -->
	<script type="text/x-mathjax-config">
	    MathJax.Hub.Config({
	        "HTML-CSS": { preferredFont: "TeX", availableFonts: ["STIX","TeX"],
	                        scale: 110, linebreaks: { automatic:true },
	                        EqnChunk: (MathJax.Hub.Browser.isMobile ? 10 : 50) },
	        tex2jax: { inlineMath: [ ["$$", "$$"], ["\\(","\\)"] ],
	                    displayMath: [ ["$$$","$$$"], ["\\[", "\\]"] ],
	                    processEscapes: true,}
	            });

	</script>
    <script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>





  <script src="https://code.jquery.com/jquery-1.10.2.min.js"></script>
  <script src="https://code.jquery.com/ui/1.11.4/jquery-ui.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" integrity="sha384-0mSbJDEHialfmuBBQP6A4Qrprq5OVfW37PRR3j5ELqxss1yVqOtnepnHVP9aJ7xS" crossorigin="anonymous"></script>
  <script src="https://think-lab.s3.amazonaws.com/s/dist/main.min.js"></script>
  



  
	
  
  
  
<script>
$(function() {
  recordHit('thread', 107);
});
</script>



<script>/* <![CDATA[ */(function(d,s,a,i,j,r,l,m,t){try{l=d.getElementsByTagName('a');t=d.createElement('textarea');for(i=0;l.length-i;i++){try{a=l[i].href;s=a.indexOf('/cdn-cgi/l/email-protection');m=a.length;if(a&&s>-1&&m>28){j=28+s;s='';if(j<m){r='0x'+a.substr(j,2)|0;for(j+=2;j<m&&a.charAt(j)!='X';j+=2)s+='%'+('0'+('0x'+a.substr(j,2)^r).toString(16)).slice(-2);j++;s=decodeURIComponent(s)+a.substr(j,m-j)}t.innerHTML=s.replace(/</g,'&lt;').replace(/\>/g,'&gt;');l[i].href='mailto:'+t.value}}catch(e){}}}catch(e){}})(document);/* ]]> */</script></body>
</html>
