<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8" /><script type="text/javascript">window.NREUM||(NREUM={}),__nr_require=function(e,n,t){function r(t){if(!n[t]){var o=n[t]={exports:{}};e[t][0].call(o.exports,function(n){var o=e[t][1][n];return r(o||n)},o,o.exports)}return n[t].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<t.length;o++)r(t[o]);return r}({1:[function(e,n,t){function r(){}function o(e,n,t){return function(){return i(e,[c.now()].concat(u(arguments)),n?null:this,t),n?void 0:this}}var i=e("handle"),a=e(2),u=e(3),f=e("ee").get("tracer"),c=e("loader"),s=NREUM;"undefined"==typeof window.newrelic&&(newrelic=s);var p=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],d="api-",l=d+"ixn-";a(p,function(e,n){s[n]=o(d+n,!0,"api")}),s.addPageAction=o(d+"addPageAction",!0),s.setCurrentRouteName=o(d+"routeName",!0),n.exports=newrelic,s.interaction=function(){return(new r).get()};var m=r.prototype={createTracer:function(e,n){var t={},r=this,o="function"==typeof n;return i(l+"tracer",[c.now(),e,t],r),function(){if(f.emit((o?"":"no-")+"fn-start",[c.now(),r,o],t),o)try{return n.apply(this,arguments)}finally{f.emit("fn-end",[c.now()],t)}}}};a("setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(e,n){m[n]=o(l+n)}),newrelic.noticeError=function(e){"string"==typeof e&&(e=new Error(e)),i("err",[e,c.now()])}},{}],2:[function(e,n,t){function r(e,n){var t=[],r="",i=0;for(r in e)o.call(e,r)&&(t[i]=n(r,e[r]),i+=1);return t}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],3:[function(e,n,t){function r(e,n,t){n||(n=0),"undefined"==typeof t&&(t=e?e.length:0);for(var r=-1,o=t-n||0,i=Array(o<0?0:o);++r<o;)i[r]=e[n+r];return i}n.exports=r},{}],4:[function(e,n,t){n.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(e,n,t){function r(){}function o(e){function n(e){return e&&e instanceof r?e:e?f(e,u,i):i()}function t(t,r,o,i){if(!d.aborted||i){e&&e(t,r,o);for(var a=n(o),u=m(t),f=u.length,c=0;c<f;c++)u[c].apply(a,r);var p=s[y[t]];return p&&p.push([b,t,r,a]),a}}function l(e,n){v[e]=m(e).concat(n)}function m(e){return v[e]||[]}function w(e){return p[e]=p[e]||o(t)}function g(e,n){c(e,function(e,t){n=n||"feature",y[t]=n,n in s||(s[n]=[])})}var v={},y={},b={on:l,emit:t,get:w,listeners:m,context:n,buffer:g,abort:a,aborted:!1};return b}function i(){return new r}function a(){(s.api||s.feature)&&(d.aborted=!0,s=d.backlog={})}var u="nr@context",f=e("gos"),c=e(2),s={},p={},d=n.exports=o();d.backlog=s},{}],gos:[function(e,n,t){function r(e,n,t){if(o.call(e,n))return e[n];var r=t();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(e,n,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return e[n]=r,r}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],handle:[function(e,n,t){function r(e,n,t,r){o.buffer([e],r),o.emit(e,n,t)}var o=e("ee").get("handle");n.exports=r,r.ee=o},{}],id:[function(e,n,t){function r(e){var n=typeof e;return!e||"object"!==n&&"function"!==n?-1:e===window?0:a(e,i,function(){return o++})}var o=1,i="nr@id",a=e("gos");n.exports=r},{}],loader:[function(e,n,t){function r(){if(!x++){var e=h.info=NREUM.info,n=d.getElementsByTagName("script")[0];if(setTimeout(s.abort,3e4),!(e&&e.licenseKey&&e.applicationID&&n))return s.abort();c(y,function(n,t){e[n]||(e[n]=t)}),f("mark",["onload",a()+h.offset],null,"api");var t=d.createElement("script");t.src="https://"+e.agent,n.parentNode.insertBefore(t,n)}}function o(){"complete"===d.readyState&&i()}function i(){f("mark",["domContent",a()+h.offset],null,"api")}function a(){return E.exists&&performance.now?Math.round(performance.now()):(u=Math.max((new Date).getTime(),u))-h.offset}var u=(new Date).getTime(),f=e("handle"),c=e(2),s=e("ee"),p=window,d=p.document,l="addEventListener",m="attachEvent",w=p.XMLHttpRequest,g=w&&w.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:w,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var v=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-1044.min.js"},b=w&&g&&g[l]&&!/CriOS/.test(navigator.userAgent),h=n.exports={offset:u,now:a,origin:v,features:{},xhrWrappable:b};e(1),d[l]?(d[l]("DOMContentLoaded",i,!1),p[l]("load",r,!1)):(d[m]("onreadystatechange",o),p[m]("onload",r)),f("mark",["firstbyte",u],null,"api");var x=0,E=e(4)},{}]},{},["loader"]);</script><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info={"beacon":"bam.nr-data.net","queueTime":0,"licenseKey":"cd1101aa94","agent":"","transactionName":"bgcDMkBZW0NZBkdcWVdNJxNcW0FZVwscV1dKB08QW11CQxYTWlBBZgEOC19dW0QCEVtHU1gG","applicationID":"6369297","errorBeacon":"bam.nr-data.net","applicationTime":466}</script>
    <title>Suggestions for additional information types? | Thinklab</title>
	  <link rel="shortcut icon" href="https://think-lab.s3.amazonaws.com/s/img/favicon.ico" type="image/x-icon" />
    <meta property="og:image" content="https://think-lab.s3.amazonaws.com/s/img" />
    
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
  <link rel="stylesheet" href="https://code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css" />

<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/highlightjs/github.css" />
<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/css/css.css" />
<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/css/bootstrap-multiselect.css" />
<!-- <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'> -->

    
    
    
    
  
    <meta name="citation_doi" content="10.15363/thinklab.d22">
    <meta name="citation_title" content="Suggestions for additional information types?">
    <meta name="citation_online_date" content="Jan. 15, 2015">
    <meta name="citation_author" content="Daniel Himmelstein, Venkat Malladi, Alessandro Didonna, Alexey Strokach">
    <meta name="citation_journal_title" content="Thinklab">
    <meta name="citation_publisher" content="Thinklab">
  

    


	<!-- GoogleAnalytics -->
	<script>
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
	(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
	m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

	ga('create', 'UA-48550230-1', 'thinklab.org');
	ga('require', 'displayfeatures');
	
	ga('send', 'pageview');

	</script>






</head>
<body >
  <form id="form_action" method="post">
    <input type='hidden' name='csrfmiddlewaretoken' value='JC3hnVSWzcwBOS226QTq2eGyHXj1YAM6' />
</form>

	

	<div class="body-wrapper">

		


<!-- <div style="height:5px;background-image:url('https://think-lab.s3.amazonaws.com/s/img/top-bg.gif');"></div> -->
<div class="gradient">
	<div class="main-centered-div header-container">

<table class="full-width" style="height:60px;"><tr>
	<td style="width:200px;vertical-align:middle;padding-bottom:15px;">
		<a href="/" style="font-weight:normal;color:#212121;"><img src="https://think-lab.s3.amazonaws.com/s/img/logo.png" width="160"></a>
	</td><td style="vertical-align:middle;text-align:left;padding-top:6px;width:100%;padding-left:40px;">
		<ul id="not-search-block"  class="main-header">


			


				<li class="dropdown">
					<a class="dropdown-toggle" data-toggle="dropdown" href="#">Browse <span class="caret"></span></a>
					<ul class="dropdown-menu" role="menu">
						<li><a href="/proposals">Proposals</a></li>
						<li><a href="/projects">Projects</a></li>
						<li><a href="/publications">Article discussion</a></li>
						<li><a href="/discussion">All discussion</a></li>
					</ul>
				</li>
				<li><a href="/leaderboard">Leaderboard</a></li>

			
				<li><a href="/about">About</a></li>
			


		</ul>

    <ul id="search-block" class="main-header" style="display: none; margin-right: 30px;">
        <li class="full-width">
            <form method="get" action="/search">
			<div class="input-group full-width">
				<input name="q" type="text" class="form-control search-input">
			</div>
            </form>
        </li>
    </ul>

	</td><td style="vertical-align:middle;text-align:right;padding-top:6px; min-width: 1%; white-space: nowrap;">
		<ul class="main-header pull-right" style="padding-left: 0;">
    		
                <!-- <li><a href="/login" style="color:silver;">Login</a></li>
                <li style="margin-right:0"><a class="btn btn-default btn-sm btn-upper" href="/signup">Join</a></li> -->
            

        </ul>
    </ul>
	</td>
</tr></table>

<!-- <div id="global-search-div" class="main-centered-div" style="display:none;">
	<input type="text" value="" class="thread-search form-control" placeholder="Search" data-action="/discussion">
</div> -->

	</div>
</div>

		
	  



	  


		

		<div class="dark-head"><div class="main-centered-div">

			

		</div></div>
		
		<div class="main-centered-div" style="margin-top:40px;padding-bottom:340px">
	    

<div style="margin-top:-20px; margin-bottom:20px;">

    <a class="side-box" href="/p/rephetio" style="background-color:#ebfdff;"><div class="side-type">Project:</div> <div class="side-title">Rephetio: Repurposing drugs on a hetnet [rephetio]</div></a><br>



</div>

<table style="margin-top:0px;"><tr>
	<td class="thread-table-content">

		<table class="full-width">
			
<tr class="comment-tr target-hl" id="1" data-comment-id="30">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="30"></a>

    

			<h2 class="page-title" style="margin-bottom:5px;margin-top:0px !important;">Suggestions for additional information types?</h2>

			<div style="margin-top:5px;color:gray;margin-bottom:20px;font-size:13px;">
				<a style="font-weight:normal;color:#212121" href="/u/dhimmel"><strong>Daniel Himmelstein</strong></a>
				<span class="is-researcher">Researcher</span>
				&nbsp; <a href="#1" class="light">Jan. 16, 2015</a>
				
			</div>

      

          <hr>
		

    <p>Are there any types of nodes or edges that you think we should include? Or do you think there is a superior resource for an information type than the one proposed?</p>

<p>When constructing edges, we prefer data that is:</p>

<ul><li>systematic, without knowledge bias</li><li>extensive in coverage</li><li>easy to process</li><li>without prohibitive reuse restrictions</li></ul><p>For node suggestions, we prefer controlled vocabularies so annotating the nodes with new edge types in the future is possible.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="2" data-comment-id="85">
  <td class="comment-profile-avatar">
		<a href="/u/vsmalladi"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/s/img/p_default_medium.png' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="85"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/vsmalladi">Venkat Malladi</a>
				
				<span class="light">&nbsp;<a href="#2" class="light">March 19, 2015</a>
          
        </span>
      </div>

    

    <p>Started discussion on changing ontology used for Tissue node. <a href="http://thinklab.com/discussion/tissue-node/41">Discussion is here</a></p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
<li id="note-58" class="inline-note toggle-parent" data-note-id="58">
	<p><a href="/u/jspauld">Jesse Spaulding</a>: Here's a <a href="http://thinklab.com/discussion/tissue-node/41">link to the new discussion</a> (Venkat, it might be a good idea to edit your post to add this link)
		
	</p>
	
	<table style="display:none;"><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d41' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>1.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d41'><a href='/d/41' class='pub-title'>Tissue Node</a><div class='pub-body'>Venkat Malladi, Daniel Himmelstein, Chris Mungall (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d41'>10.15363/thinklab.d41</a></div></div></td></tr></table>
</li>

				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="3" data-comment-id="345">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="345"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#3" class="light">July 30, 2015</a>
          
        </span>
      </div>

    

    <h1>Phenotype–Disease Associations</h1>

<p>A recent paper titled "The Human Phenotype Ontology: Semantic Unification of Common and Rare Disease" <span class="citation">[<a href="https://doi.org/10.1016/j.ajhg.2015.05.020" class="citation" data-key="10.1016/j.ajhg.2015.05.020">1</a>]</span> constructed a catalog of 132,006 phenotypic annotations for common diseases.</p>

<p>The approach relied on text mining of PubMed abstracts. Abstracts were annotated with diseases using MEDLINE topics. Phenotype annotation, however, relied on concept recognition. Thus the method appears to produce similar results to our <a href="http://thinklab.com/discussion/mining-knowledge-from-medline-articles-and-their-indexed-mesh-terms/67#2">disease–symptom MEDLINE approach</a>. The difference being that their approach achieves greater phenotype/symptom coverage by substituting manual topic annotation with concept recognition.</p>

<p>The data is <a href="http://pubmed-browser.human-phenotype-ontology.org/hp_common_annotations_all.tab">online</a>. The column names for the dataset, provided by Tudor Groza in personal communication, are:</p>

<ul><li>MeSH ID</li><li>MeSH descriptor</li><li>Disease Ontology ID</li><li>HPO ID</li><li>HPO label</li><li>Ranking score of the HPO concept in the context of the MeSH term - this is a modified version of TF-IDF (as per the paper)</li><li>Number of Publications containing this association</li><li>5 PMIDs (of the total number listed above) referring to this association</li></ul>

<p>I am still slightly unclear on what a phenotype means in the context of human disease, but we will keep this dataset on hand.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
<li id="note-204" class="inline-note toggle-parent" data-note-id="204">
	<p><a href="/u/dhimmel">Daniel Himmelstein</a>: Noting another disease–phenotype approach <span class="citation">[<a href="https://doi.org/10.1038/srep10888" class="citation" data-key="10.1038/srep10888">1</a>]</span> titled "Analysis of the human diseasome using phenotype similarity between common, genetic, and infectious diseases."
		
	</p>
	
	<table style="display:none;"><tr><td><a href='/doi/10.1038/srep10888' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>1.</div></td><td><div class='doi-body' data-key='10.1038/srep10888'><a href='/doi/10.1038/srep10888' class='pub-title'>Analysis of the human diseasome using phenotype similarity between common, genetic, and infectious diseases</a><div class='pub-body'>Robert Hoehndorf, Paul N. Schofield, Georgios V. Gkoutos (2015) <i>Sci. Rep.</i>. doi:<a class='pub-doi' href='https://doi.org/10.1038/srep10888'>10.1038/srep10888</a></div></div></td></tr></table>
</li>

				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="4" data-comment-id="346">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="346"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#4" class="light">July 31, 2015</a>
          
        </span>
      </div>

    

    <p>A recent study <span class="citation">[<a href="https://doi.org/10.1038/ng.3314" class="citation" data-key="10.1038/ng.3314">1</a>]</span> found disease-associated genes are mildly predictive of effective drug targets, which is in line with the findings of a preceding but less rigorous study <span class="citation">[<a href="https://doi.org/10.1038/nbt.3183" class="citation" data-key="10.1038/nbt.3183">2</a>]</span>.</p>

<p>The data supplement for  "The support of human genetic evidence for approved drug indications" contains two potentially useful resources:</p>

<p><strong>Gene-disease associations</strong> from GWAS and OMIM. Compared to <a href="http://dx.doi.org/10.15363/thinklab.d80">our approach</a> for converting from SNP to gene, their method incorporates experimental genomic evidence.</p>

<p><strong>Disease-target combinations</strong> extracted from <a href="https://citeline.com/products/pharmaprojects/">Pharmaprojects</a>, a commercial database. As per the paper: </p>

<blockquote><p>A target was defined as successful in treating an indication if a drug targeting that gene product was approved for the corresponding indication in the United States or the European Union, as annotated in Pharmaprojects.</p></blockquote>

<p>Unfortunately, this dataset is a step abstracted from the <em>real deal</em> (separate databases of drug targets and drug indications).</p>

<p>We performed some basic manipulation of their data, which is <a href="https://github.com/dhimmel/nelson/blob/af1066df8f8d2b599869864a2a5d7935cf67c1ba/process.ipynb">available here</a>.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
<li id="note-159" class="inline-note toggle-parent" data-note-id="159">
	<p><a href="/u/janispi">janet piñero</a>: I think that for gene-disease associations you should check DisGeNET (www.disgenet.org). I would start by checking the disease list that you already have and query DisGeNET with it. Currently, DisGeNET may be searched using MeSH, OMIMs, and UMLS CUIs. You can download the data in tab files.
		
	</p>
	
	<table style="display:none;"></table>
</li>

<li id="note-160" class="inline-note toggle-parent" data-note-id="160">
	<p><a href="/u/dhimmel">Daniel Himmelstein</a>: <a href="/u/janispi" class="username">@janispi</a>, thanks for the suggestion. We've <a href="http://thinklab.com/discussion/processing-disgenet-for-disease-gene-relationships/105" class="citation thread-citation" data-key="10.15363/thinklab.d105">begun</a> adding DisGeNET.
		
	</p>
	
	<table style="display:none;"><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d105' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>1.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d105'><a href='/d/105' class='pub-title'>Processing DisGeNET for disease-gene relationships</a><div class='pub-body'>Daniel Himmelstein, janet piñero (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d105'>10.15363/thinklab.d105</a></div></div></td></tr></table>
</li>

				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="5" data-comment-id="347">
  <td class="comment-profile-avatar">
		<a href="/u/alessandrodidonna"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/s/img/p_default_medium.png' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="347"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/alessandrodidonna">Alessandro Didonna</a>
				
				<span class="light">&nbsp;<a href="#5" class="light">July 31, 2015</a>
          
        </span>
      </div>

    

    <p>What do you think about implementing high-throughput data from RNA interference screenings? RNAi is an alternative, more precise way to control gene expression and it should have less off-target effects compared to pharmacological inhibition.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="6" data-comment-id="364">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="364"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#6" class="light">Aug. 8, 2015</a>
          
        </span>
      </div>

    

    <h1>Genetic perturbation edges</h1>

<p><a href="/u/alessandrodidonna" class="username">@alessandrodidonna</a>, thanks for the recommendation. You have motivated to us to add four new gene–gene metaedges:</p>

<ul><li>Gene → knockdown downregulates → Gene</li><li>Gene → knockdown upregulates → Gene</li><li>Gene → overexpression downregulates → Gene</li><li>Gene → overexpression upregulates → Gene</li></ul>

<p>These will be our first directed edges, so it will be exciting to stress test our support for directed edges, a feature that we <a href="https://github.com/dhimmel/hetio/blob/340b5f3572e29a766cb103b0883796323f983e97/hetio/graph.py#L322">designed</a> our implementation to support.</p>

<p>We'll be taking the data from <a href="http://www.lincscloud.org/">LINCS L1000</a> which contains a large number of genetic perturbation experiments:</p>

<p><img src="https://raw.githubusercontent.com/dhimmel/rephetio/b4ccfe08be839a4caa4b4a0e2b918b03d50cde65/figure/lincs-l1000-synopsys.png" alt=""></p>

<p><a href="http://thinklab.com/discussion/computing-consensus-transcriptional-profiles-for-compounds-using-lincs/43#6">Read more</a> about our consensus signatures for gene knockdowns (of 4,363 genes) and overexpressions (of 2,471 genes).</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="7" data-comment-id="1266">
  <td class="comment-profile-avatar">
		<a href="/u/ostrokach"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/226_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="1266"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/ostrokach">Alexey Strokach</a>
				
				<span class="light">&nbsp;<a href="#7" class="light">April 15, 2016</a>
          
        </span>
      </div>

    

    <p>What about disease-disease co-occurrence data mined from electronic health records (EHR)? I know of two studies in particular, but there are probably more: </p>

<p>Human Disease Network (HuDiNe) <span class="citation">[<a href="https://doi.org/10.1371/journal.pcbi.1000353" class="citation" data-key="10.1371/journal.pcbi.1000353">1</a>]</span></p>

<ul><li>Data compiled from Medicare claims of mostly elderly Americans. </li><li>Diseases are mapped to ICD9.</li><li>Data can be downloaded from the <a href="http://barabasilab.neu.edu/projects/hudine/resource/data/data.html">HuDiNe website</a>.</li><li>License: Free for academic use only.</li></ul>

<p>Stanford Translational Research Integrated Database Environment (STRIDE) <span class="citation">[<a href="https://doi.org/10.1038/sdata.2014.32" class="citation" data-key="10.1038/sdata.2014.32">2</a>]</span></p>

<ul><li>Data was mined from EHR at Stanford hospitals.</li><li>Diseases are mapped to UMLS CUI.</li><li>Data can be downloaded from <a href="http://datadryad.org/pages/policies">dryad</a>.</li><li>License: Looks like something liberal.</li></ul>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="8" data-comment-id="1268">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="1268"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#8" class="light">April 16, 2016</a>
          
        </span>
      </div>

    

    <p>Thanks <a href="/u/ostrokach" class="username">@ostrokach</a>. While <a href="#7">these resources</a> won't make it into the current project (corresponding to Hetionet v1.0), your suggestions will help us in the future.</p>

<h2>HuDiNe</h2>

<p>Regarding HuDiNe, I know I looked into the resource in the past. Specifically, I mention it in my Qualifying Exam proposal in 2013 <span class="citation">[<a href="https://doi.org/10.6084/m9.figshare.3180142.v1" class="citation" data-key="10.6084/m9.figshare.3180142.v1">1</a>]</span>:</p>

<blockquote><p>Disease comorbidity will be extracted from <a href="http://hudine.neu.edu/">HuDiNe</a>, a resource of co-occurring diseases constructed from Medicare claims spanning three years and 13 million patients <span class="citation">[<a href="https://doi.org/10.1371/journal.pcbi.1000353" class="citation" data-key="10.1371/journal.pcbi.1000353">2</a>]</span>. FDR adjusted p-values from the φ comorbidity statistic — a conservative metric optimized for prevalent diseases — will determine disease-disease edges.</p></blockquote>

<p>If I remember correctly, I didn't end up including comorbidity relationships from HuDiNe because I had trouble picking an inclusion threshold: prevalent diseases were always comorbid with each other, while rare diseases never had comorbidities. However, as the HuDiNe paper explains they include two measures of comorbidity <span class="citation">[<a href="https://doi.org/10.1371/journal.pcbi.1000353" class="citation" data-key="10.1371/journal.pcbi.1000353">2</a>]</span>:</p>

<blockquote><p>We will use two comorbidity measures to quantify the distance between two diseases: The Relative Risk (RR) and φ-correlation (φ). … For example, <em>RR</em> overestimates relationships involving rare diseases and underestimates the comorbidity between highly prevalent illnesses, whereas φ accurately discriminates comorbidities between pairs of diseases of similar prevalence but underestimates the comorbidity between rare and common diseases (see <a href="https://doi.org/10.1371/journal.pcbi.1000353.s001">SM Box 1</a>).</p></blockquote>

<p>Hence, I think it may make sense for us to revisit this dataset and attempt to pick a threshold that combines information from several <a href="http://barabasilab.neu.edu/projects/hudine/resource/data/data.html" title="HuDiNe Data Download Page">available columns</a> including:</p>

<ul><li>Relative Risk 99% Conf. Interval</li><li>Phi-correlation</li><li>t-test value</li></ul>

<p>Intuitively, comorbidity is an ideal relationship for our approach because it's high throughput and could potentially offer orthogonal information to existing relationships.</p>

<h2>STRIDE</h2>

<p>STRIDE is definitely be of interest <span class="citation">[<a href="https://doi.org/10.1038/sdata.2014.32" class="citation" data-key="10.1038/sdata.2014.32">3</a>]</span>. The data is licensed as CC0 <span class="citation">[<a href="https://doi.org/10.5061/dryad.jp917" class="citation" data-key="10.5061/dryad.jp917">4</a>]</span>, which is ideal. However, a good deal of exploratory analysis would be needed to determine a processing pipeline and see what types of cooccurring concepts have meaning.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

    		
		</table>

    

    &nbsp;<br>
    <!-- <a class="btn btn-primary btn-lg" href="/signup?next=/discussion/suggestions-for-additional-information-types/22&suppress_message=1" style>Join to Reply</a> -->


		<table class="references table-td-5" style="display:none;"><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d41' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>1.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d41'><a href='/d/41' class='pub-title'>Tissue Node</a><div class='pub-body'>Venkat Malladi, Daniel Himmelstein, Chris Mungall (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d41'>10.15363/thinklab.d41</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d41' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>2.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d41'><a href='/d/41' class='pub-title'>Tissue Node</a><div class='pub-body'>Venkat Malladi, Daniel Himmelstein, Chris Mungall (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d41'>10.15363/thinklab.d41</a></div></div></td></tr><tr><td><a href='/doi/10.1016/j.ajhg.2015.05.020' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>3.</div></td><td><div class='doi-body' data-key='10.1016/j.ajhg.2015.05.020'><a href='/doi/10.1016/j.ajhg.2015.05.020' class='pub-title'>The Human Phenotype Ontology: Semantic Unification of Common and Rare Disease</a><div class='pub-body'>Tudor Groza, Sebastian Köhler, Dawid Moldenhauer, Nicole Vasilevsky, Gareth Baynam, Tomasz Zemojtel, Lynn Marie Schriml, Warren Alden Kibbe, Paul N. Schofield, Tim Beck, Drashtti Vasant, Anthony J. Brookes, Andreas Zankl, Nicole L. Washington, Christopher J. Mungall, Suzanna E. Lewis, Melissa A. Haendel, Helen Parkinson, Peter N. Robinson (2015) <i>The American Journal of Human Genetics</i>. doi:<a class='pub-doi' href='https://doi.org/10.1016/j.ajhg.2015.05.020'>10.1016/j.ajhg.2015.05.020</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d67' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>4.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d67'><a href='/d/67' class='pub-title'>Mining knowledge from MEDLINE articles and their indexed MeSH terms</a><div class='pub-body'>Daniel Himmelstein, Alex Pankov (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d67'>10.15363/thinklab.d67</a></div></div></td></tr><tr><td><a href='/doi/10.1038/srep10888' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>5.</div></td><td><div class='doi-body' data-key='10.1038/srep10888'><a href='/doi/10.1038/srep10888' class='pub-title'>Analysis of the human diseasome using phenotype similarity between common, genetic, and infectious diseases</a><div class='pub-body'>Robert Hoehndorf, Paul N. Schofield, Georgios V. Gkoutos (2015) <i>Sci. Rep.</i>. doi:<a class='pub-doi' href='https://doi.org/10.1038/srep10888'>10.1038/srep10888</a></div></div></td></tr><tr><td><a href='/doi/10.1038/ng.3314' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>6.</div></td><td><div class='doi-body' data-key='10.1038/ng.3314'><a href='/doi/10.1038/ng.3314' class='pub-title'>The support of human genetic evidence for approved drug indications</a><div class='pub-body'>Matthew R Nelson, Hannah Tipney, Jeffery L Painter, Judong Shen, Paola Nicoletti, Yufeng Shen, Aris Floratos, Pak Chung Sham, Mulin Jun Li, Junwen Wang, Lon R Cardon, John C Whittaker, Philippe Sanseau (2015) <i>Nature Genetics</i>. doi:<a class='pub-doi' href='https://doi.org/10.1038/ng.3314'>10.1038/ng.3314</a></div></div></td></tr><tr><td><a href='/doi/10.1038/nbt.3183' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>7.</div></td><td><div class='doi-body' data-key='10.1038/nbt.3183'><a href='/doi/10.1038/nbt.3183' class='pub-title'>Opportunities for drug repositioning from phenome-wide association studies</a><div class='pub-body'>Majid Rastegar-Mojarad, Zhan Ye, Jill M Kolesar, Scott J Hebbring, Simon M Lin (2015) <i>Nat Biotechnol</i>. doi:<a class='pub-doi' href='https://doi.org/10.1038/nbt.3183'>10.1038/nbt.3183</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d105' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>8.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d105'><a href='/d/105' class='pub-title'>Processing DisGeNET for disease-gene relationships</a><div class='pub-body'>Daniel Himmelstein, janet piñero (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d105'>10.15363/thinklab.d105</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d43' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>9.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d43'><a href='/d/43' class='pub-title'>Computing consensus transcriptional profiles for LINCS L1000 perturbations</a><div class='pub-body'>Daniel Himmelstein, Caty Chung (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d43'>10.15363/thinklab.d43</a></div></div></td></tr><tr><td><a href='/doi/10.1371/journal.pcbi.1000353' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>10.</div></td><td><div class='doi-body' data-key='10.1371/journal.pcbi.1000353'><a href='/doi/10.1371/journal.pcbi.1000353' class='pub-title'>A Dynamic Network Approach for the Study of Human Phenotypes</a><div class='pub-body'>César A. Hidalgo, Nicholas Blumm, Albert-László Barabási, Nicholas A. Christakis (2009) <i>PLoS Comput Biol</i>. doi:<a class='pub-doi' href='https://doi.org/10.1371/journal.pcbi.1000353'>10.1371/journal.pcbi.1000353</a></div></div></td></tr><tr><td><a href='/doi/10.1038/sdata.2014.32' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>11.</div></td><td><div class='doi-body' data-key='10.1038/sdata.2014.32'><a href='/doi/10.1038/sdata.2014.32' class='pub-title'>Building the graph of medicine from millions of clinical narratives</a><div class='pub-body'>Samuel G. Finlayson, Paea LePendu, Nigam H. Shah (2014) <i>Scientific Data</i>. doi:<a class='pub-doi' href='https://doi.org/10.1038/sdata.2014.32'>10.1038/sdata.2014.32</a></div></div></td></tr><tr><td><a href='/doi/10.6084/m9.figshare.3180142.v1' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>12.</div></td><td><div class='doi-body' data-key='10.6084/m9.figshare.3180142.v1'><a href='/doi/10.6084/m9.figshare.3180142.v1' class='pub-title'>Qualifying Exam proposal: Drug repurposing on a heterogeneous network of pathogenesis</a><div class='pub-body'>Daniel Himmelstein (2016) <i>Figshare</i>. doi:<a class='pub-doi' href='https://doi.org/10.6084/m9.figshare.3180142.v1'>10.6084/m9.figshare.3180142.v1</a></div></div></td></tr><tr><td><a href='/doi/10.1371/journal.pcbi.1000353' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>13.</div></td><td><div class='doi-body' data-key='10.1371/journal.pcbi.1000353'><a href='/doi/10.1371/journal.pcbi.1000353' class='pub-title'>A Dynamic Network Approach for the Study of Human Phenotypes</a><div class='pub-body'>César A. Hidalgo, Nicholas Blumm, Albert-László Barabási, Nicholas A. Christakis (2009) <i>PLoS Comput Biol</i>. doi:<a class='pub-doi' href='https://doi.org/10.1371/journal.pcbi.1000353'>10.1371/journal.pcbi.1000353</a></div></div></td></tr><tr><td><a href='/doi/10.1038/sdata.2014.32' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>14.</div></td><td><div class='doi-body' data-key='10.1038/sdata.2014.32'><a href='/doi/10.1038/sdata.2014.32' class='pub-title'>Building the graph of medicine from millions of clinical narratives</a><div class='pub-body'>Samuel G. Finlayson, Paea LePendu, Nigam H. Shah (2014) <i>Scientific Data</i>. doi:<a class='pub-doi' href='https://doi.org/10.1038/sdata.2014.32'>10.1038/sdata.2014.32</a></div></div></td></tr><tr><td><a href='/doi/10.5061/dryad.jp917' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>15.</div></td><td><div class='doi-body' data-key='10.5061/dryad.jp917'><a href='/doi/10.5061/dryad.jp917' class='pub-title'>Data from: Building the graph of medicine from millions of clinical narratives</a><div class='pub-body'>Finlayson, Samuel G, LePendu, Paea, Shah, Nigam H (2014) <i>Dryad Digital Repository</i>. doi:<a class='pub-doi' href='https://doi.org/10.5061/dryad.jp917'>10.5061/dryad.jp917</a></div></div></td></tr></table>

    


	</td>
	<td class="thread-sidebar">


			

      <!-- <a class="btn btn-default follow-link btn-follow btn-lg is-following tt-top full-width" title="Follow discussion" data-id="22" data-ctype="thread" data-is-following="false">&nbsp;</a> -->

      
        <div class="status-box text-center"><span style="">Status:</span> 
  Open

</div>
      

      
        
      


				

					<div class="details">Views</div>
					<span style="font-size:20px;">166</span>

        

				
					<div class="details">Topics</div>
					<div style="margin-bottom:15px;">
		            <a href="/topics/chemoinformatics" class="tag">Chemoinformatics</a><a href="/topics/bioinformatics" class="tag">Bioinformatics</a><a href="/topics/databases" class="tag">Databases</a>
					</div>
				


		
			<div class="details">Referenced by</div>
			<div style="font-size:12px;"><ul style="padding-left:16px !important;">
			
				<li>
					
					<a href='/d/102' class='thread-title'>One network to rule them all</a>
				</li>
			
				<li>
					
					<a href='/d/171' class='thread-title'>Positive correlations between knockdown and overexpression profiles from LINCS L1000</a>
				</li>
			
				<li>
					
					<a href='/d/104' class='thread-title'>Renaming ‘heterogeneous networks’ to a more concise and catchy term</a>
				</li>
			
				<li>
					
					<a href='/d/48' class='thread-title'>Text as a resource for network population?</a>
				</li>
			
				</ul>
			</div>
		

		
			<div class="details">Cite this as</div>
			<span style="font-size:12px;">
				Daniel Himmelstein, Venkat Malladi, Alessandro Didonna, Alexey Strokach (2015) Suggestions for additional information types?. <i>Thinklab.</i> doi:<a href='https://doi.org/10.15363/thinklab.d22'>10.15363/thinklab.d22</a>
			</span>
		

		<div class="details">License</div>
		<p><a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0;width:88px;height:31px;" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a></p>

		<div class="details">Share</div>
		
			
				<a class="share-button fb" onclick="javascript:void(window.open('https://www.facebook.com/sharer/sharer.php?u=https%3A//thinklab.org/discussion/suggestions-for-additional-information-types/22','','width=600,height=460,menubar=no,status=no'));" href="#"></a>
<a class="share-button twitter" onclick="javascript:void(window.open('https://twitter.com/share?text=Suggestions%20for%20additional%20information%20types%3F&url=https%3A//thinklab.org/discussion/suggestions-for-additional-information-types/22','','width=600,height=300,menubar=no,status=no'));" href="#"></a>
<a class="share-button google" onclick="javascript:void(window.open('https://plus.google.com/share?url=https%3A//thinklab.org/discussion/suggestions-for-additional-information-types/22','','width=510,height=460,menubar=no,status=no'));" href="#"></a>

			
		

	</td>
</tr></table>


		</div>


		
	    	<div class="footer">
<!-- <div style="background:#dedede;">
	<div class="main-centered-div text-center" style="padding:20px;">
		<h4 style="margin-bottom:0px;font-weight:bold;color:gray;">Feedback? Questions? Email us: <a href="mailto:hello@thinklab.com">hello@thinklab.com</a></h4>
	</div>
</div> -->
<div class="main-footer">
	<div class="main-centered-div">

<table class="full-width"><tr>
	<td class="footer-nav">

        <div class="foot-head">Explore</div>
        <a href="/proposals">Proposals</a><br>
        <a href="/projects">Projects</a><br>
        <a href="/publications">Article discussion</a><br>
        <a href="/discussion">All discussion</a><br>
        <a href="/topics">Topics</a><br>
        <!-- <a href="/funders">Funders</a><br> -->
        <a href="/funding">Funding opportunities</a><br>

	</td>
	<td class="footer-nav">

        <!-- <div class="foot-head">Help &amp; feedback</div>	 -->
        <!-- <div class="foot-head">Intro</div>
        <a href="/benefits/for-science">Why</a><br>
        <a href="/how-it-works">How it works</a><br>

<br> -->

        <div class="foot-head">Community</div>
        <a href="/leaderboard">Leaderboard</a><br>
        <a href="/p/meta">Thinklab Meta</a><br>
				<a href="/help/faq">Help</a><br>
				<!-- <a href="/new-project">Start a proposal</a><br> -->
        <!-- <a href="/benefits/proposal">Benefits</a><br> -->
        <!-- <a href="/funding">Get funded</a><br> -->
        <!-- <a href="/participate">Become a contributor</a><br> -->
        <!-- <a href="/for-funders">For science funders</a><br> -->
        <!-- <a href="/for-institutions">For institutions</a><br> -->

	</td>
	<td class="footer-nav">



        <div class="foot-head">About</div>
        <a href="/about">Overview</a><br>
        <a href="/benefits">Benefits</a><br>
        <a href="/how-it-works">How it works</a><br>
        <a href="/about/story">Our story</a><br>
        <a href="/blog">Our blog</a><br>
        
        <br>

        <!-- <div class="foot-head">Help</div>
        <a href="/help">Help</a><br>
		 -->


	</td>
	<td class="footer-nav">
		<!-- <a href="http://twitter.com/thinklab" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/twitter_com.png" width="40"/></a>
		<a href="https://www.facebook.com/pages/Thinklab/310895039088185" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/facebook_com.png" width="40"/></a>
		<a href="https://plus.google.com/115578983466800753402/" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/plus_google_com.png" width="40"/></a>
		<a href="https://www.linkedin.com/company/thinklab-co" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/linkedin_com.png" width="40"/></a> -->
		<br><br>
		<span class="light" style='font-size:12px;'>&copy; 2015 Thinklab</span>

	</td>

</tr></table>

</div>
</div>
</div>

		

	</div>


  <script></script>
  <script>
  var MEDIA_URL = 'https://think-lab.s3.amazonaws.com/m/';
  var STATIC_URL = 'https://think-lab.s3.amazonaws.com/s/';
  var page_is_authenticated = false;
  var page_page = 'discussion';
  var page_profile_id = undefined;
</script>



	<!-- Note different script type. MUST come before MathJax.js -->
	<script type="text/x-mathjax-config">
	    MathJax.Hub.Config({
	        "HTML-CSS": { preferredFont: "TeX", availableFonts: ["STIX","TeX"],
	                        scale: 110, linebreaks: { automatic:true },
	                        EqnChunk: (MathJax.Hub.Browser.isMobile ? 10 : 50) },
	        tex2jax: { inlineMath: [ ["$$", "$$"], ["\\(","\\)"] ],
	                    displayMath: [ ["$$$","$$$"], ["\\[", "\\]"] ],
	                    processEscapes: true,}
	            });

	</script>
    <script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>





  <script src="https://code.jquery.com/jquery-1.10.2.min.js"></script>
  <script src="https://code.jquery.com/ui/1.11.4/jquery-ui.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" integrity="sha384-0mSbJDEHialfmuBBQP6A4Qrprq5OVfW37PRR3j5ELqxss1yVqOtnepnHVP9aJ7xS" crossorigin="anonymous"></script>
  <script src="https://think-lab.s3.amazonaws.com/s/dist/main.min.js"></script>
  



  
	
  
  
  
<script>
$(function() {
  recordHit('thread', 22);
});
</script>



</body>
</html>
