<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8" /><script type="text/javascript">window.NREUM||(NREUM={}),__nr_require=function(e,n,t){function r(t){if(!n[t]){var o=n[t]={exports:{}};e[t][0].call(o.exports,function(n){var o=e[t][1][n];return r(o||n)},o,o.exports)}return n[t].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<t.length;o++)r(t[o]);return r}({1:[function(e,n,t){function r(){}function o(e,n,t){return function(){return i(e,[c.now()].concat(u(arguments)),n?null:this,t),n?void 0:this}}var i=e("handle"),a=e(2),u=e(3),f=e("ee").get("tracer"),c=e("loader"),s=NREUM;"undefined"==typeof window.newrelic&&(newrelic=s);var p=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],d="api-",l=d+"ixn-";a(p,function(e,n){s[n]=o(d+n,!0,"api")}),s.addPageAction=o(d+"addPageAction",!0),s.setCurrentRouteName=o(d+"routeName",!0),n.exports=newrelic,s.interaction=function(){return(new r).get()};var m=r.prototype={createTracer:function(e,n){var t={},r=this,o="function"==typeof n;return i(l+"tracer",[c.now(),e,t],r),function(){if(f.emit((o?"":"no-")+"fn-start",[c.now(),r,o],t),o)try{return n.apply(this,arguments)}finally{f.emit("fn-end",[c.now()],t)}}}};a("setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(e,n){m[n]=o(l+n)}),newrelic.noticeError=function(e){"string"==typeof e&&(e=new Error(e)),i("err",[e,c.now()])}},{}],2:[function(e,n,t){function r(e,n){var t=[],r="",i=0;for(r in e)o.call(e,r)&&(t[i]=n(r,e[r]),i+=1);return t}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],3:[function(e,n,t){function r(e,n,t){n||(n=0),"undefined"==typeof t&&(t=e?e.length:0);for(var r=-1,o=t-n||0,i=Array(o<0?0:o);++r<o;)i[r]=e[n+r];return i}n.exports=r},{}],4:[function(e,n,t){n.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(e,n,t){function r(){}function o(e){function n(e){return e&&e instanceof r?e:e?f(e,u,i):i()}function t(t,r,o,i){if(!d.aborted||i){e&&e(t,r,o);for(var a=n(o),u=m(t),f=u.length,c=0;c<f;c++)u[c].apply(a,r);var p=s[y[t]];return p&&p.push([b,t,r,a]),a}}function l(e,n){v[e]=m(e).concat(n)}function m(e){return v[e]||[]}function w(e){return p[e]=p[e]||o(t)}function g(e,n){c(e,function(e,t){n=n||"feature",y[t]=n,n in s||(s[n]=[])})}var v={},y={},b={on:l,emit:t,get:w,listeners:m,context:n,buffer:g,abort:a,aborted:!1};return b}function i(){return new r}function a(){(s.api||s.feature)&&(d.aborted=!0,s=d.backlog={})}var u="nr@context",f=e("gos"),c=e(2),s={},p={},d=n.exports=o();d.backlog=s},{}],gos:[function(e,n,t){function r(e,n,t){if(o.call(e,n))return e[n];var r=t();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(e,n,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return e[n]=r,r}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],handle:[function(e,n,t){function r(e,n,t,r){o.buffer([e],r),o.emit(e,n,t)}var o=e("ee").get("handle");n.exports=r,r.ee=o},{}],id:[function(e,n,t){function r(e){var n=typeof e;return!e||"object"!==n&&"function"!==n?-1:e===window?0:a(e,i,function(){return o++})}var o=1,i="nr@id",a=e("gos");n.exports=r},{}],loader:[function(e,n,t){function r(){if(!x++){var e=h.info=NREUM.info,n=d.getElementsByTagName("script")[0];if(setTimeout(s.abort,3e4),!(e&&e.licenseKey&&e.applicationID&&n))return s.abort();c(y,function(n,t){e[n]||(e[n]=t)}),f("mark",["onload",a()+h.offset],null,"api");var t=d.createElement("script");t.src="https://"+e.agent,n.parentNode.insertBefore(t,n)}}function o(){"complete"===d.readyState&&i()}function i(){f("mark",["domContent",a()+h.offset],null,"api")}function a(){return E.exists&&performance.now?Math.round(performance.now()):(u=Math.max((new Date).getTime(),u))-h.offset}var u=(new Date).getTime(),f=e("handle"),c=e(2),s=e("ee"),p=window,d=p.document,l="addEventListener",m="attachEvent",w=p.XMLHttpRequest,g=w&&w.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:w,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var v=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-1039.min.js"},b=w&&g&&g[l]&&!/CriOS/.test(navigator.userAgent),h=n.exports={offset:u,now:a,origin:v,features:{},xhrWrappable:b};e(1),d[l]?(d[l]("DOMContentLoaded",i,!1),p[l]("load",r,!1)):(d[m]("onreadystatechange",o),p[m]("onload",r)),f("mark",["firstbyte",u],null,"api");var x=0,E=e(4)},{}]},{},["loader"]);</script><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info={"beacon":"bam.nr-data.net","queueTime":0,"licenseKey":"cd1101aa94","agent":"","transactionName":"bgcDMkBZW0NZBkdcWVdNJxNcW0FZVwscV1dKB08CV1taQlkRXEdFAz0WFFNIRVVcOkVcU049BxNcWw==","applicationID":"6369297","errorBeacon":"bam.nr-data.net","applicationTime":428}</script>
    <title>What is the function of FoxP in operant self-learning? | Thinklab</title>
	  <link rel="shortcut icon" href="https://think-lab.s3.amazonaws.com/s/img/favicon.ico" type="image/x-icon" />
    <meta property="og:image" content="https://think-lab.s3.amazonaws.com/s/img" />
    
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
  <link rel="stylesheet" href="https://code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css" />

<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/highlightjs/github.css" />
<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/css/css.css" />
<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/css/bootstrap-multiselect.css" />
<!-- <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'> -->

    
    
    
    
    


	<!-- GoogleAnalytics -->
	<script>
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
	(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
	m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

	ga('create', 'UA-48550230-1', 'thinklab.com');
	ga('require', 'displayfeatures');
	
	ga('send', 'pageview');

	</script>






</head>
<body >
  <form id="form_action" method="post">
    <input type='hidden' name='csrfmiddlewaretoken' value='pQNiZQX4gF2vyFeXrnbUl9EmuTWA5MG1' />
</form>

	

	<div class="body-wrapper">

		


<!-- <div style="height:5px;background-image:url('https://think-lab.s3.amazonaws.com/s/img/top-bg.gif');"></div> -->
<div class="gradient">
	<div class="main-centered-div header-container">

<table class="full-width" style="height:60px;"><tr>
	<td style="width:200px;vertical-align:middle;padding-bottom:15px;">
		<a href="/" style="font-weight:normal;color:#212121;"><img src="https://think-lab.s3.amazonaws.com/s/img/logo.png" width="160"></a>
	</td><td style="vertical-align:middle;text-align:left;padding-top:6px;width:100%;padding-left:40px;">
		<ul id="not-search-block"  class="main-header">


			


				<li class="dropdown">
					<a class="dropdown-toggle" data-toggle="dropdown" href="#">Browse <span class="caret"></span></a>
					<ul class="dropdown-menu" role="menu">
						<li><a href="/proposals">Proposals</a></li>
						<li><a href="/projects">Projects</a></li>
						<li><a href="/publications">Article discussion</a></li>
						<li><a href="/discussion">All discussion</a></li>
					</ul>
				</li>
				<li><a href="/leaderboard">Leaderboard</a></li>

			
				<li><a href="/about">About</a></li>
			


		</ul>

    <ul id="search-block" class="main-header" style="display: none; margin-right: 30px;">
        <li class="full-width">
            <form method="get" action="/search">
			<div class="input-group full-width">
				<input name="q" type="text" class="form-control search-input">
			</div>
            </form>
        </li>
    </ul>

	</td><td style="vertical-align:middle;text-align:right;padding-top:6px; min-width: 1%; white-space: nowrap;">
		<ul class="main-header pull-right" style="padding-left: 0;">
    		
                <li><a href="/login" style="color:silver;">Login</a></li>
                <li style="margin-right:0"><a class="btn btn-default btn-sm btn-upper" href="/signup">Join</a></li>
            

        </ul>
    </ul>
	</td>
</tr></table>

<!-- <div id="global-search-div" class="main-centered-div" style="display:none;">
	<input type="text" value="" class="thread-search form-control" placeholder="Search" data-action="/discussion">
</div> -->

	</div>
</div>

		
	  



	  


		

		<div class="dark-head"><div class="main-centered-div">

			

	


	<div class="header-object-type"></div>

	
			<a class="btn btn-lg follow-link btn-follow pull-right active tt-top" style="margin-left:20px;" title="Follow project discussion" data-id="11" data-ctype="project" data-is-following="false">&nbsp;</a>
	

	<h2 class="header-object-name">What is the function of FoxP in operant self-learning?</h2>

	
	<div class="project-team">
		<a href='/u/brembs'>Björn Brembs</a>
		 <!-- &nbsp; &nbsp; doi:<a href="https://doi.org/10.15363/thinklab.11">10.15363/thinklab.11</a> -->
	</div>
	

		<!-- <div class="project-team">
			<b>DOI:</b> <a href="https://doi.org/10.15363/thinklab.11">10.15363/thinklab.11</a>
		</div> -->

		

	<div class="project-team">
		
		<div class="proposal-status proposal-status-lg proposal-status-open-for-review">Open for Review</div>
		
		
		
	</div>




	


	<!-- <div class="project-team">
		
	</div> -->

    <ul class="nav nav-tabs project-nav" style="margin-top:25px;">

		

		

    

		
	    
	        <li class="active"><a href="/p/dFoxPoperant/proposal">Proposal</a></li>
	    
	    
		

		<li><a href="/p/dFoxPoperant/discussion"><span class="nav-fig">3</span>Discussions</a></li>

		
			<li><a href="/p/dFoxPoperant/leaderboard"><span class="nav-fig">3</span>Reviewers</a></li>
		

		

		


        <li class="dropdown">
            <a class="dropdown-toggle" data-toggle="dropdown"><span style="opacity:0.5">More <span class="caret"></span></span></a>
            <ul class="dropdown-menu" style="font-size:13px;background-color:#819ec8">

				

				

            <li><a href="/p/dFoxPoperant/followers"><span class="nav-fig">2</span>Followers</a></li>
            </ul>
        </li>

			

    </ul>



		</div></div>
		
		<div class="main-centered-div" style="margin-top:40px;padding-bottom:340px">
	    
	<h2 class="page-title">Proposal revision comparison</h2>

	<p>
		You are comparing versions dated
		<a href="/doc/8/revisions/d4fee4053363051569bf116bfdf0c798b495ff85">Dec. 3, 2015, 9:18 a.m.</a>
		and <a href="/doc/8/revisions/c5a65ee505161bb3328f6ea1119b6b89d74de91f">Jan. 13, 2016, 10:48 a.m.</a>
	</p>

	<hr>
	
    <h3>Content</h3>
        <table class="full-width diff-table">
        

            
            
            
                <tr class="repo-line" height="25px">
                    <td style="width: 35px;" colspan="4"></td>
            
            
            
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            

            
            

            
            
                    <td class="diff-marker" style="width: 35px;" colspan="3">+</td>
                    <td class="add-line diff-context" style="width: 45%;"># Abstract
<br>
<br>The Forkhead Box P2 (FOXP2) transcription factor is the first gene discovered to be specifically involved in the development of speech and language. Recent studies in birds, mice and fruit flies have shown that the gene exerts this involvement via its conserved function in a particular form of operant conditioning: operant self-learning. In operant self-learning, the subject learns about the outcomes of its own behavior and modifies the behavior accordingly. The neurobiology underlying this recently described form of learning is largely unknown, only Protein Kinase C and the FoxP gene family have been identified across taxa to be critically involved. The mechanism underlying operant self-learning has also been shown not to underlie other forms of learning, such as operant world-learning, allowing for a very rigorous behavioral analysis of the specificity of any of our neurobiological manipulations. In order to understand the mechanism by which FoxP mediates the neuronal modifications underlying operant self-learning, we need to find out which neurons express FoxP and which FoxP-dependent cellular processes underly memory formation. To this end, we will generate transgenic fly lines with which we will manipulate not only the expression of FoxP generally, but also the activity of FoxP expressing neurons. We will also generate monoclonal antibodies against the FoxP protein not only to study endogenous FoxP expression patterns, but also to validate and characterize our manipulations of FoxP expression. Behavioral experiments will then provide the information about which manipulation of FoxP expression in which neurons (or the activation/inactivation of which neurons) had a specific effect on operant self-learning (without affecting operant world-learning). The results of this research in the fruit fly will provide us with mechanistic insights into a novel form of behavioral learning (operant self-learning) which is one of the essential neurobiological underpinnings of speech and language acquisition in humans. As this form of learning is associated not only with speech and language disorders, but also with psychiatric disorders such as substance abuse, Tourette's or compulsive disorders, as well as with dyspraxias more broadly, the basic science on operant self-learning is instructive for clinical research in these areas as well. In fact, one instance of operant self-learning is already being researched as a treatment option for stroke patients, with the aim to increase their mobility.
<br>
</td>
                </tr>

            
        

            
            
            
            
            
            
                <tr class="repo-line" height="25px">
                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;"># Project Description
<br>
<br># 1 State of the art and preliminary work
<br>
</td>

                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;"># Project Description
<br>
<br># 1 State of the art and preliminary work
<br>
</td>
                </tr>
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            

            
            
                <tr class="repo-line">
                    <td class="diff-marker" style="width: 35px;" colspan="3">-</td>
                    <td class="sub-line diff-context" style="width: 45%;">The Forkhead Box P2 (FOXP2) transcription factor is the first gene discovered to be involved in the development of speech and language [@10.1038/35097076 @10.1038/nature01025]. Recent studies in birds, mice and the fruit fly *Drosophila* corroborate the early hypothesis that the gene exerts this involvement via its conserved function in a particular form of operant conditioning: motor learning (or operant self-learning) [@10.1371/journal.pbio.0050321 @10.1371/journal.pone.0100648 @10.1073/pnas.1414542111]. This conserved function allows for a mechanistic investigation using invertebrate genetic model organisms such as *Drosophila*. Studies of these forms of learning in other model organisms such as transgenic mice, songbirds and the marine snail *Aplysia* suffer from experimental difficulties (few genetic tools in birds and snails, difficult physiology and complex brains in vertebrates), which can be complemented by the fly model system. The *Drosophila* FoxP orthologue, dFoxP, expresses three isoforms, isoA, isoB and an intron-retention isoform (isoIR; [Figure](#dFoxPlocus)). The data suggest that isoB, in particular, may be specifically important for operant self-learning [@10.1371/journal.pone.0100648]. Therefore, we will focus our efforts on this isoform, but will include the other two isoforms, wherever possible.
</td>
            

            
            
        

            
            
            
                <tr class="repo-line" height="25px">
                    <td style="width: 35px;" colspan="4"></td>
            
            
            
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            

            
            

            
            
                    <td class="diff-marker" style="width: 35px;" colspan="3">+</td>
                    <td class="add-line diff-context" style="width: 45%;">One of the main mechanisms by which animals explore the world around them and learn how to choose adaptive behavior is "learning-by-doing" or operant learning. In operant conditioning experiments, at least two simultaneous components can be discerned: learning about causal relations in the environment (world-learning) and learning about the outcomes of one's own behavior (self-learning) [@10.1371/journal.pone.0100648 @10.1016/j.beproc.2011.02.005 @10.4161/cib.3.2.10334 @10.1016/j.cub.2009.06.014 @10.1016/j.cub.2008.07.041]. For instance, a rat pressing a lever for food can learn that when the lever is pressed (whatever the actual movement to press the lever), food will drop into the hopper, and/or the rat can learn that it needs to perform a series of movements to obtain food.
<br>
<br>World-learning is mediated by the same, evolutionary conserved biochemical processes underlying synaptic plasticity, whether it is brought about by operant or by classical conditioning [@10.1016/j.cub.2008.07.041]. In world-learning, the sites of plasticity are commonly found along the sensory pathway (e.g., [@10.1101/lm.65303 @10.1126/science.7455692 @10.1101/lm.5.1.146 @10.1016/0304-3940(95)11309-K @10.1037/a0033979 @10.1523/JNEUROSCI.3286-12.2012 @10.3109/01677068509100140 @10.1016/j.neuron.2015.11.003]). Conversely, sites of plasticity have been identified along the motor control pathways in self-learning experiments, such as the cerebellum [@10.1126/science.1207403], the basal ganglia [@10.1073/pnas.1414542111 @10.1371/journal.pbio.0050321], central pattern generators [@Nargeot1999 @10.1126/science.1069434 @10.1016/j.neuron.2008.07.019 @10.1016/j.cub.2014.03.004 @10.1016/j.cub.2009.05.030] or motor neurons [@Wolpaw1994 @Wolpaw1995]. Unlike in world-learning, however, only two components of the biochemical pathways mediating neuronal change have been identified as evolutionary conserved in studies in humans [@10.1038/35097076] mice [@10.1126/science.1207403 @10.1073/pnas.1414542111], songbirds [@10.1111/j.1460-9568.2006.04763.x @10.1371/journal.pbio.0050321], snails [@10.1016/j.neuron.2008.07.019] and flies [@10.1016/j.cub.2008.07.041 @10.1371/journal.pone.0100648]: Protein Kinase C (PKC) and the transcription factor family FoxP.
<br>
<br>There are two main areas of research required for a mechanistic understanding of self-learning. Which biochemical processes need to take place in which neural circuits for operant self-learning to occur? In the fruit fly *Drosophila*, both areas can be studied and this proposal addresses specific aspects of both questions.
<br>
<br>The Forkhead Box P2 (FOXP2) transcription factor is the first gene discovered to be involved in the development of speech and language [@10.1038/35097076 @10.1038/nature01025]. Recent studies in birds, mice and the fruit fly *Drosophila* have corroborated the early hypothesis that the gene exerts this involvement via its conserved function in operant self-learning [@10.1371/journal.pbio.0050321 @10.1371/journal.pone.0100648 @10.1073/pnas.1414542111]. This conserved function allows for a mechanistic investigation using invertebrate genetic model organisms such as *Drosophila*. Studies of these forms of learning in other model organisms such as transgenic mice, songbirds and the marine snail *Aplysia* suffer from experimental difficulties (few genetic tools in birds and snails, difficult physiology and complex brains in vertebrates), which can be complemented by the fly model system. The *Drosophila* FoxP orthologue, dFoxP, expresses three isoforms, isoA, isoB and an intron-retention isoform (isoIR; [Figure {n}](#dFoxPlocus)). The data suggest that isoB, in particular, may be specifically important for operant self-learning [@10.1371/journal.pone.0100648]. Therefore, we will focus our efforts on this isoform, but will include the other two isoforms, wherever possible.
</td>
                </tr>

            
        

            
            
            
            
            
            
                <tr class="repo-line" height="25px">
                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
<br>[:figure](dFoxPlocus)
<br>
<br>Our laboratory has been investigating the involvement of the FoxP gene in *Drosophila* learning since about 2007. In these years we have been using available mutant and transgenic fly lines for our research and started developing our own antibodies (see poster publication at 1.1.2). We showed that FoxP mutants are impaired in operant self-learning but not operant world-learning [@10.1371/journal.pone.0100648], providing a second component of the molecular process underlying operant self-learning (the first one being Protein Kinase C [@10.1016/j.cub.2008.07.041]). 
<br>
</td>

                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
<br>[:figure](dFoxPlocus)
<br>
<br>Our laboratory has been investigating the involvement of the FoxP gene in *Drosophila* learning since about 2007. In these years we have been using available mutant and transgenic fly lines for our research and started developing our own antibodies (see poster publication at 1.1.2). We showed that FoxP mutants are impaired in operant self-learning but not operant world-learning [@10.1371/journal.pone.0100648], providing a second component of the molecular process underlying operant self-learning (the first one being Protein Kinase C [@10.1016/j.cub.2008.07.041]). 
<br>
</td>
                </tr>
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            

            
            
                <tr class="repo-line">
                    <td class="diff-marker" style="width: 35px;" colspan="3">-</td>
                    <td class="sub-line diff-context" style="width: 45%;">For further mechanistic understanding of the processes modifying the neurons involved in operant self-learning, one needs to a) identify the neurons, b) characterize the role of FoxP/PKC in these neurons and c) understand the role activity in these neurons plays during learning. For PKC we have recently published work towards identifying the neurons where PKC is required (Colomb and Brembs, subm.). This work points to motorneurons as one cite of PKC-dependent plasticity. Given the data from other forms of self-learning, such as operant conditioning of the H-reflex (see, e.g., [@10.3389/fnint.2014.00025 @10.1146/annurev.neuro.24.1.807 @10.1016/S0166-2236(97)01133-8]), one would expect multiple sites of plasticity in operant self learning. In particular, work in vertebrate self-learning paradigms have implicated both cerebellar and basal ganglia structures. In fact, FOXP2 work in birds, mice and humans has focused almost exclusively on the basal ganglia. Hence, in order to elucidate FoxP function in operant self-learning, we need to find out if FoxP activity is required in the brain or in the ventral nerve chord of *Drosophila* for operant self-learning. However, as will be detailed below, we neither know the expression pattern of FoxP, nor do we have sufficient evidence that the available transgenic lines designed to specifically alter FoxP expression, affect FoxP.
</td>
            

            
            
        

            
            
            
                <tr class="repo-line" height="25px">
                    <td style="width: 35px;" colspan="4"></td>
            
            
            
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            

            
            

            
            
                    <td class="diff-marker" style="width: 35px;" colspan="3">+</td>
                    <td class="add-line diff-context" style="width: 45%;">For further mechanistic understanding of the processes modifying the neurons involved in operant self-learning, one needs to a) identify the neurons, b) characterize the role of FoxP and/or PKC in these neurons and c) understand the role activity in these neurons plays during learning. For PKC we have recently submitted work towards identifying the neurons where PKC is required (Colomb and Brembs, subm.). This work points to motorneurons as one site of PKC-dependent plasticity. Given the data from other forms of self-learning, such as operant conditioning of the H-reflex (see, e.g., [@10.3389/fnint.2014.00025 @10.1146/annurev.neuro.24.1.807 @10.1016/S0166-2236(97)01133-8]), one would expect multiple sites of plasticity in operant self learning. In particular, work in vertebrate self-learning paradigms have implicated both cerebellar and basal ganglia structures [@10.1126/science.1207403 @10.1073/pnas.1414542111]. In fact, FOXP2 work in birds, mice and humans has focused almost exclusively on the basal ganglia. Hence, in order to elucidate FoxP function in operant self-learning, one needs to find out where in the *Drosophila* nervous system FoxP activity is required for operant self-learning. However, as will be detailed below, we neither know the expression pattern of FoxP, nor do we have sufficient evidence that the available transgenic lines designed to specifically alter FoxP expression, actually affect FoxP.
</td>
                </tr>

            
        

            
            
            
            
            
            
                <tr class="repo-line" height="25px">
                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
</td>

                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
</td>
                </tr>
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            

            
            
                <tr class="repo-line">
                    <td class="diff-marker" style="width: 35px;" colspan="3">-</td>
                    <td class="sub-line diff-context" style="width: 45%;">Even in the light of recent public discussions (see, e.g., [@10.1038/527545a @10.1038/518027a @10.1038/521274a @10.12688/f1000research.6894.1]), monoclonal antibodies remain the gold standard for reliably detecting a protein's expression pattern. However, such antibodies do not exist for dFoxP. It is a technical hurdle that the currently available FoxP mutant lines are insertion lines, which only affect the expression level of some of the isoforms. This makes it difficult, for instance, to evaluate the specificity of antibodies against FoxP. Despite the efforts of several laboratories around the world, including ours, there are now only polyclonal antibodies for dFoxP available (one published, see below, several unpublished), none of which satisfy basic specificity criteria. For example, in a collaboration with the laboratory of Prof. Dr. Schade at the Charité, Berlin, with decades of experience in generating highly specific polyclonal chicken sera (see publication at 1.1.2), we used four different peptides from the *Drosophila* FoxP gene and immunized chicken. Even after 10 months of immunization and subsequent affinity purification, the serum still detects several bands in a Western blot and none at the expected size ([Figure](#ImmunizationStrategy)). Subsequent efforts in the laboratory of Constance Scharff were successful in generating antibodies against honeybee FoxP, but a parallel and analogous attempt failed for *Drosophila* (pers. comm.). Other laboratories working on FoxP also struggle with antibody-generation (e.g., Annette Schenk, Gero Miesenböck, pers. comm.) and the only published report of wholemount immunofluorescence in the adult brain shows little overlap with the authors’ own FoxP-GAL4 line and lacks documentation of standard validation procedures [@10.1016/j.bbr.2014.04.009], rendering the reliability of the expression reported from this antibody sub-optimal. Hence, in order to reliably detect the FoxP protein in wild type, mutant and transgenic *Drosophila*, monoclonal antibodies (ideally against all three isoforms) are required.
</td>
            

            
            
        

            
            
            
                <tr class="repo-line" height="25px">
                    <td style="width: 35px;" colspan="4"></td>
            
            
            
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            

            
            

            
            
                    <td class="diff-marker" style="width: 35px;" colspan="3">+</td>
                    <td class="add-line diff-context" style="width: 45%;">Even in the light of recent public discussions (see, e.g., [@10.1038/527545a @10.1038/518027a @10.1038/521274a @10.12688/f1000research.6894.1]), monoclonal antibodies (mABs) remain the gold standard for reliably detecting a protein's expression pattern. However, such antibodies do not exist for dFoxP. It is a technical hurdle that the currently available FoxP mutant lines are insertion lines, which only affect the expression level of some of the isoforms. This makes it difficult, for instance, to evaluate the specificity of antibodies against FoxP. Despite the efforts of several laboratories around the world, including ours, there are now only polyclonal antibodies (pABs) for dFoxP available (one published, see below, several unpublished), none of which satisfy basic specificity criteria. For example, in a collaboration with the laboratory of Prof. Dr. Schade at the Charité, Berlin, with decades of experience in generating highly specific polyclonal chicken sera (see publication at 1.1.2), we used four different peptides from the *Drosophila* FoxP gene and immunized chicken. Even after 10 months of immunization and subsequent affinity purification, the serum still detects several bands in a Western blot and none at the expected size ([Figure {n}](#ImmunizationStrategy)). Subsequent efforts in the laboratory of Constance Scharff were successful in generating antibodies against honeybee FoxP, but a parallel and analogous attempt failed for *Drosophila* (pers. comm.). Other laboratories working on FoxP also struggle with antibody-generation (e.g., Annette Schenk, Gero Miesenböck, pers. comm.) and the only published report of wholemount immunofluorescence in the adult brain shows little overlap with the authors’ own FoxP-GAL4 line and lacks documentation of standard validation procedures [@10.1016/j.bbr.2014.04.009], rendering the reliability of the expression reported from this antibody sub-optimal. Hence, in order to reliably detect the FoxP protein in wild type, mutant and transgenic *Drosophila*, mABs (ideally against all three isoforms) are required. Only mABs reproducibly achieve the sensitivity and specificity which pABs are lacking. The more sophisticated immunization and screening methods for mABs also allow for more effective antibody generation and more efficient screening.
</td>
                </tr>

            
        

            
            
            
            
            
            
                <tr class="repo-line" height="25px">
                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
<br>[:figure](ImmunizationStrategy)
<br>
<br>On the genomic level, there are efforts to generate both a complete dFoxP null-allele and isoform-specific alleles in other laboratories which are on track to be available well before the funding period of this proposal [@CastellsNobau2013]. Nevertheless, should these lines not be available, such lines are required for antibody characterization and we will generate them using CRSIPR/Cas9, an established technique in the collaborating Schneuwly laboratory.
<br>
</td>

                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
<br>[:figure](ImmunizationStrategy)
<br>
<br>On the genomic level, there are efforts to generate both a complete dFoxP null-allele and isoform-specific alleles in other laboratories which are on track to be available well before the funding period of this proposal [@CastellsNobau2013]. Nevertheless, should these lines not be available, such lines are required for antibody characterization and we will generate them using CRSIPR/Cas9, an established technique in the collaborating Schneuwly laboratory.
<br>
</td>
                </tr>
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            

            
            
                <tr class="repo-line">
                    <td class="diff-marker" style="width: 35px;" colspan="3">-</td>
                    <td class="sub-line diff-context" style="width: 45%;">In order to elucidate the role of FoxP in operant self-learning, it is necessary to manipulate FoxP action in a spatiotemporally controlled manner. However, the available transgenic lines are not reliable enough to allow for such manipulations. In our own publication, we show that the RNAi line we use, despite showing a clear behavioral phenotype that mimics the phenotype of the mutant line, does not show any evidence of a knock-down of FoxP on the mRNA level [@10.1371/journal.pone.0100648]. Hence, we tested all available RNAi lines for their knock-down using qPCR and found that none of the available lines appears to reduce the mRNA levels of any of the FoxP isoforms ([Figure](#RNAiLine)). While still preliminary, these results replicate results from the laboratory of Annette Schenck (pers. comm) who used the pan-neuronal elav-GAL4 driver line. Two laboratories using different driver lines obtaining largely concordant results inspire confidence even if the results may be preliminary.
</td>
            

            
            
        

            
            
            
                <tr class="repo-line" height="25px">
                    <td style="width: 35px;" colspan="4"></td>
            
            
            
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            

            
            

            
            
                    <td class="diff-marker" style="width: 35px;" colspan="3">+</td>
                    <td class="add-line diff-context" style="width: 45%;">In order to elucidate the role of FoxP in operant self-learning, it is necessary to manipulate FoxP action in a spatiotemporally controlled manner. However, the available transgenic lines are not reliable enough to allow for such manipulations. In our own publication, we show that the RNAi line we use, despite showing a clear behavioral phenotype that mimics the phenotype of the mutant line, does not show any evidence of a knock-down of FoxP on the mRNA level [@10.1371/journal.pone.0100648]. Hence, we tested all available RNAi lines for their knock-down using qPCR and found that none of the available lines appears to reduce the mRNA levels of any of the FoxP isoforms ([Figure {n}](#RNAiLine)). While still preliminary, these results replicate results from the laboratory of Annette Schenck (pers. comm) who used the pan-neuronal elav-GAL4 driver line. Two laboratories using different driver lines obtaining largely concordant results inspire confidence even if the results may be preliminary.
</td>
                </tr>

            
        

            
            
            
            
            
            
                <tr class="repo-line" height="25px">
                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
<br>[:figure](RNAiLine)
<br>
</td>

                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
<br>[:figure](RNAiLine)
<br>
</td>
                </tr>
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            

            
            
                <tr class="repo-line">
                    <td class="diff-marker" style="width: 35px;" colspan="3">-</td>
                    <td class="sub-line diff-context" style="width: 45%;">As detailed in our paper, the issue of template mismatches predisposing the targeted mRNA to sequestration, rather than degradation is a long-known issue. Because of these mismatches, without highly specific antibodies it is impossible to determine if the available lines mimic the FoxP mutant phenotype because of off-target effects or because of their effect on FoxP translation. Moreover, at least in vertebrates, FoxP2 appears to be an exceedingly stable protein, such that genomic manipulation of the locus needs weeks to have an effect on the protein level [@French2015]. Thus, a reliable antibody is required to assess the effectiveness of any manipulation targeting FoxP expression. Even with a specific antibody available, it would be desirable to have at least one RNAi line that reliably knocks down FoxP mRNA for experiments where immunotechniques are not feasible and as a positive control when testing the already available lines for knock down on the protein level.
</td>
            

            
            
        

            
            
            
                <tr class="repo-line" height="25px">
                    <td style="width: 35px;" colspan="4"></td>
            
            
            
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            

            
            

            
            
                    <td class="diff-marker" style="width: 35px;" colspan="3">+</td>
                    <td class="add-line diff-context" style="width: 45%;">As detailed in our paper, the issue of template mismatches predisposing the targeted mRNA to sequestration, rather than degradation is a long-known issue [@10.1371/journal.pbio.0040210 @10.1016/j.cell.2009.01.035 @10.1101/gad.1284105 @10.1016/j.molcel.2012.09.024]. Because of these mismatches, without highly specific antibodies it is impossible to determine if the available lines mimic the FoxP mutant phenotype because of off-target effects or because of their effect on FoxP translation. Moreover, at least in vertebrates, FoxP2 appears to be an exceedingly stable protein, such that genomic manipulations of the locus need weeks to have an effect on the protein level [@French2015]. Thus, a reliable antibody is required to assess the effectiveness of any manipulation targeting FoxP expression. Even with a specific antibody available, it would be desirable to have at least one RNAi line that reliably knocks down (degrades) FoxP mRNA for experiments where immunotechniques are not feasible and as a positive control when testing the already available lines for knock-down on the protein level. Such lines require dedicated, single-gene efforts (rather then genome-wide screens) to exclude template mismatches as well as off-target effects.
</td>
                </tr>

            
        

            
            
            
            
            
            
                <tr class="repo-line" height="25px">
                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
</td>

                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
</td>
                </tr>
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            

            
            
                <tr class="repo-line">
                    <td class="diff-marker" style="width: 35px;" colspan="3">-</td>
                    <td class="sub-line diff-context" style="width: 45%;">Traditionally, GAL4 lines derived from genomic promoter regions have not only served to mark and anatomically characterize important cells but also to manipulate these cells in order to study their function. The three available FoxP-GAL4 driver lines have been generated using, approx., 1, 1.5 and 2kb fragments of the dFoxP promotor region, respectively. All three lines differ widely in their expression pattern: The first published record of a FoxP-GAL4 expression pattern is from Lawton et al. [@10.1016/j.bbr.2014.04.009]. The authors describe their expression pattern with “Particularly strong expression was evident in the protocerebral bridge”, while the second published report on such a fly line described their expression pattern thusly: “FoxP-GAL4–driven transgene expression was confined to two subsets of Kenyon cells (KCs), the principal intrinsic neurons of the mushroom bodies” [@10.1126/science.1252114]. The laboratory of Constance Scharff, with whom we collaborate, generated their own FoxP-GAL4 line with yet a third expression pattern ([Figure](#ExpressionPattern)). The only way to determine which, if any, of the three FoxP-GAL4 lines mimics most closely the actual FoxP expression pattern, is to use highly specific, validated and thoroughly characterized antibodies for comparison and double-labelling. However, such experiments may also reveal that none of the expression patterns is accurate, as the existence of expression-altering P-Element insertions downstream of the FoxP gene suggest regulatory regions outside of the putative promotor region. These considerations are supported by FlyBase data indicating hotspots for jumu, sens, twi, Med, Ubx, D and dl downstream of the last FoxP exon (which one intuitively would have attributed to the downstream gene, hyd, if the insertions had had an effect on hyd expression, which they do not [@10.1371/journal.pone.0100648]). Moreover, there are multiple binding sites for known transcription factors (dl, twi, sens, inv), and different chromatin domains scattered throughout the FoxP gene region (FlyBase). Finally, in humans, an enhancer was recently discovered in the region downstream of the FOXP2 coding region [@10.1186/s13039-015-0173-0]. All of these consideration make it appear highly likely that a substantial amount of FoxP gene regulation is carried by regulatory elements within and downstream of the coding region of the gene, in which case the promotor-fragment method cannot yield reliable GAL4 lines. Specific and sensitive antibodies are required to test any transgenic lines as to their faithful reproduction of the wildtype dFoxP expression pattern. 
</td>
            

            
            
        

            
            
            
                <tr class="repo-line" height="25px">
                    <td style="width: 35px;" colspan="4"></td>
            
            
            
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            

            
            

            
            
                    <td class="diff-marker" style="width: 35px;" colspan="3">+</td>
                    <td class="add-line diff-context" style="width: 45%;">Traditionally, GAL4 lines derived from genomic promotor regions have not only served to mark and anatomically characterize important cells but also to manipulate these cells in order to study their function. The three available FoxP-GAL4 driver lines have been generated using, approx., 1, 1.5 and 2kb fragments of the dFoxP promotor region, respectively. All three lines differ widely in their expression pattern: The first published record of a FoxP-GAL4 expression pattern is from Lawton et al. [@10.1016/j.bbr.2014.04.009]. The authors describe their expression pattern with “Particularly strong expression was evident in the protocerebral bridge”, while the second published report on such a fly line described their expression pattern thusly: “FoxP-GAL4–driven transgene expression was confined to two subsets of Kenyon cells (KCs), the principal intrinsic neurons of the mushroom bodies” [@10.1126/science.1252114]. The laboratory of Constance Scharff, with whom we collaborate, generated their own FoxP-GAL4 line with yet a third expression pattern ([Figure {n}](#ExpressionPattern)). The only way to determine which, if any, of the three FoxP-GAL4 lines mimics most closely the actual FoxP expression pattern, is to use highly specific, validated and thoroughly characterized antibodies for comparison and double-labelling. However, such experiments may also reveal that none of the expression patterns is accurate, as the existence of expression-altering P-Element insertions downstream of the FoxP gene suggest regulatory regions outside of the putative promotor region. These considerations are supported by FlyBase data indicating hotspots for jumu, sens, twi, Med, Ubx, D and dl downstream of the last FoxP exon (which one intuitively would have attributed to the downstream gene, hyd, if the insertions had had an effect on hyd expression, which they do not [@10.1371/journal.pone.0100648]). Moreover, there are multiple binding sites for known transcription factors (dl, twi, sens, inv), and different chromatin domains scattered throughout the FoxP gene region (FlyBase). Finally, in humans, an enhancer was recently discovered in the region downstream of the FOXP2 coding region [@10.1186/s13039-015-0173-0]. All of these consideration make it appear highly likely that a substantial amount of FoxP gene regulation is carried by regulatory elements within and downstream of the coding region of the gene, in which case the promotor-fragment method is highly unlikely to yield reliable GAL4 lines. Specific and sensitive antibodies are required to test any transgenic lines as to their faithful reproduction of the wildtype dFoxP expression pattern. 
</td>
                </tr>

            
        

            
            
            
            
            
            
                <tr class="repo-line" height="25px">
                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
<br>[:figure](ExpressionPattern)
<br>
</td>

                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
<br>[:figure](ExpressionPattern)
<br>
</td>
                </tr>
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            
                <tr class="repo-line" height="25px">
                    <td class="" style="width: 35px;" colspan="8"><hr></td>
                </tr>
            

            
            

            
            
        

            
            
            
            
            
            
                <tr class="repo-line" height="25px">
                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
<br>However, single driver lines may not be sufficient for a mechanistic understanding of dFoxP gene function, as we have found evidence that the different isoforms may serve different functions [@10.1371/journal.pone.0100648]. Moreover, evidence from vertebrates suggests that at least different FoxP family members form both homo- and hetero-dimers and even oligomers [@10.1002/pro.626 @10.1016/j.str.2005.10.005 @10.1093/intimm/dxm043]. It is not known if the different dFoxP isoforms are expressed in the same neurons, nor if they form dimers or oligomers.
<br>
</td>

                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
<br>However, single driver lines may not be sufficient for a mechanistic understanding of dFoxP gene function, as we have found evidence that the different isoforms may serve different functions [@10.1371/journal.pone.0100648]. Moreover, evidence from vertebrates suggests that at least different FoxP family members form both homo- and hetero-dimers and even oligomers [@10.1002/pro.626 @10.1016/j.str.2005.10.005 @10.1093/intimm/dxm043]. It is not known if the different dFoxP isoforms are expressed in the same neurons, nor if they form dimers or oligomers.
<br>
</td>
                </tr>
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            

            
            
                <tr class="repo-line">
                    <td class="diff-marker" style="width: 35px;" colspan="3">-</td>
                    <td class="sub-line diff-context" style="width: 45%;">Using the by now tried-and-tested combination of operant self- and world-learning [@10.1371/journal.pone.0100648 @10.1016/j.beproc.2011.02.005 @10.4161/cib.3.2.10334 @10.1016/j.cub.2009.06.014 @10.1016/j.cub.2008.07.041], we will find out which of our manipulations of dFoxP expression have which effects specifically on operant self-learning. In particular, we will test if FoxP activity is required during adulthood or during development. We will test if FoxP activity is required in the brain or in the ventral nerve chord and in which neurons there. The identification of these neurons is necessary for a mechanistic understanding of this form of learning. Ultimately, we will study the mechanistic contribution of these neurons to operant self-learning by characterizing their functional connectivity to up- and downstream neurons within the circuit as well as their activity requirements by activating or silencing them during learning: what function do these neurons serve that makes their FoxP activity necessary for operant self-learning?
</td>
            

            
            
        

            
            
            
                <tr class="repo-line" height="25px">
                    <td style="width: 35px;" colspan="4"></td>
            
            
            
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            

            
            

            
            
                    <td class="diff-marker" style="width: 35px;" colspan="3">+</td>
                    <td class="add-line diff-context" style="width: 45%;">Using our by now tried-and-tested combination of operant self- and world-learning [@10.1371/journal.pone.0100648 @10.1016/j.beproc.2011.02.005 @10.4161/cib.3.2.10334 @10.1016/j.cub.2009.06.014 @10.1016/j.cub.2008.07.041], we will find out which of our manipulations of dFoxP expression have which effects specifically on operant self-learning. Crucial for the study of self-learning is the experimental separation of the two components. In brief, operant self-learning experiments differ from composite experiments (i.e., where both world- and self-learning might occur) in the absence of a predictive external stimulus. As such, a composite experiment provides all the general motor and sensory controls as well as a control for general learning ability, while the self-learning procedure isolates the self-learning component from all other abilities shared by both tasks. This is accomplished by allowing stationary flying flies to use their yaw torque to control a punishing heat beam. In self-learning, no other stimuli are present during the experiment, forcing the fly to rely solely on its own behavior for predicting the occurrence of the heat. In the composite situation, an additional color stimulus is made contingent on the heat (and hence the yaw torque), allowing the fly to rely on both its own behavior and the color to predict the heat. In the latter situation, flies preferentially rely on and learn the color cue, rather than their own behavior. Hence, in the initial phases of this composite situation, operant world-learning appears to occur exclusively. Only with extended training is the self-learning component thought to start being engaged (i.e., habit formation starts; [@10.1371/journal.pone.0100648 @10.1016/j.beproc.2011.02.005 @10.4161/cib.3.2.10334 @10.1016/j.cub.2009.06.014 @10.1016/j.cub.2008.07.041]). The details of the procedure are described in the cited literature and in video format here: [@10.3791/731]
<br>
<br>In particular, we will test if FoxP activity is required during adulthood or during development. We will test if FoxP activity is required in the brain or in the ventral nerve chord and in which neurons there. The identification of these neurons is necessary for a mechanistic understanding of this form of learning. Ultimately, we will study the mechanistic contribution of these neurons to operant self-learning by characterizing their functional connectivity to up- and downstream neurons within the circuit as well as their activity requirements by activating or silencing them during learning: what function do these neurons serve that makes their FoxP activity necessary for operant self-learning?
</td>
                </tr>

            
        

            
            
            
            
            
            
                <tr class="repo-line" height="25px">
                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
<br>## 1.1 Project-related publications  
<br>
</td>

                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
<br>## 1.1 Project-related publications  
<br>
</td>
                </tr>
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            
                <tr class="repo-line" height="25px">
                    <td class="" style="width: 35px;" colspan="8"><hr></td>
                </tr>
            

            
            

            
            
        

            
            
            
            
            
            
                <tr class="repo-line" height="25px">
                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">3. Generate a limited collection of genomic alterations to the dFoxP locus
<br>4. Anatomical and behavioral characterization of a selected subset of available and all resulting fly lines. The behavioral experiments will detect deficits specifically in operant self-learning.
<br>
</td>

                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">3. Generate a limited collection of genomic alterations to the dFoxP locus
<br>4. Anatomical and behavioral characterization of a selected subset of available and all resulting fly lines. The behavioral experiments will detect deficits specifically in operant self-learning.
<br>
</td>
                </tr>
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            

            
            
                <tr class="repo-line">
                    <td class="diff-marker" style="width: 35px;" colspan="3">-</td>
                    <td class="sub-line diff-context" style="width: 45%;">Objectives 1-3 provide a thorough characterization of FoxP gene expression in *Drosophila*, as well as an initial characterization of the potential phenotypic consequences of altering FoxP expression during/after development. Hence, these objectives constitute a publishable research goal in its own right, in particular if the anatomical characterizations of objective 4 will be included as necessary. 
</td>
            

            
            
        

            
            
            
                <tr class="repo-line" height="25px">
                    <td style="width: 35px;" colspan="4"></td>
            
            
            
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            

            
            

            
            
                    <td class="diff-marker" style="width: 35px;" colspan="3">+</td>
                    <td class="add-line diff-context" style="width: 45%;">If objectives 1-3 are all successfully achieved, they provide a thorough characterization of FoxP gene expression in *Drosophila*, as well as an initial characterization of the potential phenotypic consequences of altering FoxP expression during/after development. Hence, these objectives, if successful, constitute a publishable research goal in its own right, in particular if the anatomical characterizations of objective 4 are included as necessary. 
<br>
<br>Without at least some of the tools generated by the objectives 1-3, objective 4 cannot be completed. In fact, a thorough and reliable dissection of the mechanisms underlying operant self-learning is crucially dependent on the tools developed in objectives 1-3. However, in the face of catastrophic failure to reach some of these objectives, objectives 1 and 2 can, to some extent, complement the efforts from objective 3 and vice versa, such that some of the required experiments in objective 4 could take place. As such, the experimental design safeguards against such failures, but with the outcome of only partially reaching the goal of mechanistically explaining operant self-learning on a neuronal level (i.e, then requiring an additional funding period). Characterizing existing lines, as well as identifying the temporal requirement of and the neurons with FoxP expression necessary for operant self-learning constitute readily achievable, minimal research aims that are publishable on their own and would be completed by a later research phase. In this worst outcome scenario, the work of objective 4 provides the necessary scientific value to the other objectives and vice versa, i.e., neither objectives 1-3 nor objective 4 would be able to produce publishable results on their own under less than optimal conditions.
</td>
                </tr>

            
        

            
            
            
            
            
            
                <tr class="repo-line" height="25px">
                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
</td>

                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
</td>
                </tr>
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            

            
            
                <tr class="repo-line">
                    <td class="diff-marker" style="width: 35px;" colspan="3">-</td>
                    <td class="sub-line diff-context" style="width: 45%;">Without at least some of the tools generated by the objectives 1-3, objective 4 cannot be completed. In fact, a thorough and reliable dissection of the mechanisms underlying operant self-learning is crucially dependent on the tools developed in objectives 1-3. However, in the face of catastrophic failure to reach some of these objectives, objectives 1 and 2 can, to some extent, complement the efforts from objective 3 and vice versa, such that some of the required experiments in objective 4 could take place. As such, the experimental design safeguards against such failures, but with the outcome of only partially reaching the goal of mechanistically explaining operant self-learning on a neuronal level. Characterizing existing lines, as well as identifying the temporal requirement of and the neurons with FoxP expression necessary for operant self-learning constitute readily achievable, minimal research aims that are publishable on their own and would be completed by a later research phase.
<br>However, given that all the techniques we will use are established and have been used successfully in many other instances in the collaborating laboratories, there is currently no reason to expect such a failure. In fact, I anticipate objectives 1 and 3 to yield more possible manipulations than can be behaviorally characterized within the funding period and objective 2 to produce more than just one antibody. 
</td>
            

            
            
        

            
            
            
                <tr class="repo-line" height="25px">
                    <td style="width: 35px;" colspan="4"></td>
            
            
            
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            

            
            

            
            
                    <td class="diff-marker" style="width: 35px;" colspan="3">+</td>
                    <td class="add-line diff-context" style="width: 45%;">However, given that all the techniques we will use are established and have been used successfully in many other instances in the collaborating laboratories, there is currently no reason to expect such a failure. In fact, I anticipate objectives 1 and 3 to yield more possible manipulations than can be behaviorally characterized within the funding period and objective 2 to produce more than just one antibody. Thus, I expect objective 4 to extend into at least one additional funding period.
</td>
                </tr>

            
        

            
            
            
            
            
            
                <tr class="repo-line" height="25px">
                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
<br>## 2.3 Work programme incl. proposed research methods
<br>
</td>

                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
<br>## 2.3 Work programme incl. proposed research methods
<br>
</td>
                </tr>
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            
                <tr class="repo-line" height="25px">
                    <td class="" style="width: 35px;" colspan="8"><hr></td>
                </tr>
            

            
            

            
            
        

            
            
            
            
            
            
                <tr class="repo-line" height="25px">
                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
<br>Thus, at the end of this project, we will have generated at least one monoclonal antibody which recognizes isoB of dFoxP highly specifically. Potentially, we will have one additional antibody which binds to all isoforms. These antibodies will be generated so as to allow for utilization in both Western Blots and wholemount immunochemistry.
<br>
</td>

                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
<br>Thus, at the end of this project, we will have generated at least one monoclonal antibody which recognizes isoB of dFoxP highly specifically. Potentially, we will have one additional antibody which binds to all isoforms. These antibodies will be generated so as to allow for utilization in both Western Blots and wholemount immunochemistry.
<br>
</td>
                </tr>
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            

            
            
                <tr class="repo-line">
                    <td class="diff-marker" style="width: 35px;" colspan="3">-</td>
                    <td class="sub-line diff-context" style="width: 45%;">With this antibody we will then characterize the available RNAi lines as well as the available GAL4 lines. In the former case, we will quantify the effectiveness of these lines in Western blots. In the latter case, we will compare the GAL4 expression pattern with that of FoxP in whole mount double labelings. In addition, we will characterize the expression pattern of the eGFP reporter in the Mi{MIC} transposon inserted into the dFoxP gene at 3R:9728212 (FBti0154903). As the Hugo Bellen lab is currently generating a new series of MiMIC lines replacing those where the cassette failed to insert into an intron, it is possible that there will be such a new line available at the start of the project.
</td>
            

            
            
        

            
            
            
                <tr class="repo-line" height="25px">
                    <td style="width: 35px;" colspan="4"></td>
            
            
            
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            

            
            

            
            
                    <td class="diff-marker" style="width: 35px;" colspan="3">+</td>
                    <td class="add-line diff-context" style="width: 45%;">With this antibody we will then characterize the available RNAi lines as well as the available GAL4 lines. In the former case, we will quantify the effectiveness of these lines in Western blots. In the latter case, we will compare the GAL4 expression pattern with that of FoxP in whole mount double labelings. In addition, we will characterize the expression pattern of the eGFP reporter in the Mi{MIC} transposon inserted into the dFoxP gene at 3R:9728212 (FBti0154903). As the Hugo Bellen lab is currently generating a new series of MiMIC lines replacing those where the cassette failed to insert into an intron, it is likely that there will be such a new line available at the start of the project.
</td>
                </tr>

            
        

            
            
            
            
            
            
                <tr class="repo-line" height="25px">
                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
<br>#### 2.3.1.3 Genome editing using MiMIC and RMCE
<br>
<br>In the case that none of the available FoxP-GAL4 lines faithfully reproduces the endogenous FoxP expression pattern, we will first generate a MiMIC-GAL4 line by using RMCE to insert a GAL4 gene trap cassette into the Mi{MIC} insertion in the fifth exon of the dFoxP locus (FBti0154903). After testing for any phenotype due to the insertion, the resulting GAL4 line will also be tested with various effectors for behavioral phenotypes.
<br>
</td>

                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
<br>#### 2.3.1.3 Genome editing using MiMIC and RMCE
<br>
<br>In the case that none of the available FoxP-GAL4 lines faithfully reproduces the endogenous FoxP expression pattern, we will first generate a MiMIC-GAL4 line by using RMCE to insert a GAL4 gene trap cassette into the Mi{MIC} insertion in the fifth exon of the dFoxP locus (FBti0154903). After testing for any phenotype due to the insertion, the resulting GAL4 line will also be tested with various effectors for behavioral phenotypes.
<br>
</td>
                </tr>
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            

            
            
                <tr class="repo-line">
                    <td class="diff-marker" style="width: 35px;" colspan="3">-</td>
                    <td class="sub-line diff-context" style="width: 45%;">Because the FoxP gene product is a transcription factor and hence expected to be localized in the cells’ nucleus, the expression pattern of the protein is not very informative as to the neuropils in which the dFoxP neurons project. In order to characterize the neuronal structure and identity of FoxP isoB neurons, we will tag the last exon with a conditional CD8-GFP construct. Without expression of CD8-GFP, development of the fly will be wild type. When CD8-GFP expression is triggered after normal development, the construct will tether the protein to the membrane, not only labelling the entire neuron, but also preventing FoxP from entering the nucleus. The expression pattern of GFP (also in combination with immunochemistry for the FoxP protein) in these flies will reveal how successful this approach was. If the tether works, in addition to reporting the isoB neuronal identity and structure, this modification is predicted to also constitute an isoB-specific conditional mutant. We will test this prediction by subjecting these flies to behavioral experiments in operant self-learning. If, however, isoB forms di- or oligomers with other isoforms, we would expect a phenotype which is not specific to isoform B alone, e.g., the flies might show a flight deficit, as the mutant lines with compromised isoIR expression [@10.1371/journal.pone.0100648]. If there is no phenotype, the function of isoB appears to be cytosolic. If the GFP tag does not tether isoB to the membrane, we will replace exon 8 with the CD8-GFP construct either with a splice site or a basal promotor to generate an isoB specific mutant and membrane-bound reporter. Of course, different time-courses of CD8-GFP expression may thus also result in different phenotypes.
</td>
            

            
            
        

            
            
            
                <tr class="repo-line" height="25px">
                    <td style="width: 35px;" colspan="4"></td>
            
            
            
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            

            
            

            
            
                    <td class="diff-marker" style="width: 35px;" colspan="3">+</td>
                    <td class="add-line diff-context" style="width: 45%;">Because the FoxP gene product is a transcription factor and hence expected to be localized in the cells’ nucleus, the expression pattern of the protein is not very informative as to the neuropils into which the dFoxP neurons project. In order to characterize the neuronal structure and identity of FoxP isoB neurons, we will tag the last exon with a conditional CD8-GFP construct. Without expression of CD8-GFP, development of the fly will be wild type. When CD8-GFP expression is triggered after normal development, the construct will tether the protein to the membrane, not only labelling the entire neuron, but also preventing FoxP from entering the nucleus. The expression pattern of GFP (also in combination with immunochemistry for the FoxP protein) in these flies will reveal how successful this approach was. If the tether works, in addition to reporting the isoB neuronal identity and structure, this modification is predicted to also constitute an isoB-specific conditional mutant. We will test this prediction by subjecting these flies to behavioral experiments in operant self-learning. If, however, isoB forms di- or oligomers with other isoforms, we would expect a phenotype which is not specific to isoform B alone, e.g., the flies might show a flight deficit, as the mutant lines with compromised isoIR expression [@10.1371/journal.pone.0100648]. If there is no phenotype, the function of isoB appears to be cytosolic. If the GFP tag does not tether isoB to the membrane, we will replace exon 8 with the CD8-GFP construct either with a splice site or a basal promotor to generate an isoB specific mutant and membrane-bound reporter. Of course, different time-courses of CD8-GFP 'activation' may thus also result in different phenotypes.
</td>
                </tr>

            
        

            
            
            
            
            
            
                <tr class="repo-line" height="25px">
                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
<br>Should the first attempts to generate these flies be successful, we will use the same method to target the other isoforms.
<br>
</td>

                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
<br>Should the first attempts to generate these flies be successful, we will use the same method to target the other isoforms.
<br>
</td>
                </tr>
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            

            
            
                <tr class="repo-line">
                    <td class="diff-marker" style="width: 35px;" colspan="3">-</td>
                    <td class="sub-line diff-context" style="width: 45%;">The generation of the CD8-GFP tagged flies will first be attempted with a relatively simple RMCE-based approach [@10.1534/g3.114.014803], where the MiMIC cassette ([Figure](#GenomicEditing)a) is replaced with a cassette containing a portion of the genomic region of the dFoxP locus downstream of the Mi{MIC} insertion, with a conditional CD8-GFP construct inserted immediately after the last exon (in-frame). The resulting strain will have an altered dFoxP locus in that an attR site is now contained within exon five and that a conditional CD8-GFP construct is inserted in-frame at the very end of exon 8 ([Figure](#GenomicEditing)b). Moreover, the right flanking region of the MiMIC cassette will now be pushed downstream by our insertion. As the attR site in exon 5 lies outside of the forkhead box and any potential regulatory regions downstream of the gene are contained in the genomic region we will insert, this manipulation should not alter dFoxP expression or function. However, as the Mi{MIC} insertion in line FBti0154903 is contained within a coding exon which is spliced into all FoxP isoforms, it is likely that this insertion alone already leads to a null mutation at the FoxP locus. Thus, we expect our RMCE approach to rescue any FoxP mutant phenotype the MiMIC strain might have. As the Hugo Bellen lab is currently generating a new series of MiMIC lines replacing those where the cassette failed to insert into an intron, it is possible that there will be such a new line available at the start of the project. We will test gene expression by Western blot as well as wholemount immunochemistry using the antibody developed in 2.3.1.2. We will test gene function using conditioning essays testing for operant self-learning as well as analyzing brain morphology. In the unlikely case that any of these tests reveal a deviation from wild type in the edited flies, we will manipulate the dFoxP genomic region using a more sophisticated approach. In this approach, again taking advantage of the available strain with the Mi{MIC} transposon inserted into the fifth exon of the dFoxP gene, we will use the fast and efficient gene knock-in technique described by Vilain et al. (2014) to additionally excise all unwanted flanking regions using double-strand-break-mediated homologous recombination triggered by transgenically expressed restriction enzymes. Should both approaches fail, we can fall back on the slightly more complicated method developed by [@10.1534/g3.114.013979] in nearby Munich to replace a suitable region of the FoxP locus with an insertion where the conditional CD8-GFP is attached to exon 8. In fact, by the time this project will commence, there will likely be a genome editing method published, which is even more effective and efficient than the ones cited above. In this case, we will of course use that method.
</td>
            

            
            
        

            
            
            
                <tr class="repo-line" height="25px">
                    <td style="width: 35px;" colspan="4"></td>
            
            
            
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            

            
            

            
            
                    <td class="diff-marker" style="width: 35px;" colspan="3">+</td>
                    <td class="add-line diff-context" style="width: 45%;">Because the Mi{MIC} insertion into the dFoxP gene locus is exonic in nature, the originally published methods [@10.1038/nmeth.1662] cannot be used here. Therefore, the generation of the CD8-GFP tagged flies will instead first be attempted with a relatively simple RMCE-based approach [@10.1534/g3.114.014803], where the MiMIC cassette ([Figure {n}a](#GenomicEditing)) is replaced with a cassette containing a portion of the genomic region of the dFoxP locus downstream of the Mi{MIC} insertion, with a conditional CD8-GFP construct inserted immediately after the last exon (in-frame). The resulting strain will have an altered dFoxP locus in that an attR site is now contained within exon five and that a conditional CD8-GFP construct is inserted in-frame at the very end of exon 8 ([Figure {n}b](#GenomicEditing)). Moreover, the right flanking region of the MiMIC cassette will now be pushed downstream by our insertion. As the attR site in exon 5 lies outside of the forkhead box and any potential regulatory regions downstream of the gene are contained in the genomic region we will insert, this manipulation should not alter dFoxP expression or function. However, as the Mi{MIC} insertion in line FBti0154903 is contained within a coding exon which is spliced into all FoxP isoforms, it is likely that this insertion alone already leads to a null mutation at the FoxP locus. Thus, we expect our RMCE approach to rescue any FoxP mutant phenotype the MiMIC strain might have. As the Hugo Bellen lab is currently generating a new series of MiMIC lines replacing those where the cassette failed to insert into an intron, it is possible that there will be such a new line available at the start of the project. We will test gene expression by Western blot as well as wholemount immunochemistry using the antibody developed in 2.3.1.2. We will test gene function using conditioning essays testing for operant self-learning as well as analyzing brain morphology. In the unlikely case that any of these tests reveal a deviation from wild type in the edited flies, we will manipulate the dFoxP genomic region using a more sophisticated approach. In this approach, again taking advantage of the available strain with the Mi{MIC} transposon inserted into the fifth exon of the dFoxP gene, we will use the fast and efficient gene knock-in technique described by Vilain et al. (2014) to additionally excise all unwanted flanking regions using double-strand-break-mediated homologous recombination triggered by transgenically expressed restriction enzymes. Should both approaches fail, we can fall back on the slightly more complicated method developed by [@10.1534/g3.114.013979] in nearby Munich to replace a suitable region of the FoxP locus with an insertion where the conditional CD8-GFP is attached to exon 8. In fact, by the time this project will commence, there will likely be a genome editing method published, which is even more effective and efficient than the ones cited above. In this case, we will of course use that method.
</td>
                </tr>

            
        

            
            
            
            
            
            
                <tr class="repo-line" height="25px">
                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
<br>[:figure](GenomicEditing)
<br>
</td>

                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
<br>[:figure](GenomicEditing)
<br>
</td>
                </tr>
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            
                <tr class="repo-line" height="25px">
                    <td class="" style="width: 35px;" colspan="8"><hr></td>
                </tr>
            

            
            

            
            
        

            
            
            
            
            
            
                <tr class="repo-line" height="25px">
                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
<br>#### 2.3.2.1 Screening available lines
<br>
</td>

                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
<br>#### 2.3.2.1 Screening available lines
<br>
</td>
                </tr>
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            

            
            
                <tr class="repo-line">
                    <td class="diff-marker" style="width: 35px;" colspan="3">-</td>
                    <td class="sub-line diff-context" style="width: 45%;">With no detectable knockdown on the mRNA level, it is possible that all reported behavioral phenotypes of RNAi-based dFoxP manipulations may stem from off-target or other, unknown effects. This hypothesis will be tested by screening all available RNAi lines targeting dFoxP, as well as a selection of RNAi lines targeting other, unrelated genes (e.g. rutabaga) for defects in operant self-learning [@10.1371/journal.pone.0100648 @10.4161/cib.3.2.10334 @10.1016/j.cub.2008.07.041 @10.1016/j.cub.2009.06.014]. The student will also test RNAi lines against the other known required protein class, the six Protein Kinase C (PKC) genes in *Drosophila*, as our results with these genes so far have been inconclusive (see attached manuscript). This comparison of RNAi lines will not only familiarize the student with this sort of experiments, but also yield insights into which RNAi lines show a self-learning phenotype and hence warrant further scrutiny. Once available, the RNAi line generated in this project will of course also be thoroughly characterized using this approach. For instance, in addition to pan-neuronal drivers, we will use the drivers identified by Julien Colomb in the PKC project (see attached manuscript) to investigate if the same neurons require both PKC and FoxP function for operant self-learning. With ever more specific driver lines, we will narrow down the minimal set of neurons where FoxP activity is required. All of the lines with a self-learning phenotype will also be tested for their phenotype when their expression is restricted to adulthood. 
</td>
            

            
            
        

            
            
            
                <tr class="repo-line" height="25px">
                    <td style="width: 35px;" colspan="4"></td>
            
            
            
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            

            
            

            
            
                    <td class="diff-marker" style="width: 35px;" colspan="3">+</td>
                    <td class="add-line diff-context" style="width: 45%;">With no detectable knockdown on the mRNA level, it is possible that all reported behavioral phenotypes of RNAi-based dFoxP manipulations may stem from off-target or other, unknown effects. This hypothesis will be tested by screening all available RNAi lines targeting dFoxP, as well as a selection of RNAi lines targeting other, unrelated genes (e.g. radish, rutabaga) for defects in operant self-learning [@10.1371/journal.pone.0100648 @10.4161/cib.3.2.10334 @10.1016/j.cub.2008.07.041 @10.1016/j.cub.2009.06.014]. We will also test RNAi lines against the other known required protein class, the six Protein Kinase C (PKC) genes in *Drosophila*, as our results with these genes so far have been inconclusive (see attached manuscript). This comparison of RNAi lines will not only familiarize the student with this sort of experiments, but also yield insights into which RNAi lines show a self-learning phenotype and hence warrant further scrutiny. Once available, the RNAi line generated in this project will of course also be thoroughly characterized using this approach. For instance, in addition to pan-neuronal drivers, we will use the drivers identified by Julien Colomb in the PKC project (see attached manuscript) to investigate if the same neurons require both PKC and FoxP function for operant self-learning. With ever more specific driver lines, we will narrow down the minimal set of neurons where FoxP activity is required. All of the lines with a self-learning phenotype will also be tested for their phenotype when FoxP repression is restricted to adulthood. 
</td>
                </tr>

            
        

            
            
            
            
            
            
                <tr class="repo-line" height="25px">
                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">In case the lines generated in this project should not be available in time, we will use the expression pattern gathered from immunohistochemistry (2.3.2.2) to guide over- and misexpression experiments using the available UAS-FoxP line (FBtp0094414). The anatomical and behavioral consequences of these experiments will elucidate the role of potential dimer/oligomer formation in FoxP function, as well as further characterize the already established necessity of a wild type FoxP expression pattern for operant self-learning.
<br>The outcome of this large set of experiments will reveal the specificity of FoxP isoform involvement as well as circumscribe the neuronal circuitry in which FoxP manipulations can affect operant self-learning. As such, these results are publishable by themselves.
<br>
<br>#### 2.3.2.2 Anatomical characterization of GFP, GAL-4 expression in MiMIC lines and anti-body staining
<br>
</td>

                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">In case the lines generated in this project should not be available in time, we will use the expression pattern gathered from immunohistochemistry (2.3.2.2) to guide over- and misexpression experiments using the available UAS-FoxP line (FBtp0094414). The anatomical and behavioral consequences of these experiments will elucidate the role of potential dimer/oligomer formation in FoxP function, as well as further characterize the already established necessity of a wild type FoxP expression pattern for operant self-learning.
<br>The outcome of this large set of experiments will reveal the specificity of FoxP isoform involvement as well as circumscribe the neuronal circuitry in which FoxP manipulations can affect operant self-learning. As such, these results are publishable by themselves.
<br>
<br>#### 2.3.2.2 Anatomical characterization of GFP, GAL-4 expression in MiMIC lines and anti-body staining
<br>
</td>
                </tr>
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            

            
            
                <tr class="repo-line">
                    <td class="diff-marker" style="width: 35px;" colspan="3">-</td>
                    <td class="sub-line diff-context" style="width: 45%;">Using confocal microscopy, we will compare the staining patterns of the original MiMIC insertion line, the GAL4 MiMIC line, the GFP MiMIC line and the wholemount immunostaining using the monoclonal AB(s). We will compare these data to the expression patterns in all the available FoxP-GAL4 lines. Together with the behavioral results of the MiMIC lines, these data will identify the neurons involved in FoxP-dependent operant self-learning. These results will then be compared to the neurons where PKC is required for operant self-learning (see attached manuscript). Wild type, mutant and transgenic FoxP gene expression and anatomical phenotype results will be published separately before the behavioral characterization.
</td>
            

            
            
        

            
            
            
                <tr class="repo-line" height="25px">
                    <td style="width: 35px;" colspan="4"></td>
            
            
            
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            

            
            

            
            
                    <td class="diff-marker" style="width: 35px;" colspan="3">+</td>
                    <td class="add-line diff-context" style="width: 45%;">Using confocal microscopy, we will compare the staining patterns of the original MiMIC insertion line, the GAL4 MiMIC line, the GFP MiMIC line and the wholemount immunostaining using the mAB(s). We will compare these data to the expression patterns in all the available FoxP-GAL4 lines. Together with the behavioral results of the MiMIC lines, these data will identify the neurons involved in FoxP-dependent operant self-learning. These results will then be compared to the neurons where PKC is required for operant self-learning (see attached manuscript). Wild type, mutant and transgenic FoxP gene expression and anatomical phenotype results will be published separately before the behavioral characterization.
</td>
                </tr>

            
        

            
            
            
            
            
            
                <tr class="repo-line" height="25px">
                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
<br>#### 2.3.2.3 Behavioral characterization of MiMIC strains
<br>
</td>

                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
<br>#### 2.3.2.3 Behavioral characterization of MiMIC strains
<br>
</td>
                </tr>
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            
                <tr class="repo-line" height="25px">
                    <td class="" style="width: 35px;" colspan="8"><hr></td>
                </tr>
            

            
            

            
            
        

            
            
            
            
            
            
                <tr class="repo-line" height="25px">
                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
<br>![image](https://cloud.githubusercontent.com/assets/1437696/11292751/67e473aa-8f8d-11e5-89f9-47f540d5e17a.png)										
<br>
</td>

                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
<br>![image](https://cloud.githubusercontent.com/assets/1437696/11292751/67e473aa-8f8d-11e5-89f9-47f540d5e17a.png)										
<br>
</td>
                </tr>
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            

            
            
                <tr class="repo-line">
                    <td class="diff-marker" style="width: 35px;" colspan="3">-</td>
                    <td class="sub-line diff-context" style="width: 45%;">Red: Molecular Biology Project Area (2.3.1)
<br>Green: Behavioral Neurobiology Project Area (2.3.2)
</td>
            

            
            
        

            
            
            
                <tr class="repo-line" height="25px">
                    <td style="width: 35px;" colspan="4"></td>
            
            
            
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            

            
            

            
            
                    <td class="diff-marker" style="width: 35px;" colspan="3">+</td>
                    <td class="add-line diff-context" style="width: 45%;">Red: Project Area Molecular Biology (2.3.1)
<br>Green: Project Area Behavioral Neurobiology (2.3.2)
</td>
                </tr>

            
        

            
            
            
            
            
            
                <tr class="repo-line" height="25px">
                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
<br>**Schedule details for the generation of monoclonal antibodies.** Experience in the Pauly laboratory indicates that one year is a realistic timeframe:
<br>
</td>

                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
<br>**Schedule details for the generation of monoclonal antibodies.** Experience in the Pauly laboratory indicates that one year is a realistic timeframe:
<br>
</td>
                </tr>
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            
                <tr class="repo-line" height="25px">
                    <td class="" style="width: 35px;" colspan="8"><hr></td>
                </tr>
            

            
            

            
            
        

            
            
            
            
            
            
                <tr class="repo-line" height="25px">
                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
<br>#### 4.1.2.2 Travel Expenses
<br>
</td>

                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
<br>#### 4.1.2.2 Travel Expenses
<br>
</td>
                </tr>
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            

            
            
                <tr class="repo-line">
                    <td class="diff-marker" style="width: 35px;" colspan="3">-</td>
                    <td class="sub-line diff-context" style="width: 45%;">Expenses for the graduate students and the applicant for one European (e.g. FENS Forum and Göttingen Neuroscience Meeting) and one international conference (e.g. SfN, USA) per year, i.e., 1500€ each per year, or 13500€ in total for the funding period.
</td>
            

            
            
        

            
            
            
                <tr class="repo-line" height="25px">
                    <td style="width: 35px;" colspan="4"></td>
            
            
            
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            

            
            

            
            
                    <td class="diff-marker" style="width: 35px;" colspan="3">+</td>
                    <td class="add-line diff-context" style="width: 45%;">Expenses for the graduate students and the applicant for one European (e.g. FENS Forum or Göttingen Neuroscience Meeting) and one international conference (e.g. SfN, USA) per year, i.e., 1500€ each per year, or 13500€ in total for the funding period.
</td>
                </tr>

            
        

            
            
            
            
            
            
                <tr class="repo-line" height="25px">
                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
<br>#### 4.1.2.3 Visiting Researchers (excluding Mercator Fellows)	
<br>
</td>

                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
<br>#### 4.1.2.3 Visiting Researchers (excluding Mercator Fellows)	
<br>
</td>
                </tr>
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            
                <tr class="repo-line" height="25px">
                    <td class="" style="width: 35px;" colspan="8"><hr></td>
                </tr>
            

            
            

            
            
        

            
            
            
            
            
            
                <tr class="repo-line" height="25px">
                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
<br>[@CastellsNobau2013]: "Castells-Nobau, A., Fenckova, M., Van de Voet, M., De Gooyert, J., Kramer, J. M., Fisher, S. E., Vernes, S. C., and Schenck, A. (2013). Flying Through a Molecular Window into the Neurobiology of Language and Cognition. in CSHL Meeting on Neurobiology of *Drosophila*."
<br>
</td>

                    <td class="" style="width: 35px;" colspan="3"></td>
                    <td class="diff-context" style="width: 45%;">
<br>[@CastellsNobau2013]: "Castells-Nobau, A., Fenckova, M., Van de Voet, M., De Gooyert, J., Kramer, J. M., Fisher, S. E., Vernes, S. C., and Schenck, A. (2013). Flying Through a Molecular Window into the Neurobiology of Language and Cognition. in CSHL Meeting on Neurobiology of *Drosophila*."
<br>
</td>
                </tr>
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            

            
            
                <tr class="repo-line">
                    <td class="diff-marker" style="width: 35px;" colspan="3">-</td>
                    <td class="sub-line diff-context" style="width: 45%;">[@French2015]: "French, C. A., Correia, M., Fisher, S. E., and Costa, R. M. (2015). Foxp2 function in the adult brain. in Annual Meeting of the Society for Neuroscience, 684.15."</td>
            

            
            
        

            
            
            
                <tr class="repo-line" height="25px">
                    <td style="width: 35px;" colspan="4"></td>
            
            
            
            
            
            

            
            

            
            
        

            
            
            
            
            
            
            
            

            
            

            
            
                    <td class="diff-marker" style="width: 35px;" colspan="3">+</td>
                    <td class="add-line diff-context" style="width: 45%;">[@French2015]: "French, C. A., Correia, M., Fisher, S. E., and Costa, R. M. (2015). Foxp2 function in the adult brain. in Annual Meeting of the Society for Neuroscience, 684.15."
<br>
<br>[@Wolpaw1994]: "Carp JS, Wolpaw JR (1994). Motoneuron plasticity underlying operantly conditioned decrease in primate H-reflex. J Neurophysiol. 72(1):431-442."
<br>
<br>[@Wolpaw1995]: "Halter JA, Carp JS, Wolpaw JR (1995). Operantly conditioned motoneuron plasticity: possible role of sodium channels. J Neurophysiol. 73(2):867-871."
<br>
<br>[@Nargeot1999]: "Nargeot R, Baxter DA, Byrne JH (1999): In vitro analog of operant conditioning in aplysia. I. Contingent reinforcement modifies the functional dynamics of an identified neuron. J Neurosci. 19(6):2247-2260."</td>
                </tr>

            
        

            
            
            
            
            
            
            
            
                <tr class="repo-line" height="25px">
                    <td class="" style="width: 35px;" colspan="8"><hr></td>
                </tr>
            

            
            

            
            
        
        </table>

    



		</div>


		
	    	<div class="footer">
<div style="background:#dedede;">
	<div class="main-centered-div text-center" style="padding:20px;">
		<h4 style="margin-bottom:0px;font-weight:bold;color:gray;">Feedback? Questions? Email us: <a href="/cdn-cgi/l/email-protection#c2aaa7aeaead82b6aaabaca9aea3a0eca1adaf"><span class="__cf_email__" data-cfemail="442c2128282b04302c2d2a2f2825266a272b29">[email&#160;protected]</span><script data-cfhash='f9e31' type="text/javascript">/* <![CDATA[ */!function(t,e,r,n,c,a,p){try{t=document.currentScript||function(){for(t=document.getElementsByTagName('script'),e=t.length;e--;)if(t[e].getAttribute('data-cfhash'))return t[e]}();if(t&&(c=t.previousSibling)){p=t.parentNode;if(a=c.getAttribute('data-cfemail')){for(e='',r='0x'+a.substr(0,2)|0,n=2;a.length-n;n+=2)e+='%'+('0'+('0x'+a.substr(n,2)^r).toString(16)).slice(-2);p.replaceChild(document.createTextNode(decodeURIComponent(e)),c)}p.removeChild(t)}}catch(u){}}()/* ]]> */</script></a></h4>
	</div>
</div>
<div class="main-footer">
	<div class="main-centered-div">

<table class="full-width"><tr>
	<td class="footer-nav">

        <div class="foot-head">Explore</div>
        <a href="/proposals">Proposals</a><br>
        <a href="/projects">Projects</a><br>
        <a href="/publications">Article discussion</a><br>
        <a href="/discussion">All discussion</a><br>
        <a href="/topics">Topics</a><br>
        <!-- <a href="/funders">Funders</a><br> -->
        <a href="/funding">Funding opportunities</a><br>

	</td>
	<td class="footer-nav">

        <!-- <div class="foot-head">Help &amp; feedback</div>	 -->
        <!-- <div class="foot-head">Intro</div>
        <a href="/benefits/for-science">Why</a><br>
        <a href="/how-it-works">How it works</a><br>

<br> -->

        <div class="foot-head">Community</div>
        <a href="/leaderboard">Leaderboard</a><br>
        <a href="/p/meta">Thinklab Meta</a><br>
				<a href="/help/faq">Help</a><br>
				<!-- <a href="/new-project">Start a proposal</a><br> -->
        <!-- <a href="/benefits/proposal">Benefits</a><br> -->
        <!-- <a href="/funding">Get funded</a><br> -->
        <!-- <a href="/participate">Become a contributor</a><br> -->
        <!-- <a href="/for-funders">For science funders</a><br> -->
        <!-- <a href="/for-institutions">For institutions</a><br> -->

	</td>
	<td class="footer-nav">



        <div class="foot-head">About</div>
        <a href="/about">Overview</a><br>
        <a href="/benefits">Benefits</a><br>
        <a href="/how-it-works">How it works</a><br>
        <a href="/about/story">Our story</a><br>
        <a href="/blog">Our blog</a><br>
        <a href="/about/jobs">Join the team</a><br>
        <br>

        <!-- <div class="foot-head">Help</div>
        <a href="/help">Help</a><br>
		 -->


	</td>
	<td class="footer-nav">
		<a href="http://twitter.com/thinklab" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/twitter_com.png" width="40"/></a>
		<a href="https://www.facebook.com/pages/Thinklab/310895039088185" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/facebook_com.png" width="40"/></a>
		<a href="https://plus.google.com/115578983466800753402/" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/plus_google_com.png" width="40"/></a>
		<a href="https://www.linkedin.com/company/thinklab-co" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/linkedin_com.png" width="40"/></a>
		<br><br>
		<span class="light" style='font-size:12px;'>&copy; 2015 Thinklab</span>

	</td>

</tr></table>

</div>
</div>
</div>

		

	</div>


  <script></script>
  <script>
  var MEDIA_URL = 'https://think-lab.s3.amazonaws.com/m/';
  var STATIC_URL = 'https://think-lab.s3.amazonaws.com/s/';
  var page_is_authenticated = false;
  var page_page = '';
  var page_profile_id = undefined;
</script>



	<!-- Note different script type. MUST come before MathJax.js -->
	<script type="text/x-mathjax-config">
	    MathJax.Hub.Config({
	        "HTML-CSS": { preferredFont: "TeX", availableFonts: ["STIX","TeX"],
	                        scale: 110, linebreaks: { automatic:true },
	                        EqnChunk: (MathJax.Hub.Browser.isMobile ? 10 : 50) },
	        tex2jax: { inlineMath: [ ["$$", "$$"], ["\\(","\\)"] ],
	                    displayMath: [ ["$$$","$$$"], ["\\[", "\\]"] ],
	                    processEscapes: true,}
	            });

	</script>
    <script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>





  <script src="https://code.jquery.com/jquery-1.10.2.min.js"></script>
  <script src="https://code.jquery.com/ui/1.11.4/jquery-ui.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" integrity="sha384-0mSbJDEHialfmuBBQP6A4Qrprq5OVfW37PRR3j5ELqxss1yVqOtnepnHVP9aJ7xS" crossorigin="anonymous"></script>
  <script src="https://think-lab.s3.amazonaws.com/s/dist/main.min.js"></script>
  



  
	
<script>/* <![CDATA[ */(function(d,s,a,i,j,r,l,m,t){try{l=d.getElementsByTagName('a');t=d.createElement('textarea');for(i=0;l.length-i;i++){try{a=l[i].href;s=a.indexOf('/cdn-cgi/l/email-protection');m=a.length;if(a&&s>-1&&m>28){j=28+s;s='';if(j<m){r='0x'+a.substr(j,2)|0;for(j+=2;j<m&&a.charAt(j)!='X';j+=2)s+='%'+('0'+('0x'+a.substr(j,2)^r).toString(16)).slice(-2);j++;s=decodeURIComponent(s)+a.substr(j,m-j)}t.innerHTML=s.replace(/</g,'&lt;').replace(/\>/g,'&gt;');l[i].href='mailto:'+t.value}}catch(e){}}}catch(e){}})(document);/* ]]> */</script></body>
</html>
