<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8" /><script type="text/javascript">window.NREUM||(NREUM={}),__nr_require=function(e,n,t){function r(t){if(!n[t]){var o=n[t]={exports:{}};e[t][0].call(o.exports,function(n){var o=e[t][1][n];return r(o||n)},o,o.exports)}return n[t].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<t.length;o++)r(t[o]);return r}({1:[function(e,n,t){function r(){}function o(e,n,t){return function(){return i(e,[c.now()].concat(u(arguments)),n?null:this,t),n?void 0:this}}var i=e("handle"),a=e(2),u=e(3),f=e("ee").get("tracer"),c=e("loader"),s=NREUM;"undefined"==typeof window.newrelic&&(newrelic=s);var p=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],d="api-",l=d+"ixn-";a(p,function(e,n){s[n]=o(d+n,!0,"api")}),s.addPageAction=o(d+"addPageAction",!0),s.setCurrentRouteName=o(d+"routeName",!0),n.exports=newrelic,s.interaction=function(){return(new r).get()};var m=r.prototype={createTracer:function(e,n){var t={},r=this,o="function"==typeof n;return i(l+"tracer",[c.now(),e,t],r),function(){if(f.emit((o?"":"no-")+"fn-start",[c.now(),r,o],t),o)try{return n.apply(this,arguments)}finally{f.emit("fn-end",[c.now()],t)}}}};a("setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(e,n){m[n]=o(l+n)}),newrelic.noticeError=function(e){"string"==typeof e&&(e=new Error(e)),i("err",[e,c.now()])}},{}],2:[function(e,n,t){function r(e,n){var t=[],r="",i=0;for(r in e)o.call(e,r)&&(t[i]=n(r,e[r]),i+=1);return t}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],3:[function(e,n,t){function r(e,n,t){n||(n=0),"undefined"==typeof t&&(t=e?e.length:0);for(var r=-1,o=t-n||0,i=Array(o<0?0:o);++r<o;)i[r]=e[n+r];return i}n.exports=r},{}],4:[function(e,n,t){n.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(e,n,t){function r(){}function o(e){function n(e){return e&&e instanceof r?e:e?f(e,u,i):i()}function t(t,r,o,i){if(!d.aborted||i){e&&e(t,r,o);for(var a=n(o),u=m(t),f=u.length,c=0;c<f;c++)u[c].apply(a,r);var p=s[y[t]];return p&&p.push([b,t,r,a]),a}}function l(e,n){v[e]=m(e).concat(n)}function m(e){return v[e]||[]}function w(e){return p[e]=p[e]||o(t)}function g(e,n){c(e,function(e,t){n=n||"feature",y[t]=n,n in s||(s[n]=[])})}var v={},y={},b={on:l,emit:t,get:w,listeners:m,context:n,buffer:g,abort:a,aborted:!1};return b}function i(){return new r}function a(){(s.api||s.feature)&&(d.aborted=!0,s=d.backlog={})}var u="nr@context",f=e("gos"),c=e(2),s={},p={},d=n.exports=o();d.backlog=s},{}],gos:[function(e,n,t){function r(e,n,t){if(o.call(e,n))return e[n];var r=t();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(e,n,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return e[n]=r,r}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],handle:[function(e,n,t){function r(e,n,t,r){o.buffer([e],r),o.emit(e,n,t)}var o=e("ee").get("handle");n.exports=r,r.ee=o},{}],id:[function(e,n,t){function r(e){var n=typeof e;return!e||"object"!==n&&"function"!==n?-1:e===window?0:a(e,i,function(){return o++})}var o=1,i="nr@id",a=e("gos");n.exports=r},{}],loader:[function(e,n,t){function r(){if(!x++){var e=h.info=NREUM.info,n=d.getElementsByTagName("script")[0];if(setTimeout(s.abort,3e4),!(e&&e.licenseKey&&e.applicationID&&n))return s.abort();c(y,function(n,t){e[n]||(e[n]=t)}),f("mark",["onload",a()+h.offset],null,"api");var t=d.createElement("script");t.src="https://"+e.agent,n.parentNode.insertBefore(t,n)}}function o(){"complete"===d.readyState&&i()}function i(){f("mark",["domContent",a()+h.offset],null,"api")}function a(){return E.exists&&performance.now?Math.round(performance.now()):(u=Math.max((new Date).getTime(),u))-h.offset}var u=(new Date).getTime(),f=e("handle"),c=e(2),s=e("ee"),p=window,d=p.document,l="addEventListener",m="attachEvent",w=p.XMLHttpRequest,g=w&&w.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:w,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var v=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-1044.min.js"},b=w&&g&&g[l]&&!/CriOS/.test(navigator.userAgent),h=n.exports={offset:u,now:a,origin:v,features:{},xhrWrappable:b};e(1),d[l]?(d[l]("DOMContentLoaded",i,!1),p[l]("load",r,!1)):(d[m]("onreadystatechange",o),p[m]("onload",r)),f("mark",["firstbyte",u],null,"api");var x=0,E=e(4)},{}]},{},["loader"]);</script><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info={"beacon":"bam.nr-data.net","queueTime":0,"licenseKey":"cd1101aa94","agent":"","transactionName":"bgcDMkBZW0NZBkdcWVdNJxNcW0FZVwscV1dKB08QW11CQxYTWlBBZgEOC19dW0QCEVtHU1gG","applicationID":"6369297","errorBeacon":"bam.nr-data.net","applicationTime":785}</script>
    <title>Using the neo4j graph database for hetnets | Thinklab</title>
	  <link rel="shortcut icon" href="https://think-lab.s3.amazonaws.com/s/img/favicon.ico" type="image/x-icon" />
    <meta property="og:image" content="https://think-lab.s3.amazonaws.com/s/img" />
    
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
  <link rel="stylesheet" href="https://code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css" />

<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/highlightjs/github.css" />
<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/css/css.css" />
<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/css/bootstrap-multiselect.css" />
<!-- <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'> -->

    
    
    
    
  
    <meta name="citation_doi" content="10.15363/thinklab.d112">
    <meta name="citation_title" content="Using the neo4j graph database for hetnets">
    <meta name="citation_online_date" content="Oct. 2, 2015">
    <meta name="citation_author" content="Daniel Himmelstein">
    <meta name="citation_journal_title" content="Thinklab">
    <meta name="citation_publisher" content="Thinklab">
  

    


	<!-- GoogleAnalytics -->
	<script>
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
	(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
	m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

	ga('create', 'UA-48550230-1', 'thinklab.org');
	ga('require', 'displayfeatures');
	
	ga('send', 'pageview');

	</script>






</head>
<body >
  <form id="form_action" method="post">
    <input type='hidden' name='csrfmiddlewaretoken' value='JC3hnVSWzcwBOS226QTq2eGyHXj1YAM6' />
</form>

	

	<div class="body-wrapper">

		


<!-- <div style="height:5px;background-image:url('https://think-lab.s3.amazonaws.com/s/img/top-bg.gif');"></div> -->
<div class="gradient">
	<div class="main-centered-div header-container">

<table class="full-width" style="height:60px;"><tr>
	<td style="width:200px;vertical-align:middle;padding-bottom:15px;">
		<a href="/" style="font-weight:normal;color:#212121;"><img src="https://think-lab.s3.amazonaws.com/s/img/logo.png" width="160"></a>
	</td><td style="vertical-align:middle;text-align:left;padding-top:6px;width:100%;padding-left:40px;">
		<ul id="not-search-block"  class="main-header">


			


				<li class="dropdown">
					<a class="dropdown-toggle" data-toggle="dropdown" href="#">Browse <span class="caret"></span></a>
					<ul class="dropdown-menu" role="menu">
						<li><a href="/proposals">Proposals</a></li>
						<li><a href="/projects">Projects</a></li>
						<li><a href="/publications">Article discussion</a></li>
						<li><a href="/discussion">All discussion</a></li>
					</ul>
				</li>
				<li><a href="/leaderboard">Leaderboard</a></li>

			
				<li><a href="/about">About</a></li>
			


		</ul>

    <ul id="search-block" class="main-header" style="display: none; margin-right: 30px;">
        <li class="full-width">
            <form method="get" action="/search">
			<div class="input-group full-width">
				<input name="q" type="text" class="form-control search-input">
			</div>
            </form>
        </li>
    </ul>

	</td><td style="vertical-align:middle;text-align:right;padding-top:6px; min-width: 1%; white-space: nowrap;">
		<ul class="main-header pull-right" style="padding-left: 0;">
    		
                <!-- <li><a href="/login" style="color:silver;">Login</a></li>
                <li style="margin-right:0"><a class="btn btn-default btn-sm btn-upper" href="/signup">Join</a></li> -->
            

        </ul>
    </ul>
	</td>
</tr></table>

<!-- <div id="global-search-div" class="main-centered-div" style="display:none;">
	<input type="text" value="" class="thread-search form-control" placeholder="Search" data-action="/discussion">
</div> -->

	</div>
</div>

		
	  



	  


		

		<div class="dark-head"><div class="main-centered-div">

			

		</div></div>
		
		<div class="main-centered-div" style="margin-top:40px;padding-bottom:340px">
	    

<div style="margin-top:-20px; margin-bottom:20px;">

    <a class="side-box" href="/p/rephetio" style="background-color:#ebfdff;"><div class="side-type">Project:</div> <div class="side-title">Rephetio: Repurposing drugs on a hetnet [rephetio]</div></a><br>



</div>

<table style="margin-top:0px;"><tr>
	<td class="thread-table-content">

		<table class="full-width">
			
<tr class="comment-tr target-hl" id="1" data-comment-id="437">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="437"></a>

    

			<h2 class="page-title" style="margin-bottom:5px;margin-top:0px !important;">Using the neo4j graph database for hetnets</h2>

			<div style="margin-top:5px;color:gray;margin-bottom:20px;font-size:13px;">
				<a style="font-weight:normal;color:#212121" href="/u/dhimmel"><strong>Daniel Himmelstein</strong></a>
				<span class="is-researcher">Researcher</span>
				&nbsp; <a href="#1" class="light">Oct. 2, 2015</a>
				
			</div>

      

          <hr>
		

    <p>Recently, I went a two-part meetup series on the graph database <a href="http://neo4j.com/">neo4j</a>. <a href="http://nicolewhite.github.io/">Nicole White</a> led the meetups and her materials are online:</p>

<ol><li>neo4j: Intro to Graphs (<a href="https://www.dropbox.com/s/zv0s4lwc6gvwxjy/Galvanize.pptx?dl=0">slides</a>, <a href="http://www.meetup.com/SF-Data-Science/events/224956828">meetup</a>)</li><li>Data Science with Python and Neo4j (<a href="http://nicolewhite.github.io/neo4j-jupyter/main.html">tutorial</a>, <a href="https://github.com/nicolewhite/neo4j-jupyter">repository</a>, <a href="http://www.meetup.com/SF-Data-Science/events/224406352">meetup</a>)</li></ol>

<p>Currently, we store our hetnets in compressed json text files. To perform any computation or graph analyses, we must load the network into memory, a process that takes from 2–5 minutes for version one of our <a href="https://think-lab.github.io/d/102#1" class="citation thread-citation" data-key="10.15363/thinklab.d102">network</a>. In contrast neo4j provides persistent storage with immediate access.</p>

<p>Additional benefits of neo4j include a mature <a href="https://github.com/GraphGeeks/awesome-neo4j">ecosystem</a> offering broad functionality. The <a href="http://neo4j.com/developer/cypher-query-language/">Cyper</a> query language is especially exciting. Cypher uses an ASCII-art based syntax to enable advanced graph lookups and traversals with little boilerplate.</p>

<h2>Comparison to hetio</h2>

<p>There are a few differences between neo4j and our python package <a href="https://github.com/dhimmel/hetio">hetio</a> <span class="citation">[<a href="https://doi.org/10.5281/zenodo.31763" class="citation" data-key="10.5281/zenodo.31763">1</a>]</span> with regards to hetnets:</p>

<ul><li><strong>nomenclature</strong> — an edge in hetio is called a relationship in neo4j. hetio calls itself a <a href="https://think-lab.github.io/d/104" class="citation thread-citation" data-key="10.15363/thinklab.d104">hetnet</a>, while neo4j calls itself a <a href="http://neo4j.com/developer/graph-database/#property-graph">property graph</a></li><li><strong>node type</strong> — in hetio each node belongs to one metanode representing its type. in neo4j node are annotated with a label to indicate type, and a node can have ≥ 0 labels</li><li><strong>directionality</strong> — in hetio metaedges are either directed or undirected and edges conform to their metaedge's directionality. neo4j doesn't support undirected edges. The <a href="http://graphaware.com/neo4j/2013/10/11/neo4j-bidirectional-relationships.html">suggested workaround</a> is to arbitrarily choose a direction upon creation and ignore the direction when querying</li><li><strong>type graph</strong> — hetio requires a predefined graph of types called a metagraph. neo4j does not enforce or explicitly support a graph of type definitions</li><li><strong>inverted edges</strong> — hetio internally stores two copies of each edge (inverses of each other)</li></ul>

<p>We plan to create export functionality from hetio to neo4j, so we can leverage the strengths of neo4j.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="2" data-comment-id="455">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="455"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#2" class="light">Oct. 4, 2015</a>
          
        </span>
      </div>

    

    <h1>Exporting hetio hetnets to neo4j</h1>

<p>We've <a href="https://github.com/dhimmel/hetio/commit/1860faadb455c1f20546ce7923b5b78fd74796b3">added</a> neo4j export capability to hetio. Our implementation uses the <a href="http://py2neo.org/2.0/">py2neo</a> toolkit to interact with the neo4j server. </p>

<p>Adding edges is quite slow and the database size is large. However, the neo4j browser combined with cypher is great for exploratory analyses. In a short amount of time, I discovered 4 issues with our network (<a href="https://github.com/dhimmel/integrate/issues/4">1</a>, <a href="https://github.com/dhimmel/integrate/issues/5">2</a>, <a href="https://github.com/dhimmel/integrate/issues/6">3</a>, <a href="https://github.com/dhimmel/integrate/issues/7">4</a>) and created a <a href="https://twitter.com/dhimmel/status/650558967492444160">sneak-preview visualization</a>.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="3" data-comment-id="460">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="460"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#3" class="light">Oct. 6, 2015</a>
          
        </span>
      </div>

    

    <p>We <a href="https://github.com/dhimmel/integrate/blob/8d93f5409a5fba85ee1109b470bbf2bb2b1a67e6/neo4j.ipynb">exported</a> the <a href="https://github.com/dhimmel/integrate/tree/8d93f5409a5fba85ee1109b470bbf2bb2b1a67e6">current version</a> of our network, which contains 49,399 nodes and 2,997,246 edges, to neo4j. The export took 10 hours and resulted in a 3.04 GB database.</p>

<p>The <code>data/graph.db/</code>, which stores the database, contained the following files with sizes over 1 MB:</p>

<table class="table markdown-table"><thead><tr><th>file</th><th>size</th></tr></thead><tbody><tr><td><code>messages.log</code></td><td>1.3 GB</td></tr><tr><td><code>data/graph.db/neostore.transaction.db.1</code></td><td>262 MB</td></tr><tr><td><code>data/graph.db/neostore.transaction.db.2</code></td><td>262 MB</td></tr><tr><td><code>data/graph.db/neostore.transaction.db.3</code></td><td>262 MB</td></tr><tr><td><code>data/graph.db/neostore.transaction.db.4</code></td><td>262 MB</td></tr><tr><td><code>data/graph.db/neostore.transaction.db.5</code></td><td>262 MB</td></tr><tr><td><code>data/graph.db/neostore.transaction.db.6</code></td><td>262 MB</td></tr><tr><td><code>data/graph.db/neostore.transaction.db.7</code></td><td>113 MB</td></tr><tr><td><code>neostore.propertystore.db</code></td><td>127.9 MB</td></tr><tr><td><code>neostore.relationshipstore.db</code></td><td>102 MB</td></tr><tr><td><code>neostore.propertystore.db.strings</code></td><td>25 MB</td></tr><tr><td><code>neostore.relationshipgroupstore.db</code></td><td>3.2 MB</td></tr><tr><td><code>rrd</code></td><td>2.0 MB</td></tr><tr><td><code>neostore.propertystore.db.arrays</code></td><td>1.5 MB</td></tr><tr><td><code>neostore.nodestore.db</code></td><td>1.5 MB</td></tr></tbody></table>

<p>We will look into ways to speed up our write times and reduce storage.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="4" data-comment-id="481">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="481"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#4" class="light">Oct. 16, 2015</a>
          
        </span>
      </div>

    

    <h1><em>DWPC</em> in Cypher</h1>

<p>We've <a href="https://github.com/dhimmel/hetio/blob/3150399c8d2570c2c6c93f1e3866d6b7c6bac2f3/hetio/neo4j.py#L135">implemented</a> the degree-weighted path count (<em>DWPC</em> <span class="citation">[<a href="https://doi.org/10.1371/journal.pcbi.1004259" class="citation" data-key="10.1371/journal.pcbi.1004259">1</a>]</span>) in <a href="http://neo4j.com/developer/cypher-query-language/">Cypher</a>. Our implementation produces a different query for each metapath, but specifies the source node (<code>source</code>), target node (<code>target</code>), and damping exponent (<code>w</code>) as <a href="http://neo4j.com/docs/2.2.6/cypher-parameters.html">parameters</a>.</p>

<p>Below is the query for the <em>CsCuGod&gt;GuD</em> metapath:</p>

<pre><code class="no-highlight hljs">MATCH p = (n0:Compound)-[:SIMILARITY]-(n1:Compound)-[:UPREGULATION]-(n2:Gene)-[:OVEREXPRESSION_DOWNREGULATION]-&gt;(n3:Gene)-[:UPREGULATION]-(n4:Disease)
WHERE n0.name = { source }
AND n4.name = { target }
WITH [size((n0)-[:SIMILARITY]-(:Compound)),
size((:Compound)-[:SIMILARITY]-(n1)),
size((n1)-[:UPREGULATION]-(:Gene)),
size((:Compound)-[:UPREGULATION]-(n2)),
size((n2)-[:OVEREXPRESSION_DOWNREGULATION]-&gt;(:Gene)),
size((:Gene)-[:OVEREXPRESSION_DOWNREGULATION]-&gt;(n3)),
size((n3)-[:UPREGULATION]-(:Disease)),
size((:Gene)-[:UPREGULATION]-(n4))] AS degrees
RETURN sum(reduce(pdp = 1.0, d in degrees| pdp * d ^ -{ w }))</code></pre>

<p>The <em>DWPC</em> for this metapath measures the extent that compounds similar to the query compound upregulate genes whose overpression downregulates genes upregulated by the query disease. The <code>MATCH</code> clause identifies paths corresponding to the metapath. The <code>WITH</code> clause computes degrees along each path and the <code>RETURN</code> clause computes path degree products (<em>PDPs</em>) and sums them to get the <em>DWPC</em>.</p>

<h2>Comparison to hetio</h2>

<p>We configure our hetio queries to exclude paths with duplicate nodes. However, neo4j <a href="http://neo4j.com/docs/2.2.6/cypherdoc-uniqueness.html">excludes</a> duplicate relationships. Additionally, when computing features for an indicated compound–disease pair, we configure our hetio queries to ignore that indication. Our current cypher framework does not support this exclusion.</p>

<p>Our preliminary experience is that <em>DWPC</em> computations in neo4j run approximately twice as quickly as <a href="https://github.com/dhimmel/hetio/blob/3150399c8d2570c2c6c93f1e3866d6b7c6bac2f3/hetio/pathtools.py">in hetio</a>. However, hetio may have more room for improvement, since we haven't implemented path caching yet.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="5" data-comment-id="484">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="484"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#5" class="light">Oct. 21, 2015</a>
          
        </span>
      </div>

    

    <h1>GraphConnect 2015</h1>

<p>Today I attended <a href="http://graphconnect.com/">GraphConnect</a> — a conference focused on neo4j. CEO, Emil Eifrem, kicked the event off with several exciting announcements:</p>

<ul><li>Neo4j 2.3 has been released bringing speed and scalability <a href="http://neo4j.com/release-notes/neo4j-2-3-0/">improvements</a>. Specifically, the caching infrastructure has been rewritten to <a href="http://neo4j.com/blog/new-on-neo4j-the-neo4j-2-3-0-milestone-2-release-is-here/">provide</a> "significant (up to 2-3x) improvements in concurrent read scaling."</li><li>Neo4j 3.0 is in the works and will bring unified and official drivers across languages. The initial release will include a Python but not R driver.</li><li>Cypher <a href="http://neo4j.com/blog/open-cypher-sql-for-graphs/">will be</a> open sourced as <a href="http://www.opencypher.org/">openCypher</a>. This will hopefully give rise to a standard query language for all graph databases.</li></ul>

<h2>Select learnings</h2>

<p>Neo4j is designed for <strong>deep traversals</strong>. Other graph databases preferentially support big data (networks with billions of nodes) over efficient traversal. Since our network is small but our edge prediction method requires deep traversal, neo4j is a good fit for our application.</p>

<p>Neo4j doesn't enforce or specifically support a type graph (also called a schema, <strong>metagraph</strong>, or graph model). However, a metagraph can <a href="http://neo4j.com/blog/rvb-2-2-meta-graph/">easily be created</a> from an already populated graph. While neo4j won't innately reason based on the created metagraph, it can be convenient from a user standpoint.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
<li id="note-194" class="inline-note toggle-parent" data-note-id="194">
	<p><a href="/u/larsjuhljensen">Lars Juhl Jensen</a>: Neo4j 3.0 should be interesting. So far, what has mainly held me back was that I do not want to develop my bioinformatics software in Java and doing everything through Cypher was not sufficiently efficient to warrant migration from a PostgreSQL database.
		
	</p>
	
	<table style="display:none;"></table>
</li>

<li id="note-195" class="inline-note toggle-parent" data-note-id="195">
	<p><a href="/u/dhimmel">Daniel Himmelstein</a>: Milestone 1 of version 3.0 was <a href="http://neo4j.com/blog/neo4j-3-0-milestone-1-release/">released today</a> with a <a href="https://github.com/neo4j/neo4j-python-driver">python driver</a>. The update promises fast access from outside of java. However, you will still need to use Cypher (which I find easier and more powerful than SQL). <a href="/u/larsjuhljensen" class="username">@larsjuhljensen</a>, I would wait till the final 3.0 release for production use but wanted to give you a heads up of what lies ahead.
		
	</p>
	
	<table style="display:none;"></table>
</li>

<li id="note-196" class="inline-note toggle-parent" data-note-id="196">
	<p><a href="/u/larsjuhljensen">Lars Juhl Jensen</a>: Thanks - my problem was, though, that my network queries could not be expressed efficiently in Cypher. Whereas shortest path could be done very efficiently, something as simple as shortest path in a weighted graph could not. Maybe Cypher has become more powerful since?
		
	</p>
	
	<table style="display:none;"></table>
</li>

<li id="note-197" class="inline-note toggle-parent" data-note-id="197">
	<p><a href="/u/dhimmel">Daniel Himmelstein</a>: Cypher doesn't have great algorithm coverage. Shortest weighted path <a href="http://www.khalidabuhakmeh.com/finding-the-shortest-path-using-neo4j-graph-database">isn't too hard</a> to implement, albeit inefficiently. However, if you're encoding anything that resembles a hetnet, cypher will beat SQL for data interactions.
		
	</p>
	
	<table style="display:none;"></table>
</li>

<li id="note-219" class="inline-note toggle-parent" data-note-id="219">
	<p><a href="/u/dhimmel">Daniel Himmelstein</a>: Lars, you should have mentioned your article on whether graph databases are ready for bioinformatics <span class="citation">[<a href="https://doi.org/10.1093/bioinformatics/btt549" class="citation" data-key="10.1093/bioinformatics/btt549">1</a>]</span>! This is a citation that belongs in this discussion.
		
	</p>
	
	<table style="display:none;"><tr><td><a href='/doi/10.1093/bioinformatics/btt549' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>1.</div></td><td><div class='doi-body' data-key='10.1093/bioinformatics/btt549'><a href='/doi/10.1093/bioinformatics/btt549' class='pub-title'>Are graph databases ready for bioinformatics?</a><div class='pub-body'>C. T. Have, L. J. Jensen (2013) <i>Bioinformatics</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/bioinformatics/btt549'>10.1093/bioinformatics/btt549</a></div></div></td></tr></table>
</li>

<li id="note-271" class="inline-note toggle-parent" data-note-id="271">
	<p><a href="/u/dhimmel">Daniel Himmelstein</a>: <a href="/u/larsjuhljensen" class="username">@larsjuhljensen</a>, I just saw that APOC — a Neo4j plugin for "Awesome Procedures On Cypher" — contains <a href="https://neo4j-contrib.github.io/neo4j-apoc-procedures/#_graph_algorithms">several graph algorithms</a>. Installing APOC <a href="https://github.com/dhimmel/hetionet/commit/687d2e22bf40f5244adeaac49724502f40b45302">is easy</a>. It looks like <a href="https://neo4j-contrib.github.io/neo4j-apoc-procedures/#_graph_algorithms_work_in_progress">some but not all algorithms</a> have edge weight support.
		
	</p>
	
	<table style="display:none;"></table>
</li>

				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="6" data-comment-id="552">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="552"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#6" class="light">Nov. 26, 2015</a>
          
        </span>
      </div>

    

    <h1>Query Optimization</h1>

<p>Above, we <a href="#4">debuted</a> <em>DWPC</em> (degree-weighted path count) computation using Cypher. I noticed that looking up the degrees along each path was a major timesink. This finding was surprising because node degree lookup should be trivial compared to path traversal.</p>

<p>In a stroke of genius, <a href="/u/alizee" class="username">@alizee</a> <a href="https://twitter.com/dhimmel/status/662415810825056256">hypothesized</a> our inclusion of node labels was to blame. For our diagnostic query, removing node labels <a href="https://twitter.com/dhimmel/status/662440818398007296">reduced</a> database hits by 1339 fold and runtime by 8 fold.</p>

<p>Michael Hunger, caretaker general of the neo4j community, <a href="https://twitter.com/mesirii/status/662463621335818240">explained</a>:</p>

<blockquote><p><code>size(pattern)</code> uses <code>node.getDegree()</code> if pattern only contains relationship type &amp; direction</p></blockquote>

<p>More explanation is <a href="http://neo4j.com/blog/neo4j-2-2-query-tuning/">available here</a>, but the essential insight is that by using only direction and relationship type to lookup node degree, we no longer need to lookup the label on the other end of each edge.</p>

<h2>Database changes</h2>

<p>To support this optimization, we need to ensure that no two metaedges that touch a common metanode have the same relationship type. Our current hetnet is noncompliant in this regard: for example, the three Gene Ontology metaedges all have kind 'participation':</p>

<ol><li>Gene–participation–Biological Process</li><li>Gene–participation–Molecular Function</li><li>Gene–participation–Cellular Component</li></ol>

<p>Therefore, we have implemented unique neo4j relationship types for each metaedge (<a href="https://github.com/dhimmel/hetio/commit/8107d783e1b86a33123b9fb3273edf51695b5e82">primary commit</a> and bugfixes <a href="https://github.com/dhimmel/hetio/commit/d4b5f4ef223a26eb5bce23245a10b403cccf9fe5">1</a> and <a href="https://github.com/dhimmel/hetio/commit/98121d64feeba829214652960322f00bffcc6b75">2</a>) by appending the standardized metaedge abbreviation to its kind. With this change, the relationship types for the Gene Ontology metaedges become:</p>

<ol><li><code>PARTICIPATION_BPpG</code></li><li><code>PARTICIPATION_GpMF</code></li><li><code>PARTICIPATION_CCpG</code></li></ol>

<h2>Example query</h2>

<p>With the updated database, the query for calculating the <em>DWPC</em> between goserelin and lung cancer for the <code>CcSEcCdGuD</code> metapath is:</p>

<pre><code class="no-highlight hljs">MATCH paths = (n0:Compound)-[:CAUSATION_CcSE]-(n1)-[:CAUSATION_CcSE]-(n2)-[:DOWNREGULATION_CdG]-(n3)-[:UPREGULATION_DuG]-(n4:Disease)
  WHERE n0.identifier = 'DB00014' // Goserelin
  AND n4.identifier = 'DOID:1324' // lung cancer
WITH [
  size((n0)-[:CAUSATION_CcSE]-()),
  size(()-[:CAUSATION_CcSE]-(n1)),
  size((n1)-[:CAUSATION_CcSE]-()),
  size(()-[:CAUSATION_CcSE]-(n2)),
  size((n2)-[:DOWNREGULATION_CdG]-()),
  size(()-[:DOWNREGULATION_CdG]-(n3)),
  size((n3)-[:UPREGULATION_DuG]-()),
  size(()-[:UPREGULATION_DuG]-(n4))
  ] AS degrees, paths
RETURN
  COUNT(paths) AS PC,
  sum(reduce(pdp = 1.0, d in degrees| pdp * d ^ -0.4)) AS DWPC</code></pre>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="7" data-comment-id="553">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="553"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#7" class="light">Nov. 26, 2015</a>
          
        </span>
      </div>

    

    <h1>Concurrent queries using py2neo</h1>

<p>As <a href="https://twitter.com/darthvader42/status/670154181064400897">explained</a> by Stefan Armbruster, a single cypher query is limited to a single core. However, multiple queries can be fulfilled in parallel:</p>

<blockquote><p>With current versions of Neo4j, a Cypher query traverses the graph in single threaded mode. Since most graph applications out there are concurrently used by multiple users, this model saturates the available cores. [<a href="http://stackoverflow.com/a/27578860/4651668">source</a>]</p></blockquote>

<p>Currently, we perform a separate query for each compound–disease–metapath combination. Depending on the number of compound–disease pairs and metapaths considered, we will need to compute between 1 million and 1 billion <em>DWPCs</em>. </p>

<p>Our software package for hetnets, <a href="https://github.com/dhimmel/hetio">hetio</a> <span class="citation">[<a href="https://doi.org/10.5281/zenodo.31763" class="citation" data-key="10.5281/zenodo.31763">1</a>]</span>, is built in python. Despite migrating to neo4j, we are still dependent on hetio for:</p>

<ul><li>metagraph operations</li><li>edge directionality</li><li>metapath abbreviation</li><li>constructing cypher queries</li></ul>

<p>Therefore, we're using python to construct and execute queries with the <a href="http://py2neo.org/2.0/">py2neo</a> package.</p>

<p>To enable concurrent queries, we initially used the <a href="https://docs.python.org/3.5/library/multiprocessing.html">multiprocessing</a> module (<a href="https://github.com/dhimmel/learn/blob/affb391ac35cd726e1377f08557b060f4098144f/neo4j-compute.ipynb">notebook</a>), which enables parallelism by creating subprocesses. However, subprocesses require substantial overhead. Since the majority of computation is performed outside of python by the neo4j sever, we switched to the <a href="https://docs.python.org/3.5/library/threading.html">threading</a> module (<a href="https://github.com/dhimmel/learn/blob/bfc0b4b3adef8ee9bab73676181b84298f6b16fe/neo4j-compute.ipynb">notebook</a>). Threading has less overhead than multiprocessing, but is limited to a single process of pure python. However, since the cypher query releases the <a href="https://docs.python.org/3.5/glossary.html#term-global-interpreter-lock">global interpreter lock</a>, the restriction to a single process is not a time-limiting step.</p>

<p>In the end, we used <a href="https://docs.python.org/3.5/library/concurrent.futures.html#threadpoolexecutor">concurrent.futures</a> to make threading easier (<a href="https://github.com/dhimmel/learn/blob/0867341ac64e5875390532e1aa31bd3b6f38c0ad/neo4j-compute.ipynb">notebook</a>). We encountered a <a href="https://github.com/nigelsmall/py2neo/issues/449">small hiccup</a> where our queue of queries waiting to be executed grew large and consumed substantial memory. We addressed the issue by postponing new query submission until the queue dropped below a given size.</p>

<p>Performing concurrent queries led to ~1000% processor usage by neo4j, equivalent to 10 cores at full load. This benchmark was performed on a 16 core machine running neo4j-community-2.3.1 on Ubuntu 15.10. Increasing the number of concurrent python workers above 16 did not increase the ~1000% usage figure. Let us know of any methods to increase processor saturation.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="8" data-comment-id="679">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="679"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#8" class="light">Jan. 29, 2016</a>
          
        </span>
      </div>

    

    <h1>2016 GraphGist Challenge</h1>

<p>Neo4j is hosting a GraphGist <a href="http://portal.graphgist.org/challenge/index.html" title="GraphGist Challenge">challenge</a>. GraphGists provide the <a href="http://portal.graphgist.org/about" title="What is a GraphGist?">following</a>:</p>

<blockquote><p>With Neo4j GraphGists you can describe and model your domain in a simple text file (AsciiDoc) and render it as a rich, interactive, database-backed page in any browser. It is perfect to document a specific domain, use-case, question or graph problem.</p></blockquote>

<p>This years competition is Star Wars themed — a theme we adhered to in <a href="http://portal.graphgist.org/graph_gists/drug-repurposing-by-hetnet-relationship-prediction-a-new-hope" title="Drug repurposing by hetnet relationship prediction: a new hope"><strong>our submission</strong></a>. To give you a taste, our prologue begins with:</p>

<blockquote><p>A long time ago in a galaxy far, far away…​. It is a dark time for drug discovery. The Empire spends over a billion dollars in R&amp;D per new drug approval. The process takes decades, 9 out of 10 attempts fail, and the cost has been doubling every 9 years since 1970. But, a small band of Rebel scientists pursue an alternative. Using public data and open source software, the Rebels are predicting new uses for existing drugs.</p></blockquote>

<p>Our goal in creating a submission was twofold. First, we're excited to interact with other members of the neo4j community who are doing complimentary work. Second, we designed the GraphGist to be a good introduction to our project and hetnet relationship prediction in general.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="9" data-comment-id="1506">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="1506"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#9" class="light">April 27, 2017</a>
          
        </span>
      </div>

    

    <h1>Literature on using Neo4j for biomedical hetnets</h1>

<p>Here we'll compile a list of studies that discuss using Neo4j or graph databases for hetnets related to biology or medicine.</p>

<ul><li><p>"Are graph databases ready for bioinformatics?" by <a href="/u/larsjuhljensen" class="username">@larsjuhljensen</a> as <a href="#note-219">noted above</a> <span class="citation">[<a href="https://doi.org/10.1093/bioinformatics/btt549" class="citation " data-key="10.1093/bioinformatics/btt549">1</a>]</span></p></li><li><p>"Representing and querying disease networks using graph databases" which we <a href="https://think-lab.github.io/d/216#2" class="citation thread-citation" data-key="10.15363/thinklab.d216">discussed here</a> <span class="citation">[<a href="https://doi.org/10.1186/s13040-016-0102-8" class="citation " data-key="10.1186/s13040-016-0102-8">2</a>]</span></p></li><li><p>"Use of Graph Database for the Integration of Heterogeneous Biological Data" which found that Neo4j was faster than MySQL for some common queries on a biological hetnet <span class="citation">[<a href="https://doi.org/10.5808/GI.2017.15.1.19" class="citation " data-key="10.5808/GI.2017.15.1.19">3</a>]</span>.</p></li></ul>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

    		
		</table>

    

    &nbsp;<br>
    <!-- <a class="btn btn-primary btn-lg" href="/signup?next=/discussion/using-the-neo4j-graph-database-for-hetnets/112&suppress_message=1" style>Join to Reply</a> -->


		<table class="references table-td-5" style="display:none;"><tr><td><a href='/doi/10.5281/zenodo.31763' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>1.</div></td><td><div class='doi-body' data-key='10.5281/zenodo.31763'><a href='/doi/10.5281/zenodo.31763' class='pub-title'>Hetnets in Python: hetio v0.1.0 initial release</a><div class='pub-body'>Daniel Himmelstein (2015) <i>Zenodo</i>. doi:<a class='pub-doi' href='https://doi.org/10.5281/zenodo.31763'>10.5281/zenodo.31763</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d102' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>2.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d102'><a href='/d/102' class='pub-title'>One network to rule them all</a><div class='pub-body'>Daniel Himmelstein, Lars Juhl Jensen (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d102'>10.15363/thinklab.d102</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d104' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>3.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d104'><a href='/d/104' class='pub-title'>Renaming ‘heterogeneous networks’ to a more concise and catchy term</a><div class='pub-body'>Daniel Himmelstein, Casey Greene, Sergio Baranzini (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d104'>10.15363/thinklab.d104</a></div></div></td></tr><tr><td><a href='/doi/10.1371/journal.pcbi.1004259' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>4.</div></td><td><div class='doi-body' data-key='10.1371/journal.pcbi.1004259'><a href='/doi/10.1371/journal.pcbi.1004259' class='pub-title'>Heterogeneous Network Edge Prediction: A Data Integration Approach to Prioritize Disease-Associated Genes</a><div class='pub-body'>Daniel S. Himmelstein, Sergio E. Baranzini (2015) <i>PLOS Computational Biology</i>. doi:<a class='pub-doi' href='https://doi.org/10.1371/journal.pcbi.1004259'>10.1371/journal.pcbi.1004259</a></div></div></td></tr><tr><td><a href='/doi/10.1093/bioinformatics/btt549' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>5.</div></td><td><div class='doi-body' data-key='10.1093/bioinformatics/btt549'><a href='/doi/10.1093/bioinformatics/btt549' class='pub-title'>Are graph databases ready for bioinformatics?</a><div class='pub-body'>C. T. Have, L. J. Jensen (2013) <i>Bioinformatics</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/bioinformatics/btt549'>10.1093/bioinformatics/btt549</a></div></div></td></tr><tr><td><a href='/doi/10.5281/zenodo.31763' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>6.</div></td><td><div class='doi-body' data-key='10.5281/zenodo.31763'><a href='/doi/10.5281/zenodo.31763' class='pub-title'>Hetnets in Python: hetio v0.1.0 initial release</a><div class='pub-body'>Daniel Himmelstein (2015) <i>Zenodo</i>. doi:<a class='pub-doi' href='https://doi.org/10.5281/zenodo.31763'>10.5281/zenodo.31763</a></div></div></td></tr><tr><td><a href='/doi/10.1093/bioinformatics/btt549' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>7.</div></td><td><div class='doi-body' data-key='10.1093/bioinformatics/btt549'><a href='/doi/10.1093/bioinformatics/btt549' class='pub-title'>Are graph databases ready for bioinformatics?</a><div class='pub-body'>C. T. Have, L. J. Jensen (2013) <i>Bioinformatics</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/bioinformatics/btt549'>10.1093/bioinformatics/btt549</a></div></div></td></tr><tr><td><a href='/doi/10.1186/s13040-016-0102-8' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>8.</div></td><td><div class='doi-body' data-key='10.1186/s13040-016-0102-8'><a href='/doi/10.1186/s13040-016-0102-8' class='pub-title'>Representing and querying disease networks using graph databases</a><div class='pub-body'>Artem Lysenko, Irina A. Roznovăţ, Mansoor Saqi, Alexander Mazein, Christopher J Rawlings, Charles Auffray (2016) <i>BioData Mining</i>. doi:<a class='pub-doi' href='https://doi.org/10.1186/s13040-016-0102-8'>10.1186/s13040-016-0102-8</a></div></div></td></tr><tr><td><a href='/doi/10.5808/GI.2017.15.1.19' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>9.</div></td><td><div class='doi-body' data-key='10.5808/GI.2017.15.1.19'><a href='/doi/10.5808/GI.2017.15.1.19' class='pub-title'>Use of Graph Database for the Integration of Heterogeneous Biological Data</a><div class='pub-body'>Byoung-Ha Yoon, Seon-Kyu Kim, Seon-Young Kim (2017) <i>Genomics &amp; Informatics</i>. doi:<a class='pub-doi' href='https://doi.org/10.5808/GI.2017.15.1.19'>10.5808/GI.2017.15.1.19</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d216' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>10.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d216'><a href='/d/216' class='pub-title'>Hosting Hetionet in the cloud: creating a public Neo4j instance</a><div class='pub-body'>Daniel Himmelstein (2016) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d216'>10.15363/thinklab.d216</a></div></div></td></tr></table>

    


	</td>
	<td class="thread-sidebar">


			

      <!-- <a class="btn btn-default follow-link btn-follow btn-lg is-following tt-top full-width" title="Follow discussion" data-id="112" data-ctype="thread" data-is-following="false">&nbsp;</a> -->

      
        <div class="status-box text-center"><span style="">Status:</span> 
  In Progress

</div>
      

      
        

          <div class="details">
            
            Labels
          </div>

          
            <a href='/p/rephetio/discussion?q=[neo4j]' class='pl pl-sidebar ' style='background-color:#6b6b6b'><span class='glyphicon glyphicon-tag'></span>&nbsp; neo4j</a>
          
        
      


				

					<div class="details">Views</div>
					<span style="font-size:20px;">657</span>

        

				
					<div class="details">Topics</div>
					<div style="margin-bottom:15px;">
		            <a href="/topics/database" class="tag">Database</a><a href="/topics/nosql" class="tag">NoSQL</a><a href="/topics/graph-database" class="tag">Graph Database</a><a href="/topics/graphs" class="tag">Graphs</a><a href="/topics/neo4j" class="tag">Neo4J</a>
					</div>
				


		
			<div class="details">Referenced by</div>
			<div style="font-size:12px;"><ul style="padding-left:16px !important;">
			
				<li>
					
					<a href='/d/216' class='thread-title'>Hosting Hetionet in the cloud: creating a public Neo4j instance</a>
				</li>
			
				<li>
					
					<a href='/d/113' class='thread-title'>Tracking project reuse, citation, and publicity</a>
				</li>
			
				<li>
					
					<a href='/doc/7/review' class='thread-title'><span class='inbox-icon tt-top glyphicon glyphicon-flash'></span>Report Review: Rephetio: Repurposing drugs on a hetnet</a>
				</li>
			
				<li>
					
					Research report: <a href='/p/rephetio/report'>Rephetio: Repurposing drugs on a hetnet</a>
				</li>
			
				<li>
					
					<a href='/d/115' class='thread-title'>Assessing the informativeness of features</a>
				</li>
			
				<li>
					
					<a href='/d/162' class='thread-title'>Data nomenclature: naming and abbreviating our network types</a>
				</li>
			
				<li>
					[<a href="/p/meta/discussion" style="color:#333333">meta</a>] 
					<a href='/d/163' class='thread-title'>Markdown bug: hyperlink with title attribute causes excision</a>
				</li>
			
				<li>
					
					<a href='/d/130' class='thread-title'>Licensing neo4j</a>
				</li>
			
				<li>
					
					<a href='/d/136' class='thread-title'>Permuting hetnets and implementing randomized edge swaps in cypher</a>
				</li>
			
				<li>
					
					Project: <a href='/p/rephetio'>Rephetio: Repurposing drugs on a hetnet</a>
				</li>
			
				<li>
					
					<a href='/d/134' class='thread-title'>Path exclusion conditions</a>
				</li>
			
				</ul>
			</div>
		

		
			<div class="details">Cite this as</div>
			<span style="font-size:12px;">
				Daniel Himmelstein (2015) Using the neo4j graph database for hetnets. <i>Thinklab.</i> doi:<a href='https://doi.org/10.15363/thinklab.d112'>10.15363/thinklab.d112</a>
			</span>
		

		<div class="details">License</div>
		<p><a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0;width:88px;height:31px;" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a></p>

		<div class="details">Share</div>
		
			
				<a class="share-button fb" onclick="javascript:void(window.open('https://www.facebook.com/sharer/sharer.php?u=https%3A//thinklab.org/discussion/using-the-neo4j-graph-database-for-hetnets/112','','width=600,height=460,menubar=no,status=no'));" href="#"></a>
<a class="share-button twitter" onclick="javascript:void(window.open('https://twitter.com/share?text=Using%20the%20neo4j%20graph%20database%20for%20hetnets&url=https%3A//thinklab.org/discussion/using-the-neo4j-graph-database-for-hetnets/112','','width=600,height=300,menubar=no,status=no'));" href="#"></a>
<a class="share-button google" onclick="javascript:void(window.open('https://plus.google.com/share?url=https%3A//thinklab.org/discussion/using-the-neo4j-graph-database-for-hetnets/112','','width=510,height=460,menubar=no,status=no'));" href="#"></a>

			
		

	</td>
</tr></table>


		</div>


		
	    	<div class="footer">
<!-- <div style="background:#dedede;">
	<div class="main-centered-div text-center" style="padding:20px;">
		<h4 style="margin-bottom:0px;font-weight:bold;color:gray;">Feedback? Questions? Email us: <a href="mailto:hello@thinklab.com">hello@thinklab.com</a></h4>
	</div>
</div> -->
<div class="main-footer">
	<div class="main-centered-div">

<table class="full-width"><tr>
	<td class="footer-nav">

        <div class="foot-head">Explore</div>
        <a href="/proposals">Proposals</a><br>
        <a href="/projects">Projects</a><br>
        <a href="/publications">Article discussion</a><br>
        <a href="/discussion">All discussion</a><br>
        <a href="/topics">Topics</a><br>
        <!-- <a href="/funders">Funders</a><br> -->
        <a href="/funding">Funding opportunities</a><br>

	</td>
	<td class="footer-nav">

        <!-- <div class="foot-head">Help &amp; feedback</div>	 -->
        <!-- <div class="foot-head">Intro</div>
        <a href="/benefits/for-science">Why</a><br>
        <a href="/how-it-works">How it works</a><br>

<br> -->

        <div class="foot-head">Community</div>
        <a href="/leaderboard">Leaderboard</a><br>
        <a href="/p/meta">Thinklab Meta</a><br>
				<a href="/help/faq">Help</a><br>
				<!-- <a href="/new-project">Start a proposal</a><br> -->
        <!-- <a href="/benefits/proposal">Benefits</a><br> -->
        <!-- <a href="/funding">Get funded</a><br> -->
        <!-- <a href="/participate">Become a contributor</a><br> -->
        <!-- <a href="/for-funders">For science funders</a><br> -->
        <!-- <a href="/for-institutions">For institutions</a><br> -->

	</td>
	<td class="footer-nav">



        <div class="foot-head">About</div>
        <a href="/about">Overview</a><br>
        <a href="/benefits">Benefits</a><br>
        <a href="/how-it-works">How it works</a><br>
        <a href="/about/story">Our story</a><br>
        <a href="/blog">Our blog</a><br>
        
        <br>

        <!-- <div class="foot-head">Help</div>
        <a href="/help">Help</a><br>
		 -->


	</td>
	<td class="footer-nav">
		<!-- <a href="http://twitter.com/thinklab" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/twitter_com.png" width="40"/></a>
		<a href="https://www.facebook.com/pages/Thinklab/310895039088185" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/facebook_com.png" width="40"/></a>
		<a href="https://plus.google.com/115578983466800753402/" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/plus_google_com.png" width="40"/></a>
		<a href="https://www.linkedin.com/company/thinklab-co" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/linkedin_com.png" width="40"/></a> -->
		<br><br>
		<span class="light" style='font-size:12px;'>&copy; 2015 Thinklab</span>

	</td>

</tr></table>

</div>
</div>
</div>

		

	</div>


  <script></script>
  <script>
  var MEDIA_URL = 'https://think-lab.s3.amazonaws.com/m/';
  var STATIC_URL = 'https://think-lab.s3.amazonaws.com/s/';
  var page_is_authenticated = false;
  var page_page = 'discussion';
  var page_profile_id = undefined;
</script>



	<!-- Note different script type. MUST come before MathJax.js -->
	<script type="text/x-mathjax-config">
	    MathJax.Hub.Config({
	        "HTML-CSS": { preferredFont: "TeX", availableFonts: ["STIX","TeX"],
	                        scale: 110, linebreaks: { automatic:true },
	                        EqnChunk: (MathJax.Hub.Browser.isMobile ? 10 : 50) },
	        tex2jax: { inlineMath: [ ["$$", "$$"], ["\\(","\\)"] ],
	                    displayMath: [ ["$$$","$$$"], ["\\[", "\\]"] ],
	                    processEscapes: true,}
	            });

	</script>
    <script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>





  <script src="https://code.jquery.com/jquery-1.10.2.min.js"></script>
  <script src="https://code.jquery.com/ui/1.11.4/jquery-ui.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" integrity="sha384-0mSbJDEHialfmuBBQP6A4Qrprq5OVfW37PRR3j5ELqxss1yVqOtnepnHVP9aJ7xS" crossorigin="anonymous"></script>
  <script src="https://think-lab.s3.amazonaws.com/s/dist/main.min.js"></script>
  



  
	
  
  
  
<script>
$(function() {
  recordHit('thread', 112);
});
</script>



</body>
</html>
