<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8" /><script type="text/javascript">window.NREUM||(NREUM={}),__nr_require=function(e,n,t){function r(t){if(!n[t]){var o=n[t]={exports:{}};e[t][0].call(o.exports,function(n){var o=e[t][1][n];return r(o||n)},o,o.exports)}return n[t].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<t.length;o++)r(t[o]);return r}({1:[function(e,n,t){function r(){}function o(e,n,t){return function(){return i(e,[c.now()].concat(u(arguments)),n?null:this,t),n?void 0:this}}var i=e("handle"),a=e(2),u=e(3),f=e("ee").get("tracer"),c=e("loader"),s=NREUM;"undefined"==typeof window.newrelic&&(newrelic=s);var p=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],d="api-",l=d+"ixn-";a(p,function(e,n){s[n]=o(d+n,!0,"api")}),s.addPageAction=o(d+"addPageAction",!0),s.setCurrentRouteName=o(d+"routeName",!0),n.exports=newrelic,s.interaction=function(){return(new r).get()};var m=r.prototype={createTracer:function(e,n){var t={},r=this,o="function"==typeof n;return i(l+"tracer",[c.now(),e,t],r),function(){if(f.emit((o?"":"no-")+"fn-start",[c.now(),r,o],t),o)try{return n.apply(this,arguments)}finally{f.emit("fn-end",[c.now()],t)}}}};a("setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(e,n){m[n]=o(l+n)}),newrelic.noticeError=function(e){"string"==typeof e&&(e=new Error(e)),i("err",[e,c.now()])}},{}],2:[function(e,n,t){function r(e,n){var t=[],r="",i=0;for(r in e)o.call(e,r)&&(t[i]=n(r,e[r]),i+=1);return t}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],3:[function(e,n,t){function r(e,n,t){n||(n=0),"undefined"==typeof t&&(t=e?e.length:0);for(var r=-1,o=t-n||0,i=Array(o<0?0:o);++r<o;)i[r]=e[n+r];return i}n.exports=r},{}],4:[function(e,n,t){n.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(e,n,t){function r(){}function o(e){function n(e){return e&&e instanceof r?e:e?f(e,u,i):i()}function t(t,r,o,i){if(!d.aborted||i){e&&e(t,r,o);for(var a=n(o),u=m(t),f=u.length,c=0;c<f;c++)u[c].apply(a,r);var p=s[y[t]];return p&&p.push([b,t,r,a]),a}}function l(e,n){v[e]=m(e).concat(n)}function m(e){return v[e]||[]}function w(e){return p[e]=p[e]||o(t)}function g(e,n){c(e,function(e,t){n=n||"feature",y[t]=n,n in s||(s[n]=[])})}var v={},y={},b={on:l,emit:t,get:w,listeners:m,context:n,buffer:g,abort:a,aborted:!1};return b}function i(){return new r}function a(){(s.api||s.feature)&&(d.aborted=!0,s=d.backlog={})}var u="nr@context",f=e("gos"),c=e(2),s={},p={},d=n.exports=o();d.backlog=s},{}],gos:[function(e,n,t){function r(e,n,t){if(o.call(e,n))return e[n];var r=t();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(e,n,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return e[n]=r,r}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],handle:[function(e,n,t){function r(e,n,t,r){o.buffer([e],r),o.emit(e,n,t)}var o=e("ee").get("handle");n.exports=r,r.ee=o},{}],id:[function(e,n,t){function r(e){var n=typeof e;return!e||"object"!==n&&"function"!==n?-1:e===window?0:a(e,i,function(){return o++})}var o=1,i="nr@id",a=e("gos");n.exports=r},{}],loader:[function(e,n,t){function r(){if(!x++){var e=h.info=NREUM.info,n=d.getElementsByTagName("script")[0];if(setTimeout(s.abort,3e4),!(e&&e.licenseKey&&e.applicationID&&n))return s.abort();c(y,function(n,t){e[n]||(e[n]=t)}),f("mark",["onload",a()+h.offset],null,"api");var t=d.createElement("script");t.src="https://"+e.agent,n.parentNode.insertBefore(t,n)}}function o(){"complete"===d.readyState&&i()}function i(){f("mark",["domContent",a()+h.offset],null,"api")}function a(){return E.exists&&performance.now?Math.round(performance.now()):(u=Math.max((new Date).getTime(),u))-h.offset}var u=(new Date).getTime(),f=e("handle"),c=e(2),s=e("ee"),p=window,d=p.document,l="addEventListener",m="attachEvent",w=p.XMLHttpRequest,g=w&&w.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:w,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var v=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-1044.min.js"},b=w&&g&&g[l]&&!/CriOS/.test(navigator.userAgent),h=n.exports={offset:u,now:a,origin:v,features:{},xhrWrappable:b};e(1),d[l]?(d[l]("DOMContentLoaded",i,!1),p[l]("load",r,!1)):(d[m]("onreadystatechange",o),p[m]("onload",r)),f("mark",["firstbyte",u],null,"api");var x=0,E=e(4)},{}]},{},["loader"]);</script><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info={"beacon":"bam.nr-data.net","queueTime":0,"licenseKey":"cd1101aa94","agent":"","transactionName":"bgcDMkBZW0NZBkdcWVdNJxNcW0FZVwscV1dKB08QW11CQxYTWlBBZgEOC19dW0QCEVtHU1gG","applicationID":"6369297","errorBeacon":"bam.nr-data.net","applicationTime":719}</script>
    <title>Processing GTEx for tissue-specific gene expression | Thinklab</title>
	  <link rel="shortcut icon" href="https://think-lab.s3.amazonaws.com/s/img/favicon.ico" type="image/x-icon" />
    <meta property="og:image" content="https://think-lab.s3.amazonaws.com/s/img" />
    
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
  <link rel="stylesheet" href="https://code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css" />

<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/highlightjs/github.css" />
<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/css/css.css" />
<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/css/bootstrap-multiselect.css" />
<!-- <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'> -->

    
    
    
    
  
    <meta name="citation_doi" content="10.15363/thinklab.d82">
    <meta name="citation_title" content="Processing GTEx for tissue-specific gene expression">
    <meta name="citation_online_date" content="June 17, 2015">
    <meta name="citation_author" content="Daniel Himmelstein, Chris Mungall, Frederic Bastian">
    <meta name="citation_journal_title" content="Thinklab">
    <meta name="citation_publisher" content="Thinklab">
  

    


	<!-- GoogleAnalytics -->
	<script>
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
	(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
	m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

	ga('create', 'UA-48550230-1', 'thinklab.org');
	ga('require', 'displayfeatures');
	
	ga('send', 'pageview');

	</script>






</head>
<body >
  <form id="form_action" method="post">
    <input type='hidden' name='csrfmiddlewaretoken' value='JC3hnVSWzcwBOS226QTq2eGyHXj1YAM6' />
</form>

	

	<div class="body-wrapper">

		


<!-- <div style="height:5px;background-image:url('https://think-lab.s3.amazonaws.com/s/img/top-bg.gif');"></div> -->
<div class="gradient">
	<div class="main-centered-div header-container">

<table class="full-width" style="height:60px;"><tr>
	<td style="width:200px;vertical-align:middle;padding-bottom:15px;">
		<a href="/" style="font-weight:normal;color:#212121;"><img src="https://think-lab.s3.amazonaws.com/s/img/logo.png" width="160"></a>
	</td><td style="vertical-align:middle;text-align:left;padding-top:6px;width:100%;padding-left:40px;">
		<ul id="not-search-block"  class="main-header">


			


				<li class="dropdown">
					<a class="dropdown-toggle" data-toggle="dropdown" href="#">Browse <span class="caret"></span></a>
					<ul class="dropdown-menu" role="menu">
						<li><a href="/proposals">Proposals</a></li>
						<li><a href="/projects">Projects</a></li>
						<li><a href="/publications">Article discussion</a></li>
						<li><a href="/discussion">All discussion</a></li>
					</ul>
				</li>
				<li><a href="/leaderboard">Leaderboard</a></li>

			
				<li><a href="/about">About</a></li>
			


		</ul>

    <ul id="search-block" class="main-header" style="display: none; margin-right: 30px;">
        <li class="full-width">
            <form method="get" action="/search">
			<div class="input-group full-width">
				<input name="q" type="text" class="form-control search-input">
			</div>
            </form>
        </li>
    </ul>

	</td><td style="vertical-align:middle;text-align:right;padding-top:6px; min-width: 1%; white-space: nowrap;">
		<ul class="main-header pull-right" style="padding-left: 0;">
    		
                <!-- <li><a href="/login" style="color:silver;">Login</a></li>
                <li style="margin-right:0"><a class="btn btn-default btn-sm btn-upper" href="/signup">Join</a></li> -->
            

        </ul>
    </ul>
	</td>
</tr></table>

<!-- <div id="global-search-div" class="main-centered-div" style="display:none;">
	<input type="text" value="" class="thread-search form-control" placeholder="Search" data-action="/discussion">
</div> -->

	</div>
</div>

		
	  



	  


		

		<div class="dark-head"><div class="main-centered-div">

			

		</div></div>
		
		<div class="main-centered-div" style="margin-top:40px;padding-bottom:340px">
	    

<div style="margin-top:-20px; margin-bottom:20px;">

    <a class="side-box" href="/p/rephetio" style="background-color:#ebfdff;"><div class="side-type">Project:</div> <div class="side-title">Rephetio: Repurposing drugs on a hetnet [rephetio]</div></a><br>



    <a class="side-box" href="/doi/10.1126/science.aaa0355" style="background-color:#f0ffec;"><div class="side-type">Publication:</div> <div class="side-title">The human transcriptome across tissues and individuals</div></a><br>

</div>

<table style="margin-top:0px;"><tr>
	<td class="thread-table-content">

		<table class="full-width">
			
<tr class="comment-tr target-hl" id="1" data-comment-id="274">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="274"></a>

    

			<h2 class="page-title" style="margin-bottom:5px;margin-top:0px !important;">Processing GTEx for tissue-specific gene expression</h2>

			<div style="margin-top:5px;color:gray;margin-bottom:20px;font-size:13px;">
				<a style="font-weight:normal;color:#212121" href="/u/dhimmel"><strong>Daniel Himmelstein</strong></a>
				<span class="is-researcher">Researcher</span>
				&nbsp; <a href="#1" class="light">June 17, 2015</a>
				
			</div>

      

          <hr>
		

    <p>The Genotype-Tissue Expression project (<a href="http://www.gtexportal.org/home/">GTEx</a>) RNA-sequenced <span class="citation">[<a href="https://doi.org/10.1126/science.aaa0355" class="citation" data-key="10.1126/science.aaa0355">1</a>]</span>:</p>

<blockquote><p>1641 samples from 175 individuals representing 43 sites: 29 solid organ tissues, 11 brain subregions, whole blood, and two cell lines: Epstein-Barr virus–transformed lymphocytes (LCL) and cultured fibroblasts from skin.</p></blockquote>

<p>The data is <a href="http://www.gtexportal.org/home/datasets2">available online</a>. Specifically, we are interested in the <code>GTEx_Analysis_V4_RNA-seq_RNA-SeQCv1.1.8_gene_rpkm.gct.gz</code> file that contains RPKM expression values for each sample. We would like to calculate a single expression value for each gene-tissue pair. Expression values should be comparable across tissues, not just within tissues.</p>

<p>We will post our questions here. Advice appreciated.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="2" data-comment-id="275">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="275"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#2" class="light">June 17, 2015</a>
          
        </span>
      </div>

    

    <h1>Mapping GTEx sites to Uberon and CL</h1>

<p>We <a href="//thinklab.com/discussion/tissue-node/41">are using</a> <a href="https://uberon.github.io/">Uberon</a> <span class="citation">[<a href="https://doi.org/10.1186/gb-2012-13-1-r5" class="citation" data-key="10.1186/gb-2012-13-1-r5">1</a>]</span> terms to identify anatomical structures and <a href="https://github.com/obophenotype/cell-ontology">Cell Ontology</a> (CL) <span class="citation">[<a href="https://doi.org/10.1186/gb-2005-6-2-r21" class="citation" data-key="10.1186/gb-2005-6-2-r21">2</a>]</span> terms to identify cell types. Thus, we need to map GTEx sites to their corresponding ontology terms.</p>

<p>From the sample attribute documentation (<code>GTEx_Data_V4_Annotations_SampleAttributesDS.txt</code>), we <a href="https://github.com/dhimmel/gtex/blob/8acb90fea2613f8b27814401e361bb6ae4b29078/gtex.ipynb">identified</a> 54 sites using the <code>SMTSD</code> attribute. I have mapped about half of the sites to Uberon. The remainder would benefit from a skilled anatomist or GTEx consortium member.</p>

<p><strong>Bounty:</strong> Add or correct <a href="https://docs.google.com/spreadsheets/d/1aXm_RvD4aywXRpQdxVxjBxwPAR_YBBq9f3xkHQaLZXo/edit?usp=sharing">our mappings using this spreadsheet</a> and put your Thinklab username. Then leave a comment in this discussion, and we will rate its value <span class="math">$$\geq $4 \times n$$</span>, where <em>n</em> is the number of mappings provided.</p>

<p>Some additional sample site information is available in Table S1 (p. 58) of the <a href="//www.sciencemag.org/content/suppl/2015/05/06/348.6235.660.DC1/Mele.SM.pdf">supplement</a>.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="3" data-comment-id="281">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="281"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#3" class="light">June 19, 2015</a>
          
        </span>
      </div>

    

    <h1>Handing over GTEx processing responsibility to Bgee</h1>

<p>We have <a href="http://thinklab.com/discussion/tissue-specific-gene-expression-resources/81#278">decided to use Bgee</a> for tissue-specific transcript presence, over-, and under-expression. Bgee doesn't currently include GTEx data but <a href="https://github.com/obophenotype/uberon/issues/704#issuecomment-113134400">will soon</a>.</p>

<p>Therefore, we are not going to proceed with GTEx data directly for this project. However, we did already process the data into a usable gene × site format (<a href="https://github.com/dhimmel/gtex/blob/93c3800cef1ced48b100d57b8d6efab831f8533b/gtex.ipynb">notebook</a>, <a href="https://github.com/dhimmel/gtex/blob/93c3800cef1ced48b100d57b8d6efab831f8533b/data/expression-SMTSD.tsv.gz">download</a>). We converted genes to Entrez GeneIDs. The sites are still in GTEx strings rather than Uberon terms. Expression values are log-transformed. Check out the notebook for a visualization of tissue-specific transcript abundance distributions.</p>

<p><strong>Bounty:</strong> we will keep the GTEx–Uberon mapping bounty going until June 25, 2015 because these mappings will help the Bgee team and eventually us. <a href="/u/chrismungall" class="username">@chrismungall</a>, do you want to add a comment here, so you can get rewarded for your 3 mappings?</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="4" data-comment-id="282">
  <td class="comment-profile-avatar">
		<a href="/u/chrismungall"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/s/img/p_default_medium.png' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="282"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/chrismungall">Chris Mungall</a>
				
				<span class="light">&nbsp;<a href="#4" class="light">June 19, 2015</a>
          
        </span>
      </div>

    

    <p>I think the Bgee team will do a great job. Just a few general comments:</p>

<p>most of the GTEx terms correspond to 'wild-type' structures as can be found in uberon/cl. There are however, two subclasses of skin: exposed and unexposed. We <em>could</em> add these as subclasses in uberon, but this would be unusual. It would be better to either post-compose these, or to have some kind of ancillary 'sample' ontology where this is composed.</p>

<p>For 'hippocampus', the safest option is to map to the broadest term, 'hippocampal formation', but if it can be shown than the GTEx sample excludes bits of the dentate gyrus then the more specific 'ammons horn' can be used.</p>

<p>Finally, it's always best when ontologies are used prospectively rather than retrospectively, maybe future rounds of GTEx will follow the lead of FANTOM5 and ENCODE in doing this.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="5" data-comment-id="283">
  <td class="comment-profile-avatar">
		<a href="/u/fbastian"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/s/img/p_default_medium.png' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="283"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/fbastian">Frederic Bastian</a>
				
				<span class="light">&nbsp;<a href="#5" class="light">June 19, 2015</a>
          
        </span>
      </div>

    

    <p>IMO, the "exposed/unexposed" state is an experimental factor, and should not be annotated using a new anatomical term, it should be an additional "column" in an annotation (using, e.g., EFO).</p>

<p>Daniel, we will discuss next week during our lab meeting the timescale to annotate GTEx data, but as you said, it should be fast. Our problem is that we requested access to the data, and are waiting for an answer. </p>

<p>We could start working on the mapping you have, but we'd rather go through the information for each sample, to check for normality, etc. This is how we usually do. <br>(do they provide GEO or SRA identifiers for the samples BTW?)</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
<li id="note-130" class="inline-note toggle-parent" data-note-id="130">
	<p><a href="/u/dhimmel">Daniel Himmelstein</a>: What does SRA stand for?
		
	</p>
	
	<table style="display:none;"></table>
</li>

<li id="note-131" class="inline-note toggle-parent" data-note-id="131">
	<p><a href="/u/fbastian">Frederic Bastian</a>: Sequence Read Archive (http://www.ncbi.nlm.nih.gov/sra). This is where we usually download the raw data from. But I think GTEx keep them private, for their "on demand" data sharing policy...
		
	</p>
	
	<table style="display:none;"></table>
</li>

				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="6" data-comment-id="285">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="285"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#6" class="light">June 19, 2015</a>
          
        </span>
      </div>

    

    <blockquote><p>do they provide GEO or SRA identifiers for the samples BTW?</p></blockquote>

<p><a href="/u/fbastian" class="username">@fbastian</a>, I do not see any other sample identifiers than <code>SAMPID</code> (GTEx Public Sample ID) in the <a href="http://www.gtexportal.org/static/datasets/gtex_analysis_v4/annotations/GTEx_Data_V4_Annotations_SampleAttributesDD.xlsx">sample attributes documentation spreadsheet</a>. The IDs are formatted like <code>GTEX-N7MS-0007-SM-2D7W1</code> — not sure whether that corresponds with other databases.</p>

<blockquote><p>We could start working on the mapping you have, but we'd rather go through the information for each sample, to check for normality, etc.</p></blockquote>

<p><a href="/u/fbastian" class="username">@fbastian</a>, do whatever is best for you! And let it be known that your painstaking and thorough integration efforts are appreciated.</p>

<blockquote><p>There are however, two subclasses of skin: exposed and unexposed.</p></blockquote>

<p><a href="/u/chrismungall" class="username">@chrismungall</a>, by post-compose do you mean contacting GTEx and asking for more details on the skin sample sites? That seems the best to me as I assume the sample collectors had specific instructions. The skin sites are specified as <em>suprapubic</em> for sun unexposed and <em>lower leg</em> for sun exposed.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="7" data-comment-id="288">
  <td class="comment-profile-avatar">
		<a href="/u/fbastian"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/s/img/p_default_medium.png' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="288"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/fbastian">Frederic Bastian</a>
				
				<span class="light">&nbsp;<a href="#7" class="light">June 20, 2015</a>
          
        </span>
      </div>

    

    <p><a href="/u/dhimmel" class="username">@dhimmel</a>, thanks for the information. Did your lab formally request access to the data to get the actual annotations? (GTEx_Data_V4_Annotations_SampleAttributesDS.txt in your notebook)</p>

<p>Otherwise, Chris is speaking about ontology term post-composition, a way of creating a new ontology concept on-the-fly, that doesn't have any identifier or IRI ("anonymous class expression"), and that is made of the "composition" of several other terms. That would allow you to create on the fly a new concept for "exposed skin".</p>

<p>See for instance, in zebrafish ontology: <a href="https://zfin.org/action/ontology/post-composed-term-detail?superTermID=ZFA:0001117&amp;subTermID=ZFA:0000155">https://zfin.org/action/ontology/post-composed-term-detail?superTermID=ZFA:0001117&amp;subTermID=ZFA:0000155</a><br>There is no term "post-vent region somite" in the ontology, but the concept is represented by using the existing terms "post-vent region" and "somite".</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="8" data-comment-id="290">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="290"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#8" class="light">June 20, 2015</a>
          
        </span>
      </div>

    

    <p><a href="/u/fbastian" class="username">@fbastian</a>, <code>GTEx_Data_V4_Annotations_SampleAttributesDS.txt</code> is available from the GTEx <a href="http://www.gtexportal.org/home/datasets2">download page</a> which requires an account. However, <a href="http://www.gtexportal.org/static/datasets/gtex_analysis_v4/annotations/GTEx_Data_V4_Annotations_SampleAttributesDS.txt">this direct link</a> circumvents the login page. How long has it been since you submitted the data access request?</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="9" data-comment-id="291">
  <td class="comment-profile-avatar">
		<a href="/u/fbastian"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/s/img/p_default_medium.png' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="291"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/fbastian">Frederic Bastian</a>
				
				<span class="light">&nbsp;<a href="#9" class="light">June 20, 2015</a>
          
        </span>
      </div>

    

    <p>Not very long, a week or so.</p>

<p>Just to anticipate, do they provide more detailed information somewhere else, like, e.g., <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM81022">http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM81022</a> ? (extraction protocols, detailed information about the anatomical structure, etc)</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="10" data-comment-id="294">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="294"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#10" class="light">June 20, 2015</a>
          
        </span>
      </div>

    

    <blockquote><p>do they provide more detailed information somewhere else</p></blockquote>

<p>No idea, I did email the GTEx support with a link to this thread, so perhaps they'll provide some clarification.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="11" data-comment-id="296">
  <td class="comment-profile-avatar">
		<a href="/u/fbastian"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/s/img/p_default_medium.png' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="296"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/fbastian">Frederic Bastian</a>
				
				<span class="light">&nbsp;<a href="#11" class="light">June 22, 2015</a>
          
        </span>
      </div>

    

    <p>FYI, our curator Anne Niknejad has started editing your mapping, she will also create the issues on the Uberon tracker to request new terms.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="12" data-comment-id="306">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="306"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#12" class="light">June 30, 2015</a>
          
        </span>
      </div>

    

    <p><a href="/u/fbastian" class="username">@fbastian</a> and Anne Niknejad.</p>

<p>On June 17th I emailed <code>gtex-help@broadinstitute.org</code> asking for a GTEX–Uberon mapping. Today, Tim Sullivan responded and attached <a href="https://github.com/dhimmel/gtex/blob/30096fb519efba939eeb0c5681ba20ad8d43660a/download/GTEx_Uberon_Terms.xlsx">this mapping file</a>.</p>

<p>He didn't mention the methodology used, but you may want to crosscheck your work.</p>

<p>I've reproduced Tim's mapping below for quick reference:</p>

<table class="table markdown-table"><thead><tr><th>Tissue Site Detail</th><th>Uberon Code</th><th>Uberon Term</th></tr></thead><tbody><tr><td>Adipose - Subcutaneous</td><td>0002190</td><td>subcutaneous adipose tissue</td></tr><tr><td>Adipose - Visceral (Omentum)</td><td>0010414</td><td>omental fat pad</td></tr><tr><td>Adrenal Gland</td><td>0002369</td><td>adrenal gland</td></tr><tr><td>Artery - Aorta</td><td>0001496</td><td>ascending aorta</td></tr><tr><td>Artery - Coronary</td><td>0001621</td><td>coronary artery</td></tr><tr><td>Artery - Tibial</td><td>0001323</td><td>tibial nerve</td></tr><tr><td>Artery - Tibial</td><td>0007610</td><td>tibial artery</td></tr><tr><td>Bladder</td><td>0001255</td><td>urinary bladder</td></tr><tr><td>Brain - Amygdala</td><td>0001876</td><td>amygdala</td></tr><tr><td>Brain - Anterior cingulate cortex (BA24)</td><td>0009835</td><td>anterior cingulate cortex</td></tr><tr><td>Brain - Caudate (basal ganglia)</td><td>0001873</td><td>caudate nucleus</td></tr><tr><td>Brain - Cerebellar Hemisphere</td><td>0002037</td><td>cerebellum</td></tr><tr><td>Brain - Cerebellum</td><td>0002037</td><td>cerebellum</td></tr><tr><td>Brain - Cortex</td><td>0001870</td><td>frontal cortex</td></tr><tr><td>Brain - Frontal Cortex (BA9)</td><td>0009834</td><td>dorsolateral prefrontal cortex</td></tr><tr><td>Brain - Hippocampus</td><td>0001954</td><td>Ammon's horn</td></tr><tr><td>Brain - Hypothalamus</td><td>0001898</td><td>hypothalamus</td></tr><tr><td>Brain - Nucleus accumbens (basal ganglia)</td><td>0001882</td><td>nucleus accumbens</td></tr><tr><td>Brain - Putamen (basal ganglia)</td><td>0001874</td><td>putamen</td></tr><tr><td>Brain - Spinal cord (cervical c-1)</td><td>0006469</td><td>first cervical spinal cord segment</td></tr><tr><td>Brain - Substantia nigra</td><td>0002038</td><td>substantia nigra</td></tr><tr><td>Breast - Mammary Tissue</td><td>0008367</td><td>breast epithelium</td></tr><tr><td>Cells - EBV-transformed lymphocytes</td><td>EFO_0000572</td><td>lymphoblast</td></tr><tr><td>Cells - Leukemia cell line (CML)</td><td>EFO_0002067</td><td>K562</td></tr><tr><td>Cells - Transformed fibroblasts</td><td>EFO_0000496</td><td>fibroblast</td></tr><tr><td>Cervix - Ectocervix</td><td>0012249</td><td>ectocervix</td></tr><tr><td>Cervix - Endocervix</td><td>0000458</td><td>endocervix</td></tr><tr><td>Colon - Sigmoid</td><td>0001159</td><td>sigmoid colon</td></tr><tr><td>Colon - Transverse</td><td>0001157</td><td>transverse colon</td></tr><tr><td>Esophagus - Gastroesophageal Junction</td><td>0004550</td><td>gastroesophageal sphincter</td></tr><tr><td>Esophagus - Mucosa</td><td>0006920</td><td>esophagus squamous epithelium</td></tr><tr><td>Esophagus - Muscularis</td><td>0004648</td><td>esophagus muscularis mucosa</td></tr><tr><td>Fallopian Tube</td><td>0003889</td><td>fallopian tube</td></tr><tr><td>Heart - Atrial Appendage</td><td>0006631</td><td>right atrium auricular region</td></tr><tr><td>Heart - Left Ventricle</td><td>0006566</td><td>left ventricle myocardium</td></tr><tr><td>Kidney - Cortex</td><td>0001225</td><td>cortex of kidney</td></tr><tr><td>Liver</td><td>0001114</td><td>right lobe of liver</td></tr><tr><td>Lung</td><td>0008952</td><td>upper lobe of left lung</td></tr><tr><td>Minor Salivary Gland</td><td>0006330</td><td>anterior lingual gland</td></tr><tr><td>Muscle - Skeletal</td><td>0011907</td><td>gastrocnemius medialis</td></tr><tr><td>Nerve - Tibial</td><td>0001323</td><td>tibial nerve</td></tr><tr><td>Nerve - Tibial</td><td>0007610</td><td>tibial artery</td></tr><tr><td>Ovary</td><td>0002119</td><td>left ovary</td></tr><tr><td>Pancreas</td><td>0001150</td><td>body of pancreas</td></tr><tr><td>Pituitary</td><td>0000007</td><td>pituitary gland</td></tr><tr><td>Prostate</td><td>0002367</td><td>prostate gland</td></tr><tr><td>Skin - Not Sun Exposed (Suprapubic)</td><td>0001416</td><td>skin of abdomen</td></tr><tr><td>Skin - Sun Exposed (Lower leg)</td><td>0001511</td><td>skin of leg</td></tr><tr><td>Small Intestine - Terminal Ileum</td><td>0001211</td><td>Peyer's patch</td></tr><tr><td>Spleen</td><td>0002106</td><td>spleen</td></tr><tr><td>Stomach</td><td>0000945</td><td>stomach</td></tr><tr><td>Testis</td><td>0000473</td><td>testis</td></tr><tr><td>Thyroid</td><td>0002046</td><td>thyroid gland</td></tr><tr><td>Uterus</td><td>0000995</td><td>uterus</td></tr><tr><td>Vagina</td><td>0000996</td><td>vagina</td></tr><tr><td>Whole Blood</td><td>0013756</td><td>venous blood</td></tr></tbody></table>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="13" data-comment-id="308">
  <td class="comment-profile-avatar">
		<a href="/u/chrismungall"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/s/img/p_default_medium.png' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="308"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/chrismungall">Chris Mungall</a>
				
				<span class="light">&nbsp;<a href="#13" class="light">July 1, 2015</a>
          
        </span>
      </div>

    

    <p>Some seem slightly more specific than the label suggests - sometimes the increased specificity is trivial (ie their ovary sample was from a left ovary), sometimes relevant (their representative skeletal muscle sample was from gastrocnemius medialis, the esophagus mucosa sample was taken from the epithelium rather than lamina propria).</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="14" data-comment-id="333">
  <td class="comment-profile-avatar">
		<a href="/u/fbastian"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/s/img/p_default_medium.png' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="333"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/fbastian">Frederic Bastian</a>
				
				<span class="light">&nbsp;<a href="#14" class="light">July 17, 2015</a>
          
        </span>
      </div>

    

    <p>Actually, for some mappings we needed to request for new terms in Uberon, e.g.: <a href="https://github.com/obophenotype/uberon/issues/725">https://github.com/obophenotype/uberon/issues/725</a></p>

<p>Things will be slow until mid-August on our side. </p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="15" data-comment-id="426">
  <td class="comment-profile-avatar">
		<a href="/u/fbastian"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/s/img/p_default_medium.png' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="426"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/fbastian">Frederic Bastian</a>
				
				<span class="light">&nbsp;<a href="#15" class="light">Sept. 7, 2015</a>
          
        </span>
      </div>

    

    <p>To keep you posted: we were recently given access to the GTEx data, so we have started annotating/analyzing the data. We hope to have a new release of Bgee including these data in about 2 months.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
<li id="note-179" class="inline-note toggle-parent" data-note-id="179">
	<p><a href="/u/dhimmel">Daniel Himmelstein</a>: Exciting, thanks for the update. On another note, I wasn't able to find any licensing information on the Bgee website, which technically means <a href="http://choosealicense.com/no-license/">all rights reserved</a>. We're <a href="http://thinklab.com/discussion/integrating-resources-with-disparate-licensing-into-an-open-network/107#3" class="citation thread-citation" data-key="10.15363/thinklab.d107">tying to compile</a> licenses for each resource we use. It would be great if you could add a license.
		
	</p>
	
	<table style="display:none;"><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d107' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>1.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d107'><a href='/d/107' class='pub-title'>Integrating resources with disparate licensing into an open network</a><div class='pub-body'>Daniel Himmelstein, Lars Juhl Jensen, MacKenzie Smith, Katie Fortney, Caty Chung (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d107'>10.15363/thinklab.d107</a></div></div></td></tr></table>
</li>

<li id="note-180" class="inline-note toggle-parent" data-note-id="180">
	<p><a href="/u/fbastian">Frederic Bastian</a>: Yep, I wanted to reply to you after checking all our datasources. I think we are going to use a cc-by, but I'll contact you when I can confirm that for sure.
		
	</p>
	
	<table style="display:none;"></table>
</li>

				
			</ul>

      

		</div>

		


  </td>
</tr>

    		
		</table>

    

    &nbsp;<br>
    <!-- <a class="btn btn-primary btn-lg" href="/signup?next=/discussion/processing-gtex-for-tissue-specific-gene-expression/82&suppress_message=1" style>Join to Reply</a> -->


		<table class="references table-td-5" style="display:none;"><tr><td><a href='/doi/10.1126/science.aaa0355' class='pub-comments tt-right' title='Article comments'>15</a></td><td><div class='pub-order'>1.</div></td><td><div class='doi-body' data-key='10.1126/science.aaa0355'><a href='/doi/10.1126/science.aaa0355' class='pub-title'>The human transcriptome across tissues and individuals</a><div class='pub-body'>M. Mele, P. G. Ferreira, F. Reverter, D. S. DeLuca, J. Monlong, M. Sammeth, T. R. Young, J. M. Goldmann, D. D. Pervouchine, T. J. Sullivan, R. Johnson, A. V. Segre, S. Djebali, A. Niarchou, T. G. Consortium, F. A. Wright, T. Lappalainen, M. Calvo, G. Getz, E. T. Dermitzakis, K. G. Ardlie, R. Guigo (2015) <i>Science</i>. doi:<a class='pub-doi' href='https://doi.org/10.1126/science.aaa0355'>10.1126/science.aaa0355</a></div></div></td></tr><tr><td><a href='/doi/10.1186/gb-2012-13-1-r5' class='pub-comments tt-right' title='Article comments'>16</a></td><td><div class='pub-order'>2.</div></td><td><div class='doi-body' data-key='10.1186/gb-2012-13-1-r5'><a href='/doi/10.1186/gb-2012-13-1-r5' class='pub-title'>Uberon, an integrative multi-species anatomy ontology</a><div class='pub-body'>Christopher J Mungall, Carlo Torniai, Georgios V Gkoutos, Suzanna E Lewis, Melissa A Haendel (2012) <i>Genome Biol</i>. doi:<a class='pub-doi' href='https://doi.org/10.1186/gb-2012-13-1-r5'>10.1186/gb-2012-13-1-r5</a></div></div></td></tr><tr><td><a href='/doi/10.1186/gb-2005-6-2-r21' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>3.</div></td><td><div class='doi-body' data-key='10.1186/gb-2005-6-2-r21'>None</div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d41' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>4.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d41'><a href='/d/41' class='pub-title'>Tissue Node</a><div class='pub-body'>Venkat Malladi, Daniel Himmelstein, Chris Mungall (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d41'>10.15363/thinklab.d41</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d81' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>5.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d81'><a href='/d/81' class='pub-title'>Tissue-specific gene expression resources</a><div class='pub-body'>Daniel Himmelstein, Frederic Bastian (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d81'>10.15363/thinklab.d81</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d107' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>6.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d107'><a href='/d/107' class='pub-title'>Integrating resources with disparate licensing into an open network</a><div class='pub-body'>Daniel Himmelstein, Lars Juhl Jensen, MacKenzie Smith, Katie Fortney, Caty Chung (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d107'>10.15363/thinklab.d107</a></div></div></td></tr></table>

    


	</td>
	<td class="thread-sidebar">


			

      <!-- <a class="btn btn-default follow-link btn-follow btn-lg is-following tt-top full-width" title="Follow discussion" data-id="82" data-ctype="thread" data-is-following="false">&nbsp;</a> -->

      
        <div class="status-box text-center"><span style="">Status:</span> 
  Declined

</div>
      

      
        
      


				

					<div class="details">Views</div>
					<span style="font-size:20px;">686</span>

        

				
					<div class="details">Topics</div>
					<div style="margin-bottom:15px;">
		            <a href="/topics/gene-expression" class="tag">Gene Expression</a><a href="/topics/transcription" class="tag">Transcription</a><a href="/topics/gtex" class="tag">GTEx</a><a href="/topics/rna-seq" class="tag">RNA-Seq</a>
					</div>
				


		

		
			<div class="details">Cite this as</div>
			<span style="font-size:12px;">
				Daniel Himmelstein, Chris Mungall, Frederic Bastian (2015) Processing GTEx for tissue-specific gene expression. <i>Thinklab.</i> doi:<a href='https://doi.org/10.15363/thinklab.d82'>10.15363/thinklab.d82</a>
			</span>
		

		<div class="details">License</div>
		<p><a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0;width:88px;height:31px;" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a></p>

		<div class="details">Share</div>
		
			
				<a class="share-button fb" onclick="javascript:void(window.open('https://www.facebook.com/sharer/sharer.php?u=https%3A//thinklab.org/discussion/processing-gtex-for-tissue-specific-gene-expression/82','','width=600,height=460,menubar=no,status=no'));" href="#"></a>
<a class="share-button twitter" onclick="javascript:void(window.open('https://twitter.com/share?text=Processing%20GTEx%20for%20tissue-specific%20gene%20expression&url=https%3A//thinklab.org/discussion/processing-gtex-for-tissue-specific-gene-expression/82','','width=600,height=300,menubar=no,status=no'));" href="#"></a>
<a class="share-button google" onclick="javascript:void(window.open('https://plus.google.com/share?url=https%3A//thinklab.org/discussion/processing-gtex-for-tissue-specific-gene-expression/82','','width=510,height=460,menubar=no,status=no'));" href="#"></a>

			
		

	</td>
</tr></table>


		</div>


		
	    	<div class="footer">
<!-- <div style="background:#dedede;">
	<div class="main-centered-div text-center" style="padding:20px;">
		<h4 style="margin-bottom:0px;font-weight:bold;color:gray;">Feedback? Questions? Email us: <a href="mailto:hello@thinklab.com">hello@thinklab.com</a></h4>
	</div>
</div> -->
<div class="main-footer">
	<div class="main-centered-div">

<table class="full-width"><tr>
	<td class="footer-nav">

        <div class="foot-head">Explore</div>
        <a href="/proposals">Proposals</a><br>
        <a href="/projects">Projects</a><br>
        <a href="/publications">Article discussion</a><br>
        <a href="/discussion">All discussion</a><br>
        <a href="/topics">Topics</a><br>
        <!-- <a href="/funders">Funders</a><br> -->
        <a href="/funding">Funding opportunities</a><br>

	</td>
	<td class="footer-nav">

        <!-- <div class="foot-head">Help &amp; feedback</div>	 -->
        <!-- <div class="foot-head">Intro</div>
        <a href="/benefits/for-science">Why</a><br>
        <a href="/how-it-works">How it works</a><br>

<br> -->

        <div class="foot-head">Community</div>
        <a href="/leaderboard">Leaderboard</a><br>
        <a href="/p/meta">Thinklab Meta</a><br>
				<a href="/help/faq">Help</a><br>
				<!-- <a href="/new-project">Start a proposal</a><br> -->
        <!-- <a href="/benefits/proposal">Benefits</a><br> -->
        <!-- <a href="/funding">Get funded</a><br> -->
        <!-- <a href="/participate">Become a contributor</a><br> -->
        <!-- <a href="/for-funders">For science funders</a><br> -->
        <!-- <a href="/for-institutions">For institutions</a><br> -->

	</td>
	<td class="footer-nav">



        <div class="foot-head">About</div>
        <a href="/about">Overview</a><br>
        <a href="/benefits">Benefits</a><br>
        <a href="/how-it-works">How it works</a><br>
        <a href="/about/story">Our story</a><br>
        <a href="/blog">Our blog</a><br>
        <a href="/about/jobs">Join the team</a><br>
        <br>

        <!-- <div class="foot-head">Help</div>
        <a href="/help">Help</a><br>
		 -->


	</td>
	<td class="footer-nav">
		<!-- <a href="http://twitter.com/thinklab" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/twitter_com.png" width="40"/></a>
		<a href="https://www.facebook.com/pages/Thinklab/310895039088185" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/facebook_com.png" width="40"/></a>
		<a href="https://plus.google.com/115578983466800753402/" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/plus_google_com.png" width="40"/></a>
		<a href="https://www.linkedin.com/company/thinklab-co" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/linkedin_com.png" width="40"/></a> -->
		<br><br>
		<span class="light" style='font-size:12px;'>&copy; 2015 Thinklab</span>

	</td>

</tr></table>

</div>
</div>
</div>

		

	</div>


  <script></script>
  <script>
  var MEDIA_URL = 'https://think-lab.s3.amazonaws.com/m/';
  var STATIC_URL = 'https://think-lab.s3.amazonaws.com/s/';
  var page_is_authenticated = false;
  var page_page = 'discussion';
  var page_profile_id = undefined;
</script>



	<!-- Note different script type. MUST come before MathJax.js -->
	<script type="text/x-mathjax-config">
	    MathJax.Hub.Config({
	        "HTML-CSS": { preferredFont: "TeX", availableFonts: ["STIX","TeX"],
	                        scale: 110, linebreaks: { automatic:true },
	                        EqnChunk: (MathJax.Hub.Browser.isMobile ? 10 : 50) },
	        tex2jax: { inlineMath: [ ["$$", "$$"], ["\\(","\\)"] ],
	                    displayMath: [ ["$$$","$$$"], ["\\[", "\\]"] ],
	                    processEscapes: true,}
	            });

	</script>
    <script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>





  <script src="https://code.jquery.com/jquery-1.10.2.min.js"></script>
  <script src="https://code.jquery.com/ui/1.11.4/jquery-ui.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" integrity="sha384-0mSbJDEHialfmuBBQP6A4Qrprq5OVfW37PRR3j5ELqxss1yVqOtnepnHVP9aJ7xS" crossorigin="anonymous"></script>
  <script src="https://think-lab.s3.amazonaws.com/s/dist/main.min.js"></script>
  



  
	
  
  
  
<script>
$(function() {
  recordHit('thread', 82);
});
</script>



</body>
</html>
