<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8" /><script type="text/javascript">window.NREUM||(NREUM={}),__nr_require=function(e,n,t){function r(t){if(!n[t]){var o=n[t]={exports:{}};e[t][0].call(o.exports,function(n){var o=e[t][1][n];return r(o||n)},o,o.exports)}return n[t].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<t.length;o++)r(t[o]);return r}({1:[function(e,n,t){function r(){}function o(e,n,t){return function(){return i(e,[c.now()].concat(u(arguments)),n?null:this,t),n?void 0:this}}var i=e("handle"),a=e(2),u=e(3),f=e("ee").get("tracer"),c=e("loader"),s=NREUM;"undefined"==typeof window.newrelic&&(newrelic=s);var p=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],d="api-",l=d+"ixn-";a(p,function(e,n){s[n]=o(d+n,!0,"api")}),s.addPageAction=o(d+"addPageAction",!0),s.setCurrentRouteName=o(d+"routeName",!0),n.exports=newrelic,s.interaction=function(){return(new r).get()};var m=r.prototype={createTracer:function(e,n){var t={},r=this,o="function"==typeof n;return i(l+"tracer",[c.now(),e,t],r),function(){if(f.emit((o?"":"no-")+"fn-start",[c.now(),r,o],t),o)try{return n.apply(this,arguments)}finally{f.emit("fn-end",[c.now()],t)}}}};a("setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(e,n){m[n]=o(l+n)}),newrelic.noticeError=function(e){"string"==typeof e&&(e=new Error(e)),i("err",[e,c.now()])}},{}],2:[function(e,n,t){function r(e,n){var t=[],r="",i=0;for(r in e)o.call(e,r)&&(t[i]=n(r,e[r]),i+=1);return t}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],3:[function(e,n,t){function r(e,n,t){n||(n=0),"undefined"==typeof t&&(t=e?e.length:0);for(var r=-1,o=t-n||0,i=Array(o<0?0:o);++r<o;)i[r]=e[n+r];return i}n.exports=r},{}],4:[function(e,n,t){n.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(e,n,t){function r(){}function o(e){function n(e){return e&&e instanceof r?e:e?f(e,u,i):i()}function t(t,r,o,i){if(!d.aborted||i){e&&e(t,r,o);for(var a=n(o),u=m(t),f=u.length,c=0;c<f;c++)u[c].apply(a,r);var p=s[y[t]];return p&&p.push([b,t,r,a]),a}}function l(e,n){v[e]=m(e).concat(n)}function m(e){return v[e]||[]}function w(e){return p[e]=p[e]||o(t)}function g(e,n){c(e,function(e,t){n=n||"feature",y[t]=n,n in s||(s[n]=[])})}var v={},y={},b={on:l,emit:t,get:w,listeners:m,context:n,buffer:g,abort:a,aborted:!1};return b}function i(){return new r}function a(){(s.api||s.feature)&&(d.aborted=!0,s=d.backlog={})}var u="nr@context",f=e("gos"),c=e(2),s={},p={},d=n.exports=o();d.backlog=s},{}],gos:[function(e,n,t){function r(e,n,t){if(o.call(e,n))return e[n];var r=t();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(e,n,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return e[n]=r,r}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],handle:[function(e,n,t){function r(e,n,t,r){o.buffer([e],r),o.emit(e,n,t)}var o=e("ee").get("handle");n.exports=r,r.ee=o},{}],id:[function(e,n,t){function r(e){var n=typeof e;return!e||"object"!==n&&"function"!==n?-1:e===window?0:a(e,i,function(){return o++})}var o=1,i="nr@id",a=e("gos");n.exports=r},{}],loader:[function(e,n,t){function r(){if(!x++){var e=h.info=NREUM.info,n=d.getElementsByTagName("script")[0];if(setTimeout(s.abort,3e4),!(e&&e.licenseKey&&e.applicationID&&n))return s.abort();c(y,function(n,t){e[n]||(e[n]=t)}),f("mark",["onload",a()+h.offset],null,"api");var t=d.createElement("script");t.src="https://"+e.agent,n.parentNode.insertBefore(t,n)}}function o(){"complete"===d.readyState&&i()}function i(){f("mark",["domContent",a()+h.offset],null,"api")}function a(){return E.exists&&performance.now?Math.round(performance.now()):(u=Math.max((new Date).getTime(),u))-h.offset}var u=(new Date).getTime(),f=e("handle"),c=e(2),s=e("ee"),p=window,d=p.document,l="addEventListener",m="attachEvent",w=p.XMLHttpRequest,g=w&&w.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:w,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var v=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-1039.min.js"},b=w&&g&&g[l]&&!/CriOS/.test(navigator.userAgent),h=n.exports={offset:u,now:a,origin:v,features:{},xhrWrappable:b};e(1),d[l]?(d[l]("DOMContentLoaded",i,!1),p[l]("load",r,!1)):(d[m]("onreadystatechange",o),p[m]("onload",r)),f("mark",["firstbyte",u],null,"api");var x=0,E=e(4)},{}]},{},["loader"]);</script><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info={"beacon":"bam.nr-data.net","queueTime":0,"licenseKey":"cd1101aa94","agent":"","transactionName":"bgcDMkBZW0NZBkdcWVdNJxNcW0FZVwscV1dKB08QW11CQxYTWlBBZgEOC19dW0QCEVtHU1gG","applicationID":"6369297","errorBeacon":"bam.nr-data.net","applicationTime":706}</script>
    <title>Creating a catalog of protein interactions | Thinklab</title>
	  <link rel="shortcut icon" href="https://think-lab.s3.amazonaws.com/s/img/favicon.ico" type="image/x-icon" />
    <meta property="og:image" content="https://think-lab.s3.amazonaws.com/s/img" />
    
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
  <link rel="stylesheet" href="https://code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css" />

<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/highlightjs/github.css" />
<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/css/css.css" />
<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/css/bootstrap-multiselect.css" />
<!-- <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'> -->

    
    
    
    
  
    <meta name="citation_doi" content="10.15363/thinklab.d85">
    <meta name="citation_title" content="Creating a catalog of protein interactions">
    <meta name="citation_online_date" content="July 1, 2015">
    <meta name="citation_author" content="Daniel Himmelstein, Dexter Hadley, Alexey Strokach">
    <meta name="citation_journal_title" content="Thinklab">
    <meta name="citation_publisher" content="Thinklab">
  

    


	<!-- GoogleAnalytics -->
	<script>
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
	(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
	m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

	ga('create', 'UA-48550230-1', 'thinklab.com');
	ga('require', 'displayfeatures');
	
	ga('send', 'pageview');

	</script>






</head>
<body >
  <form id="form_action" method="post">
    <input type='hidden' name='csrfmiddlewaretoken' value='ypamUdfXu4FnASUyTHPbTrQWcdTFmVrB' />
</form>

	

	<div class="body-wrapper">

		


<!-- <div style="height:5px;background-image:url('https://think-lab.s3.amazonaws.com/s/img/top-bg.gif');"></div> -->
<div class="gradient">
	<div class="main-centered-div header-container">

<table class="full-width" style="height:60px;"><tr>
	<td style="width:200px;vertical-align:middle;padding-bottom:15px;">
		<a href="/" style="font-weight:normal;color:#212121;"><img src="https://think-lab.s3.amazonaws.com/s/img/logo.png" width="160"></a>
	</td><td style="vertical-align:middle;text-align:left;padding-top:6px;width:100%;padding-left:40px;">
		<ul id="not-search-block"  class="main-header">


			


				<li class="dropdown">
					<a class="dropdown-toggle" data-toggle="dropdown" href="#">Browse <span class="caret"></span></a>
					<ul class="dropdown-menu" role="menu">
						<li><a href="/proposals">Proposals</a></li>
						<li><a href="/projects">Projects</a></li>
						<li><a href="/publications">Article discussion</a></li>
						<li><a href="/discussion">All discussion</a></li>
					</ul>
				</li>
				<li><a href="/leaderboard">Leaderboard</a></li>

			
				<li><a href="/about">About</a></li>
			


		</ul>

    <ul id="search-block" class="main-header" style="display: none; margin-right: 30px;">
        <li class="full-width">
            <form method="get" action="/search">
			<div class="input-group full-width">
				<input name="q" type="text" class="form-control search-input">
			</div>
            </form>
        </li>
    </ul>

	</td><td style="vertical-align:middle;text-align:right;padding-top:6px; min-width: 1%; white-space: nowrap;">
		<ul class="main-header pull-right" style="padding-left: 0;">
    		
                <!-- <li><a href="/login" style="color:silver;">Login</a></li>
                <li style="margin-right:0"><a class="btn btn-default btn-sm btn-upper" href="/signup">Join</a></li> -->
            

        </ul>
    </ul>
	</td>
</tr></table>

<!-- <div id="global-search-div" class="main-centered-div" style="display:none;">
	<input type="text" value="" class="thread-search form-control" placeholder="Search" data-action="/discussion">
</div> -->

	</div>
</div>

		
	  



	  


		

		<div class="dark-head"><div class="main-centered-div">

			

		</div></div>
		
		<div class="main-centered-div" style="margin-top:40px;padding-bottom:340px">
	    

<div style="margin-top:-20px; margin-bottom:20px;">

    <a class="side-box" href="/p/rephetio" style="background-color:#ebfdff;"><div class="side-type">Project:</div> <div class="side-title">Rephetio: Repurposing drugs on a hetnet [rephetio]</div></a><br>



</div>

<table style="margin-top:0px;"><tr>
	<td class="thread-table-content">

		<table class="full-width">
			
<tr class="comment-tr target-hl" id="1" data-comment-id="309">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="309"></a>

    

			<h2 class="page-title" style="margin-bottom:5px;margin-top:0px !important;">Creating a catalog of protein interactions</h2>

			<div style="margin-top:5px;color:gray;margin-bottom:20px;font-size:13px;">
				<a style="font-weight:normal;color:#212121" href="/u/dhimmel"><strong>Daniel Himmelstein</strong></a>
				<span class="is-researcher">Researcher</span>
				&nbsp; <a href="#1" class="light">July 2, 2015</a>
				
			</div>

      

          <hr>
		

    <p>We would like to create a catalog of interactions between proteins (PPIs). I am currently leaning towards focusing on physical interactions, since other types of interactions will be captured by other metanodes and metaedges. If some non-physical interactions are included that is acceptable but not the goal.</p>

<p>Some previous studies have compiled PPI catalogs:</p>

<ul><li><p>the Incomplete Interactome (II) <span class="citation">[<a href="https://doi.org/10.1126/science.1257601" class="citation" data-key="10.1126/science.1257601">1</a>]</span> — compiled protein interactions of seven types</p></li><li><p>the Human Interaction Database (<a href="http://interactome.dfci.harvard.edu/H_sapiens/index.php?page=download">HID</a>) <span class="citation">[<a href="https://doi.org/10.1016/j.cell.2014.10.050" class="citation" data-key="10.1016/j.cell.2014.10.050">2</a>]</span> — systematic experimental approach for identifying PPIs</p></li><li><p>our disease-gene association study (<a href="http://het.io/disease-genes/downloads/">hetio</a>) <span class="citation">[<a href="https://doi.org/10.1371/journal.pcbi.1004259" class="citation" data-key="10.1371/journal.pcbi.1004259">3</a>]</span> — interactions from <a href="http://irefindex.org/wiki/index.php?title=iRefIndex">iRefIndex</a>, which compiles records from primary databases, processed using <a href="http://www.ncbi.nlm.nih.gov/CBBresearch/Yu/downloads/ppiTrim.html">ppiTrim</a> <span class="citation">[<a href="https://doi.org/10.1093/database/bar036" class="citation" data-key="10.1093/database/bar036">4</a>]</span>.</p></li></ul>

<p>Suggestions for other resources are welcome.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="2" data-comment-id="310">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="310"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#2" class="light">July 2, 2015</a>
          
        </span>
      </div>

    

    <h1>Methods for the <em>Incomplete Interactome</em> PPI catalog</h1>

<p><em>Here, we reproduce the methods section from the <a href="http://www.sciencemag.org/content/suppl/2015/02/18/347.6224.1257601.DC1/Menche_SM.pdf">supplement</a> of the Incomplete Interactome publication <span class="citation">[<a href="https://doi.org/10.1126/science.1257601" class="citation" data-key="10.1126/science.1257601">1</a>]</span> describing how their PPI catalog was constructed:</em></p>

<p>In building the interactome, we rely only physical protein interactions with experimental support, hence we do not include interactions extracted from gene expression data or evolutionary considerations. In order to obtain an interactome as complete as currently feasible, we combine several databases with various kinds of physical interactions:</p>

<ol><li><strong>Regulatory interactions</strong>: We use the TRANSFAC database <span class="citation">[<a href="https://doi.org/10.1093/nar/gkg108" class="citation" data-key="10.1093/nar/gkg108">2</a>]</span> that lists interactions derived from the presence of a transcription factor binding site in the promoter region of a certain gene. The resulting network consists of 271 transcription factors regulating 564 genes via 1,335 interactions.</li><li><strong>Binary interactions</strong>: We combine several yeast-two-hybrid high-throughput datasets <span class="citation">[<a href="https://doi.org/10.1016/j.cell.2014.10.050" class="citation" data-key="10.1016/j.cell.2014.10.050">3</a>, <a href="https://doi.org/10.1038/nmeth.1280" class="citation" data-key="10.1038/nmeth.1280">4</a>, <a href="https://doi.org/10.1016/j.cell.2005.08.029" class="citation" data-key="10.1016/j.cell.2005.08.029">5</a>, <a href="https://doi.org/10.1038/nmeth.1597" class="citation" data-key="10.1038/nmeth.1597">6</a>, <a href="https://doi.org/10.1038/nature04209" class="citation" data-key="10.1038/nature04209">7</a>, <a href="" class="citation" data-key="55">8</a>]</span> with binary interactions from IntAct <span class="citation">[<a href="https://doi.org/10.1093/nar/gkp878" class="citation" data-key="10.1093/nar/gkp878">9</a>]</span> and MINT databases <span class="citation">[<a href="https://doi.org/10.1093/nar/gkp983" class="citation" data-key="10.1093/nar/gkp983">10</a>]</span>. The sum of these data sources yields 28,653 interactions between 8,120 proteins. Note that IntAct and MINT provide interactions derived from both literature curation and direct submissions. </li><li><strong>Literature curated interactions</strong>: These interactions, typically obtained by low throughput experiments, are manually curated from the literature. We use IntAct <span class="citation">[<a href="https://doi.org/10.1093/nar/gkp878" class="citation" data-key="10.1093/nar/gkp878">9</a>]</span>, MINT <span class="citation">[<a href="https://doi.org/10.1093/nar/gkp983" class="citation" data-key="10.1093/nar/gkp983">10</a>]</span>, BioGRID <span class="citation">[<a href="https://doi.org/10.1093/nar/gkq1116" class="citation" data-key="10.1093/nar/gkq1116">11</a>]</span> and HPRD <span class="citation">[<a href="https://doi.org/10.1093/nar/gkn892" class="citation" data-key="10.1093/nar/gkn892">12</a>]</span>, resulting in 88,349 interactions between 11,798 proteins.</li><li><strong>Metabolic enzyme-coupled interactions</strong>: Two enzymes are assumed to be coupled if they share adjacent reactions in the KEGG and BIGG databases. In total, we use 5,325 such metabolic links between 921 enzymes from <span class="citation">[<a href="https://doi.org/10.1073/pnas.0802208105" class="citation" data-key="10.1073/pnas.0802208105">13</a>]</span>.</li><li><strong>Protein complexes</strong>: Protein complexes are single molecular units that integrate multiple gene products. The CORUM database <span class="citation">[<a href="https://doi.org/10.1093/nar/gkp914" class="citation" data-key="10.1093/nar/gkp914">14</a>]</span> is a collection of mammalian complexes derived from a variety of experimental tools, from co-immunoprecipitation to co-sedimentation and ion exchange chromatography. In total, CORUM yields 2,837 complexes with 2,069 proteins connected by 31,276 links.</li><li><strong>Kinase network (kinase-substrate pairs)</strong>: Protein kinases are important regulators in different biological processes, such as signal transduction. PhosphositePlus <span class="citation">[<a href="https://doi.org/10.1093/nar/gkr1122" class="citation" data-key="10.1093/nar/gkr1122">15</a>]</span> provides a network of peptides that can be bound by kinases, yielding in total 6,066 interactions between 1,843 proteins.</li><li><strong>Signaling interactions</strong>: The dataset from <span class="citation">[<a href="https://doi.org/10.1126/scisignal.2001699" class="citation" data-key="10.1126/scisignal.2001699">16</a>]</span> provides 32,706 interactions between 6,339 proteins that integrate several sources, both high-throughput and literature curation, into a directed network in which cellular signals are transmitted by proteins-protein interactions.</li></ol>

<p>The union of all interactions obtained from (i)-(vii) yields a network of 13,460 proteins that are interconnected by 141,296 physical interactions. Note that we treat the interactome as an undirected network (see Section 2.3 for a discussion of the impact of directionality). The interactome is approximately scale-free (Figure S1a) and shows other typical characteristics as observed previously in many other biological networks <span class="citation">[<a href="https://doi.org/10.1103/RevModPhys.74.47" class="citation" data-key="10.1103/RevModPhys.74.47">17</a>, <a href="https://doi.org/10.1137/S003614450342480" class="citation" data-key="10.1137/S003614450342480">18</a>]</span>, such as high clustering and short pathlengths (Figure S1c)</p>

<p></p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="3" data-comment-id="323">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="323"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#3" class="light">July 13, 2015</a>
          
        </span>
      </div>

    

    <h1>Human Interaction Database</h1>

<p>The Human Interaction Database (HID) 2014 <a href="http://interactome.dfci.harvard.edu/H_sapiens/index.php?page=download">release</a> <span class="citation">[<a href="https://doi.org/10.1016/j.cell.2014.10.050" class="citation" data-key="10.1016/j.cell.2014.10.050">1</a>]</span> contained two PPI datasets:</p>

<ul><li><code>HI-II-14</code> — 13,944 interactions — proteome-scale map of the human binary interactome network generated by systematically screening Space-II</li><li><code>Lit-BM-13</code> — 11,045 interactions — high-quality recurated literature binary interactions extracted from 7 public repository in 2013</li></ul>

<p>We <a href="https://github.com/dhimmel/ppi/blob/77862798448c4272c55e9c718323a3ec5d8db571/compile-PPIs.ipynb">compared</a> interactions from HID to interactions from the Incomplete Interactome (II). We found that <code>HI-II-14</code> was a subset of <code>II_binary</code>. However, only 78.2% of <code>Lit-BM-13</code> was included in <code>II_literature</code>.</p>

<p><em>To better understand <code>Lit-BM-13</code>, we have added the relevant sections of the paper <a href="http://www.sciencedirect.com/science/MiamiMultiMediaURL/1-s2.0-S0092867414014226/1-s2.0-S0092867414014226-mmc1.pdf/272196/html/S0092867414014226/2b892ecda8f249667d75023be6d13c7b/mmc1.pdf">supplement</a> below, omitting the "assignment of experimental method" sections <span class="citation">[<a href="https://doi.org/10.1016/j.cell.2014.10.050" class="citation" data-key="10.1016/j.cell.2014.10.050">1</a>]</span>:</em></p>

<p><strong>Literature datasets:</strong> We generated two datasets from literature-curated protein-protein interactions. A first dataset was generated in 2010 and used for all experiments, concomitantly with our mapping experiment, and a second dataset was extracted in 2013 to provide an updated version for all computational analyses.</p>

<p><strong>Obtaining the Lit-2010 dataset:</strong> The Lit-2010 dataset extracts human protein-protein interactions (PPIs), annotated through December 2010, from seven primary source databases: BIND <span class="citation">[<a href="https://doi.org/10.1093/nar/gkg056" class="citation" data-key="10.1093/nar/gkg056">2</a>]</span>, BioGRID <span class="citation">[<a href="https://doi.org/10.1093/nar/gks1158" class="citation" data-key="10.1093/nar/gks1158">3</a>]</span>, DIP <span class="citation">[<a href="https://doi.org/10.1093/nar/gkh086" class="citation" data-key="10.1093/nar/gkh086">4</a>]</span>, HPRD <span class="citation">[<a href="https://doi.org/10.1093/nar/gkn892" class="citation" data-key="10.1093/nar/gkn892">5</a>]</span>, MINT <span class="citation">[<a href="https://doi.org/10.1093/nar/gkr930" class="citation" data-key="10.1093/nar/gkr930">6</a>]</span>, IntAct <span class="citation">[<a href="https://doi.org/10.1093/nar/gkr1088" class="citation" data-key="10.1093/nar/gkr1088">7</a>]</span> and PDB <span class="citation">[<a href="https://doi.org/10.1093/nar/28.1.235" class="citation" data-key="10.1093/nar/28.1.235">8</a>]</span>. For each reported PPI the interacting proteins were mapped to UniProt protein identifiers and then converted to NCBI Entrez gene ID pairs using an ID mapping table downloaded on January 12, 2012 from uniprot.org. Information about the specific publications reporting each interaction was retained and reported interactions that did not have an associated PubMed ID (PMID) were not included in the Lit-2010 dataset.</p>

<p><strong>Identification of binary interactions:</strong> We divided Lit-2010 into the PPIs reported by<br>systematic high-throughput binary human interactome mapping efforts <span class="citation">[<a href="https://doi.org/10.1038/nature04209" class="citation" data-key="10.1038/nature04209">9</a>, <a href="https://doi.org/10.1016/j.cell.2005.08.029" class="citation" data-key="10.1016/j.cell.2005.08.029">10</a>, <a href="https://doi.org/10.1038/nmeth.1280" class="citation" data-key="10.1038/nmeth.1280">11</a>]</span> and those detected in small- or medium-scale experiments. A small number of PPIs that had been detected in both systematic and other studies could appear in both datasets. Removing the PPIs only seen in systematic studies resulted in a dataset of 56,743 human PPIs.</p>

<p>Next we attempted to distinguish binary interactions (direct biophysical contact between two proteins) <span class="citation">[<a href="https://doi.org/10.1002/pmic.201100598" class="citation" data-key="10.1002/pmic.201100598">12</a>, <a href="https://doi.org/10.1002/pmic.201100563" class="citation" data-key="10.1002/pmic.201100563">13</a>]</span> from indirect associations (associations between two proteins that are in the same complex, but may or may not directly interact) <span class="citation">[<a href="https://doi.org/10.1016/j.sbi.2013.02.008" class="citation" data-key="10.1016/j.sbi.2013.02.008">14</a>]</span>. We evaluated each experimental interaction detection method in the PSI-MI 2.5 and classified them as binary, that is, primarily detects binary interactions, versus indirect, that is, primarily detects association of proteins within a complex (Table S1C). Where an experimental method could be viewed as either, depending on the specific experimental implementation then the method was conservatively classified as indirect. Fewer methods were classified as binary here than in previous <span class="citation">[<a href="https://doi.org/10.1038/nmeth.1284" class="citation" data-key="10.1038/nmeth.1284">15</a>, <a href="https://doi.org/10.1126/science.1158684" class="citation" data-key="10.1126/science.1158684">16</a>]</span> or parallel <span class="citation">[<a href="https://doi.org/10.1186/1752-0509-6-92" class="citation" data-key="10.1186/1752-0509-6-92">17</a>]</span> efforts to ensure the highest confidence binary Lit dataset possible.</p>

<p>After parsing all PPI data from the source databases we obtained a binary human<br>dataset of 13,962 PPIs that contained at least one piece of binary evidence supporting each PPI (there could be other pieces of experimental evidences that were either binary or indirect) and a non-binary dataset containing 42,781 PPIs for which none of the experimental methods are binary (Lit-NB-10).</p>

<p>A paper curated independently by two or more different PPI databases is commonly annotated to different PSI-MI terms, generally to terms of different depth on the same branch of the PSI-MI ontology tree <span class="citation">[<a href="https://doi.org/10.1093/database/baq026" class="citation" data-key="10.1093/database/baq026">18</a>, <a href="https://doi.org/10.1038/nbt.1867" class="citation" data-key="10.1038/nbt.1867">19</a>]</span>. If not corrected for, these annotations would count as two or more pieces of evidence for the PPI, when actually there is only one piece of supporting evidence. For example, a yeast two-hybrid experiment might be annotated to the deeper term “two hybrid prey pooling approach” (MI:1112) by one PPI database but to the parent term “two hybrid” (MI:0018) by another database; a coimmunoprecipitation (co-IP) experiment might be annotated to the deeper term “anti-tag coimmunoprecipitation” (MI:0007) by one database but to the parent term “affinity chromatography technology” (MI:0004) by another. To compensate for variability in the annotated methods, when the same paper with the same PMID had different MI terms in two databases, we reassigned the deeper term “up” to the corresponding parent binary or nonbinary term on the same PSI-MI branch. In the examples given, the two Y2H annotations collapse to the single ID MI:0018, while the two co-IP annotations collapse to the single ID MI:0004.</p>

<p>The binary human dataset was next separated into “binary multiple” (Lit-BM-10) (Table S1A), containing all interactions supported by two or more pieces of experimental evidence, at least one of which was binary (4,906 PPIs); versus “binary single”, containing all interactions supported by exactly one piece of binary experimental evidence (Lit-BS-10) (9,056 PPIs).</p>

<p><strong>Updating the Lit dataset to 2013:</strong> To construct Lit-2013 (Figure S1B and Table S1B) we downloaded, on August 5, 2013, the updated curated PPI content of the same seven PPI databases used for Lit-2010.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="4" data-comment-id="324">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="324"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#4" class="light">July 13, 2015</a>
          
        </span>
      </div>

    

    <h1><em>hetio</em> interactions</h1>

<p>Previously, we used the following method to catalog protein interactions <span class="citation">[<a href="https://doi.org/10.1371/journal.pcbi.1004259" class="citation" data-key="10.1371/journal.pcbi.1004259">1</a>]</span>:</p>

<blockquote><p>Physical protein-protein interactions (<a href="http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004259#pcbi.1004259.s020">S8 Data</a>) were extracted from iRefIndex 12.0, a compilation of 15 primary interaction databases <span class="citation">[<a href="https://doi.org/10.1186/1471-2105-9-405" class="citation" data-key="10.1186/1471-2105-9-405">2</a>]</span>. The iRefIndex was processed with ppiTrim to convert proteins to genes, remove protein complexes, and condense duplicated entries <span class="citation">[<a href="https://doi.org/10.1093/database/bar036" class="citation" data-key="10.1093/database/bar036">3</a>]</span>.</p></blockquote>

<p>The method contributed 97,938 interactions to our network of protein-coding genes. For this project, we converted these interactions to entrez genes. Prior to filtering for coding genes, 98,119 interactions were in the hetio dataset.</p>

<p>The hetio interactions overlapped most with <code>Lit-BM-13</code>, <code>II_literature</code>, and <code>II_signaling</code> (<a href="https://github.com/dhimmel/ppi/blob/919264e834a95bf8724c7177b8657af9de8622fd/compile-PPIs.ipynb">notebook</a>).</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="5" data-comment-id="330">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="330"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#5" class="light">July 15, 2015</a>
          
        </span>
      </div>

    

    <h1>Signaling PPI</h1>

<p><em>The incomplete interactome paper includes interactions from a signaling study <span class="citation">[<a href="https://doi.org/10.1126/scisignal.2001699" class="citation" data-key="10.1126/scisignal.2001699">1</a>]</span>. We report the sources of this <code>HPPI1</code> network from the original paper's <a href="http://stke.sciencemag.org/content/sigtrans/suppl/2011/09/01/4.189.rs8.DC1/4_rs8_SM.pdf">supplement</a>:</em></p>

<p><strong>Table S3</strong>: Human PPI interaction data sets used to construct the HPPI1 network. A comprehensive HPPI1 network was created by unifying the data sets listed. Name of the data set, publication reference, number of proteins and number of interactions in the data set (after mapping to NCBI Entrez GeneID) are given. </p>

<table class="table markdown-table"><thead><tr><th>PPI Data sets</th><th>Reference</th><th>Proteins</th><th>Interactions</th></tr></thead><tbody><tr><td>Human Protein Reference Database V7.0</td><td><span class="citation">[<a href="https://doi.org/10.1007/978-1-60761-232-2_6" class="citation" data-key="10.1007/978-1-60761-232-2_6">2</a>]</span></td><td>9305</td><td>35021</td></tr><tr><td>Genome-wide Y2H screen and literature-derived PPI data</td><td><span class="citation">[<a href="https://doi.org/10.1038/nature04209" class="citation" data-key="10.1038/nature04209">3</a>]</span></td><td>3024</td><td>6221</td></tr><tr><td>Y2H screen for inherited ataxia and literature-derived PPI data</td><td><span class="citation">[<a href="https://doi.org/10.1016/j.cell.2006.03.032" class="citation" data-key="10.1016/j.cell.2006.03.032">4</a>]</span></td><td>2909</td><td>5440</td></tr><tr><td>Genome-wide Y2H screen</td><td><span class="citation">[<a href="https://doi.org/10.1016/j.cell.2005.08.029" class="citation" data-key="10.1016/j.cell.2005.08.029">5</a>]</span></td><td>1699</td><td>3150</td></tr><tr><td>Y2H PPIs</td><td>This study</td><td>1126</td><td>2626</td></tr><tr><td>Mouse signaling PPI data from AfCS</td><td><a href="http://www.signalinggateway.org/">http://www.signalinggateway.org/</a></td><td>857</td><td>1004</td></tr><tr><td>Network for Smad signaling</td><td><span class="citation">[<a href="https://doi.org/10.1101/gr.2334104" class="citation" data-key="10.1101/gr.2334104">6</a>]</span></td><td>623</td><td>874</td></tr><tr><td>PPIs of proteins in MHC class III region and mRNA decay</td><td><span class="citation">[<a href="https://doi.org/10.1101/gr.2122004" class="citation" data-key="10.1101/gr.2122004">7</a>, <a href="https://doi.org/10.1016/S0888-7543%2803%2900235-0" class="citation" data-key="10.1016/S0888-7543(03)00235-0">8</a>]</span></td><td>300</td><td>376</td></tr><tr><td>Network of nuclear receptors</td><td><span class="citation">[<a href="https://doi.org/10.1074/mcp.M400169-MCP200" class="citation" data-key="10.1074/mcp.M400169-MCP200">9</a>]</span></td><td>134</td><td>288</td></tr><tr><td>Huntingtin’s disease PPI network</td><td><span class="citation">[<a href="https://doi.org/10.1016/j.molcel.2004.09.016" class="citation" data-key="10.1016/j.molcel.2004.09.016">10</a>]</span></td><td>64</td><td>156</td></tr><tr><td>PPIs between KIAA proteins</td><td><span class="citation">[<a href="https://doi.org/10.1101/gr.406902" class="citation" data-key="10.1101/gr.406902">11</a>]</span></td><td>94</td><td>84</td></tr><tr><td>Total</td><td></td><td>9832</td><td>39641</td></tr></tbody></table>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="6" data-comment-id="334">
  <td class="comment-profile-avatar">
		<a href="/u/idrdex"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/s/img/p_default_medium.png' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="334"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/idrdex">Dexter Hadley</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#6" class="light">July 28, 2015</a>
          
        </span>
      </div>

    

    <p>Have you seen <a href="http://interactome.dfci.harvard.edu/H_sapiens/">Human Interactome</a> out of Harvard.  I used them for my <a href="http://www.nature.com/ncomms/2014/140606/ncomms5074/full/ncomms5074.html">PPI work for CNVs in GWAS for Autism</a> and worked out quite well.  They seem to have at least some of the ones you have already listed, although I think your list looks more extensive.  You want to be careful of integrating genome-wide PPI (i.e. Y2H) vs targeted PPI datasets.  It may mess up interpreting the statistics if the evidence for PPI is biased vs genome-wide.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="7" data-comment-id="336">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="336"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#7" class="light">July 28, 2015</a>
          
        </span>
      </div>

    

    <p><a href="/u/idrdex" class="username">@idrdex</a>, we're referring to this data as the Human Interaction Database (HID) and found that the systematic datasets were wholly included in the Incomplete Interactome (<code>II</code>) data (<a href="#323">comment</a>, <a href="https://github.com/dhimmel/ppi/blob/77862798448c4272c55e9c718323a3ec5d8db571/compile-PPIs.ipynb">notebook</a>).</p>

<p>You are right that once we introduce PPIs from targeted or curated analyses, we have introduce knowledge bias. While we prefer systematic data, we <a href="http://thinklab.com/discussion/text-as-a-resource-for-network-population/48#120">decided to incorporate</a> biased knowledge. We will likely perform a parallel analysis on an network built from only unbiased data sources. For this analysis, we'll use the Y2H datasets used in your autism study <span class="citation">[<a href="https://doi.org/10.1038/ncomms5074" class="citation" data-key="10.1038/ncomms5074">1</a>]</span>.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
<li id="note-151" class="inline-note toggle-parent" data-note-id="151">
	<p><a href="/u/dhimmel">Daniel Himmelstein</a>: Your study alludes that it includes <a href="https://dx.doi.org/10.1038/ncomms5074#ref51">citation 51</a> <span class="citation">[<a href="https://doi.org/10.1038/nature01156" class="citation" data-key="10.1038/nature01156">1</a>]</span> as a human interactome dataset. I don't think we'll include this study as it's old and doesn't seem to be a PPI database.
		
	</p>
	
	<table style="display:none;"><tr><td><a href='/doi/10.1038/nature01156' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>1.</div></td><td><div class='doi-body' data-key='10.1038/nature01156'><a href='/doi/10.1038/nature01156' class='pub-title'>N-CoR controls differentiation of neural stem cells into astrocytes</a><div class='pub-body'>Ola Hermanson, Kristen Jepsen, Michael G. Rosenfeld (2002) <i>Nature</i>. doi:<a class='pub-doi' href='https://doi.org/10.1038/nature01156'>10.1038/nature01156</a></div></div></td></tr></table>
</li>

				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="8" data-comment-id="391">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="391"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#8" class="light">Aug. 12, 2015</a>
          
        </span>
      </div>

    

    <h1>Unbiased PPI datasets</h1>

<p>Since we now include a <a href="http://thinklab.com/d/48#3">edge attribute for bias</a> in our network, we need to identify a subset of our PPIs that are derived from hypothesis free, i.e. unbiased, experiments.</p>

<p>The <em>Incomplete Interactome</em> <span class="citation">[<a href="https://doi.org/10.1126/science.1257601" class="citation" data-key="10.1126/science.1257601">1</a>]</span> <a href="http://www.sciencemag.org/content/suppl/2015/02/18/347.6224.1257601.DC1/Menche_SM.pdf#page=5">describes</a> their creation of an unbiased interactome:</p>

<blockquote><p>Since our interactome includes data from literature curation, it is inherently biased towards much studied disease-associated proteins and their interactions. We, therefore, complement our analysis using only interactions from well controlled and completely unbiased high-throughput yeast two-hybrid (y2h) datasets <span class="citation">[<a href="https://doi.org/10.1016/j.cell.2014.10.050" class="citation" data-key="10.1016/j.cell.2014.10.050">2</a>, <a href="https://doi.org/10.1038/nmeth.1280" class="citation" data-key="10.1038/nmeth.1280">3</a>, <a href="https://doi.org/10.1038/nmeth.1597" class="citation" data-key="10.1038/nmeth.1597">4</a>, <a href="https://doi.org/10.1038/nature04209" class="citation" data-key="10.1038/nature04209">5</a>, <a href="" class="citation" data-key="55">6</a>]</span>.</p></blockquote>

<p>Minus the last citation <span class="citation">[<a href="" class="citation" data-key="55">6</a>]</span> which is prepublication, we can use the other four resources <span class="citation">[<a href="https://doi.org/10.1016/j.cell.2014.10.050" class="citation" data-key="10.1016/j.cell.2014.10.050">2</a>, <a href="https://doi.org/10.1038/nmeth.1280" class="citation" data-key="10.1038/nmeth.1280">3</a>, <a href="https://doi.org/10.1038/nmeth.1597" class="citation" data-key="10.1038/nmeth.1597">4</a>, <a href="https://doi.org/10.1038/nature04209" class="citation" data-key="10.1038/nature04209">5</a>]</span>, two of which were used by <a href="/u/idrdex" class="username">@idrdex</a> in his study <span class="citation">[<a href="https://doi.org/10.1038/ncomms5074" class="citation" data-key="10.1038/ncomms5074">7</a>]</span>.</p>

<p></p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="9" data-comment-id="392">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="392"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#9" class="light">Aug. 12, 2015</a>
          
        </span>
      </div>

    

    <h1>Completed PPI catalog</h1>

<p>We have completed an initial version of our protein interaction catalog for this project <span class="citation">[<a href="https://doi.org/10.5281/zenodo.48443" class="citation" data-key="10.5281/zenodo.48443">1</a>]</span>, named <code>hetio-ind</code>. We defined interaction as <em>two genes whose protein products physically interact</em>. Physical associations from protein complexes were minimized.</p>

<p>Interactions were taken from the following sources:</p>

<ul><li><strong>Human Interactome Database</strong> (<a href="http://interactome.dfci.harvard.edu/H_sapiens/index.php?page=download">HID</a>): specifically the <code>HI-I-05</code> <span class="citation">[<a href="https://doi.org/10.1038/nature04209" class="citation" data-key="10.1038/nature04209">2</a>]</span>, <code>Venkatesan-09</code> <span class="citation">[<a href="https://doi.org/10.1038/nmeth.1280" class="citation" data-key="10.1038/nmeth.1280">3</a>]</span>, <code>Yu-11</code> <span class="citation">[<a href="https://doi.org/10.1038/nmeth.1597" class="citation" data-key="10.1038/nmeth.1597">4</a>]</span>, <code>HI-II-14</code> <span class="citation">[<a href="https://doi.org/10.1016/j.cell.2014.10.050" class="citation" data-key="10.1016/j.cell.2014.10.050">5</a>]</span>, <code>Lit-BM-13</code> <span class="citation">[<a href="https://doi.org/10.1016/j.cell.2014.10.050" class="citation" data-key="10.1016/j.cell.2014.10.050">5</a>]</span> datasets.</li><li><strong>Incomplete Interactome</strong> <span class="citation">[<a href="https://doi.org/10.1126/science.1257601" class="citation" data-key="10.1126/science.1257601">6</a>]</span>: specifically the <code>II_binary</code> and <code>II_literature</code> <a href="#2">subsets</a>.</li><li><strong>hetio-dag</strong> <span class="citation">[<a href="https://doi.org/10.1371/journal.pcbi.1004259" class="citation" data-key="10.1371/journal.pcbi.1004259">7</a>]</span>: our previous project (<a href="#4">details</a>). We removed all interactions that were not physical associations (<a href="http://www.ebi.ac.uk/ontology-lookup/browse.do?ontName=MI&amp;termId=MI%3A0915&amp;termName=physical%20association"><code>MI:0195</code></a>). This step excluded genetic and colocalization interactions.</li></ul>

<p>16,526 interactions reported by <code>HI-I-05</code> <span class="citation">[<a href="https://doi.org/10.1038/nature04209" class="citation" data-key="10.1038/nature04209">2</a>]</span>, <code>Venkatesan-09</code> <span class="citation">[<a href="https://doi.org/10.1038/nmeth.1280" class="citation" data-key="10.1038/nmeth.1280">3</a>]</span>, <code>Yu-11</code> <span class="citation">[<a href="https://doi.org/10.1038/nmeth.1597" class="citation" data-key="10.1038/nmeth.1597">4</a>]</span>, or <code>HI-II-14</code> <span class="citation">[<a href="https://doi.org/10.1016/j.cell.2014.10.050" class="citation" data-key="10.1016/j.cell.2014.10.050">5</a>]</span> were considered unbiased. The 135,203 other interactions were considered biased.</p>

<p>In total our dataset contains 151,729 protein interactions (<a href="https://github.com/dhimmel/ppi/blob/4012fe7af1699222539844256e3639782ae72695/compile-PPIs.ipynb">notebook</a>, <a href="https://github.com/dhimmel/ppi/tree/4012fe7af1699222539844256e3639782ae72695/data">downloads</a>).</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="10" data-comment-id="1241">
  <td class="comment-profile-avatar">
		<a href="/u/ostrokach"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/226_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="1241"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/ostrokach">Alexey Strokach</a>
				
				<span class="light">&nbsp;<a href="#10" class="light">April 11, 2016</a>
          
        </span>
      </div>

    

    <p>Have you considered the <a href="http://mentha.uniroma2.it/about.php">Mentha PPI database</a>? </p>

<ul><li>It combines protein-protein interaction data from most databases.</li><li>It assigns a reliability score to each interaction (like iRefIndex). </li><li>It uses the psicquic protocol and MITAB 2.5 format, so it compatible with all <a href="http://www.ebi.ac.uk/Tools/webservices/psicquic/registry/registry?action=STATUS">other protein interaction databases</a>.</li><li>Over the last few years, it has been <a href="http://mentha.uniroma2.it/download.php">regularly updated</a>. </li><li>It uses HGNC Gene IDs and UniProt accessions.</li></ul>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="11" data-comment-id="1242">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="1242"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#11" class="light">April 11, 2016</a>
          
        </span>
      </div>

    

    <p><a href="/u/ostrokach" class="username">@ostrokach</a>, thanks for letting us know about <a href="http://mentha.uniroma2.it/">Mentha</a> <span class="citation">[<a href="https://doi.org/10.1038/nmeth.2561" class="citation" data-key="10.1038/nmeth.2561">1</a>]</span>. We've moved past the network construction stage on this project but will keep Mentha in mind for the future.</p>

<p>I downloaded the latest release (<code>2016-04-11.zip</code>). Here are a few select rows from the top:</p>

<table class="table markdown-table"><thead><tr><th>Protein A</th><th>Gene A</th><th>Taxon A</th><th>Protein B</th><th>Gene B</th><th>Taxon B</th><th>Score</th><th>PMID</th></tr></thead><tbody><tr><td>Q86UX7</td><td>FERMT3</td><td>9606</td><td>Q06830</td><td>PRDX1</td><td>9606</td><td>0.126</td><td>26496610</td></tr><tr><td>Q9LK77</td><td>Q9LK77</td><td>3702</td><td>A0A088QD33</td><td>OEC103 {ECO:0000313|EMBL:AIN81148.1}</td><td>62715</td><td>0.236</td><td>25211078</td></tr><tr><td>Q96QD9</td><td>FYTTD1</td><td>9606</td><td>Q86UX7</td><td>FERMT3</td><td>9606</td><td>0.126</td><td>26496610</td></tr><tr><td>F4J5N9</td><td>BZIP24 {ECO:0000313|EMBL:AEE78868.1}</td><td>3702</td><td>A0A088QD42</td><td>OEC112 {ECO:0000313|EMBL:AIN81158.1}</td><td>62715</td><td>0.236</td><td>25211078</td></tr><tr><td>Q9Y5Q8</td><td>GTF3C5</td><td>9606</td><td>Q86U86</td><td>PBRM1</td><td>9606</td><td>0.155</td><td>22939629 26344197</td></tr></tbody></table>

<p>Two features that caught my eye are the reliability scores and the PubMed IDs of the original source(s). It looks like the file format could be a little bit cleaner (see for example <code>{ECO:0000313|EMBL:AEE78868.1}</code>). Also I didn't see a license on the Mentha website. Going forward I plan to avoid resources without an <a href="http://opendefinition.org/licenses/" title="Conformant Licenses: Open Definition">open license</a> — something we <a href="http://thinklab.com/discussion/integrating-resources-with-disparate-licensing-into-an-open-network/107#14" class="citation thread-citation" data-key="10.15363/thinklab.d107">learned the importance</a> of during this project.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

    		
		</table>

    

    &nbsp;<br>
    <!-- <a class="btn btn-primary btn-lg" href="/signup?next=/discussion/creating-a-catalog-of-protein-interactions/85&suppress_message=1" style>Join to Reply</a> -->


		<table class="references table-td-5" style="display:none;"><tr><td><a href='/doi/10.1126/science.1257601' class='pub-comments tt-right' title='Article comments'>5</a></td><td><div class='pub-order'>1.</div></td><td><div class='doi-body' data-key='10.1126/science.1257601'><a href='/doi/10.1126/science.1257601' class='pub-title'>Uncovering disease-disease relationships through the incomplete interactome</a><div class='pub-body'>J. Menche, A. Sharma, M. Kitsak, S. D. Ghiassian, M. Vidal, J. Loscalzo, A.-L. Barabasi (2015) <i>Science</i>. doi:<a class='pub-doi' href='https://doi.org/10.1126/science.1257601'>10.1126/science.1257601</a></div></div></td></tr><tr><td><a href='/doi/10.1016/j.cell.2014.10.050' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>2.</div></td><td><div class='doi-body' data-key='10.1016/j.cell.2014.10.050'><a href='/doi/10.1016/j.cell.2014.10.050' class='pub-title'>A Proteome-Scale Map of the Human Interactome Network</a><div class='pub-body'>Thomas Rolland, Murat Taşan, Benoit Charloteaux, Samuel J. Pevzner, Quan Zhong, Nidhi Sahni, Song Yi, Irma Lemmens, Celia Fontanillo, Roberto Mosca, Atanas Kamburov, Susan D. Ghiassian, Xinping Yang, Lila Ghamsari, Dawit Balcha, Bridget E. Begg, Pascal Braun, Marc Brehme, Martin P. Broly, Anne-Ruxandra Carvunis, Dan Convery-Zupan, Roser Corominas, Jasmin Coulombe-Huntington, Elizabeth Dann, Matija Dreze, Amélie Dricot, Changyu Fan, Eric Franzosa, Fana Gebreab, Bryan J. Gutierrez, Madeleine F. Hardy, Mike Jin, Shuli Kang, Ruth Kiros, Guan Ning Lin, Katja Luck, Andrew MacWilliams, Jörg Menche, Ryan R. Murray, Alexandre Palagi, Matthew M. Poulin, Xavier Rambout, John Rasla, Patrick Reichert, Viviana Romero, Elien Ruyssinck, Julie M. Sahalie, Annemarie Scholz, Akash A. Shah, Amitabh Sharma, Yun Shen, Kerstin Spirohn, Stanley Tam, Alexander O. Tejeda, Shelly A. Trigg, Jean-Claude Twizere, Kerwin Vega, Jennifer Walsh, Michael E. Cusick, Yu Xia, Albert-László Barabási, Lilia M. Iakoucheva, Patrick Aloy, Javier De Las Rivas, Jan Tavernier, Michael A. Calderwood, David E. Hill, Tong Hao, Frederick P. Roth, Marc Vidal (2014) <i>Cell</i>. doi:<a class='pub-doi' href='https://doi.org/10.1016/j.cell.2014.10.050'>10.1016/j.cell.2014.10.050</a></div></div></td></tr><tr><td><a href='/doi/10.1371/journal.pcbi.1004259' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>3.</div></td><td><div class='doi-body' data-key='10.1371/journal.pcbi.1004259'><a href='/doi/10.1371/journal.pcbi.1004259' class='pub-title'>Heterogeneous Network Edge Prediction: A Data Integration Approach to Prioritize Disease-Associated Genes</a><div class='pub-body'>Daniel S. Himmelstein, Sergio E. Baranzini (2015) <i>PLOS Computational Biology</i>. doi:<a class='pub-doi' href='https://doi.org/10.1371/journal.pcbi.1004259'>10.1371/journal.pcbi.1004259</a></div></div></td></tr><tr><td><a href='/doi/10.1093/database/bar036' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>4.</div></td><td><div class='doi-body' data-key='10.1093/database/bar036'><a href='/doi/10.1093/database/bar036' class='pub-title'>ppiTrim: constructing non-redundant and up-to-date interactomes</a><div class='pub-body'>A. Stojmirovic, Y.-K. Yu (2011) <i>Database</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/database/bar036'>10.1093/database/bar036</a></div></div></td></tr><tr><td><a href='/doi/10.1126/science.1257601' class='pub-comments tt-right' title='Article comments'>5</a></td><td><div class='pub-order'>5.</div></td><td><div class='doi-body' data-key='10.1126/science.1257601'><a href='/doi/10.1126/science.1257601' class='pub-title'>Uncovering disease-disease relationships through the incomplete interactome</a><div class='pub-body'>J. Menche, A. Sharma, M. Kitsak, S. D. Ghiassian, M. Vidal, J. Loscalzo, A.-L. Barabasi (2015) <i>Science</i>. doi:<a class='pub-doi' href='https://doi.org/10.1126/science.1257601'>10.1126/science.1257601</a></div></div></td></tr><tr><td><a href='/doi/10.1093/nar/gkg108' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>6.</div></td><td><div class='doi-body' data-key='10.1093/nar/gkg108'><a href='/doi/10.1093/nar/gkg108' class='pub-title'>TRANSFAC(R): transcriptional regulation, from patterns to profiles</a><div class='pub-body'>V. Matys (2003) <i>Nucleic Acids Research</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/nar/gkg108'>10.1093/nar/gkg108</a></div></div></td></tr><tr><td><a href='/doi/10.1016/j.cell.2014.10.050' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>7.</div></td><td><div class='doi-body' data-key='10.1016/j.cell.2014.10.050'><a href='/doi/10.1016/j.cell.2014.10.050' class='pub-title'>A Proteome-Scale Map of the Human Interactome Network</a><div class='pub-body'>Thomas Rolland, Murat Taşan, Benoit Charloteaux, Samuel J. Pevzner, Quan Zhong, Nidhi Sahni, Song Yi, Irma Lemmens, Celia Fontanillo, Roberto Mosca, Atanas Kamburov, Susan D. Ghiassian, Xinping Yang, Lila Ghamsari, Dawit Balcha, Bridget E. Begg, Pascal Braun, Marc Brehme, Martin P. Broly, Anne-Ruxandra Carvunis, Dan Convery-Zupan, Roser Corominas, Jasmin Coulombe-Huntington, Elizabeth Dann, Matija Dreze, Amélie Dricot, Changyu Fan, Eric Franzosa, Fana Gebreab, Bryan J. Gutierrez, Madeleine F. Hardy, Mike Jin, Shuli Kang, Ruth Kiros, Guan Ning Lin, Katja Luck, Andrew MacWilliams, Jörg Menche, Ryan R. Murray, Alexandre Palagi, Matthew M. Poulin, Xavier Rambout, John Rasla, Patrick Reichert, Viviana Romero, Elien Ruyssinck, Julie M. Sahalie, Annemarie Scholz, Akash A. Shah, Amitabh Sharma, Yun Shen, Kerstin Spirohn, Stanley Tam, Alexander O. Tejeda, Shelly A. Trigg, Jean-Claude Twizere, Kerwin Vega, Jennifer Walsh, Michael E. Cusick, Yu Xia, Albert-László Barabási, Lilia M. Iakoucheva, Patrick Aloy, Javier De Las Rivas, Jan Tavernier, Michael A. Calderwood, David E. Hill, Tong Hao, Frederick P. Roth, Marc Vidal (2014) <i>Cell</i>. doi:<a class='pub-doi' href='https://doi.org/10.1016/j.cell.2014.10.050'>10.1016/j.cell.2014.10.050</a></div></div></td></tr><tr><td><a href='/doi/10.1038/nmeth.1280' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>8.</div></td><td><div class='doi-body' data-key='10.1038/nmeth.1280'><a href='/doi/10.1038/nmeth.1280' class='pub-title'>An empirical framework for binary interactome mapping</a><div class='pub-body'>Kavitha Venkatesan, Jean-François Rual, Alexei Vazquez, Ulrich Stelzl, Irma Lemmens, Tomoko Hirozane-Kishikawa, Tong Hao, Martina Zenkner, Xiaofeng Xin, Kwang-Il Goh, Muhammed A Yildirim, Nicolas Simonis, Kathrin Heinzmann, Fana Gebreab, Julie M Sahalie, Sebiha Cevik, Christophe Simon, Anne-Sophie de Smet, Elizabeth Dann, Alex Smolyar, Arunachalam Vinayagam, Haiyuan Yu, David Szeto, Heather Borick, Amélie Dricot, Niels Klitgord, Ryan R Murray, Chenwei Lin, Maciej Lalowski, Jan Timm, Kirstin Rau, Charles Boone, Pascal Braun, Michael E Cusick, Frederick P Roth, David E Hill, Jan Tavernier, Erich E Wanker, Albert-László Barabási, Marc Vidal (2008) <i>Nat Meth</i>. doi:<a class='pub-doi' href='https://doi.org/10.1038/nmeth.1280'>10.1038/nmeth.1280</a></div></div></td></tr><tr><td><a href='/doi/10.1016/j.cell.2005.08.029' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>9.</div></td><td><div class='doi-body' data-key='10.1016/j.cell.2005.08.029'><a href='/doi/10.1016/j.cell.2005.08.029' class='pub-title'>A Human Protein-Protein Interaction Network: A Resource for Annotating the Proteome</a><div class='pub-body'>Ulrich Stelzl, Uwe Worm, Maciej Lalowski, Christian Haenig, Felix H. Brembeck, Heike Goehler, Martin Stroedicke, Martina Zenkner, Anke Schoenherr, Susanne Koeppen, Jan Timm, Sascha Mintzlaff, Claudia Abraham, Nicole Bock, Silvia Kietzmann, Astrid Goedde, Engin Toksöz, Anja Droege, Sylvia Krobitsch, Bernhard Korn, Walter Birchmeier, Hans Lehrach, Erich E. Wanker (2005) <i>Cell</i>. doi:<a class='pub-doi' href='https://doi.org/10.1016/j.cell.2005.08.029'>10.1016/j.cell.2005.08.029</a></div></div></td></tr><tr><td><a href='/doi/10.1038/nmeth.1597' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>10.</div></td><td><div class='doi-body' data-key='10.1038/nmeth.1597'><a href='/doi/10.1038/nmeth.1597' class='pub-title'>Next-generation sequencing to generate interactome datasets</a><div class='pub-body'>Haiyuan Yu, Leah Tardivo, Stanley Tam, Evan Weiner, Fana Gebreab, Changyu Fan, Nenad Svrzikapa, Tomoko Hirozane-Kishikawa, Edward Rietman, Xinping Yang, Julie Sahalie, Kourosh Salehi-Ashtiani, Tong Hao, Michael E Cusick, David E Hill, Frederick P Roth, Pascal Braun, Marc Vidal (2011) <i>Nat Meth</i>. doi:<a class='pub-doi' href='https://doi.org/10.1038/nmeth.1597'>10.1038/nmeth.1597</a></div></div></td></tr><tr><td><a href='/doi/10.1038/nature04209' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>11.</div></td><td><div class='doi-body' data-key='10.1038/nature04209'><a href='/doi/10.1038/nature04209' class='pub-title'>Towards a proteome-scale map of the human protein–protein interaction network</a><div class='pub-body'>Jean-François Rual, Kavitha Venkatesan, Tong Hao, Tomoko Hirozane-Kishikawa, Amélie Dricot, Ning Li, Gabriel F. Berriz, Francis D. Gibbons, Matija Dreze, Nono Ayivi-Guedehoussou, Niels Klitgord, Christophe Simon, Mike Boxem, Stuart Milstein, Jennifer Rosenberg, Debra S. Goldberg, Lan V. Zhang, Sharyl L. Wong, Giovanni Franklin, Siming Li, Joanna S. Albala, Janghoo Lim, Carlene Fraughton, Estelle Llamosas, Sebiha Cevik, Camille Bex, Philippe Lamesch, Robert S. Sikorski, Jean Vandenhaute, Huda Y. Zoghbi, Alex Smolyar, Stephanie Bosak, Reynaldo Sequerra, Lynn Doucette-Stamm, Michael E. Cusick, David E. Hill, Frederick P. Roth, Marc Vidal (2005) <i>Nature</i>. doi:<a class='pub-doi' href='https://doi.org/10.1038/nature04209'>10.1038/nature04209</a></div></div></td></tr><tr><td></td><td><div class='pub-order'>12.</div></td><td><div class='doi-body' data-key='55'><span class='pub-title'>Center for Cancer Systems Biology, Hi-2012 prepublication</a></div></td></tr><tr><td><a href='/doi/10.1093/nar/gkp878' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>13.</div></td><td><div class='doi-body' data-key='10.1093/nar/gkp878'><a href='/doi/10.1093/nar/gkp878' class='pub-title'>The IntAct molecular interaction database in 2010</a><div class='pub-body'>B. Aranda, P. Achuthan, Y. Alam-Faruque, I. Armean, A. Bridge, C. Derow, M. Feuermann, A. T. Ghanbarian, S. Kerrien, J. Khadake, J. Kerssemakers, C. Leroy, M. Menden, M. Michaut, L. Montecchi-Palazzi, S. N. Neuhauser, S. Orchard, V. Perreau, B. Roechert, K. van Eijk, H. Hermjakob (2009) <i>Nucleic Acids Research</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/nar/gkp878'>10.1093/nar/gkp878</a></div></div></td></tr><tr><td><a href='/doi/10.1093/nar/gkp983' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>14.</div></td><td><div class='doi-body' data-key='10.1093/nar/gkp983'><a href='/doi/10.1093/nar/gkp983' class='pub-title'>MINT, the molecular interaction database: 2009 update</a><div class='pub-body'>A. Ceol, A. Chatr Aryamontri, L. Licata, D. Peluso, L. Briganti, L. Perfetto, L. Castagnoli, G. Cesareni (2009) <i>Nucleic Acids Research</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/nar/gkp983'>10.1093/nar/gkp983</a></div></div></td></tr><tr><td><a href='/doi/10.1093/nar/gkq1116' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>15.</div></td><td><div class='doi-body' data-key='10.1093/nar/gkq1116'><a href='/doi/10.1093/nar/gkq1116' class='pub-title'>The BioGRID Interaction Database: 2011 update</a><div class='pub-body'>C. Stark, B.-J. Breitkreutz, A. Chatr-aryamontri, L. Boucher, R. Oughtred, M. S. Livstone, J. Nixon, K. Van Auken, X. Wang, X. Shi, T. Reguly, J. M. Rust, A. Winter, K. Dolinski, M. Tyers (2010) <i>Nucleic Acids Research</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/nar/gkq1116'>10.1093/nar/gkq1116</a></div></div></td></tr><tr><td><a href='/doi/10.1093/nar/gkn892' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>16.</div></td><td><div class='doi-body' data-key='10.1093/nar/gkn892'><a href='/doi/10.1093/nar/gkn892' class='pub-title'>Human Protein Reference Database--2009 update</a><div class='pub-body'>T. S. Keshava Prasad, R. Goel, K. Kandasamy, S. Keerthikumar, S. Kumar, S. Mathivanan, D. Telikicherla, R. Raju, B. Shafreen, A. Venugopal, L. Balakrishnan, A. Marimuthu, S. Banerjee, D. S. Somanathan, A. Sebastian, S. Rani, S. Ray, C. J. Harrys Kishore, S. Kanth, M. Ahmed, M. K. Kashyap, R. Mohmood, Y. L. Ramachandra, V. Krishna, B. A. Rahiman, S. Mohan, P. Ranganathan, S. Ramabadran, R. Chaerkady, A. Pandey (2009) <i>Nucleic Acids Research</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/nar/gkn892'>10.1093/nar/gkn892</a></div></div></td></tr><tr><td><a href='/doi/10.1073/pnas.0802208105' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>17.</div></td><td><div class='doi-body' data-key='10.1073/pnas.0802208105'><a href='/doi/10.1073/pnas.0802208105' class='pub-title'>The implications of human metabolic network topology for disease comorbidity</a><div class='pub-body'>D.- S. Lee, J. Park, K. A. Kay, N. A. Christakis, Z. N. Oltvai, A.- L. Barabasi (2008) <i>Proceedings of the National Academy of Sciences</i>. doi:<a class='pub-doi' href='https://doi.org/10.1073/pnas.0802208105'>10.1073/pnas.0802208105</a></div></div></td></tr><tr><td><a href='/doi/10.1093/nar/gkp914' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>18.</div></td><td><div class='doi-body' data-key='10.1093/nar/gkp914'><a href='/doi/10.1093/nar/gkp914' class='pub-title'>CORUM: the comprehensive resource of mammalian protein complexes--2009</a><div class='pub-body'>A. Ruepp, B. Waegele, M. Lechner, B. Brauner, I. Dunger-Kaltenbach, G. Fobo, G. Frishman, C. Montrone, H.-W. Mewes (2009) <i>Nucleic Acids Research</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/nar/gkp914'>10.1093/nar/gkp914</a></div></div></td></tr><tr><td><a href='/doi/10.1093/nar/gkr1122' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>19.</div></td><td><div class='doi-body' data-key='10.1093/nar/gkr1122'><a href='/doi/10.1093/nar/gkr1122' class='pub-title'>PhosphoSitePlus: a comprehensive resource for investigating the structure and function of experimentally determined post-translational modifications in man and mouse</a><div class='pub-body'>P. V. Hornbeck, J. M. Kornhauser, S. Tkachev, B. Zhang, E. Skrzypek, B. Murray, V. Latham, M. Sullivan (2011) <i>Nucleic Acids Research</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/nar/gkr1122'>10.1093/nar/gkr1122</a></div></div></td></tr><tr><td><a href='/doi/10.1126/scisignal.2001699' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>20.</div></td><td><div class='doi-body' data-key='10.1126/scisignal.2001699'><a href='/doi/10.1126/scisignal.2001699' class='pub-title'>A Directed Protein Interaction Network for Investigating Intracellular Signal Transduction</a><div class='pub-body'>A. Vinayagam, U. Stelzl, R. Foulle, S. Plassmann, M. Zenkner, J. Timm, H. E. Assmus, M. A. Andrade-Navarro, E. E. Wanker (2011) <i>Science Signaling</i>. doi:<a class='pub-doi' href='https://doi.org/10.1126/scisignal.2001699'>10.1126/scisignal.2001699</a></div></div></td></tr><tr><td><a href='/doi/10.1103/RevModPhys.74.47' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>21.</div></td><td><div class='doi-body' data-key='10.1103/RevModPhys.74.47'><a href='/doi/10.1103/RevModPhys.74.47' class='pub-title'>Statistical mechanics of complex networks</a><div class='pub-body'>Réka Albert, Albert-László Barabási (2002) <i>Reviews of Modern Physics</i>. doi:<a class='pub-doi' href='https://doi.org/10.1103/RevModPhys.74.47'>10.1103/RevModPhys.74.47</a></div></div></td></tr><tr><td><a href='/doi/10.1137/S003614450342480' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>22.</div></td><td><div class='doi-body' data-key='10.1137/S003614450342480'><a href='/doi/10.1137/S003614450342480' class='pub-title'>The Structure and Function of Complex Networks</a><div class='pub-body'>M. E. J. Newman (2003) <i>SIAM Rev.</i>. doi:<a class='pub-doi' href='https://doi.org/10.1137/S003614450342480'>10.1137/S003614450342480</a></div></div></td></tr><tr><td><a href='/doi/10.1016/j.cell.2014.10.050' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>23.</div></td><td><div class='doi-body' data-key='10.1016/j.cell.2014.10.050'><a href='/doi/10.1016/j.cell.2014.10.050' class='pub-title'>A Proteome-Scale Map of the Human Interactome Network</a><div class='pub-body'>Thomas Rolland, Murat Taşan, Benoit Charloteaux, Samuel J. Pevzner, Quan Zhong, Nidhi Sahni, Song Yi, Irma Lemmens, Celia Fontanillo, Roberto Mosca, Atanas Kamburov, Susan D. Ghiassian, Xinping Yang, Lila Ghamsari, Dawit Balcha, Bridget E. Begg, Pascal Braun, Marc Brehme, Martin P. Broly, Anne-Ruxandra Carvunis, Dan Convery-Zupan, Roser Corominas, Jasmin Coulombe-Huntington, Elizabeth Dann, Matija Dreze, Amélie Dricot, Changyu Fan, Eric Franzosa, Fana Gebreab, Bryan J. Gutierrez, Madeleine F. Hardy, Mike Jin, Shuli Kang, Ruth Kiros, Guan Ning Lin, Katja Luck, Andrew MacWilliams, Jörg Menche, Ryan R. Murray, Alexandre Palagi, Matthew M. Poulin, Xavier Rambout, John Rasla, Patrick Reichert, Viviana Romero, Elien Ruyssinck, Julie M. Sahalie, Annemarie Scholz, Akash A. Shah, Amitabh Sharma, Yun Shen, Kerstin Spirohn, Stanley Tam, Alexander O. Tejeda, Shelly A. Trigg, Jean-Claude Twizere, Kerwin Vega, Jennifer Walsh, Michael E. Cusick, Yu Xia, Albert-László Barabási, Lilia M. Iakoucheva, Patrick Aloy, Javier De Las Rivas, Jan Tavernier, Michael A. Calderwood, David E. Hill, Tong Hao, Frederick P. Roth, Marc Vidal (2014) <i>Cell</i>. doi:<a class='pub-doi' href='https://doi.org/10.1016/j.cell.2014.10.050'>10.1016/j.cell.2014.10.050</a></div></div></td></tr><tr><td><a href='/doi/10.1093/nar/gkg056' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>24.</div></td><td><div class='doi-body' data-key='10.1093/nar/gkg056'><a href='/doi/10.1093/nar/gkg056' class='pub-title'>BIND: the Biomolecular Interaction Network Database</a><div class='pub-body'>G. D. Bader (2003) <i>Nucleic Acids Research</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/nar/gkg056'>10.1093/nar/gkg056</a></div></div></td></tr><tr><td><a href='/doi/10.1093/nar/gks1158' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>25.</div></td><td><div class='doi-body' data-key='10.1093/nar/gks1158'><a href='/doi/10.1093/nar/gks1158' class='pub-title'>The BioGRID interaction database: 2013 update</a><div class='pub-body'>A. Chatr-aryamontri, B.-J. Breitkreutz, S. Heinicke, L. Boucher, A. Winter, C. Stark, J. Nixon, L. Ramage, N. Kolas, L. O&#39;Donnell, T. Reguly, A. Breitkreutz, A. Sellam, D. Chen, C. Chang, J. Rust, M. Livstone, R. Oughtred, K. Dolinski, M. Tyers (2012) <i>Nucleic Acids Research</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/nar/gks1158'>10.1093/nar/gks1158</a></div></div></td></tr><tr><td><a href='/doi/10.1093/nar/gkh086' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>26.</div></td><td><div class='doi-body' data-key='10.1093/nar/gkh086'><a href='/doi/10.1093/nar/gkh086' class='pub-title'>The Database of Interacting Proteins: 2004 update</a><div class='pub-body'>L. Salwinski (2004) <i>Nucleic Acids Research</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/nar/gkh086'>10.1093/nar/gkh086</a></div></div></td></tr><tr><td><a href='/doi/10.1093/nar/gkn892' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>27.</div></td><td><div class='doi-body' data-key='10.1093/nar/gkn892'><a href='/doi/10.1093/nar/gkn892' class='pub-title'>Human Protein Reference Database--2009 update</a><div class='pub-body'>T. S. Keshava Prasad, R. Goel, K. Kandasamy, S. Keerthikumar, S. Kumar, S. Mathivanan, D. Telikicherla, R. Raju, B. Shafreen, A. Venugopal, L. Balakrishnan, A. Marimuthu, S. Banerjee, D. S. Somanathan, A. Sebastian, S. Rani, S. Ray, C. J. Harrys Kishore, S. Kanth, M. Ahmed, M. K. Kashyap, R. Mohmood, Y. L. Ramachandra, V. Krishna, B. A. Rahiman, S. Mohan, P. Ranganathan, S. Ramabadran, R. Chaerkady, A. Pandey (2009) <i>Nucleic Acids Research</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/nar/gkn892'>10.1093/nar/gkn892</a></div></div></td></tr><tr><td><a href='/doi/10.1093/nar/gkr930' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>28.</div></td><td><div class='doi-body' data-key='10.1093/nar/gkr930'><a href='/doi/10.1093/nar/gkr930' class='pub-title'>MINT, the molecular interaction database: 2012 update</a><div class='pub-body'>L. Licata, L. Briganti, D. Peluso, L. Perfetto, M. Iannuccelli, E. Galeota, F. Sacco, A. Palma, A. P. Nardozza, E. Santonico, L. Castagnoli, G. Cesareni (2011) <i>Nucleic Acids Research</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/nar/gkr930'>10.1093/nar/gkr930</a></div></div></td></tr><tr><td><a href='/doi/10.1093/nar/gkr1088' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>29.</div></td><td><div class='doi-body' data-key='10.1093/nar/gkr1088'><a href='/doi/10.1093/nar/gkr1088' class='pub-title'>The IntAct molecular interaction database in 2012</a><div class='pub-body'>S. Kerrien, B. Aranda, L. Breuza, A. Bridge, F. Broackes-Carter, C. Chen, M. Duesbury, M. Dumousseau, M. Feuermann, U. Hinz, C. Jandrasits, R. C. Jimenez, J. Khadake, U. Mahadevan, P. Masson, I. Pedruzzi, E. Pfeiffenberger, P. Porras, A. Raghunath, B. Roechert, S. Orchard, H. Hermjakob (2011) <i>Nucleic Acids Research</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/nar/gkr1088'>10.1093/nar/gkr1088</a></div></div></td></tr><tr><td><a href='/doi/10.1093/nar/28.1.235' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>30.</div></td><td><div class='doi-body' data-key='10.1093/nar/28.1.235'><a href='/doi/10.1093/nar/28.1.235' class='pub-title'>The Protein Data Bank</a><div class='pub-body'>H. M. Berman (2000) <i>Nucleic Acids Research</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/nar/28.1.235'>10.1093/nar/28.1.235</a></div></div></td></tr><tr><td><a href='/doi/10.1038/nature04209' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>31.</div></td><td><div class='doi-body' data-key='10.1038/nature04209'><a href='/doi/10.1038/nature04209' class='pub-title'>Towards a proteome-scale map of the human protein–protein interaction network</a><div class='pub-body'>Jean-François Rual, Kavitha Venkatesan, Tong Hao, Tomoko Hirozane-Kishikawa, Amélie Dricot, Ning Li, Gabriel F. Berriz, Francis D. Gibbons, Matija Dreze, Nono Ayivi-Guedehoussou, Niels Klitgord, Christophe Simon, Mike Boxem, Stuart Milstein, Jennifer Rosenberg, Debra S. Goldberg, Lan V. Zhang, Sharyl L. Wong, Giovanni Franklin, Siming Li, Joanna S. Albala, Janghoo Lim, Carlene Fraughton, Estelle Llamosas, Sebiha Cevik, Camille Bex, Philippe Lamesch, Robert S. Sikorski, Jean Vandenhaute, Huda Y. Zoghbi, Alex Smolyar, Stephanie Bosak, Reynaldo Sequerra, Lynn Doucette-Stamm, Michael E. Cusick, David E. Hill, Frederick P. Roth, Marc Vidal (2005) <i>Nature</i>. doi:<a class='pub-doi' href='https://doi.org/10.1038/nature04209'>10.1038/nature04209</a></div></div></td></tr><tr><td><a href='/doi/10.1016/j.cell.2005.08.029' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>32.</div></td><td><div class='doi-body' data-key='10.1016/j.cell.2005.08.029'><a href='/doi/10.1016/j.cell.2005.08.029' class='pub-title'>A Human Protein-Protein Interaction Network: A Resource for Annotating the Proteome</a><div class='pub-body'>Ulrich Stelzl, Uwe Worm, Maciej Lalowski, Christian Haenig, Felix H. Brembeck, Heike Goehler, Martin Stroedicke, Martina Zenkner, Anke Schoenherr, Susanne Koeppen, Jan Timm, Sascha Mintzlaff, Claudia Abraham, Nicole Bock, Silvia Kietzmann, Astrid Goedde, Engin Toksöz, Anja Droege, Sylvia Krobitsch, Bernhard Korn, Walter Birchmeier, Hans Lehrach, Erich E. Wanker (2005) <i>Cell</i>. doi:<a class='pub-doi' href='https://doi.org/10.1016/j.cell.2005.08.029'>10.1016/j.cell.2005.08.029</a></div></div></td></tr><tr><td><a href='/doi/10.1038/nmeth.1280' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>33.</div></td><td><div class='doi-body' data-key='10.1038/nmeth.1280'><a href='/doi/10.1038/nmeth.1280' class='pub-title'>An empirical framework for binary interactome mapping</a><div class='pub-body'>Kavitha Venkatesan, Jean-François Rual, Alexei Vazquez, Ulrich Stelzl, Irma Lemmens, Tomoko Hirozane-Kishikawa, Tong Hao, Martina Zenkner, Xiaofeng Xin, Kwang-Il Goh, Muhammed A Yildirim, Nicolas Simonis, Kathrin Heinzmann, Fana Gebreab, Julie M Sahalie, Sebiha Cevik, Christophe Simon, Anne-Sophie de Smet, Elizabeth Dann, Alex Smolyar, Arunachalam Vinayagam, Haiyuan Yu, David Szeto, Heather Borick, Amélie Dricot, Niels Klitgord, Ryan R Murray, Chenwei Lin, Maciej Lalowski, Jan Timm, Kirstin Rau, Charles Boone, Pascal Braun, Michael E Cusick, Frederick P Roth, David E Hill, Jan Tavernier, Erich E Wanker, Albert-László Barabási, Marc Vidal (2008) <i>Nat Meth</i>. doi:<a class='pub-doi' href='https://doi.org/10.1038/nmeth.1280'>10.1038/nmeth.1280</a></div></div></td></tr><tr><td><a href='/doi/10.1002/pmic.201100598' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>34.</div></td><td><div class='doi-body' data-key='10.1002/pmic.201100598'><a href='/doi/10.1002/pmic.201100598' class='pub-title'>Interactome mapping for analysis of complex phenotypes: Insights from benchmarking binary interaction assays</a><div class='pub-body'>Pascal Braun (2012) <i>PROTEOMICS</i>. doi:<a class='pub-doi' href='https://doi.org/10.1002/pmic.201100598'>10.1002/pmic.201100598</a></div></div></td></tr><tr><td><a href='/doi/10.1002/pmic.201100563' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>35.</div></td><td><div class='doi-body' data-key='10.1002/pmic.201100563'><a href='/doi/10.1002/pmic.201100563' class='pub-title'>History of protein-protein interactions: From egg-white to complex networks</a><div class='pub-body'>Pascal Braun, Anne-Claude Gingras (2012) <i>PROTEOMICS</i>. doi:<a class='pub-doi' href='https://doi.org/10.1002/pmic.201100563'>10.1002/pmic.201100563</a></div></div></td></tr><tr><td><a href='/doi/10.1016/j.sbi.2013.02.008' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>36.</div></td><td><div class='doi-body' data-key='10.1016/j.sbi.2013.02.008'><a href='/doi/10.1016/j.sbi.2013.02.008' class='pub-title'>Mass spectrometry supported determination of protein complex structure</a><div class='pub-body'>Thomas Walzthoeni, Alexander Leitner, Florian Stengel, Ruedi Aebersold (2013) <i>Current Opinion in Structural Biology</i>. doi:<a class='pub-doi' href='https://doi.org/10.1016/j.sbi.2013.02.008'>10.1016/j.sbi.2013.02.008</a></div></div></td></tr><tr><td><a href='/doi/10.1038/nmeth.1284' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>37.</div></td><td><div class='doi-body' data-key='10.1038/nmeth.1284'><a href='/doi/10.1038/nmeth.1284' class='pub-title'>Literature-curated protein interaction datasets</a><div class='pub-body'>Michael E Cusick, Haiyuan Yu, Alex Smolyar, Kavitha Venkatesan, Anne-Ruxandra Carvunis, Nicolas Simonis, Jean-François Rual, Heather Borick, Pascal Braun, Matija Dreze, Jean Vandenhaute, Mary Galli, Junshi Yazaki, David E Hill, Joseph R Ecker, Frederick P Roth, Marc Vidal (2009) <i>Nat Meth</i>. doi:<a class='pub-doi' href='https://doi.org/10.1038/nmeth.1284'>10.1038/nmeth.1284</a></div></div></td></tr><tr><td><a href='/doi/10.1126/science.1158684' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>38.</div></td><td><div class='doi-body' data-key='10.1126/science.1158684'><a href='/doi/10.1126/science.1158684' class='pub-title'>High-Quality Binary Protein Interaction Map of the Yeast Interactome Network</a><div class='pub-body'>H. Yu, P. Braun, M. A. Yildirim, I. Lemmens, K. Venkatesan, J. Sahalie, T. Hirozane-Kishikawa, F. Gebreab, N. Li, N. Simonis, T. Hao, J.-F. Rual, A. Dricot, A. Vazquez, R. R. Murray, C. Simon, L. Tardivo, S. Tam, N. Svrzikapa, C. Fan, A.-S. de Smet, A. Motyl, M. E. Hudson, J. Park, X. Xin, M. E. Cusick, T. Moore, C. Boone, M. Snyder, F. P. Roth, A.-L. Barabasi, J. Tavernier, D. E. Hill, M. Vidal (2008) <i>Science</i>. doi:<a class='pub-doi' href='https://doi.org/10.1126/science.1158684'>10.1126/science.1158684</a></div></div></td></tr><tr><td><a href='/doi/10.1186/1752-0509-6-92' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>39.</div></td><td><div class='doi-body' data-key='10.1186/1752-0509-6-92'><a href='/doi/10.1186/1752-0509-6-92' class='pub-title'>HINT: High-quality protein interactomes and their applications in understanding human disease</a><div class='pub-body'>Jishnu Das, Haiyuan Yu (2012) <i>BMC Syst Biol</i>. doi:<a class='pub-doi' href='https://doi.org/10.1186/1752-0509-6-92'>10.1186/1752-0509-6-92</a></div></div></td></tr><tr><td><a href='/doi/10.1093/database/baq026' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>40.</div></td><td><div class='doi-body' data-key='10.1093/database/baq026'><a href='/doi/10.1093/database/baq026' class='pub-title'>Literature curation of protein interactions: measuring agreement across major public databases</a><div class='pub-body'>A. L. Turinsky, S. Razick, B. Turner, I. M. Donaldson, S. J. Wodak (2010) <i>Database</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/database/baq026'>10.1093/database/baq026</a></div></div></td></tr><tr><td><a href='/doi/10.1038/nbt.1867' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>41.</div></td><td><div class='doi-body' data-key='10.1038/nbt.1867'><a href='/doi/10.1038/nbt.1867' class='pub-title'>Interaction databases on the same page</a><div class='pub-body'>Andrei L Turinsky, Sabry Razick, Brian Turner, Ian M Donaldson, Shoshana J Wodak (2011) <i>Nat Biotechnol</i>. doi:<a class='pub-doi' href='https://doi.org/10.1038/nbt.1867'>10.1038/nbt.1867</a></div></div></td></tr><tr><td><a href='/doi/10.1371/journal.pcbi.1004259' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>42.</div></td><td><div class='doi-body' data-key='10.1371/journal.pcbi.1004259'><a href='/doi/10.1371/journal.pcbi.1004259' class='pub-title'>Heterogeneous Network Edge Prediction: A Data Integration Approach to Prioritize Disease-Associated Genes</a><div class='pub-body'>Daniel S. Himmelstein, Sergio E. Baranzini (2015) <i>PLOS Computational Biology</i>. doi:<a class='pub-doi' href='https://doi.org/10.1371/journal.pcbi.1004259'>10.1371/journal.pcbi.1004259</a></div></div></td></tr><tr><td><a href='/doi/10.1186/1471-2105-9-405' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>43.</div></td><td><div class='doi-body' data-key='10.1186/1471-2105-9-405'><a href='/doi/10.1186/1471-2105-9-405' class='pub-title'>iRefIndex: A consolidated protein interaction database with provenance</a><div class='pub-body'>Sabry Razick, George Magklaras, Ian M Donaldson (2008) <i>BMC Bioinformatics</i>. doi:<a class='pub-doi' href='https://doi.org/10.1186/1471-2105-9-405'>10.1186/1471-2105-9-405</a></div></div></td></tr><tr><td><a href='/doi/10.1093/database/bar036' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>44.</div></td><td><div class='doi-body' data-key='10.1093/database/bar036'><a href='/doi/10.1093/database/bar036' class='pub-title'>ppiTrim: constructing non-redundant and up-to-date interactomes</a><div class='pub-body'>A. Stojmirovic, Y.-K. Yu (2011) <i>Database</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/database/bar036'>10.1093/database/bar036</a></div></div></td></tr><tr><td><a href='/doi/10.1126/scisignal.2001699' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>45.</div></td><td><div class='doi-body' data-key='10.1126/scisignal.2001699'><a href='/doi/10.1126/scisignal.2001699' class='pub-title'>A Directed Protein Interaction Network for Investigating Intracellular Signal Transduction</a><div class='pub-body'>A. Vinayagam, U. Stelzl, R. Foulle, S. Plassmann, M. Zenkner, J. Timm, H. E. Assmus, M. A. Andrade-Navarro, E. E. Wanker (2011) <i>Science Signaling</i>. doi:<a class='pub-doi' href='https://doi.org/10.1126/scisignal.2001699'>10.1126/scisignal.2001699</a></div></div></td></tr><tr><td><a href='/doi/10.1007/978-1-60761-232-2_6' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>46.</div></td><td><div class='doi-body' data-key='10.1007/978-1-60761-232-2_6'><a href='/doi/10.1007/978-1-60761-232-2_6' class='pub-title'>Human Protein Reference Database and Human Proteinpedia as Discovery Tools for Systems Biology</a><div class='pub-body'>T. S. Keshava Prasad, Kumaran Kandasamy, Akhilesh Pandey (2009) <i>Methods in Molecular Biology</i>. doi:<a class='pub-doi' href='https://doi.org/10.1007/978-1-60761-232-2_6'>10.1007/978-1-60761-232-2_6</a></div></div></td></tr><tr><td><a href='/doi/10.1038/nature04209' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>47.</div></td><td><div class='doi-body' data-key='10.1038/nature04209'><a href='/doi/10.1038/nature04209' class='pub-title'>Towards a proteome-scale map of the human protein–protein interaction network</a><div class='pub-body'>Jean-François Rual, Kavitha Venkatesan, Tong Hao, Tomoko Hirozane-Kishikawa, Amélie Dricot, Ning Li, Gabriel F. Berriz, Francis D. Gibbons, Matija Dreze, Nono Ayivi-Guedehoussou, Niels Klitgord, Christophe Simon, Mike Boxem, Stuart Milstein, Jennifer Rosenberg, Debra S. Goldberg, Lan V. Zhang, Sharyl L. Wong, Giovanni Franklin, Siming Li, Joanna S. Albala, Janghoo Lim, Carlene Fraughton, Estelle Llamosas, Sebiha Cevik, Camille Bex, Philippe Lamesch, Robert S. Sikorski, Jean Vandenhaute, Huda Y. Zoghbi, Alex Smolyar, Stephanie Bosak, Reynaldo Sequerra, Lynn Doucette-Stamm, Michael E. Cusick, David E. Hill, Frederick P. Roth, Marc Vidal (2005) <i>Nature</i>. doi:<a class='pub-doi' href='https://doi.org/10.1038/nature04209'>10.1038/nature04209</a></div></div></td></tr><tr><td><a href='/doi/10.1016/j.cell.2006.03.032' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>48.</div></td><td><div class='doi-body' data-key='10.1016/j.cell.2006.03.032'><a href='/doi/10.1016/j.cell.2006.03.032' class='pub-title'>A Protein–Protein Interaction Network for Human Inherited Ataxias and Disorders of Purkinje Cell Degeneration</a><div class='pub-body'>Janghoo Lim, Tong Hao, Chad Shaw, Akash J. Patel, Gábor Szabó, Jean-François Rual, C. Joseph Fisk, Ning Li, Alex Smolyar, David E. Hill, Albert-László Barabási, Marc Vidal, Huda Y. Zoghbi (2006) <i>Cell</i>. doi:<a class='pub-doi' href='https://doi.org/10.1016/j.cell.2006.03.032'>10.1016/j.cell.2006.03.032</a></div></div></td></tr><tr><td><a href='/doi/10.1016/j.cell.2005.08.029' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>49.</div></td><td><div class='doi-body' data-key='10.1016/j.cell.2005.08.029'><a href='/doi/10.1016/j.cell.2005.08.029' class='pub-title'>A Human Protein-Protein Interaction Network: A Resource for Annotating the Proteome</a><div class='pub-body'>Ulrich Stelzl, Uwe Worm, Maciej Lalowski, Christian Haenig, Felix H. Brembeck, Heike Goehler, Martin Stroedicke, Martina Zenkner, Anke Schoenherr, Susanne Koeppen, Jan Timm, Sascha Mintzlaff, Claudia Abraham, Nicole Bock, Silvia Kietzmann, Astrid Goedde, Engin Toksöz, Anja Droege, Sylvia Krobitsch, Bernhard Korn, Walter Birchmeier, Hans Lehrach, Erich E. Wanker (2005) <i>Cell</i>. doi:<a class='pub-doi' href='https://doi.org/10.1016/j.cell.2005.08.029'>10.1016/j.cell.2005.08.029</a></div></div></td></tr><tr><td><a href='/doi/10.1101/gr.2334104' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>50.</div></td><td><div class='doi-body' data-key='10.1101/gr.2334104'><a href='/doi/10.1101/gr.2334104' class='pub-title'>Functional Proteomics Mapping of a Human Signaling Pathway</a><div class='pub-body'>F. Colland (2004) <i>Genome Research</i>. doi:<a class='pub-doi' href='https://doi.org/10.1101/gr.2334104'>10.1101/gr.2334104</a></div></div></td></tr><tr><td><a href='/doi/10.1101/gr.2122004' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>51.</div></td><td><div class='doi-body' data-key='10.1101/gr.2122004'><a href='/doi/10.1101/gr.2122004' class='pub-title'>A Protein Interaction Framework for Human mRNA Degradation</a><div class='pub-body'>B. Lehner (2004) <i>Genome Research</i>. doi:<a class='pub-doi' href='https://doi.org/10.1101/gr.2122004'>10.1101/gr.2122004</a></div></div></td></tr><tr><td><a href='/doi/10.1016/S0888-7543(03)00235-0' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>52.</div></td><td><div class='doi-body' data-key='10.1016/S0888-7543(03)00235-0'><a href='/doi/10.1016/S0888-7543(03)00235-0' class='pub-title'>Analysis of a high-throughput yeast two-hybrid system and its use to predict the function of intracellular proteins encoded within the human MHC class III region</a><div class='pub-body'>Ben Lehner, Jennifer I Semple, Stephanie E Brown, Damian Counsell, R.Duncan Campbell, Christopher M Sanderson (2004) <i>Genomics</i>. doi:<a class='pub-doi' href='https://doi.org/10.1016/S0888-7543(03)00235-0'>10.1016/S0888-7543(03)00235-0</a></div></div></td></tr><tr><td><a href='/doi/10.1074/mcp.M400169-MCP200' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>53.</div></td><td><div class='doi-body' data-key='10.1074/mcp.M400169-MCP200'><a href='/doi/10.1074/mcp.M400169-MCP200' class='pub-title'>Automated Yeast Two-hybrid Screening for Nuclear Receptor-interacting Proteins</a><div class='pub-body'>M. Albers (2004) <i>Molecular &amp; Cellular Proteomics</i>. doi:<a class='pub-doi' href='https://doi.org/10.1074/mcp.M400169-MCP200'>10.1074/mcp.M400169-MCP200</a></div></div></td></tr><tr><td><a href='/doi/10.1016/j.molcel.2004.09.016' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>54.</div></td><td><div class='doi-body' data-key='10.1016/j.molcel.2004.09.016'><a href='/doi/10.1016/j.molcel.2004.09.016' class='pub-title'>A Protein Interaction Network Links GIT1, an Enhancer of Huntingtin Aggregation, to Huntington&#39;s Disease</a><div class='pub-body'>Heike Goehler, Maciej Lalowski, Ulrich Stelzl, Stephanie Waelter, Martin Stroedicke, Uwe Worm, Anja Droege, Katrin S. Lindenberg, Maria Knoblich, Christian Haenig, Martin Herbst, Jaana Suopanki, Eberhard Scherzinger, Claudia Abraham, Bianca Bauer, Renate Hasenbank, Anja Fritzsche, Andreas H. Ludewig, Konrad Buessow, Sarah H. Coleman, Claire-Anne Gutekunst, Bernhard G. Landwehrmeyer, Hans Lehrach, Erich E. Wanker (2004) <i>Molecular Cell</i>. doi:<a class='pub-doi' href='https://doi.org/10.1016/j.molcel.2004.09.016'>10.1016/j.molcel.2004.09.016</a></div></div></td></tr><tr><td><a href='/doi/10.1101/gr.406902' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>55.</div></td><td><div class='doi-body' data-key='10.1101/gr.406902'><a href='/doi/10.1101/gr.406902' class='pub-title'>Protein-Protein Interactions Between Large Proteins: Two-Hybrid Screening Using a Functionally Classified Library Composed of Long cDNAs</a><div class='pub-body'>M. Nakayama (2002) <i>Genome Research</i>. doi:<a class='pub-doi' href='https://doi.org/10.1101/gr.406902'>10.1101/gr.406902</a></div></div></td></tr><tr><td><a href='/doi/10.1038/ncomms5074' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>56.</div></td><td><div class='doi-body' data-key='10.1038/ncomms5074'><a href='/doi/10.1038/ncomms5074' class='pub-title'>The impact of the metabotropic glutamate receptor and other gene family interaction networks on autism</a><div class='pub-body'>Dexter Hadley, Zhi-liang Wu, Charlly Kao, Akshata Kini, Alisha Mohamed-Hadley, Kelly Thomas, Lyam Vazquez, Haijun Qiu, Frank Mentch, Renata Pellegrino, Cecilia Kim, John Connolly, Dalila Pinto, Alison Merikangas, Lambertus Klei, Jacob A.S. Vorstman, Ann Thompson, Regina Regan, Alistair T. Pagnamenta, Bárbara Oliveira, Tiago R. Magalhaes, John Gilbert, Eftichia Duketis, Maretha V. De Jonge, Michael Cuccaro, Catarina T. Correia, Judith Conroy, Inês C. Conceição, Andreas G. Chiocchetti, Jillian P. Casey, Nadia Bolshakova, Elena Bacchelli, Richard Anney, Lonnie Zwaigenbaum, Kerstin Wittemeyer, Simon Wallace, Herman van Engeland, Latha Soorya, Bernadette Rogé, Wendy Roberts, Fritz Poustka, Susana Mouga, Nancy Minshew, Susan G. McGrew, Catherine Lord, Marion Leboyer, Ann S. Le Couteur, Alexander Kolevzon, Suma Jacob, Stephen Guter, Jonathan Green, Andrew Green, Christopher Gillberg, Bridget A. Fernandez, Frederico Duque, Richard Delorme, Geraldine Dawson, Cátia Café, Sean Brennan, Thomas Bourgeron, Patrick F. Bolton, Sven Bölte, Raphael Bernier, Gillian Baird, Anthony J. Bailey, Evdokia Anagnostou, Joana Almeida, Ellen M. Wijsman, Veronica J. Vieland, Astrid M. Vicente, Gerard D. Schellenberg, Margaret Pericak-Vance, Andrew D. Paterson, Jeremy R. Parr, Guiomar Oliveira, Joana Almeida, Cátia Café, Susana Mouga, Catarina Correia, John I. Nurnberger, Anthony P. Monaco, Elena Maestrini, Sabine M. Klauck, Hakon Hakonarson, Jonathan L. Haines, Daniel H. Geschwind, Christine M. Freitag, Susan E. Folstein, Sean Ennis, Hilary Coon, Agatino Battaglia, Peter Szatmari, James S. Sutcliffe, Joachim Hallmayer, Michael Gill, Edwin H. Cook, Joseph D. Buxbaum, Bernie Devlin, Louise Gallagher, Catalina Betancur, Stephen W. Scherer, Joseph Glessner, Hakon Hakonarson (2014) <i>Nature Communications</i>. doi:<a class='pub-doi' href='https://doi.org/10.1038/ncomms5074'>10.1038/ncomms5074</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d48' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>57.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d48'><a href='/d/48' class='pub-title'>Text as a resource for network population?</a><div class='pub-body'>Benjamin Good, Daniel Himmelstein (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d48'>10.15363/thinklab.d48</a></div></div></td></tr><tr><td><a href='/doi/10.1038/nature01156' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>58.</div></td><td><div class='doi-body' data-key='10.1038/nature01156'><a href='/doi/10.1038/nature01156' class='pub-title'>N-CoR controls differentiation of neural stem cells into astrocytes</a><div class='pub-body'>Ola Hermanson, Kristen Jepsen, Michael G. Rosenfeld (2002) <i>Nature</i>. doi:<a class='pub-doi' href='https://doi.org/10.1038/nature01156'>10.1038/nature01156</a></div></div></td></tr><tr><td><a href='/doi/10.1126/science.1257601' class='pub-comments tt-right' title='Article comments'>5</a></td><td><div class='pub-order'>59.</div></td><td><div class='doi-body' data-key='10.1126/science.1257601'><a href='/doi/10.1126/science.1257601' class='pub-title'>Uncovering disease-disease relationships through the incomplete interactome</a><div class='pub-body'>J. Menche, A. Sharma, M. Kitsak, S. D. Ghiassian, M. Vidal, J. Loscalzo, A.-L. Barabasi (2015) <i>Science</i>. doi:<a class='pub-doi' href='https://doi.org/10.1126/science.1257601'>10.1126/science.1257601</a></div></div></td></tr><tr><td><a href='/doi/10.1016/j.cell.2014.10.050' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>60.</div></td><td><div class='doi-body' data-key='10.1016/j.cell.2014.10.050'><a href='/doi/10.1016/j.cell.2014.10.050' class='pub-title'>A Proteome-Scale Map of the Human Interactome Network</a><div class='pub-body'>Thomas Rolland, Murat Taşan, Benoit Charloteaux, Samuel J. Pevzner, Quan Zhong, Nidhi Sahni, Song Yi, Irma Lemmens, Celia Fontanillo, Roberto Mosca, Atanas Kamburov, Susan D. Ghiassian, Xinping Yang, Lila Ghamsari, Dawit Balcha, Bridget E. Begg, Pascal Braun, Marc Brehme, Martin P. Broly, Anne-Ruxandra Carvunis, Dan Convery-Zupan, Roser Corominas, Jasmin Coulombe-Huntington, Elizabeth Dann, Matija Dreze, Amélie Dricot, Changyu Fan, Eric Franzosa, Fana Gebreab, Bryan J. Gutierrez, Madeleine F. Hardy, Mike Jin, Shuli Kang, Ruth Kiros, Guan Ning Lin, Katja Luck, Andrew MacWilliams, Jörg Menche, Ryan R. Murray, Alexandre Palagi, Matthew M. Poulin, Xavier Rambout, John Rasla, Patrick Reichert, Viviana Romero, Elien Ruyssinck, Julie M. Sahalie, Annemarie Scholz, Akash A. Shah, Amitabh Sharma, Yun Shen, Kerstin Spirohn, Stanley Tam, Alexander O. Tejeda, Shelly A. Trigg, Jean-Claude Twizere, Kerwin Vega, Jennifer Walsh, Michael E. Cusick, Yu Xia, Albert-László Barabási, Lilia M. Iakoucheva, Patrick Aloy, Javier De Las Rivas, Jan Tavernier, Michael A. Calderwood, David E. Hill, Tong Hao, Frederick P. Roth, Marc Vidal (2014) <i>Cell</i>. doi:<a class='pub-doi' href='https://doi.org/10.1016/j.cell.2014.10.050'>10.1016/j.cell.2014.10.050</a></div></div></td></tr><tr><td><a href='/doi/10.1038/nmeth.1280' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>61.</div></td><td><div class='doi-body' data-key='10.1038/nmeth.1280'><a href='/doi/10.1038/nmeth.1280' class='pub-title'>An empirical framework for binary interactome mapping</a><div class='pub-body'>Kavitha Venkatesan, Jean-François Rual, Alexei Vazquez, Ulrich Stelzl, Irma Lemmens, Tomoko Hirozane-Kishikawa, Tong Hao, Martina Zenkner, Xiaofeng Xin, Kwang-Il Goh, Muhammed A Yildirim, Nicolas Simonis, Kathrin Heinzmann, Fana Gebreab, Julie M Sahalie, Sebiha Cevik, Christophe Simon, Anne-Sophie de Smet, Elizabeth Dann, Alex Smolyar, Arunachalam Vinayagam, Haiyuan Yu, David Szeto, Heather Borick, Amélie Dricot, Niels Klitgord, Ryan R Murray, Chenwei Lin, Maciej Lalowski, Jan Timm, Kirstin Rau, Charles Boone, Pascal Braun, Michael E Cusick, Frederick P Roth, David E Hill, Jan Tavernier, Erich E Wanker, Albert-László Barabási, Marc Vidal (2008) <i>Nat Meth</i>. doi:<a class='pub-doi' href='https://doi.org/10.1038/nmeth.1280'>10.1038/nmeth.1280</a></div></div></td></tr><tr><td><a href='/doi/10.1038/nmeth.1597' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>62.</div></td><td><div class='doi-body' data-key='10.1038/nmeth.1597'><a href='/doi/10.1038/nmeth.1597' class='pub-title'>Next-generation sequencing to generate interactome datasets</a><div class='pub-body'>Haiyuan Yu, Leah Tardivo, Stanley Tam, Evan Weiner, Fana Gebreab, Changyu Fan, Nenad Svrzikapa, Tomoko Hirozane-Kishikawa, Edward Rietman, Xinping Yang, Julie Sahalie, Kourosh Salehi-Ashtiani, Tong Hao, Michael E Cusick, David E Hill, Frederick P Roth, Pascal Braun, Marc Vidal (2011) <i>Nat Meth</i>. doi:<a class='pub-doi' href='https://doi.org/10.1038/nmeth.1597'>10.1038/nmeth.1597</a></div></div></td></tr><tr><td><a href='/doi/10.1038/nature04209' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>63.</div></td><td><div class='doi-body' data-key='10.1038/nature04209'><a href='/doi/10.1038/nature04209' class='pub-title'>Towards a proteome-scale map of the human protein–protein interaction network</a><div class='pub-body'>Jean-François Rual, Kavitha Venkatesan, Tong Hao, Tomoko Hirozane-Kishikawa, Amélie Dricot, Ning Li, Gabriel F. Berriz, Francis D. Gibbons, Matija Dreze, Nono Ayivi-Guedehoussou, Niels Klitgord, Christophe Simon, Mike Boxem, Stuart Milstein, Jennifer Rosenberg, Debra S. Goldberg, Lan V. Zhang, Sharyl L. Wong, Giovanni Franklin, Siming Li, Joanna S. Albala, Janghoo Lim, Carlene Fraughton, Estelle Llamosas, Sebiha Cevik, Camille Bex, Philippe Lamesch, Robert S. Sikorski, Jean Vandenhaute, Huda Y. Zoghbi, Alex Smolyar, Stephanie Bosak, Reynaldo Sequerra, Lynn Doucette-Stamm, Michael E. Cusick, David E. Hill, Frederick P. Roth, Marc Vidal (2005) <i>Nature</i>. doi:<a class='pub-doi' href='https://doi.org/10.1038/nature04209'>10.1038/nature04209</a></div></div></td></tr><tr><td></td><td><div class='pub-order'>64.</div></td><td><div class='doi-body' data-key='55'><span class='pub-title'>Center for Cancer Systems Biology, Hi-2012 prepublication</a></div></td></tr><tr><td><a href='/doi/10.1038/ncomms5074' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>65.</div></td><td><div class='doi-body' data-key='10.1038/ncomms5074'><a href='/doi/10.1038/ncomms5074' class='pub-title'>The impact of the metabotropic glutamate receptor and other gene family interaction networks on autism</a><div class='pub-body'>Dexter Hadley, Zhi-liang Wu, Charlly Kao, Akshata Kini, Alisha Mohamed-Hadley, Kelly Thomas, Lyam Vazquez, Haijun Qiu, Frank Mentch, Renata Pellegrino, Cecilia Kim, John Connolly, Dalila Pinto, Alison Merikangas, Lambertus Klei, Jacob A.S. Vorstman, Ann Thompson, Regina Regan, Alistair T. Pagnamenta, Bárbara Oliveira, Tiago R. Magalhaes, John Gilbert, Eftichia Duketis, Maretha V. De Jonge, Michael Cuccaro, Catarina T. Correia, Judith Conroy, Inês C. Conceição, Andreas G. Chiocchetti, Jillian P. Casey, Nadia Bolshakova, Elena Bacchelli, Richard Anney, Lonnie Zwaigenbaum, Kerstin Wittemeyer, Simon Wallace, Herman van Engeland, Latha Soorya, Bernadette Rogé, Wendy Roberts, Fritz Poustka, Susana Mouga, Nancy Minshew, Susan G. McGrew, Catherine Lord, Marion Leboyer, Ann S. Le Couteur, Alexander Kolevzon, Suma Jacob, Stephen Guter, Jonathan Green, Andrew Green, Christopher Gillberg, Bridget A. Fernandez, Frederico Duque, Richard Delorme, Geraldine Dawson, Cátia Café, Sean Brennan, Thomas Bourgeron, Patrick F. Bolton, Sven Bölte, Raphael Bernier, Gillian Baird, Anthony J. Bailey, Evdokia Anagnostou, Joana Almeida, Ellen M. Wijsman, Veronica J. Vieland, Astrid M. Vicente, Gerard D. Schellenberg, Margaret Pericak-Vance, Andrew D. Paterson, Jeremy R. Parr, Guiomar Oliveira, Joana Almeida, Cátia Café, Susana Mouga, Catarina Correia, John I. Nurnberger, Anthony P. Monaco, Elena Maestrini, Sabine M. Klauck, Hakon Hakonarson, Jonathan L. Haines, Daniel H. Geschwind, Christine M. Freitag, Susan E. Folstein, Sean Ennis, Hilary Coon, Agatino Battaglia, Peter Szatmari, James S. Sutcliffe, Joachim Hallmayer, Michael Gill, Edwin H. Cook, Joseph D. Buxbaum, Bernie Devlin, Louise Gallagher, Catalina Betancur, Stephen W. Scherer, Joseph Glessner, Hakon Hakonarson (2014) <i>Nature Communications</i>. doi:<a class='pub-doi' href='https://doi.org/10.1038/ncomms5074'>10.1038/ncomms5074</a></div></div></td></tr><tr><td><a href='/doi/10.5281/zenodo.48443' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>66.</div></td><td><div class='doi-body' data-key='10.5281/zenodo.48443'><a href='/doi/10.5281/zenodo.48443' class='pub-title'>dhimmel/ppi v1.0: Compiling a human protein interaction catalog</a><div class='pub-body'>Daniel S. Himmelstein, Sergio E. Baranzini (2016) <i>Zenodo</i>. doi:<a class='pub-doi' href='https://doi.org/10.5281/zenodo.48443'>10.5281/zenodo.48443</a></div></div></td></tr><tr><td><a href='/doi/10.1038/nature04209' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>67.</div></td><td><div class='doi-body' data-key='10.1038/nature04209'><a href='/doi/10.1038/nature04209' class='pub-title'>Towards a proteome-scale map of the human protein–protein interaction network</a><div class='pub-body'>Jean-François Rual, Kavitha Venkatesan, Tong Hao, Tomoko Hirozane-Kishikawa, Amélie Dricot, Ning Li, Gabriel F. Berriz, Francis D. Gibbons, Matija Dreze, Nono Ayivi-Guedehoussou, Niels Klitgord, Christophe Simon, Mike Boxem, Stuart Milstein, Jennifer Rosenberg, Debra S. Goldberg, Lan V. Zhang, Sharyl L. Wong, Giovanni Franklin, Siming Li, Joanna S. Albala, Janghoo Lim, Carlene Fraughton, Estelle Llamosas, Sebiha Cevik, Camille Bex, Philippe Lamesch, Robert S. Sikorski, Jean Vandenhaute, Huda Y. Zoghbi, Alex Smolyar, Stephanie Bosak, Reynaldo Sequerra, Lynn Doucette-Stamm, Michael E. Cusick, David E. Hill, Frederick P. Roth, Marc Vidal (2005) <i>Nature</i>. doi:<a class='pub-doi' href='https://doi.org/10.1038/nature04209'>10.1038/nature04209</a></div></div></td></tr><tr><td><a href='/doi/10.1038/nmeth.1280' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>68.</div></td><td><div class='doi-body' data-key='10.1038/nmeth.1280'><a href='/doi/10.1038/nmeth.1280' class='pub-title'>An empirical framework for binary interactome mapping</a><div class='pub-body'>Kavitha Venkatesan, Jean-François Rual, Alexei Vazquez, Ulrich Stelzl, Irma Lemmens, Tomoko Hirozane-Kishikawa, Tong Hao, Martina Zenkner, Xiaofeng Xin, Kwang-Il Goh, Muhammed A Yildirim, Nicolas Simonis, Kathrin Heinzmann, Fana Gebreab, Julie M Sahalie, Sebiha Cevik, Christophe Simon, Anne-Sophie de Smet, Elizabeth Dann, Alex Smolyar, Arunachalam Vinayagam, Haiyuan Yu, David Szeto, Heather Borick, Amélie Dricot, Niels Klitgord, Ryan R Murray, Chenwei Lin, Maciej Lalowski, Jan Timm, Kirstin Rau, Charles Boone, Pascal Braun, Michael E Cusick, Frederick P Roth, David E Hill, Jan Tavernier, Erich E Wanker, Albert-László Barabási, Marc Vidal (2008) <i>Nat Meth</i>. doi:<a class='pub-doi' href='https://doi.org/10.1038/nmeth.1280'>10.1038/nmeth.1280</a></div></div></td></tr><tr><td><a href='/doi/10.1038/nmeth.1597' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>69.</div></td><td><div class='doi-body' data-key='10.1038/nmeth.1597'><a href='/doi/10.1038/nmeth.1597' class='pub-title'>Next-generation sequencing to generate interactome datasets</a><div class='pub-body'>Haiyuan Yu, Leah Tardivo, Stanley Tam, Evan Weiner, Fana Gebreab, Changyu Fan, Nenad Svrzikapa, Tomoko Hirozane-Kishikawa, Edward Rietman, Xinping Yang, Julie Sahalie, Kourosh Salehi-Ashtiani, Tong Hao, Michael E Cusick, David E Hill, Frederick P Roth, Pascal Braun, Marc Vidal (2011) <i>Nat Meth</i>. doi:<a class='pub-doi' href='https://doi.org/10.1038/nmeth.1597'>10.1038/nmeth.1597</a></div></div></td></tr><tr><td><a href='/doi/10.1016/j.cell.2014.10.050' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>70.</div></td><td><div class='doi-body' data-key='10.1016/j.cell.2014.10.050'><a href='/doi/10.1016/j.cell.2014.10.050' class='pub-title'>A Proteome-Scale Map of the Human Interactome Network</a><div class='pub-body'>Thomas Rolland, Murat Taşan, Benoit Charloteaux, Samuel J. Pevzner, Quan Zhong, Nidhi Sahni, Song Yi, Irma Lemmens, Celia Fontanillo, Roberto Mosca, Atanas Kamburov, Susan D. Ghiassian, Xinping Yang, Lila Ghamsari, Dawit Balcha, Bridget E. Begg, Pascal Braun, Marc Brehme, Martin P. Broly, Anne-Ruxandra Carvunis, Dan Convery-Zupan, Roser Corominas, Jasmin Coulombe-Huntington, Elizabeth Dann, Matija Dreze, Amélie Dricot, Changyu Fan, Eric Franzosa, Fana Gebreab, Bryan J. Gutierrez, Madeleine F. Hardy, Mike Jin, Shuli Kang, Ruth Kiros, Guan Ning Lin, Katja Luck, Andrew MacWilliams, Jörg Menche, Ryan R. Murray, Alexandre Palagi, Matthew M. Poulin, Xavier Rambout, John Rasla, Patrick Reichert, Viviana Romero, Elien Ruyssinck, Julie M. Sahalie, Annemarie Scholz, Akash A. Shah, Amitabh Sharma, Yun Shen, Kerstin Spirohn, Stanley Tam, Alexander O. Tejeda, Shelly A. Trigg, Jean-Claude Twizere, Kerwin Vega, Jennifer Walsh, Michael E. Cusick, Yu Xia, Albert-László Barabási, Lilia M. Iakoucheva, Patrick Aloy, Javier De Las Rivas, Jan Tavernier, Michael A. Calderwood, David E. Hill, Tong Hao, Frederick P. Roth, Marc Vidal (2014) <i>Cell</i>. doi:<a class='pub-doi' href='https://doi.org/10.1016/j.cell.2014.10.050'>10.1016/j.cell.2014.10.050</a></div></div></td></tr><tr><td><a href='/doi/10.1126/science.1257601' class='pub-comments tt-right' title='Article comments'>5</a></td><td><div class='pub-order'>71.</div></td><td><div class='doi-body' data-key='10.1126/science.1257601'><a href='/doi/10.1126/science.1257601' class='pub-title'>Uncovering disease-disease relationships through the incomplete interactome</a><div class='pub-body'>J. Menche, A. Sharma, M. Kitsak, S. D. Ghiassian, M. Vidal, J. Loscalzo, A.-L. Barabasi (2015) <i>Science</i>. doi:<a class='pub-doi' href='https://doi.org/10.1126/science.1257601'>10.1126/science.1257601</a></div></div></td></tr><tr><td><a href='/doi/10.1371/journal.pcbi.1004259' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>72.</div></td><td><div class='doi-body' data-key='10.1371/journal.pcbi.1004259'><a href='/doi/10.1371/journal.pcbi.1004259' class='pub-title'>Heterogeneous Network Edge Prediction: A Data Integration Approach to Prioritize Disease-Associated Genes</a><div class='pub-body'>Daniel S. Himmelstein, Sergio E. Baranzini (2015) <i>PLOS Computational Biology</i>. doi:<a class='pub-doi' href='https://doi.org/10.1371/journal.pcbi.1004259'>10.1371/journal.pcbi.1004259</a></div></div></td></tr><tr><td><a href='/doi/10.1038/nmeth.2561' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>73.</div></td><td><div class='doi-body' data-key='10.1038/nmeth.2561'><a href='/doi/10.1038/nmeth.2561' class='pub-title'>mentha: a resource for browsing integrated protein-interaction networks</a><div class='pub-body'>Alberto Calderone, Luisa Castagnoli, Gianni Cesareni (2013) <i>Nature Methods</i>. doi:<a class='pub-doi' href='https://doi.org/10.1038/nmeth.2561'>10.1038/nmeth.2561</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d107' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>74.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d107'><a href='/d/107' class='pub-title'>Integrating resources with disparate licensing into an open network</a><div class='pub-body'>Daniel Himmelstein, Lars Juhl Jensen, MacKenzie Smith, Katie Fortney, Caty Chung (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d107'>10.15363/thinklab.d107</a></div></div></td></tr></table>

    


	</td>
	<td class="thread-sidebar">


			

      <!-- <a class="btn btn-default follow-link btn-follow btn-lg is-following tt-top full-width" title="Follow discussion" data-id="85" data-ctype="thread" data-is-following="false">&nbsp;</a> -->

      
        <div class="status-box text-center"><span style="">Status:</span> 
  Completed

</div>
      

      
        
      


				

					<div class="details">Views</div>
					<span style="font-size:20px;">181</span>

        

				
					<div class="details">Topics</div>
					<div style="margin-bottom:15px;">
		            <a href="/topics/protein-interactions" class="tag">Protein Interactions</a><a href="/topics/interactions" class="tag">Interactions</a><a href="/topics/ppi" class="tag">PPI</a>
					</div>
				


		
			<div class="details">Referenced by</div>
			<div style="font-size:12px;"><ul style="padding-left:16px !important;">
			
				<li>
					
					<a href='/d/107' class='thread-title'>Integrating resources with disparate licensing into an open network</a>
				</li>
			
				<li>
					
					<a href='/d/227' class='thread-title'>Brainstorming future directions for Hetionet</a>
				</li>
			
				<li>
					
					Research report: <a href='/p/rephetio/report'>Rephetio: Repurposing drugs on a hetnet</a>
				</li>
			
				<li>
					
					<a href='/d/111' class='thread-title'>Incomplete Interactome licensing</a>
				</li>
			
				<li>
					
					<a href='/d/48' class='thread-title'>Text as a resource for network population?</a>
				</li>
			
				</ul>
			</div>
		

		
			<div class="details">Cite this as</div>
			<span style="font-size:12px;">
				Daniel Himmelstein, Dexter Hadley, Alexey Strokach (2015) Creating a catalog of protein interactions. <i>Thinklab.</i> doi:<a href='https://doi.org/10.15363/thinklab.d85'>10.15363/thinklab.d85</a>
			</span>
		

		<div class="details">License</div>
		<p><a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0;width:88px;height:31px;" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a></p>

		<div class="details">Share</div>
		
			
				<a class="share-button fb" onclick="javascript:void(window.open('https://www.facebook.com/sharer/sharer.php?u=https%3A//thinklab.com/discussion/creating-a-catalog-of-protein-interactions/85','','width=600,height=460,menubar=no,status=no'));" href="#"></a>
<a class="share-button twitter" onclick="javascript:void(window.open('https://twitter.com/share?text=Creating%20a%20catalog%20of%20protein%20interactions&url=https%3A//thinklab.com/discussion/creating-a-catalog-of-protein-interactions/85','','width=600,height=300,menubar=no,status=no'));" href="#"></a>
<a class="share-button google" onclick="javascript:void(window.open('https://plus.google.com/share?url=https%3A//thinklab.com/discussion/creating-a-catalog-of-protein-interactions/85','','width=510,height=460,menubar=no,status=no'));" href="#"></a>

			
		

	</td>
</tr></table>


		</div>


		
	    	<div class="footer">
<!-- <div style="background:#dedede;">
	<div class="main-centered-div text-center" style="padding:20px;">
		<h4 style="margin-bottom:0px;font-weight:bold;color:gray;">Feedback? Questions? Email us: <a href="mailto:hello@thinklab.com">hello@thinklab.com</a></h4>
	</div>
</div> -->
<div class="main-footer">
	<div class="main-centered-div">

<table class="full-width"><tr>
	<td class="footer-nav">

        <div class="foot-head">Explore</div>
        <a href="/proposals">Proposals</a><br>
        <a href="/projects">Projects</a><br>
        <a href="/publications">Article discussion</a><br>
        <a href="/discussion">All discussion</a><br>
        <a href="/topics">Topics</a><br>
        <!-- <a href="/funders">Funders</a><br> -->
        <a href="/funding">Funding opportunities</a><br>

	</td>
	<td class="footer-nav">

        <!-- <div class="foot-head">Help &amp; feedback</div>	 -->
        <!-- <div class="foot-head">Intro</div>
        <a href="/benefits/for-science">Why</a><br>
        <a href="/how-it-works">How it works</a><br>

<br> -->

        <div class="foot-head">Community</div>
        <a href="/leaderboard">Leaderboard</a><br>
        <a href="/p/meta">Thinklab Meta</a><br>
				<a href="/help/faq">Help</a><br>
				<!-- <a href="/new-project">Start a proposal</a><br> -->
        <!-- <a href="/benefits/proposal">Benefits</a><br> -->
        <!-- <a href="/funding">Get funded</a><br> -->
        <!-- <a href="/participate">Become a contributor</a><br> -->
        <!-- <a href="/for-funders">For science funders</a><br> -->
        <!-- <a href="/for-institutions">For institutions</a><br> -->

	</td>
	<td class="footer-nav">



        <div class="foot-head">About</div>
        <a href="/about">Overview</a><br>
        <a href="/benefits">Benefits</a><br>
        <a href="/how-it-works">How it works</a><br>
        <a href="/about/story">Our story</a><br>
        <a href="/blog">Our blog</a><br>
        <a href="/about/jobs">Join the team</a><br>
        <br>

        <!-- <div class="foot-head">Help</div>
        <a href="/help">Help</a><br>
		 -->


	</td>
	<td class="footer-nav">
		<!-- <a href="http://twitter.com/thinklab" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/twitter_com.png" width="40"/></a>
		<a href="https://www.facebook.com/pages/Thinklab/310895039088185" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/facebook_com.png" width="40"/></a>
		<a href="https://plus.google.com/115578983466800753402/" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/plus_google_com.png" width="40"/></a>
		<a href="https://www.linkedin.com/company/thinklab-co" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/linkedin_com.png" width="40"/></a> -->
		<br><br>
		<span class="light" style='font-size:12px;'>&copy; 2015 Thinklab</span>

	</td>

</tr></table>

</div>
</div>
</div>

		

	</div>


  <script></script>
  <script>
  var MEDIA_URL = 'https://think-lab.s3.amazonaws.com/m/';
  var STATIC_URL = 'https://think-lab.s3.amazonaws.com/s/';
  var page_is_authenticated = false;
  var page_page = 'discussion';
  var page_profile_id = undefined;
</script>



	<!-- Note different script type. MUST come before MathJax.js -->
	<script type="text/x-mathjax-config">
	    MathJax.Hub.Config({
	        "HTML-CSS": { preferredFont: "TeX", availableFonts: ["STIX","TeX"],
	                        scale: 110, linebreaks: { automatic:true },
	                        EqnChunk: (MathJax.Hub.Browser.isMobile ? 10 : 50) },
	        tex2jax: { inlineMath: [ ["$$", "$$"], ["\\(","\\)"] ],
	                    displayMath: [ ["$$$","$$$"], ["\\[", "\\]"] ],
	                    processEscapes: true,}
	            });

	</script>
    <script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>





  <script src="https://code.jquery.com/jquery-1.10.2.min.js"></script>
  <script src="https://code.jquery.com/ui/1.11.4/jquery-ui.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" integrity="sha384-0mSbJDEHialfmuBBQP6A4Qrprq5OVfW37PRR3j5ELqxss1yVqOtnepnHVP9aJ7xS" crossorigin="anonymous"></script>
  <script src="https://think-lab.s3.amazonaws.com/s/dist/main.min.js"></script>
  



  
	
  
  
  
<script>
$(function() {
  recordHit('thread', 85);
});
</script>



</body>
</html>
