<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8" /><script type="text/javascript">window.NREUM||(NREUM={}),__nr_require=function(e,n,t){function r(t){if(!n[t]){var o=n[t]={exports:{}};e[t][0].call(o.exports,function(n){var o=e[t][1][n];return r(o||n)},o,o.exports)}return n[t].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<t.length;o++)r(t[o]);return r}({1:[function(e,n,t){function r(){}function o(e,n,t){return function(){return i(e,[c.now()].concat(u(arguments)),n?null:this,t),n?void 0:this}}var i=e("handle"),a=e(2),u=e(3),f=e("ee").get("tracer"),c=e("loader"),s=NREUM;"undefined"==typeof window.newrelic&&(newrelic=s);var p=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],d="api-",l=d+"ixn-";a(p,function(e,n){s[n]=o(d+n,!0,"api")}),s.addPageAction=o(d+"addPageAction",!0),s.setCurrentRouteName=o(d+"routeName",!0),n.exports=newrelic,s.interaction=function(){return(new r).get()};var m=r.prototype={createTracer:function(e,n){var t={},r=this,o="function"==typeof n;return i(l+"tracer",[c.now(),e,t],r),function(){if(f.emit((o?"":"no-")+"fn-start",[c.now(),r,o],t),o)try{return n.apply(this,arguments)}finally{f.emit("fn-end",[c.now()],t)}}}};a("setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(e,n){m[n]=o(l+n)}),newrelic.noticeError=function(e){"string"==typeof e&&(e=new Error(e)),i("err",[e,c.now()])}},{}],2:[function(e,n,t){function r(e,n){var t=[],r="",i=0;for(r in e)o.call(e,r)&&(t[i]=n(r,e[r]),i+=1);return t}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],3:[function(e,n,t){function r(e,n,t){n||(n=0),"undefined"==typeof t&&(t=e?e.length:0);for(var r=-1,o=t-n||0,i=Array(o<0?0:o);++r<o;)i[r]=e[n+r];return i}n.exports=r},{}],4:[function(e,n,t){n.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(e,n,t){function r(){}function o(e){function n(e){return e&&e instanceof r?e:e?f(e,u,i):i()}function t(t,r,o,i){if(!d.aborted||i){e&&e(t,r,o);for(var a=n(o),u=m(t),f=u.length,c=0;c<f;c++)u[c].apply(a,r);var p=s[y[t]];return p&&p.push([b,t,r,a]),a}}function l(e,n){v[e]=m(e).concat(n)}function m(e){return v[e]||[]}function w(e){return p[e]=p[e]||o(t)}function g(e,n){c(e,function(e,t){n=n||"feature",y[t]=n,n in s||(s[n]=[])})}var v={},y={},b={on:l,emit:t,get:w,listeners:m,context:n,buffer:g,abort:a,aborted:!1};return b}function i(){return new r}function a(){(s.api||s.feature)&&(d.aborted=!0,s=d.backlog={})}var u="nr@context",f=e("gos"),c=e(2),s={},p={},d=n.exports=o();d.backlog=s},{}],gos:[function(e,n,t){function r(e,n,t){if(o.call(e,n))return e[n];var r=t();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(e,n,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return e[n]=r,r}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],handle:[function(e,n,t){function r(e,n,t,r){o.buffer([e],r),o.emit(e,n,t)}var o=e("ee").get("handle");n.exports=r,r.ee=o},{}],id:[function(e,n,t){function r(e){var n=typeof e;return!e||"object"!==n&&"function"!==n?-1:e===window?0:a(e,i,function(){return o++})}var o=1,i="nr@id",a=e("gos");n.exports=r},{}],loader:[function(e,n,t){function r(){if(!x++){var e=h.info=NREUM.info,n=d.getElementsByTagName("script")[0];if(setTimeout(s.abort,3e4),!(e&&e.licenseKey&&e.applicationID&&n))return s.abort();c(y,function(n,t){e[n]||(e[n]=t)}),f("mark",["onload",a()+h.offset],null,"api");var t=d.createElement("script");t.src="https://"+e.agent,n.parentNode.insertBefore(t,n)}}function o(){"complete"===d.readyState&&i()}function i(){f("mark",["domContent",a()+h.offset],null,"api")}function a(){return E.exists&&performance.now?Math.round(performance.now()):(u=Math.max((new Date).getTime(),u))-h.offset}var u=(new Date).getTime(),f=e("handle"),c=e(2),s=e("ee"),p=window,d=p.document,l="addEventListener",m="attachEvent",w=p.XMLHttpRequest,g=w&&w.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:w,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var v=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-1039.min.js"},b=w&&g&&g[l]&&!/CriOS/.test(navigator.userAgent),h=n.exports={offset:u,now:a,origin:v,features:{},xhrWrappable:b};e(1),d[l]?(d[l]("DOMContentLoaded",i,!1),p[l]("load",r,!1)):(d[m]("onreadystatechange",o),p[m]("onload",r)),f("mark",["firstbyte",u],null,"api");var x=0,E=e(4)},{}]},{},["loader"]);</script><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info={"beacon":"bam.nr-data.net","queueTime":0,"licenseKey":"cd1101aa94","agent":"","transactionName":"bgcDMkBZW0NZBkdcWVdNJxNcW0FZVwscV1dKB08QW11CQxYTWlBBZgEOC19dW0QCEVtHU1gG","applicationID":"6369297","errorBeacon":"bam.nr-data.net","applicationTime":556}</script>
    <title>Unifying disease vocabularies | Thinklab</title>
	  <link rel="shortcut icon" href="https://think-lab.s3.amazonaws.com/s/img/favicon.ico" type="image/x-icon" />
    <meta property="og:image" content="https://think-lab.s3.amazonaws.com/s/img" />
    
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
  <link rel="stylesheet" href="https://code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css" />

<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/highlightjs/github.css" />
<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/css/css.css" />
<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/css/bootstrap-multiselect.css" />
<!-- <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'> -->

    
    
    
    
  
    <meta name="citation_doi" content="10.15363/thinklab.d44">
    <meta name="citation_title" content="Unifying disease vocabularies">
    <meta name="citation_online_date" content="March 30, 2015">
    <meta name="citation_author" content="Daniel Himmelstein, Tong Shu Li">
    <meta name="citation_journal_title" content="Thinklab">
    <meta name="citation_publisher" content="Thinklab">
  

    


	<!-- GoogleAnalytics -->
	<script>
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
	(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
	m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

	ga('create', 'UA-48550230-1', 'thinklab.com');
	ga('require', 'displayfeatures');
	
	ga('send', 'pageview');

	</script>






</head>
<body >
  <form id="form_action" method="post">
    <input type='hidden' name='csrfmiddlewaretoken' value='pQNiZQX4gF2vyFeXrnbUl9EmuTWA5MG1' />
</form>

	

	<div class="body-wrapper">

		


<!-- <div style="height:5px;background-image:url('https://think-lab.s3.amazonaws.com/s/img/top-bg.gif');"></div> -->
<div class="gradient">
	<div class="main-centered-div header-container">

<table class="full-width" style="height:60px;"><tr>
	<td style="width:200px;vertical-align:middle;padding-bottom:15px;">
		<a href="/" style="font-weight:normal;color:#212121;"><img src="https://think-lab.s3.amazonaws.com/s/img/logo.png" width="160"></a>
	</td><td style="vertical-align:middle;text-align:left;padding-top:6px;width:100%;padding-left:40px;">
		<ul id="not-search-block"  class="main-header">


			


				<li class="dropdown">
					<a class="dropdown-toggle" data-toggle="dropdown" href="#">Browse <span class="caret"></span></a>
					<ul class="dropdown-menu" role="menu">
						<li><a href="/proposals">Proposals</a></li>
						<li><a href="/projects">Projects</a></li>
						<li><a href="/publications">Article discussion</a></li>
						<li><a href="/discussion">All discussion</a></li>
					</ul>
				</li>
				<li><a href="/leaderboard">Leaderboard</a></li>

			
				<li><a href="/about">About</a></li>
			


		</ul>

    <ul id="search-block" class="main-header" style="display: none; margin-right: 30px;">
        <li class="full-width">
            <form method="get" action="/search">
			<div class="input-group full-width">
				<input name="q" type="text" class="form-control search-input">
			</div>
            </form>
        </li>
    </ul>

	</td><td style="vertical-align:middle;text-align:right;padding-top:6px; min-width: 1%; white-space: nowrap;">
		<ul class="main-header pull-right" style="padding-left: 0;">
    		
                <li><a href="/login" style="color:silver;">Login</a></li>
                <li style="margin-right:0"><a class="btn btn-default btn-sm btn-upper" href="/signup">Join</a></li>
            

        </ul>
    </ul>
	</td>
</tr></table>

<!-- <div id="global-search-div" class="main-centered-div" style="display:none;">
	<input type="text" value="" class="thread-search form-control" placeholder="Search" data-action="/discussion">
</div> -->

	</div>
</div>

		
	  



	  


		

		<div class="dark-head"><div class="main-centered-div">

			

		</div></div>
		
		<div class="main-centered-div" style="margin-top:40px;padding-bottom:340px">
	    

<div style="margin-top:-20px; margin-bottom:20px;">

    <a class="side-box" href="/p/rephetio" style="background-color:#ebfdff;"><div class="side-type">Project:</div> <div class="side-title">Rephetio: Repurposing drugs on a hetnet [rephetio]</div></a><br>



</div>

<table style="margin-top:0px;"><tr>
	<td class="thread-table-content">

		<table class="full-width">
			
<tr class="comment-tr target-hl" id="1" data-comment-id="94">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="94"></a>

    

			<h2 class="page-title" style="margin-bottom:5px;margin-top:0px !important;">Unifying disease vocabularies</h2>

			<div style="margin-top:5px;color:gray;margin-bottom:20px;font-size:13px;">
				<a style="font-weight:normal;color:#212121" href="/u/dhimmel"><strong>Daniel Himmelstein</strong></a>
				<span class="is-researcher">Researcher</span>
				&nbsp; <a href="#1" class="light">829 days ago</a>
				
			</div>

      

          <hr>
		

    <p>This discussion will explore how to unify the variety of disease vocabularies used by our resources. These resources are listed below:</p>

<table class="table markdown-table"><thead><tr><th>type</th><th>resource</th><th>vocabulary</th></tr></thead><tbody><tr><td>indications</td><td>MEDI <span class="citation">[<a href="https://doi.org/10.1136/amiajnl-2012-001431" class="citation" data-key="10.1136/amiajnl-2012-001431">1</a>]</span></td><td>ICD-9</td></tr><tr><td>indications</td><td>LabeledIn <span class="citation">[<a href="https://doi.org/10.1016/j.jbi.2014.08.004" class="citation" data-key="10.1016/j.jbi.2014.08.004">2</a>, <a href="https://doi.org/10.1093/database/bav016" class="citation" data-key="10.1093/database/bav016">3</a>]</span></td><td>UMLS <span class="citation">[<a href="https://doi.org/10.1093/nar/gkh061" class="citation" data-key="10.1093/nar/gkh061">4</a>]</span></td></tr><tr><td>transcriptional signatures</td><td><a href="http://dev.stargeo.io/">STAR-GEO</a></td><td>custom</td></tr><tr><td>symptoms</td><td>HSDN <span class="citation">[<a href="https://doi.org/10.1038/ncomms5212" class="citation" data-key="10.1038/ncomms5212">5</a>]</span></td><td><a href="//www.nlm.nih.gov/mesh/filelist.html">MeSH</a> <span class="citation">[<a href="https://doi.org/10.1001/jama.271.14.1103" class="citation" data-key="10.1001/jama.271.14.1103">6</a>]</span> (2011 release)</td></tr><tr><td>gene associations</td><td><a href="//het.io/disease-genes/downloads/">het.io</a> <span class="citation">[<a href="https://doi.org/10.1101/011569" class="citation" data-key="10.1101/011569">7</a>]</span></td><td><a href="//disease-ontology.org/">Disease Ontology</a> <span class="citation">[<a href="https://doi.org/10.1093/nar/gkr972" class="citation" data-key="10.1093/nar/gkr972">8</a>, <a href="https://doi.org/10.1093/nar/gku1011" class="citation" data-key="10.1093/nar/gku1011">9</a>]</span></td></tr><tr><td>pathophysiology</td><td><a href="//het.io/disease-genes/downloads/">het.io</a> <span class="citation">[<a href="https://doi.org/10.1101/011569" class="citation" data-key="10.1101/011569">7</a>]</span></td><td><a href="//disease-ontology.org/">Disease Ontology</a> <span class="citation">[<a href="https://doi.org/10.1093/nar/gkr972" class="citation" data-key="10.1093/nar/gkr972">8</a>, <a href="https://doi.org/10.1093/nar/gku1011" class="citation" data-key="10.1093/nar/gku1011">9</a>]</span></td></tr><tr><td>tissue localization</td><td><a href="//het.io/disease-genes/downloads/">het.io</a> <span class="citation">[<a href="https://doi.org/10.1101/011569" class="citation" data-key="10.1101/011569">7</a>]</span></td><td><a href="//disease-ontology.org/">Disease Ontology</a> <span class="citation">[<a href="https://doi.org/10.1093/nar/gkr972" class="citation" data-key="10.1093/nar/gkr972">8</a>, <a href="https://doi.org/10.1093/nar/gku1011" class="citation" data-key="10.1093/nar/gku1011">9</a>]</span></td></tr></tbody></table>

<p>Our current plan is to use the Disease Ontology (DO) as our primary vocabulary. Therefore, we will have to map resources to DO terms.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="2" data-comment-id="98">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="98"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#2" class="light">827 days</a>
          
        </span>
      </div>

    

    <h1>Disease Ontology Resources</h1>

<p>Since we plan to use the DO as our primary disease vocabulary, I thought I would keep track of related papers and projects here. These may be slightly off-topic but valuable to keep track of.</p>

<table class="table markdown-table"><thead><tr><th>name</th><th>description</th><th>cite</th></tr></thead><tbody><tr><td><a href="//disease-ontology.org">Disease Ontology</a></td><td>Main resource</td><td><span class="citation">[<a href="https://doi.org/10.1093/nar/gkr972" class="citation" data-key="10.1093/nar/gkr972">1</a>, <a href="https://doi.org/10.1093/nar/gku1011" class="citation" data-key="10.1093/nar/gku1011">2</a>]</span></td></tr><tr><td><a href="http://django.nubic.northwestern.edu/fundo/">DOLite</a></td><td>DO terms are grouped using associated-gene similarity to produce a simplified vocabulary with little redundancy.</td><td><span class="citation">[<a href="https://doi.org/10.1093/bioinformatics/btp193" class="citation" data-key="10.1093/bioinformatics/btp193">3</a>]</span></td></tr><tr><td>DO_cancer_slim</td><td>Created a DO subset named <code>TOPNodes_DOcancerslim</code> composed of 63 non-redundant upper-level cancer terms</td><td><span class="citation">[<a href="https://doi.org/10.1093/database/bav032" class="citation" data-key="10.1093/database/bav032">4</a>]</span></td></tr><tr><td><a href="http://doa.nubic.northwestern.edu/pages/search.php">DOAF</a></td><td>Provides gene annotations (extracted from GeneRIF) to the Disease Ontology.</td><td><span class="citation">[<a href="https://doi.org/10.1371/journal.pone.0049686" class="citation" data-key="10.1371/journal.pone.0049686">5</a>]</span></td></tr><tr><td><a href="http://django.nubic.northwestern.edu/fundo/">FunDO</a></td><td>Provides gene annotations (extracted from GeneRIF) to the Disease Ontology. Uses diseases from DOLite. Potentially outdated.</td><td><span class="citation">[<a href="https://doi.org/10.1186/1471-2164-10-S1-S6" class="citation" data-key="10.1186/1471-2164-10-S1-S6">6</a>]</span></td></tr><tr><td><a href="https://github.com/GuangchuangYu/DOSE">DOSE</a></td><td>DOSE is an R package to compute semantic similarity between DO terms. The result is pairwise similarities between DO terms based only on the ontology structure.</td><td><span class="citation">[<a href="https://doi.org/10.1093/bioinformatics/btu684" class="citation" data-key="10.1093/bioinformatics/btu684">7</a>]</span></td></tr><tr><td><a href="http://cran.r-project.org/web/packages/DOSim/index.html">DOsim</a></td><td>Similar to DOSE but allegedly unmaintained or outdated</td><td><span class="citation">[<a href="https://doi.org/10.1186/1471-2105-12-266" class="citation" data-key="10.1186/1471-2105-12-266">8</a>]</span></td></tr></tbody></table>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="3" data-comment-id="121">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="121"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#3" class="light">820 days</a>
          
        </span>
      </div>

    

    <h1>Seeking a Slim DO with distinct terms.</h1>

<p>The Disease Ontology is a hierarchy of human diseases. However, our current method has been designed for <em>distinct</em> nodes (especially with respect to diseases and compounds, since we will be predicting indications). By distinct we mean non-redundant — in a non-redundant set of terms, no terms should be an ancestor or descendant of any other term. </p>

<p>Additionally, we would like to pick terms at an appropriate level of specificity. Below, I show lineages of DO terms and bold the term whose specificity I prefer:</p>

<ul><li>cancer &gt; organ system cancer &gt; respiratory system cancer &gt; <strong>lung cancer</strong> &gt; Pancoast tumor &gt; lung superior sulcus carcinoma</li><li>neurodegenerative disease &gt; demyelinating disease &gt; <strong>multiple sclerosis</strong> &gt; relapsing-remitting multiple sclerosis</li><li>glucose metabolism disease &gt; diabetes mellitus &gt; <strong>type 2 diabetes mellitus</strong> &gt; diabetic peripheral angiopathy</li></ul>

<p>Ideally, we chose a level of specificity such that:</p>

<ul><li>a term and its descendants form a cohesive and differentiated disease concept</li><li>disease data is collected at a similar level of specificity</li><li>sufficient data exists for the term, after propagating data annotated to more specific terms</li></ul>

<p>These are competing aims and we will most likely have to make difficult subjective decisions. In the past <span class="citation">[<a href="https://doi.org/10.1101/011569" class="citation" data-key="10.1101/011569">1</a>]</span>, we identified <a href="http://het.io/disease-genes/downloads/files/diseases.txt">108 distinct, complex diseases</a> by investigating only diseases with GWAS. However, GWAS may be too restrictive of a filter for the present study. We are more concerned with omitting diseases that would be poorly connected in the network. Some diseases may be well connected but lacking GWAS. We also would prefer to focus on diseases with indications as indications are needed to train our model.</p>

<p>Two previous DO studies may be helpful here:</p>

<ol><li><strong>DOLite</strong> took a data-driven approach to selecting a consolidated set of DO terms <span class="citation">[<a href="https://doi.org/10.1093/bioinformatics/btp193" class="citation" data-key="10.1093/bioinformatics/btp193">2</a>]</span>. I have not been able to locate the list of DO identifiers composing DOLite. These terms may also be obsolete as the project is dated. <em><strong>Any information regarding DOLite would be appreciated.</strong></em></li><li><strong>DO_cancer_slim</strong> created a DO subset named <code>TOPNodes_DOcancerslim</code> composed of 63 non-redundant upper-level cancer terms <span class="citation">[<a href="https://doi.org/10.1093/database/bav032" class="citation" data-key="10.1093/database/bav032">3</a>]</span>. <em><strong>We would like a similar term list but encompassing all diseases, not just cancers.</strong></em></li></ol>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="4" data-comment-id="130">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="130"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#4" class="light">819 days</a>
          
        </span>
      </div>

    

    <h1>Selecting the top nodes from DO Cancer Slim</h1>

<p>I emailed <a href="http://medschool.umaryland.edu/FACULTYRESEARCHPROFILE/viewprofile.aspx?id=20337">Lynn Schriml</a>, a lead investigator for the DO, asking about the creation of a consolidated and distinct set of high-level cancer terms. This work was <a href="https://dx.doi.org/10.1093/database/bav032">recently published</a> in <em>Database</em> <span class="citation">[<a href="https://doi.org/10.1093/database/bav032" class="citation" data-key="10.1093/database/bav032">1</a>]</span>. I wrote:</p>

<blockquote><p>My main question is how did you decide which terms should be included in TopNodes_DOcancerslim? I want to generate a similar top-level term set but for all diseases. Was this a very difficult task that required medical expertise? Do you have any plans to create a DO-wide slim set? (Ideally, I would prefer to rely on an existing effort than to recreate the wheel).</p></blockquote>

<p><strong>Response by Lynn Schriml</strong> (posted with permission):</p>

<p>We are not planning on creating a DO wide slim at this time.</p>

<p>Creating the DO cancer slim and the TOP nodes slim was a couple of months work by a team, including MDs, cancer and disease experts.</p>

<p>We started with a set of terms from multiple cancer sources that we wanted align. We worked to identify how each term mapped to the current nomenclature of disease (disease names change over time), we then worked to define each term, figure out if the term was represented in DO, and where to place the term in DO, then we added the terms, creating DO definitions. We also reviewed and edited related terms. Once we had the larger set of terms defined, we looked at their parent nodes up to the top node for cancer. </p>

<p>We wanted to identify body system level parents that reflected both the most specific we could be (not mapping the new TOP node parent all the way up to cancer), and for the TOP node to be biologically informative. There was also much discussion in our work group to finalize these choices. When that was done, we then edited the DO file, adding the new subtypes (slims) and adding each term in (one at a time).</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="5" data-comment-id="131">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="131"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#5" class="light">819 days</a>
          
        </span>
      </div>

    

    <h1>Reconstructing DO Lite</h1>

<p>DO Lite is an outdated project which provided a consolidated disease terminology <span class="citation">[<a href="https://doi.org/10.1093/bioinformatics/btp193" class="citation" data-key="10.1093/bioinformatics/btp193">1</a>]</span>. The only data release we could find is a mapping of disease names to implicated genes. Unfortunately, <a href="http://django.nubic.northwestern.edu/fundo/media/data/do_lite.txt">this dataset</a> omits the actual DO identifiers.</p>

<p>We extracted these disease names found 561 different terms. We <a href="http://nbviewer.ipython.org/github/dhimmel/disease-ontology/blob/gh-pages/dolite/dolite.ipynb">used a text matching paradigm to match</a> these disease names to current DO terms. The paradigm consisted of the following steps:</p>

<ol><li>creating a mapping of names (including synonyms) to DO identifiers [<a href="http://git.dhimmel.com/disease-ontology/data/term-names.tsv">tsv</a>]</li><li>mapping DOLite names to current DO names (via exact lowercase match) [<a href="http://git.dhimmel.com/disease-ontology/dolite/dolite_to_doid.tsv">tsv</a>]</li></ol>

<p>A majority (<code>66.3% =  372 / 561</code>) of DOLite terms were matched to a current DO identifier. We may consider using these terms as a reference when manually constructing a DO Slim.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="6" data-comment-id="144">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="144"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#6" class="light">812 days</a>
          
        </span>
      </div>

    

    <h1>Creating a slim DO</h1>

<p>We created a slim DO with <a href="https://github.com/dhimmel/disease-ontology/blob/75050ea2d4f60e745d3f3578ae03560a2cc0e444/data/slim-terms.tsv" title="TSV of DO Slim diseases">137 terms</a> where:</p>

<ol><li>no terms were descendants/ancestors of other terms</li><li>terms were specific enough to be clinically relevant</li><li>terms were general enough to be well annotated</li></ol>

<p>To create this slim term set, we combined the diseases from:</p>

<ol><li>hetio <span class="citation">[<a href="https://doi.org/10.1371/journal.pcbi.1004259" class="citation" data-key="10.1371/journal.pcbi.1004259">1</a>]</span> — 108 complex diseases contained in the GWAS Catalog.</li><li>TOPNodes_DOcancerslim <span class="citation">[<a href="https://doi.org/10.1093/database/bav032" class="citation" data-key="10.1093/database/bav032">2</a>]</span> — a body system focused set of 63 cancer terms.</li></ol>

<p>We found that both sources contained overlapping nodes, and we removed 34 nodes to create a non-overlapping term set. We chose the following rules to resolve overlapping nodes:</p>

<ol><li>For cancers in TOPNodes_DOcancerslim, retain only the most specific cancer</li><li>Remove hetio terms that descend from TOPNodes_DOcancerslim terms</li><li>For the remaining overlapping hetio terms, choose the term with greater clinical interest or GWAS annotations. For 4 out of the 5 conflicts under this rule, we chose to retain the more general term.</li></ol>

<h3>Other notes:</h3>

<p>The repository with our DO analysis is <a href="https://github.com/dhimmel/disease-ontology">here</a> and contains notebooks for <a href="https://github.com/dhimmel/disease-ontology/blob/75050ea2d4f60e745d3f3578ae03560a2cc0e444/DO-xrefs.ipynb">extracting xrefs</a> and <a href="https://github.com/dhimmel/disease-ontology/blob/75050ea2d4f60e745d3f3578ae03560a2cc0e444/slim.ipynb">evaluating our slim DO</a> <span class="citation">[<a href="https://doi.org/10.5281/zenodo.45584" class="citation" data-key="10.5281/zenodo.45584">3</a>]</span>.</p>

<p>We plan to propagate annotations from more specific terms to our slim terms. To facilitate this process, we created a <a href="https://github.com/dhimmel/disease-ontology/blob/75050ea2d4f60e745d3f3578ae03560a2cc0e444/data/xrefs-prop-slim.tsv">propagated DO slim xref mapping file</a>.</p>

<p>Pleural cancer (<code>DOID:9917</code>) was a TOPNodes_DOcancerslim term but was not found in the ontology version we downloaded (subversion revision 2810).</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="7" data-comment-id="906">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="906"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#7" class="light">502 days</a>
          
        </span>
      </div>

    

    <h1>Redundant terms removed from the slim DO</h1>

<p>My <a href="#6">above post</a> on creating the slim DO didn't specify which diseases were removed to "resolve overlapping nodes". The table below shows which diseases we removed and why (rules above). The exclusions counts by rule are: 7 for rule 1, 22 for rule 2, and 5 for rule 3.</p>

<table class="table markdown-table"><thead><tr><th>ID</th><th>Name</th><th>Source</th><th>Removed by</th></tr></thead><tbody><tr><td>DOID:201</td><td>Connective tissue cancer</td><td>DOcancerslim</td><td>rule 1</td></tr><tr><td>DOID:10155</td><td>Intestinal cancer</td><td>DOcancerslim</td><td>rule 1</td></tr><tr><td>DOID:5672</td><td>Large intestine cancer</td><td>DOcancerslim</td><td>rule 1</td></tr><tr><td>DOID:3119</td><td>Gastrointestinal system cancer</td><td>DOcancerslim</td><td>rule 1</td></tr><tr><td>DOID:8618</td><td>Oral cavity cancer</td><td>DOcancerslim</td><td>rule 1</td></tr><tr><td>DOID:170</td><td>Endocrine gland cancer</td><td>DOcancerslim</td><td>rule 1</td></tr><tr><td>DOID:3996</td><td>Urinary system cancer</td><td>DOcancerslim</td><td>rule 1</td></tr><tr><td>DOID:3459</td><td>breast carcinoma</td><td>hetio</td><td>rule 2</td></tr><tr><td>DOID:10286</td><td>prostate carcinoma</td><td>hetio</td><td>rule 2</td></tr><tr><td>DOID:1040</td><td>chronic lymphocytic leukemia</td><td>hetio</td><td>rule 2</td></tr><tr><td>DOID:3905</td><td>lung carcinoma</td><td>hetio</td><td>rule 2</td></tr><tr><td>DOID:1909</td><td>melanoma</td><td>hetio</td><td>rule 2</td></tr><tr><td>DOID:4001</td><td>ovarian carcinoma</td><td>hetio</td><td>rule 2</td></tr><tr><td>DOID:1107</td><td>esophageal carcinoma</td><td>hetio</td><td>rule 2</td></tr><tr><td>DOID:4007</td><td>bladder carcinoma</td><td>hetio</td><td>rule 2</td></tr><tr><td>DOID:289</td><td>endometriosis</td><td>hetio</td><td>rule 2</td></tr><tr><td>DOID:4450</td><td>renal cell carcinoma</td><td>hetio</td><td>rule 2</td></tr><tr><td>DOID:769</td><td>neuroblastoma</td><td>hetio</td><td>rule 2</td></tr><tr><td>DOID:8567</td><td>Hodgkin's lymphoma</td><td>hetio</td><td>rule 2</td></tr><tr><td>DOID:3963</td><td>thyroid carcinoma</td><td>hetio</td><td>rule 2</td></tr><tr><td>DOID:9538</td><td>multiple myeloma</td><td>hetio</td><td>rule 2</td></tr><tr><td>DOID:9952</td><td>acute lymphocytic leukemia</td><td>hetio</td><td>rule 2</td></tr><tr><td>DOID:5517</td><td>stomach carcinoma</td><td>hetio</td><td>rule 2</td></tr><tr><td>DOID:684</td><td>hepatocellular carcinoma</td><td>hetio</td><td>rule 2</td></tr><tr><td>DOID:1380</td><td>endometrial cancer</td><td>hetio</td><td>rule 2</td></tr><tr><td>DOID:4905</td><td>pancreatic carcinoma</td><td>hetio</td><td>rule 2</td></tr><tr><td>DOID:4960</td><td>bone marrow cancer</td><td>hetio</td><td>rule 2</td></tr><tr><td>DOID:706</td><td>mature B-cell neoplasm</td><td>hetio</td><td>rule 2</td></tr><tr><td>DOID:8552</td><td>chronic myeloid leukemia</td><td>hetio</td><td>rule 2</td></tr><tr><td>DOID:5844</td><td>myocardial infarction</td><td>hetio</td><td>rule 3</td></tr><tr><td>DOID:6713</td><td>cerebrovascular disease</td><td>hetio</td><td>rule 3</td></tr><tr><td>DOID:11829</td><td>degenerative myopia</td><td>hetio</td><td>rule 3</td></tr><tr><td>DOID:13641</td><td>exfoliation syndrome</td><td>hetio</td><td>rule 3</td></tr><tr><td>DOID:3324</td><td>mood disorder</td><td>hetio</td><td>rule 3</td></tr></tbody></table>

<p>See the <a href="https://github.com/dhimmel/disease-ontology/blob/5cb93c38568536222b0a14fbcb7fb644a348931d/data/slim-terms.tsv">remaining 137 diseases here</a>.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="8" data-comment-id="1360">
  <td class="comment-profile-avatar">
		<a href="/u/tongli"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/176_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="1360"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/tongli">Tong Shu Li</a>
				
				<span class="light">&nbsp;<a href="#8" class="light">289 days</a>
          
        </span>
      </div>

    

    <p>Hi Daniel,</p>

<p>In cell 2 of <a href="https://github.com/dhimmel/integrate/blob/master/compile/DaG-association.ipynb">this notebook</a> I notice that you're reading the <code>data/slim-terms.tsv</code> file from the <code>dhimmel/disease-ontology</code> repository. However, I could not determine how the <code>slim-terms.tsv</code> file was generated by looking at the files in the <code>disease-ontology</code> repository.</p>

<p>The only mention of a <code>slim-terms.tsv</code> file in the DO repository is in cell 42 of <a href="https://github.com/dhimmel/disease-ontology/blob/72614ade9f1cc5a5317b8f6836e1e464b31d5587/slim.ipynb">this notebook</a>, but here the file seems to have been created already.</p>

<p>Do you know how this file was generated? I am trying to replicate the Rephetio workflow with up-to-date data.</p>

<p>Thanks!</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="9" data-comment-id="1362">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="1362"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#9" class="light">287 days</a>
          
        </span>
      </div>

    

    <p><a href="/u/tongli" class="username">@tongli</a>, <code>slim-terms.tsv</code> was created manually according to the <a href="#6">post above</a>. In short, I created the table in a <a href="https://docs.google.com/spreadsheets/d/1jVRHiBH4lvTHRVpVBm1UbWt7y_0jIYpVWKDWkM_Lnq8/edit?usp=sharing" title="DO Slim on Google Drive">spreadsheet</a> by combining the two disease sets (hetio-dag <span class="citation">[<a href="https://doi.org/10.1371/journal.pcbi.1004259" class="citation" data-key="10.1371/journal.pcbi.1004259">1</a>]</span> and TOPNodes_DOcancerslim <span class="citation">[<a href="https://doi.org/10.1093/database/bav032" class="citation" data-key="10.1093/database/bav032">2</a>]</span>) and removing any problematic terms based on ad hoc rules. The pathophysiology classification was manually done by <a href="/u/pouyakhankhanian" class="username">@pouyakhankhanian</a> and I — I believe primarily for hetio-dag <span class="citation">[<a href="https://doi.org/10.1371/journal.pcbi.1004259" class="citation" data-key="10.1371/journal.pcbi.1004259">1</a>]</span> — and was not used in Project Rephetio.</p>

<blockquote><p>I am trying to replicate the Rephetio workflow with up-to-date data.</p></blockquote>

<p>The problem with the slim approach we took is that it's difficult to automate and potentially requires some clinical expertise. In other words, it's difficult to update. Therefore, I'm thinking of alternatives for the future. One option would be to include the entire ontology. You could then pick your slim term set at analysis time rather than network construction time. This would make the network more versatile. Since Cypher is capable of <a href="http://graphaware.com/graphaware/2015/05/19/neo4j-cypher-variable-length-relationships-by-example.html">variable length relationship</a> queries, you could do on-the-fly propagation. However, this would require some additional implementation and theoretical work.</p>

<p>Along these ends, I started playing around with <a href="https://github.com/greenelab/hetontology">creating Hetontology</a> — a public Neo4j database of open biomedical ontologies. Depending on how much ground you're looking to break, this may be of interest.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

    		
		</table>

    

    &nbsp;<br>
    <a class="btn btn-primary btn-lg" href="/signup?next=/discussion/unifying-disease-vocabularies/44&suppress_message=1" style>Join to Reply</a>


		<table class="references table-td-5" style="display:none;"><tr><td><a href='/doi/10.1136/amiajnl-2012-001431' class='pub-comments tt-right' title='Article comments'>2</a></td><td><div class='pub-order'>1.</div></td><td><div class='doi-body' data-key='10.1136/amiajnl-2012-001431'><a href='/doi/10.1136/amiajnl-2012-001431' class='pub-title'>Development and evaluation of an ensemble resource linking medications to their indications</a><div class='pub-body'>W.-Q. Wei, R. M. Cronin, H. Xu, T. A. Lasko, L. Bastarache, J. C. Denny (2013) <i>Journal of the American Medical Informatics Association</i>. doi:<a class='pub-doi' href='https://doi.org/10.1136/amiajnl-2012-001431'>10.1136/amiajnl-2012-001431</a></div></div></td></tr><tr><td><a href='/doi/10.1016/j.jbi.2014.08.004' class='pub-comments tt-right' title='Article comments'>5</a></td><td><div class='pub-order'>2.</div></td><td><div class='doi-body' data-key='10.1016/j.jbi.2014.08.004'><a href='/doi/10.1016/j.jbi.2014.08.004' class='pub-title'>LabeledIn: Cataloging labeled indications for human drugs</a><div class='pub-body'>Ritu Khare, Jiao Li, Zhiyong Lu (2014) <i>Journal of Biomedical Informatics</i>. doi:<a class='pub-doi' href='https://doi.org/10.1016/j.jbi.2014.08.004'>10.1016/j.jbi.2014.08.004</a></div></div></td></tr><tr><td><a href='/doi/10.1093/database/bav016' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>3.</div></td><td><div class='doi-body' data-key='10.1093/database/bav016'><a href='/doi/10.1093/database/bav016' class='pub-title'>Scaling drug indication curation through crowdsourcing</a><div class='pub-body'>R. Khare, J. D. Burger, J. S. Aberdeen, D. W. Tresner-Kirsch, T. J. Corrales, L. Hirchman, Z. Lu (2015) <i>Database</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/database/bav016'>10.1093/database/bav016</a></div></div></td></tr><tr><td><a href='/doi/10.1093/nar/gkh061' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>4.</div></td><td><div class='doi-body' data-key='10.1093/nar/gkh061'><a href='/doi/10.1093/nar/gkh061' class='pub-title'>The Unified Medical Language System (UMLS): integrating biomedical terminology</a><div class='pub-body'>O. Bodenreider (2004) <i>Nucleic Acids Research</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/nar/gkh061'>10.1093/nar/gkh061</a></div></div></td></tr><tr><td><a href='/doi/10.1038/ncomms5212' class='pub-comments tt-right' title='Article comments'>4</a></td><td><div class='pub-order'>5.</div></td><td><div class='doi-body' data-key='10.1038/ncomms5212'><a href='/doi/10.1038/ncomms5212' class='pub-title'>Human symptoms–disease network</a><div class='pub-body'>XueZhong Zhou, Jörg Menche, Albert-László Barabási, Amitabh Sharma (2014) <i>Nat Comms</i>. doi:<a class='pub-doi' href='https://doi.org/10.1038/ncomms5212'>10.1038/ncomms5212</a></div></div></td></tr><tr><td><a href='/doi/10.1001/jama.271.14.1103' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>6.</div></td><td><div class='doi-body' data-key='10.1001/jama.271.14.1103'><a href='/doi/10.1001/jama.271.14.1103' class='pub-title'>Understanding and using the medical subject headings (MeSH) vocabulary to perform literature searches</a><div class='pub-body'>H. J. Lowe (1994) <i>JAMA: The Journal of the American Medical Association</i>. doi:<a class='pub-doi' href='https://doi.org/10.1001/jama.271.14.1103'>10.1001/jama.271.14.1103</a></div></div></td></tr><tr><td><a href='/doi/10.1101/011569' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>7.</div></td><td><div class='doi-body' data-key='10.1101/011569'><a href='/doi/10.1101/011569' class='pub-title'>Heterogeneous Network Edge Prediction: A Data Integration Approach to Prioritize Disease-Associated Genes</a><div class='pub-body'>D. S. Himmelstein, S. E. Baranzini (2014) <i>Cold Spring Harbor Laboratory Press</i>. doi:<a class='pub-doi' href='https://doi.org/10.1101/011569'>10.1101/011569</a></div></div></td></tr><tr><td><a href='/doi/10.1093/nar/gkr972' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>8.</div></td><td><div class='doi-body' data-key='10.1093/nar/gkr972'><a href='/doi/10.1093/nar/gkr972' class='pub-title'>Disease Ontology: a backbone for disease semantic integration</a><div class='pub-body'>L. M. Schriml, C. Arze, S. Nadendla, Y.-W. W. Chang, M. Mazaitis, V. Felix, G. Feng, W. A. Kibbe (2011) <i>Nucleic Acids Research</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/nar/gkr972'>10.1093/nar/gkr972</a></div></div></td></tr><tr><td><a href='/doi/10.1093/nar/gku1011' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>9.</div></td><td><div class='doi-body' data-key='10.1093/nar/gku1011'><a href='/doi/10.1093/nar/gku1011' class='pub-title'>Disease Ontology 2015 update: an expanded and updated database of human diseases for linking biomedical knowledge through disease data</a><div class='pub-body'>W. A. Kibbe, C. Arze, V. Felix, E. Mitraka, E. Bolton, G. Fu, C. J. Mungall, J. X. Binder, J. Malone, D. Vasant, H. Parkinson, L. M. Schriml (2014) <i>Nucleic Acids Research</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/nar/gku1011'>10.1093/nar/gku1011</a></div></div></td></tr><tr><td><a href='/doi/10.1093/nar/gkr972' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>10.</div></td><td><div class='doi-body' data-key='10.1093/nar/gkr972'><a href='/doi/10.1093/nar/gkr972' class='pub-title'>Disease Ontology: a backbone for disease semantic integration</a><div class='pub-body'>L. M. Schriml, C. Arze, S. Nadendla, Y.-W. W. Chang, M. Mazaitis, V. Felix, G. Feng, W. A. Kibbe (2011) <i>Nucleic Acids Research</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/nar/gkr972'>10.1093/nar/gkr972</a></div></div></td></tr><tr><td><a href='/doi/10.1093/nar/gku1011' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>11.</div></td><td><div class='doi-body' data-key='10.1093/nar/gku1011'><a href='/doi/10.1093/nar/gku1011' class='pub-title'>Disease Ontology 2015 update: an expanded and updated database of human diseases for linking biomedical knowledge through disease data</a><div class='pub-body'>W. A. Kibbe, C. Arze, V. Felix, E. Mitraka, E. Bolton, G. Fu, C. J. Mungall, J. X. Binder, J. Malone, D. Vasant, H. Parkinson, L. M. Schriml (2014) <i>Nucleic Acids Research</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/nar/gku1011'>10.1093/nar/gku1011</a></div></div></td></tr><tr><td><a href='/doi/10.1093/bioinformatics/btp193' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>12.</div></td><td><div class='doi-body' data-key='10.1093/bioinformatics/btp193'><a href='/doi/10.1093/bioinformatics/btp193' class='pub-title'>From disease ontology to disease-ontology lite: statistical methods to adapt a general-purpose ontology for the test of gene-ontology associations</a><div class='pub-body'>P. Du, G. Feng, J. Flatow, J. Song, M. Holko, W. A. Kibbe, S. M. Lin (2009) <i>Bioinformatics</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/bioinformatics/btp193'>10.1093/bioinformatics/btp193</a></div></div></td></tr><tr><td><a href='/doi/10.1093/database/bav032' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>13.</div></td><td><div class='doi-body' data-key='10.1093/database/bav032'><a href='/doi/10.1093/database/bav032' class='pub-title'>Generating a focused view of disease ontology cancer terms for pan-cancer data integration and analysis</a><div class='pub-body'>T.-J. Wu, L. M. Schriml, Q.-R. Chen, M. Colbert, D. J. Crichton, R. Finney, Y. Hu, W. A. Kibbe, H. Kincaid, D. Meerzaman, E. Mitraka, Y. Pan, K. M. Smith, S. Srivastava, S. Ward, C. Yan, R. Mazumder (2015) <i>Database</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/database/bav032'>10.1093/database/bav032</a></div></div></td></tr><tr><td><a href='/doi/10.1371/journal.pone.0049686' class='pub-comments tt-right' title='Article comments'>1</a></td><td><div class='pub-order'>14.</div></td><td><div class='doi-body' data-key='10.1371/journal.pone.0049686'><a href='/doi/10.1371/journal.pone.0049686' class='pub-title'>A Framework for Annotating Human Genome in Disease Context</a><div class='pub-body'>Wei Xu, Huisong Wang, Wenqing Cheng, Dong Fu, Tian Xia, Warren A. Kibbe, Simon M. Lin (2012) <i>PLoS ONE</i>. doi:<a class='pub-doi' href='https://doi.org/10.1371/journal.pone.0049686'>10.1371/journal.pone.0049686</a></div></div></td></tr><tr><td><a href='/doi/10.1186/1471-2164-10-S1-S6' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>15.</div></td><td><div class='doi-body' data-key='10.1186/1471-2164-10-S1-S6'><a href='/doi/10.1186/1471-2164-10-S1-S6' class='pub-title'>Annotating the human genome with Disease Ontology</a><div class='pub-body'>John D Osborne, Jared Flatow, Michelle Holko, Simon M Lin, Warren A Kibbe, Lihua Zhu, Maria I Danila, Gang Feng, Rex L Chisholm (2009) <i>BMC Genomics</i>. doi:<a class='pub-doi' href='https://doi.org/10.1186/1471-2164-10-S1-S6'>10.1186/1471-2164-10-S1-S6</a></div></div></td></tr><tr><td><a href='/doi/10.1093/bioinformatics/btu684' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>16.</div></td><td><div class='doi-body' data-key='10.1093/bioinformatics/btu684'><a href='/doi/10.1093/bioinformatics/btu684' class='pub-title'>DOSE: an R/Bioconductor package for disease ontology semantic and enrichment analysis</a><div class='pub-body'>G. Yu, L.-G. Wang, G.-R. Yan, Q.-Y. He (2014) <i>Bioinformatics</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/bioinformatics/btu684'>10.1093/bioinformatics/btu684</a></div></div></td></tr><tr><td><a href='/doi/10.1186/1471-2105-12-266' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>17.</div></td><td><div class='doi-body' data-key='10.1186/1471-2105-12-266'><a href='/doi/10.1186/1471-2105-12-266' class='pub-title'>DOSim: An R package for similarity between diseases based on Disease Ontology</a><div class='pub-body'>Jiang Li, Binsheng Gong, Xi Chen, Tao Liu, Chao Wu, Fan Zhang, Chunquan Li, Xiang Li, Shaoqi Rao, Xia Li (2011) <i>BMC Bioinformatics</i>. doi:<a class='pub-doi' href='https://doi.org/10.1186/1471-2105-12-266'>10.1186/1471-2105-12-266</a></div></div></td></tr><tr><td><a href='/doi/10.1101/011569' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>18.</div></td><td><div class='doi-body' data-key='10.1101/011569'><a href='/doi/10.1101/011569' class='pub-title'>Heterogeneous Network Edge Prediction: A Data Integration Approach to Prioritize Disease-Associated Genes</a><div class='pub-body'>D. S. Himmelstein, S. E. Baranzini (2014) <i>Cold Spring Harbor Laboratory Press</i>. doi:<a class='pub-doi' href='https://doi.org/10.1101/011569'>10.1101/011569</a></div></div></td></tr><tr><td><a href='/doi/10.1093/bioinformatics/btp193' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>19.</div></td><td><div class='doi-body' data-key='10.1093/bioinformatics/btp193'><a href='/doi/10.1093/bioinformatics/btp193' class='pub-title'>From disease ontology to disease-ontology lite: statistical methods to adapt a general-purpose ontology for the test of gene-ontology associations</a><div class='pub-body'>P. Du, G. Feng, J. Flatow, J. Song, M. Holko, W. A. Kibbe, S. M. Lin (2009) <i>Bioinformatics</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/bioinformatics/btp193'>10.1093/bioinformatics/btp193</a></div></div></td></tr><tr><td><a href='/doi/10.1093/database/bav032' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>20.</div></td><td><div class='doi-body' data-key='10.1093/database/bav032'><a href='/doi/10.1093/database/bav032' class='pub-title'>Generating a focused view of disease ontology cancer terms for pan-cancer data integration and analysis</a><div class='pub-body'>T.-J. Wu, L. M. Schriml, Q.-R. Chen, M. Colbert, D. J. Crichton, R. Finney, Y. Hu, W. A. Kibbe, H. Kincaid, D. Meerzaman, E. Mitraka, Y. Pan, K. M. Smith, S. Srivastava, S. Ward, C. Yan, R. Mazumder (2015) <i>Database</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/database/bav032'>10.1093/database/bav032</a></div></div></td></tr><tr><td><a href='/doi/10.1093/database/bav032' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>21.</div></td><td><div class='doi-body' data-key='10.1093/database/bav032'><a href='/doi/10.1093/database/bav032' class='pub-title'>Generating a focused view of disease ontology cancer terms for pan-cancer data integration and analysis</a><div class='pub-body'>T.-J. Wu, L. M. Schriml, Q.-R. Chen, M. Colbert, D. J. Crichton, R. Finney, Y. Hu, W. A. Kibbe, H. Kincaid, D. Meerzaman, E. Mitraka, Y. Pan, K. M. Smith, S. Srivastava, S. Ward, C. Yan, R. Mazumder (2015) <i>Database</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/database/bav032'>10.1093/database/bav032</a></div></div></td></tr><tr><td><a href='/doi/10.1093/bioinformatics/btp193' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>22.</div></td><td><div class='doi-body' data-key='10.1093/bioinformatics/btp193'><a href='/doi/10.1093/bioinformatics/btp193' class='pub-title'>From disease ontology to disease-ontology lite: statistical methods to adapt a general-purpose ontology for the test of gene-ontology associations</a><div class='pub-body'>P. Du, G. Feng, J. Flatow, J. Song, M. Holko, W. A. Kibbe, S. M. Lin (2009) <i>Bioinformatics</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/bioinformatics/btp193'>10.1093/bioinformatics/btp193</a></div></div></td></tr><tr><td><a href='/doi/10.1371/journal.pcbi.1004259' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>23.</div></td><td><div class='doi-body' data-key='10.1371/journal.pcbi.1004259'><a href='/doi/10.1371/journal.pcbi.1004259' class='pub-title'>Heterogeneous Network Edge Prediction: A Data Integration Approach to Prioritize Disease-Associated Genes</a><div class='pub-body'>Daniel S. Himmelstein, Sergio E. Baranzini (2015) <i>PLOS Computational Biology</i>. doi:<a class='pub-doi' href='https://doi.org/10.1371/journal.pcbi.1004259'>10.1371/journal.pcbi.1004259</a></div></div></td></tr><tr><td><a href='/doi/10.1093/database/bav032' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>24.</div></td><td><div class='doi-body' data-key='10.1093/database/bav032'><a href='/doi/10.1093/database/bav032' class='pub-title'>Generating a focused view of disease ontology cancer terms for pan-cancer data integration and analysis</a><div class='pub-body'>T.-J. Wu, L. M. Schriml, Q.-R. Chen, M. Colbert, D. J. Crichton, R. Finney, Y. Hu, W. A. Kibbe, H. Kincaid, D. Meerzaman, E. Mitraka, Y. Pan, K. M. Smith, S. Srivastava, S. Ward, C. Yan, R. Mazumder (2015) <i>Database</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/database/bav032'>10.1093/database/bav032</a></div></div></td></tr><tr><td><a href='/doi/10.5281/zenodo.45584' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>25.</div></td><td><div class='doi-body' data-key='10.5281/zenodo.45584'><a href='/doi/10.5281/zenodo.45584' class='pub-title'>User-friendly extensions to the Disease Ontology v1.0</a><div class='pub-body'>Daniel S. Himmelstein (2016) <i>Zenodo</i>. doi:<a class='pub-doi' href='https://doi.org/10.5281/zenodo.45584'>10.5281/zenodo.45584</a></div></div></td></tr><tr><td><a href='/doi/10.1371/journal.pcbi.1004259' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>26.</div></td><td><div class='doi-body' data-key='10.1371/journal.pcbi.1004259'><a href='/doi/10.1371/journal.pcbi.1004259' class='pub-title'>Heterogeneous Network Edge Prediction: A Data Integration Approach to Prioritize Disease-Associated Genes</a><div class='pub-body'>Daniel S. Himmelstein, Sergio E. Baranzini (2015) <i>PLOS Computational Biology</i>. doi:<a class='pub-doi' href='https://doi.org/10.1371/journal.pcbi.1004259'>10.1371/journal.pcbi.1004259</a></div></div></td></tr><tr><td><a href='/doi/10.1093/database/bav032' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>27.</div></td><td><div class='doi-body' data-key='10.1093/database/bav032'><a href='/doi/10.1093/database/bav032' class='pub-title'>Generating a focused view of disease ontology cancer terms for pan-cancer data integration and analysis</a><div class='pub-body'>T.-J. Wu, L. M. Schriml, Q.-R. Chen, M. Colbert, D. J. Crichton, R. Finney, Y. Hu, W. A. Kibbe, H. Kincaid, D. Meerzaman, E. Mitraka, Y. Pan, K. M. Smith, S. Srivastava, S. Ward, C. Yan, R. Mazumder (2015) <i>Database</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/database/bav032'>10.1093/database/bav032</a></div></div></td></tr></table>

    


	</td>
	<td class="thread-sidebar">


			

      <a class="btn btn-default follow-link btn-follow btn-lg is-following tt-top full-width" title="Follow discussion" data-id="44" data-ctype="thread" data-is-following="false">&nbsp;</a>

      
        <div class="status-box text-center"><span style="">Status:</span> 
  Completed

</div>
      

      
        

          <div class="details">
            
            Labels
          </div>

          
            <a href='/p/rephetio/discussion?q=[data integration]' class='pl pl-sidebar ' style='background-color:#6b6b6b'><span class='glyphicon glyphicon-tag'></span>&nbsp; data integration</a>
          
        
      


				

					<div class="details">Views</div>
					<span style="font-size:20px;">313</span>

        

				
					<div class="details">Topics</div>
					<div style="margin-bottom:15px;">
		            <a href="/topics/umls" class="tag">UMLS</a><a href="/topics/disease-ontology" class="tag">Disease Ontology</a><a href="/topics/mapping" class="tag">Mapping</a><a href="/topics/terminologies" class="tag">Terminologies</a><a href="/topics/do" class="tag">DO</a><a href="/topics/vocabularies" class="tag">Vocabularies</a><a href="/topics/diseases" class="tag">Diseases</a><a href="/topics/mesh" class="tag">MeSH</a>
					</div>
				


		
			<div class="details">Referenced by</div>
			<div style="font-size:12px;"><ul style="padding-left:16px !important;">
			
				<li>
					
					<a href='/discussion/incorporating-drugcentral-data-in-our-network/186' class='thread-title'>Incorporating DrugCentral data in our network</a>
				</li>
			
				<li>
					
					<a href='/discussion/stargeo-expression-signatures-for-disease-using-crowdsourced-geo-annotation/96' class='thread-title'>STARGEO: expression signatures for disease using crowdsourced GEO annotation</a>
				</li>
			
				<li>
					
					Research report: <a href='/p/rephetio/report'>Rephetio: Repurposing drugs on a hetnet</a>
				</li>
			
				<li>
					
					<a href='/discussion/predictions-of-whether-a-compound-treats-a-disease/203' class='thread-title'>Predictions of whether a compound treats a disease</a>
				</li>
			
				<li>
					
					<a href='/discussion/disease-ontology-feature-requests/68' class='thread-title'>Disease Ontology feature requests</a>
				</li>
			
				<li>
					
					<a href='/discussion/cataloging-drugdisease-therapies-in-the-clinicaltrialsgov-database/212' class='thread-title'>Cataloging drug–disease therapies in the ClinicalTrials.gov database</a>
				</li>
			
				<li>
					
					<a href='/discussion/how-should-we-construct-a-catalog-of-drug-indications/21' class='thread-title'>How should we construct a catalog of drug indications?</a>
				</li>
			
				<li>
					
					<a href='/discussion/the-adeptus-resource-for-expression-signatures-of-disease/101' class='thread-title'>The ADEPTUS resource for expression signatures of disease</a>
				</li>
			
				<li>
					
					<a href='/discussion/announcing-pharmacotherapydb-the-open-catalog-of-drug-therapies-for-disease/182' class='thread-title'>Announcing PharmacotherapyDB: the Open Catalog of Drug Therapies for Disease</a>
				</li>
			
				<li>
					
					<a href='/discussion/expert-curation-of-our-indication-catalog-for-disease-modifying-treatments/95' class='thread-title'>Expert curation of our indication catalog for disease-modifying treatments</a>
				</li>
			
				<li>
					
					<a href='/discussion/tissue-specific-gene-expression-resources/81' class='thread-title'>Tissue-specific gene expression resources</a>
				</li>
			
				<li>
					
					<a href='/discussion/processing-the-diseases-resource-for-diseasegene-relationships/106' class='thread-title'>Processing the DISEASES resource for disease–gene relationships</a>
				</li>
			
				<li>
					
					<a href='/discussion/processing-disgenet-for-disease-gene-relationships/105' class='thread-title'>Processing DisGeNET for disease-gene relationships</a>
				</li>
			
				<li>
					
					<a href='/discussion/disease-similarity-from-medline-topic-cooccurrence/93' class='thread-title'>Disease similarity from MEDLINE topic cooccurrence</a>
				</li>
			
				<li>
					
					<a href='/discussion/extracting-disease-gene-associations-from-the-gwas-catalog/80' class='thread-title'>Extracting disease-gene associations from the GWAS Catalog</a>
				</li>
			
				<li>
					
					<a href='/discussion/human-symptom-disease-network-mesh-id-matching/52' class='thread-title'>Human Symptom Disease Network-MeSH ID Matching</a>
				</li>
			
				</ul>
			</div>
		

		
			<div class="details">Cite this as</div>
			<span style="font-size:12px;">
				Daniel Himmelstein, Tong Shu Li (2015) Unifying disease vocabularies. <i>Thinklab.</i> doi:<a href='https://doi.org/10.15363/thinklab.d44'>10.15363/thinklab.d44</a>
			</span>
		

		<div class="details">License</div>
		<p><a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0;width:88px;height:31px;" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a></p>

		<div class="details">Share</div>
		
			
				<a class="share-button fb" onclick="javascript:void(window.open('https://www.facebook.com/sharer/sharer.php?u=https%3A//thinklab.com/discussion/unifying-disease-vocabularies/44','','width=600,height=460,menubar=no,status=no'));" href="#"></a>
<a class="share-button twitter" onclick="javascript:void(window.open('https://twitter.com/share?text=Unifying%20disease%20vocabularies&url=https%3A//thinklab.com/discussion/unifying-disease-vocabularies/44','','width=600,height=300,menubar=no,status=no'));" href="#"></a>
<a class="share-button google" onclick="javascript:void(window.open('https://plus.google.com/share?url=https%3A//thinklab.com/discussion/unifying-disease-vocabularies/44','','width=510,height=460,menubar=no,status=no'));" href="#"></a>

			
		

	</td>
</tr></table>


		</div>


		
	    	<div class="footer">
<div style="background:#dedede;">
	<div class="main-centered-div text-center" style="padding:20px;">
		<h4 style="margin-bottom:0px;font-weight:bold;color:gray;">Feedback? Questions? Email us: <a href="/cdn-cgi/l/email-protection#f199949d9d9eb18599989f9a9d9093df929e9c"><span class="__cf_email__" data-cfemail="2f474a4343406f5b47464144434e4d014c4042">[email&#160;protected]</span><script data-cfhash='f9e31' type="text/javascript">/* <![CDATA[ */!function(t,e,r,n,c,a,p){try{t=document.currentScript||function(){for(t=document.getElementsByTagName('script'),e=t.length;e--;)if(t[e].getAttribute('data-cfhash'))return t[e]}();if(t&&(c=t.previousSibling)){p=t.parentNode;if(a=c.getAttribute('data-cfemail')){for(e='',r='0x'+a.substr(0,2)|0,n=2;a.length-n;n+=2)e+='%'+('0'+('0x'+a.substr(n,2)^r).toString(16)).slice(-2);p.replaceChild(document.createTextNode(decodeURIComponent(e)),c)}p.removeChild(t)}}catch(u){}}()/* ]]> */</script></a></h4>
	</div>
</div>
<div class="main-footer">
	<div class="main-centered-div">

<table class="full-width"><tr>
	<td class="footer-nav">

        <div class="foot-head">Explore</div>
        <a href="/proposals">Proposals</a><br>
        <a href="/projects">Projects</a><br>
        <a href="/publications">Article discussion</a><br>
        <a href="/discussion">All discussion</a><br>
        <a href="/topics">Topics</a><br>
        <!-- <a href="/funders">Funders</a><br> -->
        <a href="/funding">Funding opportunities</a><br>

	</td>
	<td class="footer-nav">

        <!-- <div class="foot-head">Help &amp; feedback</div>	 -->
        <!-- <div class="foot-head">Intro</div>
        <a href="/benefits/for-science">Why</a><br>
        <a href="/how-it-works">How it works</a><br>

<br> -->

        <div class="foot-head">Community</div>
        <a href="/leaderboard">Leaderboard</a><br>
        <a href="/p/meta">Thinklab Meta</a><br>
				<a href="/help/faq">Help</a><br>
				<!-- <a href="/new-project">Start a proposal</a><br> -->
        <!-- <a href="/benefits/proposal">Benefits</a><br> -->
        <!-- <a href="/funding">Get funded</a><br> -->
        <!-- <a href="/participate">Become a contributor</a><br> -->
        <!-- <a href="/for-funders">For science funders</a><br> -->
        <!-- <a href="/for-institutions">For institutions</a><br> -->

	</td>
	<td class="footer-nav">



        <div class="foot-head">About</div>
        <a href="/about">Overview</a><br>
        <a href="/benefits">Benefits</a><br>
        <a href="/how-it-works">How it works</a><br>
        <a href="/about/story">Our story</a><br>
        <a href="/blog">Our blog</a><br>
        <a href="/about/jobs">Join the team</a><br>
        <br>

        <!-- <div class="foot-head">Help</div>
        <a href="/help">Help</a><br>
		 -->


	</td>
	<td class="footer-nav">
		<a href="http://twitter.com/thinklab" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/twitter_com.png" width="40"/></a>
		<a href="https://www.facebook.com/pages/Thinklab/310895039088185" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/facebook_com.png" width="40"/></a>
		<a href="https://plus.google.com/115578983466800753402/" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/plus_google_com.png" width="40"/></a>
		<a href="https://www.linkedin.com/company/thinklab-co" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/linkedin_com.png" width="40"/></a>
		<br><br>
		<span class="light" style='font-size:12px;'>&copy; 2015 Thinklab</span>

	</td>

</tr></table>

</div>
</div>
</div>

		

	</div>


  <script></script>
  <script>
  var MEDIA_URL = 'https://think-lab.s3.amazonaws.com/m/';
  var STATIC_URL = 'https://think-lab.s3.amazonaws.com/s/';
  var page_is_authenticated = false;
  var page_page = 'discussion';
  var page_profile_id = undefined;
</script>



	<!-- Note different script type. MUST come before MathJax.js -->
	<script type="text/x-mathjax-config">
	    MathJax.Hub.Config({
	        "HTML-CSS": { preferredFont: "TeX", availableFonts: ["STIX","TeX"],
	                        scale: 110, linebreaks: { automatic:true },
	                        EqnChunk: (MathJax.Hub.Browser.isMobile ? 10 : 50) },
	        tex2jax: { inlineMath: [ ["$$", "$$"], ["\\(","\\)"] ],
	                    displayMath: [ ["$$$","$$$"], ["\\[", "\\]"] ],
	                    processEscapes: true,}
	            });

	</script>
    <script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>





  <script src="https://code.jquery.com/jquery-1.10.2.min.js"></script>
  <script src="https://code.jquery.com/ui/1.11.4/jquery-ui.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" integrity="sha384-0mSbJDEHialfmuBBQP6A4Qrprq5OVfW37PRR3j5ELqxss1yVqOtnepnHVP9aJ7xS" crossorigin="anonymous"></script>
  <script src="https://think-lab.s3.amazonaws.com/s/dist/main.min.js"></script>
  



  
	
  
  
  
<script>
$(function() {
  recordHit('thread', 44);
});
</script>



<script>/* <![CDATA[ */(function(d,s,a,i,j,r,l,m,t){try{l=d.getElementsByTagName('a');t=d.createElement('textarea');for(i=0;l.length-i;i++){try{a=l[i].href;s=a.indexOf('/cdn-cgi/l/email-protection');m=a.length;if(a&&s>-1&&m>28){j=28+s;s='';if(j<m){r='0x'+a.substr(j,2)|0;for(j+=2;j<m&&a.charAt(j)!='X';j+=2)s+='%'+('0'+('0x'+a.substr(j,2)^r).toString(16)).slice(-2);j++;s=decodeURIComponent(s)+a.substr(j,m-j)}t.innerHTML=s.replace(/</g,'&lt;').replace(/\>/g,'&gt;');l[i].href='mailto:'+t.value}}catch(e){}}}catch(e){}})(document);/* ]]> */</script></body>
</html>
