<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8" /><script type="text/javascript">window.NREUM||(NREUM={}),__nr_require=function(e,n,t){function r(t){if(!n[t]){var o=n[t]={exports:{}};e[t][0].call(o.exports,function(n){var o=e[t][1][n];return r(o||n)},o,o.exports)}return n[t].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<t.length;o++)r(t[o]);return r}({1:[function(e,n,t){function r(){}function o(e,n,t){return function(){return i(e,[c.now()].concat(u(arguments)),n?null:this,t),n?void 0:this}}var i=e("handle"),a=e(2),u=e(3),f=e("ee").get("tracer"),c=e("loader"),s=NREUM;"undefined"==typeof window.newrelic&&(newrelic=s);var p=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],d="api-",l=d+"ixn-";a(p,function(e,n){s[n]=o(d+n,!0,"api")}),s.addPageAction=o(d+"addPageAction",!0),s.setCurrentRouteName=o(d+"routeName",!0),n.exports=newrelic,s.interaction=function(){return(new r).get()};var m=r.prototype={createTracer:function(e,n){var t={},r=this,o="function"==typeof n;return i(l+"tracer",[c.now(),e,t],r),function(){if(f.emit((o?"":"no-")+"fn-start",[c.now(),r,o],t),o)try{return n.apply(this,arguments)}finally{f.emit("fn-end",[c.now()],t)}}}};a("setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(e,n){m[n]=o(l+n)}),newrelic.noticeError=function(e){"string"==typeof e&&(e=new Error(e)),i("err",[e,c.now()])}},{}],2:[function(e,n,t){function r(e,n){var t=[],r="",i=0;for(r in e)o.call(e,r)&&(t[i]=n(r,e[r]),i+=1);return t}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],3:[function(e,n,t){function r(e,n,t){n||(n=0),"undefined"==typeof t&&(t=e?e.length:0);for(var r=-1,o=t-n||0,i=Array(o<0?0:o);++r<o;)i[r]=e[n+r];return i}n.exports=r},{}],4:[function(e,n,t){n.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(e,n,t){function r(){}function o(e){function n(e){return e&&e instanceof r?e:e?f(e,u,i):i()}function t(t,r,o,i){if(!d.aborted||i){e&&e(t,r,o);for(var a=n(o),u=m(t),f=u.length,c=0;c<f;c++)u[c].apply(a,r);var p=s[y[t]];return p&&p.push([b,t,r,a]),a}}function l(e,n){v[e]=m(e).concat(n)}function m(e){return v[e]||[]}function w(e){return p[e]=p[e]||o(t)}function g(e,n){c(e,function(e,t){n=n||"feature",y[t]=n,n in s||(s[n]=[])})}var v={},y={},b={on:l,emit:t,get:w,listeners:m,context:n,buffer:g,abort:a,aborted:!1};return b}function i(){return new r}function a(){(s.api||s.feature)&&(d.aborted=!0,s=d.backlog={})}var u="nr@context",f=e("gos"),c=e(2),s={},p={},d=n.exports=o();d.backlog=s},{}],gos:[function(e,n,t){function r(e,n,t){if(o.call(e,n))return e[n];var r=t();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(e,n,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return e[n]=r,r}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],handle:[function(e,n,t){function r(e,n,t,r){o.buffer([e],r),o.emit(e,n,t)}var o=e("ee").get("handle");n.exports=r,r.ee=o},{}],id:[function(e,n,t){function r(e){var n=typeof e;return!e||"object"!==n&&"function"!==n?-1:e===window?0:a(e,i,function(){return o++})}var o=1,i="nr@id",a=e("gos");n.exports=r},{}],loader:[function(e,n,t){function r(){if(!x++){var e=h.info=NREUM.info,n=d.getElementsByTagName("script")[0];if(setTimeout(s.abort,3e4),!(e&&e.licenseKey&&e.applicationID&&n))return s.abort();c(y,function(n,t){e[n]||(e[n]=t)}),f("mark",["onload",a()+h.offset],null,"api");var t=d.createElement("script");t.src="https://"+e.agent,n.parentNode.insertBefore(t,n)}}function o(){"complete"===d.readyState&&i()}function i(){f("mark",["domContent",a()+h.offset],null,"api")}function a(){return E.exists&&performance.now?Math.round(performance.now()):(u=Math.max((new Date).getTime(),u))-h.offset}var u=(new Date).getTime(),f=e("handle"),c=e(2),s=e("ee"),p=window,d=p.document,l="addEventListener",m="attachEvent",w=p.XMLHttpRequest,g=w&&w.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:w,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var v=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-1044.min.js"},b=w&&g&&g[l]&&!/CriOS/.test(navigator.userAgent),h=n.exports={offset:u,now:a,origin:v,features:{},xhrWrappable:b};e(1),d[l]?(d[l]("DOMContentLoaded",i,!1),p[l]("load",r,!1)):(d[m]("onreadystatechange",o),p[m]("onload",r)),f("mark",["firstbyte",u],null,"api");var x=0,E=e(4)},{}]},{},["loader"]);</script><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info={"beacon":"bam.nr-data.net","queueTime":0,"licenseKey":"cd1101aa94","agent":"","transactionName":"bgcDMkBZW0NZBkdcWVdNJxNcW0FZVwscV1dKB08QW11CQxYTWlBBZgEOC19dW0QCEVtHU1gG","applicationID":"6369297","errorBeacon":"bam.nr-data.net","applicationTime":481}</script>
    <title>Human Symptom Disease Network-MeSH ID Matching | Thinklab</title>
	  <link rel="shortcut icon" href="https://think-lab.s3.amazonaws.com/s/img/favicon.ico" type="image/x-icon" />
    <meta property="og:image" content="https://think-lab.s3.amazonaws.com/s/img" />
    
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
  <link rel="stylesheet" href="https://code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css" />

<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/highlightjs/github.css" />
<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/css/css.css" />
<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/css/bootstrap-multiselect.css" />
<!-- <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'> -->

    
    
    
    
  
    <meta name="citation_doi" content="10.15363/thinklab.d52">
    <meta name="citation_title" content="Human Symptom Disease Network-MeSH ID Matching">
    <meta name="citation_online_date" content="April 9, 2015">
    <meta name="citation_author" content="Leo Brueggeman, Daniel Himmelstein">
    <meta name="citation_journal_title" content="Thinklab">
    <meta name="citation_publisher" content="Thinklab">
  

    


	<!-- GoogleAnalytics -->
	<script>
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
	(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
	m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

	ga('create', 'UA-48550230-1', 'thinklab.org');
	ga('require', 'displayfeatures');
	
	ga('send', 'pageview');

	</script>






</head>
<body >
  <form id="form_action" method="post">
    <input type='hidden' name='csrfmiddlewaretoken' value='JC3hnVSWzcwBOS226QTq2eGyHXj1YAM6' />
</form>

	

	<div class="body-wrapper">

		


<!-- <div style="height:5px;background-image:url('https://think-lab.s3.amazonaws.com/s/img/top-bg.gif');"></div> -->
<div class="gradient">
	<div class="main-centered-div header-container">

<table class="full-width" style="height:60px;"><tr>
	<td style="width:200px;vertical-align:middle;padding-bottom:15px;">
		<a href="/" style="font-weight:normal;color:#212121;"><img src="https://think-lab.s3.amazonaws.com/s/img/logo.png" width="160"></a>
	</td><td style="vertical-align:middle;text-align:left;padding-top:6px;width:100%;padding-left:40px;">
		<ul id="not-search-block"  class="main-header">


			


				<li class="dropdown">
					<a class="dropdown-toggle" data-toggle="dropdown" href="#">Browse <span class="caret"></span></a>
					<ul class="dropdown-menu" role="menu">
						<li><a href="/proposals">Proposals</a></li>
						<li><a href="/projects">Projects</a></li>
						<li><a href="/publications">Article discussion</a></li>
						<li><a href="/discussion">All discussion</a></li>
					</ul>
				</li>
				<li><a href="/leaderboard">Leaderboard</a></li>

			
				<li><a href="/about">About</a></li>
			


		</ul>

    <ul id="search-block" class="main-header" style="display: none; margin-right: 30px;">
        <li class="full-width">
            <form method="get" action="/search">
			<div class="input-group full-width">
				<input name="q" type="text" class="form-control search-input">
			</div>
            </form>
        </li>
    </ul>

	</td><td style="vertical-align:middle;text-align:right;padding-top:6px; min-width: 1%; white-space: nowrap;">
		<ul class="main-header pull-right" style="padding-left: 0;">
    		
                <!-- <li><a href="/login" style="color:silver;">Login</a></li>
                <li style="margin-right:0"><a class="btn btn-default btn-sm btn-upper" href="/signup">Join</a></li> -->
            

        </ul>
    </ul>
	</td>
</tr></table>

<!-- <div id="global-search-div" class="main-centered-div" style="display:none;">
	<input type="text" value="" class="thread-search form-control" placeholder="Search" data-action="/discussion">
</div> -->

	</div>
</div>

		
	  



	  


		

		<div class="dark-head"><div class="main-centered-div">

			

		</div></div>
		
		<div class="main-centered-div" style="margin-top:40px;padding-bottom:340px">
	    

<div style="margin-top:-20px; margin-bottom:20px;">

    <a class="side-box" href="/p/rephetio" style="background-color:#ebfdff;"><div class="side-type">Project:</div> <div class="side-title">Rephetio: Repurposing drugs on a hetnet [rephetio]</div></a><br>



    <a class="side-box" href="/doi/10.1038/ncomms5212" style="background-color:#f0ffec;"><div class="side-type">Publication:</div> <div class="side-title">Human symptoms–disease network</div></a><br>

</div>

<table style="margin-top:0px;"><tr>
	<td class="thread-table-content">

		<table class="full-width">
			
<tr class="comment-tr target-hl" id="1" data-comment-id="129">
  <td class="comment-profile-avatar">
		<a href="/u/leobrueggeman"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/21_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="129"></a>

    

			<h2 class="page-title" style="margin-bottom:5px;margin-top:0px !important;">Human Symptom Disease Network-MeSH ID Matching</h2>

			<div style="margin-top:5px;color:gray;margin-bottom:20px;font-size:13px;">
				<a style="font-weight:normal;color:#212121" href="/u/leobrueggeman"><strong>Leo Brueggeman</strong></a>
				<span class="is-researcher">Researcher</span>
				&nbsp; <a href="#1" class="light">April 9, 2015</a>
				
			</div>

      

          <hr>
		

    <p>One of the edge types we plan to incorporate in our network is that between diseases and symptoms. This data will come from the work of <em>Zhou et al.</em> in their <a href="https://dx.doi.org/10.1038/ncomms5212">"Human symptoms–disease network"</a> paper. The supplementary data released by <em>Zhou et al.</em> identifies diseases and symptoms by their MeSH names, but does not include the associated MeSH IDs. To ease interoperability we have performed the minor task of appending the relevant MeSH IDs to these files. The result can be found <a href="https://github.com/LABrueggs/HSDN">here</a>.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="2" data-comment-id="145">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="145"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#2" class="light">April 20, 2015</a>
          
        </span>
      </div>

    

    <p>We mapped the MeSH diseases from the HSDN <span class="citation">[<a href="https://doi.org/10.1038/ncomms5212" class="citation" data-key="10.1038/ncomms5212">1</a>]</span> to <a href="http://thinklab.com/discussion/unifying-disease-vocabularies/44#144">our slim DO</a>. See the <a href="https://cdn.rawgit.com/dhimmel/hsdn/51d31d738fe3cad0a12c736b8def333729c3c7f4/index.html">notebook</a> for more info or download the <a href="https://raw.githubusercontent.com/dhimmel/hsdn/51d31d738fe3cad0a12c736b8def333729c3c7f4/data/symptoms-DO.tsv">mapped data</a>.</p>

<p>Each disease-symptom relationship includes a <code>tfidf_score</code> (term frequency-inverse document frequency). This score, <span class="math">$$w_{i,j}$$</span>, between symptom <em>i</em> and disease <em>j</em> was calculated with:</p>

<p></p><div class="math">$$$
w_{i,j} = W_{i,j} \times \log{\frac{N}{n_i}}
$$$</div>

<p>where <span class="math">$$W_{i,j}$$</span> is the number of co-occurrences in PubMed, <em>N</em> is the total number of diseases, and <span class="math">$$n_i$$</span> is the number of diseases where symptom <em>i</em> appears.</p>

<p>At some point we will set an inclusion threshold for symptom edges based on their <code>tfidf_score</code>.</p>

<p>We used a propagated slim DO mapping, so symptoms for MeSH term "relapsing-remitting multiple sclerosis" for example were included as symptoms for DO term "multiple sclerosis".</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="3" data-comment-id="167">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="167"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#3" class="light">April 28, 2015</a>
          
        </span>
      </div>

    

    <p>The above formula used to calculate <code>tfidf_score</code> adjusts for the frequency of the symptom, but not the frequency of disease. Therefore we speculate that the scores are comparable within but not across diseases. Since we want to adopt a single inclusion threshold for all symptom-disease pairs, we would like to reformulate the metric to adjust for disease frequency.</p>

<p>We added a <a href="https://cdn.rawgit.com/dhimmel/hsdn/af2237af712be4b5319fa3669527e3fa1dbdfe44/index.html">new visualization and table</a> to investigate a disease-frequency bias. It appears that diseases that occur in more PubMed records have a higher number of symptoms exceeding a given <code>tfidf_score</code>.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="4" data-comment-id="176">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="176"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#4" class="light">April 30, 2015</a>
          
        </span>
      </div>

    

    <p><em>Below, I've copied the <a href="http://www.nature.com/ncomms/2014/140626/ncomms5212/extref/ncomms5212-s1.pdf">supplementary methods section</a> from the HSDN <span class="citation">[<a href="https://doi.org/10.1038/ncomms5212" class="citation" data-key="10.1038/ncomms5212">1</a>]</span> describing how the literature mining was accomplished. I think a similar method could help us if we choose to perform our own <a href="http://thinklab.com/discussion/text-as-a-resource-for-network-population/48">text mining</a> for network population.</em></p>

<p>We use the Medical Subject Headings (MeSH) <span class="citation">[<a href="https://doi.org/10.1001/jama.1994.03510380059038" class="citation" data-key="10.1001/jama.1994.03510380059038">2</a>]</span> terminology to generate symptom-disease relationships from the metadata extracted from PubMed <span class="citation">[<a href="https://doi.org/10.1093/nar/gkl1031" class="citation" data-key="10.1093/nar/gkl1031">3</a>]</span> bibliographic records. PubMed is currently the most comprehensive literature database on biomedical sciences. It includes MEDLINE <span class="citation">[<a href="https://doi.org/10.3163/1536-5050.95.4.416" class="citation" data-key="10.3163/1536-5050.95.4.416">4</a>]</span> and uses MeSH for each citation to facilitate information retrieval. MeSH is a controlled thesaurus that is used for the annotation of published articles, resulting in a high quality representation of their main topics and contributions. The MeSH terms are assigned manually by trained indexers and have been used in numerous biomedical text mining and literature-based discovery studies <span class="citation">[<a href="https://doi.org/10.1038/ng895" class="citation" data-key="10.1038/ng895">5</a>, <a href="https://doi.org/10.1093/bioinformatics/btr223" class="citation" data-key="10.1093/bioinformatics/btr223">6</a>, <a href="https://doi.org/10.1002/asi.20438" class="citation" data-key="10.1002/asi.20438">7</a>, <a href="https://doi.org/10.1038/nrg1768" class="citation" data-key="10.1038/nrg1768">8</a>]</span>.</p>

<p>We downloaded the <a href="http://www.nlm.nih.gov/mesh/2011/download/termscon.html">2011 ASCII version of MeSH</a> that contains 26,142 distinct terms and their unified identifiers. The MeSH vocabulary is structured as a hierarchical tree with 16 top nodes, representing general categories, such as ‘Anatomy’, ‘Diseases’ and ‘Phenomena and Processes.’ The broad category ‘Diseases’ contains the sub-category ‘Symptoms and Signs’ (MeSH tree code C23.888) that incorporates terms related to clinical manifestations observed by physicians or perceived by patients. We used all terms contained in the ‘Disease’ category (Table S1), excluding ‘Animal diseases’, as well as twenty terms, which only represent unspecific disease information, such as ‘Diseases’ itself, ‘Syndrome’, ‘Chronic diseases’ and ‘Infection’. In total, we obtained 4,442 distinct MeSH disease terms and 327 distinct MeSH symptom terms to be used for the PubMed query. To ensure that we only retrieve records with the corresponding indexed disease terms as a major topic, we search MEDLINE with the constraint “[Majr:NoExp]”, which filters for bibliographic records with the study of a specific disease as a main contribution. Using the E-Utility API web service interface of the National Center for Biotechnology Information, we developed a JAVA program to automatically search all MEDLINE bibliographic records published between 1966 and October 2011 (Figure S4). The total number of corresponding PubMed records was 7,109,429, of which 6,553,494 included a disease and 1,405,038 a symptom term. The number of records that contain both a disease, as well as a symptom term was 849,103. They included all 4,442 MeSH disease terms and almost all (322, i.e. 98%) symptom terms.</p>

<p><img src="https://raw.githubusercontent.com/dhimmel/hsdn/ca31229cf7174d8ee22567a455f386412a592144/figure/figS4.png" alt="Figure S4 from the Human symptoms–disease network"></p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

    		
		</table>

    

    &nbsp;<br>
    <!-- <a class="btn btn-primary btn-lg" href="/signup?next=/discussion/human-symptom-disease-network-mesh-id-matching/52&suppress_message=1" style>Join to Reply</a> -->


		<table class="references table-td-5" style="display:none;"><tr><td><a href='/doi/10.1038/ncomms5212' class='pub-comments tt-right' title='Article comments'>4</a></td><td><div class='pub-order'>1.</div></td><td><div class='doi-body' data-key='10.1038/ncomms5212'><a href='/doi/10.1038/ncomms5212' class='pub-title'>Human symptoms–disease network</a><div class='pub-body'>XueZhong Zhou, Jörg Menche, Albert-László Barabási, Amitabh Sharma (2014) <i>Nat Comms</i>. doi:<a class='pub-doi' href='https://doi.org/10.1038/ncomms5212'>10.1038/ncomms5212</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d44' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>2.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d44'><a href='/d/44' class='pub-title'>Unifying disease vocabularies</a><div class='pub-body'>Daniel Himmelstein, Tong Shu Li (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d44'>10.15363/thinklab.d44</a></div></div></td></tr><tr><td><a href='/doi/10.1038/ncomms5212' class='pub-comments tt-right' title='Article comments'>4</a></td><td><div class='pub-order'>3.</div></td><td><div class='doi-body' data-key='10.1038/ncomms5212'><a href='/doi/10.1038/ncomms5212' class='pub-title'>Human symptoms–disease network</a><div class='pub-body'>XueZhong Zhou, Jörg Menche, Albert-László Barabási, Amitabh Sharma (2014) <i>Nat Comms</i>. doi:<a class='pub-doi' href='https://doi.org/10.1038/ncomms5212'>10.1038/ncomms5212</a></div></div></td></tr><tr><td><a href='/doi/10.1001/jama.1994.03510380059038' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>4.</div></td><td><div class='doi-body' data-key='10.1001/jama.1994.03510380059038'><a href='/doi/10.1001/jama.1994.03510380059038' class='pub-title'>Understanding and Using the Medical Subject Headings (MeSH) Vocabulary to Perform Literature Searches</a><div class='pub-body'>Henry J. Lowe (1994) <i>JAMA: The Journal of the American Medical Association</i>. doi:<a class='pub-doi' href='https://doi.org/10.1001/jama.1994.03510380059038'>10.1001/jama.1994.03510380059038</a></div></div></td></tr><tr><td><a href='/doi/10.1093/nar/gkl1031' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>5.</div></td><td><div class='doi-body' data-key='10.1093/nar/gkl1031'><a href='/doi/10.1093/nar/gkl1031' class='pub-title'>Database resources of the National Center for Biotechnology Information</a><div class='pub-body'>D. L. Wheeler, T. Barrett, D. A. Benson, S. H. Bryant, K. Canese, V. Chetvernin, D. M. Church, M. DiCuccio, R. Edgar, S. Federhen, L. Y. Geer, Y. Kapustin, O. Khovayko, D. Landsman, D. J. Lipman, T. L. Madden, D. R. Maglott, J. Ostell, V. Miller, K. D. Pruitt, G. D. Schuler, E. Sequeira, S. T. Sherry, K. Sirotkin, A. Souvorov, G. Starchenko, R. L. Tatusov, T. A. Tatusova, L. Wagner, E. Yaschenko (2007) <i>Nucleic Acids Research</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/nar/gkl1031'>10.1093/nar/gkl1031</a></div></div></td></tr><tr><td><a href='/doi/10.3163/1536-5050.95.4.416' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>6.</div></td><td><div class='doi-body' data-key='10.3163/1536-5050.95.4.416'><a href='/doi/10.3163/1536-5050.95.4.416' class='pub-title'>The development of the Medical Literature Analysis and Retrieval System (MEDLARS)</a><div class='pub-body'>Cheryl Rae Dee (2007) <i>Journal of the Medical Library Association : JMLA</i>. doi:<a class='pub-doi' href='https://doi.org/10.3163/1536-5050.95.4.416'>10.3163/1536-5050.95.4.416</a></div></div></td></tr><tr><td><a href='/doi/10.1038/ng895' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>7.</div></td><td><div class='doi-body' data-key='10.1038/ng895'><a href='/doi/10.1038/ng895' class='pub-title'>Association of genes to genetically inherited diseases using data mining</a><div class='pub-body'>Carolina Perez-Iratxeta, Peer Bork, Miguel A. Andrade (2002) <i>Nat. Genet.</i>. doi:<a class='pub-doi' href='https://doi.org/10.1038/ng895'>10.1038/ng895</a></div></div></td></tr><tr><td><a href='/doi/10.1093/bioinformatics/btr223' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>8.</div></td><td><div class='doi-body' data-key='10.1093/bioinformatics/btr223'><a href='/doi/10.1093/bioinformatics/btr223' class='pub-title'>MeSH: a window into full text for document summarization</a><div class='pub-body'>S. Bhattacharya, V. Ha-Thuc, P. Srinivasan (2011) <i>Bioinformatics</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/bioinformatics/btr223'>10.1093/bioinformatics/btr223</a></div></div></td></tr><tr><td><a href='/doi/10.1002/asi.20438' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>9.</div></td><td><div class='doi-body' data-key='10.1002/asi.20438'><a href='/doi/10.1002/asi.20438' class='pub-title'>Ranking indirect connections in literature-based discovery: The role of medical subject headings</a><div class='pub-body'>Don R. Swanson, Neil R. Smalheiser, Vetle I. Torvik (2006) <i>Journal of the American Society for Information Science and Technology</i>. doi:<a class='pub-doi' href='https://doi.org/10.1002/asi.20438'>10.1002/asi.20438</a></div></div></td></tr><tr><td><a href='/doi/10.1038/nrg1768' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>10.</div></td><td><div class='doi-body' data-key='10.1038/nrg1768'><a href='/doi/10.1038/nrg1768' class='pub-title'>Literature mining for the biologist: from information retrieval to biological discovery</a><div class='pub-body'>Lars Juhl Jensen, Jasmin Saric, Peer Bork (2006) <i>Nat Rev Genet</i>. doi:<a class='pub-doi' href='https://doi.org/10.1038/nrg1768'>10.1038/nrg1768</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d48' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>11.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d48'><a href='/d/48' class='pub-title'>Text as a resource for network population?</a><div class='pub-body'>Benjamin Good, Daniel Himmelstein (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d48'>10.15363/thinklab.d48</a></div></div></td></tr></table>

    


	</td>
	<td class="thread-sidebar">


			

      <!-- <a class="btn btn-default follow-link btn-follow btn-lg is-following tt-top full-width" title="Follow discussion" data-id="52" data-ctype="thread" data-is-following="false">&nbsp;</a> -->

      
        <div class="status-box text-center"><span style="">Status:</span> 
  Completed

</div>
      

      
        

          <div class="details">
            
            Labels
          </div>

          
            <a href='/p/rephetio/discussion?q=[resource]' class='pl pl-sidebar ' style='background-color:#6b6b6b'><span class='glyphicon glyphicon-tag'></span>&nbsp; resource</a>
          
        
      


				

					<div class="details">Views</div>
					<span style="font-size:20px;">383</span>

        

				
					<div class="details">Topics</div>
					<div style="margin-bottom:15px;">
		            <a href="/topics/hsdn" class="tag">HSDN</a>
					</div>
				


		
			<div class="details">Referenced by</div>
			<div style="font-size:12px;"><ul style="padding-left:16px !important;">
			
				<li>
					
					<a href='/d/67' class='thread-title'>Mining knowledge from MEDLINE articles and their indexed MeSH terms</a>
				</li>
			
				</ul>
			</div>
		

		
			<div class="details">Cite this as</div>
			<span style="font-size:12px;">
				Leo Brueggeman, Daniel Himmelstein (2015) Human Symptom Disease Network-MeSH ID Matching. <i>Thinklab.</i> doi:<a href='https://doi.org/10.15363/thinklab.d52'>10.15363/thinklab.d52</a>
			</span>
		

		<div class="details">License</div>
		<p><a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0;width:88px;height:31px;" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a></p>

		<div class="details">Share</div>
		
			
				<a class="share-button fb" onclick="javascript:void(window.open('https://www.facebook.com/sharer/sharer.php?u=https%3A//thinklab.org/discussion/human-symptom-disease-network-mesh-id-matching/52','','width=600,height=460,menubar=no,status=no'));" href="#"></a>
<a class="share-button twitter" onclick="javascript:void(window.open('https://twitter.com/share?text=Human%20Symptom%20Disease%20Network-MeSH%20ID%20Matching&url=https%3A//thinklab.org/discussion/human-symptom-disease-network-mesh-id-matching/52','','width=600,height=300,menubar=no,status=no'));" href="#"></a>
<a class="share-button google" onclick="javascript:void(window.open('https://plus.google.com/share?url=https%3A//thinklab.org/discussion/human-symptom-disease-network-mesh-id-matching/52','','width=510,height=460,menubar=no,status=no'));" href="#"></a>

			
		

	</td>
</tr></table>


		</div>


		
	    	<div class="footer">
<!-- <div style="background:#dedede;">
	<div class="main-centered-div text-center" style="padding:20px;">
		<h4 style="margin-bottom:0px;font-weight:bold;color:gray;">Feedback? Questions? Email us: <a href="mailto:hello@thinklab.com">hello@thinklab.com</a></h4>
	</div>
</div> -->
<div class="main-footer">
	<div class="main-centered-div">

<table class="full-width"><tr>
	<td class="footer-nav">

        <div class="foot-head">Explore</div>
        <a href="/proposals">Proposals</a><br>
        <a href="/projects">Projects</a><br>
        <a href="/publications">Article discussion</a><br>
        <a href="/discussion">All discussion</a><br>
        <a href="/topics">Topics</a><br>
        <!-- <a href="/funders">Funders</a><br> -->
        <a href="/funding">Funding opportunities</a><br>

	</td>
	<td class="footer-nav">

        <!-- <div class="foot-head">Help &amp; feedback</div>	 -->
        <!-- <div class="foot-head">Intro</div>
        <a href="/benefits/for-science">Why</a><br>
        <a href="/how-it-works">How it works</a><br>

<br> -->

        <div class="foot-head">Community</div>
        <a href="/leaderboard">Leaderboard</a><br>
        <a href="/p/meta">Thinklab Meta</a><br>
				<a href="/help/faq">Help</a><br>
				<!-- <a href="/new-project">Start a proposal</a><br> -->
        <!-- <a href="/benefits/proposal">Benefits</a><br> -->
        <!-- <a href="/funding">Get funded</a><br> -->
        <!-- <a href="/participate">Become a contributor</a><br> -->
        <!-- <a href="/for-funders">For science funders</a><br> -->
        <!-- <a href="/for-institutions">For institutions</a><br> -->

	</td>
	<td class="footer-nav">



        <div class="foot-head">About</div>
        <a href="/about">Overview</a><br>
        <a href="/benefits">Benefits</a><br>
        <a href="/how-it-works">How it works</a><br>
        <a href="/about/story">Our story</a><br>
        <a href="/blog">Our blog</a><br>
        <a href="/about/jobs">Join the team</a><br>
        <br>

        <!-- <div class="foot-head">Help</div>
        <a href="/help">Help</a><br>
		 -->


	</td>
	<td class="footer-nav">
		<!-- <a href="http://twitter.com/thinklab" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/twitter_com.png" width="40"/></a>
		<a href="https://www.facebook.com/pages/Thinklab/310895039088185" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/facebook_com.png" width="40"/></a>
		<a href="https://plus.google.com/115578983466800753402/" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/plus_google_com.png" width="40"/></a>
		<a href="https://www.linkedin.com/company/thinklab-co" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/linkedin_com.png" width="40"/></a> -->
		<br><br>
		<span class="light" style='font-size:12px;'>&copy; 2015 Thinklab</span>

	</td>

</tr></table>

</div>
</div>
</div>

		

	</div>


  <script></script>
  <script>
  var MEDIA_URL = 'https://think-lab.s3.amazonaws.com/m/';
  var STATIC_URL = 'https://think-lab.s3.amazonaws.com/s/';
  var page_is_authenticated = false;
  var page_page = 'discussion';
  var page_profile_id = undefined;
</script>



	<!-- Note different script type. MUST come before MathJax.js -->
	<script type="text/x-mathjax-config">
	    MathJax.Hub.Config({
	        "HTML-CSS": { preferredFont: "TeX", availableFonts: ["STIX","TeX"],
	                        scale: 110, linebreaks: { automatic:true },
	                        EqnChunk: (MathJax.Hub.Browser.isMobile ? 10 : 50) },
	        tex2jax: { inlineMath: [ ["$$", "$$"], ["\\(","\\)"] ],
	                    displayMath: [ ["$$$","$$$"], ["\\[", "\\]"] ],
	                    processEscapes: true,}
	            });

	</script>
    <script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>





  <script src="https://code.jquery.com/jquery-1.10.2.min.js"></script>
  <script src="https://code.jquery.com/ui/1.11.4/jquery-ui.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" integrity="sha384-0mSbJDEHialfmuBBQP6A4Qrprq5OVfW37PRR3j5ELqxss1yVqOtnepnHVP9aJ7xS" crossorigin="anonymous"></script>
  <script src="https://think-lab.s3.amazonaws.com/s/dist/main.min.js"></script>
  



  
	
  
  
  
<script>
$(function() {
  recordHit('thread', 52);
});
</script>



</body>
</html>
