<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8" /><script type="text/javascript">window.NREUM||(NREUM={}),__nr_require=function(e,n,t){function r(t){if(!n[t]){var o=n[t]={exports:{}};e[t][0].call(o.exports,function(n){var o=e[t][1][n];return r(o||n)},o,o.exports)}return n[t].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<t.length;o++)r(t[o]);return r}({1:[function(e,n,t){function r(){}function o(e,n,t){return function(){return i(e,[c.now()].concat(u(arguments)),n?null:this,t),n?void 0:this}}var i=e("handle"),a=e(2),u=e(3),f=e("ee").get("tracer"),c=e("loader"),s=NREUM;"undefined"==typeof window.newrelic&&(newrelic=s);var p=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],d="api-",l=d+"ixn-";a(p,function(e,n){s[n]=o(d+n,!0,"api")}),s.addPageAction=o(d+"addPageAction",!0),s.setCurrentRouteName=o(d+"routeName",!0),n.exports=newrelic,s.interaction=function(){return(new r).get()};var m=r.prototype={createTracer:function(e,n){var t={},r=this,o="function"==typeof n;return i(l+"tracer",[c.now(),e,t],r),function(){if(f.emit((o?"":"no-")+"fn-start",[c.now(),r,o],t),o)try{return n.apply(this,arguments)}finally{f.emit("fn-end",[c.now()],t)}}}};a("setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(e,n){m[n]=o(l+n)}),newrelic.noticeError=function(e){"string"==typeof e&&(e=new Error(e)),i("err",[e,c.now()])}},{}],2:[function(e,n,t){function r(e,n){var t=[],r="",i=0;for(r in e)o.call(e,r)&&(t[i]=n(r,e[r]),i+=1);return t}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],3:[function(e,n,t){function r(e,n,t){n||(n=0),"undefined"==typeof t&&(t=e?e.length:0);for(var r=-1,o=t-n||0,i=Array(o<0?0:o);++r<o;)i[r]=e[n+r];return i}n.exports=r},{}],4:[function(e,n,t){n.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(e,n,t){function r(){}function o(e){function n(e){return e&&e instanceof r?e:e?f(e,u,i):i()}function t(t,r,o,i){if(!d.aborted||i){e&&e(t,r,o);for(var a=n(o),u=m(t),f=u.length,c=0;c<f;c++)u[c].apply(a,r);var p=s[y[t]];return p&&p.push([b,t,r,a]),a}}function l(e,n){v[e]=m(e).concat(n)}function m(e){return v[e]||[]}function w(e){return p[e]=p[e]||o(t)}function g(e,n){c(e,function(e,t){n=n||"feature",y[t]=n,n in s||(s[n]=[])})}var v={},y={},b={on:l,emit:t,get:w,listeners:m,context:n,buffer:g,abort:a,aborted:!1};return b}function i(){return new r}function a(){(s.api||s.feature)&&(d.aborted=!0,s=d.backlog={})}var u="nr@context",f=e("gos"),c=e(2),s={},p={},d=n.exports=o();d.backlog=s},{}],gos:[function(e,n,t){function r(e,n,t){if(o.call(e,n))return e[n];var r=t();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(e,n,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return e[n]=r,r}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],handle:[function(e,n,t){function r(e,n,t,r){o.buffer([e],r),o.emit(e,n,t)}var o=e("ee").get("handle");n.exports=r,r.ee=o},{}],id:[function(e,n,t){function r(e){var n=typeof e;return!e||"object"!==n&&"function"!==n?-1:e===window?0:a(e,i,function(){return o++})}var o=1,i="nr@id",a=e("gos");n.exports=r},{}],loader:[function(e,n,t){function r(){if(!x++){var e=h.info=NREUM.info,n=d.getElementsByTagName("script")[0];if(setTimeout(s.abort,3e4),!(e&&e.licenseKey&&e.applicationID&&n))return s.abort();c(y,function(n,t){e[n]||(e[n]=t)}),f("mark",["onload",a()+h.offset],null,"api");var t=d.createElement("script");t.src="https://"+e.agent,n.parentNode.insertBefore(t,n)}}function o(){"complete"===d.readyState&&i()}function i(){f("mark",["domContent",a()+h.offset],null,"api")}function a(){return E.exists&&performance.now?Math.round(performance.now()):(u=Math.max((new Date).getTime(),u))-h.offset}var u=(new Date).getTime(),f=e("handle"),c=e(2),s=e("ee"),p=window,d=p.document,l="addEventListener",m="attachEvent",w=p.XMLHttpRequest,g=w&&w.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:w,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var v=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-1039.min.js"},b=w&&g&&g[l]&&!/CriOS/.test(navigator.userAgent),h=n.exports={offset:u,now:a,origin:v,features:{},xhrWrappable:b};e(1),d[l]?(d[l]("DOMContentLoaded",i,!1),p[l]("load",r,!1)):(d[m]("onreadystatechange",o),p[m]("onload",r)),f("mark",["firstbyte",u],null,"api");var x=0,E=e(4)},{}]},{},["loader"]);</script><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info={"beacon":"bam.nr-data.net","queueTime":0,"licenseKey":"cd1101aa94","agent":"","transactionName":"bgcDMkBZW0NZBkdcWVdNJxNcW0FZVwscV1dKB08QW11CQxYTWlBBZgEOC19dW0QCEVtHU1gG","applicationID":"6369297","errorBeacon":"bam.nr-data.net","applicationTime":575}</script>
    <title>Processing Bgee for tissue-specific gene presence and over/under-expression | Thinklab</title>
	  <link rel="shortcut icon" href="https://think-lab.s3.amazonaws.com/s/img/favicon.ico" type="image/x-icon" />
    <meta property="og:image" content="https://think-lab.s3.amazonaws.com/s/img" />
    
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
  <link rel="stylesheet" href="https://code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css" />

<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/highlightjs/github.css" />
<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/css/css.css" />
<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/css/bootstrap-multiselect.css" />
<!-- <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'> -->

    
    
    
    
  
    <meta name="citation_doi" content="10.15363/thinklab.d124">
    <meta name="citation_title" content="Processing Bgee for tissue-specific gene presence and over/under-expression">
    <meta name="citation_online_date" content="Nov. 3, 2015">
    <meta name="citation_author" content="Daniel Himmelstein, Frederic Bastian">
    <meta name="citation_journal_title" content="Thinklab">
    <meta name="citation_publisher" content="Thinklab">
  

    


	<!-- GoogleAnalytics -->
	<script>
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
	(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
	m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

	ga('create', 'UA-48550230-1', 'thinklab.com');
	ga('require', 'displayfeatures');
	
	ga('send', 'pageview');

	</script>






</head>
<body >
  <form id="form_action" method="post">
    <input type='hidden' name='csrfmiddlewaretoken' value='pQNiZQX4gF2vyFeXrnbUl9EmuTWA5MG1' />
</form>

	

	<div class="body-wrapper">

		


<!-- <div style="height:5px;background-image:url('https://think-lab.s3.amazonaws.com/s/img/top-bg.gif');"></div> -->
<div class="gradient">
	<div class="main-centered-div header-container">

<table class="full-width" style="height:60px;"><tr>
	<td style="width:200px;vertical-align:middle;padding-bottom:15px;">
		<a href="/" style="font-weight:normal;color:#212121;"><img src="https://think-lab.s3.amazonaws.com/s/img/logo.png" width="160"></a>
	</td><td style="vertical-align:middle;text-align:left;padding-top:6px;width:100%;padding-left:40px;">
		<ul id="not-search-block"  class="main-header">


			


				<li class="dropdown">
					<a class="dropdown-toggle" data-toggle="dropdown" href="#">Browse <span class="caret"></span></a>
					<ul class="dropdown-menu" role="menu">
						<li><a href="/proposals">Proposals</a></li>
						<li><a href="/projects">Projects</a></li>
						<li><a href="/publications">Article discussion</a></li>
						<li><a href="/discussion">All discussion</a></li>
					</ul>
				</li>
				<li><a href="/leaderboard">Leaderboard</a></li>

			
				<li><a href="/about">About</a></li>
			


		</ul>

    <ul id="search-block" class="main-header" style="display: none; margin-right: 30px;">
        <li class="full-width">
            <form method="get" action="/search">
			<div class="input-group full-width">
				<input name="q" type="text" class="form-control search-input">
			</div>
            </form>
        </li>
    </ul>

	</td><td style="vertical-align:middle;text-align:right;padding-top:6px; min-width: 1%; white-space: nowrap;">
		<ul class="main-header pull-right" style="padding-left: 0;">
    		
                <li><a href="/login" style="color:silver;">Login</a></li>
                <li style="margin-right:0"><a class="btn btn-default btn-sm btn-upper" href="/signup">Join</a></li>
            

        </ul>
    </ul>
	</td>
</tr></table>

<!-- <div id="global-search-div" class="main-centered-div" style="display:none;">
	<input type="text" value="" class="thread-search form-control" placeholder="Search" data-action="/discussion">
</div> -->

	</div>
</div>

		
	  



	  


		

		<div class="dark-head"><div class="main-centered-div">

			

		</div></div>
		
		<div class="main-centered-div" style="margin-top:40px;padding-bottom:340px">
	    

<div style="margin-top:-20px; margin-bottom:20px;">

    <a class="side-box" href="/p/rephetio" style="background-color:#ebfdff;"><div class="side-type">Project:</div> <div class="side-title">Rephetio: Repurposing drugs on a hetnet [rephetio]</div></a><br>



    <a class="side-box" href="/doi/10.1007/978-3-540-69828-9_12" style="background-color:#f0ffec;"><div class="side-type">Publication:</div> <div class="side-title">Bgee: Integrating and Comparing Heterogeneous Transcriptome Data Among Species</div></a><br>

</div>

<table style="margin-top:0px;"><tr>
	<td class="thread-table-content">

		<table class="full-width">
			
<tr class="comment-tr target-hl" id="1" data-comment-id="538">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="538"></a>

    

			<h2 class="page-title" style="margin-bottom:5px;margin-top:0px !important;">Processing Bgee for tissue-specific gene presence and over/under-expression</h2>

			<div style="margin-top:5px;color:gray;margin-bottom:20px;font-size:13px;">
				<a style="font-weight:normal;color:#212121" href="/u/dhimmel"><strong>Daniel Himmelstein</strong></a>
				<span class="is-researcher">Researcher</span>
				&nbsp; <a href="#1" class="light">611 days ago</a>
				
			</div>

      

          <hr>
		

    <p><a href="http://bgee.org">Bgee</a> is an integrative and comparative resource for gene expression <span class="citation">[<a href="https://doi.org/10.1007/978-3-540-69828-9_12" class="citation" data-key="10.1007/978-3-540-69828-9_12">1</a>]</span>. We extract the following edges from Bgee for our network:</p>

<ul><li>Gene–Anatomy <strong>expression</strong> — whether a gene is present in an anatomy</li><li>Gene–Anatomy <strong>upregulation</strong> — whether a gene is upregulated (overexpressed) in an anatomy</li><li>Gene–Anatomy <strong>downregulation</strong> — whether a gene is downregulated (underexpressed) in an anatomy</li></ul>

<p>We have already substantially discussed Bgee on <em>Thinklab</em> in a more general gene expression thread <span class="citation">[<a href="/discussion/tissue-specific-gene-expression-resources/81" class="citation" data-key="10.15363/thinklab.d81">2</a>]</span>. The comments consisted of:</p>

<ul><li>an <a href="http://thinklab.com/discussion/tissue-specific-gene-expression-resources/81#3" class="citation thread-citation" data-key="10.15363/thinklab.d81">introduction</a> by <a href="/u/dhimmel" class="username">@dhimmel</a> </li><li><a href="http://thinklab.com/discussion/tissue-specific-gene-expression-resources/81#4" class="citation thread-citation" data-key="10.15363/thinklab.d81">questions</a> on processing parameters by <a href="/u/dhimmel" class="username">@dhimmel</a></li><li><a href="http://thinklab.com/discussion/tissue-specific-gene-expression-resources/81#5" class="citation thread-citation" data-key="10.15363/thinklab.d81">answers</a> by <a href="/u/fbastian" class="username">@fbastian</a> </li><li>preliminary <a href="http://thinklab.com/discussion/tissue-specific-gene-expression-resources/81#6" class="citation thread-citation" data-key="10.15363/thinklab.d81">analysis</a> by <a href="/u/dhimmel" class="username">@dhimmel</a></li><li>additional <a href="http://thinklab.com/discussion/tissue-specific-gene-expression-resources/81#7" class="citation thread-citation" data-key="10.15363/thinklab.d81">guidance</a> by <a href="/u/fbastian" class="username">@fbastian</a> </li><li>an initial <a href="http://thinklab.com/discussion/tissue-specific-gene-expression-resources/81#8" class="citation thread-citation" data-key="10.15363/thinklab.d81">complete analysis</a> and feedback by <a href="/u/dhimmel" class="username">@dhimmel</a> </li><li><a href="http://thinklab.com/discussion/tissue-specific-gene-expression-resources/81#9" class="citation thread-citation" data-key="10.15363/thinklab.d81">response</a> to feedback by <a href="/u/fbastian" class="username">@fbastian</a> </li><li><a href="http://thinklab.com/discussion/tissue-specific-gene-expression-resources/81#10" class="citation thread-citation" data-key="10.15363/thinklab.d81">tying up</a> loose ends by <a href="/u/dhimmel" class="username">@dhimmel</a></li></ul>

<p>We are revisiting our Bgee analysis. Since the original discussion has become cluttered and covers a general topic, we are starting a designated Bgee discussion. Stay tuned.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="2" data-comment-id="540">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="540"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#2" class="light">611 days</a>
          
        </span>
      </div>

    

    <h1>Quality control filters</h1>

<p>Initially, we <a href="http://thinklab.com/discussion/tissue-specific-gene-expression-resources/81#5" class="citation thread-citation" data-key="10.15363/thinklab.d81">chose permissive filters</a> for including Bgee edges based largely on the <a href="http://thinklab.com/discussion/tissue-specific-gene-expression-resources/81#5" class="citation thread-citation" data-key="10.15363/thinklab.d81">recommendations</a> of <a href="/u/fbastian" class="username">@fbastian</a>. Several developments are making us reevaluate our permissiveness:</p>

<ol><li><em>Gene–expression–Anatomy</em> edges are the most prevalent type in the network (see last figure in <a href="https://github.com/dhimmel/integrate/blob/1a43ef2718283cbfc9ae869aee1856bf20e2ad6e/integrate.ipynb">this notebook</a>). Now that we're traversing the network to extract paths, we're noticing high-degree nodes are computationally problematic. Several anatomies have 20,000 expressed genes and 5000 differentially expressed genes (see <a href="https://github.com/dhimmel/integrate/blob/1a43ef2718283cbfc9ae869aee1856bf20e2ad6e/viz/degrees.pdf">page 2 here</a>). Therefore downstream constraints favor rigorous thresholds for extremely high-degree edge types.</li><li>We <a href="http://doi.org/10.15363/thinklab.d91">also extract</a> <em>Gene–expression–Anatomy</em> edges from <a href="http://tissues.jensenlab.org/About">TISSUES</a> <span class="citation">[<a href="https://doi.org/10.7717/peerj.1054" class="citation" data-key="10.7717/peerj.1054">1</a>]</span>. <a href="https://github.com/dhimmel/tissues/blob/d6b0c99352db27469f2c3399cecb6f9fae2db547/bgee-combine.ipynb">Currently</a>, TISSUES contributes 321,516 expression edges, while Bgee contributes 5,406,177. I suspect our TISSUES inclusion threshold (score ≥ 3) is much more stringent than our <a href="http://thinklab.com/discussion/tissue-specific-gene-expression-resources/81#6" class="citation thread-citation" data-key="10.15363/thinklab.d81">Bgee theshold</a>.</li><li>TISSUES provides a score for each tissue–gene relationship. These scores have been calibrated on a gold standard allowing evidence-based weighting of each study. In contrast, Bgee has different categories of evidence based on ambiguity and quality. These measures are codependent and have been difficult to understand. Therefore, it's difficult to conclude whether low quality or high ambiguity relationships have merit.</li></ol>

<h2>Ambiguity and call quality</h2>

<p>We <a href="https://github.com/dhimmel/bgee/blob/add20b29b8f926004ce69b9bacff2edf69cd383c/bgee.ipynb">looked into</a> the relationship between ambiguity and call quality.</p>

<p>Below, we show the contingency table for <code>Expression</code> (columns) and <code>Call quality</code> (rows) in the <code>Homo_sapiens_expr-complete</code> dataset (<a href="http://bgee.org/?page=doc&amp;action=call_files#single_expr_complete_col7">documentation</a>). Each cell contains the percentage of observations in the corresponding category:</p>

<table class="table markdown-table"><thead><tr><th></th><th>absent</th><th>high ambiguity</th><th>low ambiguity</th><th>present</th></tr></thead><tbody><tr><td>NA</td><td>0</td><td>0.001</td><td>0.001</td><td>0</td></tr><tr><td>poor quality</td><td>0</td><td>0</td><td>0</td><td>0.23</td></tr><tr><td>high quality</td><td>0.11</td><td>0</td><td>0</td><td>0.67</td></tr></tbody></table>

<p>Below, we show the contingency table for <code>Differential expression</code> (columns) and <code>Call quality</code> (rows) in the <code>Homo_sapiens_diffexpr-anatomy-simple</code> dataset (<a href="http://bgee.org/?page=doc&amp;action=call_files#single_diff_simple_col7">documentation</a>). Each cell contains the percentage of observations in the corresponding category:</p>

<table class="table markdown-table"><thead><tr><th></th><th>high ambiguity</th><th>low ambiguity</th><th>over-expression</th><th>under-expression</th></tr></thead><tbody><tr><td>NA</td><td>0.025</td><td>0.16</td><td>0</td><td>0</td></tr><tr><td>low quality</td><td>0</td><td>0</td><td>0.18</td><td>0.24</td></tr><tr><td>high quality</td><td>0</td><td>0</td><td>0.21</td><td>0.20</td></tr></tbody></table>

<h2>Conclusions</h2>

<p><strong>Presence:</strong> <code>Homo_sapiens_expr-complete</code> uses the value <code>poor quality</code> while <code>Homo_sapiens_diffexpr-anatomy-simple</code> uses <code>low quality</code>. In our previous processing, we used <code>low quality</code> as the value for both datasets. Therefore, we accidentally omitted the 23% of observations that were <code>present</code> with <code>poor quality</code>. Our proposed solution is to take only <code>high quality</code> and <code>present</code> observations. The observations with ambiguous call qualities are rare, and thus I am not worried about excluding them for simplicity.</p>

<p><strong>Differential expression:</strong> In <code>Homo_sapiens_diffexpr-anatomy-simple</code> differentially expressed observations are split between low and high quality. Low quality is <a href="http://bgee.org/?page=doc&amp;action=call_files#single_diff_simple_col8">explained as</a>:</p>

<blockquote><p>differential expression reported as low quality, or there exists a conflict for the same gene, anatomical entity and developmental stage, from different analyses of a same data type (conflicts between different data types are treated differently). For instance, an analysis showed a gene to be over-expressed in a condition, while another analysis showed the same gene to be under-expressed or not differentially expressed in the same condition. Such conflicts are resolved by a voting system based on the number of conditions compared, weighted by p-value. Note that in one case, this quality level is used to reconcile conflicting calls from different data types: when a data type produced an under-expression call, while a different data type has shown that the same gene was never seen as expressed in the same condition. In that case, the overall summary is under-expression low quality.</p></blockquote>

<p>We are undecided whether to omit low quality differential expression edges.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="3" data-comment-id="541">
  <td class="comment-profile-avatar">
		<a href="/u/fbastian"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/s/img/p_default_medium.png' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="541"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/fbastian">Frederic Bastian</a>
				
				<span class="light">&nbsp;<a href="#3" class="light">611 days</a>
          
        </span>
      </div>

    

    <p>For expression calls, besides being more stringent, it is also possible to discard anatomical entities close to the root of the ontology, that are less informative, and that benefits from the propagation from lots of substructures. See also <a href="https://github.com/owlcollab/owltools/issues/145">https://github.com/owlcollab/owltools/issues/145</a> for a related discussion.</p>

<p>For differential expression, well, as said before, I'm not really surprised with this number of 5,000 differentially expressed genes in some structures. We are still willing to update our FDR computation to take into account all analyses, as mentionned <a href="http://thinklab.com/discussion/tissue-specific-gene-expression-resources/81#9" class="citation thread-citation" data-key="10.15363/thinklab.d81">here</a> (we are currently updating our differential expression pipeline).</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="4" data-comment-id="1143">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="1143"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#4" class="light">493 days</a>
          
        </span>
      </div>

    

    <h1>Methods for reducing the number of Bgee expression relationships</h1>

<p><a href="/u/fbastian" class="username">@fbastian</a>, we're looking to reduce the number of expression relationships extracted from Bgee. Our motivation is twofold:</p>

<ol><li>Our hetnet <a href="https://github.com/dhimmel/integrate/blob/dff453c020bbea953adc6cc3225235e445ba94f9/data/summary/metaedges.tsv#L3" title="hetio-ind metaedge summaries">currently contains</a> over 1 million <em>Anatomy–expresses–Gene</em> relationships. This high number of relationships is causing computational bottlenecks.</li><li>Our network permutations <a href="http://thinklab.com/discussion/assessing-the-effectiveness-of-our-hetnet-permutations/178#1" class="citation thread-citation" data-key="10.15363/thinklab.d178">do little to randomize the expression relationships</a>. In other words, too many expression relationships connect super anatomies — anatomies which express most genes — limiting the information content of the edge.</li></ol>

<p>From our previous conversations, it appears that there are three ways to proceed:</p>

<ol><li>Exclude relationships for general anatomies, as <a href="#3">suggested above</a>.</li><li>Use only RNA-Seq experiments. <a href="/u/fbastian" class="username">@fbastian</a> <a href="http://thinklab.com/discussion/tissue-specific-gene-expression-resources/81#5" class="citation thread-citation" data-key="10.15363/thinklab.d81">mentions</a> that using only RNA-Seq data avoids the ambiguity states. So what happens when a gene is present in one experiment but not the other for the same data type and conditions? With RNA-seq we should be able to adjust the RPKM inclusion threshold.</li><li>Use unpropagated relationships, so for example genes expressed in for brain gray matter would not automatically be transmitted to brain.</li></ol>

<p>These options are not mutually exclusive. We can choose any combination of the three.</p>

<p>I am leaning towards option 3, because I think studies will often be performed on clinically relevant anatomies. In other words, we may accomplish the goal of 1 (removing overly broad anatomies) by including only relationships to their directly annotated anatomies. Additionally, since our hetnet contains <a href="http://thinklab.com/discussion/tissue-node/41#16" class="citation thread-citation" data-key="10.15363/thinklab.d41">many nested levels Uberon terms</a>,  we would not be throwing out too many experiments entirely. In the future, we can even write queries that perform the propagation in realtime.</p>

<p><strong>Data complications:</strong> In <code>Homo_sapiens_expr-complete.tsv</code> version 13.1, all the values for <code>In situ call quality</code> and <code>In situ call quality</code> equal <code>no data</code>. Additionally, all values for <code>Including in situ observed data</code> are <code>no</code>. <a href="/u/fbastian" class="username">@fbastian</a>, I assume this is a bug? Is there a workaround? Advice on the specific filters to apply on which files to achieve options 2 or 3 would be appreciated.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="5" data-comment-id="1147">
  <td class="comment-profile-avatar">
		<a href="/u/fbastian"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/s/img/p_default_medium.png' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="1147"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/fbastian">Frederic Bastian</a>
				
				<span class="light">&nbsp;<a href="#5" class="light">491 days</a>
          
        </span>
      </div>

    

    <p>For me, the two working solutions would either be:  </p>

<ul><li>"3. Use unpropagated relationships": actually, you would still use the propagated data, but you would consider only the anatomical structures described in the experiments. This is what our "simple" expression file contains.</li><li>or manually select a list of 50-100 anatomical structures you are interested in, and use the propagated data in them. Maybe you can also include all cell types. This is very similar to the previous solution, except that you'd have more freedom about the choice of the organs. Although you should already have lots of organs experimentally described once we include the GTEx data (see bottom of this message), so the previous solution might be good enough.</li></ul>

<p>"2. Use only RNA-Seq experiments" is incorrect: you'll still get the ambiguity states if two libraries provide contradicting information. And, it would be sad not to use other data types – lots of good information come from Affymetrix data.<br>And adjusting the RPKM threshold will not solve your problem: your problem comes from the propagation of data to upper level terms.</p>

<p>To implement solution "3", you can either use our "simple" expression file, or use the "complete" file, but filter thanks to the column "Observed data": <a href="http://bgee.org/?page=doc&amp;action=call_files#single_expr_complete_col9">http://bgee.org/?page=doc&amp;action=call_files#single_expr_complete_col9</a></p>

<p>Otherwise, we can think of alternative solutions: </p>

<ul><li>Using only over-/under-expression data. We should have a lot of them thanks to the GTEx data in the near future</li><li>Using only the best-ranked anatomical structures for each gene, rather than all data, and do the propagation based on that (e.g., in TISSUES the anatomical structures are ranked – we are going to release a gene page next week also providing ranked anatomical structures). </li><li>Using a completely different approach, based on gene lists rather than individual genes: see our new GO-like expression enrichment test: <a href="http://bgee.org/?page=top_anat">http://bgee.org/?page=top_anat</a>. I don't know if it can be applied to your network, though, but maybe you can link groups of genes to anatomical structures, rather than individual genes.</li></ul>

<p><strong>About in situ data:</strong> we don't have in situ data for human. I don't know of any resource providing systematic gene expression analyses in human using in situ hybridization. This data type is not well suited for human (e.g., I think you're not supposed to use frozen tissues). </p>

<p><strong>Additional information:</strong> we have started the pipeline for Bgee 14, that will include the GTEx data. The full pipeline run should take a few months, but you'll get the data ultimately. Also, I'll let you know about the status of our work of re-annotation in the other thread, as we have completed it.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="6" data-comment-id="1148">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="1148"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#6" class="light">490 days</a>
          
        </span>
      </div>

    

    <h2>Clarification</h2>

<p>Let's see if I understand: It is not possible to tell whether gene presence for a specific condition (anatomy–developmental stage) was from an experiment on that exact condition or was propagated. However, the simple file (or complete file filtered for <code>Observed data</code>) contains only conditions with directly annotated experiments, but any given gene presence call may be from propagation.</p>

<h2>Proposed gene presence method</h2>

<p>I think the ideal setup for our network would be <em>propagation by developmental stage but not by anatomical structure</em>. Using just the simple or complete datasets, this doesn't currently seem to be possible. However, I've what about the following workaround: using all adult stages on the simple dataset.</p>

<p>Using the simple dataset, I found all gene–anatomy pairs where <code>Expression</code> is <code>present</code> and <code>Call quality</code> is <code>high quality</code> for any adult developmental stage. To identify adult developmental stages, I filtered for <code>HsapDv:0000087</code> and its descendants (<a href="https://github.com/dhimmel/bgee/blob/5cfed945305d7ec118fe79a1486c28d6a97ee8ee/developmental-stages.ipynb">notebook</a>, <a href="https://github.com/dhimmel/bgee/blob/5cfed945305d7ec118fe79a1486c28d6a97ee8ee/data/stages.tsv#L76">dataset</a>).</p>

<p>The resulting presence/absence matrix of gene expression is 16,257 genes × 188 anatomies  (<a href="https://github.com/dhimmel/bgee/blob/5cfed945305d7ec118fe79a1486c28d6a97ee8ee/bgee.ipynb">notebook</a>, <a href="https://github.com/dhimmel/bgee/blob/5cfed945305d7ec118fe79a1486c28d6a97ee8ee/data/present-in-adult.tsv.gz">dataset</a>) compared to the <a href="http://thinklab.com/discussion/tissue-specific-gene-expression-resources/81#8" class="citation thread-citation" data-key="10.15363/thinklab.d81">previous dimensions</a> of 16,278 genes × 666 anatomies. We hope that this pruning of anatomies will help address the network problems we <a href="#4">were facing</a>.</p>

<h2>Miscellaneous</h2>

<p>Thanks <a href="/u/fbastian" class="username">@fbastian</a> for suggesting other possible approaches. For future networks, we will revisit these options. For now we're looking for the most straightforward and immediately actionable option.</p>

<blockquote><p>We don't have in situ data for human. I don't know of any resource providing systematic gene expression analyses in human using in situ hybridization.</p></blockquote>

<p>I had misinterpreted <code>Including in situ observed data</code> to mean <code>Observed data</code></p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="7" data-comment-id="1149">
  <td class="comment-profile-avatar">
		<a href="/u/fbastian"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/s/img/p_default_medium.png' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="1149"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/fbastian">Frederic Bastian</a>
				
				<span class="light">&nbsp;<a href="#7" class="light">490 days</a>
          
        </span>
      </div>

    

    <blockquote><p> the simple file (or complete file filtered for Observed data) contains only conditions with directly annotated experiments, but any given gene presence call may be from propagation.</p></blockquote>

<p>No, you have the guarantee that there exists an unpropagated call for that very gene. What you can't know for sure from these files, is where the quality level comes from. For instance: </p>

<pre><code>gene A expressed in brain with low quality from experiment A
gene A expressed in brain substructure with high quality from experiment B
=&gt; gene A expressed in brain with high quality in Bgee files</code></pre>

<p>Or </p>

<pre><code>gene A NOT expressed in brain from RNA-Seq experiment A
gene A expressed in brain substructure from Affymetrix experiment B
=&gt; gene A with lowly conflicting status in brain substructure, in Bgee files </code></pre>

<p>Data in adult for human must represent 90% of the data, so you shouldn't add much by considering all developmental stages. </p>

<p>If you want, I can provide you with a completely unpropagated dataset. But I think it's good to benefit from propagation (e.g., to determine that 2 genes are both expressed in brain, which you might miss if they were expressed in different brain substructures). </p>

<blockquote><p>I had misinterpreted <code>Including in situ observed data</code> to mean <code>Observed data</code></p></blockquote>

<p>I see, we should rename these fields with <code>in situ hybridization</code> then.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="8" data-comment-id="1152">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="1152"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#8" class="light">486 days</a>
          
        </span>
      </div>

    

    <h1>Releasing <code>dhimmel/bgee v1.0</code></h1>

<p>We've released <a href="https://github.com/dhimmel/bgee/tree/08ba54e83ee8e28dec22b4351d29e23f1d034d30" title="GitHub repository">version 1.0</a> of our Bgee processing <span class="citation">[<a href="https://doi.org/10.5281/zenodo.47157" class="citation" data-key="10.5281/zenodo.47157">1</a>]</span>.</p>

<h2>Datasets</h2>

<p>Genes are identified with Entrez GeneIDs and anatomical structures (anatomies) are identified by Cell Ontology or Uberon terms.</p>

<ul><li><a href="https://github.com/dhimmel/bgee/blob/08ba54e83ee8e28dec22b4351d29e23f1d034d30/data/present-in-adult.tsv.gz"><code>present-in-adult.tsv.gz</code></a> indicates whether a gene is present (<code>1</code>) or absent (<code>0</code>) for a given anatomy in human adults. The dataset is a matrix of 16,257 genes × 188 anatomies.</li><li><a href="https://github.com/dhimmel/bgee/blob/08ba54e83ee8e28dec22b4351d29e23f1d034d30/data/diffex.tsv.gz"><code>diffex.tsv.gz</code></a> indicates whether a gene is over-expressed (<code>1</code>), under-expressed (<code>-1</code>), or non-deferentially expressed (<code>0</code>) for a given anatomy. The dataset is a matrix of 18,620 genes × 98 anatomies.</li></ul>

<h2>Changelog</h2>

<p>Compared to <a href="http://thinklab.com/discussion/tissue-specific-gene-expression-resources/81#8" class="citation thread-citation" data-key="10.15363/thinklab.d81">version 0</a> — the Bgee data in the <a href="http://thinklab.com/discussion/one-network-to-rule-them-all/102" class="citation thread-citation" data-key="10.15363/thinklab.d102">initial version</a> of our hetnet — the following change was made:</p>

<ul><li>We adopted the modification <a href="#6">proposed above</a>: gene presence was extracted from the simple Bgee dataset, which <a href="#6">guarantees</a> that each <em>Anatomy–expresses–Gene</em> relationship contains direct (unpropagated) experimental data.</li></ul>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

    		
		</table>

    

    &nbsp;<br>
    <a class="btn btn-primary btn-lg" href="/signup?next=/discussion/processing-bgee-for-tissue-specific-gene-presence-and-overunder-expression/124&suppress_message=1" style>Join to Reply</a>


		<table class="references table-td-5" style="display:none;"><tr><td><a href='/doi/10.1007/978-3-540-69828-9_12' class='pub-comments tt-right' title='Article comments'>8</a></td><td><div class='pub-order'>1.</div></td><td><div class='doi-body' data-key='10.1007/978-3-540-69828-9_12'><a href='/doi/10.1007/978-3-540-69828-9_12' class='pub-title'>Bgee: Integrating and Comparing Heterogeneous Transcriptome Data Among Species</a><div class='pub-body'>Frederic Bastian, Gilles Parmentier, Julien Roux, Sebastien Moretti, Vincent Laudet, Marc Robinson-Rechavi (2008) <i>Data Integration in the Life Sciences</i>. doi:<a class='pub-doi' href='https://doi.org/10.1007/978-3-540-69828-9_12'>10.1007/978-3-540-69828-9_12</a></div></div></td></tr><tr><td><a href='/doi/10.15363/thinklab.d81' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>2.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d81'><a href='/discussion/tissue-specific-gene-expression-resources/81' class='pub-title'>Tissue-specific gene expression resources</a><div class='pub-body'>Daniel Himmelstein, Frederic Bastian (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d81'>10.15363/thinklab.d81</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d81' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>3.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d81'><a href='/discussion/tissue-specific-gene-expression-resources/81' class='pub-title'>Tissue-specific gene expression resources</a><div class='pub-body'>Daniel Himmelstein, Frederic Bastian (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d81'>10.15363/thinklab.d81</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d81' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>4.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d81'><a href='/discussion/tissue-specific-gene-expression-resources/81' class='pub-title'>Tissue-specific gene expression resources</a><div class='pub-body'>Daniel Himmelstein, Frederic Bastian (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d81'>10.15363/thinklab.d81</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d81' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>5.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d81'><a href='/discussion/tissue-specific-gene-expression-resources/81' class='pub-title'>Tissue-specific gene expression resources</a><div class='pub-body'>Daniel Himmelstein, Frederic Bastian (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d81'>10.15363/thinklab.d81</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d81' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>6.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d81'><a href='/discussion/tissue-specific-gene-expression-resources/81' class='pub-title'>Tissue-specific gene expression resources</a><div class='pub-body'>Daniel Himmelstein, Frederic Bastian (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d81'>10.15363/thinklab.d81</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d81' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>7.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d81'><a href='/discussion/tissue-specific-gene-expression-resources/81' class='pub-title'>Tissue-specific gene expression resources</a><div class='pub-body'>Daniel Himmelstein, Frederic Bastian (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d81'>10.15363/thinklab.d81</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d81' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>8.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d81'><a href='/discussion/tissue-specific-gene-expression-resources/81' class='pub-title'>Tissue-specific gene expression resources</a><div class='pub-body'>Daniel Himmelstein, Frederic Bastian (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d81'>10.15363/thinklab.d81</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d81' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>9.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d81'><a href='/discussion/tissue-specific-gene-expression-resources/81' class='pub-title'>Tissue-specific gene expression resources</a><div class='pub-body'>Daniel Himmelstein, Frederic Bastian (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d81'>10.15363/thinklab.d81</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d81' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>10.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d81'><a href='/discussion/tissue-specific-gene-expression-resources/81' class='pub-title'>Tissue-specific gene expression resources</a><div class='pub-body'>Daniel Himmelstein, Frederic Bastian (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d81'>10.15363/thinklab.d81</a></div></div></td></tr><tr><td><a href='/doi/10.7717/peerj.1054' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>11.</div></td><td><div class='doi-body' data-key='10.7717/peerj.1054'><a href='/doi/10.7717/peerj.1054' class='pub-title'>Comprehensive comparison of large-scale tissue expression datasets</a><div class='pub-body'>Alberto Santos, Kalliopi Tsafou, Christian Stolte, Sune Pletscher-Frankild, Seán I. O’Donoghue, Lars Juhl Jensen (2015) <i>PeerJ</i>. doi:<a class='pub-doi' href='https://doi.org/10.7717/peerj.1054'>10.7717/peerj.1054</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d81' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>12.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d81'><a href='/discussion/tissue-specific-gene-expression-resources/81' class='pub-title'>Tissue-specific gene expression resources</a><div class='pub-body'>Daniel Himmelstein, Frederic Bastian (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d81'>10.15363/thinklab.d81</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d81' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>13.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d81'><a href='/discussion/tissue-specific-gene-expression-resources/81' class='pub-title'>Tissue-specific gene expression resources</a><div class='pub-body'>Daniel Himmelstein, Frederic Bastian (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d81'>10.15363/thinklab.d81</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d81' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>14.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d81'><a href='/discussion/tissue-specific-gene-expression-resources/81' class='pub-title'>Tissue-specific gene expression resources</a><div class='pub-body'>Daniel Himmelstein, Frederic Bastian (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d81'>10.15363/thinklab.d81</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d81' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>15.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d81'><a href='/discussion/tissue-specific-gene-expression-resources/81' class='pub-title'>Tissue-specific gene expression resources</a><div class='pub-body'>Daniel Himmelstein, Frederic Bastian (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d81'>10.15363/thinklab.d81</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d178' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>16.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d178'><a href='/discussion/assessing-the-effectiveness-of-our-hetnet-permutations/178' class='pub-title'>Assessing the effectiveness of our hetnet permutations</a><div class='pub-body'>Daniel Himmelstein (2016) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d178'>10.15363/thinklab.d178</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d81' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>17.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d81'><a href='/discussion/tissue-specific-gene-expression-resources/81' class='pub-title'>Tissue-specific gene expression resources</a><div class='pub-body'>Daniel Himmelstein, Frederic Bastian (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d81'>10.15363/thinklab.d81</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d41' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>18.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d41'><a href='/discussion/tissue-node/41' class='pub-title'>Tissue Node</a><div class='pub-body'>Venkat Malladi, Daniel Himmelstein, Chris Mungall (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d41'>10.15363/thinklab.d41</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d81' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>19.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d81'><a href='/discussion/tissue-specific-gene-expression-resources/81' class='pub-title'>Tissue-specific gene expression resources</a><div class='pub-body'>Daniel Himmelstein, Frederic Bastian (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d81'>10.15363/thinklab.d81</a></div></div></td></tr><tr><td><a href='/doi/10.5281/zenodo.47157' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>20.</div></td><td><div class='doi-body' data-key='10.5281/zenodo.47157'><a href='/doi/10.5281/zenodo.47157' class='pub-title'>dhimmel/bgee v1.0: Anatomy-specific gene expression in humans from Bgee</a><div class='pub-body'>Daniel Himmelstein, Frederic Bastian, Sergio Baranzini (2016) <i>Zenodo</i>. doi:<a class='pub-doi' href='https://doi.org/10.5281/zenodo.47157'>10.5281/zenodo.47157</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d81' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>21.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d81'><a href='/discussion/tissue-specific-gene-expression-resources/81' class='pub-title'>Tissue-specific gene expression resources</a><div class='pub-body'>Daniel Himmelstein, Frederic Bastian (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d81'>10.15363/thinklab.d81</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d102' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>22.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d102'><a href='/discussion/one-network-to-rule-them-all/102' class='pub-title'>One network to rule them all</a><div class='pub-body'>Daniel Himmelstein, Lars Juhl Jensen (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d102'>10.15363/thinklab.d102</a></div></div></td></tr></table>

    


	</td>
	<td class="thread-sidebar">


			

      <a class="btn btn-default follow-link btn-follow btn-lg is-following tt-top full-width" title="Follow discussion" data-id="124" data-ctype="thread" data-is-following="false">&nbsp;</a>

      
        <div class="status-box text-center"><span style="">Status:</span> 
  Completed

</div>
      

      
        
      


				

					<div class="details">Views</div>
					<span style="font-size:20px;">139</span>

        

				
					<div class="details">Topics</div>
					<div style="margin-bottom:15px;">
		            <a href="/topics/gene-expression" class="tag">Gene Expression</a><a href="/topics/over-expression" class="tag">Over-Expression</a><a href="/topics/bgee" class="tag">Bgee</a><a href="/topics/under-expression" class="tag">Under-Expression</a><a href="/topics/transcriptome" class="tag">Transcriptome</a>
					</div>
				


		
			<div class="details">Referenced by</div>
			<div style="font-size:12px;"><ul style="padding-left:16px !important;">
			
				<li>
					
					Research report: <a href='/p/rephetio/report'>Rephetio: Repurposing drugs on a hetnet</a>
				</li>
			
				<li>
					
					<a href='/discussion/exploring-the-power-of-hetionet-a-cypher-query-depot/220' class='thread-title'>Exploring the power of Hetionet: a Cypher query depot</a>
				</li>
			
				<li>
					
					<a href='/discussion/assessing-the-effectiveness-of-our-hetnet-permutations/178' class='thread-title'>Assessing the effectiveness of our hetnet permutations</a>
				</li>
			
				<li>
					
					<a href='/discussion/tissue-specific-gene-expression-resources/81' class='thread-title'>Tissue-specific gene expression resources</a>
				</li>
			
				</ul>
			</div>
		

		
			<div class="details">Cite this as</div>
			<span style="font-size:12px;">
				Daniel Himmelstein, Frederic Bastian (2015) Processing Bgee for tissue-specific gene presence and over/under-expression. <i>Thinklab.</i> doi:<a href='https://doi.org/10.15363/thinklab.d124'>10.15363/thinklab.d124</a>
			</span>
		

		<div class="details">License</div>
		<p><a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0;width:88px;height:31px;" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a></p>

		<div class="details">Share</div>
		
			
				<a class="share-button fb" onclick="javascript:void(window.open('https://www.facebook.com/sharer/sharer.php?u=https%3A//thinklab.com/discussion/processing-bgee-for-tissue-specific-gene-presence-and-overunder-expression/124','','width=600,height=460,menubar=no,status=no'));" href="#"></a>
<a class="share-button twitter" onclick="javascript:void(window.open('https://twitter.com/share?text=Processing%20Bgee%20for%20tissue-specific%20gene%20presence%20and%20over/under-expression&url=https%3A//thinklab.com/discussion/processing-bgee-for-tissue-specific-gene-presence-and-overunder-expression/124','','width=600,height=300,menubar=no,status=no'));" href="#"></a>
<a class="share-button google" onclick="javascript:void(window.open('https://plus.google.com/share?url=https%3A//thinklab.com/discussion/processing-bgee-for-tissue-specific-gene-presence-and-overunder-expression/124','','width=510,height=460,menubar=no,status=no'));" href="#"></a>

			
		

	</td>
</tr></table>


		</div>


		
	    	<div class="footer">
<div style="background:#dedede;">
	<div class="main-centered-div text-center" style="padding:20px;">
		<h4 style="margin-bottom:0px;font-weight:bold;color:gray;">Feedback? Questions? Email us: <a href="/cdn-cgi/l/email-protection#254d4049494a65514d4c4b4e4944470b464a48"><span class="__cf_email__" data-cfemail="0b636e6767644b7f63626560676a6925686466">[email&#160;protected]</span><script data-cfhash='f9e31' type="text/javascript">/* <![CDATA[ */!function(t,e,r,n,c,a,p){try{t=document.currentScript||function(){for(t=document.getElementsByTagName('script'),e=t.length;e--;)if(t[e].getAttribute('data-cfhash'))return t[e]}();if(t&&(c=t.previousSibling)){p=t.parentNode;if(a=c.getAttribute('data-cfemail')){for(e='',r='0x'+a.substr(0,2)|0,n=2;a.length-n;n+=2)e+='%'+('0'+('0x'+a.substr(n,2)^r).toString(16)).slice(-2);p.replaceChild(document.createTextNode(decodeURIComponent(e)),c)}p.removeChild(t)}}catch(u){}}()/* ]]> */</script></a></h4>
	</div>
</div>
<div class="main-footer">
	<div class="main-centered-div">

<table class="full-width"><tr>
	<td class="footer-nav">

        <div class="foot-head">Explore</div>
        <a href="/proposals">Proposals</a><br>
        <a href="/projects">Projects</a><br>
        <a href="/publications">Article discussion</a><br>
        <a href="/discussion">All discussion</a><br>
        <a href="/topics">Topics</a><br>
        <!-- <a href="/funders">Funders</a><br> -->
        <a href="/funding">Funding opportunities</a><br>

	</td>
	<td class="footer-nav">

        <!-- <div class="foot-head">Help &amp; feedback</div>	 -->
        <!-- <div class="foot-head">Intro</div>
        <a href="/benefits/for-science">Why</a><br>
        <a href="/how-it-works">How it works</a><br>

<br> -->

        <div class="foot-head">Community</div>
        <a href="/leaderboard">Leaderboard</a><br>
        <a href="/p/meta">Thinklab Meta</a><br>
				<a href="/help/faq">Help</a><br>
				<!-- <a href="/new-project">Start a proposal</a><br> -->
        <!-- <a href="/benefits/proposal">Benefits</a><br> -->
        <!-- <a href="/funding">Get funded</a><br> -->
        <!-- <a href="/participate">Become a contributor</a><br> -->
        <!-- <a href="/for-funders">For science funders</a><br> -->
        <!-- <a href="/for-institutions">For institutions</a><br> -->

	</td>
	<td class="footer-nav">



        <div class="foot-head">About</div>
        <a href="/about">Overview</a><br>
        <a href="/benefits">Benefits</a><br>
        <a href="/how-it-works">How it works</a><br>
        <a href="/about/story">Our story</a><br>
        <a href="/blog">Our blog</a><br>
        <a href="/about/jobs">Join the team</a><br>
        <br>

        <!-- <div class="foot-head">Help</div>
        <a href="/help">Help</a><br>
		 -->


	</td>
	<td class="footer-nav">
		<a href="http://twitter.com/thinklab" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/twitter_com.png" width="40"/></a>
		<a href="https://www.facebook.com/pages/Thinklab/310895039088185" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/facebook_com.png" width="40"/></a>
		<a href="https://plus.google.com/115578983466800753402/" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/plus_google_com.png" width="40"/></a>
		<a href="https://www.linkedin.com/company/thinklab-co" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/linkedin_com.png" width="40"/></a>
		<br><br>
		<span class="light" style='font-size:12px;'>&copy; 2015 Thinklab</span>

	</td>

</tr></table>

</div>
</div>
</div>

		

	</div>


  <script></script>
  <script>
  var MEDIA_URL = 'https://think-lab.s3.amazonaws.com/m/';
  var STATIC_URL = 'https://think-lab.s3.amazonaws.com/s/';
  var page_is_authenticated = false;
  var page_page = 'discussion';
  var page_profile_id = undefined;
</script>



	<!-- Note different script type. MUST come before MathJax.js -->
	<script type="text/x-mathjax-config">
	    MathJax.Hub.Config({
	        "HTML-CSS": { preferredFont: "TeX", availableFonts: ["STIX","TeX"],
	                        scale: 110, linebreaks: { automatic:true },
	                        EqnChunk: (MathJax.Hub.Browser.isMobile ? 10 : 50) },
	        tex2jax: { inlineMath: [ ["$$", "$$"], ["\\(","\\)"] ],
	                    displayMath: [ ["$$$","$$$"], ["\\[", "\\]"] ],
	                    processEscapes: true,}
	            });

	</script>
    <script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>





  <script src="https://code.jquery.com/jquery-1.10.2.min.js"></script>
  <script src="https://code.jquery.com/ui/1.11.4/jquery-ui.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" integrity="sha384-0mSbJDEHialfmuBBQP6A4Qrprq5OVfW37PRR3j5ELqxss1yVqOtnepnHVP9aJ7xS" crossorigin="anonymous"></script>
  <script src="https://think-lab.s3.amazonaws.com/s/dist/main.min.js"></script>
  



  
	
  
  
  
<script>
$(function() {
  recordHit('thread', 124);
});
</script>



<script>/* <![CDATA[ */(function(d,s,a,i,j,r,l,m,t){try{l=d.getElementsByTagName('a');t=d.createElement('textarea');for(i=0;l.length-i;i++){try{a=l[i].href;s=a.indexOf('/cdn-cgi/l/email-protection');m=a.length;if(a&&s>-1&&m>28){j=28+s;s='';if(j<m){r='0x'+a.substr(j,2)|0;for(j+=2;j<m&&a.charAt(j)!='X';j+=2)s+='%'+('0'+('0x'+a.substr(j,2)^r).toString(16)).slice(-2);j++;s=decodeURIComponent(s)+a.substr(j,m-j)}t.innerHTML=s.replace(/</g,'&lt;').replace(/\>/g,'&gt;');l[i].href='mailto:'+t.value}}catch(e){}}}catch(e){}})(document);/* ]]> */</script></body>
</html>
