<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8" /><script type="text/javascript">window.NREUM||(NREUM={}),__nr_require=function(e,n,t){function r(t){if(!n[t]){var o=n[t]={exports:{}};e[t][0].call(o.exports,function(n){var o=e[t][1][n];return r(o||n)},o,o.exports)}return n[t].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<t.length;o++)r(t[o]);return r}({1:[function(e,n,t){function r(){}function o(e,n,t){return function(){return i(e,[c.now()].concat(u(arguments)),n?null:this,t),n?void 0:this}}var i=e("handle"),a=e(2),u=e(3),f=e("ee").get("tracer"),c=e("loader"),s=NREUM;"undefined"==typeof window.newrelic&&(newrelic=s);var p=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],d="api-",l=d+"ixn-";a(p,function(e,n){s[n]=o(d+n,!0,"api")}),s.addPageAction=o(d+"addPageAction",!0),s.setCurrentRouteName=o(d+"routeName",!0),n.exports=newrelic,s.interaction=function(){return(new r).get()};var m=r.prototype={createTracer:function(e,n){var t={},r=this,o="function"==typeof n;return i(l+"tracer",[c.now(),e,t],r),function(){if(f.emit((o?"":"no-")+"fn-start",[c.now(),r,o],t),o)try{return n.apply(this,arguments)}finally{f.emit("fn-end",[c.now()],t)}}}};a("setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(e,n){m[n]=o(l+n)}),newrelic.noticeError=function(e){"string"==typeof e&&(e=new Error(e)),i("err",[e,c.now()])}},{}],2:[function(e,n,t){function r(e,n){var t=[],r="",i=0;for(r in e)o.call(e,r)&&(t[i]=n(r,e[r]),i+=1);return t}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],3:[function(e,n,t){function r(e,n,t){n||(n=0),"undefined"==typeof t&&(t=e?e.length:0);for(var r=-1,o=t-n||0,i=Array(o<0?0:o);++r<o;)i[r]=e[n+r];return i}n.exports=r},{}],4:[function(e,n,t){n.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(e,n,t){function r(){}function o(e){function n(e){return e&&e instanceof r?e:e?f(e,u,i):i()}function t(t,r,o,i){if(!d.aborted||i){e&&e(t,r,o);for(var a=n(o),u=m(t),f=u.length,c=0;c<f;c++)u[c].apply(a,r);var p=s[y[t]];return p&&p.push([b,t,r,a]),a}}function l(e,n){v[e]=m(e).concat(n)}function m(e){return v[e]||[]}function w(e){return p[e]=p[e]||o(t)}function g(e,n){c(e,function(e,t){n=n||"feature",y[t]=n,n in s||(s[n]=[])})}var v={},y={},b={on:l,emit:t,get:w,listeners:m,context:n,buffer:g,abort:a,aborted:!1};return b}function i(){return new r}function a(){(s.api||s.feature)&&(d.aborted=!0,s=d.backlog={})}var u="nr@context",f=e("gos"),c=e(2),s={},p={},d=n.exports=o();d.backlog=s},{}],gos:[function(e,n,t){function r(e,n,t){if(o.call(e,n))return e[n];var r=t();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(e,n,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return e[n]=r,r}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],handle:[function(e,n,t){function r(e,n,t,r){o.buffer([e],r),o.emit(e,n,t)}var o=e("ee").get("handle");n.exports=r,r.ee=o},{}],id:[function(e,n,t){function r(e){var n=typeof e;return!e||"object"!==n&&"function"!==n?-1:e===window?0:a(e,i,function(){return o++})}var o=1,i="nr@id",a=e("gos");n.exports=r},{}],loader:[function(e,n,t){function r(){if(!x++){var e=h.info=NREUM.info,n=d.getElementsByTagName("script")[0];if(setTimeout(s.abort,3e4),!(e&&e.licenseKey&&e.applicationID&&n))return s.abort();c(y,function(n,t){e[n]||(e[n]=t)}),f("mark",["onload",a()+h.offset],null,"api");var t=d.createElement("script");t.src="https://"+e.agent,n.parentNode.insertBefore(t,n)}}function o(){"complete"===d.readyState&&i()}function i(){f("mark",["domContent",a()+h.offset],null,"api")}function a(){return E.exists&&performance.now?Math.round(performance.now()):(u=Math.max((new Date).getTime(),u))-h.offset}var u=(new Date).getTime(),f=e("handle"),c=e(2),s=e("ee"),p=window,d=p.document,l="addEventListener",m="attachEvent",w=p.XMLHttpRequest,g=w&&w.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:w,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var v=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-1039.min.js"},b=w&&g&&g[l]&&!/CriOS/.test(navigator.userAgent),h=n.exports={offset:u,now:a,origin:v,features:{},xhrWrappable:b};e(1),d[l]?(d[l]("DOMContentLoaded",i,!1),p[l]("load",r,!1)):(d[m]("onreadystatechange",o),p[m]("onload",r)),f("mark",["firstbyte",u],null,"api");var x=0,E=e(4)},{}]},{},["loader"]);</script><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info={"beacon":"bam.nr-data.net","queueTime":0,"licenseKey":"cd1101aa94","agent":"","transactionName":"bgcDMkBZW0NZBkdcWVdNJxNcW0FZVwscV1dKB08QW11CQxYTWlBBZgEOC19dW0QCEVtHU1gG","applicationID":"6369297","errorBeacon":"bam.nr-data.net","applicationTime":484}</script>
    <title>STARGEO: expression signatures for disease using crowdsourced GEO annotation | Thinklab</title>
	  <link rel="shortcut icon" href="https://think-lab.s3.amazonaws.com/s/img/favicon.ico" type="image/x-icon" />
    <meta property="og:image" content="https://think-lab.s3.amazonaws.com/s/img" />
    
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
  <link rel="stylesheet" href="https://code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css" />

<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/highlightjs/github.css" />
<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/css/css.css" />
<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/css/bootstrap-multiselect.css" />
<!-- <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'> -->

    
    
    
    
  
    <meta name="citation_doi" content="10.15363/thinklab.d96">
    <meta name="citation_title" content="STARGEO: expression signatures for disease using crowdsourced GEO annotation">
    <meta name="citation_online_date" content="July 28, 2015">
    <meta name="citation_author" content="Daniel Himmelstein, Frederic Bastian, Dexter Hadley, Casey Greene">
    <meta name="citation_journal_title" content="Thinklab">
    <meta name="citation_publisher" content="Thinklab">
  

    


	<!-- GoogleAnalytics -->
	<script>
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
	(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
	m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

	ga('create', 'UA-48550230-1', 'thinklab.com');
	ga('require', 'displayfeatures');
	
	ga('send', 'pageview');

	</script>






</head>
<body >
  <form id="form_action" method="post">
    <input type='hidden' name='csrfmiddlewaretoken' value='pQNiZQX4gF2vyFeXrnbUl9EmuTWA5MG1' />
</form>

	

	<div class="body-wrapper">

		


<!-- <div style="height:5px;background-image:url('https://think-lab.s3.amazonaws.com/s/img/top-bg.gif');"></div> -->
<div class="gradient">
	<div class="main-centered-div header-container">

<table class="full-width" style="height:60px;"><tr>
	<td style="width:200px;vertical-align:middle;padding-bottom:15px;">
		<a href="/" style="font-weight:normal;color:#212121;"><img src="https://think-lab.s3.amazonaws.com/s/img/logo.png" width="160"></a>
	</td><td style="vertical-align:middle;text-align:left;padding-top:6px;width:100%;padding-left:40px;">
		<ul id="not-search-block"  class="main-header">


			


				<li class="dropdown">
					<a class="dropdown-toggle" data-toggle="dropdown" href="#">Browse <span class="caret"></span></a>
					<ul class="dropdown-menu" role="menu">
						<li><a href="/proposals">Proposals</a></li>
						<li><a href="/projects">Projects</a></li>
						<li><a href="/publications">Article discussion</a></li>
						<li><a href="/discussion">All discussion</a></li>
					</ul>
				</li>
				<li><a href="/leaderboard">Leaderboard</a></li>

			
				<li><a href="/about">About</a></li>
			


		</ul>

    <ul id="search-block" class="main-header" style="display: none; margin-right: 30px;">
        <li class="full-width">
            <form method="get" action="/search">
			<div class="input-group full-width">
				<input name="q" type="text" class="form-control search-input">
			</div>
            </form>
        </li>
    </ul>

	</td><td style="vertical-align:middle;text-align:right;padding-top:6px; min-width: 1%; white-space: nowrap;">
		<ul class="main-header pull-right" style="padding-left: 0;">
    		
                <li><a href="/login" style="color:silver;">Login</a></li>
                <li style="margin-right:0"><a class="btn btn-default btn-sm btn-upper" href="/signup">Join</a></li>
            

        </ul>
    </ul>
	</td>
</tr></table>

<!-- <div id="global-search-div" class="main-centered-div" style="display:none;">
	<input type="text" value="" class="thread-search form-control" placeholder="Search" data-action="/discussion">
</div> -->

	</div>
</div>

		
	  



	  


		

		<div class="dark-head"><div class="main-centered-div">

			

		</div></div>
		
		<div class="main-centered-div" style="margin-top:40px;padding-bottom:340px">
	    

<div style="margin-top:-20px; margin-bottom:20px;">

    <a class="side-box" href="/p/rephetio" style="background-color:#ebfdff;"><div class="side-type">Project:</div> <div class="side-title">Rephetio: Repurposing drugs on a hetnet [rephetio]</div></a><br>



</div>

<table style="margin-top:0px;"><tr>
	<td class="thread-table-content">

		<table class="full-width">
			
<tr class="comment-tr target-hl" id="1" data-comment-id="335">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="335"></a>

    

			<h2 class="page-title" style="margin-bottom:5px;margin-top:0px !important;">STARGEO: expression signatures for disease using crowdsourced GEO annotation</h2>

			<div style="margin-top:5px;color:gray;margin-bottom:20px;font-size:13px;">
				<a style="font-weight:normal;color:#212121" href="/u/dhimmel"><strong>Daniel Himmelstein</strong></a>
				<span class="is-researcher">Researcher</span>
				&nbsp; <a href="#1" class="light">710 days ago</a>
				
			</div>

      

          <hr>
		

    <p>A 2011 study <span class="citation">[<a href="https://doi.org/10.1126/scitranslmed.3001318" class="citation" data-key="10.1126/scitranslmed.3001318">1</a>]</span> introduced the idea of large-scale drug repurposing based disease expression profiles. However, the field has faced a great impediment: results from differential expression experiments are only available on a <em>per study</em> basis. Our project requires a consensus signature (that aggregates many experiments) for <em>each of 137</em> diseases.</p>

<p>A forthcoming project called <a href="http://dev.stargeo.io/">STARGEO</a> aims to provide disease-specific expression signatures on a broad scale. The webapp crowdsources <a href="http://www.ncbi.nlm.nih.gov/geo/">GEO</a> <span class="citation">[<a href="https://doi.org/10.1093/nar/gks1193" class="citation" data-key="10.1093/nar/gks1193">2</a>, <a href="https://doi.org/10.1093/nar/30.1.207" class="citation" data-key="10.1093/nar/30.1.207">3</a>]</span> annotation and performs case-control analyses based on user queries. The following video introduces the project:</p>

<p></p><div class="iframe-container"><iframe src="https://www.youtube.com/embed/61lw_d6Eoik" frameborder="0" allowfullscreen="true"></iframe></div>

<p>We now join forces with STARGEO and welcome its creator <a href="/u/idrdex" class="username">@idrdex</a> to the team! The first stage will be tagging all GEO datasets containing <a href="http://thinklab.com/discussion/unifying-disease-vocabularies/44#144" class="citation thread-citation" data-key="10.15363/thinklab.d44">DO Slim</a> diseases. STARGEO's current DO Slim coverage is <a href="https://github.com/dhimmel/stargeo/blob/master/data/DO-tag-mapping.tsv">available here</a>.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="2" data-comment-id="390">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="390"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#2" class="light">695 days</a>
          
        </span>
      </div>

    

    <h1>Ontologies for disease-centric GEO annotation</h1>

<p>STARGEO allows users to define arbitrary "tags" for sample annotation. We have been adding Disease Ontology (DO) IDs to our tag descriptions. This suffices for simple case-control comparisons, but is insufficient for more complex comparisons.</p>

<p>For example, many cancer studies, will compare tumors to healthy tissue but all samples are from cases. Therefore the contrast is not case versus control, but healthy versus diseased tissue. In general, we will want to incorporate these contrasts into our disease-specific expression signatures.</p>

<p><a href="/u/fbastian" class="username">@fbastian</a> and <a href="/u/chrismungall" class="username">@chrismungall</a>, do you know of any ontologies that could help with our annotation task? I know that Bgee focuses on healthy tissue, but I though you may be able to direct us in the right direction.</p>

<p><strong>Summary</strong>: we are gathering disease-specific expression signatures. What terminologies should we use to create contrast between samples within a study (GEO Series)?</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="3" data-comment-id="395">
  <td class="comment-profile-avatar">
		<a href="/u/fbastian"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/s/img/p_default_medium.png' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="395"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/fbastian">Frederic Bastian</a>
				
				<span class="light">&nbsp;<a href="#3" class="light">693 days</a>
          
        </span>
      </div>

    

    <p>To annotate the condition of a sample, you can use the Experimental Factor Ontology (EFO). But not sure what you mean by "contrast between samples" (do you want to annotate each sample, or have terms directly representing, e.g. "healthy vs. diseased contrast")</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="4" data-comment-id="409">
  <td class="comment-profile-avatar">
		<a href="/u/idrdex"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/s/img/p_default_medium.png' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="409"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/idrdex">Dexter Hadley</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#4" class="light">688 days</a>
          
        </span>
      </div>

    

    <p><a href="/u/fbastian" class="username">@fbastian</a></p>

<p>For now STARGEO annotations really funnel towards performing classical meta-analysis across studies given an standardized set of "cases" vs "controls".  This kind of fits with micro-array experimental design which usually "contrast" some type of case vs control.  So there is a concept of a control for a disease which may or may not be a "healthy" control.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="5" data-comment-id="410">
  <td class="comment-profile-avatar">
		<a href="/u/idrdex"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/s/img/p_default_medium.png' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="410"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/idrdex">Dexter Hadley</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#5" class="light">688 days</a>
          
        </span>
      </div>

    

    <p><a href="/u/dhimmel" class="username">@dhimmel</a> </p>

<p>We will soon have analyses working on the dev site: <a href="http://dev.stargeo.io">http://dev.stargeo.io</a>.  We will probably shut down the old site by the end of the month.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
<li id="note-163" class="inline-note toggle-parent" data-note-id="163">
	<p><a href="/u/dhimmel">Daniel Himmelstein</a>: I am not able to resolve <a href="http://dev.stargeo.org/">http://dev.stargeo.org/</a>
		
	</p>
	
	<table style="display:none;"></table>
</li>

<li id="note-164" class="inline-note toggle-parent" data-note-id="164">
	<p><a href="/u/idrdex">Dexter Hadley</a>: Make that: <a href="http://dev.stargeo.io">http://dev.stargeo.io</a>. But not live quite yet.  I'll get an update tonight on the UI.  Current only working from command line, but at least its working :)
		
	</p>
	
	<table style="display:none;"></table>
</li>

<li id="note-165" class="inline-note toggle-parent" data-note-id="165">
	<p><a href="/u/dhimmel">Daniel Himmelstein</a>: Okay, I'll switch the hyperlinks on Thinklab over to <a href="http://dev.stargeo.io/">http://dev.stargeo.io/</a>
		
	</p>
	
	<table style="display:none;"></table>
</li>

<li id="note-168" class="inline-note toggle-parent" data-note-id="168">
	<p><a href="/u/idrdex">Dexter Hadley</a>: I guess.  This is mainly for testing.  By the end of the month or soon thereafter it should revert back to .org (public marketed site) and .io will remain for testing among a chosen few (and relatively private)
		
	</p>
	
	<table style="display:none;"></table>
</li>

<li id="note-175" class="inline-note toggle-parent" data-note-id="175">
	<p><a href="/u/dhimmel">Daniel Himmelstein</a>: Okay, we can change links to <a href="http://stargeo.org">http://stargeo.org</a> now or once the public site goes live. Your call.
		
	</p>
	
	<table style="display:none;"></table>
</li>

				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="6" data-comment-id="411">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="411"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#6" class="light">688 days</a>
          
        </span>
      </div>

    

    <p>The Experimental Factor Ontology (<a href="http://www.ebi.ac.uk/efo/">EFO</a>) <span class="citation">[<a href="https://doi.org/10.1093/bioinformatics/btq099" class="citation" data-key="10.1093/bioinformatics/btq099">1</a>]</span> has a few terms related to what we need. For example,</p>

<ul><li>case control design (<a href="http://www.ebi.ac.uk/efo/EFO_0001427"><code>EFO:0001427</code></a>)</li><li>control (<a href="http://www.ebi.ac.uk/efo/EFO_0001461"><code>EFO:0001461</code></a>)</li><li>individual (<a href="http://www.ebi.ac.uk/efo/EFO_0000542"><code>EFO:0000542</code></a>)</li><li>tissue specimen (<a href="http://purl.obolibrary.org/obo/OBI_0001479"><code>OBI:0001479</code></a>)</li></ul>

<p>Ultimately, for a given disease, we want to be able to differentiate the following:</p>

<ul><li>a healthy sample from healthy individual</li><li>a healthy sample from diseased individual</li><li>a disease sample from diseased individual</li></ul>

<p>Essentially, we want to support two types of case-control analyses based on a single set of annotations:</p>

<ol><li>samples from healthy individuals versus diseased individuals</li><li>samples from healthy tissue versus diseased tissue, where all samples may come from diseased individuals</li></ol>

<p>It seems that most existing ontologies are good at describing the characteristics of a single sample — for example, its tissue of origin, the developmental stage of the donor, the phenotypes/diseases of the donor — but they are not good at allowing tagging for the sole purpose of contrast.</p>

<p><a href="/u/idrdex" class="username">@idrdex</a> suggested that we could use "qualifiers for the tags: like <code>PC_individual_case</code> vs <code>PC_individual_control</code> or <code>PC_tissue_case</code> vs <code>PC_tissue_control</code>" I like this idea and think it is a good immediate solution.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
<li id="note-167" class="inline-note toggle-parent" data-note-id="167">
	<p><a href="/u/idrdex">Dexter Hadley</a>: I think whatever we do is ultimately arbitrary and will become our 'protocol'.  We can assume that tags reference the sample itself which probably is some type of tissue and applies to the individual.  Like diabetic pancreas tissue is from a diabetic individual.  But cancer is a "mosaic" disease which puts tissue vs individual out of sync.  I think eventually for every set of annotations made on a GSE, we can allow for qualifiers from EFO to be set.
		
	</p>
	
	<table style="display:none;"></table>
</li>

				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="7" data-comment-id="413">
  <td class="comment-profile-avatar">
		<a href="/u/caseygreene"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/22_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="413"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/caseygreene">Casey Greene</a>
				
				<span class="light">&nbsp;<a href="#7" class="light">687 days</a>
          
        </span>
      </div>

    

    <p><a href="/u/idrdex" class="username">@idrdex</a>: We've been doing some extensive curation (i.e. back to the literature describing the experiments) on ~1000 samples that matter a lot to a project that we're working on. A couple questions about STARGEO:<br>1) Is it going to/does it also include samples from organisms other than human?<br>2) What's the best way to contribute these annotations? Anything programmatic and/or spreadsheet friendly?<br>3) What's the API like to extract annotations?</p>

<p>Thanks!<br>Casey</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
<li id="note-170" class="inline-note toggle-parent" data-note-id="170">
	<p><a href="/u/idrdex">Dexter Hadley</a>: 1) Yes multi species are on the books.  But for now we are focusing on humans to get the site launched.<br>2) Lets get you an account which for now is through me.  I'm idrdex at both google hangouts and Skype. </p>

<p>3) Everything for now is accessed through the site.  We are working on an API to serve the data for the computational folks.  Basically allow retrival of annotations and served matrices with matching gene ids.
		
	</p>
	
	<table style="display:none;"></table>
</li>

				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="8" data-comment-id="414">
  <td class="comment-profile-avatar">
		<a href="/u/idrdex"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/s/img/p_default_medium.png' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="414"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/idrdex">Dexter Hadley</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#8" class="light">687 days</a>
          
        </span>
      </div>

    

    <p>Analysis is working now here: <a href="http://dev.stargeo.io/analysis/">http://dev.stargeo.io/analysis/</a></p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
<li id="note-169" class="inline-note toggle-parent" data-note-id="169">
	<p><a href="/u/caseygreene">Casey Greene</a>: I took a look at that but it doesn't look like it provides hooks to the underlying annotations. Are those going to be available? For the types of analyses that we work on, those are much more valuable than the profiles.
		
	</p>
	
	<table style="display:none;"></table>
</li>

<li id="note-171" class="inline-note toggle-parent" data-note-id="171">
	<p><a href="/u/idrdex">Dexter Hadley</a>: The "annotations" interface is not ready yet.  But imagine a searchable list where you could put GSEs in for instance and get all our annotations across GSMs.  For now I could probably generate a .csv for you.   We currently have 400K+ annotations  over 100+ tags.  Some GSEs have been done up to quadruplicate and ones with multiple annotations we have kappa inter-related sats calculated.
		
	</p>
	
	<table style="display:none;"></table>
</li>

<li id="note-172" class="inline-note toggle-parent" data-note-id="172">
	<p><a href="/u/caseygreene">Casey Greene</a>: The annotations interface sounds very helpful! I'm looking forward to it! The analysis that we're doing right now isn't for human datasets, so if your curations are generally there we can't check the overlap with our own curations. We do a lot of human work though, so it'll be very helpful to us to have these annotations available.
		
	</p>
	
	<table style="display:none;"></table>
</li>

<li id="note-173" class="inline-note toggle-parent" data-note-id="173">
	<p><a href="/u/idrdex">Dexter Hadley</a>: I've got a postgres table that you can query if you like.  Let's hangout and discuss :)
		
	</p>
	
	<table style="display:none;"></table>
</li>

				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="9" data-comment-id="796">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="796"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#9" class="light">513 days</a>
          
        </span>
      </div>

    

    <h1>Gene handling quality control</h1>

<p>Currently, we have STARGEO case-control queries for 66 of our diseases. Of these 66 queries, 37 return differential expression results. The rest either have insufficient samples or fail <a href="https://github.com/idrdex/star_api/issues/13#issue-123599787" title="idrdex/star_geo#13 Specific failing analyses">due to errors</a>.</p>

<p>In the past, I remember coming across a STARGEO output (gene rows × meta-analysis columns) where many rows contained duplicate gene symbols. <a href="/u/idrdex" class="username">@idrdex</a> had also mentioned to me that mapping the probe/gene names deposited in GEO can get complicated. Therefore, I wanted to do a few quality controls before proceeding.</p>

<p>I checked into our current STARGEO analysis of differential expression for 37 diseases. I looked for three occurrences which could be due to problems with gene handling (<a href="https://github.com/dhimmel/stargeo/blob/6c5a348f2ea22edf475dc14a5eed93eb95290d7d/gene-fidelity.ipynb">notebook</a>):</p>

<ul><li>GeneID–Symbol mappings that don't exist in Entrez Gene (<a href="https://github.com/dhimmel/stargeo/blob/6c5a348f2ea22edf475dc14a5eed93eb95290d7d/diagnose/discord.tsv">results</a>). There were 2,274 ID-Symbol pairs that didn't exist in my parsing of Entrez Gene. However, most of these were not protein-coding and appeared to stem from updates to the database over time.</li><li>Rows with duplicate GeneIDs (<a href="https://github.com/dhimmel/stargeo/blob/6c5a348f2ea22edf475dc14a5eed93eb95290d7d/diagnose/duplicate_ids.tsv">results</a>). Only two rows were affected by this issue.</li><li>Rows with duplicate symbols (<a href="https://github.com/dhimmel/stargeo/blob/6c5a348f2ea22edf475dc14a5eed93eb95290d7d/diagnose/duplicate_symbols.tsv">results</a>). 72 rows were affected by this issue. It did seem however that many gene symbols were not the approved symbols but rather synonyms. Since we <a href="http://thinklab.com/discussion/using-entrez-gene-as-our-gene-vocabulary/34" class="citation thread-citation" data-key="10.15363/thinklab.d34">use Entrez Gene</a> IDs for mapping, synonyms are not a major concern for us.</li></ul>

<p>So in conclusion, I didn't detect any major issues with the gene handling in STARGEO. These quality controls do not assess the probe–gene mapping, but instead whether the gene information reported for the meta-analyses makes sense.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="10" data-comment-id="1145">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="1145"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#10" class="light">491 days</a>
          
        </span>
      </div>

    

    <h1>Initial release of STARGEO analyses</h1>

<p>We've released the first complete version of our STARGEO analysis (<a href="https://github.com/dhimmel/stargeo">repository</a> <span class="citation">[<a href="https://doi.org/10.5281/zenodo.46866" class="citation" data-key="10.5281/zenodo.46866">1</a>]</span>). Thanks <a href="/u/idrdex" class="username">@idrdex</a> for helping us get all of the queries <a href="https://github.com/idrdex/star_api/issues/13#issuecomment-191414422">running smoothly</a>.</p>

<p>In summary, we defined case-control queries for <a href="https://github.com/dhimmel/stargeo/blob/1a11633b5e0095454453335be82012a9f0f482e4/data/queries.tsv">66 diseases</a>. Of these diseases, 49 contained sufficient data (multiple studies with at least 3 samples per class). We used STARGEO's random effects meta-analysis and applied an FDR <em>p</em>-value threshold of 0.05 to identify deferentially expressed genes for each disease. </p>

<p>48,688 <em>Disease–downregulates–Gene</em> and 50,287 <em>Disease–upregulates–Gene</em> relationships were identified (<a href="https://github.com/dhimmel/stargeo/blob/1a11633b5e0095454453335be82012a9f0f482e4/data/diffex.tsv">dataset</a>). The number of dysregulated genes <a href="https://github.com/dhimmel/stargeo/blob/1a11633b5e0095454453335be82012a9f0f482e4/data/summary.tsv">varied widely</a> by disease. No deferentially expressed genes were identified for endogenous depression, which had a combined sample size of <a href="https://github.com/dhimmel/stargeo/blob/1a11633b5e0095454453335be82012a9f0f482e4/data/doslim/DOID_1595/samples.tsv">533 cases and controls</a>.</p>

<p>Now we will integrate these relationships into our hetnet. We may choose to limit each disease to the 500 most significantly up and 500 most significantly down-regulated genes.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="11" data-comment-id="1398">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="1398"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#11" class="light">218 days</a>
          
        </span>
      </div>

    

    <h1>Additional resources</h1>

<p>Here are some additional resources that claim to do similar things:</p>

<ul><li>CRowd Extracted Expression of Differential Signatures (<a href="http://amp.pharm.mssm.edu/creeds">CREEDS</a>) <span class="citation">[<a href="https://doi.org/10.1038/ncomms12846" class="citation" data-key="10.1038/ncomms12846">1</a>]</span></li><li>OMics Compendia Commons (<a href="https://omicc.niaid.nih.gov/">OMiCC</a>) <span class="citation">[<a href="https://doi.org/10.1038/nbt.3603" class="citation" data-key="10.1038/nbt.3603">2</a>]</span></li></ul>

<p>I haven't had time to look into them, but wanted to note them here in case I ever do!</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

    		
		</table>

    

    &nbsp;<br>
    <a class="btn btn-primary btn-lg" href="/signup?next=/discussion/stargeo-expression-signatures-for-disease-using-crowdsourced-geo-annotation/96&suppress_message=1" style>Join to Reply</a>


		<table class="references table-td-5" style="display:none;"><tr><td><a href='/doi/10.1126/scitranslmed.3001318' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>1.</div></td><td><div class='doi-body' data-key='10.1126/scitranslmed.3001318'><a href='/doi/10.1126/scitranslmed.3001318' class='pub-title'>Discovery and Preclinical Validation of Drug Indications Using Compendia of Public Gene Expression Data</a><div class='pub-body'>M. Sirota, J. T. Dudley, J. Kim, A. P. Chiang, A. A. Morgan, A. Sweet-Cordero, J. Sage, A. J. Butte (2011) <i>Science Translational Medicine</i>. doi:<a class='pub-doi' href='https://doi.org/10.1126/scitranslmed.3001318'>10.1126/scitranslmed.3001318</a></div></div></td></tr><tr><td><a href='/doi/10.1093/nar/gks1193' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>2.</div></td><td><div class='doi-body' data-key='10.1093/nar/gks1193'><a href='/doi/10.1093/nar/gks1193' class='pub-title'>NCBI GEO: archive for functional genomics data sets--update</a><div class='pub-body'>T. Barrett, S. E. Wilhite, P. Ledoux, C. Evangelista, I. F. Kim, M. Tomashevsky, K. A. Marshall, K. H. Phillippy, P. M. Sherman, M. Holko, A. Yefanov, H. Lee, N. Zhang, C. L. Robertson, N. Serova, S. Davis, A. Soboleva (2012) <i>Nucleic Acids Research</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/nar/gks1193'>10.1093/nar/gks1193</a></div></div></td></tr><tr><td><a href='/doi/10.1093/nar/30.1.207' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>3.</div></td><td><div class='doi-body' data-key='10.1093/nar/30.1.207'><a href='/doi/10.1093/nar/30.1.207' class='pub-title'>Gene Expression Omnibus: NCBI gene expression and hybridization array data repository</a><div class='pub-body'>R. Edgar (2002) <i>Nucleic Acids Research</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/nar/30.1.207'>10.1093/nar/30.1.207</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d44' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>4.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d44'><a href='/discussion/unifying-disease-vocabularies/44' class='pub-title'>Unifying disease vocabularies</a><div class='pub-body'>Daniel Himmelstein, Tong Shu Li (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d44'>10.15363/thinklab.d44</a></div></div></td></tr><tr><td><a href='/doi/10.1093/bioinformatics/btq099' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>5.</div></td><td><div class='doi-body' data-key='10.1093/bioinformatics/btq099'><a href='/doi/10.1093/bioinformatics/btq099' class='pub-title'>Modeling sample variables with an Experimental Factor Ontology</a><div class='pub-body'>J. Malone, E. Holloway, T. Adamusiak, M. Kapushesky, J. Zheng, N. Kolesnikov, A. Zhukova, A. Brazma, H. Parkinson (2010) <i>Bioinformatics</i>. doi:<a class='pub-doi' href='https://doi.org/10.1093/bioinformatics/btq099'>10.1093/bioinformatics/btq099</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d34' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>6.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d34'><a href='/discussion/using-entrez-gene-as-our-gene-vocabulary/34' class='pub-title'>Using Entrez Gene as our gene vocabulary</a><div class='pub-body'>Daniel Himmelstein, Casey Greene, Alexander Pico (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d34'>10.15363/thinklab.d34</a></div></div></td></tr><tr><td><a href='/doi/10.5281/zenodo.46866' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>7.</div></td><td><div class='doi-body' data-key='10.5281/zenodo.46866'><a href='/doi/10.5281/zenodo.46866' class='pub-title'>dhimmel/stargeo v1.0: differentially expressed genes for 48 diseases from STARGEO</a><div class='pub-body'>Daniel Himmelstein, Dexter Hadley, Alexander Schepanovski (2016) <i>Zenodo</i>. doi:<a class='pub-doi' href='https://doi.org/10.5281/zenodo.46866'>10.5281/zenodo.46866</a></div></div></td></tr><tr><td><a href='/doi/10.1038/ncomms12846' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>8.</div></td><td><div class='doi-body' data-key='10.1038/ncomms12846'><a href='/doi/10.1038/ncomms12846' class='pub-title'>Extraction and analysis of signatures from the Gene Expression Omnibus by the crowd</a><div class='pub-body'>Zichen Wang, Caroline D. Monteiro, Kathleen M. Jagodnik, Nicolas F. Fernandez, Gregory W. Gundersen, Andrew D. Rouillard, Sherry L. Jenkins, Axel S. Feldmann, Kevin S. Hu, Michael G. McDermott, Qiaonan Duan, Neil R. Clark, Matthew R. Jones, Yan Kou, Troy Goff, Holly Woodland, Fabio M R. Amaral, Gregory L. Szeto, Oliver Fuchs, Sophia M. Schüssler-Fiorenza Rose, Shvetank Sharma, Uwe Schwartz, Xabier Bengoetxea Bausela, Maciej Szymkiewicz, Vasileios Maroulis, Anton Salykin, Carolina M. Barra, Candice D. Kruth, Nicholas J. Bongio, Vaibhav Mathur, Radmila D Todoric, Udi E. Rubin, Apostolos Malatras, Carl T. Fulp, John A. Galindo, Ruta Motiejunaite, Christoph Jüschke, Philip C. Dishuck, Katharina Lahl, Mohieddin Jafari, Sara Aibar, Apostolos Zaravinos, Linda H. Steenhuizen, Lindsey R. Allison, Pablo Gamallo, Fernando de Andres Segura, Tyler Dae Devlin, Vicente Pérez-García, Avi Ma’ayan (2016) <i>Nature Communications</i>. doi:<a class='pub-doi' href='https://doi.org/10.1038/ncomms12846'>10.1038/ncomms12846</a></div></div></td></tr><tr><td><a href='/doi/10.1038/nbt.3603' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>9.</div></td><td><div class='doi-body' data-key='10.1038/nbt.3603'><a href='/doi/10.1038/nbt.3603' class='pub-title'>A crowdsourcing approach for reusing and meta-analyzing gene expression data</a><div class='pub-body'>Naisha Shah, Yongjian Guo, Katherine V Wendelsdorf, Yong Lu, Rachel Sparks, John S Tsang (2016) <i>Nature Biotechnology</i>. doi:<a class='pub-doi' href='https://doi.org/10.1038/nbt.3603'>10.1038/nbt.3603</a></div></div></td></tr></table>

    


	</td>
	<td class="thread-sidebar">


			

      <a class="btn btn-default follow-link btn-follow btn-lg is-following tt-top full-width" title="Follow discussion" data-id="96" data-ctype="thread" data-is-following="false">&nbsp;</a>

      
        <div class="status-box text-center"><span style="">Status:</span> 
  Completed

</div>
      

      
        
      


				

					<div class="details">Views</div>
					<span style="font-size:20px;">358</span>

        

				
					<div class="details">Topics</div>
					<div style="margin-bottom:15px;">
		            <a href="/topics/transcriptome" class="tag">Transcriptome</a><a href="/topics/gene-expression" class="tag">Gene Expression</a><a href="/topics/geo" class="tag">GEO</a><a href="/topics/stargeo" class="tag">STARGEO</a><a href="/topics/star-geo" class="tag">STAR-GEO</a><a href="/topics/gene-expression-omnibus" class="tag">Gene Expression Omnibus</a>
					</div>
				


		
			<div class="details">Referenced by</div>
			<div style="font-size:12px;"><ul style="padding-left:16px !important;">
			
				<li>
					
					<a href='/discussion/one-network-to-rule-them-all/102' class='thread-title'>One network to rule them all</a>
				</li>
			
				<li>
					
					<a href='/discussion/brainstorming-future-directions-for-hetionet/227' class='thread-title'>Brainstorming future directions for Hetionet</a>
				</li>
			
				<li>
					
					Research report: <a href='/p/rephetio/report'>Rephetio: Repurposing drugs on a hetnet</a>
				</li>
			
				<li>
					
					<a href='/discussion/the-adeptus-resource-for-expression-signatures-of-disease/101' class='thread-title'>The ADEPTUS resource for expression signatures of disease</a>
				</li>
			
				<li>
					
					<a href='/discussion/text-as-a-resource-for-network-population/48' class='thread-title'>Text as a resource for network population?</a>
				</li>
			
				</ul>
			</div>
		

		
			<div class="details">Cite this as</div>
			<span style="font-size:12px;">
				Daniel Himmelstein, Frederic Bastian, Dexter Hadley, Casey Greene (2015) STARGEO: expression signatures for disease using crowdsourced GEO annotation. <i>Thinklab.</i> doi:<a href='https://doi.org/10.15363/thinklab.d96'>10.15363/thinklab.d96</a>
			</span>
		

		<div class="details">License</div>
		<p><a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0;width:88px;height:31px;" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a></p>

		<div class="details">Share</div>
		
			
				<a class="share-button fb" onclick="javascript:void(window.open('https://www.facebook.com/sharer/sharer.php?u=https%3A//thinklab.com/discussion/stargeo-expression-signatures-for-disease-using-crowdsourced-geo-annotation/96','','width=600,height=460,menubar=no,status=no'));" href="#"></a>
<a class="share-button twitter" onclick="javascript:void(window.open('https://twitter.com/share?text=STARGEO%3A%20expression%20signatures%20for%20disease%20using%20crowdsourced%20GEO%20annotation&url=https%3A//thinklab.com/discussion/stargeo-expression-signatures-for-disease-using-crowdsourced-geo-annotation/96','','width=600,height=300,menubar=no,status=no'));" href="#"></a>
<a class="share-button google" onclick="javascript:void(window.open('https://plus.google.com/share?url=https%3A//thinklab.com/discussion/stargeo-expression-signatures-for-disease-using-crowdsourced-geo-annotation/96','','width=510,height=460,menubar=no,status=no'));" href="#"></a>

			
		

	</td>
</tr></table>


		</div>


		
	    	<div class="footer">
<div style="background:#dedede;">
	<div class="main-centered-div text-center" style="padding:20px;">
		<h4 style="margin-bottom:0px;font-weight:bold;color:gray;">Feedback? Questions? Email us: <a href="/cdn-cgi/l/email-protection#5a323f3636351a2e32333431363b3874393537"><span class="__cf_email__" data-cfemail="08606d646467487c6061666364696a266b6765">[email&#160;protected]</span><script data-cfhash='f9e31' type="text/javascript">/* <![CDATA[ */!function(t,e,r,n,c,a,p){try{t=document.currentScript||function(){for(t=document.getElementsByTagName('script'),e=t.length;e--;)if(t[e].getAttribute('data-cfhash'))return t[e]}();if(t&&(c=t.previousSibling)){p=t.parentNode;if(a=c.getAttribute('data-cfemail')){for(e='',r='0x'+a.substr(0,2)|0,n=2;a.length-n;n+=2)e+='%'+('0'+('0x'+a.substr(n,2)^r).toString(16)).slice(-2);p.replaceChild(document.createTextNode(decodeURIComponent(e)),c)}p.removeChild(t)}}catch(u){}}()/* ]]> */</script></a></h4>
	</div>
</div>
<div class="main-footer">
	<div class="main-centered-div">

<table class="full-width"><tr>
	<td class="footer-nav">

        <div class="foot-head">Explore</div>
        <a href="/proposals">Proposals</a><br>
        <a href="/projects">Projects</a><br>
        <a href="/publications">Article discussion</a><br>
        <a href="/discussion">All discussion</a><br>
        <a href="/topics">Topics</a><br>
        <!-- <a href="/funders">Funders</a><br> -->
        <a href="/funding">Funding opportunities</a><br>

	</td>
	<td class="footer-nav">

        <!-- <div class="foot-head">Help &amp; feedback</div>	 -->
        <!-- <div class="foot-head">Intro</div>
        <a href="/benefits/for-science">Why</a><br>
        <a href="/how-it-works">How it works</a><br>

<br> -->

        <div class="foot-head">Community</div>
        <a href="/leaderboard">Leaderboard</a><br>
        <a href="/p/meta">Thinklab Meta</a><br>
				<a href="/help/faq">Help</a><br>
				<!-- <a href="/new-project">Start a proposal</a><br> -->
        <!-- <a href="/benefits/proposal">Benefits</a><br> -->
        <!-- <a href="/funding">Get funded</a><br> -->
        <!-- <a href="/participate">Become a contributor</a><br> -->
        <!-- <a href="/for-funders">For science funders</a><br> -->
        <!-- <a href="/for-institutions">For institutions</a><br> -->

	</td>
	<td class="footer-nav">



        <div class="foot-head">About</div>
        <a href="/about">Overview</a><br>
        <a href="/benefits">Benefits</a><br>
        <a href="/how-it-works">How it works</a><br>
        <a href="/about/story">Our story</a><br>
        <a href="/blog">Our blog</a><br>
        <a href="/about/jobs">Join the team</a><br>
        <br>

        <!-- <div class="foot-head">Help</div>
        <a href="/help">Help</a><br>
		 -->


	</td>
	<td class="footer-nav">
		<a href="http://twitter.com/thinklab" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/twitter_com.png" width="40"/></a>
		<a href="https://www.facebook.com/pages/Thinklab/310895039088185" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/facebook_com.png" width="40"/></a>
		<a href="https://plus.google.com/115578983466800753402/" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/plus_google_com.png" width="40"/></a>
		<a href="https://www.linkedin.com/company/thinklab-co" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/linkedin_com.png" width="40"/></a>
		<br><br>
		<span class="light" style='font-size:12px;'>&copy; 2015 Thinklab</span>

	</td>

</tr></table>

</div>
</div>
</div>

		

	</div>


  <script></script>
  <script>
  var MEDIA_URL = 'https://think-lab.s3.amazonaws.com/m/';
  var STATIC_URL = 'https://think-lab.s3.amazonaws.com/s/';
  var page_is_authenticated = false;
  var page_page = 'discussion';
  var page_profile_id = undefined;
</script>



	<!-- Note different script type. MUST come before MathJax.js -->
	<script type="text/x-mathjax-config">
	    MathJax.Hub.Config({
	        "HTML-CSS": { preferredFont: "TeX", availableFonts: ["STIX","TeX"],
	                        scale: 110, linebreaks: { automatic:true },
	                        EqnChunk: (MathJax.Hub.Browser.isMobile ? 10 : 50) },
	        tex2jax: { inlineMath: [ ["$$", "$$"], ["\\(","\\)"] ],
	                    displayMath: [ ["$$$","$$$"], ["\\[", "\\]"] ],
	                    processEscapes: true,}
	            });

	</script>
    <script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>





  <script src="https://code.jquery.com/jquery-1.10.2.min.js"></script>
  <script src="https://code.jquery.com/ui/1.11.4/jquery-ui.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" integrity="sha384-0mSbJDEHialfmuBBQP6A4Qrprq5OVfW37PRR3j5ELqxss1yVqOtnepnHVP9aJ7xS" crossorigin="anonymous"></script>
  <script src="https://think-lab.s3.amazonaws.com/s/dist/main.min.js"></script>
  



  
	
  
  
  
<script>
$(function() {
  recordHit('thread', 96);
});
</script>



<script>/* <![CDATA[ */(function(d,s,a,i,j,r,l,m,t){try{l=d.getElementsByTagName('a');t=d.createElement('textarea');for(i=0;l.length-i;i++){try{a=l[i].href;s=a.indexOf('/cdn-cgi/l/email-protection');m=a.length;if(a&&s>-1&&m>28){j=28+s;s='';if(j<m){r='0x'+a.substr(j,2)|0;for(j+=2;j<m&&a.charAt(j)!='X';j+=2)s+='%'+('0'+('0x'+a.substr(j,2)^r).toString(16)).slice(-2);j++;s=decodeURIComponent(s)+a.substr(j,m-j)}t.innerHTML=s.replace(/</g,'&lt;').replace(/\>/g,'&gt;');l[i].href='mailto:'+t.value}}catch(e){}}}catch(e){}})(document);/* ]]> */</script></body>
</html>
