<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8" /><script type="text/javascript">window.NREUM||(NREUM={}),__nr_require=function(e,n,t){function r(t){if(!n[t]){var o=n[t]={exports:{}};e[t][0].call(o.exports,function(n){var o=e[t][1][n];return r(o||n)},o,o.exports)}return n[t].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<t.length;o++)r(t[o]);return r}({1:[function(e,n,t){function r(){}function o(e,n,t){return function(){return i(e,[c.now()].concat(u(arguments)),n?null:this,t),n?void 0:this}}var i=e("handle"),a=e(2),u=e(3),f=e("ee").get("tracer"),c=e("loader"),s=NREUM;"undefined"==typeof window.newrelic&&(newrelic=s);var p=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],d="api-",l=d+"ixn-";a(p,function(e,n){s[n]=o(d+n,!0,"api")}),s.addPageAction=o(d+"addPageAction",!0),s.setCurrentRouteName=o(d+"routeName",!0),n.exports=newrelic,s.interaction=function(){return(new r).get()};var m=r.prototype={createTracer:function(e,n){var t={},r=this,o="function"==typeof n;return i(l+"tracer",[c.now(),e,t],r),function(){if(f.emit((o?"":"no-")+"fn-start",[c.now(),r,o],t),o)try{return n.apply(this,arguments)}finally{f.emit("fn-end",[c.now()],t)}}}};a("setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(e,n){m[n]=o(l+n)}),newrelic.noticeError=function(e){"string"==typeof e&&(e=new Error(e)),i("err",[e,c.now()])}},{}],2:[function(e,n,t){function r(e,n){var t=[],r="",i=0;for(r in e)o.call(e,r)&&(t[i]=n(r,e[r]),i+=1);return t}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],3:[function(e,n,t){function r(e,n,t){n||(n=0),"undefined"==typeof t&&(t=e?e.length:0);for(var r=-1,o=t-n||0,i=Array(o<0?0:o);++r<o;)i[r]=e[n+r];return i}n.exports=r},{}],4:[function(e,n,t){n.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(e,n,t){function r(){}function o(e){function n(e){return e&&e instanceof r?e:e?f(e,u,i):i()}function t(t,r,o,i){if(!d.aborted||i){e&&e(t,r,o);for(var a=n(o),u=m(t),f=u.length,c=0;c<f;c++)u[c].apply(a,r);var p=s[y[t]];return p&&p.push([b,t,r,a]),a}}function l(e,n){v[e]=m(e).concat(n)}function m(e){return v[e]||[]}function w(e){return p[e]=p[e]||o(t)}function g(e,n){c(e,function(e,t){n=n||"feature",y[t]=n,n in s||(s[n]=[])})}var v={},y={},b={on:l,emit:t,get:w,listeners:m,context:n,buffer:g,abort:a,aborted:!1};return b}function i(){return new r}function a(){(s.api||s.feature)&&(d.aborted=!0,s=d.backlog={})}var u="nr@context",f=e("gos"),c=e(2),s={},p={},d=n.exports=o();d.backlog=s},{}],gos:[function(e,n,t){function r(e,n,t){if(o.call(e,n))return e[n];var r=t();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(e,n,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return e[n]=r,r}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],handle:[function(e,n,t){function r(e,n,t,r){o.buffer([e],r),o.emit(e,n,t)}var o=e("ee").get("handle");n.exports=r,r.ee=o},{}],id:[function(e,n,t){function r(e){var n=typeof e;return!e||"object"!==n&&"function"!==n?-1:e===window?0:a(e,i,function(){return o++})}var o=1,i="nr@id",a=e("gos");n.exports=r},{}],loader:[function(e,n,t){function r(){if(!x++){var e=h.info=NREUM.info,n=d.getElementsByTagName("script")[0];if(setTimeout(s.abort,3e4),!(e&&e.licenseKey&&e.applicationID&&n))return s.abort();c(y,function(n,t){e[n]||(e[n]=t)}),f("mark",["onload",a()+h.offset],null,"api");var t=d.createElement("script");t.src="https://"+e.agent,n.parentNode.insertBefore(t,n)}}function o(){"complete"===d.readyState&&i()}function i(){f("mark",["domContent",a()+h.offset],null,"api")}function a(){return E.exists&&performance.now?Math.round(performance.now()):(u=Math.max((new Date).getTime(),u))-h.offset}var u=(new Date).getTime(),f=e("handle"),c=e(2),s=e("ee"),p=window,d=p.document,l="addEventListener",m="attachEvent",w=p.XMLHttpRequest,g=w&&w.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:w,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var v=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-1044.min.js"},b=w&&g&&g[l]&&!/CriOS/.test(navigator.userAgent),h=n.exports={offset:u,now:a,origin:v,features:{},xhrWrappable:b};e(1),d[l]?(d[l]("DOMContentLoaded",i,!1),p[l]("load",r,!1)):(d[m]("onreadystatechange",o),p[m]("onload",r)),f("mark",["firstbyte",u],null,"api");var x=0,E=e(4)},{}]},{},["loader"]);</script><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info={"beacon":"bam.nr-data.net","queueTime":0,"licenseKey":"cd1101aa94","agent":"","transactionName":"bgcDMkBZW0NZBkdcWVdNJxNcW0FZVwscV1dKB08QW11CQxYTWlBBZgEOC19dW0QCEVtHU1gG","applicationID":"6369297","errorBeacon":"bam.nr-data.net","applicationTime":434}</script>
    <title>Computing consensus transcriptional profiles for LINCS L1000 perturbations | Thinklab</title>
	  <link rel="shortcut icon" href="https://think-lab.s3.amazonaws.com/s/img/favicon.ico" type="image/x-icon" />
    <meta property="og:image" content="https://think-lab.s3.amazonaws.com/s/img" />
    
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
  <link rel="stylesheet" href="https://code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css" />

<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/highlightjs/github.css" />
<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/css/css.css" />
<link rel="stylesheet" href="https://think-lab.s3.amazonaws.com/s/css/bootstrap-multiselect.css" />
<!-- <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'> -->

    
    
    
    
  
    <meta name="citation_doi" content="10.15363/thinklab.d43">
    <meta name="citation_title" content="Computing consensus transcriptional profiles for LINCS L1000 perturbations">
    <meta name="citation_online_date" content="March 26, 2015">
    <meta name="citation_author" content="Daniel Himmelstein, Caty Chung">
    <meta name="citation_journal_title" content="Thinklab">
    <meta name="citation_publisher" content="Thinklab">
  

    


	<!-- GoogleAnalytics -->
	<script>
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
	(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
	m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

	ga('create', 'UA-48550230-1', 'thinklab.org');
	ga('require', 'displayfeatures');
	
	ga('send', 'pageview');

	</script>






</head>
<body >
  <form id="form_action" method="post">
    <input type='hidden' name='csrfmiddlewaretoken' value='JC3hnVSWzcwBOS226QTq2eGyHXj1YAM6' />
</form>

	

	<div class="body-wrapper">

		


<!-- <div style="height:5px;background-image:url('https://think-lab.s3.amazonaws.com/s/img/top-bg.gif');"></div> -->
<div class="gradient">
	<div class="main-centered-div header-container">

<table class="full-width" style="height:60px;"><tr>
	<td style="width:200px;vertical-align:middle;padding-bottom:15px;">
		<a href="/" style="font-weight:normal;color:#212121;"><img src="https://think-lab.s3.amazonaws.com/s/img/logo.png" width="160"></a>
	</td><td style="vertical-align:middle;text-align:left;padding-top:6px;width:100%;padding-left:40px;">
		<ul id="not-search-block"  class="main-header">


			


				<li class="dropdown">
					<a class="dropdown-toggle" data-toggle="dropdown" href="#">Browse <span class="caret"></span></a>
					<ul class="dropdown-menu" role="menu">
						<li><a href="/proposals">Proposals</a></li>
						<li><a href="/projects">Projects</a></li>
						<li><a href="/publications">Article discussion</a></li>
						<li><a href="/discussion">All discussion</a></li>
					</ul>
				</li>
				<li><a href="/leaderboard">Leaderboard</a></li>

			
				<li><a href="/about">About</a></li>
			


		</ul>

    <ul id="search-block" class="main-header" style="display: none; margin-right: 30px;">
        <li class="full-width">
            <form method="get" action="/search">
			<div class="input-group full-width">
				<input name="q" type="text" class="form-control search-input">
			</div>
            </form>
        </li>
    </ul>

	</td><td style="vertical-align:middle;text-align:right;padding-top:6px; min-width: 1%; white-space: nowrap;">
		<ul class="main-header pull-right" style="padding-left: 0;">
    		
                <!-- <li><a href="/login" style="color:silver;">Login</a></li>
                <li style="margin-right:0"><a class="btn btn-default btn-sm btn-upper" href="/signup">Join</a></li> -->
            

        </ul>
    </ul>
	</td>
</tr></table>

<!-- <div id="global-search-div" class="main-centered-div" style="display:none;">
	<input type="text" value="" class="thread-search form-control" placeholder="Search" data-action="/discussion">
</div> -->

	</div>
</div>

		
	  



	  


		

		<div class="dark-head"><div class="main-centered-div">

			

		</div></div>
		
		<div class="main-centered-div" style="margin-top:40px;padding-bottom:340px">
	    

<div style="margin-top:-20px; margin-bottom:20px;">

    <a class="side-box" href="/p/rephetio" style="background-color:#ebfdff;"><div class="side-type">Project:</div> <div class="side-title">Rephetio: Repurposing drugs on a hetnet [rephetio]</div></a><br>



</div>

<table style="margin-top:0px;"><tr>
	<td class="thread-table-content">

		<table class="full-width">
			
<tr class="comment-tr target-hl" id="1" data-comment-id="93">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="93"></a>

    

			<h2 class="page-title" style="margin-bottom:5px;margin-top:0px !important;">Computing consensus transcriptional profiles for LINCS L1000 perturbations</h2>

			<div style="margin-top:5px;color:gray;margin-bottom:20px;font-size:13px;">
				<a style="font-weight:normal;color:#212121" href="/u/dhimmel"><strong>Daniel Himmelstein</strong></a>
				<span class="is-researcher">Researcher</span>
				&nbsp; <a href="#1" class="light">March 27, 2015</a>
				
			</div>

      

          <hr>
		

    <p><a href="//www.lincscloud.org/">LINCS</a> (Library of Integrated Cellular Signatures) provided "perturbational profiles across multiple cell and perturbation types, as well as read-outs, at a massive scale." We plan to compute transcriptional profiles, i.e. expression signatures — sets of up and down-regulated genes — for the compounds in our network. We will be following a workflow suggested to us by Ted Natoli during the <a href="//www.lincscloud.org/training/">online office hours</a>.</p>

<p>For a given compound in our network, there may be multiple matched LINCS compounds. Additionally, each LINCS compound may have been assayed across multiple cell lines, dosages, and replicates. To calculate a single consensus transcriptional profile across multiple signatures we will</p>

<ol><li>calculate pairwise correlations between signatures</li><li>calculate mean correlation with other signatures for each signature</li><li>scale similarities to sum to 1</li><li>multiply z-score signature vectors by their similarity weights</li><li>sum weighted z-score signature vectors</li></ol>

<p>The z-score signature vectors are retrieved from the <code>/xchip/cogs/data/build/a2y13q1/modzs.gctx</code> file on the <a href="http://c3.lincscloud.org/">C3 cloud</a>.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
<li id="note-69" class="inline-note toggle-parent" data-note-id="69">
	<p><a href="/u/dhimmel">Daniel Himmelstein</a>: Saw this paper <span class="citation">[<a href="https://doi.org/10.1371/journal.pcbi.1004068" class="citation" data-key="10.1371/journal.pcbi.1004068">1</a>]</span> which explores when disease/compound expression profiles are reliable therapeutic indicators, specifically with regards to lung cancer.
		
	</p>
	
	<table style="display:none;"><tr><td><a href='/doi/10.1371/journal.pcbi.1004068' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>1.</div></td><td><div class='doi-body' data-key='10.1371/journal.pcbi.1004068'><a href='/doi/10.1371/journal.pcbi.1004068' class='pub-title'>Prioritizing Therapeutics for Lung Cancer: An Integrative Meta-analysis of Cancer Gene Signatures and Chemogenomic Data</a><div class='pub-body'>Kristen Fortney, Joshua Griesman, Max Kotlyar, Chiara Pastrello, Marc Angeli, Ming Sound-Tsao, Igor Jurisica (2015) <i>PLOS Computational Biology</i>. doi:<a class='pub-doi' href='https://doi.org/10.1371/journal.pcbi.1004068'>10.1371/journal.pcbi.1004068</a></div></div></td></tr></table>
</li>

				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="2" data-comment-id="139">
  <td class="comment-profile-avatar">
		<a href="/u/cchung"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/s/img/p_default_medium.png' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="139"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/cchung">Caty Chung</a>
				
				<span class="light">&nbsp;<a href="#2" class="light">April 15, 2015</a>
          
        </span>
      </div>

    

    <p>The <a href="https://sites.google.com/site/bd2klincsdatascience/">BD2K LINCS Data Science Research webinars</a> serve as a general forum to engage data scientists within and outside of LINCS project to work on problems related to LINCS data analysis and integration. </p>

<p>The <a href="http://lincs-dcic.org/">BD2K-LINCS DCIC</a> engages the research community by delivering high quality educational materials through the web as well as through mentoring, seminars and symposia. Also, Center investigators actively engage in the education of a new generation of Big Data Scientists by developing a graduate-level Big Data Science MOOC that will be delivered to graduate students in Big Data Biostatistics and other Biomedical Informatics graduate programs.</p>

<p>See <a href="http://lincs-dcic.org/#/training">BD2K-LINCS DCIC Training and Outreach</a>.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="3" data-comment-id="142">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="142"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#3" class="light">April 16, 2015</a>
          
        </span>
      </div>

    

    <p><a href="/u/leobrueggeman" class="username">@leobrueggeman</a> and I attended the online LINCS office hours today. We spoke primarily with Ted who provided some helpful insights and suggestions.</p>

<ol><li><p><strong>Spearman's correlation</strong> is preferred to the Pearson's correlation when calculating correlations between transcriptional profiles. Spearman's correlation is rank based and therefore less prone to excessive influence of extremes. We will switch from Pearson's to Spearman's correlation in step 2 <a href="#">above</a>.</p></li><li><p><strong>Gold signatures</strong> — Multiple replicates (often 3) are performed for each signature. Higher quality signatures are considered gold (<code>is_gold</code>) based on a heuristic that values reproducibility across replicates and distinctness of that signature. They require the aggregate correlation within replicates to exceed a threshold among other criteria. Around half of the dataset is gold. The informativeness of nongold signatures is dubious, thus Ted suggests restricting to gold signatures.</p></li><li><p><strong>Z-score threshold</strong> — when setting a DEG (deferentially expressed gene) threshold, the LINCS team uses <code>&gt; 2.0</code> or <code>&lt; -2.0</code> and is pleased with the results.</p></li><li><p><strong>Signature number bias in DEG counts</strong> — We found that some perturbagens had an extremely high number of DEGs. We speculate that this occurs for perturbagens with few signatures, since with many signatures across varying contexts, a large set of consistently dysregulated genes seems unlikely. Two possible corrections for this bias are: a) choose a constant number of signatures to include per perturbagen. However, this would lead to information loss, which is undesirable. b) develop an empirical method for adjusting for expected DEG numbers. We will explore option (b) in a future post.</p></li><li><p><strong>Best inferred gene set</strong> — (<code>is_bing</code>) refers to genes that are reliably imputed from the 1000 assayed genes (L1000). We plan to switch to only included the ~7,500 bing genes in our DEG analysis because we would like to avoid unreliable data. This may also help reduce the excess of DEG for certain perturbagens. Whether a gene is part of the bing subset can be retrieved through the <code>geneinfo</code> <a href="http://api.lincscloud.org/a2/docs/geneinfo">API query</a>. </p></li></ol>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="4" data-comment-id="203">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="203"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#4" class="light">May 7, 2015</a>
          
        </span>
      </div>

    

    <p>We talked to Dave and Ted at the LINCS office hours today. Here are the meeting notes:</p>

<h2>Probes and genes</h2>

<p>There have been two versions of landmark genes (the genes that are measured by the L1000 platform). In the first version (<code>pr_pool_id = 'deltap'</code>), there were 979 landmark genes. In the current version (<code>pr_pool_id = 'epsilon'</code>), there are 978.</p>

<p>The L1000 platform is designed to imitate the Affymetrix HG-U133A array <span class="citation">[<a href="https://doi.org/10.1186/gb-2006-7-7-r61" class="citation" data-key="10.1186/gb-2006-7-7-r61">1</a>]</span>, so L1000 output is in probe-space. The LINCS team performs their analyses in probe-space. For landmark genes, the probe-to-gene correspondence is one-to-one. However, other genes may consist of multiple probes. We plan to average z-scores across probes to convert our observations into gene-space.</p>

<h2>Computing consensus signatures</h2>

<p>When drugs have multiple signatures, we find the average correlation value for each signature. We have noticed that some signatures have negative correlations and were thus contributing their inverse signature to the consensus. We are uncomfortable with negative weights and therefore plan to set a minimum correlation threshold for each signature. A minimum of <code>0</code> would exclude all negatively correlated signatures. Another option is <code>0.1</code>, which is used by the LINCS team when processing shRNA data.</p>

<h1>Miscellaneous</h1>

<ul><li>The API returns <code>-666</code> for missing values.</li><li>Instances refer to the replicates that compose a signature.</li><li><a href="/u/leobrueggeman" class="username">@leobrueggeman</a>, will be presenting on LINCS at lab meeting today (<a href="http://slides.com/leoo/lincs">presentation</a>).</li><li><code>is_summly</code> refers to whether a perturbagen has been profiled across a broad range of cell lines.</li></ul>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="5" data-comment-id="231">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="231"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#5" class="light">May 21, 2015</a>
          
        </span>
      </div>

    

    <h1>Combining z-scores across multiple signatures</h1>

<p>To create consensus signatures for a compound, we have been taking a weighted average of z-scores (steps 4–5 <a href="#3">above</a>). It has occurred to us that this is an underpowered method, just as averaging p-values is a weak method of meta-analysis. </p>

<p>Instead we can use <a href="https://en.wikipedia.org/wiki/Fisher%27s_method#Relation_to_Stouffer.27s_Z-score_method">Stouffer's method</a> to meta-analyze z-scores <span class="citation">[<a href="http://press.princeton.edu/titles/2692.html" class="citation" data-key="stouffer">1</a>]</span>. This method accepts weights (calculated in steps 2–3 <a href="#3">above</a>). The formula is below for weight vector <em>w</em> and z-score vector <em>Z</em>:</p>

<div class="math">$$$
Z \sim \frac{\sum_{i=1}^k w_iZ_i}{\sqrt{\sum_{i=1}^k w_i^2}}
$$$</div>



		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="6" data-comment-id="363">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="363"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#6" class="light">Aug. 8, 2015</a>
          
        </span>
      </div>

    

    <h1>Initial release of consensus signatures</h1>

<p>We have computed consensus transcriptional signatures for LINCS L1000 perturbations. We have released <a href="https://github.com/dhimmel/lincs/tree/b36dfd82b6b1aa0b0c45ec905cb2548ddf7dc53e/data/consensi">datasets</a> <span class="citation">[<a href="https://doi.org/10.5281/zenodo.27229" class="citation" data-key="10.5281/zenodo.27229">1</a>]</span> for the following pertubation sets:</p>

<ul><li>LINCS pert_ids: 38,327 consensi (<a href="https://github.com/dhimmel/lincs/blob/b36dfd82b6b1aa0b0c45ec905cb2548ddf7dc53e/data/consensi/consensi-pert_id.tsv.gz">download</a>)</li><li>DrugBank compounds: 1,170 consensi (<a href="https://github.com/dhimmel/lincs/blob/b36dfd82b6b1aa0b0c45ec905cb2548ddf7dc53e/data/consensi/consensi-drugbank.tsv.gz">download</a>)</li><li>Gene knockdowns: 4,363 consensi (<a href="https://github.com/dhimmel/lincs/blob/b36dfd82b6b1aa0b0c45ec905cb2548ddf7dc53e/data/consensi/consensi-knockdown.tsv.gz">download</a>)</li><li>Gene over-expressions: 2,471 consensi (<a href="https://github.com/dhimmel/lincs/blob/b36dfd82b6b1aa0b0c45ec905cb2548ddf7dc53e/data/consensi/consensi-overexpression.tsv.gz">download</a>)</li></ul>

<p>The datasets are tsv-formatted with perturbations as rows and genes as columns. We only report expression values for the 978 assayed genes. Non-gold signatures were omitted. We set a <a href="#4">minimum signature weight</a> of 0.05 and combined z-scores using <a href="#5">Stouffer's method</a>.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

<tr class="comment-tr target-hl" id="7" data-comment-id="1150">
  <td class="comment-profile-avatar">
		<a href="/u/dhimmel"><img class='profile_image photo-medium' src='https://think-lab.s3.amazonaws.com/m/profile/17_medium.jpg' width='60' height='60'></a>
  </td>
	<td class="comment-body">
    <a name="1150"></a>

    

      <div class="comment-header">
        <a style="font-weight:bold;color:#212121" href="/u/dhimmel">Daniel Himmelstein</a>
				<span class="is-researcher">Researcher</span>
				<span class="light">&nbsp;<a href="#7" class="light">March 7, 2016</a>
          
        </span>
      </div>

    

    <h1>Concensus signatures version 2.0</h1>

<p>We've released <code>v2.0</code> of our analysis of LINCS L1000. This release brings major updates including:</p>

<ul><li><strong>Inferred genes</strong>. <a href="#6">Previously</a>, dysregulation scores were only reported for the 978 landmark (directly measured) genes. Now we've expanded our analysis to include 6,489 imputed genes from the best inferred gene set (<code>is_bing</code>), which covers genes imputed with high accuracy. However, we've maintained backwards compatibility by only using landmark probes for signature weighting.</li><li><strong>Significantly dysregulated genes</strong>. We now report significantly down or upregulated perturbagen–gene pairs.</li><li><strong>Improved knockdown and overexpression pipeline</strong>. We now convert gene symbols to entrez genes at the earliest stage. Now two genetic perturbations with different symbols that map to the same entrez gene will benefit from <a href="#5"><em>z</em>-score meta-analysis</a>.</li></ul>

<h2>Consensus signature datasets</h2>

<p>Our consensus signatures are available on <a href="https://github.com/dhimmel/lincs/tree/abcb12f942f93e3ee839e5e3593f930df2c56845/data/consensi">GitHub</a> <span class="citation">[<a href="https://doi.org/10.5281/zenodo.47223" class="citation" data-key="10.5281/zenodo.47223">1</a>]</span> and <a href="https://doi.org/10.6084/m9.figshare.3085426" title="Consensus signatures for LINCS L1000 perturbations · figshare data deposition">figshare</a> <span class="citation">[<a href="https://doi.org/10.6084/m9.figshare.3085426" class="citation" data-key="10.6084/m9.figshare.3085426">2</a>]</span>. Each consensus signature measures the transcriptional response to a perturbation at <a href="https://github.com/dhimmel/lincs/blob/abcb12f942f93e3ee839e5e3593f930df2c56845/data/consensi/genes.tsv">7,467 genes</a>. Genes are identified by their Entrez GeneID. Consensi are produced for:</p>

<ul><li><strong>DrugBank compounds</strong> — 1,170 small molecule compounds identified by their DrugBank ID. L1000 compounds were mapped to DrugBank <a href="http://thinklab.com/discussion/unichem-mapping-to-lincs-small-molecules/51#8" class="citation thread-citation" data-key="10.15363/thinklab.d51">using atomic connectivity</a>.</li><li><strong>Gene knockdowns</strong> — 4,326 knocked-down genes identified by their Entrez GeneID.</li><li><strong>Gene overexpressions</strong> — 2,413 overexpressed genes identified by their Entrez GeneID</li><li><strong>All L1000 pertubations</strong> — 38,327 perturbagens identified by their L1000 <code>pert_id</code>.</li></ul>

<h2>Methodology</h2>

<p>To recap our methodology, our objective is to compute a consensus signature from multiple input signatures. Each input signature measures the dysregulation caused by a specific perturbation and condition. The purpose of computing consensi is to combine signatures for the same perturbation under different conditions (for examples different dosages, cell types, or time points). </p>

<p>Our method, developed in consultation with the L1000 team, arrives at a consensus signature from a set of input signatures by:</p>

<p>A) Starting with a probe × signature matrix of dysregulation <em>z</em>-scores with the following filters:</p>

<ul><li><strong>Initial probe filter</strong>: include all landmark or <code>is_bing</code> probes with the following exclusions: a) inferred probes for genes with a landmark probe. b) probes with non-existent Entrez GeneIDs.</li><li><strong>Initial signature filter</strong>: use only <a href="#3">gold signatures</a> to remove non-replicating or indistinct signatures</li></ul>

<p>B) Then a gene-level consensus signature is computed by:</p>

<ol><li>Calculating an input signature weight. Each input signature gets a weight equal to its average Spearman's correlation with other input signatures. We set a minimum correlation value of 0.05 to ensure all signatures make at least a small contribution and to prevent negative weights. Weights are computed using only landmark probes.</li><li><a href="#5">Meta-analyzing <em>z</em>-scores</a> with Stouffer's method to compute a probe-level consensus signature.</li><li>Condensing to a gene-level consensus by averaging probe <em>z</em>-scores for the same entrez gene.</li></ol>

<p>C) Finally, significant <em>Perturbagen–regulates–Gene</em> relationships are extracted for a given perturbation by:</p>

<ol><li>Converting gene <em>z</em>-scores to <em>p</em>-values by via a normal distribution</li><li>Correcting p-values for multiple testing using a Bonferroni adjustment. The correction is applied separately to measured genes and inferred genes. Hence, inferred genes are more heavily penalized for multiple testing.</li><li>Filter genes for corrected <em>p</em>-value ≤ 0.05.</li><li>Allowing at most 1000 significant inferred genes. In cases with more than 1000 significant inferred genes, filter to the 1000 smallest <em>p</em>-values.</li></ol>

<p>The methods described above are executed in the <a href="https://github.com/dhimmel/lincs/blob/abcb12f942f93e3ee839e5e3593f930df2c56845/consensi.ipynb"><code>consensi.ipynb</code></a> and <a href="https://github.com/dhimmel/lincs/blob/abcb12f942f93e3ee839e5e3593f930df2c56845/significance.ipynb"><code>significance.ipynb</code></a> notebooks.</p>

		<div style="margin-left:30px;">

      


      <ul class="comment-notes">
				
				
			</ul>

      

		</div>

		


  </td>
</tr>

    		
		</table>

    

    &nbsp;<br>
    <!-- <a class="btn btn-primary btn-lg" href="/signup?next=/discussion/computing-consensus-transcriptional-profiles-for-lincs-l1000-perturbations/43&suppress_message=1" style>Join to Reply</a> -->


		<table class="references table-td-5" style="display:none;"><tr><td><a href='/doi/10.1371/journal.pcbi.1004068' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>1.</div></td><td><div class='doi-body' data-key='10.1371/journal.pcbi.1004068'><a href='/doi/10.1371/journal.pcbi.1004068' class='pub-title'>Prioritizing Therapeutics for Lung Cancer: An Integrative Meta-analysis of Cancer Gene Signatures and Chemogenomic Data</a><div class='pub-body'>Kristen Fortney, Joshua Griesman, Max Kotlyar, Chiara Pastrello, Marc Angeli, Ming Sound-Tsao, Igor Jurisica (2015) <i>PLOS Computational Biology</i>. doi:<a class='pub-doi' href='https://doi.org/10.1371/journal.pcbi.1004068'>10.1371/journal.pcbi.1004068</a></div></div></td></tr><tr><td><a href='/doi/10.1186/gb-2006-7-7-r61' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>2.</div></td><td><div class='doi-body' data-key='10.1186/gb-2006-7-7-r61'>None</div></td></tr><tr><td></td><td><div class='pub-order'>3.</div></td><td><div class='doi-body' data-key='stouffer'><a href='http://press.princeton.edu/titles/2692.html' class='pub-title'>Stouffer SA, Suchman EA, DeVinney LC, Star SA, Williams RM. (1949) *The American Soldier, Vol.1: Adjustment during Army Life*. Princeton University Press</a></div></td></tr><tr><td><a href='/doi/10.5281/zenodo.27229' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>4.</div></td><td><div class='doi-body' data-key='10.5281/zenodo.27229'><a href='/doi/10.5281/zenodo.27229' class='pub-title'>Transcriptional signatures of perturbation from LINCS L1000</a><div class='pub-body'>Daniel Himmelstein, Leo Brueggeman, Sergio Baranzini (2015) <i>Zenodo</i>. doi:<a class='pub-doi' href='https://doi.org/10.5281/zenodo.27229'>10.5281/zenodo.27229</a></div></div></td></tr><tr><td><a href='/doi/10.5281/zenodo.47223' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>5.</div></td><td><div class='doi-body' data-key='10.5281/zenodo.47223'><a href='/doi/10.5281/zenodo.47223' class='pub-title'>dhimmel/lincs v2.0: Refined consensus signatures from LINCS L1000</a><div class='pub-body'>Daniel Himmelstein, Leo Brueggeman, Sergio Baranzini (2016) <i>Zenodo</i>. doi:<a class='pub-doi' href='https://doi.org/10.5281/zenodo.47223'>10.5281/zenodo.47223</a></div></div></td></tr><tr><td><a href='/doi/10.6084/m9.figshare.3085426' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>6.</div></td><td><div class='doi-body' data-key='10.6084/m9.figshare.3085426'><a href='/doi/10.6084/m9.figshare.3085426' class='pub-title'>Consensus signatures for LINCS L1000 perturbations</a><div class='pub-body'>Daniel Himmelstein, Leo Brueggeman, Sergio Baranzini (2016) <i>Figshare</i>. doi:<a class='pub-doi' href='https://doi.org/10.6084/m9.figshare.3085426'>10.6084/m9.figshare.3085426</a></div></div></td></tr><tr style=&#39;display:none;&#39;><td><a href='/doi/10.15363/thinklab.d51' class='pub-comments tt-right zero-items' title='Article comments'>0</a></td><td><div class='pub-order'>7.</div></td><td><div class='doi-body' data-key='10.15363/thinklab.d51'><a href='/d/51' class='pub-title'>UniChem Mapping to LINCS Small Molecules</a><div class='pub-body'>Leo Brueggeman, Daniel Himmelstein, Caty Chung (2015) <i>Thinklab</i>. doi:<a href='https://doi.org/10.15363/thinklab.d51'>10.15363/thinklab.d51</a></div></div></td></tr></table>

    


	</td>
	<td class="thread-sidebar">


			

      <!-- <a class="btn btn-default follow-link btn-follow btn-lg is-following tt-top full-width" title="Follow discussion" data-id="43" data-ctype="thread" data-is-following="false">&nbsp;</a> -->

      
        <div class="status-box text-center"><span style="">Status:</span> 
  Completed

</div>
      

      
        
      


				

					<div class="details">Views</div>
					<span style="font-size:20px;">680</span>

        

				
					<div class="details">Topics</div>
					<div style="margin-bottom:15px;">
		            <a href="/topics/expression-signatures" class="tag">Expression Signatures</a><a href="/topics/transcriptional-profiles" class="tag">Transcriptional Profiles</a><a href="/topics/lincs" class="tag">LINCS</a>
					</div>
				


		
			<div class="details">Referenced by</div>
			<div style="font-size:12px;"><ul style="padding-left:16px !important;">
			
				<li>
					
					<a href='/d/227' class='thread-title'>Brainstorming future directions for Hetionet</a>
				</li>
			
				<li>
					
					Research report: <a href='/p/rephetio/report'>Rephetio: Repurposing drugs on a hetnet</a>
				</li>
			
				<li>
					
					<a href='/d/220' class='thread-title'>Exploring the power of Hetionet: a Cypher query depot</a>
				</li>
			
				<li>
					
					<a href='/d/162' class='thread-title'>Data nomenclature: naming and abbreviating our network types</a>
				</li>
			
				<li>
					
					<a href='/d/22' class='thread-title'>Suggestions for additional information types?</a>
				</li>
			
				<li>
					
					<a href='/d/187' class='thread-title'>Estimating the complexity of hetnet traversal</a>
				</li>
			
				<li>
					
					<a href='/d/185' class='thread-title'>Assessing the imputation quality of gene expression in LINCS L1000</a>
				</li>
			
				<li>
					
					<a href='/d/171' class='thread-title'>Positive correlations between knockdown and overexpression profiles from LINCS L1000</a>
				</li>
			
				<li>
					
					<a href='/d/181' class='thread-title'>Workshop to analyze LINCS data for the Systems Pharmacology course at UCSF</a>
				</li>
			
				<li>
					
					Project: <a href='/p/rephetio'>Rephetio: Repurposing drugs on a hetnet</a>
				</li>
			
				<li>
					
					<a href='/d/110' class='thread-title'>LINCS L1000 licensing</a>
				</li>
			
				<li>
					
					<a href='/d/48' class='thread-title'>Text as a resource for network population?</a>
				</li>
			
				<li>
					
					<a href='/d/30' class='thread-title'>Assessing the quality and applicability of the SIDER 2 resource</a>
				</li>
			
				</ul>
			</div>
		

		
			<div class="details">Cite this as</div>
			<span style="font-size:12px;">
				Daniel Himmelstein, Caty Chung (2015) Computing consensus transcriptional profiles for LINCS L1000 perturbations. <i>Thinklab.</i> doi:<a href='https://doi.org/10.15363/thinklab.d43'>10.15363/thinklab.d43</a>
			</span>
		

		<div class="details">License</div>
		<p><a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0;width:88px;height:31px;" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a></p>

		<div class="details">Share</div>
		
			
				<a class="share-button fb" onclick="javascript:void(window.open('https://www.facebook.com/sharer/sharer.php?u=https%3A//thinklab.org/discussion/computing-consensus-transcriptional-profiles-for-lincs-l1000-perturbations/43','','width=600,height=460,menubar=no,status=no'));" href="#"></a>
<a class="share-button twitter" onclick="javascript:void(window.open('https://twitter.com/share?text=Computing%20consensus%20transcriptional%20profiles%20for%20LINCS%20L1000%20perturbations&url=https%3A//thinklab.org/discussion/computing-consensus-transcriptional-profiles-for-lincs-l1000-perturbations/43','','width=600,height=300,menubar=no,status=no'));" href="#"></a>
<a class="share-button google" onclick="javascript:void(window.open('https://plus.google.com/share?url=https%3A//thinklab.org/discussion/computing-consensus-transcriptional-profiles-for-lincs-l1000-perturbations/43','','width=510,height=460,menubar=no,status=no'));" href="#"></a>

			
		

	</td>
</tr></table>


		</div>


		
	    	<div class="footer">
<!-- <div style="background:#dedede;">
	<div class="main-centered-div text-center" style="padding:20px;">
		<h4 style="margin-bottom:0px;font-weight:bold;color:gray;">Feedback? Questions? Email us: <a href="mailto:hello@thinklab.com">hello@thinklab.com</a></h4>
	</div>
</div> -->
<div class="main-footer">
	<div class="main-centered-div">

<table class="full-width"><tr>
	<td class="footer-nav">

        <div class="foot-head">Explore</div>
        <a href="/proposals">Proposals</a><br>
        <a href="/projects">Projects</a><br>
        <a href="/publications">Article discussion</a><br>
        <a href="/discussion">All discussion</a><br>
        <a href="/topics">Topics</a><br>
        <!-- <a href="/funders">Funders</a><br> -->
        <a href="/funding">Funding opportunities</a><br>

	</td>
	<td class="footer-nav">

        <!-- <div class="foot-head">Help &amp; feedback</div>	 -->
        <!-- <div class="foot-head">Intro</div>
        <a href="/benefits/for-science">Why</a><br>
        <a href="/how-it-works">How it works</a><br>

<br> -->

        <div class="foot-head">Community</div>
        <a href="/leaderboard">Leaderboard</a><br>
        <a href="/p/meta">Thinklab Meta</a><br>
				<a href="/help/faq">Help</a><br>
				<!-- <a href="/new-project">Start a proposal</a><br> -->
        <!-- <a href="/benefits/proposal">Benefits</a><br> -->
        <!-- <a href="/funding">Get funded</a><br> -->
        <!-- <a href="/participate">Become a contributor</a><br> -->
        <!-- <a href="/for-funders">For science funders</a><br> -->
        <!-- <a href="/for-institutions">For institutions</a><br> -->

	</td>
	<td class="footer-nav">



        <div class="foot-head">About</div>
        <a href="/about">Overview</a><br>
        <a href="/benefits">Benefits</a><br>
        <a href="/how-it-works">How it works</a><br>
        <a href="/about/story">Our story</a><br>
        <a href="/blog">Our blog</a><br>
        <a href="/about/jobs">Join the team</a><br>
        <br>

        <!-- <div class="foot-head">Help</div>
        <a href="/help">Help</a><br>
		 -->


	</td>
	<td class="footer-nav">
		<!-- <a href="http://twitter.com/thinklab" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/twitter_com.png" width="40"/></a>
		<a href="https://www.facebook.com/pages/Thinklab/310895039088185" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/facebook_com.png" width="40"/></a>
		<a href="https://plus.google.com/115578983466800753402/" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/plus_google_com.png" width="40"/></a>
		<a href="https://www.linkedin.com/company/thinklab-co" class="social-icon"><img src="https://think-lab.s3.amazonaws.com/s/img/icons/linkedin_com.png" width="40"/></a> -->
		<br><br>
		<span class="light" style='font-size:12px;'>&copy; 2015 Thinklab</span>

	</td>

</tr></table>

</div>
</div>
</div>

		

	</div>


  <script></script>
  <script>
  var MEDIA_URL = 'https://think-lab.s3.amazonaws.com/m/';
  var STATIC_URL = 'https://think-lab.s3.amazonaws.com/s/';
  var page_is_authenticated = false;
  var page_page = 'discussion';
  var page_profile_id = undefined;
</script>



	<!-- Note different script type. MUST come before MathJax.js -->
	<script type="text/x-mathjax-config">
	    MathJax.Hub.Config({
	        "HTML-CSS": { preferredFont: "TeX", availableFonts: ["STIX","TeX"],
	                        scale: 110, linebreaks: { automatic:true },
	                        EqnChunk: (MathJax.Hub.Browser.isMobile ? 10 : 50) },
	        tex2jax: { inlineMath: [ ["$$", "$$"], ["\\(","\\)"] ],
	                    displayMath: [ ["$$$","$$$"], ["\\[", "\\]"] ],
	                    processEscapes: true,}
	            });

	</script>
    <script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>





  <script src="https://code.jquery.com/jquery-1.10.2.min.js"></script>
  <script src="https://code.jquery.com/ui/1.11.4/jquery-ui.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" integrity="sha384-0mSbJDEHialfmuBBQP6A4Qrprq5OVfW37PRR3j5ELqxss1yVqOtnepnHVP9aJ7xS" crossorigin="anonymous"></script>
  <script src="https://think-lab.s3.amazonaws.com/s/dist/main.min.js"></script>
  



  
	
  
  
  
<script>
$(function() {
  recordHit('thread', 43);
});
</script>



</body>
</html>
